{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import requests\n",
    "import json\n",
    "import re"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pprint"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Obtaining/Requesting .xml Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "Entrez.email = 'dxiang@ucsd.edu'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get a list of ids from search with a term that should be contained in the abstract\n",
    "\n",
    "def search_from_pmc(query):\n",
    "    '''\n",
    "    The function takes a term that needed to be contained in the abstract, and searched for the ids from pubmed central\n",
    "\n",
    "    Returns: the ids from the query.\n",
    "\n",
    "    Default query: synthetic biology ethics\n",
    "    '''\n",
    "    search_query = query\n",
    "    handle = Entrez.esearch(db='pmc',\n",
    "                            sort='relevance',\n",
    "                            retmax='100000',\n",
    "                            term=search_query,\n",
    "                            usehistory='y')\n",
    "    results = Entrez.read(handle)\n",
    "    ids = results['IdList']\n",
    "    return ids, query\n",
    "\n",
    "def write_to_folder(ids, term):\n",
    "    '''\n",
    "    The funtion writes the output from the search to xml files.\n",
    "\n",
    "    input: output directory\n",
    "    ids: ids of papers that need to be fetched.\n",
    "    term: abstract term contained.\n",
    "    '''\n",
    "    for i in ids:\n",
    "        handle = Entrez.efetch(db='pmc', id=i, rettype='full',retmode='xml')\n",
    "        with open('../PMC data/{term}/pmc{ids}.xml'.format(term = term, ids = i), 'w') as f:\n",
    "            f.write(handle.read().decode('UTF-8'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# uncomment when needed\n",
    "# for ids in results['IdList']:\n",
    "#     handle = Entrez.efetch(db='pmc', id=ids, rettype='full',retmode='xml')\n",
    "#     with open('../PMC data/ethics/pmc{}.xml'.format(ids), 'w') as f:\n",
    "#         f.write(handle.read())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# for t in ['ethics', 'dilemma', 'safe', 'security']:\n",
    "#     ids,term = search_from_pmc(t)\n",
    "#     write_to_folder(ids=ids, term=term)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Request directly using BioC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "def parse_json(j_file):\n",
    "    \n",
    "    \n",
    "    body = ''\n",
    "    for t in j_file['documents'][0]['passages']:\n",
    "        if t['infons']['type'] == 'paragraph':\n",
    "            body += t['text'] + '\\n'\n",
    "            \n",
    "    title = ''\n",
    "    for t in j_file['documents'][0]['passages']:\n",
    "        if t['infons']['section_type'] == 'TITLE':\n",
    "            title = t['text']\n",
    "            break\n",
    "                \n",
    "    keywords = ''\n",
    "    for t in j_file['documents'][0]['passages']:\n",
    "        if t['infons']['section_type'] == 'TITLE':\n",
    "            keywords = t['infons']['kwd']\n",
    "            break\n",
    "            \n",
    "    abstract = ''\n",
    "    for t in j_file['documents'][0]['passages']:\n",
    "        if t['infons']['section_type'] == 'ABSTRACT':\n",
    "            abstract += t['text'] + '\\n'\n",
    "            \n",
    "    pmc_id = j_file['documents'][0]['passages'][0]['infons']['article-id_pmc']\n",
    "    \n",
    "    def split_names(name):\n",
    "        last_name = name.split(';')[0].split(':')[1]\n",
    "        first_name = name.split(';')[1].split(':')[1]\n",
    "        return first_name + ' ' + last_name\n",
    "    \n",
    "    temp = j_file['documents'][0]['passages'][0]['infons']\n",
    "    authors = [v for k, v in temp.items() if k.startswith('name')]\n",
    "    authors = list(map(split_names, authors))\n",
    "    \n",
    "    output = {\n",
    "            \"title\": title,\n",
    "            \"pmc_id\": pmc_id,\n",
    "            \"authors\": authors,\n",
    "            \"abstract\": abstract,\n",
    "            \"key-words\": keywords,\n",
    "            \"body\": body\n",
    "        }\n",
    "    return output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "# asbtract contain ethics\n",
    "ids, query = search_from_pmc('(((\"synthetic biology\"[MeSH Terms] OR (\"synthetic\"[All Fields] AND \"biology\"[All Fields]) OR \"synthetic biology\"[All Fields]) AND (\"ethics\"[Subheading] OR \"ethics\"[All Fields] OR \"ethics\"[MeSH Terms])) AND ethics[Abstract]) AND synthetic biology[Abstract] AND \"open access\"[filter]')\n",
    "\n",
    "for i in ids:\n",
    "    r = requests.get('https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/PMC{}/unicode'.format(i))\n",
    "    with open('../PMC data/full text/pmc{}.json'.format(i), 'w') as f:\n",
    "        j_file = parse_json(r.json())\n",
    "        json.dump(j_file,f, indent=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "# abstract contain safe\n",
    "ids, query = search_from_pmc('(((\"synthetic biology\"[MeSH Terms] OR (\"synthetic\"[All Fields] AND \"biology\"[All Fields]) OR \"synthetic biology\"[All Fields]) AND (\"ethics\"[Subheading] OR \"ethics\"[All Fields] OR \"ethics\"[MeSH Terms])) AND safe[Abstract]) AND synthetic biology[Abstract] AND \"open access\"[filter]')\n",
    "\n",
    "\n",
    "for i in ids:\n",
    "    r = requests.get('https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/PMC{}/unicode'.format(i))\n",
    "    with open('../PMC data/full text/pmc{}.json'.format(i), 'w') as f:\n",
    "        j_file = parse_json(r.json())\n",
    "        json.dump(j_file,f, indent=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "# abstract contain security\n",
    "ids, query = search_from_pmc('(((\"synthetic biology\"[MeSH Terms] OR (\"synthetic\"[All Fields] AND \"biology\"[All Fields]) OR \"synthetic biology\"[All Fields]) AND (\"ethics\"[Subheading] OR \"ethics\"[All Fields] OR \"ethics\"[MeSH Terms])) AND security[Abstract]) AND synthetic biology[Abstract] AND \"open access\"[filter]')\n",
    "\n",
    "ids = ids[:-1]\n",
    "for i in ids:\n",
    "    r = requests.get('https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/PMC{}/unicode'.format(i))\n",
    "    with open('../PMC data/full text/pmc{}.json'.format(i), 'w') as f:\n",
    "        j_file = parse_json(r.json())\n",
    "        json.dump(j_file,f, indent=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Dual use is not a concept that is unique for the life sciences. The (possibility of) dual use is as old as engineering and designing. Literally dual use means nothing more and nothing less than that a certain activity or a certain object can be applied in at least two ways. This is the case with almost everything that has been designed or developed, but also with objects that are not human made, like natural herbs. To give some examples: a kitchen knife can be used to cut, but also sometimes as an alternative for a screw driver and indeed also to stab someone.1 Palliative pills are meant to alleviate pain, but if someone takes enough of these pills they can be used for committing suicide. This list can be continued endlessly. Almost every artefact and many natural products can be applied in a dual or even multiple use way. The dual or multiple ways an artefact can be used are not always intended by the designer. A screwdriver is not designed to stab a person. In pharmaceutical research unexpected or unintended effects of medicines also can lead to dual use. Sometimes the original function even is displaced by the unintended one. A well-known example is the Viagra pill. This pill had been designed against angina pectoris, but is now used for almost 100% to solve erection problems.\\nAlthough serendipity surely is an interesting phenomenon in (life) sciences most scientists and engineers do not spend a lot of time on thinking about the unintended or unexpected side-effects that can occur when their products are used (van Andel).2 Even less they think of intended misuse. Making scientists, engineers and other designers aware of the possible misuse of their ‘brainchild’ is the main goal of the dual use policy that has been developed in the life sciences during the past years.\\nBased on the intention of the engineers and designers some distinctions can be made when declaring an artefact, an activity or a natural product as dual use. If the starting point is the military field, dual use is a concept indicating that new technological developments in the military field are also useful for non-military purposes. History shows some well known examples of this spin-off effect of military technology. Areas of spin-off include variable fields as communications, fuels, weather observation, power sources, protective clothing, and displays.\\nThe concept of dual use that is at stake in present day discussions refers to (civil) products and technologies that can be used for beneficial purposes as well as for malicious purposes. As was said above this possibility of dual use of artefacts and of knowledge is possible with almost everything that is designed or produced. But in the present situation the most important application of the concept of dual use is related to weapons of mass destruction (WMD). As nuclear weapons are the most important of the WMD the concept was first applied in debates about the risk of misuse of nuclear technology. The concept of dual use in relation to nuclear weapons is most significant because of the regime of arms control. Inspections of the International Atomic Energy Agency (IAEA) are mainly directed at discovering intended or perhaps unintended activities of countries that can be seen as a breach of the Non Proliferation Treaty (NPT). Because of this link with the NPT states are the main actors that IAEA focuses upon, as is illustrated by the recent discussions about Iran and North Korea. But of course the risks of (nuclear) terrorism have not gone unnoticed by IAEA. In its mid term strategy for the period 2006–2011 IAEA has as one of its goals to “develop a comprehensive set of recommendations and guidelines for the international community, for the prevention, detection and response to acts of nuclear terrorism or other malicious acts, along with an appropriate review process” (IAEA).\\nEach State Party to this Convention undertakes never in any circumstances to develop, produce, stockpile or otherwise acquire or retain:\\n(1) Microbial or other biological agents, or toxins whatever their origin or method of production, of types and in quantities that have no justification for prophylactic, protective or other peaceful purposes;\\n(2) Weapons, equipment or means of delivery designed to use such agents or toxins for hostile purposes or in armed conflict.\\nHow is the dual use concept introduced and applied in relation to the life sciences? To gain insight in this question it is useful to start with describing the arms control context of biological weapons. This context is determined by the Biological and Toxin Weapons Convention (BTWC). The BTWC was signed in 1972 and came into force in 1975. In those years BTWC was one of a greater number of arms control agreements during the Cold War period. The BTWC became possible—as Fidler and Gostin write—when the United States decided unilaterally to terminate its offensive program on biological weapons (Fidler and Gostin, p. 47). The Soviet Union and other main powers followed. And so the first treaty that required real disarmament came into being.3 The main article of the BTWC is article I: \\nHas the concept of dual use been an important element of the Convention? This was not the case until after the anthrax attacks of 2001. The words “dual use” do not appear in the text of the Convention nor in any of the final declarations of the six review conferences that have been organized since 1980. However, during the Sixth Review Conference (2006) explicit attention was given to what is called the misuse of biotechnology: “The Conference recognizes that while recent scientific and technological developments in the field of biotechnology would increase the potential for cooperation among States Parties and thereby strengthen the Convention, they could also increase the potential for the misuse of both science and technology” (Final Document Sixth Review Conference BTWC, p. 19). The Conference decides to contribute one of the intersessional meetings to “Oversight, education, awareness raising, and adoption and/or development of codes of conduct with the aim of preventing misuse in the context of advances in bio-science and bio-technology research with the potential of use for purposes prohibited by the Convention” (Final Document Sixth Review Conference BTWC, p. 25). In 2008 an Expert Meeting and an Intersessional Meeting of States Parties were organized about the risks of misuse of biotechnology. In a report which synthesizes the discussions of the Expert Meeting 2008 the concept of dual use is mentioned in relation to the development of codes of conduct (Document BWC/MSP, p. 7–8). Dual use is linked to three different words: biological agents or toxins, potential (of research) and directly to research. These different ways to apply dual use indicate that—even on the level of state parties of the BTWC—dual use is not an unequivocal concept. This can be illustrated by the recent discussions on the dual use character of synthetic biology.\\nIt is remarkable that dual use was seen as an important issue almost from the beginning of the development of synthetic biology (De Vriend, p. 54–57). What is the dual use risk of synthetic biology? Tucker and Zilinskas argue that “at present, the primary threat of misuse [of synthetic biology] appears to come from state-level biological warfare programs”. They refer to former Soviet scholars who could be engaged in such projects. But at the same page they state that these developments are “extremely unlikely” (Tucker and Zilinskas, p. 38). Tucker and Zilinskas point to other risks: one of the possible scenarios for the deliberate misuse of synthetic biology involves a “lone operator,” such as a highly trained molecular biologist who develops an obsessive grudge against certain individuals or groups (or society as a whole) (Tucker and Zilinskas, p. 17). The other risk comes from so called biohackers, e.g. college kids who are eager to demonstrate their technological abilities (Tucker and Zilinskas, p. 18). The growth of the number of do-it-yourself-biologists has indeed got attention of the authorities. The American Federal Bureau of Investigation (FBI) “has adopted what some call a ‘neighbourhood watch’ stance. The approach relies on biohackers monitoring their own community and reporting behaviour they find threatening” (Ledford). If indeed second hand tools for genome assembly are becoming available to the public at affordable costs then this would seem to add weight to the concerns over possible misuse of synthetic biology research (Balmer and Martin).\\nInsist That All Commercial Gene Synthesis Houses Adopt Current Best Practice Screening Procedures.\\nCreate and Endorse New Watch-Lists To Improve Industry Screening Programs.\\nCreate a Confidential Hotline For Biosafety and Biosecurity Issues.\\nAffirm Members’ Ethical Obligation to Investigate and Report Dangerous Behavior.\\nCreate a Community-Wide Clearinghouse for Identifying and Tracking Potential Biosafety/Biosecurity Issues.\\nEndorse Biosecurity/Biosafety Research&Development Priorities.\\nIs there a relevant difference between dual use issues in ‘classic’ biology and in synthetic biology. Tucker and Zilinskas state that “the most likely misapplication of synthetic biology for hostile purposes involves the recreation of known pathogenic viruses in the laboratory” (Tucker and Zilinskas, p. 16). If this is the case, the new element is in the aspect of recreation (e.g. of the Spanish flu virus). Problems can arise if such a recreated virus is misused for biological weapons or bioterrorism. But there is essentially no difference with the dual use issue in traditional biology. Although there still are many unknown factors Tucker and Zilinskas think it “likely that, given the difficulty of anticipating and assessing the risks associated with synthetic organisms, synthetic biology will require a new approach to regulation that differs significantly from the NIH [National Institutes of Health] Guidelines on recombinant DNA [Deoxyribonucleic acid]” (Tucker and Zilinskas, p. 19). Already in 2006 the United States synthetic biology community proposed some measures that can be seen as an addendum of existing guidelines (Maurer et al.): \\nMost of these measures have a procedural character that is not specific for synthetic biology. These measures could be applied in a general biosecurity policy. In fact some of them are common practice already in a number of countries and laboratories. An important question regarding the possible dual use of synthetic biology is a rather pragmatic one: why take the long and complex way of synthesizing a biological weapon, if in practice there are many more and easier ways to reach the same result? This is confirmed by the American biologist Drew Endy. He estimates the risks for the short and medium term very low. “From a security perspective, many people are concerned that it is now possible to directly construct harmful pathogens from DNA sequence information. This seems to me a real but remote possibility, and is likely best addressed by improvements in our capacity to respond to emerging infectious diseases, natural or otherwise, and to our public health systems. The more pressing security concern is to ensure that the tools and policies defining the future of biotechnology do not directly or inadvertently lead to a remilitarization of biology by nations.” (Endy)\\nThe example of the discussion on synthetic biology shows that attention for dual use has spread widely in the life sciences during the past decade. The example also makes clear that the emphasis in the discussion is placed more on the possible consequences of misusing synthetic biology (new diseases etc.) than on the question how (un)realistic such a threat is.\\nDemonstrate how to render a vaccine ineffective;\\nConfer resistance to therapeutically useful antibiotics or antiviral agents;\\nEnhance the virulence of a pathogen or render a non-pathogen virulent;\\nIncrease the transmissibility of a pathogen;\\nAlter the host range of a pathogen;\\nEnable the invasion of diagnosis and/or detection by established methods;\\nEnable the weaponization of a biological agent or toxin.\\nThe discussion on synthetic biology confirms an assertion of Pustovit and Williams. They perceive two main approaches to dual use technology: the Anglo-American pragmatic approach and the continental metaphysical approach (Pustovit and Williams). According to the authors these approaches are based on different understandings of technology: the Anglo-American approach “is associated with technology notion as a sequence of processes and operations aimed to products with necessary and useful for people properties. The second one with enough neutral definition-understanding of technology as use of organized knowledge for aiming at practical goals by systems of machines and people” (Pustovit and Williams, p. 2). In their view the Anglo-American approach is directed at concrete technologies, the continental one also at the role and intentions of people. The authors illustrate the ‘American’ view by referring to the phrase “dual use research of concern” that has been developed in the—already mentioned—famous Fink-report of the National Research Council of the National Academies, the starting point and analytical fundament of the present biosecurity debate in the United States, and not only there. Dual Use Research of Concern is defined as “research that, based on current understanding, can be reasonably anticipated to provide knowledge, products, or technologies that could be directly misapplied by others to pose a threat to public health and safety, agriculture, plants, animals, the environment, or material” (National Research Council of the National Academies). The list of experiments of concern has now an almost official status. Experiments of concern are those that: \\nGenetic sequencing of pathogens;\\nSynthesis of pathogenic micro-organisms;\\nAny experiment with variola virus (smallpox);\\nAttempts to recover/revive past pathogens.\\nIt is referred to in many other publications, such as the report that Miller and Selgelid wrote for the Australian government. Miller and Selgelid add some other categories: \\nAccording to Pustovit and Williams in declaring activities or technologies as ‘dual use’, the emphasis is laid upon the possible consequences of applications of the technology as such. Indeed, it is not difficult to describe those consequences for all of the above ‘experiments of concern’. An example is altering the host range of a pathogen. If a pathogen can be changed in such a way that it can survive not only in e.g. animals, but also in human beings, this can lead to new diseases for mankind. Animal diseases become human diseases. And because antibiotics or other remedies for such new diseases do not (yet) exist, the consequences can be disastrous. The risk of the transfer of animal diseases to humans is not imaginary, as is shown by the bird flu and more recently in The Netherlands by the Q-fever that is transmitted to humans by pregnant goats and sheep.\\nHowever, a consequence of putting (too) much emphasis on possible consequences of (mis)use of life science technologies, could be that the aspect of threat or intention gets too less weight in declaring a technology as dual use. The assignment of the label “dual use” should not only be determined by the biological, chemical or physical properties of the technology as such, but also by realistic interpretations and expectations about the way the technology will be used. In other words: an artefact, a technology or a natural product will become “dual use” only by a combination of (technical) properties and intentions. Let us make the comparison with the kitchen knife: only in very exceptional circumstances a knife becomes a weapon.\\nThe question is if Pustovit and Williams are right in making this sharp distinction between an Anglo-American and a continental approach. The existing or perceived threats were the starting point for the Fink Committee. The Committee was charged “to consider ways to minimize threats from biological warfare and bioterrorism without hindering the progress of biotechnology, which is essential for the health of the nation” (National Research Council of the National Academies, p. vii). Given the professional and scientific background of the Committee members it is fully understandable that they concentrated on the scientific and technical aspects of the problem and not on the political and security aspects. In fact, Miller and Selgelid’s approach also clearly emphasises both consequences and intentions. For example, they explicitly state: “For something to be an instance of a dual use dilemma both outcomes (the two horns of the dual use dilemma) need to be (actually or potentially) intended…” (Miller and Selgelid, p. 12). So Pustovit and Williams’ presentation of a sharp dichotomy between the views expressed in the literature is somehow misleading. That said, it is important to make clear that both intentions and consequences are in play.\\nThe relevance of identifying a threat for defining dual use is also the point of view of John Forge: “to classify something as dual use should not simply be the flag that the item could have some bad use, that some bad use is in theory possible (…) for artefacts at least, there has to be some threat to make and use an improvised weapon for it to be dual use”. Furthermore Forge remarks that threats come and go. This means that a technology or an artefact that is labelled as dual use today does not have to be necessarily a dual use issue tomorrow. This is a very relevant remark, because it focuses attention on questions as: is there a threat?; what kind of a threat is it?; who determines if there is a threat?; is this threat serious enough to declare a technology or an artefact as dual use?; etc. Maybe the most important, but often neglected question is: when does a threat, and thus a dual use marking disappear and who is to decide this? Sometimes it seems that since the terrorist attacks and the anthrax letters of 2001 almost irreversible steps have been taken to counter a possible terrorist or—more specific—bioterrorist threat. Now, almost 10\\xa0years later the question could be raised if the events since 2001 still justify this focus on bioterrorism and biosecurity.\\nBefore further elaborating on these questions, some remarks will be made on the relationship between threats and intentions. This will be done by introducing two translations of the word threat in Dutch: dreiging and bedreiging. They mark a difference that does not exist in the same way in English. Bedreiging is used if an actor is threatening to act in a certain way, e.g. with using violence. Threat as bedreiging looks at a situation from the perspective of the subject making a threat. Dreiging on the other hand focuses at the perspective of the person or group that is or feels threatened. The dreiging perspective of threat does not have to coincide with that of bedreiging. Someone can feel threatened while in fact there is no real threat, because no one has the intention to commit an action against this person. The reverse is also possible: someone does not feel threatened, while in fact a real threat exists, because an actor has the intention to commit a crime or a terrorist action. There are indications that the aspect of dreiging is most influential in the recent debates on biosecurity. Because of that the chance exists that in declaring an activity as dual use an emphasis is laid on the dual use potentiality and that there is an underestimation of the aspect of intentionality (bedreiging).\\nRecent history seems to confirm this assertion. If we take a look at the examples of threats that are presented as illustrations (or perhaps even proofs) of the threat, it is remarkable that—even in the recent National Strategy for Countering Biological Threats of the United States(US) National Security Council —only three already well known examples and one suspicion are recorded. These examples are: the Rajneeshee attack with a contaminated salad in Oregon (1984); contamination with anthrax spores by Aum Shinrykio (Japan) and the anthrax letters in the US (2001). The suspicion that Al Qaida might be preparing a bioterrorist attack was found after the occupation of Afghanistan. Three examples in more than 25\\xa0years! Of course the seriousness of each of these examples should not be underestimated. But for politicians and for decision makers in the life sciences it should be a serious consideration if these cases really justify the whole range of measures that haven been taken during the past years. Judith Reppy summarizes the developments in life sciences of the past decade with the conclusion that “concerns about bioterrorism raise the issues of dual-use technology in a field that until recently was not of much interest to the military” (Reppy, p. 7). These developments have lead to what is called a securitization of life sciences and of public health (Kelle).4 \\nSecuritization is a concept that has been developed in the 1990\\xa0s by the Danish political scientist Ole Wæver. Because of that securitization is seen as a concept of the Copenhagen School of International Relations. This school is closely related to the English School, represented by theorists as Barry Buzan (Buzan). For Wæver security is a speech act. “It is by labeling something a security issue that it becomes one” (Wæver, p. 13). Stating that a particular referent object is threatened in its existence claims a right to extraordinary measures to ensure the referent objects survival. The issue is then moved out of the sphere of normal politics into the realm of emergency politics, where it can be dealt with swiftly and without the normal (democratic) rules and regulations of policy making. For the content of security this means that it has no longer any given meaning but that it can be anything a securitizing actor says it is. Security—understood in this way—is a social construction, with the meaning of security dependent on what is done with it (Taureck). “‘Security’ is the move that takes politics beyond the established rules of the game and frames the issue as a special kind of politics or as above politics. Securitization can thus be seen as a more extreme version of politicization. In theory, any public issue can be located on the spectrum ranging from nonpoliticized (…) through politicized (…), to securitized (…). This link between politicization and securitization does not imply that securitization always goes through the state, politicization as well as securitization can be enacted in other for a as well” (Taureck, p. 23–24). Securitization studies aim to understand “who securitizes, on what issues (threats), for whom (referent object), why, with what results, and not least, under what conditions” (Buzan et al., p. 32). Translated to the dual use issue the main questions are: (a) did a securitization process take place in the life sciences and (b) if so how did it influence the dual use issue?\\nAccording to Fidler and Gostin securitization of public health means “that the theory and practice of public health are increasingly considered in security terms” (Fidler and Gostin, p. 121). This is a rather new development, not only in health care, but also in the broader world of the life sciences. Public health and life sciences on the one hand and security on the other hand were until a few years ago almost completely separated worlds. “Biologists and other life scientists were—unlike the physicists—not involved in security politics, except for a relatively limited group of biologists and other life scientists who were working in Defence laboratories in order to develop biological weapons or to contribute to biodefense research” (van der Bruggen, p. 69). But most of these life scientists did not take part in public debates on biological weapons or—broader—weapons of mass destruction, like e.g. physicists did on nuclear weapons.5 This has changed in the past decade. The following reasons can be distinguished for the securitization of the life sciences (Van der Bruggen).\\nA first reason for securitization is the reference to the events of 11 September 2001 and the anthrax letters in the same period. Undoubtedly these events were an important reason for seriously considering if life sciences could be a source for terrorist attacks. But some qualification has to be considered. First of all: the anthrax attack was the result of the actions of one deranged researcher who was working in a Defence laboratory. He had no link with any terrorist or radicalized (Muslim) organization. This makes his deeds no less repugnant, but this fact should have influenced the threat analysis. Of course it has taken some years before this conclusion could be drawn definitively, but from the beginning the experts knew that the anthrax spore came from one of the American Defence laboratories. Nevertheless it is a legitimate question to investigate the risks of a comparable threat by terrorist groups. But how credible is it to expect a bioterrorist threat from these groups? Are there really any indications for such an expectation? Of course intelligence services will have to perform their inquiries. And it is understandable that they do not make all their results public. So it cannot be excluded that there really are some indications for a threat.\\nA second reason for securitization can be found in the initiatives of the BTWC State Parties during the 5th and 6th review conferences and the intersessional meetings to stimulate awareness raising among scientists, e.g. by developing codes of conduct. Attention—also within BTWC—shifted from state actors to non state actors and the risks of bioterrorism. And, of course, most of these discussions were held in the same time that the world was confronted with a series of terrorist attacks. This gave an extra argument to involve the scientific world in the BTWC activities (Revill and Dando).\\nA next important reason is the occurrence of (new) infectious diseases that threaten humans as well as animals: Acquired Immune Deficiency Syndrome (AIDS), Severe Acute Respiratory Syndrome (SARS) and the Bird Flu are the most well-known examples. Some authors—as Fidler and Gostin—put naturally occurring infectious diseases under the heading of biosecurity (Fidler and Gostin, p. 2). They see this broadening of the security concept as an effort to release the concept of security from the “traditional state centred military-biased perspective” (Fidler and Gostin, p. 6). In their opinion public health and security are unjustly two almost completely separated worlds. They refer to the concept of human security to defend their view (Human Security Centre). Although there are divergent interpretations of human security, a common denominator is that the concept of security should be broader than national or military security. All proponents agree that the primary goal of human security is the protection of human individuals. This broadening of the concept of security implies that there should not only be attention for prevention of possible threats of intentional spreading diseases. Security issues are also at stake because of the effects of diseases, irrespective if these are intentionally causes or not. Pandemic explosions can lead to societal unrest and even upheavals.\\nGlobalization has in at least two ways led to more attention for biosecurity issues. First, a growing number of international personal and commercial contacts may contribute to a faster and more extensive spread of viruses around the world. Local epidemics can become national or even global epidemics. There are examples of sources for diseases (such as the malaria mosquito) that are spread to new regions (e.g. because of climatic changes or as a ‘stowaway’ in a plane or ship). Globalization also promotes the international contacts between scientists and researchers. Personal exchange, appointments abroad and international conferences have grown exponentially during the past decades. The advantages of this development are obvious. Science and technology flourish because of it. More people from more countries are able to contribute to science. But there is the other side that intentional or unintentional misuse is made of scientific results. In The Netherlands the example of the Pakistani nuclear scientist Dr. Khan is well known. He was from May 1972 to December 1975 working at an engineering firm based in Amsterdam and a subcontractor to the Uranium Enrichment Corporation (URENCO) specializing in the manufacture of nuclear equipment. URENCO’s primary enrichment facility was at Almelo, Netherlands. A. Q. Khan, in his capacity would eventually have an office at that facility by late 1974. In early 1976, Dr. Khan left the Netherlands with secret URENCO blueprints for a uranium centrifuge.\\nLast but not least another consequence of globalization is that terrorist activities are no longer limited to regional and local conflicts. Unlike the Irish Republican Army (IRA) or Euskadi Ta Askatasuna (ETA), groups as Al Qaeda have made the entire world to their working area.\\nThe preceding paragraphs provide answers to the questions that were formulated above: (a) did a securitization process take place in the life sciences and (b) if so how did it influence the dual use issue? There has to be no doubt that indeed a process of securitization has taken place in the life sciences. The main actors of this securitization process are politicians and policy makers as well as the scientific community itself. The main reasons are political (9/11; BTWC, globalization) as well as scientific (new epidemics, scientific developments).In relation to the second question it can be established that due to this securitization process dual use has become an issue for the life sciences. The publication of the Fink report in 2004 is a first milestone in this development. A great range of national and international activities have taken place since then, varying from expert meetings of the BTWC to national codes of conduct and education and training courses.\\nTaking all these developments into account, the question remains if the biosecurity risk indeed is significantly greater than 10 or 25\\xa0years ago. Can the securitization process be declared and justified. Is it really necessary to mobilise the life sciences community and make them aware, or are we pulled along in a whirlpool of developments that have led to an overexposure of perhaps not or hardly existing biosecurity risks? It is not easy to give a validated answer to this question, if only because there is much confidential information held by intelligence services and other government officials on the occurrence of incidents. Moreover it is hard to prove that or why some incidents have not happened. Comparable discussions took place on the effectiveness of nuclear deterrence during the Cold War: it cannot be proven that nuclear deterrence really has prevented the occurrence of (nuclear) war. But did a nuclear war not occur because of the effectiveness of nuclear deterrence or maybe because there was no real threat?\\nCan the same be said of dual use policy for the life sciences? Some caveats are in place.\\nThe first caveat is about the reaction of the state and that of the public to the terrorist threat. When after September 2001 the threat of a terrorist attack moved up the chart of security risks, it was clear from the beginning that government saw a prominent role for itself in reducing that threat. This is understandable because terrorism belongs to the kinds of threats that are directed against the state as such, which is not the case with e.g. airplane crashes. This link with the ‘core business’ of government led to a revival of the role of the state as the ultimate guardian of security. Shortly after September 11 Francis Fukuyama (one of the prophets of neo-liberalism) wrote very strikingly that it is not the Microsofts of this world that send aircraft carriers (Fukuyama). Therefore a state with its monopoly of violence is necessary! The new security issues made clear that there are domains where public and political responsibilities come first. In the years after 9–11 governments took up numerous activities and institutions to reduce terrorist threats. In general these activities were supported by the population, even if it led to measures that could be seen as limitations of individual freedom or privacy. This acceptance can be related to a high level of subjective insecurity. Many people feared new terrorist attacks.\\nBut how does this subjective (feeling of) insecurity (dreiging) relate to the objective level of insecurity that is caused by potential bioterrorism (bedreiging)? Certainly the risk of dying because of a bioterrorist attack is many times smaller than the chance of dying in a traffic accident. But the social shock and distress are much greater. This can be explained by the fact that terrorist attacks hit society as such. If 3,000 people would have been killed after the World Trade Centre had fallen down because of an earthquake, the shock would have been great also, but incomparable with what happened after 9–11. The essence of the state and the capability of the state to protect its citizens were at stake! The terrorist attack was seen as an act of war, comparable with the attack on Pearl Harbour in 1941. This view explains the reaction of governments, not only in the United States, but also in many other states. The reactions to 9/11 clearly show that governments are willing to spend more financial and personal means to counter terrorist threats than other security risks, which do not directly affect its core tasks.\\nAnother phenomenon that can be observed is the tendency of policy makers to think in worst case analyses rather than in probabilities. Again a plausible explanation can be given. Our views and images of terrorism have been formed by the attack on the World Trade Centre and the massive attacks in Madrid and London. For these kinds of terrorist attacks even a new name has been created, catastrophic terrorism: attacks using weapons of mass destruction or more simple methods to kill thousands of civilians for political pressure, political theatre, or crazed self-expression (Fearon). Although political authorities are aware of the low probability of terrorist attacks they are prepared for the worst: policy is developed that is based on such worst case scenarios.\\nA third caveat has a more psychological character: a phenomenon that can be identified as anticipated decision regret. Anticipated decision regret is an attitude of people which leads them to take actions that are directed at preventing possible future incidents. If I take this preventive measure now, it will mean that I do not have to blame myself (or get blamed by others) for not having done everything to prevent that incident from happening. This attitude also can be observed in health care. A growing number of preventive screening tests are offered that provide information of the chance of developing some kind of disease. It is often not taken in consideration that the chance of really getting this disease is very small. And it is possible that the measures that are taken to prevent the disease negatively influence lifestyles. The Dutch medical sociologist Tjeerd Tijmstra gives some—often hilarious—examples of anticipated decision regret: if a mother gets a screening test for her child offered for a disease for which the risk is 1 in 90,000, there is a great willingness to participate. And even if it is explained to people that the chance of having a car accident while driving to the clinic is about as much as that particular risk, still many people decide to go on. Their motivation seems to be: suppose our baby does develop that disease, then we should not forgive ourselves not to have done everything to prevent it. There are signals that this anticipated decision regret has become an attitude in security issues also. But in the field of security governments and other public agencies (such as scientific organizations) are bearers of the attitude more than individual citizens. After 9/11—let it be repeated again—security measures against possible terrorist attacks have been given an enormous priority. It looks as if governments are willing to invest a lot of energy in minimizing the chances of terrorist attacks. They do not want to take the risk that they have not done everything to prevent an assault. This attitude can be based on experience. Officials of the Dutch government were reproached for not having done enough to prevent the murder of film director Theo van Gogh (2004). These reproaches have led to decisions that are directed at minimizing the chance of new attacks. With—in general—societal and political support a good deal of money and a lot of energy are devoted to this topic. It is not farfetched to suppose that this is one of the effects of our common anticipated decision regret.\\nThe considerations that have been presented in the previous pages might the reader lead to the assumption that there is no much need to worry about bioterrorism and that most measures are highly exaggerated. That would not be a right assumption. There are reasons to be cautious for the risks of bioterrorism. But one should not stop thinking! Anti terrorist policy should not be carried out on autopilot. Fortunately, many politicians and officials of the Intelligence services are among the first ones to stress that possible acts of Chemical, Biological, Radiological or Nuclear terrorism (CBRN) do almost certainly not imply the use of weapons of mass destruction with thousands of victims, but of less destructive (although of course still very dangerous and harmful) weapons.\\nBut still it is possible that politicians and officials are so much preoccupied with anti-terrorism policy that all information is interpreted in such a way that it strengthens the conviction that already exists. In that case information that contradicts this conviction or that is not directly relevant could be underestimated or neglected. A focus on security issues can lead to an attitude where other policy issues are subordinated to security issues or will be judged only in their relation to security issues. To give a fictitious, but not unrealistic example from the life sciences: why is a student from a Middle East country coming to this European laboratory for his PhD research? The idea that this person just wants to become a good scientist in order to help his country in fighting serious diseases could be set aside by the bias driven view that he or she could be a potential terrorist, so “we will watch him and prevent him from stealing materials”.\\nThis example brings us back to the dual use policy as it has been developed in the past years. The conclusion is that there is nothing wrong with more awareness of the potential dual use that can be made with the materials or the results of life sciences research, but this awareness should not become so predominant that distrust is the default attitude in a laboratory. Codes of conduct, like the Dutch Code of conduct for biosecurity have proven to be an appropriate method to make life scientists and other people that are involved in biotechnological research aware of the dual use issue without exaggerating the risks (Koninklijke Nederlandse Akademie van Wetenschappen).\\nIn almost all discussions that were entered in relation to the Code of Conduct a conclusion was that such a Code alone is not enough to limit the risk of the misuse of dual use technology. Measures have to be taken to reduce the risk of dual use research and technology beyond the spheres of awareness. It will be important to go on with this policy on the basis of cooperation between all involved parties: scientists, funding organisations, universities, hospitals, politicians, officials of ministries and of course experts on terrorism and anti-terrorism. Such cooperation does not only take away possible misunderstandings, but it is also an appropriate remedy against possible tunnel visions, that could lead to an overemphasis of biosecurity. Although it is very useful to pay attention to possible developments in an early stage, possible dual use should not get such a high priority that promising developments in e.g. the field of synthetic biology are hampered or that talented researchers from abroad (especially from suspected countries) do not get the chance to participate in research.\\nAt the end of this article, I translate the main conclusions into a proposal for an acceptable, adequate and applicable definition of the dual use concept for researchers, universities, companies and policy makers. The reason for proposing such a new definition is not academic in nature. The intention is to offer a definition that will be consistent with existing definitions of dual use, but wider. The conclusion of the above cannot be otherwise than that a definition of dual use should encompass more than only the technical or physical properties of a biological agent. Dual use policy should be based upon a deliberate consideration of technical possibilities, threats and intentions and possible consequences.\\nThat which has both civilian and military applications;\\nThat which can be used for both beneficial/good and harmful/bad purposes, and\\nThat which has both beneficial/good and harmful/bad purposes—where the harmful/bad purposes involve weapons, and usually weapons of mass destruction.\\nThe most common and authoritative definition still is the earlier mentioned one in the Fink-report (National Research Council of the National Academies): Dual Use Research of Concern is: \"Research that, based on current understanding, can be reasonably anticipated to provide knowledge, products, or technologies that could be directly misapplied by others to pose a threat to public health and safety, agriculture, plants, animals, the environment, or material\". The definition is explained by the list of seven experiments of concern. In a recent publication Selgelid proposes three plausible definitions of dual use science and technology: \\nAs the authors of the Fink report and Selgelid are aware, uncertainties are inherent to both threats and expected damage. Moreover, as argued above, both intentions and consequences need to be involved in defining dual use. Further, too much emphasis on worst case scenarios is problematic. These kinds of scenarios describe the disasters that (can) take place if indeed the biological agent is misused for terrorist or criminal intentions. But worst cases are not by definition realistic scenarios. Because of that a definition of dual use should not only be based on elements that point to worst cases. So—with all necessary restraint—this definition will be proposed:\\nresearch, based on current understanding, can be reasonably anticipated to provide knowledge, products, or technologies that could be misapplied and;\\nthere is a recognizable threat and a not negligible chance of such misuse and;\\nthere are serious consequences for society and science (public health and safety, agriculture, plants, animals, the environment, or material).\\nA dual use problem arises when \\n'"
      ]
     },
     "execution_count": 205,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "j_file['body']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## XML Parsing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Example for pmc6692427"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n",
      "<!DOCTYPE pmc-articleset PUBLIC \"-//NLM//DTD ARTICLE SET 2.0//EN\" \"https://dtd.nlm.nih.gov/ncbi/pmc/articleset/nlm-articleset-2.0.dtd\">\n",
      "<pmc-articleset>\n",
      " <article article-type=\"brief-report\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
      "  <?properties open_access?>\n",
      "  <front>\n",
      "   <journal-meta>\n",
      "    <journal-id journal-id-type=\"nlm-ta\">\n",
      "     Front Bioeng Biotechnol\n",
      "    </journal-id>\n",
      "    <journal-id journal-id-type=\"iso-abbrev\">\n",
      "     Front Bioeng Biotechnol\n",
      "    </journal-id>\n",
      "    <journal-id journal-id-type=\"publisher-id\">\n",
      "     Front. Bioeng. Biotechnol.\n",
      "    </journal-id>\n",
      "    <journal-title-group>\n",
      "     <journal-title>\n",
      "      Frontiers in Bioengineering and Biotechnology\n",
      "     </journal-title>\n",
      "    </journal-title-group>\n",
      "    <issn pub-type=\"epub\">\n",
      "     2296-4185\n",
      "    </issn>\n",
      "    <publisher>\n",
      "     <publisher-name>\n",
      "      Frontiers Media S.A.\n",
      "     </publisher-name>\n",
      "    </publisher>\n",
      "   </journal-meta>\n",
      "   <article-meta>\n",
      "    <article-id pub-id-type=\"pmid\">\n",
      "     31448268\n",
      "    </article-id>\n",
      "    <article-id pub-id-type=\"pmc\">\n",
      "     6692427\n",
      "    </article-id>\n",
      "    <article-id pub-id-type=\"doi\">\n",
      "     10.3389/fbioe.2019.00175\n",
      "    </article-id>\n",
      "    <article-categories>\n",
      "     <subj-group subj-group-type=\"heading\">\n",
      "      <subject>\n",
      "       Bioengineering and Biotechnology\n",
      "      </subject>\n",
      "      <subj-group>\n",
      "       <subject>\n",
      "        Perspective\n",
      "       </subject>\n",
      "      </subj-group>\n",
      "     </subj-group>\n",
      "    </article-categories>\n",
      "    <title-group>\n",
      "     <article-title>\n",
      "      Future Trends in Synthetic Biology—A Report\n",
      "     </article-title>\n",
      "    </title-group>\n",
      "    <contrib-group>\n",
      "     <contrib contrib-type=\"author\">\n",
      "      <name>\n",
      "       <surname>\n",
      "        El Karoui\n",
      "       </surname>\n",
      "       <given-names>\n",
      "        Meriem\n",
      "       </given-names>\n",
      "      </name>\n",
      "      <xref ref-type=\"aff\" rid=\"aff1\">\n",
      "       <sup>\n",
      "        1\n",
      "       </sup>\n",
      "      </xref>\n",
      "      <xref ref-type=\"corresp\" rid=\"c001\">\n",
      "       <sup>\n",
      "        *\n",
      "       </sup>\n",
      "      </xref>\n",
      "      <uri xlink:href=\"http://loop.frontiersin.org/people/306990/overview\" xlink:type=\"simple\"/>\n",
      "     </contrib>\n",
      "     <contrib contrib-type=\"author\">\n",
      "      <name>\n",
      "       <surname>\n",
      "        Hoyos-Flight\n",
      "       </surname>\n",
      "       <given-names>\n",
      "        Monica\n",
      "       </given-names>\n",
      "      </name>\n",
      "      <xref ref-type=\"aff\" rid=\"aff2\">\n",
      "       <sup>\n",
      "        2\n",
      "       </sup>\n",
      "      </xref>\n",
      "      <uri xlink:href=\"http://loop.frontiersin.org/people/737621/overview\" xlink:type=\"simple\"/>\n",
      "     </contrib>\n",
      "     <contrib contrib-type=\"author\">\n",
      "      <name>\n",
      "       <surname>\n",
      "        Fletcher\n",
      "       </surname>\n",
      "       <given-names>\n",
      "        Liz\n",
      "       </given-names>\n",
      "      </name>\n",
      "      <xref ref-type=\"aff\" rid=\"aff1\">\n",
      "       <sup>\n",
      "        1\n",
      "       </sup>\n",
      "      </xref>\n",
      "      <uri xlink:href=\"http://loop.frontiersin.org/people/737489/overview\" xlink:type=\"simple\"/>\n",
      "     </contrib>\n",
      "    </contrib-group>\n",
      "    <aff id=\"aff1\">\n",
      "     <sup>\n",
      "      1\n",
      "     </sup>\n",
      "     <institution>\n",
      "      SynthSys-Centre for Synthetic and Systems Biology, School of Biological Sciences, University of Edinburgh\n",
      "     </institution>\n",
      "     ,\n",
      "     <addr-line>\n",
      "      Edinburgh\n",
      "     </addr-line>\n",
      "     ,\n",
      "     <country>\n",
      "      United Kingdom\n",
      "     </country>\n",
      "    </aff>\n",
      "    <aff id=\"aff2\">\n",
      "     <sup>\n",
      "      2\n",
      "     </sup>\n",
      "     <institution>\n",
      "      Innogen Institute, School of Social and Political Sciences, University of Edinburgh\n",
      "     </institution>\n",
      "     ,\n",
      "     <addr-line>\n",
      "      Edinburgh\n",
      "     </addr-line>\n",
      "     ,\n",
      "     <country>\n",
      "      United Kingdom\n",
      "     </country>\n",
      "    </aff>\n",
      "    <author-notes>\n",
      "     <fn fn-type=\"edited-by\">\n",
      "      <p>\n",
      "       Edited by: Jean Marie François, UMR5504 Laboratoire d'Ingénierie des Systèmes Biologiques et des Procédés (LISBP), France\n",
      "      </p>\n",
      "     </fn>\n",
      "     <fn fn-type=\"edited-by\">\n",
      "      <p>\n",
      "       Reviewed by: Vijai Singh, Indrashil University, India; Chris John Myers, The University of Utah, United States\n",
      "      </p>\n",
      "     </fn>\n",
      "     <corresp id=\"c001\">\n",
      "      *Correspondence: Meriem El Karoui\n",
      "      <email>\n",
      "       Meriem.Elkaroui@ed.ac.uk\n",
      "      </email>\n",
      "     </corresp>\n",
      "     <fn fn-type=\"other\" id=\"fn001\">\n",
      "      <p>\n",
      "       This article was submitted to Synthetic Biology, a section of the journal Frontiers in Bioengineering and Biotechnology\n",
      "      </p>\n",
      "     </fn>\n",
      "    </author-notes>\n",
      "    <pub-date pub-type=\"epub\">\n",
      "     <day>\n",
      "      07\n",
      "     </day>\n",
      "     <month>\n",
      "      8\n",
      "     </month>\n",
      "     <year>\n",
      "      2019\n",
      "     </year>\n",
      "    </pub-date>\n",
      "    <pub-date pub-type=\"collection\">\n",
      "     <year>\n",
      "      2019\n",
      "     </year>\n",
      "    </pub-date>\n",
      "    <volume>\n",
      "     7\n",
      "    </volume>\n",
      "    <elocation-id>\n",
      "     175\n",
      "    </elocation-id>\n",
      "    <history>\n",
      "     <date date-type=\"received\">\n",
      "      <day>\n",
      "       28\n",
      "      </day>\n",
      "      <month>\n",
      "       5\n",
      "      </month>\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "     </date>\n",
      "     <date date-type=\"accepted\">\n",
      "      <day>\n",
      "       08\n",
      "      </day>\n",
      "      <month>\n",
      "       7\n",
      "      </month>\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "     </date>\n",
      "    </history>\n",
      "    <permissions>\n",
      "     <copyright-statement>\n",
      "      Copyright © 2019 El Karoui, Hoyos-Flight and Fletcher.\n",
      "     </copyright-statement>\n",
      "     <copyright-year>\n",
      "      2019\n",
      "     </copyright-year>\n",
      "     <copyright-holder>\n",
      "      El Karoui, Hoyos-Flight and Fletcher\n",
      "     </copyright-holder>\n",
      "     <license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\">\n",
      "      <license-p>\n",
      "       This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.\n",
      "      </license-p>\n",
      "     </license>\n",
      "    </permissions>\n",
      "    <abstract>\n",
      "     <p>\n",
      "      Leading researchers working on synthetic biology and its applications gathered at the University of Edinburgh in May 2018 to discuss the latest challenges and opportunities in the field. In addition to the potential socio-economic benefits of synthetic biology, they also examined the ethics and security risks arising from the development of these technologies. Speakers from industry, academia and not-for-profit organizations presented their vision for the future of the field and provided guidance to funding and regulatory bodies to ensure that synthetic biology research is carried out responsibly and can realize its full potential. This report aims to capture the collective views and recommendations that emerged from the discussions that took place. The meeting was held under the Chatham House Rule (i.e., a private invite-only meeting where comments can be freely used but not attributed) to promote open discussion; the findings and quotes included in the report are therefore not attributed to individuals. The goal of the meeting was to identify research priorities and bottlenecks. It also provided the opportunity to discuss how best to manage risk and earn public acceptance of this emerging and disruptive technology.\n",
      "     </p>\n",
      "    </abstract>\n",
      "    <kwd-group>\n",
      "     <kwd>\n",
      "      synthetic biology\n",
      "     </kwd>\n",
      "     <kwd>\n",
      "      biosystem\n",
      "     </kwd>\n",
      "     <kwd>\n",
      "      future trends and developments\n",
      "     </kwd>\n",
      "     <kwd>\n",
      "      biodesign automation\n",
      "     </kwd>\n",
      "     <kwd>\n",
      "      responsible research and innovation (RRI)\n",
      "     </kwd>\n",
      "    </kwd-group>\n",
      "    <funding-group>\n",
      "     <award-group>\n",
      "      <funding-source id=\"cn001\">\n",
      "       Biotechnology and Biological Sciences Research Council\n",
      "       <named-content content-type=\"fundref-id\">\n",
      "        10.13039/501100000268\n",
      "       </named-content>\n",
      "      </funding-source>\n",
      "     </award-group>\n",
      "    </funding-group>\n",
      "    <counts>\n",
      "     <fig-count count=\"1\"/>\n",
      "     <table-count count=\"0\"/>\n",
      "     <equation-count count=\"0\"/>\n",
      "     <ref-count count=\"40\"/>\n",
      "     <page-count count=\"8\"/>\n",
      "     <word-count count=\"5356\"/>\n",
      "    </counts>\n",
      "   </article-meta>\n",
      "  </front>\n",
      "  <body>\n",
      "   <sec id=\"s1\" sec-type=\"intro\">\n",
      "    <title>\n",
      "     Introduction\n",
      "    </title>\n",
      "    <p>\n",
      "     Synthetic Biology offers innovative approaches for engineering new biological systems or re-designing existing ones for useful purposes (see\n",
      "     <xref ref-type=\"fig\" rid=\"F1\">\n",
      "      Figure 1\n",
      "     </xref>\n",
      "     ). It has been described as a disruptive technology at the heart of the so-called Bioeconomy, capable of delivering new solutions to global healthcare, agriculture, manufacturing, and environmental challenges (Cameron et al.,\n",
      "     <xref ref-type=\"bibr\" rid=\"B6\">\n",
      "      2014\n",
      "     </xref>\n",
      "     ; Bueso and Tangney,\n",
      "     <xref ref-type=\"bibr\" rid=\"B3\">\n",
      "      2017\n",
      "     </xref>\n",
      "     ; French,\n",
      "     <xref ref-type=\"bibr\" rid=\"B19\">\n",
      "      2019\n",
      "     </xref>\n",
      "     ). However, despite successes in the production of some high value chemicals and drugs, there is a perception that synthetic biology is still not yet delivering on its promise.\n",
      "    </p>\n",
      "    <fig id=\"F1\" position=\"float\">\n",
      "     <label>\n",
      "      Figure 1\n",
      "     </label>\n",
      "     <caption>\n",
      "      <p>\n",
      "       Synthetic biology is developing into a biodesign platform where it will be possible to apply the “design-build-test-iterate (or deploy)” to predictably create cells or organisms able to produce a wide variety of novel molecules, materials or even cells for multiple applications.\n",
      "      </p>\n",
      "     </caption>\n",
      "     <graphic xlink:href=\"fbioe-07-00175-g0001\"/>\n",
      "    </fig>\n",
      "    <p>\n",
      "     Moreover, there are some concerns from governments that synthetic biology expands the pool of agents of concern, which increases the need to develop detection, identification and monitoring systems, and proactively build countermeasures against chemical and biological threats (Wang and Zhang,\n",
      "     <xref ref-type=\"bibr\" rid=\"B37\">\n",
      "      2019\n",
      "     </xref>\n",
      "     ). The participation of representatives from various government organizations at this meeting is testament to their commitment to maintaining an active dialogue with the synthetic biology community. In this way, they aim to keep abreast of the changing nature of threats and provide the best advice to government about investment in science and technology and the introduction or amendment of regulatory processes.\n",
      "    </p>\n",
      "    <p>\n",
      "     The cost of DNA sequencing and synthesis have decreased dramatically (Carlson,\n",
      "     <xref ref-type=\"bibr\" rid=\"B7\">\n",
      "      2014\n",
      "     </xref>\n",
      "     ; Kosuri and Church,\n",
      "     <xref ref-type=\"bibr\" rid=\"B25\">\n",
      "      2014\n",
      "     </xref>\n",
      "     ) and we have access to more genetic information and more powerful genetic engineering capabilities than ever before. Critical investments in infrastructure are bearing fruit and, as is described below, synthetic biology is increasingly becoming, at least part of, the solution to many of our present and future needs in medicine, food and energy production, remediation, manufacturing, and national security. So what is the potential of synthetic biology and what challenges does it still face to realize this?\n",
      "    </p>\n",
      "   </sec>\n",
      "   <sec id=\"s2\">\n",
      "    <title>\n",
      "     Advances in Synthetic Biology: The State of Play\n",
      "    </title>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Small Molecules: Production on Demand a Reality\n",
      "     </title>\n",
      "     <p>\n",
      "      Despite the lack of predictability in biology, and current technical constraints that limit data collection and analyses, we can now produce small molecules on demand using synthetic biology approaches.\n",
      "     </p>\n",
      "     <p>\n",
      "      Probably the most impressive examples come from the Foundry at the Broad Institute of MIT and Harvard. When the Defense Advanced Research Projects Agency (DARPA) put the MIT-Broad Institute Foundry's design capabilities to the test, its researchers were able to deliver 6 out of 10 molecules of interest to the US Department of Defense in 90 days. This “pressure test” confirms the potential of synthetic biology to address shortages of key compounds quickly (Casini et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B8\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      Indeed, many labs can now design and construct relatively complex gene networks capable of generating a wide variety of “designer” molecules in a range of host cells; however, this is often a slow iterative process of trial and error.\n",
      "     </p>\n",
      "     <p>\n",
      "      As yet, very few small molecules in medicine are manufactured using a synthetic biology process; it remains very difficult to engineer microbes to carry out processes that Nature did not intend. This is to be expected: the performance of microbes is “good enough” from an evolutionary perspective. Microbes evolved to address the specific needs and challenges of their natural environments not those of industrial fermenters and bioreactors. Gene Transfer from one system to another may sound easy but in practice is hard work and rarely generates sufficient reward (i.e., increased yield) to justify the investment made. The application of automation and artificial intelligence (e.g., in designing and building plasmids) may help to reduce the time and cost—and improve return on investment—in the future (Zhang et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B40\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        Scale up is product specific – we need more synthetic biology in the production process”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "     <p>\n",
      "      Plants make alternative production platforms. Improvements in mining plant genomes and the development of effective transient expression systems have enabled large-scale production of, for example, vaccines in tobacco plants in just a few weeks (Dirisala et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B12\">\n",
      "       2017\n",
      "      </xref>\n",
      "      ; Emmanuel et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B16\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ). Directing the production of synthetic biological materials to plant chloroplasts also shows promise (Boehm and Bock,\n",
      "      <xref ref-type=\"bibr\" rid=\"B1\">\n",
      "       2019\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      The photosynthetic reducing power generated in plant chloroplasts can be harnessed for the light-driven synthesis of bioactive molecules such as dhurrin, which protects plants against insects (Gnanasekaran et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B21\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      However, underlying all these platforms is a knowledge gap in our understanding in how nature works. This makes it very hard to apply the design/build/test/learn cycles used in conventional engineering to the production of synthetic biological materials whatever the production platform (yeast, bacteria, plants, or human cells) if the platform itself is not well-understood (Sauro et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B33\">\n",
      "       2006\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      What we need now are instruments able to measure and characterize outputs, assisted by progress in robotics and automation, and the application of machine learning approaches to analyse the data generated. This will help us to generate more robust models of biological systems, so we can improve experimental design for future engineering strategies.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        We can do ‘build’. ‘Test’ is the challenge when we want to learn from the iterative design process”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Healthcare: Reimagining Medicine\n",
      "     </title>\n",
      "     <p>\n",
      "      Synthetic biology is driving significant advances in biomedicine, which will lead to transformational improvements in healthcare. Already, patients are benefiting from so-called CAR (for chimeric antigen receptor) technology, which engineers the immune cells (T-cells) of the patient to recognize and attack cancer cells (June et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B22\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      Genetically engineered viruses are being used to correct defective genes in patients with inherited diseases such as Severe Combined Immune Deficiency (SCID) or epidermolysis bullosa (Dunbar et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B15\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      The ability to reprogramme somatic cells from patients into induced pluripotent stem cells is furthering our understanding of their disease, reducing the use of animals in research, and paving the way for the development of personalized medicines and cell therapies. In principle at least, we could engineer a patient's own cells to multiply, differentiate into different cell types and even self-assemble into new tissues, or even organs, to repair those damaged through disease or injury (Davies and Cachat,\n",
      "      <xref ref-type=\"bibr\" rid=\"B10\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ; Satoshi et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B32\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      Work on new vectors that are able to deliver large genetic loads to target tissues is helping to produce more efficient therapeutics and vaccines that will have fewer side effects and a smaller risk of resistance. Furthermore, optimizing antibody or vaccine production, or example, so that they are in an edible format (e.g., plant based), could greatly reduce the cost and increase the speed of vaccine production in an epidemic.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        We have the tools but need the creativity to make stuff that can't be made without synthetic biology”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "     <p>\n",
      "      In the next few years, genetically engineering pigs to be virus resistant and have human-like immune profiles could make xenotransplantation a clinical reality (Burkard et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B4\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ). Engineering the microbiome is expected to lead to the development of synthetic probiotics (Dou and Bennett,\n",
      "      <xref ref-type=\"bibr\" rid=\"B14\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      The synthetic biology initiative known as Human Genome Project-write (HGP-write) has set its sights even higher, rallying scientists to build entire human chromosomes (Boeke et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B2\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ). Concerns have been raised about the ethics of creating “synthetic humans” and indeed the scientific and commercial value of such a project. More recently, HGP-write champions have proposed a more focused project to build a virus-resistant chromosome, making at least 400,000 changes to the human genome to remove DNA sequences that viruses use to hijack cells and replicate (Dolgin,\n",
      "      <xref ref-type=\"bibr\" rid=\"B13\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      One of the many exciting opportunities that synthetic biology offers medicine is in the production of theranostic cell lines that can sense a disease state and produce an appropriate therapeutic response (Teixeira and Fussenegger,\n",
      "      <xref ref-type=\"bibr\" rid=\"B35\">\n",
      "       2019\n",
      "      </xref>\n",
      "      ). Several obstacles need to be overcome to achieve this goal: first, to expand the range of molecules that can be recognized by cellular “sensors” as inputs; and second, to better understand the genetic control factors that regulate gene expression in space and time so we can engineer better activator systems.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        At present we need pills because we can't swallow a chemistry kit”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "     <p>\n",
      "      Metabolomics is shedding light on many disease biomarkers. Because some biomarkers are shared between seemingly unrelated diseases, an accurate diagnosis will require the detection of multiple markers to provide a more unique “disease fingerprint.” Work in whole cell and cell free systems to develop sensors of multiple disease-specific biomarkers could assist in earlier detection of disease and prognostic monitoring.\n",
      "     </p>\n",
      "     <p>\n",
      "      To expand the range of biologically detectable molecules, it is possible to design metabolic pathways that transform currently undetectable molecules of interest (e.g., hippuric acid, the prostate cancer biomarker) into molecules for which sensors already exist (in this case benzoic acid) (Libis et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B27\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      Cybergenetics is an emerging field that is developing experimental tools for the computer control of cellular processes at the gene level in real time. Cybergenetic control can be achieved by interfacing living cells with a digital computer that switches on or off the embedded “genetic switch” using light (optogenetics) or chemicals (Gabriele et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B20\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ; Maysam et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B29\">\n",
      "       2019\n",
      "      </xref>\n",
      "      ). Such systems could help to maintain cellular homeostasis by monitoring the state of the body and triggering an appropriate response upon the detection of dysregulation; for example, they could trigger the release of insulin when blood glucose levels rise as detected by a wireless diagnostic tool (Ye et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B39\">\n",
      "       2011\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Advanced Materials: Inspired by Nature, Improved by Synthetic Biology\n",
      "     </title>\n",
      "     <p>\n",
      "      Synthetic biology offers the opportunity to create responsive and multifunctional materials (Le Feuvre and Scrutton,\n",
      "      <xref ref-type=\"bibr\" rid=\"B26\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ). The integration of biochemical components from living systems with inorganic components can lead to new materials that are able to sense the environment (or internal signals) and change their properties. These features could be particularly useful for improving protective clothing or building materials.\n",
      "     </p>\n",
      "     <p>\n",
      "      An issue when using microbes to produce composite materials is regulating the assembly of these materials to achieve specific desired properties. By understanding how microbes communicate with each other, it is possible to make them work better together and combine them with other production systems so that the properties of materials can be tailored for particular functions.\n",
      "     </p>\n",
      "     <p>\n",
      "      Interestingly, rather than modifying, or improving existing protein-based materials, an alternative approach involves using computational techniques to design completely novel proteins that self-assemble into predicted shapes (Ljubetič et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B28\">\n",
      "       2017\n",
      "      </xref>\n",
      "      ). Such “programmable” proteins open up even further opportunities for synthetic biology not only for materials science but also for medicine and chemistry.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        The tools are there, we just don't know what we want to make”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "    </sec>\n",
      "   </sec>\n",
      "   <sec id=\"s3\">\n",
      "    <title>\n",
      "     Tackling the Challenges\n",
      "    </title>\n",
      "    <p>\n",
      "     Regardless of the research areas involved, there are some common challenges for the community to address.\n",
      "    </p>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Design With the End in Mind\n",
      "     </title>\n",
      "     <p>\n",
      "      There was consensus that while the production of small molecules using synthetically engineered cells at the bench is becoming more tractable, these processes often do not translate well into mass production. Scalability needs to be incorporated into the initial design process by including features, for example, that reduce toxicity of the molecule to the production host or “chassis” and/or by introducing modifications that favor its extraction. Careful consideration of the right chassis could also greatly improve yields and significantly reduce cost of production.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        Nature has developed its own ways to concentrate and solubilise chemicals…we are not learning from this”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Expanding the Host Repertoire\n",
      "     </title>\n",
      "     <p>\n",
      "      The number of microbes that are currently in use for the production of synthetic biological materials is only a tiny fraction of the total diversity that exists in Nature; only ten microbes are “domesticated” for industrial use. To identify the most appropriate chassis, it is worth turning to nature to identify species that have unique metabolic networks suited to host particular types of chemical reactions. For example, the soil bacterium\n",
      "      <italic>\n",
      "       Pseudomonas putida\n",
      "      </italic>\n",
      "      , which has adapted to harsh environmental conditions, is ideally suited to host redox-intensive reactions (Pablo and de Lorenzo,\n",
      "      <xref ref-type=\"bibr\" rid=\"B30\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ). A treasure trove of, as yet, unexplored natural products, with novel and beneficial properties, exist in the plant kingdom. In addition, as noted above, plants are naturally excellent production hosts and it is now possible to “plug and play” combinations of plant pathways to generate novel molecules (Sainsbury and Lomonossoff,\n",
      "      <xref ref-type=\"bibr\" rid=\"B31\">\n",
      "       2014\n",
      "      </xref>\n",
      "      ; Evangelos and O'connor,\n",
      "      <xref ref-type=\"bibr\" rid=\"B17\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Developing a Universal Production System\n",
      "     </title>\n",
      "     <p>\n",
      "      To circumvent the problem of the impact of different host chassis on synthetic gene circuits, researchers would benefit from a universal synthetic expression system that permits the testing of new constructs. This would aid in the identification of optimal production platforms and decrease the need for organism-specific technologies. Combining this technology with DNA editing techniques, such as CRISPR/Cas9, will make the establishment of host production platforms and the generation of complex biosynthetic products easier and faster. One articulation of this could be in cell-free formats, in which the essential cellular machinery is reconstituted\n",
      "      <italic>\n",
      "       in vitro\n",
      "      </italic>\n",
      "      and used as a manufacturing platform (Villarreal and Tan,\n",
      "      <xref ref-type=\"bibr\" rid=\"B36\">\n",
      "       2017\n",
      "      </xref>\n",
      "      ; Koch et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B24\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        We need to reduce the burden of synthetic networks on endogenous circuits”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Move to Cell-Free Environments\n",
      "     </title>\n",
      "     <p>\n",
      "      Cell-free environments, interfaced with semiconductors, offer a powerful route for flexible and controllable production systems.\n",
      "     </p>\n",
      "     <p>\n",
      "      For example, nanoparticles made of semiconductor materials or quantum dots can be used to enhance enzyme activity in a cell free environment with a minimal set of ingredients. Multistep enzymatic pathways can be tethered to nanoparticle surfaces and, by avoiding the diffusion effect that takes place in cells, reaction rates can be increased 100-fold (Wang et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B38\">\n",
      "       2017\n",
      "      </xref>\n",
      "      ). This approach can be used to access non-natural materials and circumvents the potential issue of toxicity in cells (as well as the regulatory issues of genetically modified cells).\n",
      "     </p>\n",
      "     <p>\n",
      "      Research on silicon chips containing immobilized genes and cell lysates allows detailed examination of gene expression in space and time. When the compartments in which DNA-driven reactions take place are linked, with materials flowing into and diffusing between the compartments, it is possible to recreate oscillating protein expression patterns and protein gradients akin to those observed in cells (Karzbrun et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B23\">\n",
      "       2014\n",
      "      </xref>\n",
      "      ). Gene expression in these “artificial cells” can be controlled with electrodes that prevent the assembly of proteins by ribosomes. By standardizing outputs, this strategy is improving the predictability of engineered genetic circuits.\n",
      "     </p>\n",
      "    </sec>\n",
      "    <sec>\n",
      "     <title>\n",
      "      Systems Modeling, Standards, and Metrology\n",
      "     </title>\n",
      "     <p>\n",
      "      Whatever the system being explored, a robust model of that biological system is needed if predictable modifications of genetic circuits are to be designed and implemented with any confidence. To create such models, large sets of data arising from measuring multiple parameters of cell behavior under different conditions are essential (Fletcher et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B18\">\n",
      "       2016\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      There is great expectation that improvements in high-throughput data measurement and collection systems will generate exactly the large data sets needed. These can be analyzed using artificial intelligence or a machine learning approach to optimize the design of synthetic biological products and move away from the inefficient trial and error process (Decoene et al.,\n",
      "      <xref ref-type=\"bibr\" rid=\"B11\">\n",
      "       2018\n",
      "      </xref>\n",
      "      ).\n",
      "     </p>\n",
      "     <p>\n",
      "      Finally, agreement on standards of design, assembly, data transfer, data measurement and regulatory rules, as well as on the language that is used, will help to improve the interdisciplinary and international collaborations that are required to drive the field forward. This is challenging for a community with such diverse interests and perspectives, and where data sharing, curating and quality control is not common practice.\n",
      "     </p>\n",
      "     <p>\n",
      "      However, without some form of agreed standards, many of the products and processes of synthetic biology will not translate well to industrial settings dependent on reproducible processes and beholden to exacting regulatory requirements.\n",
      "     </p>\n",
      "     <p>\n",
      "      Typically, academic researchers are driven by a need to understand the complexity of nature (and publish their work in high impact journals). Standardization and scaling up production are important, but have less of an academic pull than discovering a new product. Access to funding, industrial partnerships, and academic recognition are examples of potential incentives for carrying out this type of research.\n",
      "     </p>\n",
      "     <disp-quote>\n",
      "      <p>\n",
      "       “\n",
      "       <italic>\n",
      "        Standards restrict flexibility but enable interoperability”\n",
      "       </italic>\n",
      "      </p>\n",
      "     </disp-quote>\n",
      "    </sec>\n",
      "   </sec>\n",
      "   <sec id=\"s4\">\n",
      "    <title>\n",
      "     Tackling Risk\n",
      "    </title>\n",
      "    <p>\n",
      "     Synthetic biology is an example of a dual-use technology: it promises numerous beneficial applications, but it can also cause harm. This has led to fears that it could, intentionally or unintentionally, harm humans or damage the environment. For example, there is huge value in our ability to engineer viruses to be more effective and specific shuttles for gene therapies of devastating inherited disorders; however, engineering viruses may also lead to the creation of even more deadly pathogens by those intent on harm.\n",
      "    </p>\n",
      "    <disp-quote>\n",
      "     <p>\n",
      "      “\n",
      "      <italic>\n",
      "       Synthetic biology should be regarded as an extension of earlier developments and technologies”\n",
      "      </italic>\n",
      "     </p>\n",
      "    </disp-quote>\n",
      "    <p>\n",
      "     Some would argue that synthetic biology poses an existential risk and needs to be treated with extreme caution. However, many new technological advances across the decades have met similar concerns. The uncertainty and remote possibility of such risks could hamper the development of useful technology. Scientists, their host institutions and funding bodies should (and indeed already do) consider whether the research planned could be misused. Measures that reduce the likelihood of misuse and its consequences should be implemented and clearly communicated. The synthetic biology community needs to be aware of, and respond to, these challenges by engaging in horizon scanning exercises as well as open dialogue with regulatory bodies and the media.\n",
      "    </p>\n",
      "    <disp-quote>\n",
      "     <p>\n",
      "      “\n",
      "      <italic>\n",
      "       Don't avoid risk – manage it”\n",
      "      </italic>\n",
      "     </p>\n",
      "    </disp-quote>\n",
      "    <p>\n",
      "     Being more open about risks, and how they are controlled, provides an opportunity to shift discourse toward the benefits of synthetic biology in addressing urgent global needs, such as the production of biofuels, food security and more effective medicines, and potentially improve public acceptance.\n",
      "    </p>\n",
      "    <disp-quote>\n",
      "     <p>\n",
      "      “\n",
      "      <italic>\n",
      "       The questions should not be ‘what’s the next big thing for synthetic biology' but ‘where is the greatest unmet need’.”\n",
      "      </italic>\n",
      "     </p>\n",
      "    </disp-quote>\n",
      "    <p>\n",
      "     Despite the efforts by individual countries to establish synthetic biology research roadmaps, broader, international agreement on common standards (and red lines) across the field may help establish trust and to advance the best pre-competitive research into useful applications.\n",
      "    </p>\n",
      "    <p>\n",
      "     Meeting participants highlighted the importance of training in responsible research conduct and ethics. Given students' future role as science ambassadors and influencers, their training should not only convey skills and knowledge but also awareness and critical thinking about the prospects and potential for dual use of synthetic biology. All researchers must remain vigilant regardless of the many pressures and distractions of running a successful research lab; they may not have specialist training in identifying the risks of misuse but they are the people best placed to maintain informed oversight of risks.\n",
      "    </p>\n",
      "    <p>\n",
      "     One example of current synthetic biology research with potential dual use is gene drive technology, which can be used to propagate a particular suite of genes throughout a population. The benefits of using gene drive technology include the eradication of disease-carrying insect populations and the elimination of invading pest species but it has raised concerns about the unintended ecological impacts of reducing or eliminating a population (Callaway,\n",
      "     <xref ref-type=\"bibr\" rid=\"B5\">\n",
      "      2018\n",
      "     </xref>\n",
      "     ; Collins,\n",
      "     <xref ref-type=\"bibr\" rid=\"B9\">\n",
      "      2018\n",
      "     </xref>\n",
      "     ).\n",
      "    </p>\n",
      "    <p>\n",
      "     Similar release concerns surround research that is harnessing the ability of pathogens to target particular tissues in the body or particular chemicals in the environment, which could greatly aid efforts to deliver targeted therapies or clean-up contaminated sites. To date, such large-scale release for environmental bioremediation interventions has not been possible.\n",
      "    </p>\n",
      "    <disp-quote>\n",
      "     <p>\n",
      "      “\n",
      "      <italic>\n",
      "       We need to mind the gap between R&amp;D scale up and communications …. One bad blog can kill a commercial product”\n",
      "      </italic>\n",
      "     </p>\n",
      "    </disp-quote>\n",
      "    <p>\n",
      "     There was consensus that the need for regulation over this community remains important. Regulation needs to keep up to speed with the emerging technologies and should focus on the product rather than the process used to create it (Tait et al.,\n",
      "     <xref ref-type=\"bibr\" rid=\"B34\">\n",
      "      2017\n",
      "     </xref>\n",
      "     ). Unsuitable regulatory frameworks (as well as unfavorable public perception) could discourage private sector investment in synthetic biology.\n",
      "    </p>\n",
      "    <p>\n",
      "     Open and balanced two-way communication between researchers, funders, companies and governments and the public will be vital. Consumers and activists may have no interest in the difference between making a chemical (e.g., a flavor or fragrance) synthetically and one made using genetically modified bacteria but they may instinctively distrust the latter.\n",
      "    </p>\n",
      "   </sec>\n",
      "   <sec id=\"s5\">\n",
      "    <title>\n",
      "     Conclusions and Recommendations\n",
      "    </title>\n",
      "    <p>\n",
      "     The short talks presented at this 2-day meeting suggest that synthetic biology is at the cusp of many major breakthroughs and that it is perhaps timely to re-define the meaning of “success” in synthetic biology.\n",
      "    </p>\n",
      "    <p>\n",
      "     There are many hurdles to overcome but the potential for synthetic biology to deliver solutions to many global challenges—improving healthcare, limiting environmental damage, and creating a wide variety of more sustainable processes—is great.\n",
      "    </p>\n",
      "    <p>\n",
      "     Meeting participants suggested that as a community they should support the measures listed below to help synthetic biology move beyond the proof-of-concept stage and to ensure that potential risks are minimized and dialog with the public can be optimized.\n",
      "    </p>\n",
      "    <list list-type=\"bullet\">\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Larger and longer investment in better big data management and processing (Artificial Intelligence and Machine Learning) systems, and in fundamental research on biosystems modeling, chassis development, and genome mining.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Support standardization initiatives; while arguably not attractive academically, the community needs to agree and support efforts toward creating interoperability around biosystem modeling, and standards around DNA design (if not DNA bioparts). Without some degree of “standardization” the ability to pool data and models, essential for improving accuracy and reproducibility, will be challenging.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       The creation and funding (ideally internationally) of a “Grand Challenge,” such as the development of generic sensors or the creation of protein-based electronic components, could help focus the community toward a target goal.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Improve the assessment, communication, and management of risk and harm among all audiences.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Ensure that early career researchers are trained in responsible research conduct and ethics as well as being cognizant of existing rules and regulations around GM (regionally dependent) and the issue of misuse and harm.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Coordinate the efforts of academia, government and industry through focused meetings that foster interdisciplinary collaborations around shared objectives.\n",
      "      </p>\n",
      "     </list-item>\n",
      "     <list-item>\n",
      "      <p>\n",
      "       Improve platforms for knowledge sharing and recognize the value of failures.\n",
      "      </p>\n",
      "     </list-item>\n",
      "    </list>\n",
      "    <p>\n",
      "     The workshop highlighted just how much more we have to learn from Nature itself. Synthetic biology is giving us insights of the criteria and processes that underpin all living systems; in turn, we can take this insight and design and use it to build a “better biology.” However, we need to take the public with us on this journey, create meaningful and considered dialogue about the work we may do and the impact it might have on our world.\n",
      "    </p>\n",
      "   </sec>\n",
      "   <sec id=\"s6\">\n",
      "    <title>\n",
      "     Author's Note\n",
      "    </title>\n",
      "    <p>\n",
      "     The meeting was hosted by the UK Center for Mammalian Synthetic Biology (the Center), based at the University of Edinburgh. The Center is building expertise in cell engineering tool generation, whole-cell modeling, computer-assisted design and construction of DNA and high-throughput phenotyping to enable synthetic biology for medicine and healthcare. The Centre's research will not only advance basic understanding of mammalian biology and pathology but also generate products and services for near-term commercial exploitation by the pharmaceutical and drug testing industries, such as diagnostics, novel therapeutics, protein-based drugs, and regenerative medicines. The Center is funded through the Research Council's UK “Synthetic Biology for Growth” programme and by the Biological and Biotechnology Sciences Research Council (BBSRC), the Engineering and Physical Sciences Research Council and the Medical Research Council.\n",
      "    </p>\n",
      "   </sec>\n",
      "   <sec id=\"s7\">\n",
      "    <title>\n",
      "     Author Contributions\n",
      "    </title>\n",
      "    <p>\n",
      "     This is a meeting report and was drafted by MH-F with support from ME and LF.\n",
      "    </p>\n",
      "   </sec>\n",
      "   <sec>\n",
      "    <title>\n",
      "     Conflict of Interest Statement\n",
      "    </title>\n",
      "    <p>\n",
      "     The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\n",
      "    </p>\n",
      "   </sec>\n",
      "  </body>\n",
      "  <back>\n",
      "   <ack>\n",
      "    <p>\n",
      "     We are grateful to all colleagues that contributed to discussions at the workshop held in Edinburgh in May 2018 and whose views helped inform this report. We also gratefully acknowledge sponsorship for this workshop from the Office of Naval Research Global and the UK Centre for Mammalian Synthetic Biology (BBSRC Grant BB/M018040/1).\n",
      "    </p>\n",
      "   </ack>\n",
      "   <ref-list>\n",
      "    <title>\n",
      "     References\n",
      "    </title>\n",
      "    <ref id=\"B1\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Boehm\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Bock\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Recent advances and current challenges in synthetic biology of the plastid genetic system and metabolism\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Plant Physiol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       179\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       794\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       802\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1104/pp.18.00767\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30181342\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B2\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Boeke\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J. D.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Church\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Hessel\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kelley\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         N. J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Arkin\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Cai\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Genome engineering: the genome project-write\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       8\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       126\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       127\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.aaf6850\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B3\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Bueso\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Y. F.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Tangney\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic biology in the driving seat of the bioeconomy\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Trends Biotechnol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       35\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       373\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       378\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.tibtech.2017.02.002\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       28249675\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B4\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Burkard\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Opriessnig\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Mileham\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Stadejek\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ait-Ali\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Lillico\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S. G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      . (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Pigs lacking the scavenger receptor cysteine-rich domain 5 of CD163 are resistant to porcine reproductive and respiratory syndrome virus 1 infection\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       J. Virol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       92\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       00415\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       00418\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1128/JVI.00415-18\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29925651\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B5\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Callaway\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Ban on ‘Gene Drives' is back on the UN's agenda - worrying scientists\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nature\n",
      "      </source>\n",
      "      <volume>\n",
      "       563\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       454\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       455\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1038/d41586-018-07436-4\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30459378\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B6\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Cameron\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         D. E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Bashor\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Collins\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J. J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2014\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       A brief history of synthetic biology\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nat. Rev. Microbiol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       12\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       381\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       390\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1038/nrmicro3239\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       24686414\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B7\">\n",
      "     <mixed-citation publication-type=\"webpage\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Carlson\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2014\n",
      "      </year>\n",
      "      ).\n",
      "      <source>\n",
      "       Time for New DNA Sequencing And Synthesis Cost Curves\n",
      "      </source>\n",
      "      . Available online at:\n",
      "      <ext-link ext-link-type=\"uri\" xlink:href=\"https://synbiobeta.com/time-new-dna-synthesis-sequencing-cost-curves-rob-carlson/\">\n",
      "       https://synbiobeta.com/time-new-dna-synthesis-sequencing-cost-curves-rob-carlson/\n",
      "      </ext-link>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B8\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Casini\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Chang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         F. Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Eluere\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         King\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Young\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Dudley\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Q. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      . (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       A pressure test to make 10 molecules in 90 days: external evaluation of methods to engineer biology\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       J Am Chem Soc.\n",
      "      </source>\n",
      "      <volume>\n",
      "       140\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       4302\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       4316\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1021/jacs.7b13292\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29480720\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B9\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Collins\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Gene drives in our future: challenges of and opportunities for using a self-sustaining technology in pest and vector management\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       BMC Proc.\n",
      "      </source>\n",
      "      <volume>\n",
      "       12\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       37\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       41\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1186/s12919-018-0110-4\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30263046\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B10\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Davies\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J. A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Cachat\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic biology meets tissue engineering\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Biochem. Soc. Transac.\n",
      "      </source>\n",
      "      <volume>\n",
      "       44\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       696\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       701\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1042/BST20150289\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       27284030\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B11\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Decoene\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         De Paepe\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         B.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Maertens\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Coussement\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Peters\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         de Maeseneire\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      . (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Standardization in synthetic biology: an engineering discipline coming of age\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Crit. Rev. Biotechnol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       38\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       647\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       656\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1080/07388551.2017.1380600\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       28954542\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B12\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Dirisala\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V. R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Nair\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R. R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Srirama\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         K.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Reddy\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P. N.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Rao\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         K. R. S. S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kumar\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P. G\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Recombinant pharmaceutical protein production in plants: unraveling the therapeutic potential of molecular pharming\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Acta Physiol. Plant\n",
      "      </source>\n",
      "      <volume>\n",
      "       39\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       18\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       26\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1007/s11738-016-2315-3\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B13\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Dolgin\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Scientists downsize bold plan to make human genome from Scratch\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nature\n",
      "      </source>\n",
      "      <volume>\n",
      "       557\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       16\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       17\n",
      "      </lpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1038/d41586-018-05043-x\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29717256\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B14\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Dou\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Bennett\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M. R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic biology and the gut microbiome\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Biotechnol J.\n",
      "      </source>\n",
      "      <volume>\n",
      "       13\n",
      "      </volume>\n",
      "      :\n",
      "      <fpage>\n",
      "       1700159\n",
      "      </fpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1002/biot.201700159\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       28976641\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B15\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Dunbar\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         High\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         K. A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Joung\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J. K.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kohn\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         D. B.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ozawa\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         K.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Sadelain\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Gene therapy comes of age\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       359\n",
      "      </volume>\n",
      "      :\n",
      "      <fpage>\n",
      "       6175\n",
      "      </fpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.aan4672\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29326244\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B16\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Emmanuel\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Chapman\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Williamson\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. L.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Rybicki\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E. P.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Meyers\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Production of complex viral glycoproteins in plants as vaccine immunogens\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Plant Biotechnol. J.\n",
      "      </source>\n",
      "      <volume>\n",
      "       16\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1531\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       1545\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1111/pbi.12963\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B17\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Evangelos\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T. C.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         O'connor\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S. E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       New developments in engineering plant metabolic pathways\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Curr. Opin. Biotechnol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       42\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       126\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       132\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.copbio.2016.04.012\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       27132124\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B18\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Fletcher\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Krivoruchko\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Nielsen\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Industrial systems biology and its impact on synthetic biology of yeast cell factories\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Biotechnol. Bioeng.\n",
      "      </source>\n",
      "      <volume>\n",
      "       113\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1164\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       1170\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1002/bit.25870\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       26524089\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B19\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         French\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         K. E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Harnessing synthetic biology for sustainable development\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nat. Sustain.\n",
      "      </source>\n",
      "      <volume>\n",
      "       2\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       250\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       252\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1038/s41893-019-0270-x\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B20\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Gabriele\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         L.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Benenson\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Khammash\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic control systems for high performance gene expression in mammalian cells\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nucleic Acids Res.\n",
      "      </source>\n",
      "      <volume>\n",
      "       46\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       9855\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       9863\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1093/nar/gky795\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30203050\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B21\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Gnanasekaran\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Karcher\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         D.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Nielsen\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. Z.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Martens\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         H. J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ruf\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Olsen\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      . (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Transfer of the cytochrome P450-dependent dhurrin pathway from sorghum bicolor into\n",
      "       <italic>\n",
      "        Nicotiana tabacum\n",
      "       </italic>\n",
      "       chloroplasts for light-driven synthesis\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       J. Exp. Botany\n",
      "      </source>\n",
      "      <volume>\n",
      "       67\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       2495\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       2506\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1093/jxb/erw067\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       26969746\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B22\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         June\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C. H.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         O'Connor\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R. S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kawalekar\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         O. U.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ghassemi\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Milone\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M. C.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      . (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       CAR T cell immunotherapy for human cancer\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       359\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1361\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       1365\n",
      "      </lpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.aar6711\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29567707\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B23\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Karzbrun\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Tayar\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Noireaux\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Bar-Ziv\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R. H.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2014\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic biology. Programmable On-chip DNA compartments as artificial cells\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       345\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       829\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       832\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.1255550\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       25124443\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B24\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Koch\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Faulon\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.-L.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Borkowski\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         O.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Models for cell-free synthetic biology: make prototyping easier, better, and faster\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Front. Bioeng. Biotech.\n",
      "      </source>\n",
      "      <volume>\n",
      "       6\n",
      "      </volume>\n",
      "      :\n",
      "      <fpage>\n",
      "       182\n",
      "      </fpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.3389/fbioe.2018.00182\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30555825\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B25\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kosuri\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Church\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2014\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Large-scale\n",
      "       <italic>\n",
      "        de novo\n",
      "       </italic>\n",
      "       DNA synthesis: technologies and applications\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Nat. Methods\n",
      "      </source>\n",
      "      <volume>\n",
      "       11\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       499\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       507\n",
      "      </lpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1038/nmeth.2918\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       24781323\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B26\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Le Feuvre\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Scrutton\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         N.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       A living foundry for synthetic biological materials: a synthetic biology roadmap to new advanced materials\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Synthet. Syst. Biotechnol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       3\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       105\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       112\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.synbio.2018.04.002\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29900423\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B27\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Libis\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Baudoin\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         D.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Faulon\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J. L.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2016\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Expanding biosensing abilities through computer-aided design of metabolic pathways\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       ACS Synthet. Biol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       5\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1076\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       1085\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1021/acssynbio.5b00225\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       27028723\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B28\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ljubetič\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Gradišar\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         H.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Jerala\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Advances in design of protein folds and assemblies\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Curr. Opin. Chem. Biol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       40\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       65\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       71\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.cbpa.2017.06.020\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       28709120\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B29\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Maysam\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Strittmatter\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Fussenegger\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Light-controlled mammalian cells and their therapeutic applications in synthetic biology\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Adv. Sci.\n",
      "      </source>\n",
      "      <volume>\n",
      "       6\n",
      "      </volume>\n",
      "      :\n",
      "      <fpage>\n",
      "       1800952\n",
      "      </fpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1002/advs.201800952\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B30\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Pablo\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         N.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         de Lorenzo\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       <italic>\n",
      "        Pseudomonas putida\n",
      "       </italic>\n",
      "       as a functional chassis for industrial biocatalysis: from native biochemistry to trans-metabolism\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Metabol. Eng.\n",
      "      </source>\n",
      "      <volume>\n",
      "       50\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       142\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       155\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.ymben.2018.05.005\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B31\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Sainsbury\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         F.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Lomonossoff\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2014\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Transient expressions of synthetic biology in plants\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Curr. Opin. Plant Biol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       19\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       7\n",
      "      </lpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.pbi.2014.02.003\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       24631883\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B32\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Satoshi\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         T.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Blauch\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         L. R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Tang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         S. K. Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Morsut\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         L.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Lim\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         W.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Programming self-organizing multicellular structures with synthetic cell-cell signaling\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       361\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       156\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       162\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.aat0271\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       29853554\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B33\">\n",
      "     <mixed-citation publication-type=\"webpage\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Sauro\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         H. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Hare\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         D.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Kwiatkowska\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Uhrmacher\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Hucka\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Mendes\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <etal/>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2006\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Challenges for modeling and simulation methods in systems biology\n",
      "      </article-title>\n",
      "      , in\n",
      "      <source>\n",
      "       Proceedings of the 2006 Winter Simulation Conference\n",
      "      </source>\n",
      "      (\n",
      "      <year>\n",
      "       2006\n",
      "      </year>\n",
      "      ). Available online at:\n",
      "      <ext-link ext-link-type=\"uri\" xlink:href=\"http://qav.comlab.ox.ac.uk/papers/winter_sim06_panel.pdf\">\n",
      "       http://qav.comlab.ox.ac.uk/papers/winter_sim06_panel.pdf\n",
      "      </ext-link>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B34\">\n",
      "     <mixed-citation publication-type=\"webpage\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Tait\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         G.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Banda\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Watkins\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <source>\n",
      "       Proportionate and Adaptive Governance of Innovative Technologies (PAGIT): A Framework to Guide Policy and Regulatory Decision Making. Innogen Institute\n",
      "      </source>\n",
      "      . Available online at:\n",
      "      <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.innogen.ac.uk/reports/1222\">\n",
      "       https://www.innogen.ac.uk/reports/1222\n",
      "      </ext-link>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B35\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Teixeira\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Fussenegger\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Engineering mammalian cells for disease diagnosis and treatment\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Curr. Opin. Biotechnol.\n",
      "      </source>\n",
      "      <volume>\n",
      "       55\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       87\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       94\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.copbio.2018.08.008\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30195188\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B36\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Villarreal\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         F.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Tan\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Cell-free systems in the new age of synthetic biology\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Front. Chem. Sci. Eng.\n",
      "      </source>\n",
      "      <volume>\n",
      "       11\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       58\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       65\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1007/s11705-017-1610-x\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B37\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Wang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         F.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Zhang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         W.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2019\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Synthetic biology: recent progress, biosafety and biosecurity concerns, and possible solutions\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       J. Biosafety Biosecur.\n",
      "      </source>\n",
      "      <volume>\n",
      "       1\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       22\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       30\n",
      "      </lpage>\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.jobb.2018.12.003\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B38\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Wang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         P.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Chang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         A. Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Novosad\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Chupin\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         V. V.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Schaller\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R. D.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Rozhkova\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         E. A.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2017\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Cell-free synthetic biology chassis for nanocatalytic photon-to-hydrogen conversion\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       ACS Nano\n",
      "      </source>\n",
      "      .\n",
      "      <volume>\n",
      "       11\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       6739\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       6745\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1021/acsnano.7b01142\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       28602073\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B39\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Ye\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         H.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Daoud-El Baba\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Peng\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.-W.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Fussenegger\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         M.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2011\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       A Synthetic optogenetic transcription device enhances blood-glucose homeostasis in mice\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Science\n",
      "      </source>\n",
      "      <volume>\n",
      "       332\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       1565\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       1568\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1126/science.1203535\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       21700876\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "    <ref id=\"B40\">\n",
      "     <mixed-citation publication-type=\"journal\">\n",
      "      <person-group person-group-type=\"author\">\n",
      "       <name>\n",
      "        <surname>\n",
      "         Zhang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         R.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Li\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         C.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Wang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         J.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Yang\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "       <name>\n",
      "        <surname>\n",
      "         Yan\n",
      "        </surname>\n",
      "        <given-names>\n",
      "         Y.\n",
      "        </given-names>\n",
      "       </name>\n",
      "      </person-group>\n",
      "      (\n",
      "      <year>\n",
      "       2018\n",
      "      </year>\n",
      "      ).\n",
      "      <article-title>\n",
      "       Microbial production of small medicinal molecules and biologics: from nature to synthetic pathways\n",
      "      </article-title>\n",
      "      .\n",
      "      <source>\n",
      "       Biotechnol Adv\n",
      "      </source>\n",
      "      .\n",
      "      <volume>\n",
      "       36\n",
      "      </volume>\n",
      "      ,\n",
      "      <fpage>\n",
      "       2219\n",
      "      </fpage>\n",
      "      –\n",
      "      <lpage>\n",
      "       2231\n",
      "      </lpage>\n",
      "      .\n",
      "      <pub-id pub-id-type=\"doi\">\n",
      "       10.1016/j.biotechadv.2018.10.009\n",
      "      </pub-id>\n",
      "      <pub-id pub-id-type=\"pmid\">\n",
      "       30385278\n",
      "      </pub-id>\n",
      "     </mixed-citation>\n",
      "    </ref>\n",
      "   </ref-list>\n",
      "  </back>\n",
      " </article>\n",
      "</pmc-articleset>\n"
     ]
    }
   ],
   "source": [
    "from bs4 import BeautifulSoup\n",
    "infile = open(\"./ethics/pmc6692427.xml\",\"r\")\n",
    "contents = infile.read()\n",
    "soup = BeautifulSoup(contents,'xml')\n",
    "print(soup.prettify())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding Title"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Future Trends in Synthetic Biology—A Report'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "title = soup.find('article-title').text\n",
    "title"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding Authors"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['El Karoui M', 'Hoyos-Flight M', 'Fletcher L']"
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Entrez.read(Entrez.esummary(db='pmc', id='6692427'))[0]['AuthorList']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding Abstract"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'NoneType' object has no attribute 'text'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-56-4f0069208991>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mabstract\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msoup\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'abstract'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtext\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0mabstract\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'text'"
     ]
    }
   ],
   "source": [
    "abstract = soup.find('abstract').text\n",
    "abstract"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding keywords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['synthetic biology',\n",
       " 'biosystem',\n",
       " 'future trends and developments',\n",
       " 'biodesign automation',\n",
       " 'responsible research and innovation (RRI)']"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "key = soup.find('kwd-group').find_all('kwd')\n",
    "key_words = [k.text for k in key]\n",
    "key_words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding Body"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "body = soup.find('body')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Synthetic Biology offers innovative approaches for engineering new biological systems or re-designing existing ones for useful purposes (see Figure 1). It has been described as a disruptive technology at the heart of the so-called Bioeconomy, capable of delivering new solutions to global healthcare, agriculture, manufacturing, and environmental challenges (Cameron et al., 2014; Bueso and Tangney, 2017; French, 2019). However, despite successes in the production of some high value chemicals and drugs, there is a perception that synthetic biology is still not yet delivering on its promise.',\n",
       " 'Synthetic biology is developing into a biodesign platform where it will be possible to apply the “design-build-test-iterate (or deploy)” to predictably create cells or organisms able to produce a wide variety of novel molecules, materials or even cells for multiple applications.',\n",
       " 'Moreover, there are some concerns from governments that synthetic biology expands the pool of agents of concern, which increases the need to develop detection, identification and monitoring systems, and proactively build countermeasures against chemical and biological threats (Wang and Zhang, 2019). The participation of representatives from various government organizations at this meeting is testament to their commitment to maintaining an active dialogue with the synthetic biology community. In this way, they aim to keep abreast of the changing nature of threats and provide the best advice to government about investment in science and technology and the introduction or amendment of regulatory processes.',\n",
       " 'The cost of DNA sequencing and synthesis have decreased dramatically (Carlson, 2014; Kosuri and Church, 2014) and we have access to more genetic information and more powerful genetic engineering capabilities than ever before. Critical investments in infrastructure are bearing fruit and, as is described below, synthetic biology is increasingly becoming, at least part of, the solution to many of our present and future needs in medicine, food and energy production, remediation, manufacturing, and national security. So what is the potential of synthetic biology and what challenges does it still face to realize this?',\n",
       " 'Despite the lack of predictability in biology, and current technical constraints that limit data collection and analyses, we can now produce small molecules on demand using synthetic biology approaches.',\n",
       " \"Probably the most impressive examples come from the Foundry at the Broad Institute of MIT and Harvard. When the Defense Advanced Research Projects Agency (DARPA) put the MIT-Broad Institute Foundry's design capabilities to the test, its researchers were able to deliver 6 out of 10 molecules of interest to the US Department of Defense in 90 days. This “pressure test” confirms the potential of synthetic biology to address shortages of key compounds quickly (Casini et al., 2018).\",\n",
       " 'Indeed, many labs can now design and construct relatively complex gene networks capable of generating a wide variety of “designer” molecules in a range of host cells; however, this is often a slow iterative process of trial and error.',\n",
       " 'As yet, very few small molecules in medicine are manufactured using a synthetic biology process; it remains very difficult to engineer microbes to carry out processes that Nature did not intend. This is to be expected: the performance of microbes is “good enough” from an evolutionary perspective. Microbes evolved to address the specific needs and challenges of their natural environments not those of industrial fermenters and bioreactors. Gene Transfer from one system to another may sound easy but in practice is hard work and rarely generates sufficient reward (i.e., increased yield) to justify the investment made. The application of automation and artificial intelligence (e.g., in designing and building plasmids) may help to reduce the time and cost—and improve return on investment—in the future (Zhang et al., 2018).',\n",
       " '“Scale up is product specific – we need more synthetic biology in the production process”',\n",
       " 'Plants make alternative production platforms. Improvements in mining plant genomes and the development of effective transient expression systems have enabled large-scale production of, for example, vaccines in tobacco plants in just a few weeks (Dirisala et al., 2017; Emmanuel et al., 2018). Directing the production of synthetic biological materials to plant chloroplasts also shows promise (Boehm and Bock, 2019).',\n",
       " 'The photosynthetic reducing power generated in plant chloroplasts can be harnessed for the light-driven synthesis of bioactive molecules such as dhurrin, which protects plants against insects (Gnanasekaran et al., 2016).',\n",
       " 'However, underlying all these platforms is a knowledge gap in our understanding in how nature works. This makes it very hard to apply the design/build/test/learn cycles used in conventional engineering to the production of synthetic biological materials whatever the production platform (yeast, bacteria, plants, or human cells) if the platform itself is not well-understood (Sauro et al., 2006).',\n",
       " 'What we need now are instruments able to measure and characterize outputs, assisted by progress in robotics and automation, and the application of machine learning approaches to analyse the data generated. This will help us to generate more robust models of biological systems, so we can improve experimental design for future engineering strategies.',\n",
       " '“We can do ‘build’. ‘Test’ is the challenge when we want to learn from the iterative design process”',\n",
       " 'Synthetic biology is driving significant advances in biomedicine, which will lead to transformational improvements in healthcare. Already, patients are benefiting from so-called CAR (for chimeric antigen receptor) technology, which engineers the immune cells (T-cells) of the patient to recognize and attack cancer cells (June et al., 2018).',\n",
       " 'Genetically engineered viruses are being used to correct defective genes in patients with inherited diseases such as Severe Combined Immune Deficiency (SCID) or epidermolysis bullosa (Dunbar et al., 2018).',\n",
       " \"The ability to reprogramme somatic cells from patients into induced pluripotent stem cells is furthering our understanding of their disease, reducing the use of animals in research, and paving the way for the development of personalized medicines and cell therapies. In principle at least, we could engineer a patient's own cells to multiply, differentiate into different cell types and even self-assemble into new tissues, or even organs, to repair those damaged through disease or injury (Davies and Cachat, 2016; Satoshi et al., 2018).\",\n",
       " 'Work on new vectors that are able to deliver large genetic loads to target tissues is helping to produce more efficient therapeutics and vaccines that will have fewer side effects and a smaller risk of resistance. Furthermore, optimizing antibody or vaccine production, or example, so that they are in an edible format (e.g., plant based), could greatly reduce the cost and increase the speed of vaccine production in an epidemic.',\n",
       " \"“We have the tools but need the creativity to make stuff that can't be made without synthetic biology”\",\n",
       " 'In the next few years, genetically engineering pigs to be virus resistant and have human-like immune profiles could make xenotransplantation a clinical reality (Burkard et al., 2018). Engineering the microbiome is expected to lead to the development of synthetic probiotics (Dou and Bennett, 2018).',\n",
       " 'The synthetic biology initiative known as Human Genome Project-write (HGP-write) has set its sights even higher, rallying scientists to build entire human chromosomes (Boeke et al., 2016). Concerns have been raised about the ethics of creating “synthetic humans” and indeed the scientific and commercial value of such a project. More recently, HGP-write champions have proposed a more focused project to build a virus-resistant chromosome, making at least 400,000 changes to the human genome to remove DNA sequences that viruses use to hijack cells and replicate (Dolgin, 2018).',\n",
       " 'One of the many exciting opportunities that synthetic biology offers medicine is in the production of theranostic cell lines that can sense a disease state and produce an appropriate therapeutic response (Teixeira and Fussenegger, 2019). Several obstacles need to be overcome to achieve this goal: first, to expand the range of molecules that can be recognized by cellular “sensors” as inputs; and second, to better understand the genetic control factors that regulate gene expression in space and time so we can engineer better activator systems.',\n",
       " \"“At present we need pills because we can't swallow a chemistry kit”\",\n",
       " 'Metabolomics is shedding light on many disease biomarkers. Because some biomarkers are shared between seemingly unrelated diseases, an accurate diagnosis will require the detection of multiple markers to provide a more unique “disease fingerprint.” Work in whole cell and cell free systems to develop sensors of multiple disease-specific biomarkers could assist in earlier detection of disease and prognostic monitoring.',\n",
       " 'To expand the range of biologically detectable molecules, it is possible to design metabolic pathways that transform currently undetectable molecules of interest (e.g., hippuric acid, the prostate cancer biomarker) into molecules for which sensors already exist (in this case benzoic acid) (Libis et al., 2016).',\n",
       " 'Cybergenetics is an emerging field that is developing experimental tools for the computer control of cellular processes at the gene level in real time. Cybergenetic control can be achieved by interfacing living cells with a digital computer that switches on or off the embedded “genetic switch” using light (optogenetics) or chemicals (Gabriele et al., 2018; Maysam et al., 2019). Such systems could help to maintain cellular homeostasis by monitoring the state of the body and triggering an appropriate response upon the detection of dysregulation; for example, they could trigger the release of insulin when blood glucose levels rise as detected by a wireless diagnostic tool (Ye et al., 2011).',\n",
       " 'Synthetic biology offers the opportunity to create responsive and multifunctional materials (Le Feuvre and Scrutton, 2018). The integration of biochemical components from living systems with inorganic components can lead to new materials that are able to sense the environment (or internal signals) and change their properties. These features could be particularly useful for improving protective clothing or building materials.',\n",
       " 'An issue when using microbes to produce composite materials is regulating the assembly of these materials to achieve specific desired properties. By understanding how microbes communicate with each other, it is possible to make them work better together and combine them with other production systems so that the properties of materials can be tailored for particular functions.',\n",
       " 'Interestingly, rather than modifying, or improving existing protein-based materials, an alternative approach involves using computational techniques to design completely novel proteins that self-assemble into predicted shapes (Ljubetič et al., 2017). Such “programmable” proteins open up even further opportunities for synthetic biology not only for materials science but also for medicine and chemistry.',\n",
       " \"“The tools are there, we just don't know what we want to make”\",\n",
       " 'Regardless of the research areas involved, there are some common challenges for the community to address.',\n",
       " 'There was consensus that while the production of small molecules using synthetically engineered cells at the bench is becoming more tractable, these processes often do not translate well into mass production. Scalability needs to be incorporated into the initial design process by including features, for example, that reduce toxicity of the molecule to the production host or “chassis” and/or by introducing modifications that favor its extraction. Careful consideration of the right chassis could also greatly improve yields and significantly reduce cost of production.',\n",
       " '“Nature has developed its own ways to concentrate and solubilise chemicals…we are not learning from this”',\n",
       " \"The number of microbes that are currently in use for the production of synthetic biological materials is only a tiny fraction of the total diversity that exists in Nature; only ten microbes are “domesticated” for industrial use. To identify the most appropriate chassis, it is worth turning to nature to identify species that have unique metabolic networks suited to host particular types of chemical reactions. For example, the soil bacterium Pseudomonas putida, which has adapted to harsh environmental conditions, is ideally suited to host redox-intensive reactions (Pablo and de Lorenzo, 2018). A treasure trove of, as yet, unexplored natural products, with novel and beneficial properties, exist in the plant kingdom. In addition, as noted above, plants are naturally excellent production hosts and it is now possible to “plug and play” combinations of plant pathways to generate novel molecules (Sainsbury and Lomonossoff, 2014; Evangelos and O'connor, 2016).\",\n",
       " 'To circumvent the problem of the impact of different host chassis on synthetic gene circuits, researchers would benefit from a universal synthetic expression system that permits the testing of new constructs. This would aid in the identification of optimal production platforms and decrease the need for organism-specific technologies. Combining this technology with DNA editing techniques, such as CRISPR/Cas9, will make the establishment of host production platforms and the generation of complex biosynthetic products easier and faster. One articulation of this could be in cell-free formats, in which the essential cellular machinery is reconstituted in vitro and used as a manufacturing platform (Villarreal and Tan, 2017; Koch et al., 2018).',\n",
       " '“We need to reduce the burden of synthetic networks on endogenous circuits”',\n",
       " 'Cell-free environments, interfaced with semiconductors, offer a powerful route for flexible and controllable production systems.',\n",
       " 'For example, nanoparticles made of semiconductor materials or quantum dots can be used to enhance enzyme activity in a cell free environment with a minimal set of ingredients. Multistep enzymatic pathways can be tethered to nanoparticle surfaces and, by avoiding the diffusion effect that takes place in cells, reaction rates can be increased 100-fold (Wang et al., 2017). This approach can be used to access non-natural materials and circumvents the potential issue of toxicity in cells (as well as the regulatory issues of genetically modified cells).',\n",
       " 'Research on silicon chips containing immobilized genes and cell lysates allows detailed examination of gene expression in space and time. When the compartments in which DNA-driven reactions take place are linked, with materials flowing into and diffusing between the compartments, it is possible to recreate oscillating protein expression patterns and protein gradients akin to those observed in cells (Karzbrun et al., 2014). Gene expression in these “artificial cells” can be controlled with electrodes that prevent the assembly of proteins by ribosomes. By standardizing outputs, this strategy is improving the predictability of engineered genetic circuits.',\n",
       " 'Whatever the system being explored, a robust model of that biological system is needed if predictable modifications of genetic circuits are to be designed and implemented with any confidence. To create such models, large sets of data arising from measuring multiple parameters of cell behavior under different conditions are essential (Fletcher et al., 2016).',\n",
       " 'There is great expectation that improvements in high-throughput data measurement and collection systems will generate exactly the large data sets needed. These can be analyzed using artificial intelligence or a machine learning approach to optimize the design of synthetic biological products and move away from the inefficient trial and error process (Decoene et al., 2018).',\n",
       " 'Finally, agreement on standards of design, assembly, data transfer, data measurement and regulatory rules, as well as on the language that is used, will help to improve the interdisciplinary and international collaborations that are required to drive the field forward. This is challenging for a community with such diverse interests and perspectives, and where data sharing, curating and quality control is not common practice.',\n",
       " 'However, without some form of agreed standards, many of the products and processes of synthetic biology will not translate well to industrial settings dependent on reproducible processes and beholden to exacting regulatory requirements.',\n",
       " 'Typically, academic researchers are driven by a need to understand the complexity of nature (and publish their work in high impact journals). Standardization and scaling up production are important, but have less of an academic pull than discovering a new product. Access to funding, industrial partnerships, and academic recognition are examples of potential incentives for carrying out this type of research.',\n",
       " '“Standards restrict flexibility but enable interoperability”',\n",
       " 'Synthetic biology is an example of a dual-use technology: it promises numerous beneficial applications, but it can also cause harm. This has led to fears that it could, intentionally or unintentionally, harm humans or damage the environment. For example, there is huge value in our ability to engineer viruses to be more effective and specific shuttles for gene therapies of devastating inherited disorders; however, engineering viruses may also lead to the creation of even more deadly pathogens by those intent on harm.',\n",
       " '“Synthetic biology should be regarded as an extension of earlier developments and technologies”',\n",
       " 'Some would argue that synthetic biology poses an existential risk and needs to be treated with extreme caution. However, many new technological advances across the decades have met similar concerns. The uncertainty and remote possibility of such risks could hamper the development of useful technology. Scientists, their host institutions and funding bodies should (and indeed already do) consider whether the research planned could be misused. Measures that reduce the likelihood of misuse and its consequences should be implemented and clearly communicated. The synthetic biology community needs to be aware of, and respond to, these challenges by engaging in horizon scanning exercises as well as open dialogue with regulatory bodies and the media.',\n",
       " \"“Don't avoid risk – manage it”\",\n",
       " 'Being more open about risks, and how they are controlled, provides an opportunity to shift discourse toward the benefits of synthetic biology in addressing urgent global needs, such as the production of biofuels, food security and more effective medicines, and potentially improve public acceptance.',\n",
       " \"“The questions should not be ‘what’s the next big thing for synthetic biology' but ‘where is the greatest unmet need’.”\",\n",
       " 'Despite the efforts by individual countries to establish synthetic biology research roadmaps, broader, international agreement on common standards (and red lines) across the field may help establish trust and to advance the best pre-competitive research into useful applications.',\n",
       " \"Meeting participants highlighted the importance of training in responsible research conduct and ethics. Given students' future role as science ambassadors and influencers, their training should not only convey skills and knowledge but also awareness and critical thinking about the prospects and potential for dual use of synthetic biology. All researchers must remain vigilant regardless of the many pressures and distractions of running a successful research lab; they may not have specialist training in identifying the risks of misuse but they are the people best placed to maintain informed oversight of risks.\",\n",
       " 'One example of current synthetic biology research with potential dual use is gene drive technology, which can be used to propagate a particular suite of genes throughout a population. The benefits of using gene drive technology include the eradication of disease-carrying insect populations and the elimination of invading pest species but it has raised concerns about the unintended ecological impacts of reducing or eliminating a population (Callaway, 2018; Collins, 2018).',\n",
       " 'Similar release concerns surround research that is harnessing the ability of pathogens to target particular tissues in the body or particular chemicals in the environment, which could greatly aid efforts to deliver targeted therapies or clean-up contaminated sites. To date, such large-scale release for environmental bioremediation interventions has not been possible.',\n",
       " '“We need to mind the gap between R&D scale up and communications …. One bad blog can kill a commercial product”',\n",
       " 'There was consensus that the need for regulation over this community remains important. Regulation needs to keep up to speed with the emerging technologies and should focus on the product rather than the process used to create it (Tait et al., 2017). Unsuitable regulatory frameworks (as well as unfavorable public perception) could discourage private sector investment in synthetic biology.',\n",
       " 'Open and balanced two-way communication between researchers, funders, companies and governments and the public will be vital. Consumers and activists may have no interest in the difference between making a chemical (e.g., a flavor or fragrance) synthetically and one made using genetically modified bacteria but they may instinctively distrust the latter.',\n",
       " 'The short talks presented at this 2-day meeting suggest that synthetic biology is at the cusp of many major breakthroughs and that it is perhaps timely to re-define the meaning of “success” in synthetic biology.',\n",
       " 'There are many hurdles to overcome but the potential for synthetic biology to deliver solutions to many global challenges—improving healthcare, limiting environmental damage, and creating a wide variety of more sustainable processes—is great.',\n",
       " 'Meeting participants suggested that as a community they should support the measures listed below to help synthetic biology move beyond the proof-of-concept stage and to ensure that potential risks are minimized and dialog with the public can be optimized.',\n",
       " 'Larger and longer investment in better big data management and processing (Artificial Intelligence and Machine Learning) systems, and in fundamental research on biosystems modeling, chassis development, and genome mining.',\n",
       " 'Support standardization initiatives; while arguably not attractive academically, the community needs to agree and support efforts toward creating interoperability around biosystem modeling, and standards around DNA design (if not DNA bioparts). Without some degree of “standardization” the ability to pool data and models, essential for improving accuracy and reproducibility, will be challenging.',\n",
       " 'The creation and funding (ideally internationally) of a “Grand Challenge,” such as the development of generic sensors or the creation of protein-based electronic components, could help focus the community toward a target goal.',\n",
       " 'Improve the assessment, communication, and management of risk and harm among all audiences.',\n",
       " 'Ensure that early career researchers are trained in responsible research conduct and ethics as well as being cognizant of existing rules and regulations around GM (regionally dependent) and the issue of misuse and harm.',\n",
       " 'Coordinate the efforts of academia, government and industry through focused meetings that foster interdisciplinary collaborations around shared objectives.',\n",
       " 'Improve platforms for knowledge sharing and recognize the value of failures.',\n",
       " 'The workshop highlighted just how much more we have to learn from Nature itself. Synthetic biology is giving us insights of the criteria and processes that underpin all living systems; in turn, we can take this insight and design and use it to build a “better biology.” However, we need to take the public with us on this journey, create meaningful and considered dialogue about the work we may do and the impact it might have on our world.',\n",
       " \"The meeting was hosted by the UK Center for Mammalian Synthetic Biology (the Center), based at the University of Edinburgh. The Center is building expertise in cell engineering tool generation, whole-cell modeling, computer-assisted design and construction of DNA and high-throughput phenotyping to enable synthetic biology for medicine and healthcare. The Centre's research will not only advance basic understanding of mammalian biology and pathology but also generate products and services for near-term commercial exploitation by the pharmaceutical and drug testing industries, such as diagnostics, novel therapeutics, protein-based drugs, and regenerative medicines. The Center is funded through the Research Council's UK “Synthetic Biology for Growth” programme and by the Biological and Biotechnology Sciences Research Council (BBSRC), the Engineering and Physical Sciences Research Council and the Medical Research Council.\",\n",
       " 'This is a meeting report and was drafted by MH-F with support from ME and LF.',\n",
       " 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.']"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "body_text = [i.text for i in body.find_all('p')]\n",
    "body_text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Synthetic Biology offers innovative approaches for engineering new biological systems or re-designing existing ones for useful purposes (see Figure 1). It has been described as a disruptive technology at the heart of the so-called Bioeconomy, capable of delivering new solutions to global healthcare, agriculture, manufacturing, and environmental challenges (Cameron et al., 2014; Bueso and Tangney, 2017; French, 2019). However, despite successes in the production of some high value chemicals and drugs, there is a perception that synthetic biology is still not yet delivering on its promise.\n",
      "Synthetic biology is developing into a biodesign platform where it will be possible to apply the “design-build-test-iterate (or deploy)” to predictably create cells or organisms able to produce a wide variety of novel molecules, materials or even cells for multiple applications.\n",
      "Moreover, there are some concerns from governments that synthetic biology expands the pool of agents of concern, which increases the need to develop detection, identification and monitoring systems, and proactively build countermeasures against chemical and biological threats (Wang and Zhang, 2019). The participation of representatives from various government organizations at this meeting is testament to their commitment to maintaining an active dialogue with the synthetic biology community. In this way, they aim to keep abreast of the changing nature of threats and provide the best advice to government about investment in science and technology and the introduction or amendment of regulatory processes.\n",
      "The cost of DNA sequencing and synthesis have decreased dramatically (Carlson, 2014; Kosuri and Church, 2014) and we have access to more genetic information and more powerful genetic engineering capabilities than ever before. Critical investments in infrastructure are bearing fruit and, as is described below, synthetic biology is increasingly becoming, at least part of, the solution to many of our present and future needs in medicine, food and energy production, remediation, manufacturing, and national security. So what is the potential of synthetic biology and what challenges does it still face to realize this?\n",
      "Despite the lack of predictability in biology, and current technical constraints that limit data collection and analyses, we can now produce small molecules on demand using synthetic biology approaches.\n",
      "Probably the most impressive examples come from the Foundry at the Broad Institute of MIT and Harvard. When the Defense Advanced Research Projects Agency (DARPA) put the MIT-Broad Institute Foundry's design capabilities to the test, its researchers were able to deliver 6 out of 10 molecules of interest to the US Department of Defense in 90 days. This “pressure test” confirms the potential of synthetic biology to address shortages of key compounds quickly (Casini et al., 2018).\n",
      "Indeed, many labs can now design and construct relatively complex gene networks capable of generating a wide variety of “designer” molecules in a range of host cells; however, this is often a slow iterative process of trial and error.\n",
      "As yet, very few small molecules in medicine are manufactured using a synthetic biology process; it remains very difficult to engineer microbes to carry out processes that Nature did not intend. This is to be expected: the performance of microbes is “good enough” from an evolutionary perspective. Microbes evolved to address the specific needs and challenges of their natural environments not those of industrial fermenters and bioreactors. Gene Transfer from one system to another may sound easy but in practice is hard work and rarely generates sufficient reward (i.e., increased yield) to justify the investment made. The application of automation and artificial intelligence (e.g., in designing and building plasmids) may help to reduce the time and cost—and improve return on investment—in the future (Zhang et al., 2018).\n",
      "“Scale up is product specific – we need more synthetic biology in the production process”\n",
      "Plants make alternative production platforms. Improvements in mining plant genomes and the development of effective transient expression systems have enabled large-scale production of, for example, vaccines in tobacco plants in just a few weeks (Dirisala et al., 2017; Emmanuel et al., 2018). Directing the production of synthetic biological materials to plant chloroplasts also shows promise (Boehm and Bock, 2019).\n",
      "The photosynthetic reducing power generated in plant chloroplasts can be harnessed for the light-driven synthesis of bioactive molecules such as dhurrin, which protects plants against insects (Gnanasekaran et al., 2016).\n",
      "However, underlying all these platforms is a knowledge gap in our understanding in how nature works. This makes it very hard to apply the design/build/test/learn cycles used in conventional engineering to the production of synthetic biological materials whatever the production platform (yeast, bacteria, plants, or human cells) if the platform itself is not well-understood (Sauro et al., 2006).\n",
      "What we need now are instruments able to measure and characterize outputs, assisted by progress in robotics and automation, and the application of machine learning approaches to analyse the data generated. This will help us to generate more robust models of biological systems, so we can improve experimental design for future engineering strategies.\n",
      "“We can do ‘build’. ‘Test’ is the challenge when we want to learn from the iterative design process”\n",
      "Synthetic biology is driving significant advances in biomedicine, which will lead to transformational improvements in healthcare. Already, patients are benefiting from so-called CAR (for chimeric antigen receptor) technology, which engineers the immune cells (T-cells) of the patient to recognize and attack cancer cells (June et al., 2018).\n",
      "Genetically engineered viruses are being used to correct defective genes in patients with inherited diseases such as Severe Combined Immune Deficiency (SCID) or epidermolysis bullosa (Dunbar et al., 2018).\n",
      "The ability to reprogramme somatic cells from patients into induced pluripotent stem cells is furthering our understanding of their disease, reducing the use of animals in research, and paving the way for the development of personalized medicines and cell therapies. In principle at least, we could engineer a patient's own cells to multiply, differentiate into different cell types and even self-assemble into new tissues, or even organs, to repair those damaged through disease or injury (Davies and Cachat, 2016; Satoshi et al., 2018).\n",
      "Work on new vectors that are able to deliver large genetic loads to target tissues is helping to produce more efficient therapeutics and vaccines that will have fewer side effects and a smaller risk of resistance. Furthermore, optimizing antibody or vaccine production, or example, so that they are in an edible format (e.g., plant based), could greatly reduce the cost and increase the speed of vaccine production in an epidemic.\n",
      "“We have the tools but need the creativity to make stuff that can't be made without synthetic biology”\n",
      "In the next few years, genetically engineering pigs to be virus resistant and have human-like immune profiles could make xenotransplantation a clinical reality (Burkard et al., 2018). Engineering the microbiome is expected to lead to the development of synthetic probiotics (Dou and Bennett, 2018).\n",
      "The synthetic biology initiative known as Human Genome Project-write (HGP-write) has set its sights even higher, rallying scientists to build entire human chromosomes (Boeke et al., 2016). Concerns have been raised about the ethics of creating “synthetic humans” and indeed the scientific and commercial value of such a project. More recently, HGP-write champions have proposed a more focused project to build a virus-resistant chromosome, making at least 400,000 changes to the human genome to remove DNA sequences that viruses use to hijack cells and replicate (Dolgin, 2018).\n",
      "One of the many exciting opportunities that synthetic biology offers medicine is in the production of theranostic cell lines that can sense a disease state and produce an appropriate therapeutic response (Teixeira and Fussenegger, 2019). Several obstacles need to be overcome to achieve this goal: first, to expand the range of molecules that can be recognized by cellular “sensors” as inputs; and second, to better understand the genetic control factors that regulate gene expression in space and time so we can engineer better activator systems.\n",
      "“At present we need pills because we can't swallow a chemistry kit”\n",
      "Metabolomics is shedding light on many disease biomarkers. Because some biomarkers are shared between seemingly unrelated diseases, an accurate diagnosis will require the detection of multiple markers to provide a more unique “disease fingerprint.” Work in whole cell and cell free systems to develop sensors of multiple disease-specific biomarkers could assist in earlier detection of disease and prognostic monitoring.\n",
      "To expand the range of biologically detectable molecules, it is possible to design metabolic pathways that transform currently undetectable molecules of interest (e.g., hippuric acid, the prostate cancer biomarker) into molecules for which sensors already exist (in this case benzoic acid) (Libis et al., 2016).\n",
      "Cybergenetics is an emerging field that is developing experimental tools for the computer control of cellular processes at the gene level in real time. Cybergenetic control can be achieved by interfacing living cells with a digital computer that switches on or off the embedded “genetic switch” using light (optogenetics) or chemicals (Gabriele et al., 2018; Maysam et al., 2019). Such systems could help to maintain cellular homeostasis by monitoring the state of the body and triggering an appropriate response upon the detection of dysregulation; for example, they could trigger the release of insulin when blood glucose levels rise as detected by a wireless diagnostic tool (Ye et al., 2011).\n",
      "Synthetic biology offers the opportunity to create responsive and multifunctional materials (Le Feuvre and Scrutton, 2018). The integration of biochemical components from living systems with inorganic components can lead to new materials that are able to sense the environment (or internal signals) and change their properties. These features could be particularly useful for improving protective clothing or building materials.\n",
      "An issue when using microbes to produce composite materials is regulating the assembly of these materials to achieve specific desired properties. By understanding how microbes communicate with each other, it is possible to make them work better together and combine them with other production systems so that the properties of materials can be tailored for particular functions.\n",
      "Interestingly, rather than modifying, or improving existing protein-based materials, an alternative approach involves using computational techniques to design completely novel proteins that self-assemble into predicted shapes (Ljubetič et al., 2017). Such “programmable” proteins open up even further opportunities for synthetic biology not only for materials science but also for medicine and chemistry.\n",
      "“The tools are there, we just don't know what we want to make”\n",
      "Regardless of the research areas involved, there are some common challenges for the community to address.\n",
      "There was consensus that while the production of small molecules using synthetically engineered cells at the bench is becoming more tractable, these processes often do not translate well into mass production. Scalability needs to be incorporated into the initial design process by including features, for example, that reduce toxicity of the molecule to the production host or “chassis” and/or by introducing modifications that favor its extraction. Careful consideration of the right chassis could also greatly improve yields and significantly reduce cost of production.\n",
      "“Nature has developed its own ways to concentrate and solubilise chemicals…we are not learning from this”\n",
      "The number of microbes that are currently in use for the production of synthetic biological materials is only a tiny fraction of the total diversity that exists in Nature; only ten microbes are “domesticated” for industrial use. To identify the most appropriate chassis, it is worth turning to nature to identify species that have unique metabolic networks suited to host particular types of chemical reactions. For example, the soil bacterium Pseudomonas putida, which has adapted to harsh environmental conditions, is ideally suited to host redox-intensive reactions (Pablo and de Lorenzo, 2018). A treasure trove of, as yet, unexplored natural products, with novel and beneficial properties, exist in the plant kingdom. In addition, as noted above, plants are naturally excellent production hosts and it is now possible to “plug and play” combinations of plant pathways to generate novel molecules (Sainsbury and Lomonossoff, 2014; Evangelos and O'connor, 2016).\n",
      "To circumvent the problem of the impact of different host chassis on synthetic gene circuits, researchers would benefit from a universal synthetic expression system that permits the testing of new constructs. This would aid in the identification of optimal production platforms and decrease the need for organism-specific technologies. Combining this technology with DNA editing techniques, such as CRISPR/Cas9, will make the establishment of host production platforms and the generation of complex biosynthetic products easier and faster. One articulation of this could be in cell-free formats, in which the essential cellular machinery is reconstituted in vitro and used as a manufacturing platform (Villarreal and Tan, 2017; Koch et al., 2018).\n",
      "“We need to reduce the burden of synthetic networks on endogenous circuits”\n",
      "Cell-free environments, interfaced with semiconductors, offer a powerful route for flexible and controllable production systems.\n",
      "For example, nanoparticles made of semiconductor materials or quantum dots can be used to enhance enzyme activity in a cell free environment with a minimal set of ingredients. Multistep enzymatic pathways can be tethered to nanoparticle surfaces and, by avoiding the diffusion effect that takes place in cells, reaction rates can be increased 100-fold (Wang et al., 2017). This approach can be used to access non-natural materials and circumvents the potential issue of toxicity in cells (as well as the regulatory issues of genetically modified cells).\n",
      "Research on silicon chips containing immobilized genes and cell lysates allows detailed examination of gene expression in space and time. When the compartments in which DNA-driven reactions take place are linked, with materials flowing into and diffusing between the compartments, it is possible to recreate oscillating protein expression patterns and protein gradients akin to those observed in cells (Karzbrun et al., 2014). Gene expression in these “artificial cells” can be controlled with electrodes that prevent the assembly of proteins by ribosomes. By standardizing outputs, this strategy is improving the predictability of engineered genetic circuits.\n",
      "Whatever the system being explored, a robust model of that biological system is needed if predictable modifications of genetic circuits are to be designed and implemented with any confidence. To create such models, large sets of data arising from measuring multiple parameters of cell behavior under different conditions are essential (Fletcher et al., 2016).\n",
      "There is great expectation that improvements in high-throughput data measurement and collection systems will generate exactly the large data sets needed. These can be analyzed using artificial intelligence or a machine learning approach to optimize the design of synthetic biological products and move away from the inefficient trial and error process (Decoene et al., 2018).\n",
      "Finally, agreement on standards of design, assembly, data transfer, data measurement and regulatory rules, as well as on the language that is used, will help to improve the interdisciplinary and international collaborations that are required to drive the field forward. This is challenging for a community with such diverse interests and perspectives, and where data sharing, curating and quality control is not common practice.\n",
      "However, without some form of agreed standards, many of the products and processes of synthetic biology will not translate well to industrial settings dependent on reproducible processes and beholden to exacting regulatory requirements.\n",
      "Typically, academic researchers are driven by a need to understand the complexity of nature (and publish their work in high impact journals). Standardization and scaling up production are important, but have less of an academic pull than discovering a new product. Access to funding, industrial partnerships, and academic recognition are examples of potential incentives for carrying out this type of research.\n",
      "“Standards restrict flexibility but enable interoperability”\n",
      "Synthetic biology is an example of a dual-use technology: it promises numerous beneficial applications, but it can also cause harm. This has led to fears that it could, intentionally or unintentionally, harm humans or damage the environment. For example, there is huge value in our ability to engineer viruses to be more effective and specific shuttles for gene therapies of devastating inherited disorders; however, engineering viruses may also lead to the creation of even more deadly pathogens by those intent on harm.\n",
      "“Synthetic biology should be regarded as an extension of earlier developments and technologies”\n",
      "Some would argue that synthetic biology poses an existential risk and needs to be treated with extreme caution. However, many new technological advances across the decades have met similar concerns. The uncertainty and remote possibility of such risks could hamper the development of useful technology. Scientists, their host institutions and funding bodies should (and indeed already do) consider whether the research planned could be misused. Measures that reduce the likelihood of misuse and its consequences should be implemented and clearly communicated. The synthetic biology community needs to be aware of, and respond to, these challenges by engaging in horizon scanning exercises as well as open dialogue with regulatory bodies and the media.\n",
      "“Don't avoid risk – manage it”\n",
      "Being more open about risks, and how they are controlled, provides an opportunity to shift discourse toward the benefits of synthetic biology in addressing urgent global needs, such as the production of biofuels, food security and more effective medicines, and potentially improve public acceptance.\n",
      "“The questions should not be ‘what’s the next big thing for synthetic biology' but ‘where is the greatest unmet need’.”\n",
      "Despite the efforts by individual countries to establish synthetic biology research roadmaps, broader, international agreement on common standards (and red lines) across the field may help establish trust and to advance the best pre-competitive research into useful applications.\n",
      "Meeting participants highlighted the importance of training in responsible research conduct and ethics. Given students' future role as science ambassadors and influencers, their training should not only convey skills and knowledge but also awareness and critical thinking about the prospects and potential for dual use of synthetic biology. All researchers must remain vigilant regardless of the many pressures and distractions of running a successful research lab; they may not have specialist training in identifying the risks of misuse but they are the people best placed to maintain informed oversight of risks.\n",
      "One example of current synthetic biology research with potential dual use is gene drive technology, which can be used to propagate a particular suite of genes throughout a population. The benefits of using gene drive technology include the eradication of disease-carrying insect populations and the elimination of invading pest species but it has raised concerns about the unintended ecological impacts of reducing or eliminating a population (Callaway, 2018; Collins, 2018).\n",
      "Similar release concerns surround research that is harnessing the ability of pathogens to target particular tissues in the body or particular chemicals in the environment, which could greatly aid efforts to deliver targeted therapies or clean-up contaminated sites. To date, such large-scale release for environmental bioremediation interventions has not been possible.\n",
      "“We need to mind the gap between R&D scale up and communications …. One bad blog can kill a commercial product”\n",
      "There was consensus that the need for regulation over this community remains important. Regulation needs to keep up to speed with the emerging technologies and should focus on the product rather than the process used to create it (Tait et al., 2017). Unsuitable regulatory frameworks (as well as unfavorable public perception) could discourage private sector investment in synthetic biology.\n",
      "Open and balanced two-way communication between researchers, funders, companies and governments and the public will be vital. Consumers and activists may have no interest in the difference between making a chemical (e.g., a flavor or fragrance) synthetically and one made using genetically modified bacteria but they may instinctively distrust the latter.\n",
      "The short talks presented at this 2-day meeting suggest that synthetic biology is at the cusp of many major breakthroughs and that it is perhaps timely to re-define the meaning of “success” in synthetic biology.\n",
      "There are many hurdles to overcome but the potential for synthetic biology to deliver solutions to many global challenges—improving healthcare, limiting environmental damage, and creating a wide variety of more sustainable processes—is great.\n",
      "Meeting participants suggested that as a community they should support the measures listed below to help synthetic biology move beyond the proof-of-concept stage and to ensure that potential risks are minimized and dialog with the public can be optimized.\n",
      "Larger and longer investment in better big data management and processing (Artificial Intelligence and Machine Learning) systems, and in fundamental research on biosystems modeling, chassis development, and genome mining.\n",
      "Support standardization initiatives; while arguably not attractive academically, the community needs to agree and support efforts toward creating interoperability around biosystem modeling, and standards around DNA design (if not DNA bioparts). Without some degree of “standardization” the ability to pool data and models, essential for improving accuracy and reproducibility, will be challenging.\n",
      "The creation and funding (ideally internationally) of a “Grand Challenge,” such as the development of generic sensors or the creation of protein-based electronic components, could help focus the community toward a target goal.\n",
      "Improve the assessment, communication, and management of risk and harm among all audiences.\n",
      "Ensure that early career researchers are trained in responsible research conduct and ethics as well as being cognizant of existing rules and regulations around GM (regionally dependent) and the issue of misuse and harm.\n",
      "Coordinate the efforts of academia, government and industry through focused meetings that foster interdisciplinary collaborations around shared objectives.\n",
      "Improve platforms for knowledge sharing and recognize the value of failures.\n",
      "The workshop highlighted just how much more we have to learn from Nature itself. Synthetic biology is giving us insights of the criteria and processes that underpin all living systems; in turn, we can take this insight and design and use it to build a “better biology.” However, we need to take the public with us on this journey, create meaningful and considered dialogue about the work we may do and the impact it might have on our world.\n",
      "The meeting was hosted by the UK Center for Mammalian Synthetic Biology (the Center), based at the University of Edinburgh. The Center is building expertise in cell engineering tool generation, whole-cell modeling, computer-assisted design and construction of DNA and high-throughput phenotyping to enable synthetic biology for medicine and healthcare. The Centre's research will not only advance basic understanding of mammalian biology and pathology but also generate products and services for near-term commercial exploitation by the pharmaceutical and drug testing industries, such as diagnostics, novel therapeutics, protein-based drugs, and regenerative medicines. The Center is funded through the Research Council's UK “Synthetic Biology for Growth” programme and by the Biological and Biotechnology Sciences Research Council (BBSRC), the Engineering and Physical Sciences Research Council and the Medical Research Council.\n",
      "This is a meeting report and was drafted by MH-F with support from ME and LF.\n",
      "The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\n"
     ]
    }
   ],
   "source": [
    "#Join the text\n",
    "body_text = \"\\n\".join(body_text)\n",
    "print(body_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Save as a JSON file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [],
   "source": [
    "def save_into_dict(file_path):\n",
    "    file_id = file_path[(file_path.index('pmc')+3):file_path.index('.xml')]\n",
    "    infile = open(file_path,\"r\")\n",
    "    contents = infile.read()\n",
    "    soup = BeautifulSoup(contents,'xml')\n",
    "    \n",
    "    title = soup.find('article-title').text\n",
    "    authors = Entrez.read(Entrez.esummary(db='pmc', id=file_id))[0]['AuthorList']\n",
    "    abstract = soup.find('abstract').text\n",
    "    \n",
    "    try:\n",
    "        key = soup.find('kwd-group').find_all('kwd')\n",
    "        key_words = [k.text for k in key]\n",
    "    except:\n",
    "        key_words = []\n",
    "    \n",
    "    body = soup.find('body')\n",
    "    body_text = [i.text for i in body.find_all('p')]\n",
    "    body_text = \"\\n\".join(body_text)\n",
    "    \n",
    "    output = {\n",
    "        \"title\": title,\n",
    "        \"authors\": authors,\n",
    "        \"abstract\": abstract,\n",
    "        \"key-words\": key_words,\n",
    "        \"body\": body_text\n",
    "    }\n",
    "    return output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': 'Future Trends in Synthetic Biology—A Report', 'authors': ['El Karoui M', 'Hoyos-Flight M', 'Fletcher L'], 'abstract': '\\nLeading researchers working on synthetic biology and its applications gathered at the University of Edinburgh in May 2018 to discuss the latest challenges and opportunities in the field. In addition to the potential socio-economic benefits of synthetic biology, they also examined the ethics and security risks arising from the development of these technologies. Speakers from industry, academia and not-for-profit organizations presented their vision for the future of the field and provided guidance to funding and regulatory bodies to ensure that synthetic biology research is carried out responsibly and can realize its full potential. This report aims to capture the collective views and recommendations that emerged from the discussions that took place. The meeting was held under the Chatham House Rule (i.e., a private invite-only meeting where comments can be freely used but not attributed) to promote open discussion; the findings and quotes included in the report are therefore not attributed to individuals. The goal of the meeting was to identify research priorities and bottlenecks. It also provided the opportunity to discuss how best to manage risk and earn public acceptance of this emerging and disruptive technology.\\n', 'key-words': ['synthetic biology', 'biosystem', 'future trends and developments', 'biodesign automation', 'responsible research and innovation (RRI)'], 'body': \"Synthetic Biology offers innovative approaches for engineering new biological systems or re-designing existing ones for useful purposes (see Figure 1). It has been described as a disruptive technology at the heart of the so-called Bioeconomy, capable of delivering new solutions to global healthcare, agriculture, manufacturing, and environmental challenges (Cameron et al., 2014; Bueso and Tangney, 2017; French, 2019). However, despite successes in the production of some high value chemicals and drugs, there is a perception that synthetic biology is still not yet delivering on its promise.\\nSynthetic biology is developing into a biodesign platform where it will be possible to apply the “design-build-test-iterate (or deploy)” to predictably create cells or organisms able to produce a wide variety of novel molecules, materials or even cells for multiple applications.\\nMoreover, there are some concerns from governments that synthetic biology expands the pool of agents of concern, which increases the need to develop detection, identification and monitoring systems, and proactively build countermeasures against chemical and biological threats (Wang and Zhang, 2019). The participation of representatives from various government organizations at this meeting is testament to their commitment to maintaining an active dialogue with the synthetic biology community. In this way, they aim to keep abreast of the changing nature of threats and provide the best advice to government about investment in science and technology and the introduction or amendment of regulatory processes.\\nThe cost of DNA sequencing and synthesis have decreased dramatically (Carlson, 2014; Kosuri and Church, 2014) and we have access to more genetic information and more powerful genetic engineering capabilities than ever before. Critical investments in infrastructure are bearing fruit and, as is described below, synthetic biology is increasingly becoming, at least part of, the solution to many of our present and future needs in medicine, food and energy production, remediation, manufacturing, and national security. So what is the potential of synthetic biology and what challenges does it still face to realize this?\\nDespite the lack of predictability in biology, and current technical constraints that limit data collection and analyses, we can now produce small molecules on demand using synthetic biology approaches.\\nProbably the most impressive examples come from the Foundry at the Broad Institute of MIT and Harvard. When the Defense Advanced Research Projects Agency (DARPA) put the MIT-Broad Institute Foundry's design capabilities to the test, its researchers were able to deliver 6 out of 10 molecules of interest to the US Department of Defense in 90 days. This “pressure test” confirms the potential of synthetic biology to address shortages of key compounds quickly (Casini et al., 2018).\\nIndeed, many labs can now design and construct relatively complex gene networks capable of generating a wide variety of “designer” molecules in a range of host cells; however, this is often a slow iterative process of trial and error.\\nAs yet, very few small molecules in medicine are manufactured using a synthetic biology process; it remains very difficult to engineer microbes to carry out processes that Nature did not intend. This is to be expected: the performance of microbes is “good enough” from an evolutionary perspective. Microbes evolved to address the specific needs and challenges of their natural environments not those of industrial fermenters and bioreactors. Gene Transfer from one system to another may sound easy but in practice is hard work and rarely generates sufficient reward (i.e., increased yield) to justify the investment made. The application of automation and artificial intelligence (e.g., in designing and building plasmids) may help to reduce the time and cost—and improve return on investment—in the future (Zhang et al., 2018).\\n“Scale up is product specific – we need more synthetic biology in the production process”\\nPlants make alternative production platforms. Improvements in mining plant genomes and the development of effective transient expression systems have enabled large-scale production of, for example, vaccines in tobacco plants in just a few weeks (Dirisala et al., 2017; Emmanuel et al., 2018). Directing the production of synthetic biological materials to plant chloroplasts also shows promise (Boehm and Bock, 2019).\\nThe photosynthetic reducing power generated in plant chloroplasts can be harnessed for the light-driven synthesis of bioactive molecules such as dhurrin, which protects plants against insects (Gnanasekaran et al., 2016).\\nHowever, underlying all these platforms is a knowledge gap in our understanding in how nature works. This makes it very hard to apply the design/build/test/learn cycles used in conventional engineering to the production of synthetic biological materials whatever the production platform (yeast, bacteria, plants, or human cells) if the platform itself is not well-understood (Sauro et al., 2006).\\nWhat we need now are instruments able to measure and characterize outputs, assisted by progress in robotics and automation, and the application of machine learning approaches to analyse the data generated. This will help us to generate more robust models of biological systems, so we can improve experimental design for future engineering strategies.\\n“We can do ‘build’. ‘Test’ is the challenge when we want to learn from the iterative design process”\\nSynthetic biology is driving significant advances in biomedicine, which will lead to transformational improvements in healthcare. Already, patients are benefiting from so-called CAR (for chimeric antigen receptor) technology, which engineers the immune cells (T-cells) of the patient to recognize and attack cancer cells (June et al., 2018).\\nGenetically engineered viruses are being used to correct defective genes in patients with inherited diseases such as Severe Combined Immune Deficiency (SCID) or epidermolysis bullosa (Dunbar et al., 2018).\\nThe ability to reprogramme somatic cells from patients into induced pluripotent stem cells is furthering our understanding of their disease, reducing the use of animals in research, and paving the way for the development of personalized medicines and cell therapies. In principle at least, we could engineer a patient's own cells to multiply, differentiate into different cell types and even self-assemble into new tissues, or even organs, to repair those damaged through disease or injury (Davies and Cachat, 2016; Satoshi et al., 2018).\\nWork on new vectors that are able to deliver large genetic loads to target tissues is helping to produce more efficient therapeutics and vaccines that will have fewer side effects and a smaller risk of resistance. Furthermore, optimizing antibody or vaccine production, or example, so that they are in an edible format (e.g., plant based), could greatly reduce the cost and increase the speed of vaccine production in an epidemic.\\n“We have the tools but need the creativity to make stuff that can't be made without synthetic biology”\\nIn the next few years, genetically engineering pigs to be virus resistant and have human-like immune profiles could make xenotransplantation a clinical reality (Burkard et al., 2018). Engineering the microbiome is expected to lead to the development of synthetic probiotics (Dou and Bennett, 2018).\\nThe synthetic biology initiative known as Human Genome Project-write (HGP-write) has set its sights even higher, rallying scientists to build entire human chromosomes (Boeke et al., 2016). Concerns have been raised about the ethics of creating “synthetic humans” and indeed the scientific and commercial value of such a project. More recently, HGP-write champions have proposed a more focused project to build a virus-resistant chromosome, making at least 400,000 changes to the human genome to remove DNA sequences that viruses use to hijack cells and replicate (Dolgin, 2018).\\nOne of the many exciting opportunities that synthetic biology offers medicine is in the production of theranostic cell lines that can sense a disease state and produce an appropriate therapeutic response (Teixeira and Fussenegger, 2019). Several obstacles need to be overcome to achieve this goal: first, to expand the range of molecules that can be recognized by cellular “sensors” as inputs; and second, to better understand the genetic control factors that regulate gene expression in space and time so we can engineer better activator systems.\\n“At present we need pills because we can't swallow a chemistry kit”\\nMetabolomics is shedding light on many disease biomarkers. Because some biomarkers are shared between seemingly unrelated diseases, an accurate diagnosis will require the detection of multiple markers to provide a more unique “disease fingerprint.” Work in whole cell and cell free systems to develop sensors of multiple disease-specific biomarkers could assist in earlier detection of disease and prognostic monitoring.\\nTo expand the range of biologically detectable molecules, it is possible to design metabolic pathways that transform currently undetectable molecules of interest (e.g., hippuric acid, the prostate cancer biomarker) into molecules for which sensors already exist (in this case benzoic acid) (Libis et al., 2016).\\nCybergenetics is an emerging field that is developing experimental tools for the computer control of cellular processes at the gene level in real time. Cybergenetic control can be achieved by interfacing living cells with a digital computer that switches on or off the embedded “genetic switch” using light (optogenetics) or chemicals (Gabriele et al., 2018; Maysam et al., 2019). Such systems could help to maintain cellular homeostasis by monitoring the state of the body and triggering an appropriate response upon the detection of dysregulation; for example, they could trigger the release of insulin when blood glucose levels rise as detected by a wireless diagnostic tool (Ye et al., 2011).\\nSynthetic biology offers the opportunity to create responsive and multifunctional materials (Le Feuvre and Scrutton, 2018). The integration of biochemical components from living systems with inorganic components can lead to new materials that are able to sense the environment (or internal signals) and change their properties. These features could be particularly useful for improving protective clothing or building materials.\\nAn issue when using microbes to produce composite materials is regulating the assembly of these materials to achieve specific desired properties. By understanding how microbes communicate with each other, it is possible to make them work better together and combine them with other production systems so that the properties of materials can be tailored for particular functions.\\nInterestingly, rather than modifying, or improving existing protein-based materials, an alternative approach involves using computational techniques to design completely novel proteins that self-assemble into predicted shapes (Ljubetič et al., 2017). Such “programmable” proteins open up even further opportunities for synthetic biology not only for materials science but also for medicine and chemistry.\\n“The tools are there, we just don't know what we want to make”\\nRegardless of the research areas involved, there are some common challenges for the community to address.\\nThere was consensus that while the production of small molecules using synthetically engineered cells at the bench is becoming more tractable, these processes often do not translate well into mass production. Scalability needs to be incorporated into the initial design process by including features, for example, that reduce toxicity of the molecule to the production host or “chassis” and/or by introducing modifications that favor its extraction. Careful consideration of the right chassis could also greatly improve yields and significantly reduce cost of production.\\n“Nature has developed its own ways to concentrate and solubilise chemicals…we are not learning from this”\\nThe number of microbes that are currently in use for the production of synthetic biological materials is only a tiny fraction of the total diversity that exists in Nature; only ten microbes are “domesticated” for industrial use. To identify the most appropriate chassis, it is worth turning to nature to identify species that have unique metabolic networks suited to host particular types of chemical reactions. For example, the soil bacterium Pseudomonas putida, which has adapted to harsh environmental conditions, is ideally suited to host redox-intensive reactions (Pablo and de Lorenzo, 2018). A treasure trove of, as yet, unexplored natural products, with novel and beneficial properties, exist in the plant kingdom. In addition, as noted above, plants are naturally excellent production hosts and it is now possible to “plug and play” combinations of plant pathways to generate novel molecules (Sainsbury and Lomonossoff, 2014; Evangelos and O'connor, 2016).\\nTo circumvent the problem of the impact of different host chassis on synthetic gene circuits, researchers would benefit from a universal synthetic expression system that permits the testing of new constructs. This would aid in the identification of optimal production platforms and decrease the need for organism-specific technologies. Combining this technology with DNA editing techniques, such as CRISPR/Cas9, will make the establishment of host production platforms and the generation of complex biosynthetic products easier and faster. One articulation of this could be in cell-free formats, in which the essential cellular machinery is reconstituted in vitro and used as a manufacturing platform (Villarreal and Tan, 2017; Koch et al., 2018).\\n“We need to reduce the burden of synthetic networks on endogenous circuits”\\nCell-free environments, interfaced with semiconductors, offer a powerful route for flexible and controllable production systems.\\nFor example, nanoparticles made of semiconductor materials or quantum dots can be used to enhance enzyme activity in a cell free environment with a minimal set of ingredients. Multistep enzymatic pathways can be tethered to nanoparticle surfaces and, by avoiding the diffusion effect that takes place in cells, reaction rates can be increased 100-fold (Wang et al., 2017). This approach can be used to access non-natural materials and circumvents the potential issue of toxicity in cells (as well as the regulatory issues of genetically modified cells).\\nResearch on silicon chips containing immobilized genes and cell lysates allows detailed examination of gene expression in space and time. When the compartments in which DNA-driven reactions take place are linked, with materials flowing into and diffusing between the compartments, it is possible to recreate oscillating protein expression patterns and protein gradients akin to those observed in cells (Karzbrun et al., 2014). Gene expression in these “artificial cells” can be controlled with electrodes that prevent the assembly of proteins by ribosomes. By standardizing outputs, this strategy is improving the predictability of engineered genetic circuits.\\nWhatever the system being explored, a robust model of that biological system is needed if predictable modifications of genetic circuits are to be designed and implemented with any confidence. To create such models, large sets of data arising from measuring multiple parameters of cell behavior under different conditions are essential (Fletcher et al., 2016).\\nThere is great expectation that improvements in high-throughput data measurement and collection systems will generate exactly the large data sets needed. These can be analyzed using artificial intelligence or a machine learning approach to optimize the design of synthetic biological products and move away from the inefficient trial and error process (Decoene et al., 2018).\\nFinally, agreement on standards of design, assembly, data transfer, data measurement and regulatory rules, as well as on the language that is used, will help to improve the interdisciplinary and international collaborations that are required to drive the field forward. This is challenging for a community with such diverse interests and perspectives, and where data sharing, curating and quality control is not common practice.\\nHowever, without some form of agreed standards, many of the products and processes of synthetic biology will not translate well to industrial settings dependent on reproducible processes and beholden to exacting regulatory requirements.\\nTypically, academic researchers are driven by a need to understand the complexity of nature (and publish their work in high impact journals). Standardization and scaling up production are important, but have less of an academic pull than discovering a new product. Access to funding, industrial partnerships, and academic recognition are examples of potential incentives for carrying out this type of research.\\n“Standards restrict flexibility but enable interoperability”\\nSynthetic biology is an example of a dual-use technology: it promises numerous beneficial applications, but it can also cause harm. This has led to fears that it could, intentionally or unintentionally, harm humans or damage the environment. For example, there is huge value in our ability to engineer viruses to be more effective and specific shuttles for gene therapies of devastating inherited disorders; however, engineering viruses may also lead to the creation of even more deadly pathogens by those intent on harm.\\n“Synthetic biology should be regarded as an extension of earlier developments and technologies”\\nSome would argue that synthetic biology poses an existential risk and needs to be treated with extreme caution. However, many new technological advances across the decades have met similar concerns. The uncertainty and remote possibility of such risks could hamper the development of useful technology. Scientists, their host institutions and funding bodies should (and indeed already do) consider whether the research planned could be misused. Measures that reduce the likelihood of misuse and its consequences should be implemented and clearly communicated. The synthetic biology community needs to be aware of, and respond to, these challenges by engaging in horizon scanning exercises as well as open dialogue with regulatory bodies and the media.\\n“Don't avoid risk – manage it”\\nBeing more open about risks, and how they are controlled, provides an opportunity to shift discourse toward the benefits of synthetic biology in addressing urgent global needs, such as the production of biofuels, food security and more effective medicines, and potentially improve public acceptance.\\n“The questions should not be ‘what’s the next big thing for synthetic biology' but ‘where is the greatest unmet need’.”\\nDespite the efforts by individual countries to establish synthetic biology research roadmaps, broader, international agreement on common standards (and red lines) across the field may help establish trust and to advance the best pre-competitive research into useful applications.\\nMeeting participants highlighted the importance of training in responsible research conduct and ethics. Given students' future role as science ambassadors and influencers, their training should not only convey skills and knowledge but also awareness and critical thinking about the prospects and potential for dual use of synthetic biology. All researchers must remain vigilant regardless of the many pressures and distractions of running a successful research lab; they may not have specialist training in identifying the risks of misuse but they are the people best placed to maintain informed oversight of risks.\\nOne example of current synthetic biology research with potential dual use is gene drive technology, which can be used to propagate a particular suite of genes throughout a population. The benefits of using gene drive technology include the eradication of disease-carrying insect populations and the elimination of invading pest species but it has raised concerns about the unintended ecological impacts of reducing or eliminating a population (Callaway, 2018; Collins, 2018).\\nSimilar release concerns surround research that is harnessing the ability of pathogens to target particular tissues in the body or particular chemicals in the environment, which could greatly aid efforts to deliver targeted therapies or clean-up contaminated sites. To date, such large-scale release for environmental bioremediation interventions has not been possible.\\n“We need to mind the gap between R&D scale up and communications …. One bad blog can kill a commercial product”\\nThere was consensus that the need for regulation over this community remains important. Regulation needs to keep up to speed with the emerging technologies and should focus on the product rather than the process used to create it (Tait et al., 2017). Unsuitable regulatory frameworks (as well as unfavorable public perception) could discourage private sector investment in synthetic biology.\\nOpen and balanced two-way communication between researchers, funders, companies and governments and the public will be vital. Consumers and activists may have no interest in the difference between making a chemical (e.g., a flavor or fragrance) synthetically and one made using genetically modified bacteria but they may instinctively distrust the latter.\\nThe short talks presented at this 2-day meeting suggest that synthetic biology is at the cusp of many major breakthroughs and that it is perhaps timely to re-define the meaning of “success” in synthetic biology.\\nThere are many hurdles to overcome but the potential for synthetic biology to deliver solutions to many global challenges—improving healthcare, limiting environmental damage, and creating a wide variety of more sustainable processes—is great.\\nMeeting participants suggested that as a community they should support the measures listed below to help synthetic biology move beyond the proof-of-concept stage and to ensure that potential risks are minimized and dialog with the public can be optimized.\\nLarger and longer investment in better big data management and processing (Artificial Intelligence and Machine Learning) systems, and in fundamental research on biosystems modeling, chassis development, and genome mining.\\nSupport standardization initiatives; while arguably not attractive academically, the community needs to agree and support efforts toward creating interoperability around biosystem modeling, and standards around DNA design (if not DNA bioparts). Without some degree of “standardization” the ability to pool data and models, essential for improving accuracy and reproducibility, will be challenging.\\nThe creation and funding (ideally internationally) of a “Grand Challenge,” such as the development of generic sensors or the creation of protein-based electronic components, could help focus the community toward a target goal.\\nImprove the assessment, communication, and management of risk and harm among all audiences.\\nEnsure that early career researchers are trained in responsible research conduct and ethics as well as being cognizant of existing rules and regulations around GM (regionally dependent) and the issue of misuse and harm.\\nCoordinate the efforts of academia, government and industry through focused meetings that foster interdisciplinary collaborations around shared objectives.\\nImprove platforms for knowledge sharing and recognize the value of failures.\\nThe workshop highlighted just how much more we have to learn from Nature itself. Synthetic biology is giving us insights of the criteria and processes that underpin all living systems; in turn, we can take this insight and design and use it to build a “better biology.” However, we need to take the public with us on this journey, create meaningful and considered dialogue about the work we may do and the impact it might have on our world.\\nThe meeting was hosted by the UK Center for Mammalian Synthetic Biology (the Center), based at the University of Edinburgh. The Center is building expertise in cell engineering tool generation, whole-cell modeling, computer-assisted design and construction of DNA and high-throughput phenotyping to enable synthetic biology for medicine and healthcare. The Centre's research will not only advance basic understanding of mammalian biology and pathology but also generate products and services for near-term commercial exploitation by the pharmaceutical and drug testing industries, such as diagnostics, novel therapeutics, protein-based drugs, and regenerative medicines. The Center is funded through the Research Council's UK “Synthetic Biology for Growth” programme and by the Biological and Biotechnology Sciences Research Council (BBSRC), the Engineering and Physical Sciences Research Council and the Medical Research Council.\\nThis is a meeting report and was drafted by MH-F with support from ME and LF.\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\"}\n"
     ]
    }
   ],
   "source": [
    "print(save_into_dict('./ethics/pmc6692427.xml'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Extraction Process for all .xml files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import glob\n",
    "from tqdm import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [],
   "source": [
    "ethics_files = glob.glob('./ethics/*.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_files(category):\n",
    "    files_paths = glob.glob('./ethics/*.xml')\n",
    "    \n",
    "    for file in tqdm(files_paths):\n",
    "        file = file[file.index('pmc'):]\n",
    "        with open('./processed/'+category+'/'+file[:-4]+'.json', 'w') as f:\n",
    "            json.dump(save_into_dict('./'+category+'/'+file), fp=f, indent=6)\n",
    "    return"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 238/238 [01:41<00:00,  2.35it/s]\n"
     ]
    }
   ],
   "source": [
    "extract_files('ethics')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Retrieving directly from Pubmed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('./abstracts only/pubmed-syntheticb-set.txt', 'r') as f:\n",
    "    text = f.readlines()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "size = len(text)\n",
    "idx_list = [idx + 1 for idx, val in\n",
    "            enumerate(text) if val == '\\n']\n",
    "  \n",
    "\n",
    "text = [text[i: j] for i, j in\n",
    "        zip([0] + idx_list, idx_list + \n",
    "        ([size] if idx_list[-1] != size else []))]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_from_pmc_text(text):\n",
    "    authors = []\n",
    "    affiliation = ''\n",
    "    abstract = ''\n",
    "    title = ''\n",
    "    no_more_AD = False\n",
    "    prev_row = ' '\n",
    "    for row in text:\n",
    "        if prev_row != ' ' and row.startswith(' '):\n",
    "            if prev_row == 'TI':\n",
    "                title += row[6:].strip('\\n')\n",
    "            if prev_row == 'AB':\n",
    "                abstract += row[6:].strip('\\n')\n",
    "            if prev_row == 'AD':\n",
    "                affiliation += row[6:].strip('\\n')\n",
    "            continue\n",
    "\n",
    "        if row.startswith('PMID'):\n",
    "            pmc_id = row[6:-1]\n",
    "            prev_row = 'PMID'\n",
    "\n",
    "        if row.startswith('TI'):\n",
    "            title = row[6:].strip('\\n')\n",
    "            prev_row = 'TI'\n",
    "\n",
    "        if row.startswith('AB'):\n",
    "            abstract = row[6:].strip('\\n')\n",
    "            prev_row = 'AB'\n",
    "\n",
    "        if row.startswith('FAU'):\n",
    "            author = row[6:].strip('\\n')\n",
    "            author = author.split(', ')\n",
    "            author = author[0] + ' ' + author[1]\n",
    "            authors.append(author)\n",
    "            prev_row = 'FAU'\n",
    "\n",
    "        if row.startswith('AD') and no_more_AD == False:\n",
    "            affiliation += row[6:]\n",
    "            prev_row = 'AD'\n",
    "            no_more_AD = True\n",
    "    \n",
    "    dict_output = {\n",
    "        'PMID': pmc_id,\n",
    "        'title': title,\n",
    "        'authors': authors,\n",
    "        'affiliation': affiliation,\n",
    "        'abstract': abstract\n",
    "    }\n",
    "    if len(dict_output['authors']) == 0:\n",
    "        dict_output['authors'] = ['unknown']\n",
    "    return dict_output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "articles = [extract_from_pmc_text(t) for t in text]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "for a in articles:\n",
    "    with open('./abstracts only/pmid'+a['PMID']+'.json', 'w') as f:\n",
    "        json.dump(a, f, indent=6)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['PMID- 30244697\\n',\n",
       " 'TI  - Synthetic Biology Ethics at iGEM: iGEMer Perspectives.\\n',\n",
       " 'AB  - The Human Practice (HP) work of the international Genetically Engineered Machine \\n',\n",
       " '      (iGEM) competition can serve as a great example of integrating ethical \\n',\n",
       " '      considerations into synthetic biology research. By highlighting three independent \\n',\n",
       " '      perspectives from those involved in various aspects of iGEM, here we aim to provide \\n',\n",
       " '      an informative picture of how ethical issues are approached within the iGEM \\n',\n",
       " '      competition.\\n',\n",
       " 'FAU - Whitford, Christopher M\\n',\n",
       " 'AU  - Whitford CM\\n',\n",
       " 'AD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany. Electronic \\n',\n",
       " '      address: whitford@cebitec.uni-bielefeld.de.\\n',\n",
       " 'FAU - Lübke, Nils-Christian\\n',\n",
       " 'AU  - Lübke NC\\n',\n",
       " 'AD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany; \\n',\n",
       " '      Teutolab-Biotechnology, Faculty of Biology, Bielefeld University, 33615 Bielefeld, \\n',\n",
       " '      Germany.\\n',\n",
       " 'FAU - Rückert, Christian\\n',\n",
       " 'AU  - Rückert C\\n',\n",
       " 'AD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany; Department \\n',\n",
       " '      of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\n',\n",
       " '      2018 Jul 7.\\n']"
      ]
     },
     "execution_count": 205,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "extract"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'30244697'"
      ]
     },
     "execution_count": 206,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pmc_id = text[0][6:-1]\n",
    "pmc_id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'OWN - NLM\\n'"
      ]
     },
     "execution_count": 207,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "title = text[1]\n",
    "title"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 160,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Whitford, Christopher M\\nAU  - Whitford CM\\nAD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany. Electronic \\n      address: whitford@cebitec.uni-bielefeld.de.\\nFAU - Lübke, Nils-Christian\\nAU  - Lübke NC\\nAD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany; \\n      Teutolab-Biotechnology, Faculty of Biology, Bielefeld University, 33615 Bielefeld, \\n      Germany.\\nFAU - Rückert, Christian\\nAU  - Rückert C\\nAD  - Center for Biotechnology, Bielefeld University, 33615 Bielefeld, Germany; Department \\n      of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20180707\\nPL  - England\\nTA  - Trends Biotechnol\\nJT  - Trends in biotechnology\\nJID - 8310903\\nMH  - Genetic Engineering/*ethics\\nMH  - Humans\\nMH  - Synthetic Biology/*ethics\\nOTO - NOTNLM\\nOT  - *ethics\\nOT  - *human practice\\nOT  - *iGEM\\nOT  - *synthetic biology\\nEDAT- 2018/09/25 06:00\\nMHDA- 2019/07/25 06:00\\nCRDT- 2018/09/25 06:00\\nPHST- 2018/04/16 00:00 [received]\\nPHST- 2018/06/15 00:00 [revised]\\nPHST- 2018/06/18 00:00 [accepted]\\nPHST- 2018/09/25 06:00 [entrez]\\nPHST- 2018/09/25 06:00 [pubmed]\\nPHST- 2019/07/25 06:00 [medline]\\nAID - S0167-7799(18)30168-9 [pii]\\nAID - 10.1016/j.tibtech.2018.06.004 [doi]\\nPST - ppublish\\nSO  - Trends Biotechnol. 2018 Oct;36(10):985-987. doi: 10.1016/j.tibtech.2018.06.004. Epub \\n      2018 Jul 7.\\n\\n'"
      ]
     },
     "execution_count": 160,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "authors = []\n",
    "name = text[re.search('FAU - ', text).end():]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Retrieving information about Citations using doi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "from bs4 import BeautifulSoup\n",
    "import requests\n",
    "import json\n",
    "import time\n",
    "from tqdm import tqdm\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### ACS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [],
   "source": [
    "dict_acs = {}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [],
   "source": [
    "list_of_acs = os.listdir('../topic modeling/data/json-files')\n",
    "list_of_dois = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "metadata": {},
   "outputs": [],
   "source": [
    "for iD in list_of_acs:\n",
    "    with open('../topic modeling/data/json-files/'+iD, 'r') as fp:\n",
    "        file = json.load(fp)\n",
    "        if file['is_research']:\n",
    "            list_of_dois.append(file['article_id'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_doi_from_acs(iD):\n",
    "    # get original html\n",
    "    contents = requests.get('https://pubs.acs.org/doi/{}'.format(iD)).content\n",
    "    \n",
    "    # look for cited content (div)\n",
    "    soup = BeautifulSoup(contents,'lxml')\n",
    "    citation_parts = soup.find_all('div', class_ = 'cited-by__content')\n",
    "    refs = citation_parts[0].find_all(attrs = {'title': 'DOI URL'})\n",
    "    \n",
    "    return [i.text for i in refs]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 907/907 [56:58<00:00,  3.77s/it]  \n"
     ]
    }
   ],
   "source": [
    "for i in tqdm(list_of_dois):\n",
    "    dict_acs[i] = get_doi_from_acs(i)\n",
    "    time.sleep(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('./references_acs.json', 'w') as f:\n",
    "    json.dump(dict_acs, f, indent = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Ethics (Pubmed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [],
   "source": [
    "pubmed_ids = [f.split('.')[0][4:] for f in os.listdir('./abstracts only/') if f.endswith('.json')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_doi(text):\n",
    "    if 'doi' in text:\n",
    "        return 'https://doi.org/' + text.split('doi: ')[1].split(' ')[0].strip('.')\n",
    "    return None\n",
    "\n",
    "def get_doi_from_pubmed(iD):\n",
    "    # fetch the original html\n",
    "    contents2 = requests.get('https://pubmed.ncbi.nlm.nih.gov/{}'.format(iD)).content\n",
    "    soup2 = BeautifulSoup(contents2,'lxml')\n",
    "    \n",
    "    # try going to the link that provides all the citations\n",
    "    try:\n",
    "        link_citations = 'https://pubmed.ncbi.nlm.nih.gov/' + soup2.find_all('a',class_='usa-button show-all-linked-articles')[1].get('data-href')\n",
    "        citations = requests.get(link_citations).content\n",
    "        soup3 = BeautifulSoup(citations,'lxml')\n",
    "        result = [extract_doi(i.text) for i in soup3.find_all('span', class_ = 'docsum-journal-citation full-journal-citation')]\n",
    "    except:\n",
    "    # if the webpage only contains a few citations, it doesn't have a full link\n",
    "        try:\n",
    "            result = soup2.find_all(class_ = 'citedby-articles')[0].find_all(class_ = 'docsum-journal-citation full-journal-citation')\n",
    "            result = [extract_doi(i.text) for i in result]\n",
    "        except:\n",
    "    # for edge cases, we will just do nothing.\n",
    "            print(iD)\n",
    "            return []\n",
    "    return [r for r in result if r is not None]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {},
   "outputs": [],
   "source": [
    "dict_pubmed = {}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 10%|▉         | 19/200 [00:56<08:20,  2.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33123514\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|█         | 20/200 [00:59<08:14,  2.75s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28692329\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 13%|█▎        | 26/200 [01:16<08:01,  2.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29761363\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 14%|█▍        | 28/200 [01:21<07:38,  2.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29900853\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 15%|█▌        | 30/200 [01:27<07:39,  2.70s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29672769\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|█▌        | 31/200 [01:29<07:34,  2.69s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32722674\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 17%|█▋        | 34/200 [01:38<07:32,  2.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "27325057\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 19%|█▉        | 38/200 [01:51<08:12,  3.04s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33633695\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 22%|██▏       | 44/200 [02:08<07:36,  2.92s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "23502566\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 24%|██▍       | 48/200 [02:20<07:18,  2.88s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31399264\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|██▍       | 49/200 [02:22<07:00,  2.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31625897\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 26%|██▌       | 52/200 [02:30<06:39,  2.70s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26160061\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|██▋       | 53/200 [02:33<06:27,  2.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21189731\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 30%|███       | 60/200 [02:54<06:29,  2.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24800469\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 31%|███       | 62/200 [03:00<06:38,  2.89s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361719\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 34%|███▍      | 68/200 [03:19<06:50,  3.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "27633830\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|███▍      | 69/200 [03:22<06:24,  2.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361723\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 38%|███▊      | 77/200 [03:45<05:40,  2.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25845210\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 40%|████      | 81/200 [03:57<05:37,  2.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29076055\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 42%|████▎     | 85/200 [04:09<05:37,  2.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26208557\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|████▎     | 86/200 [04:11<05:21,  2.82s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26644292\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 44%|████▍     | 88/200 [04:17<05:05,  2.73s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26152897\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|████▍     | 89/200 [04:19<04:57,  2.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "26238764\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 46%|████▋     | 93/200 [04:31<05:10,  2.90s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21535064\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|████▋     | 94/200 [04:34<04:52,  2.76s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33132748\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|████▊     | 95/200 [04:36<04:44,  2.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "30634968\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|████▊     | 96/200 [04:39<04:39,  2.69s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361722\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 50%|█████     | 100/200 [04:50<04:31,  2.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361718\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|█████     | 101/200 [04:52<04:24,  2.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29457239\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|█████     | 102/200 [04:55<04:17,  2.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33476383\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|█████▏    | 103/200 [04:58<04:25,  2.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25032374\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 52%|█████▎    | 105/200 [05:04<04:42,  2.97s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29368114\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|█████▎    | 106/200 [05:07<04:27,  2.85s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361721\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 56%|█████▋    | 113/200 [05:27<04:06,  2.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "23834596\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 59%|█████▉    | 118/200 [05:42<03:53,  2.84s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28543647\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|█████▉    | 119/200 [05:44<03:48,  2.82s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29255953\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|██████    | 120/200 [05:47<03:39,  2.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33869403\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 62%|██████▏   | 124/200 [05:58<03:32,  2.80s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25845204\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 63%|██████▎   | 126/200 [06:04<03:30,  2.84s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "30198056\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 64%|██████▍   | 128/200 [06:10<03:15,  2.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25032403\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 65%|██████▌   | 130/200 [06:15<03:09,  2.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28979291\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|██████▌   | 131/200 [06:18<03:06,  2.70s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "22653442\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 70%|███████   | 141/200 [06:46<02:45,  2.81s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32075446\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|███████   | 142/200 [06:49<02:37,  2.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25845209\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 72%|███████▏  | 144/200 [06:54<02:37,  2.81s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28749058\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 75%|███████▌  | 150/200 [07:11<02:13,  2.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24340832\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|███████▌  | 151/200 [07:14<02:08,  2.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28361720\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 77%|███████▋  | 154/200 [07:23<02:12,  2.88s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "23502565\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 78%|███████▊  | 156/200 [07:28<02:01,  2.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33934835\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 79%|███████▉  | 158/200 [07:34<01:55,  2.75s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28543657\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 80%|████████  | 160/200 [07:39<01:47,  2.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32648715\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 83%|████████▎ | 166/200 [07:56<01:33,  2.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29453838\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 85%|████████▌ | 170/200 [08:06<01:18,  2.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28941334\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 87%|████████▋ | 174/200 [08:18<01:10,  2.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24872969\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 88%|████████▊ | 176/200 [08:23<01:04,  2.69s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "29936670\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 89%|████████▉ | 178/200 [08:28<00:59,  2.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32442798\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 91%|█████████ | 182/200 [08:39<00:47,  2.66s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "33977723\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|█████████▏| 183/200 [08:42<00:46,  2.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32309535\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|█████████▏| 184/200 [08:44<00:42,  2.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24010855\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 94%|█████████▍| 189/200 [09:00<00:32,  2.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "30006902\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 97%|█████████▋| 194/200 [09:14<00:16,  2.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "23502564\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 200/200 [09:30<00:00,  2.85s/it]\n"
     ]
    }
   ],
   "source": [
    "for i in tqdm(pubmed_ids):\n",
    "    time.sleep(2)\n",
    "    dict_pubmed['pmid'+i] = get_doi_from_pubmed(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('./references_pubmed.json', 'w') as f:\n",
    "    json.dump(dict_pubmed, f, indent = 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Appendix"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Attributes of PMC .xml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['sub-article', 'response', 'front', 'body'])"
      ]
     },
     "execution_count": 103,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 108,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0]['sub-article']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['list', 'notes', 'def-list', 'journal-meta', 'article-meta'])"
      ]
     },
     "execution_count": 104,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0]['front'].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['table-wrap-group', 'ack', 'chem-struct', 'mml:math', 'related-article', 'boxed-text', 'media', 'table-wrap', 'preformat', 'chem-struct-wrapper', 'disp-formula', 'array', 'list', 'tex-math', 'sec', 'supplementary-material', 'statement', 'graphic', 'verse-group', 'def-list', 'p', 'fig', 'disp-quote', 'fig-group', 'speech'])"
      ]
     },
     "execution_count": 105,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0]['body'].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pprint"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'abstract': [{'ack': [],\n",
      "               'array': [],\n",
      "               'boxed-text': [],\n",
      "               'chem-struct': [],\n",
      "               'chem-struct-wrapper': [],\n",
      "               'def-list': [],\n",
      "               'disp-formula': [],\n",
      "               'disp-quote': [],\n",
      "               'fig': [],\n",
      "               'fig-group': [],\n",
      "               'fn-group': [],\n",
      "               'glossary': [],\n",
      "               'graphic': [],\n",
      "               'list': [],\n",
      "               'media': [],\n",
      "               'mml:math': [],\n",
      "               'notes': [],\n",
      "               'object-id': [],\n",
      "               'p': ['The 12th meeting of the Scientific Group on Methodologies for the Safety Evaluation of Chemicals (SGOMSEC) considered the topic of methodologies for determining human and ecosystem susceptibility to environmental hazards. The report prepared at the meeting describes measurement of susceptibility through the use of biological markers of exposure, biological markers of effect, and biomarkers directly indicative of susceptibility of humans or of ecosystems. The utility and validity of these biological markers for the study of susceptibility are evaluated, as are opportunities for developing newer approaches for the study of humans or of ecosystems. For the first time a SGOMSEC workshop also formally considered the issue of ethics in relation to methodology, an issue of particular concern for studies of susceptibility.'],\n",
      "               'preformat': [],\n",
      "               'ref-list': [],\n",
      "               'related-article': [],\n",
      "               'sec': [],\n",
      "               'speech': [],\n",
      "               'statement': [],\n",
      "               'supplementary-material': [],\n",
      "               'table-wrap': [],\n",
      "               'table-wrap-group': [],\n",
      "               'tex-math': [],\n",
      "               'verse-group': []}],\n",
      " 'aff': ['National Institute of Environmental Health Sciences, Research Triangle Park, North Carolina, USA.'],\n",
      " 'article-categories': {'series-title': [],\n",
      "                        'subj-group': [{'subj-group': [],\n",
      "                                        'subject': ['Research Article']}]},\n",
      " 'article-id': [StringElement('9255554', attributes={'pub-id-type': 'pmid'}),\n",
      "                StringElement('1470055', attributes={'pub-id-type': 'pmc'})],\n",
      " 'conference': [],\n",
      " 'contract-num': [],\n",
      " 'contract-sponsor': [],\n",
      " 'contrib-group': [{'address': [],\n",
      "                    'aff': [],\n",
      "                    'author-comment': [],\n",
      "                    'bio': [],\n",
      "                    'contrib': [{'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'J C',\n",
      "                                           'surname': 'Barrett'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'H',\n",
      "                                           'surname': 'Vainio'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'D',\n",
      "                                           'surname': 'Peakall'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'B D',\n",
      "                                           'surname': 'Goldstein'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []}],\n",
      "                    'email': [],\n",
      "                    'etal': [],\n",
      "                    'ext-link': [],\n",
      "                    'fn': [],\n",
      "                    'on-behalf-of': [],\n",
      "                    'role': [],\n",
      "                    'uri': [],\n",
      "                    'x': [],\n",
      "                    'xref': []}],\n",
      " 'email': [],\n",
      " 'ext-link': [],\n",
      " 'fpage': '699',\n",
      " 'issue': 'Suppl 4',\n",
      " 'issue-id': [],\n",
      " 'issue-title': [],\n",
      " 'kwd-group': [],\n",
      " 'lpage': '737',\n",
      " 'product': [],\n",
      " 'pub-date': [{'month': '6',\n",
      "               'year': '1997'}],\n",
      " 'related-article': [],\n",
      " 'self-uri': [],\n",
      " 'supplementary-material': [],\n",
      " 'title-group': {'alt-title': [],\n",
      "                 'article-title': '12th meeting of the Scientific Group on Methodologies for the Safety Evaluation of Chemicals: susceptibility to environmental hazards.',\n",
      "                 'subtitle': [],\n",
      "                 'trans-title': []},\n",
      " 'trans-abstract': [],\n",
      " 'uri': [],\n",
      " 'volume': '105',\n",
      " 'volume-id': [],\n",
      " 'x': []}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(parsed[0]['front']['article-meta'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "def alternating_sign_product(values):\n",
    "    product=1\n",
    "    for i in range(0,len(values)):\n",
    "        product=product*values[i]\n",
    "    print(product)\n",
    "    if len(values)%2==0:\n",
    "        return product\n",
    "    else:\n",
    "        return -product"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "688.5\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "-688.5"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "alternating_sign_product(np.array([2, 8.5, 1, 9, 4.5]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
