{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pickle as pickle\n",
    "from pprint import pprint\n",
    "import pandas as pd\n",
    "import re\n",
    "import pyLDAvis\n",
    "import pyLDAvis.gensim_models as gensimvis\n",
    "import matplotlib.pyplot as plt\n",
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "%config InlineBackend.figure_format = 'retina'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load Model\n",
    "with open(\"./data/saved_results/model/lda_k=5.pickle\", \"rb\") as fp:\n",
    "    lda_5 = pickle.load(fp)\n",
    "    \n",
    "# datawords\n",
    "with open(\"./data/saved_results/data_words/ner=20210205_q=0.005.pickle\", \"rb\") as fp:\n",
    "    data_word = pickle.load(fp)\n",
    "\n",
    "# load corpus\n",
    "with open(\"./data/saved_results/data_words/ner=20210205_q=0.005_corpus.pickle\", \"rb\") as fp:\n",
    "    corpus = pickle.load(fp)\n",
    "    \n",
    "# load doc_lda\n",
    "with open(\"./data/saved_results/model/lda_k=5_abstract_doc_lda.pickle\", \"rb\") as fp:\n",
    "    doc_lda = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[(0, 0.34567901234567905),\n",
       "  (1, 0.10789049919484703),\n",
       "  (2, 0.3332566520972319),\n",
       "  (3, 0.16455793267387472),\n",
       "  (4, 0.04861590368836746)],\n",
       " [(0, 0.2620721987810595),\n",
       "  (1, 0.03234880450070323),\n",
       "  (2, 0.01915477864844953),\n",
       "  (3, 0.6599022168642421),\n",
       "  (4, 0.026522001205545504)],\n",
       " [(0, 0.7624306527712965),\n",
       "  (1, 0.09571213770933787),\n",
       "  (2, 0.06175143879296934),\n",
       "  (3, 0.032975579405817385),\n",
       "  (4, 0.047130191320578614)],\n",
       " [(0, 0.020822397200349955),\n",
       "  (1, 0.023272090988626423),\n",
       "  (2, 0.931350247885681),\n",
       "  (3, 0.014756488772236804),\n",
       "  (4, 0.009798775153105862)],\n",
       " [(0, 0.03267195767195767),\n",
       "  (1, 0.5948853615520282),\n",
       "  (2, 0.014109347442680775),\n",
       "  (3, 0.136331569664903),\n",
       "  (4, 0.22200176366843033)],\n",
       " [(0, 0.048171966178397044),\n",
       "  (1, 0.03924020483506014),\n",
       "  (2, 0.18065975943789447),\n",
       "  (3, 0.12355603191616053),\n",
       "  (4, 0.6083720376324878)],\n",
       " [(0, 0.04344418318217445),\n",
       "  (1, 0.8630239241592954),\n",
       "  (2, 0.012018064419811146),\n",
       "  (3, 0.05617138804911732),\n",
       "  (4, 0.025342440189601766)],\n",
       " [(0, 0.016300845369422647),\n",
       "  (1, 0.6474089237651162),\n",
       "  (2, 0.009818416164373176),\n",
       "  (3, 0.026043443648356646),\n",
       "  (4, 0.30042837105273135)],\n",
       " [(0, 0.0558859624280185),\n",
       "  (1, 0.02152364769187199),\n",
       "  (2, 0.022373265363919568),\n",
       "  (3, 0.6898895497026338),\n",
       "  (4, 0.21032757481355613)],\n",
       " [(0, 0.020974706971005553),\n",
       "  (1, 0.03893344300500377),\n",
       "  (2, 0.884296387689355),\n",
       "  (3, 0.027760641579272053),\n",
       "  (4, 0.02803482075536363)],\n",
       " [(0, 0.03960159837776585),\n",
       "  (1, 0.028627661477903025),\n",
       "  (2, 0.11665772052245482),\n",
       "  (3, 0.11182680264805869),\n",
       "  (4, 0.7032862169738177)],\n",
       " [(0, 0.6445605149308853),\n",
       "  (1, 0.1208715838345468),\n",
       "  (2, 0.033976996939959904),\n",
       "  (3, 0.150627835813021),\n",
       "  (4, 0.049963068481587)],\n",
       " [(0, 0.04708390963577686),\n",
       "  (1, 0.7902259105578608),\n",
       "  (2, 0.03567312125403412),\n",
       "  (3, 0.028872752420470268),\n",
       "  (4, 0.09814430613185801)],\n",
       " [(0, 0.00858648056923919),\n",
       "  (1, 0.005541871921182266),\n",
       "  (2, 0.6178845101258894),\n",
       "  (3, 0.3609058565955118),\n",
       "  (4, 0.00708128078817734)],\n",
       " [(0, 0.020820449391877962),\n",
       "  (1, 0.025080739366453653),\n",
       "  (2, 0.9097780526351955),\n",
       "  (3, 0.02370645227788085),\n",
       "  (4, 0.02061430632859204)],\n",
       " [(0, 0.1946082136558327),\n",
       "  (1, 0.051196775006298816),\n",
       "  (2, 0.011438649533887629),\n",
       "  (3, 0.12249937011841774),\n",
       "  (4, 0.6202569916855631)],\n",
       " [(0, 0.49625715543813304),\n",
       "  (1, 0.011546553158177994),\n",
       "  (2, 0.3695386271344),\n",
       "  (3, 0.08425069719653605),\n",
       "  (4, 0.038406967072753075)],\n",
       " [(0, 0.015944839474251238),\n",
       "  (1, 0.06277382747970983),\n",
       "  (2, 0.01185089420383538),\n",
       "  (3, 0.8173525820584644),\n",
       "  (4, 0.09207785678373914)],\n",
       " [(0, 0.1648612400701537),\n",
       "  (1, 0.06876096151862168),\n",
       "  (2, 0.40730424017332095),\n",
       "  (3, 0.32956772928917777),\n",
       "  (4, 0.029505828948725884)],\n",
       " [(0, 0.2676261458065622),\n",
       "  (1, 0.18332904994431598),\n",
       "  (2, 0.1667095005568406),\n",
       "  (3, 0.2903281075987322),\n",
       "  (4, 0.09200719609354922)],\n",
       " [(0, 0.33208056278307796),\n",
       "  (1, 0.017683338151681605),\n",
       "  (2, 0.015515081430085767),\n",
       "  (3, 0.5232244386624265),\n",
       "  (4, 0.11149657897272815)],\n",
       " [(0, 0.02103376721373326),\n",
       "  (1, 0.8499339747217506),\n",
       "  (2, 0.011743067345783814),\n",
       "  (3, 0.09950952650443312),\n",
       "  (4, 0.01777966421429919)],\n",
       " [(0, 0.07735870427001926),\n",
       "  (1, 0.13886057311133765),\n",
       "  (2, 0.5760561785026617),\n",
       "  (3, 0.0345452486125269),\n",
       "  (4, 0.1731792955034545)],\n",
       " [(0, 0.0541343669250646),\n",
       "  (1, 0.10142118863049096),\n",
       "  (2, 0.05749354005167959),\n",
       "  (3, 0.25045219638242894),\n",
       "  (4, 0.536498708010336)],\n",
       " [(0, 0.06623093681917211),\n",
       "  (1, 0.3748148148148148),\n",
       "  (2, 0.054422657952069714),\n",
       "  (3, 0.11389978213507625),\n",
       "  (4, 0.3906318082788671)],\n",
       " [(0, 0.09108367626886145),\n",
       "  (1, 0.17386831275720166),\n",
       "  (2, 0.27866941015089164),\n",
       "  (3, 0.0803840877914952),\n",
       "  (4, 0.37599451303155007)],\n",
       " [(0, 0.015891523081573603),\n",
       "  (1, 0.2765470484086885),\n",
       "  (2, 0.6759511162931294),\n",
       "  (3, 0.0190439176059075),\n",
       "  (4, 0.012566394610700865)],\n",
       " [(0, 0.324769647696477),\n",
       "  (1, 0.034688346883468835),\n",
       "  (2, 0.06997289972899728),\n",
       "  (3, 0.50710027100271),\n",
       "  (4, 0.06346883468834688)],\n",
       " [(0, 0.04427549194991056),\n",
       "  (1, 0.4683959451401312),\n",
       "  (2, 0.01888292585967005),\n",
       "  (3, 0.249950308089843),\n",
       "  (4, 0.21849532896044527)],\n",
       " [(0, 0.7804366722912512),\n",
       "  (1, 0.12082630260264866),\n",
       "  (2, 0.022498338190929078),\n",
       "  (3, 0.04167305824001636),\n",
       "  (4, 0.034565628675154676)],\n",
       " [(0, 0.010325604966425947),\n",
       "  (1, 0.24832129735208414),\n",
       "  (2, 0.006746484226529836),\n",
       "  (3, 0.008171797795515013),\n",
       "  (4, 0.7264348156594451)],\n",
       " [(0, 0.020090026060175313),\n",
       "  (1, 0.013361762615493959),\n",
       "  (2, 0.8772802653399668),\n",
       "  (3, 0.06453447050461976),\n",
       "  (4, 0.024733475479744138)],\n",
       " [(0, 0.19214667244117223),\n",
       "  (1, 0.01174149463452192),\n",
       "  (2, 0.7456811510514412),\n",
       "  (3, 0.04022905538713248),\n",
       "  (4, 0.01020162648573216)],\n",
       " [(0, 0.5755853994490359),\n",
       "  (1, 0.014003673094582185),\n",
       "  (2, 0.017447199265381085),\n",
       "  (3, 0.380050505050505),\n",
       "  (4, 0.012913223140495868)],\n",
       " [(0, 0.01570271714555843),\n",
       "  (1, 0.14925802606157768),\n",
       "  (2, 0.009988901220865706),\n",
       "  (3, 0.012907469067291486),\n",
       "  (4, 0.8121428865047067)],\n",
       " [(0, 0.25678270306369483),\n",
       "  (1, 0.6076467150847317),\n",
       "  (2, 0.04958677685950415),\n",
       "  (3, 0.027214291677101602),\n",
       "  (4, 0.058769513314967874)],\n",
       " [(0, 0.021227559437166425),\n",
       "  (1, 0.04338508814491347),\n",
       "  (2, 0.0576580947759987),\n",
       "  (3, 0.03990781174187288),\n",
       "  (4, 0.8378214459000485)],\n",
       " [(0, 0.02207505518763797),\n",
       "  (1, 0.06130073017490236),\n",
       "  (2, 0.011999773589177563),\n",
       "  (3, 0.8903039565291221),\n",
       "  (4, 0.014320484519160017)],\n",
       " [(0, 0.10111937495304636),\n",
       "  (1, 0.09071444669821953),\n",
       "  (2, 0.02775899631883405),\n",
       "  (3, 0.7689504920742244),\n",
       "  (4, 0.011456689955675758)],\n",
       " [(0, 0.2726658559654101),\n",
       "  (1, 0.014277349907670135),\n",
       "  (2, 0.021933972886546863),\n",
       "  (3, 0.2224474170157186),\n",
       "  (4, 0.4686754042246543)],\n",
       " [(0, 0.03987203495630462),\n",
       "  (1, 0.3693820224719101),\n",
       "  (2, 0.02200374531835206),\n",
       "  (3, 0.5465823970037453),\n",
       "  (4, 0.02215980024968789)],\n",
       " [(0, 0.41605082328536236),\n",
       "  (1, 0.09490470633994555),\n",
       "  (2, 0.2711655646311422),\n",
       "  (3, 0.0338389731621937),\n",
       "  (4, 0.18403993258135615)],\n",
       " [(0, 0.10360252416327183),\n",
       "  (1, 0.09281891524882178),\n",
       "  (2, 0.020289160476076363),\n",
       "  (3, 0.7012540937774583),\n",
       "  (4, 0.08203530633437175)],\n",
       " [(0, 0.24210430092783033),\n",
       "  (1, 0.21070432835138717),\n",
       "  (2, 0.13583801819095936),\n",
       "  (3, 0.16751222633575574),\n",
       "  (4, 0.24384112619406736)],\n",
       " [(0, 0.1058806898501555),\n",
       "  (1, 0.11867401752897937),\n",
       "  (2, 0.3771557817359344),\n",
       "  (3, 0.08396946564885496),\n",
       "  (4, 0.31432004523607576)],\n",
       " [(0, 0.020950729663343447),\n",
       "  (1, 0.6308336945528102),\n",
       "  (2, 0.026104127534556664),\n",
       "  (3, 0.11607185859461543),\n",
       "  (4, 0.20603958965467417)],\n",
       " [(0, 0.042775791285922404),\n",
       "  (1, 0.8291617004370283),\n",
       "  (2, 0.046042466781441756),\n",
       "  (3, 0.04696949631395401),\n",
       "  (4, 0.03505054518165365)],\n",
       " [(0, 0.025119011445356023),\n",
       "  (1, 0.13825584928593132),\n",
       "  (2, 0.00891319760964246),\n",
       "  (3, 0.7342752962625342),\n",
       "  (4, 0.09343664539653601)],\n",
       " [(0, 0.13485872768244064),\n",
       "  (1, 0.08545675160339747),\n",
       "  (2, 0.022794245103137458),\n",
       "  (3, 0.734009360374415),\n",
       "  (4, 0.022880915236609463)],\n",
       " [(0, 0.18864763436536658),\n",
       "  (1, 0.11808068623508913),\n",
       "  (2, 0.019032301300093823),\n",
       "  (3, 0.6277978823214047),\n",
       "  (4, 0.046441495778045835)],\n",
       " [(0, 0.013785350846053722),\n",
       "  (1, 0.21890900485149686),\n",
       "  (2, 0.01668441604543841),\n",
       "  (3, 0.06809844988758727),\n",
       "  (4, 0.6825227783694238)],\n",
       " [(0, 0.009938090583251874),\n",
       "  (1, 0.7528918214402085),\n",
       "  (2, 0.13864450961225155),\n",
       "  (3, 0.045739654610622356),\n",
       "  (4, 0.05278592375366569)],\n",
       " [(0, 0.13444990135583262),\n",
       "  (1, 0.025521554799983213),\n",
       "  (2, 0.5440540653989843),\n",
       "  (3, 0.15422071107753016),\n",
       "  (4, 0.14175376736766993)],\n",
       " [(0, 0.3753280839895013),\n",
       "  (1, 0.021405657626130066),\n",
       "  (2, 0.0979294254884806),\n",
       "  (3, 0.06275882181393992),\n",
       "  (4, 0.4425780110819481)],\n",
       " [(0, 0.13528318005929946),\n",
       "  (1, 0.6861651339263279),\n",
       "  (2, 0.119503492637821),\n",
       "  (3, 0.02969998492386552),\n",
       "  (4, 0.029348208452686066)],\n",
       " [(0, 0.07857142857142857),\n",
       "  (1, 0.32185185185185183),\n",
       "  (2, 0.017566137566137566),\n",
       "  (3, 0.5664021164021164),\n",
       "  (4, 0.015608465608465608)],\n",
       " [(0, 0.22198046610579247),\n",
       "  (1, 0.37979885891113047),\n",
       "  (2, 0.06479063920317184),\n",
       "  (3, 0.06885214195919157),\n",
       "  (4, 0.26457789382071367)],\n",
       " [(0, 0.2096505823627288),\n",
       "  (1, 0.656128674431503),\n",
       "  (2, 0.02323288346582856),\n",
       "  (3, 0.05922228384790781),\n",
       "  (4, 0.0517655758920318)],\n",
       " [(0, 0.043791956835435096),\n",
       "  (1, 0.018068887634105024),\n",
       "  (2, 0.10929167450906581),\n",
       "  (3, 0.027166070644331512),\n",
       "  (4, 0.8016814103770625)],\n",
       " [(0, 0.10162327442780701),\n",
       "  (1, 0.7142857142857143),\n",
       "  (2, 0.06304240298574576),\n",
       "  (3, 0.0947884347317775),\n",
       "  (4, 0.02626017356895544)],\n",
       " [(0, 0.030243261012491782),\n",
       "  (1, 0.07955292570677186),\n",
       "  (2, 0.023997370151216304),\n",
       "  (3, 0.1494630725399956),\n",
       "  (4, 0.7167433705895244)],\n",
       " [(0, 0.06897219800445606),\n",
       "  (1, 0.08883076625012107),\n",
       "  (2, 0.6330524072459556),\n",
       "  (3, 0.17204301075268816),\n",
       "  (4, 0.03710161774677903)],\n",
       " [(0, 0.521085617165923),\n",
       "  (1, 0.21855746760144465),\n",
       "  (2, 0.18403441682600383),\n",
       "  (3, 0.017155300616103677),\n",
       "  (4, 0.05916719779052475)],\n",
       " [(0, 0.10987928061098792),\n",
       "  (1, 0.3246119733924612),\n",
       "  (2, 0.48016752894801673),\n",
       "  (3, 0.023059866962305987),\n",
       "  (4, 0.062281350086228135)],\n",
       " [(0, 0.019321953532479848),\n",
       "  (1, 0.8532874980243401),\n",
       "  (2, 0.010550023707918445),\n",
       "  (3, 0.06729097518571203),\n",
       "  (4, 0.04954954954954955)],\n",
       " [(0, 0.6628151260504201),\n",
       "  (1, 0.027252567693744165),\n",
       "  (2, 0.2456816059757236),\n",
       "  (3, 0.020074696545284782),\n",
       "  (4, 0.04417600373482727)],\n",
       " [(0, 0.07501858045336306),\n",
       "  (1, 0.701783723522854),\n",
       "  (2, 0.027452619843924192),\n",
       "  (3, 0.040551839464882944),\n",
       "  (4, 0.15519323671497584)],\n",
       " [(0, 0.033343012001548586),\n",
       "  (1, 0.048828881145954316),\n",
       "  (2, 0.8293650793650794),\n",
       "  (3, 0.05361982191250484),\n",
       "  (4, 0.03484320557491289)],\n",
       " [(0, 0.021112199368810536),\n",
       "  (1, 0.009359016215039722),\n",
       "  (2, 0.026825552290782456),\n",
       "  (3, 0.13576014800304712),\n",
       "  (4, 0.8069430841223202)],\n",
       " [(0, 0.046476437527021186),\n",
       "  (1, 0.11062472978815391),\n",
       "  (2, 0.7134673584089927),\n",
       "  (3, 0.023670557717250323),\n",
       "  (4, 0.10576091655858193)],\n",
       " [(0, 0.1375506893755069),\n",
       "  (1, 0.028764530954311977),\n",
       "  (2, 0.08775344687753447),\n",
       "  (3, 0.04460665044606651),\n",
       "  (4, 0.7013246823465802)],\n",
       " [(0, 0.013909670268772514),\n",
       "  (1, 0.9255749515101136),\n",
       "  (2, 0.009531726239955667),\n",
       "  (3, 0.034746467165419785),\n",
       "  (4, 0.016237184815738433)],\n",
       " [(0, 0.2785070785070785),\n",
       "  (1, 0.03105963105963106),\n",
       "  (2, 0.29000429000429),\n",
       "  (3, 0.37125697125697127),\n",
       "  (4, 0.02917202917202917)],\n",
       " [(0, 0.14360878984834416),\n",
       "  (1, 0.2793562364593005),\n",
       "  (2, 0.05954812751470133),\n",
       "  (3, 0.48765088207985147),\n",
       "  (4, 0.029835964097802536)],\n",
       " [(0, 0.18112373737373738),\n",
       "  (1, 0.24867424242424244),\n",
       "  (2, 0.01792929292929293),\n",
       "  (3, 0.522159090909091),\n",
       "  (4, 0.030113636363636363)],\n",
       " [(0, 0.03028574145390577),\n",
       "  (1, 0.02039651975467123),\n",
       "  (2, 0.04773451243284363),\n",
       "  (3, 0.050492083868207104),\n",
       "  (4, 0.8510911424903723)],\n",
       " [(0, 0.028904338869576416),\n",
       "  (1, 0.015385605767992791),\n",
       "  (2, 0.9060126174842282),\n",
       "  (3, 0.022853096433629462),\n",
       "  (4, 0.0268443414445732)],\n",
       " [(0, 0.03554522764437559),\n",
       "  (1, 0.03472760134262845),\n",
       "  (2, 0.8825200103279112),\n",
       "  (3, 0.019364833462432222),\n",
       "  (4, 0.027842327222652553)],\n",
       " [(0, 0.33306853087596655),\n",
       "  (1, 0.45752568583836456),\n",
       "  (2, 0.022667090350598454),\n",
       "  (3, 0.12106768350810296),\n",
       "  (4, 0.06567100942696748)],\n",
       " [(0, 0.1764574691110219),\n",
       "  (1, 0.3277577055176413),\n",
       "  (2, 0.024934207591774834),\n",
       "  (3, 0.033989027164458716),\n",
       "  (4, 0.43686159061510327)],\n",
       " [(0, 0.06056521543783979),\n",
       "  (1, 0.01779215527699151),\n",
       "  (2, 0.6470773239879589),\n",
       "  (3, 0.15972503032753743),\n",
       "  (4, 0.11484027496967247)],\n",
       " [(0, 0.09730572255622358),\n",
       "  (1, 0.47316855934090407),\n",
       "  (2, 0.01454761374601054),\n",
       "  (3, 0.38603132190306544),\n",
       "  (4, 0.028946782453796484)],\n",
       " [(0, 0.029419881878898275),\n",
       "  (1, 0.02726720759507645),\n",
       "  (2, 0.840260528785119),\n",
       "  (3, 0.03654026604846278),\n",
       "  (4, 0.06651211569244357)],\n",
       " [(0, 0.017063957274977205),\n",
       "  (1, 0.01762841387694846),\n",
       "  (2, 0.01016021883548261),\n",
       "  (3, 0.9252746298467284),\n",
       "  (4, 0.0298727801658634)],\n",
       " [(0, 0.03111360518767926),\n",
       "  (1, 0.06578126948497319),\n",
       "  (2, 0.07438583364509291),\n",
       "  (3, 0.021137298915076694),\n",
       "  (4, 0.8075819927671779)],\n",
       " [(0, 0.6319691187436943),\n",
       "  (1, 0.0189498618239242),\n",
       "  (2, 0.028073869368776594),\n",
       "  (3, 0.014607185155941572),\n",
       "  (4, 0.3063999649076633)],\n",
       " [(0, 0.7644113549285961),\n",
       "  (1, 0.019026471612678505),\n",
       "  (2, 0.17681121560431903),\n",
       "  (3, 0.02285788923719958),\n",
       "  (4, 0.016893068617206543)],\n",
       " [(0, 0.003654188948306595),\n",
       "  (1, 0.00368389780154486),\n",
       "  (2, 0.0038324420677361853),\n",
       "  (3, 0.9842840166369579),\n",
       "  (4, 0.004545454545454545)],\n",
       " [(0, 0.044812768569674644),\n",
       "  (1, 0.017955801104972375),\n",
       "  (2, 0.04304788213627993),\n",
       "  (3, 0.18615715162676488),\n",
       "  (4, 0.7080263965623081)],\n",
       " [(0, 0.061925810365747964),\n",
       "  (1, 0.025914369908129658),\n",
       "  (2, 0.03406136245449818),\n",
       "  (3, 0.027084416710001735),\n",
       "  (4, 0.8510140405616224)],\n",
       " [(0, 0.03422683383772878),\n",
       "  (1, 0.022121343133016284),\n",
       "  (2, 0.02601239371667387),\n",
       "  (3, 0.09302493154633232),\n",
       "  (4, 0.8246144977662487)],\n",
       " [(0, 0.4637020144431775),\n",
       "  (1, 0.17737235525148867),\n",
       "  (2, 0.12454073229443811),\n",
       "  (3, 0.050107690358545544),\n",
       "  (4, 0.18427720765235017)],\n",
       " [(0, 0.049153242461792646),\n",
       "  (1, 0.031350681536555146),\n",
       "  (2, 0.8362247005369682),\n",
       "  (3, 0.0503510945890128),\n",
       "  (4, 0.03292028087567121)],\n",
       " [(0, 0.19771676016140144),\n",
       "  (1, 0.10274579273693533),\n",
       "  (2, 0.4010924121641571),\n",
       "  (3, 0.030607223698454877),\n",
       "  (4, 0.26783781123905126)],\n",
       " [(0, 0.07950666226186545),\n",
       "  (1, 0.15835260433872925),\n",
       "  (2, 0.7218368021143047),\n",
       "  (3, 0.02356568659839225),\n",
       "  (4, 0.016738244686708516)],\n",
       " [(0, 0.03938661603084182),\n",
       "  (1, 0.03135968697853732),\n",
       "  (2, 0.517808849761206),\n",
       "  (3, 0.11571436791530007),\n",
       "  (4, 0.2957304793141147)],\n",
       " [(0, 0.011036385583721331),\n",
       "  (1, 0.01776168304880152),\n",
       "  (2, 0.9239524055871702),\n",
       "  (3, 0.018279013623038454),\n",
       "  (4, 0.028970512157268494)],\n",
       " [(0, 0.05300898002948667),\n",
       "  (1, 0.09254791582897735),\n",
       "  (2, 0.7455434928293795),\n",
       "  (3, 0.0223830585712371),\n",
       "  (4, 0.08651655274091945)],\n",
       " [(0, 0.015524311657879321),\n",
       "  (1, 0.01166764303846905),\n",
       "  (2, 0.010788908416324936),\n",
       "  (3, 0.9489357547354033),\n",
       "  (4, 0.013083382151923452)],\n",
       " [(0, 0.021088938162108894),\n",
       "  (1, 0.6910569105691057),\n",
       "  (2, 0.01931510224193151),\n",
       "  (3, 0.0546440009854644),\n",
       "  (4, 0.2138950480413895)],\n",
       " [(0, 0.2423909057572424),\n",
       "  (1, 0.03822882288228823),\n",
       "  (2, 0.051063439677301065),\n",
       "  (3, 0.03777044371103777),\n",
       "  (4, 0.6305463879721306)],\n",
       " [(0, 0.06625793266645154),\n",
       "  (1, 0.12950414112079164),\n",
       "  (2, 0.22319027643325803),\n",
       "  (3, 0.051521996342906315),\n",
       "  (4, 0.5295256534365924)],\n",
       " [(0, 0.029540987397386986),\n",
       "  (1, 0.03561105330095966),\n",
       "  (2, 0.016244652560989715),\n",
       "  (3, 0.8970979303965777),\n",
       "  (4, 0.021505376344086027)],\n",
       " [(0, 0.12320294215981277),\n",
       "  (1, 0.040621865596790374),\n",
       "  (2, 0.6973698874400981),\n",
       "  (3, 0.12426167391062075),\n",
       "  (4, 0.014543630892678034)],\n",
       " [(0, 0.07971014492753624),\n",
       "  (1, 0.10854926072317378),\n",
       "  (2, 0.5838091055482361),\n",
       "  (3, 0.18986971160884208),\n",
       "  (4, 0.03806177719221198)],\n",
       " [(0, 0.8879509379509379),\n",
       "  (1, 0.01883116883116883),\n",
       "  (2, 0.016666666666666666),\n",
       "  (3, 0.02215007215007215),\n",
       "  (4, 0.0544011544011544)],\n",
       " [(0, 0.06391833529642717),\n",
       "  (1, 0.2907734589713388),\n",
       "  (2, 0.037220259128386335),\n",
       "  (3, 0.10467216332940714),\n",
       "  (4, 0.5034157832744405)],\n",
       " [(0, 0.5767195767195767),\n",
       "  (1, 0.12397232397232397),\n",
       "  (2, 0.028897028897028897),\n",
       "  (3, 0.1820919820919821),\n",
       "  (4, 0.08831908831908832)],\n",
       " [(0, 0.061549540390562124),\n",
       "  (1, 0.02317109331977803),\n",
       "  (2, 0.8144618121743551),\n",
       "  (3, 0.03376117253357055),\n",
       "  (4, 0.06705638158173423)],\n",
       " [(0, 0.12921880925293894),\n",
       "  (1, 0.021852483883200605),\n",
       "  (2, 0.03446150929086083),\n",
       "  (3, 0.01962457337883959),\n",
       "  (4, 0.7948426241941601)],\n",
       " [(0, 0.01659814222628293),\n",
       "  (1, 0.05248464544946957),\n",
       "  (2, 0.02233389168062535),\n",
       "  (3, 0.20242627277803157),\n",
       "  (4, 0.7061570478655905)],\n",
       " [(0, 0.027525160664484054),\n",
       "  (1, 0.022917424518006546),\n",
       "  (2, 0.17400266763671637),\n",
       "  (3, 0.021341093731053716),\n",
       "  (4, 0.7542136534497393)],\n",
       " [(0, 0.7926736586107311),\n",
       "  (1, 0.019091047271621826),\n",
       "  (2, 0.029274965800273597),\n",
       "  (3, 0.026265389876880985),\n",
       "  (4, 0.1326949384404925)],\n",
       " [(0, 0.06698731353687438),\n",
       "  (1, 0.6118778753659556),\n",
       "  (2, 0.022933221803987176),\n",
       "  (3, 0.07590966122961104),\n",
       "  (4, 0.22229192806357173)],\n",
       " [(0, 0.21030225627926777),\n",
       "  (1, 0.32098765432098764),\n",
       "  (2, 0.029516106144458636),\n",
       "  (3, 0.1884489853838513),\n",
       "  (4, 0.25074499787143467)],\n",
       " [(0, 0.15064613205877148),\n",
       "  (1, 0.03735174367144627),\n",
       "  (2, 0.0633150410102083),\n",
       "  (3, 0.07499852481265121),\n",
       "  (4, 0.6736885584469228)],\n",
       " [(0, 0.06431818181818182),\n",
       "  (1, 0.09353535353535354),\n",
       "  (2, 0.03166666666666667),\n",
       "  (3, 0.7279545454545454),\n",
       "  (4, 0.08252525252525253)],\n",
       " [(0, 0.05479542600304052),\n",
       "  (1, 0.014739903496595942),\n",
       "  (2, 0.8716372529578954),\n",
       "  (3, 0.021283627470421045),\n",
       "  (4, 0.03754379007204706)],\n",
       " [(0, 0.18390522875816995),\n",
       "  (1, 0.2099673202614379),\n",
       "  (2, 0.16315359477124183),\n",
       "  (3, 0.08243464052287582),\n",
       "  (4, 0.3605392156862745)],\n",
       " [(0, 0.015678524374176547),\n",
       "  (1, 0.7993412384716733),\n",
       "  (2, 0.00720245937637242),\n",
       "  (3, 0.1702239789196311),\n",
       "  (4, 0.007553798858146684)],\n",
       " [(0, 0.03553756185335133),\n",
       "  (1, 0.7540260908681962),\n",
       "  (2, 0.013945119208277103),\n",
       "  (3, 0.05519568151147099),\n",
       "  (4, 0.14129554655870447)],\n",
       " [(0, 0.06247297881539127),\n",
       "  (1, 0.014429312581063555),\n",
       "  (2, 0.7839926502377864),\n",
       "  (3, 0.05658236057068742),\n",
       "  (4, 0.08252269779507133)],\n",
       " [(0, 0.006029810298102982),\n",
       "  (1, 0.006537940379403795),\n",
       "  (2, 0.005758807588075882),\n",
       "  (3, 0.9734417344173443),\n",
       "  (4, 0.008231707317073173)],\n",
       " [(0, 0.10379707438530968),\n",
       "  (1, 0.011515717398070339),\n",
       "  (2, 0.051457620085071064),\n",
       "  (3, 0.014316837846249611),\n",
       "  (4, 0.8189127502852993)],\n",
       " [(0, 0.2068268408366379),\n",
       "  (1, 0.03436746753751652),\n",
       "  (2, 0.6476945805147345),\n",
       "  (3, 0.04906305886011974),\n",
       "  (4, 0.062048052250991366)],\n",
       " [(0, 0.10215505177721802),\n",
       "  (1, 0.06633081444164568),\n",
       "  (2, 0.5255387629443045),\n",
       "  (3, 0.10894206549118388),\n",
       "  (4, 0.19703330534564792)],\n",
       " [(0, 0.016228131888748182),\n",
       "  (1, 0.0072229257539515035),\n",
       "  (2, 0.007129121523380705),\n",
       "  (3, 0.957834998358426),\n",
       "  (4, 0.011584822475493644)],\n",
       " [(0, 0.6219099661722612),\n",
       "  (1, 0.0205048139474369),\n",
       "  (2, 0.013739266198282592),\n",
       "  (3, 0.3338017174082748),\n",
       "  (4, 0.010044236273744471)],\n",
       " [(0, 0.5510865486303005),\n",
       "  (1, 0.25422167638931525),\n",
       "  (2, 0.05362808310306006),\n",
       "  (3, 0.03247705796063179),\n",
       "  (4, 0.10858663391669222)],\n",
       " [(0, 0.020424256762085008),\n",
       "  (1, 0.011828720132481665),\n",
       "  (2, 0.2402807349578109),\n",
       "  (3, 0.014194464158977998),\n",
       "  (4, 0.7132718239886444)],\n",
       " [(0, 0.9203364153998852),\n",
       "  (1, 0.0262236848978739),\n",
       "  (2, 0.012798411952149612),\n",
       "  (3, 0.02658935381079246),\n",
       "  (4, 0.014052133939298963)],\n",
       " [(0, 0.03499609820517438),\n",
       "  (1, 0.008463893390959841),\n",
       "  (2, 0.13566240470616484),\n",
       "  (3, 0.11807431418452488),\n",
       "  (4, 0.702803289513176)],\n",
       " [(0, 0.01013531726777724),\n",
       "  (1, 0.00938055959890021),\n",
       "  (2, 0.00927273707477492),\n",
       "  (3, 0.9605369561701439),\n",
       "  (4, 0.010674429888403688)],\n",
       " [(0, 0.16121212121212122),\n",
       "  (1, 0.5657575757575758),\n",
       "  (2, 0.07676767676767676),\n",
       "  (3, 0.08904040404040404),\n",
       "  (4, 0.10722222222222222)],\n",
       " [(0, 0.06396534148827727),\n",
       "  (1, 0.046572375127421),\n",
       "  (2, 0.041539245667686034),\n",
       "  (3, 0.35977319062181445),\n",
       "  (4, 0.48814984709480125)],\n",
       " [(0, 0.012751992498827944),\n",
       "  (1, 0.01669010782934834),\n",
       "  (2, 0.015518049695264886),\n",
       "  (3, 0.945475855602438),\n",
       "  (4, 0.009563994374120957)],\n",
       " [(0, 0.015972463936264843),\n",
       "  (1, 0.8992535870363925),\n",
       "  (2, 0.008469660053370461),\n",
       "  (3, 0.06756390919286846),\n",
       "  (4, 0.008740379781103763)],\n",
       " [(0, 0.11145648312611012),\n",
       "  (1, 0.033353068877047566),\n",
       "  (2, 0.03192224195776593),\n",
       "  (3, 0.024422735346358793),\n",
       "  (4, 0.7988454706927176)],\n",
       " [(0, 0.020135052179251073),\n",
       "  (1, 0.029343155310006138),\n",
       "  (2, 0.9060159607120933),\n",
       "  (3, 0.014487415592387969),\n",
       "  (4, 0.03001841620626151)],\n",
       " [(0, 0.5197945437805883),\n",
       "  (1, 0.0641081552551243),\n",
       "  (2, 0.01003052769297863),\n",
       "  (3, 0.11372777050927944),\n",
       "  (4, 0.29233900276202934)],\n",
       " [(0, 0.0345855694692904),\n",
       "  (1, 0.020936858146160472),\n",
       "  (2, 0.827602199695223),\n",
       "  (3, 0.04704167494865169),\n",
       "  (4, 0.06983369774067448)],\n",
       " [(0, 0.13368747233289066),\n",
       "  (1, 0.6717257952317713),\n",
       "  (2, 0.011762473913868336),\n",
       "  (3, 0.081135774362866),\n",
       "  (4, 0.10168848415860368)],\n",
       " [(0, 0.038093529482119366),\n",
       "  (1, 0.9028525296017222),\n",
       "  (2, 0.010794163377586413),\n",
       "  (3, 0.0299007295778017),\n",
       "  (4, 0.018359047960770244)],\n",
       " [(0, 0.09385455185772143),\n",
       "  (1, 0.15055467511885895),\n",
       "  (2, 0.2095439337911604),\n",
       "  (3, 0.12026765275576685),\n",
       "  (4, 0.4257791864764923)],\n",
       " [(0, 0.014411379374883026),\n",
       "  (1, 0.022147357913781276),\n",
       "  (2, 0.011229646266142618),\n",
       "  (3, 0.9424168694241688),\n",
       "  (4, 0.009794747021024395)],\n",
       " [(0, 0.03604836869723933),\n",
       "  (1, 0.04502243516617233),\n",
       "  (2, 0.6799756635485589),\n",
       "  (3, 0.08685071108069055),\n",
       "  (4, 0.15210282150733895)],\n",
       " [(0, 0.3124546114742193),\n",
       "  (1, 0.04260469619946744),\n",
       "  (2, 0.0793996610990075),\n",
       "  (3, 0.32492132655531347),\n",
       "  (4, 0.24061970467199226)],\n",
       " [(0, 0.15132990132990135),\n",
       "  (1, 0.045795795795795805),\n",
       "  (2, 0.03045903045903046),\n",
       "  (3, 0.11851136851136852),\n",
       "  (4, 0.653903903903904)],\n",
       " [(0, 0.022087985844163768),\n",
       "  (1, 0.17450729147599),\n",
       "  (2, 0.017999877966929038),\n",
       "  (3, 0.08102995911892123),\n",
       "  (4, 0.704374885593996)],\n",
       " [(0, 0.2616310892172961),\n",
       "  (1, 0.05604816639299398),\n",
       "  (2, 0.03601532567049808),\n",
       "  (3, 0.09589490968801313),\n",
       "  (4, 0.5504105090311987)],\n",
       " [(0, 0.2478923936724448),\n",
       "  (1, 0.021218149095386947),\n",
       "  (2, 0.12885605127719366),\n",
       "  (3, 0.06295980550029996),\n",
       "  (4, 0.5390736004546747)],\n",
       " [(0, 0.05598331729226821),\n",
       "  (1, 0.7993797454817666),\n",
       "  (2, 0.027376751149609666),\n",
       "  (3, 0.02518447224895733),\n",
       "  (4, 0.09207571382739814)],\n",
       " [(0, 0.6795647483701515),\n",
       "  (1, 0.0816096805965949),\n",
       "  (2, 0.14004971624220253),\n",
       "  (3, 0.04685521317011397),\n",
       "  (4, 0.051920641620937104)],\n",
       " [(0, 0.8763278541876328),\n",
       "  (1, 0.012076484401207649),\n",
       "  (2, 0.02038838570203884),\n",
       "  (3, 0.07749077490774908),\n",
       "  (4, 0.01371650080137165)],\n",
       " [(0, 0.2808944989870656),\n",
       "  (1, 0.032024310425432446),\n",
       "  (2, 0.019791179678977717),\n",
       "  (3, 0.6366682250272713),\n",
       "  (4, 0.030621785881252923)],\n",
       " [(0, 0.26463894061659426),\n",
       "  (1, 0.04319263397475688),\n",
       "  (2, 0.35924891371818746),\n",
       "  (3, 0.05141733912683633),\n",
       "  (4, 0.2815021725636251)],\n",
       " [(0, 0.0872866597004528),\n",
       "  (1, 0.042702890978753044),\n",
       "  (2, 0.6796934865900383),\n",
       "  (3, 0.04019505398815743),\n",
       "  (4, 0.15012190874259837)],\n",
       " [(0, 0.051135005973715654),\n",
       "  (1, 0.078215850258861),\n",
       "  (2, 0.027638391079251295),\n",
       "  (3, 0.8208681800079649),\n",
       "  (4, 0.022142572680207088)],\n",
       " [(0, 0.08510152863335615),\n",
       "  (1, 0.045098486576926),\n",
       "  (2, 0.7837858392273177),\n",
       "  (3, 0.02357593733363754),\n",
       "  (4, 0.06243820822876264)],\n",
       " [(0, 0.08407391733671886),\n",
       "  (1, 0.1524897728875723),\n",
       "  (2, 0.10128367893920158),\n",
       "  (3, 0.15681572389147505),\n",
       "  (4, 0.5053369069450322)],\n",
       " [(0, 0.8526746114389294),\n",
       "  (1, 0.019090593543908366),\n",
       "  (2, 0.09406456091634849),\n",
       "  (3, 0.012958463496471133),\n",
       "  (4, 0.02121177060434263)],\n",
       " [(0, 0.034411276948590384),\n",
       "  (1, 0.024098258706467663),\n",
       "  (2, 0.020107794361525703),\n",
       "  (3, 0.8974398839137645),\n",
       "  (4, 0.02394278606965174)],\n",
       " [(0, 0.1685418045444518),\n",
       "  (1, 0.02544304728288845),\n",
       "  (2, 0.633428928597691),\n",
       "  (3, 0.10000735348187367),\n",
       "  (4, 0.07257886609309508)],\n",
       " [(0, 0.1906733285748275),\n",
       "  (1, 0.3447061622650488),\n",
       "  (2, 0.014132762312633832),\n",
       "  (3, 0.16078991196764217),\n",
       "  (4, 0.28969783487984774)],\n",
       " [(0, 0.13186364564196773),\n",
       "  (1, 0.2164387289923113),\n",
       "  (2, 0.22297067428726947),\n",
       "  (3, 0.08144519289650948),\n",
       "  (4, 0.34728175818194185)],\n",
       " [(0, 0.02475452196382429),\n",
       "  (1, 0.7599483204134367),\n",
       "  (2, 0.08361757105943153),\n",
       "  (3, 0.04950904392764858),\n",
       "  (4, 0.08217054263565891)],\n",
       " [(0, 0.027374184038745),\n",
       "  (1, 0.023934863480030884),\n",
       "  (2, 0.8816592966940409),\n",
       "  (3, 0.049694672562644766),\n",
       "  (4, 0.0173369832245385)],\n",
       " [(0, 0.08589287638754176),\n",
       "  (1, 0.039605560944067246),\n",
       "  (2, 0.010022631749110895),\n",
       "  (3, 0.741728634551137),\n",
       "  (4, 0.12275029636814312)],\n",
       " [(0, 0.025488400488400488),\n",
       "  (1, 0.22909035409035408),\n",
       "  (2, 0.2317867317867318),\n",
       "  (3, 0.018162393162393164),\n",
       "  (4, 0.49547212047212047)],\n",
       " [(0, 0.04209844559585492),\n",
       "  (1, 0.02748992515831894),\n",
       "  (2, 0.0768566493955095),\n",
       "  (3, 0.02259643062751871),\n",
       "  (4, 0.8309585492227979)],\n",
       " [(0, 0.14535588219798745),\n",
       "  (1, 0.03545650914071967),\n",
       "  (2, 0.26410621147463254),\n",
       "  (3, 0.07633949739212897),\n",
       "  (4, 0.47874189979453136)],\n",
       " [(0, 0.4029908186266425),\n",
       "  (1, 0.058441915443399856),\n",
       "  (2, 0.4061795590741657),\n",
       "  (3, 0.024685249326516025),\n",
       "  (4, 0.10770245752927593)],\n",
       " [(0, 0.06204752911551779),\n",
       "  (1, 0.21694995278564683),\n",
       "  (2, 0.6413676424299654),\n",
       "  (3, 0.02533836953100409),\n",
       "  (4, 0.054296506137865914)],\n",
       " [(0, 0.35288403157255616),\n",
       "  (1, 0.1902246508803886),\n",
       "  (2, 0.3536126290224651),\n",
       "  (3, 0.04068002428658166),\n",
       "  (4, 0.0625986642380085)],\n",
       " [(0, 0.13736239247671259),\n",
       "  (1, 0.14676650176048492),\n",
       "  (2, 0.5063511164594197),\n",
       "  (3, 0.137451530953336),\n",
       "  (4, 0.0720684583500468)],\n",
       " [(0, 0.15627822944896116),\n",
       "  (1, 0.1111111111111111),\n",
       "  (2, 0.34610917537746805),\n",
       "  (3, 0.04542521615692347),\n",
       "  (4, 0.3410762679055362)],\n",
       " [(0, 0.12329651140720495),\n",
       "  (1, 0.04585658429184224),\n",
       "  (2, 0.024484682837945564),\n",
       "  (3, 0.1518050335952625),\n",
       "  (4, 0.6545571878677447)],\n",
       " [(0, 0.17886178861788618),\n",
       "  (1, 0.0631123618928497),\n",
       "  (2, 0.6411819887429644),\n",
       "  (3, 0.06973108192620388),\n",
       "  (4, 0.047112778820095895)],\n",
       " [(0, 0.3749784890724488),\n",
       "  (1, 0.054379624849423505),\n",
       "  (2, 0.028910686628807435),\n",
       "  (3, 0.40778982389720647),\n",
       "  (4, 0.1339413755521138)],\n",
       " [(0, 0.037080028374282585),\n",
       "  (1, 0.8169213903398466),\n",
       "  (2, 0.013929193267556587),\n",
       "  (3, 0.07235442058425227),\n",
       "  (4, 0.05971496743406204)],\n",
       " [(0, 0.013315430233859237),\n",
       "  (1, 0.014881951437842676),\n",
       "  (2, 0.013539218977285442),\n",
       "  (3, 0.9417030323374734),\n",
       "  (4, 0.016560367013539218)],\n",
       " [(0, 0.12895853738165197),\n",
       "  (1, 0.17412123190771575),\n",
       "  (2, 0.04440091413646752),\n",
       "  (3, 0.6058330612689085),\n",
       "  (4, 0.04668625530525628)],\n",
       " [(0, 0.0590909090909091),\n",
       "  (1, 0.7463203463203464),\n",
       "  (2, 0.0336940836940837),\n",
       "  (3, 0.13080808080808085),\n",
       "  (4, 0.03008658008658009)],\n",
       " [(0, 0.14975438596491228),\n",
       "  (1, 0.1135906432748538),\n",
       "  (2, 0.01319298245614035),\n",
       "  (3, 0.056514619883040934),\n",
       "  (4, 0.6669473684210526)],\n",
       " [(0, 0.1253904506916555),\n",
       "  (1, 0.009705488621151271),\n",
       "  (2, 0.7906068719321732),\n",
       "  (3, 0.013833110218652387),\n",
       "  (4, 0.06046407853636769)],\n",
       " [(0, 0.025409414702210472),\n",
       "  (1, 0.7833590364984945),\n",
       "  (2, 0.10384078725123008),\n",
       "  (3, 0.06080634500991408),\n",
       "  (4, 0.02658441653815084)],\n",
       " [(0, 0.10982658959537572),\n",
       "  (1, 0.047877620131955394),\n",
       "  (2, 0.6054183453027384),\n",
       "  (3, 0.21217959946283646),\n",
       "  (4, 0.02469784550709406)],\n",
       " [(0, 0.053375249298583644),\n",
       "  (1, 0.26853260318426125),\n",
       "  (2, 0.014467768650914377),\n",
       "  (3, 0.10269411486326606),\n",
       "  (4, 0.5609302640029746)],\n",
       " [(0, 0.5063158741652587),\n",
       "  (1, 0.20749859200257462),\n",
       "  (2, 0.08810041033067825),\n",
       "  (3, 0.0712848982219004),\n",
       "  (4, 0.12680022527958806)],\n",
       " [(0, 0.021589283391858498),\n",
       "  (1, 0.011401569341483505),\n",
       "  (2, 0.886374474357307),\n",
       "  (3, 0.032557332986517536),\n",
       "  (4, 0.04807733992283349)],\n",
       " [(0, 0.16296641308285462),\n",
       "  (1, 0.543756145526057),\n",
       "  (2, 0.04248822646586969),\n",
       "  (3, 0.11866687367386017),\n",
       "  (4, 0.1321223412513585)],\n",
       " [(0, 0.43303227613246514),\n",
       "  (1, 0.3259819365679479),\n",
       "  (2, 0.01855352516978226),\n",
       "  (3, 0.1270741440873766),\n",
       "  (4, 0.09535811804242807)],\n",
       " [(0, 0.02075624625577052),\n",
       "  (1, 0.06998625647531452),\n",
       "  (2, 0.010325263417556471),\n",
       "  (3, 0.8437819360749903),\n",
       "  (4, 0.055150297776368186)],\n",
       " [(0, 0.22157213283493146),\n",
       "  (1, 0.6061000054174116),\n",
       "  (2, 0.034888130451270385),\n",
       "  (3, 0.11219459342326236),\n",
       "  (4, 0.02524513787312422)],\n",
       " [(0, 0.09807516040329974),\n",
       "  (1, 0.6856095325389552),\n",
       "  (2, 0.027294021794480094),\n",
       "  (3, 0.16366228740197578),\n",
       "  (4, 0.02535899786128934)],\n",
       " [(0, 0.18464510500405062),\n",
       "  (1, 0.12226584408300618),\n",
       "  (2, 0.5926341372219107),\n",
       "  (3, 0.06325169813672339),\n",
       "  (4, 0.03720321555430922)],\n",
       " [(0, 0.08693808312128923),\n",
       "  (1, 0.04664970313825276),\n",
       "  (2, 0.49943454905286966),\n",
       "  (3, 0.024125907077561022),\n",
       "  (4, 0.34285175761002734)],\n",
       " [(0, 0.05167636464394569),\n",
       "  (1, 0.2315045719035744),\n",
       "  (2, 0.01170684400110834),\n",
       "  (3, 0.6840537545026323),\n",
       "  (4, 0.021058464948739263)],\n",
       " [(0, 0.06864006864006864),\n",
       "  (1, 0.12714500214500216),\n",
       "  (2, 0.0834942084942085),\n",
       "  (3, 0.08102745602745602),\n",
       "  (4, 0.6396932646932647)],\n",
       " [(0, 0.15802873249681756),\n",
       "  (1, 0.012729587197672301),\n",
       "  (2, 0.7761411165666483),\n",
       "  (3, 0.025822876886706668),\n",
       "  (4, 0.02727768685215493)],\n",
       " [(0, 0.030715005035246726),\n",
       "  (1, 0.019385699899295065),\n",
       "  (2, 0.8487747566297416),\n",
       "  (3, 0.014853977844914401),\n",
       "  (4, 0.08627056059080228)],\n",
       " [(0, 0.06143445032333921),\n",
       "  (1, 0.02927689594356261),\n",
       "  (2, 0.8097589653145209),\n",
       "  (3, 0.03333333333333333),\n",
       "  (4, 0.06619635508524398)],\n",
       " [(0, 0.1123463652646532),\n",
       "  (1, 0.07955036748811067),\n",
       "  (2, 0.013464270273608796),\n",
       "  (3, 0.7718485578407758),\n",
       "  (4, 0.022790439132851583)],\n",
       " [(0, 0.07389660863546824),\n",
       "  (1, 0.032476175264867166),\n",
       "  (2, 0.020071341106319543),\n",
       "  (3, 0.8033328009370175),\n",
       "  (4, 0.07022307405632754)],\n",
       " [(0, 0.01930274836915838),\n",
       "  (1, 0.012191209496310555),\n",
       "  (2, 0.01507860121912095),\n",
       "  (3, 0.31328200192492783),\n",
       "  (4, 0.6401454389904823)],\n",
       " [(0, 0.15104166666666669),\n",
       "  (1, 0.0476388888888889),\n",
       "  (2, 0.622638888888889),\n",
       "  (3, 0.05159722222222223),\n",
       "  (4, 0.12708333333333335)],\n",
       " [(0, 0.6319411105776925),\n",
       "  (1, 0.1179922120872673),\n",
       "  (2, 0.022403584573531768),\n",
       "  (3, 0.13314130260841736),\n",
       "  (4, 0.09452179015309117)],\n",
       " [(0, 0.4043693874815708),\n",
       "  (1, 0.0406781932716794),\n",
       "  (2, 0.4727248358128937),\n",
       "  (3, 0.027141133896260553),\n",
       "  (4, 0.0550864495375955)],\n",
       " [(0, 0.0298390263054574),\n",
       "  (1, 0.022143698468786808),\n",
       "  (2, 0.03855516293678838),\n",
       "  (3, 0.02701217118178249),\n",
       "  (4, 0.8824499411071849)],\n",
       " [(0, 0.13306559571619814),\n",
       "  (1, 0.15093708165997322),\n",
       "  (2, 0.5483935742971887),\n",
       "  (3, 0.06546184738955824),\n",
       "  (4, 0.10214190093708166)],\n",
       " [(0, 0.0998263888888889),\n",
       "  (1, 0.03236607142857143),\n",
       "  (2, 0.012214781746031746),\n",
       "  (3, 0.8307911706349206),\n",
       "  (4, 0.0248015873015873)],\n",
       " [(0, 0.5129122681457625),\n",
       "  (1, 0.013061370549293254),\n",
       "  (2, 0.42112482853223593),\n",
       "  (3, 0.025884177252937318),\n",
       "  (4, 0.02701735551977098)],\n",
       " [(0, 0.09255156796140401),\n",
       "  (1, 0.7957957957957956),\n",
       "  (2, 0.01275045537340619),\n",
       "  (3, 0.08433023187121547),\n",
       "  (4, 0.014571948998178503)],\n",
       " [(0, 0.07774244030491191),\n",
       "  (1, 0.37366853450451815),\n",
       "  (2, 0.03160179716290575),\n",
       "  (3, 0.12888081175223384),\n",
       "  (4, 0.38810641627543035)],\n",
       " [(0, 0.4739728710641007),\n",
       "  (1, 0.016747465843984136),\n",
       "  (2, 0.12129670437294941),\n",
       "  (3, 0.030067087801772686),\n",
       "  (4, 0.35791587091719307)],\n",
       " [(0, 0.06432337434094904),\n",
       "  (1, 0.062097246631517285),\n",
       "  (2, 0.010857254442491701),\n",
       "  (3, 0.03366529974614333),\n",
       "  (4, 0.8290568248388986)],\n",
       " [(0, 0.2977718754611185),\n",
       "  (1, 0.039471742658993655),\n",
       "  (2, 0.07304116865869854),\n",
       "  (3, 0.4617087206728641),\n",
       "  (4, 0.12800649254832522)],\n",
       " [(0, 0.2611048194287152),\n",
       "  (1, 0.05077387693469234),\n",
       "  (2, 0.02730590159808733),\n",
       "  (3, 0.6075248521454637),\n",
       "  (4, 0.0532905498930414)],\n",
       " [(0, 0.034218820351193155),\n",
       "  (1, 0.2107659212566912),\n",
       "  (2, 0.014257841812997149),\n",
       "  (3, 0.041772975136324976),\n",
       "  (4, 0.6989844414427936)],\n",
       " [(0, 0.07771889982336613),\n",
       "  (1, 0.04188745899571032),\n",
       "  (2, 0.3366977878711414),\n",
       "  (3, 0.24905374716124148),\n",
       "  (4, 0.2946421061485407)],\n",
       " [(0, 0.022722400857449088),\n",
       "  (1, 0.02586638085030368),\n",
       "  (2, 0.8636655948553055),\n",
       "  (3, 0.04258663808503037),\n",
       "  (4, 0.045158985351911395)],\n",
       " [(0, 0.32080689767366194),\n",
       "  (1, 0.07523995444932488),\n",
       "  (2, 0.1997722466243696),\n",
       "  (3, 0.08312998210509191),\n",
       "  (4, 0.3210509191475516)],\n",
       " [(0, 0.044476100031655585),\n",
       "  (1, 0.03888361295768703),\n",
       "  (2, 0.3914740951777989),\n",
       "  (3, 0.08462593647778834),\n",
       "  (4, 0.4405402553550702)],\n",
       " [(0, 0.04038123902683722),\n",
       "  (1, 0.10067720090293454),\n",
       "  (2, 0.03491346877351392),\n",
       "  (3, 0.046300476548783547),\n",
       "  (4, 0.7777276147479307)],\n",
       " [(0, 0.12040777072513945),\n",
       "  (1, 0.43534012951208567),\n",
       "  (2, 0.037186638456113356),\n",
       "  (3, 0.34147592485734435),\n",
       "  (4, 0.06558953644931717)],\n",
       " [(0, 0.024121118809340522),\n",
       "  (1, 0.011483192199127535),\n",
       "  (2, 0.9220554272517322),\n",
       "  (3, 0.020207852193995384),\n",
       "  (4, 0.02213240954580447)],\n",
       " [(0, 0.21811300351446336),\n",
       "  (1, 0.048932143822654775),\n",
       "  (2, 0.011570694782373615),\n",
       "  (3, 0.029521492295214922),\n",
       "  (4, 0.6918626655852933)],\n",
       " [(0, 0.21083837305822364),\n",
       "  (1, 0.4816791177516898),\n",
       "  (2, 0.04950788568718131),\n",
       "  (3, 0.020218190442309974),\n",
       "  (4, 0.23775643306059527)],\n",
       " [(0, 0.02504425700301989),\n",
       "  (1, 0.8855565968968031),\n",
       "  (2, 0.036134541289180463),\n",
       "  (3, 0.022284702697073833),\n",
       "  (4, 0.030979902113922732)],\n",
       " [(0, 0.13679616512842527),\n",
       "  (1, 0.038634900193174504),\n",
       "  (2, 0.669242326679545),\n",
       "  (3, 0.12957000787007225),\n",
       "  (4, 0.025756600128783)],\n",
       " [(0, 0.10393389199255121),\n",
       "  (1, 0.016410614525139665),\n",
       "  (2, 0.008690254500310366),\n",
       "  (3, 0.8566495965238982),\n",
       "  (4, 0.01431564245810056)],\n",
       " [(0, 0.023670082967301123),\n",
       "  (1, 0.01378721327476818),\n",
       "  (2, 0.024646168862859932),\n",
       "  (3, 0.12554904831625183),\n",
       "  (4, 0.812347486578819)],\n",
       " [(0, 0.5135895593869731),\n",
       "  (1, 0.014487547892720307),\n",
       "  (2, 0.10973419540229885),\n",
       "  (3, 0.10985392720306514),\n",
       "  (4, 0.25233477011494254)],\n",
       " [(0, 0.014167533561202756),\n",
       "  (1, 0.016495764600980834),\n",
       "  (2, 0.2186060335859712),\n",
       "  (3, 0.7276465051765989),\n",
       "  (4, 0.02308416307524645)],\n",
       " [(0, 0.11970709964979306),\n",
       "  (1, 0.14845135762041206),\n",
       "  (2, 0.015736571610497112),\n",
       "  (3, 0.632373675353618),\n",
       "  (4, 0.08373129576567973)],\n",
       " [(0, 0.04580939755486817),\n",
       "  (1, 0.012225659154514656),\n",
       "  (2, 0.03657877939804586),\n",
       "  (3, 0.19055334609908184),\n",
       "  (4, 0.7148328177934895)],\n",
       " [(0, 0.06180555555555556),\n",
       "  (1, 0.7112962962962963),\n",
       "  (2, 0.06138888888888889),\n",
       "  (3, 0.017037037037037038),\n",
       "  (4, 0.14847222222222223)],\n",
       " [(0, 0.22573735376503543),\n",
       "  (1, 0.0544296149832482),\n",
       "  (2, 0.04553193826550228),\n",
       "  (3, 0.06459054209919261),\n",
       "  (4, 0.6097105508870215)],\n",
       " [(0, 0.678801437190699),\n",
       "  (1, 0.13392312385600977),\n",
       "  (2, 0.026540573520439294),\n",
       "  (3, 0.13246559555284387),\n",
       "  (4, 0.028269269880008135)],\n",
       " [(0, 0.06785907859078591),\n",
       "  (1, 0.669810298102981),\n",
       "  (2, 0.023197831978319785),\n",
       "  (3, 0.1319241192411924),\n",
       "  (4, 0.10720867208672087)],\n",
       " [(0, 0.6315172817809023),\n",
       "  (1, 0.019332161687170477),\n",
       "  (2, 0.01803033261732735),\n",
       "  (3, 0.3114626049599688),\n",
       "  (4, 0.01965761895463126)],\n",
       " [(0, 0.08749689286602039),\n",
       "  (1, 0.2662192393736018),\n",
       "  (2, 0.5582898334576187),\n",
       "  (3, 0.03047477007208551),\n",
       "  (4, 0.05751926423067363)],\n",
       " [(0, 0.5702542107849708),\n",
       "  (1, 0.018764240718402363),\n",
       "  (2, 0.00696957512397802),\n",
       "  (3, 0.38998346959746244),\n",
       "  (4, 0.014028503775186527)],\n",
       " [(0, 0.06532861078315624),\n",
       "  (1, 0.01547947002492457),\n",
       "  (2, 0.8304473304473304),\n",
       "  (3, 0.03469762560671651),\n",
       "  (4, 0.05404696313787223)],\n",
       " [(0, 0.420337003670337),\n",
       "  (1, 0.09943276609943276),\n",
       "  (2, 0.035618952285618954),\n",
       "  (3, 0.10076743410076744),\n",
       "  (4, 0.34384384384384387)],\n",
       " [(0, 0.08061374756543632),\n",
       "  (1, 0.016246259085079092),\n",
       "  (2, 0.021899197187782053),\n",
       "  (3, 0.8521685430620873),\n",
       "  (4, 0.02907225309961522)],\n",
       " [(0, 0.03429355281207133),\n",
       "  (1, 0.7736625514403292),\n",
       "  (2, 0.02801519468186135),\n",
       "  (3, 0.14303049488234673),\n",
       "  (4, 0.02099820618339137)],\n",
       " [(0, 0.15112166798627608),\n",
       "  (1, 0.21393507521773555),\n",
       "  (2, 0.5271047769860121),\n",
       "  (3, 0.042174716283979945),\n",
       "  (4, 0.0656637635259963)],\n",
       " [(0, 0.18512366186784796),\n",
       "  (1, 0.04138981173864895),\n",
       "  (2, 0.7237449243263198),\n",
       "  (3, 0.025609080841638984),\n",
       "  (4, 0.024132521225544486)],\n",
       " [(0, 0.3402223586385649),\n",
       "  (1, 0.01882545528954369),\n",
       "  (2, 0.013164177068412795),\n",
       "  (3, 0.6075983902871565),\n",
       "  (4, 0.020189618716322216)],\n",
       " [(0, 0.037897954880877084),\n",
       "  (1, 0.01728863588446131),\n",
       "  (2, 0.9010120177103099),\n",
       "  (3, 0.020345772717689226),\n",
       "  (4, 0.02345561880666245)],\n",
       " [(0, 0.020638733208789253),\n",
       "  (1, 0.7141268570411885),\n",
       "  (2, 0.06636420247308958),\n",
       "  (3, 0.023574415087625655),\n",
       "  (4, 0.175295792189307)],\n",
       " [(0, 0.03315053604070946),\n",
       "  (1, 0.09401709401709403),\n",
       "  (2, 0.04574872782965269),\n",
       "  (3, 0.8048021342819032),\n",
       "  (4, 0.02228150783064078)],\n",
       " [(0, 0.011115975870791984),\n",
       "  (1, 0.009048453006421485),\n",
       "  (2, 0.004426931309593307),\n",
       "  (3, 0.9712249464876436),\n",
       "  (4, 0.004183693325549719)],\n",
       " [(0, 0.08619941748776105),\n",
       "  (1, 0.03718163227365682),\n",
       "  (2, 0.10144388671996035),\n",
       "  (3, 0.7453677883125737),\n",
       "  (4, 0.029807275206048214)],\n",
       " [(0, 0.2573611719472429),\n",
       "  (1, 0.16211612400837105),\n",
       "  (2, 0.03679369250985545),\n",
       "  (3, 0.15880663843870152),\n",
       "  (4, 0.38492237309582905)],\n",
       " [(0, 0.04650183480719485),\n",
       "  (1, 0.024303675630150994),\n",
       "  (2, 0.02033327317572039),\n",
       "  (3, 0.3972207182818986),\n",
       "  (4, 0.5116404981050352)],\n",
       " [(0, 0.014694597912829958),\n",
       "  (1, 0.012124002455494168),\n",
       "  (2, 0.03426181706568447),\n",
       "  (3, 0.049340085942295886),\n",
       "  (4, 0.8895794966236955)],\n",
       " [(0, 0.8336242001163467),\n",
       "  (1, 0.06551774287376382),\n",
       "  (2, 0.01199825479930192),\n",
       "  (3, 0.07278941244909831),\n",
       "  (4, 0.016070389761489237)],\n",
       " [(0, 0.23594891757153588),\n",
       "  (1, 0.09192105442873728),\n",
       "  (2, 0.33258212115003755),\n",
       "  (3, 0.06433107969678345),\n",
       "  (4, 0.27521682715290585)],\n",
       " [(0, 0.7789813210090506),\n",
       "  (1, 0.021615636433660698),\n",
       "  (2, 0.033169651453880226),\n",
       "  (3, 0.14278836895821298),\n",
       "  (4, 0.023445022145195456)],\n",
       " [(0, 0.02570672132547613),\n",
       "  (1, 0.02092407549748057),\n",
       "  (2, 0.046032966094457256),\n",
       "  (3, 0.036553078828251775),\n",
       "  (4, 0.8707831582543343)],\n",
       " [(0, 0.08275617119516959),\n",
       "  (1, 0.0504942875747351),\n",
       "  (2, 0.6175930858935654),\n",
       "  (3, 0.02065944474042503),\n",
       "  (4, 0.2284970105961049)],\n",
       " [(0, 0.55323469683783),\n",
       "  (1, 0.0587467362924282),\n",
       "  (2, 0.014541630403249203),\n",
       "  (3, 0.3494342906875544),\n",
       "  (4, 0.024042645778938207)],\n",
       " [(0, 0.023887079261672096),\n",
       "  (1, 0.7249366630474122),\n",
       "  (2, 0.029768367716250452),\n",
       "  (3, 0.1320123054650742),\n",
       "  (4, 0.08939558450959102)],\n",
       " [(0, 0.12925739810161924),\n",
       "  (1, 0.024846454494695705),\n",
       "  (2, 0.016192071468453383),\n",
       "  (3, 0.8033686953284944),\n",
       "  (4, 0.026335380606737395)],\n",
       " [(0, 0.11893124796350603),\n",
       "  (1, 0.01775822743564679),\n",
       "  (2, 0.016291951775822745),\n",
       "  (3, 0.8339035516454871),\n",
       "  (4, 0.013115021179537308)],\n",
       " [(0, 0.4706482899438489),\n",
       "  (1, 0.264624808575804),\n",
       "  (2, 0.04199421473540922),\n",
       "  (3, 0.20210992002722478),\n",
       "  (4, 0.02062276671771312)],\n",
       " [(0, 0.09489452346595204),\n",
       "  (1, 0.041778327492613204),\n",
       "  (2, 0.05868205868205868),\n",
       "  (3, 0.025974025974025976),\n",
       "  (4, 0.77867106438535)],\n",
       " [(0, 0.2069799222117103),\n",
       "  (1, 0.1483233469988437),\n",
       "  (2, 0.35908756438557765),\n",
       "  (3, 0.15746872700515085),\n",
       "  (4, 0.12814043939871755)],\n",
       " [(0, 0.07734473496683661),\n",
       "  (1, 0.3225346708326482),\n",
       "  (2, 0.5356575124705367),\n",
       "  (3, 0.028668530395220087),\n",
       "  (4, 0.03579455133475854)],\n",
       " [(0, 0.04439834024896266),\n",
       "  (1, 0.026924850161364686),\n",
       "  (2, 0.19552789303826648),\n",
       "  (3, 0.01516828031350853),\n",
       "  (4, 0.7179806362378977)],\n",
       " [(0, 0.3960222464030952),\n",
       "  (1, 0.08656752508765568),\n",
       "  (2, 0.17126103252327413),\n",
       "  (3, 0.02224640309515174),\n",
       "  (4, 0.3239027928908234)],\n",
       " [(0, 0.15119418483904468),\n",
       "  (1, 0.16365524402907583),\n",
       "  (2, 0.4112668743509866),\n",
       "  (3, 0.24293873312564904),\n",
       "  (4, 0.03094496365524403)],\n",
       " [(0, 0.4486735870818915),\n",
       "  (1, 0.0754105563794145),\n",
       "  (2, 0.035316087219201396),\n",
       "  (3, 0.12621519195913658),\n",
       "  (4, 0.31438457736035585)],\n",
       " [(0, 0.08811981380287391),\n",
       "  (1, 0.015705322809147947),\n",
       "  (2, 0.1600080955272212),\n",
       "  (3, 0.14563853470957297),\n",
       "  (4, 0.590528233151184)],\n",
       " [(0, 0.026727312107429065),\n",
       "  (1, 0.8571366688325752),\n",
       "  (2, 0.017240632445310808),\n",
       "  (3, 0.08161143599740091),\n",
       "  (4, 0.01728395061728395)],\n",
       " [(0, 0.1963690681374239),\n",
       "  (1, 0.6381324447508523),\n",
       "  (2, 0.047186793825883803),\n",
       "  (3, 0.07334635566279828),\n",
       "  (4, 0.04496533762304187)],\n",
       " [(0, 0.19231403595579746),\n",
       "  (1, 0.04178349551926989),\n",
       "  (2, 0.4184946945956347),\n",
       "  (3, 0.14701193028753642),\n",
       "  (4, 0.2003958436417615)],\n",
       " [(0, 0.033805835647133325),\n",
       "  (1, 0.6230698036923377),\n",
       "  (2, 0.019143650445710945),\n",
       "  (3, 0.14472210044327535),\n",
       "  (4, 0.1792586097715427)],\n",
       " [(0, 0.03441109661205355),\n",
       "  (1, 0.9020347010777633),\n",
       "  (2, 0.008747764728625972),\n",
       "  (3, 0.04151563481707023),\n",
       "  (4, 0.013290802764486975)],\n",
       " [(0, 0.021205586362702814),\n",
       "  (1, 0.008471965495995071),\n",
       "  (2, 0.01884370507291025),\n",
       "  (3, 0.019613883754364347),\n",
       "  (4, 0.9318648593140275)],\n",
       " [(0, 0.7949754319226502),\n",
       "  (1, 0.020129973054366777),\n",
       "  (2, 0.03399904897765098),\n",
       "  (3, 0.07100966872721509),\n",
       "  (4, 0.07988587731811697)],\n",
       " [(0, 0.5915532879818595),\n",
       "  (1, 0.01951733074182054),\n",
       "  (2, 0.31081956592160676),\n",
       "  (3, 0.01032555879494655),\n",
       "  (4, 0.06778425655976676)],\n",
       " [(0, 0.2502410800385728),\n",
       "  (1, 0.5654666238079932),\n",
       "  (2, 0.06739526411657559),\n",
       "  (3, 0.04805528768884603),\n",
       "  (4, 0.06884174434801243)],\n",
       " [(0, 0.11514919663351186),\n",
       "  (1, 0.019850378304854203),\n",
       "  (2, 0.21503868060868825),\n",
       "  (3, 0.01942531667091728),\n",
       "  (4, 0.6305364277820283)],\n",
       " [(0, 0.6963036412714152),\n",
       "  (1, 0.021043353715639288),\n",
       "  (2, 0.009970803723902385),\n",
       "  (3, 0.22470115132484988),\n",
       "  (4, 0.047981049964193244)],\n",
       " [(0, 0.0694499146645661),\n",
       "  (1, 0.6322261607807098),\n",
       "  (2, 0.027044768281475645),\n",
       "  (3, 0.03763511443700494),\n",
       "  (4, 0.2336440418362435)],\n",
       " [(0, 0.22655739651998338),\n",
       "  (1, 0.06484945661856405),\n",
       "  (2, 0.1714472355840608),\n",
       "  (3, 0.17655442722251916),\n",
       "  (4, 0.3605914840548726)],\n",
       " [(0, 0.10244238046095631),\n",
       "  (1, 0.6101135190918473),\n",
       "  (2, 0.22180942552459582),\n",
       "  (3, 0.025937392500859992),\n",
       "  (4, 0.03969728242174063)],\n",
       " [(0, 0.020861678004535148),\n",
       "  (1, 0.0854119425547997),\n",
       "  (2, 0.017535903250188965),\n",
       "  (3, 0.05356512975560595),\n",
       "  (4, 0.8226253464348703)],\n",
       " [(0, 0.011614997010846357),\n",
       "  (1, 0.008711247758134769),\n",
       "  (2, 0.010846357502775643),\n",
       "  (3, 0.9559313348706123),\n",
       "  (4, 0.012896062857630883)],\n",
       " [(0, 0.04546957671957672),\n",
       "  (1, 0.08928571428571429),\n",
       "  (2, 0.14260912698412698),\n",
       "  (3, 0.05133928571428571),\n",
       "  (4, 0.6712962962962963)],\n",
       " [(0, 0.03405730600591414),\n",
       "  (1, 0.01067264878828048),\n",
       "  (2, 0.22191631827606131),\n",
       "  (3, 0.7212195370653615),\n",
       "  (4, 0.012134189864382585)],\n",
       " [(0, 0.054781420765027324),\n",
       "  (1, 0.03524590163934426),\n",
       "  (2, 0.01999089253187614),\n",
       "  (3, 0.03278688524590164),\n",
       "  (4, 0.8571948998178507)],\n",
       " [(0, 0.049878003003003005),\n",
       "  (1, 0.11477102102102102),\n",
       "  (2, 0.6901745495495496),\n",
       "  (3, 0.015765765765765764),\n",
       "  (4, 0.12941066066066065)],\n",
       " [(0, 0.7651597087028625),\n",
       "  (1, 0.046686855577186534),\n",
       "  (2, 0.05025596654409114),\n",
       "  (3, 0.03987309827673228),\n",
       "  (4, 0.09802437089912755)],\n",
       " [(0, 0.015928972450711582),\n",
       "  (1, 0.018213866039952998),\n",
       "  (2, 0.012664838751795273),\n",
       "  (3, 0.9422901162031597),\n",
       "  (4, 0.010902206554380468)],\n",
       " [(0, 0.05681282413955682),\n",
       "  (1, 0.015165802294515168),\n",
       "  (2, 0.85989313217036),\n",
       "  (3, 0.02577400597202578),\n",
       "  (4, 0.04235423542354236)],\n",
       " [(0, 0.4974703810438681),\n",
       "  (1, 0.10765289785462696),\n",
       "  (2, 0.26692283061159144),\n",
       "  (3, 0.07031700288184438),\n",
       "  (4, 0.05763688760806916)],\n",
       " [(0, 0.3125508607198748),\n",
       "  (1, 0.025289514866979657),\n",
       "  (2, 0.35805946791862286),\n",
       "  (3, 0.26579029733959314),\n",
       "  (4, 0.03830985915492958)],\n",
       " [(0, 0.0642369258042651),\n",
       "  (1, 0.5172323895927838),\n",
       "  (2, 0.0952604411074613),\n",
       "  (3, 0.15855884039835239),\n",
       "  (4, 0.16471140309713753)],\n",
       " [(0, 0.2751918158567775),\n",
       "  (1, 0.030122193805058255),\n",
       "  (2, 0.08951406649616368),\n",
       "  (3, 0.08974140380789997),\n",
       "  (4, 0.5154305200341006)],\n",
       " [(0, 0.06488282616299407),\n",
       "  (1, 0.7208814270724031),\n",
       "  (2, 0.10510668065757259),\n",
       "  (3, 0.07001282499708524),\n",
       "  (4, 0.03911624110994521)],\n",
       " [(0, 0.8006877283473028),\n",
       "  (1, 0.018439716312056736),\n",
       "  (2, 0.011648398882441435),\n",
       "  (3, 0.1531485063399957),\n",
       "  (4, 0.01607565011820331)],\n",
       " [(0, 0.10267140963939457),\n",
       "  (1, 0.08683825068005858),\n",
       "  (2, 0.13440747715700635),\n",
       "  (3, 0.06947060054404687),\n",
       "  (4, 0.6066122619794936)],\n",
       " [(0, 0.0560188402439966),\n",
       "  (1, 0.19666435024322446),\n",
       "  (2, 0.10431626901397575),\n",
       "  (3, 0.14238282758088178),\n",
       "  (4, 0.5006177129179213)],\n",
       " [(0, 0.04621780506024879),\n",
       "  (1, 0.08473805579914905),\n",
       "  (2, 0.024261911721718796),\n",
       "  (3, 0.825619539445906),\n",
       "  (4, 0.019162687972977363)],\n",
       " [(0, 0.025986795898300323),\n",
       "  (1, 0.08442196937772159),\n",
       "  (2, 0.012361286697569883),\n",
       "  (3, 0.8526478437982863),\n",
       "  (4, 0.024582104228121928)],\n",
       " [(0, 0.6968933669185559),\n",
       "  (1, 0.04869857262804366),\n",
       "  (2, 0.039672544080604534),\n",
       "  (3, 0.041561712846347604),\n",
       "  (4, 0.17317380352644837)],\n",
       " [(0, 0.12620329580682005),\n",
       "  (1, 0.2753711861641377),\n",
       "  (2, 0.4262114537444934),\n",
       "  (3, 0.0810491107847936),\n",
       "  (4, 0.09116495349975526)],\n",
       " [(0, 0.30484037075180226),\n",
       "  (1, 0.29929053667467675),\n",
       "  (2, 0.141091658084449),\n",
       "  (3, 0.15390776976770798),\n",
       "  (4, 0.100869664721364)],\n",
       " [(0, 0.06819378620326487),\n",
       "  (1, 0.025451992276636828),\n",
       "  (2, 0.8294716517465333),\n",
       "  (3, 0.019396173424609444),\n",
       "  (4, 0.05748639634895559)],\n",
       " [(0, 0.032711218141681715),\n",
       "  (1, 0.30583985550872966),\n",
       "  (2, 0.3770151849622048),\n",
       "  (3, 0.2687136263295204),\n",
       "  (4, 0.0157201150578634)],\n",
       " [(0, 0.28414465546096196),\n",
       "  (1, 0.01862766499308739),\n",
       "  (2, 0.0366732154551408),\n",
       "  (3, 0.19173397365931746),\n",
       "  (4, 0.4688204904314924)],\n",
       " [(0, 0.15101289134438306),\n",
       "  (1, 0.03860582497783235),\n",
       "  (2, 0.057499488438714955),\n",
       "  (3, 0.04222085805879544),\n",
       "  (4, 0.7106609371802742)],\n",
       " [(0, 0.04585060993930653),\n",
       "  (1, 0.662400096148068),\n",
       "  (2, 0.009674899344991286),\n",
       "  (3, 0.25905895078420765),\n",
       "  (4, 0.023015443783426476)],\n",
       " [(0, 0.03768714506969541),\n",
       "  (1, 0.013652268685825733),\n",
       "  (2, 0.011415132220501348),\n",
       "  (3, 0.13181896403372914),\n",
       "  (4, 0.8054264899902484)],\n",
       " [(0, 0.0315234328179312),\n",
       "  (1, 0.019896919573294976),\n",
       "  (2, 0.01881817092173079),\n",
       "  (3, 0.8602421191417955),\n",
       "  (4, 0.06951935754524752)],\n",
       " [(0, 0.052463522088840385),\n",
       "  (1, 0.8830281718604746),\n",
       "  (2, 0.009013378602320035),\n",
       "  (3, 0.038155288791883915),\n",
       "  (4, 0.017339638656481145)],\n",
       " [(0, 0.19504778453518679),\n",
       "  (1, 0.015879911188338643),\n",
       "  (2, 0.008398494063133507),\n",
       "  (3, 0.7678830002896032),\n",
       "  (4, 0.012790809923737812)],\n",
       " [(0, 0.6084291187739465),\n",
       "  (1, 0.0050659855257556416),\n",
       "  (2, 0.020945083014048536),\n",
       "  (3, 0.35687526607066844),\n",
       "  (4, 0.0086845466155811)],\n",
       " [(0, 0.03914083575100524),\n",
       "  (1, 0.012775487351758538),\n",
       "  (2, 0.8620654552857943),\n",
       "  (3, 0.021071919377004124),\n",
       "  (4, 0.06494630223443783)],\n",
       " [(0, 0.08532198409079395),\n",
       "  (1, 0.014034554390231544),\n",
       "  (2, 0.013375892992856056),\n",
       "  (3, 0.8657850737194103),\n",
       "  (4, 0.021482494806708212)],\n",
       " [(0, 0.008975398756420654),\n",
       "  (1, 0.009029467423628007),\n",
       "  (2, 0.009407948094079481),\n",
       "  (3, 0.9615571776155718),\n",
       "  (4, 0.01103000811030008)],\n",
       " [(0, 0.0887028581092508),\n",
       "  (1, 0.18662269575511584),\n",
       "  (2, 0.3180280737358363),\n",
       "  (3, 0.025621511922881784),\n",
       "  (4, 0.38102486047691525)],\n",
       " [(0, 0.024077046548956663),\n",
       "  (1, 0.8924201890494026),\n",
       "  (2, 0.014588906723738184),\n",
       "  (3, 0.028749777064383805),\n",
       "  (4, 0.04016408061351882)],\n",
       " [(0, 0.1703740870825668),\n",
       "  (1, 0.09304788983665051),\n",
       "  (2, 0.6636003791046441),\n",
       "  (3, 0.05814796231253833),\n",
       "  (4, 0.01482968166360038)],\n",
       " [(0, 0.21273527625986643),\n",
       "  (1, 0.03498785670916819),\n",
       "  (2, 0.1145264116575592),\n",
       "  (3, 0.588949605343048),\n",
       "  (4, 0.04880085003035823)],\n",
       " [(0, 0.539971529182588),\n",
       "  (1, 0.07866936390200047),\n",
       "  (2, 0.26208136659923587),\n",
       "  (3, 0.07934367273544618),\n",
       "  (4, 0.03993406758072976)],\n",
       " [(0, 0.3827984913126816),\n",
       "  (1, 0.04470413652383602),\n",
       "  (2, 0.016508996475607492),\n",
       "  (3, 0.3168861683052),\n",
       "  (4, 0.23910220738267482)],\n",
       " [(0, 0.0303359004953538),\n",
       "  (1, 0.820117872509672),\n",
       "  (2, 0.08721119427269769),\n",
       "  (3, 0.01713851827747044),\n",
       "  (4, 0.04519651444480602)],\n",
       " [(0, 0.0771797293043752),\n",
       "  (1, 0.1225055083412024),\n",
       "  (2, 0.6356940509915014),\n",
       "  (3, 0.04375196726471514),\n",
       "  (4, 0.12086874409820586)],\n",
       " [(0, 0.024230517354289455),\n",
       "  (1, 0.14172669722767955),\n",
       "  (2, 0.7108709888670596),\n",
       "  (3, 0.045241213708797204),\n",
       "  (4, 0.0779305828421742)],\n",
       " [(0, 0.23376777956317324),\n",
       "  (1, 0.07040409450693329),\n",
       "  (2, 0.13205975123489852),\n",
       "  (3, 0.4311134916383979),\n",
       "  (4, 0.13265488305659703)],\n",
       " [(0, 0.746728527242446),\n",
       "  (1, 0.012015227218653345),\n",
       "  (2, 0.009398049012610042),\n",
       "  (3, 0.04948846062336427),\n",
       "  (4, 0.18236973590292652)],\n",
       " [(0, 0.036217633562766305),\n",
       "  (1, 0.0429367420517863),\n",
       "  (2, 0.8139954113405441),\n",
       "  (3, 0.024745984922976073),\n",
       "  (4, 0.08210422812192723)],\n",
       " [(0, 0.35691849012404836),\n",
       "  (1, 0.07091518926689028),\n",
       "  (2, 0.3211414577011127),\n",
       "  (3, 0.20140552627375818),\n",
       "  (4, 0.04961933663419049)],\n",
       " [(0, 0.17928104575163398),\n",
       "  (1, 0.032745098039215687),\n",
       "  (2, 0.7298692810457517),\n",
       "  (3, 0.020784313725490194),\n",
       "  (4, 0.037320261437908495)],\n",
       " [(0, 0.09536541889483066),\n",
       "  (1, 0.028104575163398694),\n",
       "  (2, 0.017052881758764113),\n",
       "  (3, 0.8034462269756387),\n",
       "  (4, 0.05603089720736779)],\n",
       " [(0, 0.010157545605306802),\n",
       "  (1, 0.011608623548922057),\n",
       "  (2, 0.006892620232172472),\n",
       "  (3, 0.9636712271973467),\n",
       "  (4, 0.007669983416252074)],\n",
       " [(0, 0.041028750442120156),\n",
       "  (1, 0.03228740336516599),\n",
       "  (2, 0.026678793390935273),\n",
       "  (3, 0.10161184376736901),\n",
       "  (4, 0.7983932090344096)],\n",
       " [(0, 0.012457912457912458),\n",
       "  (1, 0.5231060606060606),\n",
       "  (2, 0.006818181818181818),\n",
       "  (3, 0.45046296296296295),\n",
       "  (4, 0.007154882154882155)],\n",
       " [(0, 0.6643960036330608),\n",
       "  (1, 0.06789282470481381),\n",
       "  (2, 0.016500151377535573),\n",
       "  (3, 0.23599757795943083),\n",
       "  (4, 0.015213442325158947)],\n",
       " [(0, 0.8832402593234483),\n",
       "  (1, 0.015469542332627257),\n",
       "  (2, 0.03697284806470249),\n",
       "  (3, 0.03479042300532769),\n",
       "  (4, 0.02952692727389435)],\n",
       " [(0, 0.4648685624938923),\n",
       "  (1, 0.06249389230919574),\n",
       "  (2, 0.261946643213134),\n",
       "  (3, 0.04612528095377699),\n",
       "  (4, 0.16456562103000097)],\n",
       " [(0, 0.011359126984126984),\n",
       "  (1, 0.883234126984127),\n",
       "  (2, 0.01597222222222222),\n",
       "  (3, 0.05223214285714286),\n",
       "  (4, 0.03720238095238095)],\n",
       " [(0, 0.17537561494482118),\n",
       "  (1, 0.6558968222310863),\n",
       "  (2, 0.08314497185657936),\n",
       "  (3, 0.048220538049018304),\n",
       "  (4, 0.037362052918494884)],\n",
       " [(0, 0.01851851851851852),\n",
       "  (1, 0.7256162915326904),\n",
       "  (2, 0.011670834821960225),\n",
       "  (3, 0.052221031320709786),\n",
       "  (4, 0.19197332380612125)],\n",
       " [(0, 0.7328567173071194),\n",
       "  (1, 0.015370866845397676),\n",
       "  (2, 0.020673220137027108),\n",
       "  (3, 0.11105153410783437),\n",
       "  (4, 0.12004766160262138)],\n",
       " [(0, 0.014229774141668415),\n",
       "  (1, 0.9052863436123348),\n",
       "  (2, 0.0062932662051604785),\n",
       "  (3, 0.012271869100062933),\n",
       "  (4, 0.06191874694077337)],\n",
       " [(0, 0.08625548625548625),\n",
       "  (1, 0.4985446985446986),\n",
       "  (2, 0.3041349041349041),\n",
       "  (3, 0.021621621621621623),\n",
       "  (4, 0.08944328944328944)],\n",
       " [(0, 0.033245149911816575),\n",
       "  (1, 0.02054673721340388),\n",
       "  (2, 0.18421516754850087),\n",
       "  (3, 0.07954144620811288),\n",
       "  (4, 0.6824514991181658)],\n",
       " [(0, 0.31555389779932863),\n",
       "  (1, 0.22346553939243236),\n",
       "  (2, 0.0432674375233122),\n",
       "  (3, 0.17655103816983714),\n",
       "  (4, 0.2411620871150897)],\n",
       " [(0, 0.01675014811224215),\n",
       "  (1, 0.04804222545376205),\n",
       "  (2, 0.6487854796143696),\n",
       "  (3, 0.05218936823396348),\n",
       "  (4, 0.23423277858566274)],\n",
       " [(0, 0.015693659761456372),\n",
       "  (1, 0.9397934143696856),\n",
       "  (2, 0.01027221366204417),\n",
       "  (3, 0.02128630942190264),\n",
       "  (4, 0.01295440278491126)],\n",
       " [(0, 0.020494512759486976),\n",
       "  (1, 0.023050817576799332),\n",
       "  (2, 0.011194852130988584),\n",
       "  (3, 0.1570364493807572),\n",
       "  (4, 0.7882233681519679)],\n",
       " [(0, 0.06490709979082072),\n",
       "  (1, 0.8093392395717978),\n",
       "  (2, 0.039559493047865144),\n",
       "  (3, 0.05364833271810016),\n",
       "  (4, 0.03254583487141627)],\n",
       " [(0, 0.07402777777777778),\n",
       "  (1, 0.03361111111111111),\n",
       "  (2, 0.7891666666666667),\n",
       "  (3, 0.04138888888888889),\n",
       "  (4, 0.06180555555555556)],\n",
       " [(0, 0.19931696402284638),\n",
       "  (1, 0.026026026026026026),\n",
       "  (2, 0.6569510687157746),\n",
       "  (3, 0.07130660071836542),\n",
       "  (4, 0.046399340516987575)],\n",
       " [(0, 0.14135358379962695),\n",
       "  (1, 0.03765876187938538),\n",
       "  (2, 0.7204458655297984),\n",
       "  (3, 0.08739674926725287),\n",
       "  (4, 0.013145039523936407)],\n",
       " [(0, 0.8358957031030017),\n",
       "  (1, 0.08323478064988456),\n",
       "  (2, 0.014980007884214675),\n",
       "  (3, 0.020330010700005632),\n",
       "  (4, 0.04555949766289351)],\n",
       " [(0, 0.6552028218694885),\n",
       "  (1, 0.03380364491475603),\n",
       "  (2, 0.1970899470899471),\n",
       "  (3, 0.05526161081716637),\n",
       "  (4, 0.05864197530864197)],\n",
       " [(0, 0.044649139640504064),\n",
       "  (1, 0.010298727051749505),\n",
       "  (2, 0.019829847118275443),\n",
       "  (3, 0.912748672679588),\n",
       "  (4, 0.01247361350988294)],\n",
       " [(0, 0.025984689318747135),\n",
       "  (1, 0.812794320075346),\n",
       "  (2, 0.010649858726363835),\n",
       "  (3, 0.0973459875872395),\n",
       "  (4, 0.053225144292303614)],\n",
       " [(0, 0.8169257340241797),\n",
       "  (1, 0.05284974093264249),\n",
       "  (2, 0.018595279217040874),\n",
       "  (3, 0.07380541162924582),\n",
       "  (4, 0.03782383419689119)],\n",
       " [(0, 0.035842293906810034),\n",
       "  (1, 0.01971326164874552),\n",
       "  (2, 0.9050179211469535),\n",
       "  (3, 0.022263578715191617),\n",
       "  (4, 0.01716294458229942)],\n",
       " [(0, 0.031829209121785544),\n",
       "  (1, 0.03270257156720039),\n",
       "  (2, 0.019941775836972344),\n",
       "  (3, 0.017467248908296942),\n",
       "  (4, 0.8980591945657448)],\n",
       " [(0, 0.049255441008018326),\n",
       "  (1, 0.036382479681448754),\n",
       "  (2, 0.856706485572465),\n",
       "  (3, 0.039818905798287244),\n",
       "  (4, 0.017836687939780722)],\n",
       " [(0, 0.016335604852351263),\n",
       "  (1, 0.14252573582717123),\n",
       "  (2, 0.022135227876854673),\n",
       "  (3, 0.4663863515538157),\n",
       "  (4, 0.3526170798898072)],\n",
       " [(0, 0.026876876876876878),\n",
       "  (1, 0.888088088088088),\n",
       "  (2, 0.013763763763763764),\n",
       "  (3, 0.023773773773773772),\n",
       "  (4, 0.047497497497497494)],\n",
       " [(0, 0.009361482342307444),\n",
       "  (1, 0.008715862870424172),\n",
       "  (2, 0.06165665956485247),\n",
       "  (3, 0.9126476854541933),\n",
       "  (4, 0.00761830976822261)],\n",
       " [(0, 0.10871905274488698),\n",
       "  (1, 0.23462105808715067),\n",
       "  (2, 0.21149782721365068),\n",
       "  (3, 0.37017103217318503),\n",
       "  (4, 0.07499102978112666)],\n",
       " [(0, 0.022911597729115978),\n",
       "  (1, 0.07137064071370641),\n",
       "  (2, 0.018856447688564478),\n",
       "  (3, 0.8688158961881589),\n",
       "  (4, 0.018045417680454177)],\n",
       " [(0, 0.09535732916014608),\n",
       "  (1, 0.5519561815336465),\n",
       "  (2, 0.007668231611893585),\n",
       "  (3, 0.2801251956181534),\n",
       "  (4, 0.06489306207616068)],\n",
       " [(0, 0.025485881921525486),\n",
       "  (1, 0.051063439677301065),\n",
       "  (2, 0.021635496883021636),\n",
       "  (3, 0.8700953428676201),\n",
       "  (4, 0.03171983865053172)],\n",
       " [(0, 0.02506913278027109),\n",
       "  (1, 0.016863865089079288),\n",
       "  (2, 0.018677183915862008),\n",
       "  (3, 0.9164513350559862),\n",
       "  (4, 0.022938483158801395)],\n",
       " [(0, 0.09823754789272031),\n",
       "  (1, 0.028620689655172414),\n",
       "  (2, 0.03517241379310345),\n",
       "  (3, 0.029846743295019157),\n",
       "  (4, 0.8081226053639847)],\n",
       " [(0, 0.014287014287014286),\n",
       "  (1, 0.20434070434070434),\n",
       "  (2, 0.28665028665028663),\n",
       "  (3, 0.04095004095004095),\n",
       "  (4, 0.4537719537719538)],\n",
       " [(0, 0.15489283699943598),\n",
       "  (1, 0.03722504230118443),\n",
       "  (2, 0.038987591652566274),\n",
       "  (3, 0.6888747884940778),\n",
       "  (4, 0.08001974055273547)],\n",
       " [(0, 0.05392689106044156),\n",
       "  (1, 0.014235734105440948),\n",
       "  (2, 0.01194353963083605),\n",
       "  (3, 0.907889974665219),\n",
       "  (4, 0.012003860538062494)],\n",
       " [(0, 0.015770252324037184),\n",
       "  (1, 0.34877158034528555),\n",
       "  (2, 0.01195219123505976),\n",
       "  (3, 0.6116091190792387),\n",
       "  (4, 0.011896857016378928)],\n",
       " [(0, 0.027244305493523895),\n",
       "  (1, 0.19234777430400476),\n",
       "  (2, 0.05091558731576597),\n",
       "  (3, 0.08778720659024367),\n",
       "  (4, 0.6417051262964617)],\n",
       " [(0, 0.03207489257213014),\n",
       "  (1, 0.05847145488029466),\n",
       "  (2, 0.027010435850214856),\n",
       "  (3, 0.8532842234499693),\n",
       "  (4, 0.029158993247391036)],\n",
       " [(0, 0.05452886151038102),\n",
       "  (1, 0.7252262529469922),\n",
       "  (2, 0.04631530914898471),\n",
       "  (3, 0.06768575557076584),\n",
       "  (4, 0.10624382082287627)],\n",
       " [(0, 0.7436980706942685),\n",
       "  (1, 0.0108435431629348),\n",
       "  (2, 0.16469511336431492),\n",
       "  (3, 0.03738910012674272),\n",
       "  (4, 0.0433741726517392)],\n",
       " [(0, 0.10307328605200945),\n",
       "  (1, 0.06274907125970956),\n",
       "  (2, 0.7566362715298885),\n",
       "  (3, 0.02998986828774063),\n",
       "  (4, 0.04755150287065181)],\n",
       " [(0, 0.030203784570596796),\n",
       "  (1, 0.027110625909752547),\n",
       "  (2, 0.8420063076176614),\n",
       "  (3, 0.03766375545851528),\n",
       "  (4, 0.06301552644347404)],\n",
       " [(0, 0.1643579919441988),\n",
       "  (1, 0.6384222418705177),\n",
       "  (2, 0.07210924452303762),\n",
       "  (3, 0.04764711661263384),\n",
       "  (4, 0.07746340504961194)],\n",
       " [(0, 0.15458648612770476),\n",
       "  (1, 0.03424930306650737),\n",
       "  (2, 0.13547059604407274),\n",
       "  (3, 0.03093057214921014),\n",
       "  (4, 0.6447630426125049)],\n",
       " [(0, 0.03616192411924119),\n",
       "  (1, 0.5599593495934959),\n",
       "  (2, 0.011178861788617886),\n",
       "  (3, 0.17869241192411925),\n",
       "  (4, 0.21400745257452575)],\n",
       " [(0, 0.17053467515574725),\n",
       "  (1, 0.037653179982200316),\n",
       "  (2, 0.014205517902375573),\n",
       "  (3, 0.7523105360443623),\n",
       "  (4, 0.025296090915314574)],\n",
       " [(0, 0.19924349881796694),\n",
       "  (1, 0.02799054373522459),\n",
       "  (2, 0.5620803782505911),\n",
       "  (3, 0.06458628841607567),\n",
       "  (4, 0.14609929078014186)],\n",
       " [(0, 0.03850385038503851),\n",
       "  (1, 0.015873015873015872),\n",
       "  (2, 0.828906700193829),\n",
       "  (3, 0.014144271570014143),\n",
       "  (4, 0.10257216197810257)],\n",
       " [(0, 0.02541417295515656),\n",
       "  (1, 0.14459189869025935),\n",
       "  (2, 0.028970422413045364),\n",
       "  (3, 0.7239136091595108),\n",
       "  (4, 0.07710989678202793)],\n",
       " [(0, 0.017928142846815587),\n",
       "  (1, 0.009614689510590616),\n",
       "  (2, 0.014169016120870383),\n",
       "  (3, 0.9389864816019663),\n",
       "  (4, 0.019301669919757104)],\n",
       " [(0, 0.12285714285714283),\n",
       "  (1, 0.09650793650793649),\n",
       "  (2, 0.7109523809523809),\n",
       "  (3, 0.03698412698412698),\n",
       "  (4, 0.03269841269841269)],\n",
       " [(0, 0.7835472578763127),\n",
       "  (1, 0.04226630364319979),\n",
       "  (2, 0.03584856735381823),\n",
       "  (3, 0.028393621159082068),\n",
       "  (4, 0.10994424996758718)],\n",
       " [(0, 0.058047385620915035),\n",
       "  (1, 0.0843545751633987),\n",
       "  (2, 0.7086601307189543),\n",
       "  (3, 0.047344771241830064),\n",
       "  (4, 0.10159313725490196)],\n",
       " [(0, 0.10675618460049598),\n",
       "  (1, 0.021230266739248774),\n",
       "  (2, 0.6777959233049053),\n",
       "  (3, 0.01989959474989415),\n",
       "  (4, 0.17431803060545575)],\n",
       " [(0, 0.40115000845594456),\n",
       "  (1, 0.03624781554766335),\n",
       "  (2, 0.32809064772535096),\n",
       "  (3, 0.022436439483623657),\n",
       "  (4, 0.21207508878741757)],\n",
       " [(0, 0.09321931101407084),\n",
       "  (1, 0.7573386705482775),\n",
       "  (2, 0.025230470645317808),\n",
       "  (3, 0.08794274623968946),\n",
       "  (4, 0.03626880155264435)],\n",
       " [(0, 0.1414141414141414),\n",
       "  (1, 0.5213792055897319),\n",
       "  (2, 0.05232778916989443),\n",
       "  (3, 0.22328548644338117),\n",
       "  (4, 0.06159337738285107)],\n",
       " [(0, 0.17012700585595864),\n",
       "  (1, 0.6167008898015058),\n",
       "  (2, 0.031409232641265494),\n",
       "  (3, 0.06167769412122595),\n",
       "  (4, 0.12008517758004411)],\n",
       " [(0, 0.018684472559011303),\n",
       "  (1, 0.9312950272359867),\n",
       "  (2, 0.011304398758273297),\n",
       "  (3, 0.02594740233116617),\n",
       "  (4, 0.012768699115562583)],\n",
       " [(0, 0.05108792033285588),\n",
       "  (1, 0.1299365664006548),\n",
       "  (2, 0.05633994952595321),\n",
       "  (3, 0.04665438919582566),\n",
       "  (4, 0.7159811745447104)],\n",
       " [(0, 0.5148975791433893),\n",
       "  (1, 0.05348644734119595),\n",
       "  (2, 0.33571280777984697),\n",
       "  (3, 0.06465963169873785),\n",
       "  (4, 0.03124353403683013)],\n",
       " [(0, 0.1392718275008837),\n",
       "  (1, 0.12907976905856014),\n",
       "  (2, 0.03835277483209615),\n",
       "  (3, 0.6719099799693649),\n",
       "  (4, 0.021385648639095086)],\n",
       " [(0, 0.35112563543936093),\n",
       "  (1, 0.07320261437908497),\n",
       "  (2, 0.021713870733478576),\n",
       "  (3, 0.49840232389252),\n",
       "  (4, 0.05555555555555555)],\n",
       " [(0, 0.01138811942136042),\n",
       "  (1, 0.6296168051708217),\n",
       "  (2, 0.024469067405355493),\n",
       "  (3, 0.010041551246537396),\n",
       "  (4, 0.3244844567559249)],\n",
       " [(0, 0.06874464114591214),\n",
       "  (1, 0.01044030866999546),\n",
       "  (2, 0.6930952741211479),\n",
       "  (3, 0.021889342815352802),\n",
       "  (4, 0.20583043324759168)],\n",
       " [(0, 0.01836139169472503),\n",
       "  (1, 0.014590347923681257),\n",
       "  (2, 0.00835016835016835),\n",
       "  (3, 0.026980920314253646),\n",
       "  (4, 0.9317171717171717)],\n",
       " [(0, 0.03361268811390466),\n",
       "  (1, 0.7137064071370641),\n",
       "  (2, 0.0267639902676399),\n",
       "  (3, 0.03126971253491935),\n",
       "  (4, 0.19464720194647203)],\n",
       " [(0, 0.04981703260146374),\n",
       "  (1, 0.05181304058549568),\n",
       "  (2, 0.12204757152361943),\n",
       "  (3, 0.04150033266799734),\n",
       "  (4, 0.7348220226214238)],\n",
       " [(0, 0.1956583833190517),\n",
       "  (1, 0.01649528706083976),\n",
       "  (2, 0.35718366181091116),\n",
       "  (3, 0.02849185946872322),\n",
       "  (4, 0.40217080834047414)],\n",
       " [(0, 0.03621192158741566),\n",
       "  (1, 0.07721483610811675),\n",
       "  (2, 0.010739809158781491),\n",
       "  (3, 0.06272208248492833),\n",
       "  (4, 0.8131113506607578)],\n",
       " [(0, 0.04818253534801869),\n",
       "  (1, 0.024880150494568844),\n",
       "  (2, 0.16912434006917895),\n",
       "  (3, 0.019540020632319924),\n",
       "  (4, 0.7382729534559136)],\n",
       " [(0, 0.016288539848749273),\n",
       "  (1, 0.010739696603570948),\n",
       "  (2, 0.014409092943124356),\n",
       "  (3, 0.9488074461896452),\n",
       "  (4, 0.00975522441491028)],\n",
       " [(0, 0.4687625957275292),\n",
       "  (1, 0.02924448027229164),\n",
       "  (2, 0.46146267186170453),\n",
       "  (3, 0.0177347843611447),\n",
       "  (4, 0.02279546777732993)],\n",
       " [(0, 0.046798029556650245),\n",
       "  (1, 0.6516146688560481),\n",
       "  (2, 0.02079912424740011),\n",
       "  (3, 0.24010217113665389),\n",
       "  (4, 0.04068600620324758)],\n",
       " [(0, 0.16290620122037747),\n",
       "  (1, 0.038420604512558534),\n",
       "  (2, 0.09582091670214275),\n",
       "  (3, 0.08798070100752094),\n",
       "  (4, 0.6148715765574003)],\n",
       " [(0, 0.6945019553715206),\n",
       "  (1, 0.012330342765125374),\n",
       "  (2, 0.09786059351276742),\n",
       "  (3, 0.09353577179664137),\n",
       "  (4, 0.10177133655394525)],\n",
       " [(0, 0.025734767025089606),\n",
       "  (1, 0.031827956989247314),\n",
       "  (2, 0.022867383512544803),\n",
       "  (3, 0.13756272401433692),\n",
       "  (4, 0.7820071684587814)],\n",
       " [(0, 0.037628682996095134),\n",
       "  (1, 0.04251962292430876),\n",
       "  (2, 0.8763854376207943),\n",
       "  (3, 0.022719204827831026),\n",
       "  (4, 0.020747051630970695)],\n",
       " [(0, 0.02193497845671759),\n",
       "  (1, 0.6465596030813423),\n",
       "  (2, 0.04778691735213475),\n",
       "  (3, 0.22346259302781044),\n",
       "  (4, 0.06025590808199505)],\n",
       " [(0, 0.8067713142505386),\n",
       "  (1, 0.03397968605724838),\n",
       "  (2, 0.0164973838104032),\n",
       "  (3, 0.08599569098184057),\n",
       "  (4, 0.05675592489996922)],\n",
       " [(0, 0.26905669972031526),\n",
       "  (1, 0.30658530383930843),\n",
       "  (2, 0.013323162979913553),\n",
       "  (3, 0.05736079328756674),\n",
       "  (4, 0.353674040172896)],\n",
       " [(0, 0.02454724504011145),\n",
       "  (1, 0.008983042705481099),\n",
       "  (2, 0.9165585819282318),\n",
       "  (3, 0.016572993226689727),\n",
       "  (4, 0.03333813709948601)],\n",
       " [(0, 0.6528704939919894),\n",
       "  (1, 0.06809078771695595),\n",
       "  (2, 0.12421500271967563),\n",
       "  (3, 0.08060129555456659),\n",
       "  (4, 0.07422242001681255)],\n",
       " [(0, 0.7857288481141692),\n",
       "  (1, 0.03289160720353381),\n",
       "  (2, 0.01746517159361196),\n",
       "  (3, 0.1109072375127421),\n",
       "  (4, 0.053007135575942915)],\n",
       " [(0, 0.22072297184740963),\n",
       "  (1, 0.24250374812593703),\n",
       "  (2, 0.3089705147426287),\n",
       "  (3, 0.05867899383641512),\n",
       "  (4, 0.16912377144760954)],\n",
       " [(0, 0.04111012169783319),\n",
       "  (1, 0.011724547343425349),\n",
       "  (2, 0.007964776887305828),\n",
       "  (3, 0.09691303057287029),\n",
       "  (4, 0.8422875234985654)],\n",
       " [(0, 0.5556617295747731),\n",
       "  (1, 0.018898975420714555),\n",
       "  (2, 0.06306736741519352),\n",
       "  (3, 0.33556298773690085),\n",
       "  (4, 0.02680893985241812)],\n",
       " [(0, 0.012547735951991271),\n",
       "  (1, 0.011335394314117718),\n",
       "  (2, 0.009031945202157969),\n",
       "  (3, 0.9549008910711039),\n",
       "  (4, 0.012184033460629205)],\n",
       " [(0, 0.5948578547966927),\n",
       "  (1, 0.022086306489976216),\n",
       "  (2, 0.00781515460414543),\n",
       "  (3, 0.3564390078151546),\n",
       "  (4, 0.018801676294031035)],\n",
       " [(0, 0.16857843976488043),\n",
       "  (1, 0.36660389202762084),\n",
       "  (2, 0.012669063516521143),\n",
       "  (3, 0.413856074873024),\n",
       "  (4, 0.038292529817953544)],\n",
       " [(0, 0.25731620271234834),\n",
       "  (1, 0.06721389483702117),\n",
       "  (2, 0.01986676183678325),\n",
       "  (3, 0.24559838210801807),\n",
       "  (4, 0.4100047585058292)],\n",
       " [(0, 0.04330922945338621),\n",
       "  (1, 0.13949148588756707),\n",
       "  (2, 0.04035456029857709),\n",
       "  (3, 0.7630044320037322),\n",
       "  (4, 0.013840292356737423)],\n",
       " [(0, 0.0851246341709557),\n",
       "  (1, 0.15869411645978404),\n",
       "  (2, 0.016449692199011),\n",
       "  (3, 0.05454637198506408),\n",
       "  (4, 0.6851851851851852)],\n",
       " [(0, 0.14759073454725632),\n",
       "  (1, 0.6657995788430572),\n",
       "  (2, 0.05487427226557662),\n",
       "  (3, 0.11228787315743839),\n",
       "  (4, 0.019447541186671625)],\n",
       " [(0, 0.4869091169930421),\n",
       "  (1, 0.01348540276881142),\n",
       "  (2, 0.45319561007101355),\n",
       "  (3, 0.025894842550749586),\n",
       "  (4, 0.02051502761638333)],\n",
       " [(0, 0.555331879438573),\n",
       "  (1, 0.026114186657719623),\n",
       "  (2, 0.015209975954817424),\n",
       "  (3, 0.21769278085332439),\n",
       "  (4, 0.1856511770955656)],\n",
       " [(0, 0.02347633249216181),\n",
       "  (1, 0.030205704672325456),\n",
       "  (2, 0.854706737019194),\n",
       "  (3, 0.018123422803395274),\n",
       "  (4, 0.07348780301292346)],\n",
       " [(0, 0.040394435640077477),\n",
       "  (1, 0.011163937312907202),\n",
       "  (2, 0.00552914245465751),\n",
       "  (3, 0.012326113752421201),\n",
       "  (4, 0.9305863708399366)],\n",
       " [(0, 0.12055302647108414),\n",
       "  (1, 0.2975467880627213),\n",
       "  (2, 0.44912325071657394),\n",
       "  (3, 0.05096105209914011),\n",
       "  (4, 0.08181588265048052)],\n",
       " [(0, 0.838326820089738),\n",
       "  (1, 0.01794760457374439),\n",
       "  (2, 0.03160129299946929),\n",
       "  (3, 0.04192599025425773),\n",
       "  (4, 0.07019829208279056)],\n",
       " [(0, 0.09997379454926625),\n",
       "  (1, 0.023322851153039833),\n",
       "  (2, 0.09741876310272536),\n",
       "  (3, 0.06577568134171907),\n",
       "  (4, 0.7135089098532494)],\n",
       " [(0, 0.15988914084152178),\n",
       "  (1, 0.015016376921138826),\n",
       "  (2, 0.21103552532123962),\n",
       "  (3, 0.5439153439153439),\n",
       "  (4, 0.07014361300075586)],\n",
       " [(0, 0.16453953953953954),\n",
       "  (1, 0.2909159159159159),\n",
       "  (2, 0.20839589589589588),\n",
       "  (3, 0.2296046046046046),\n",
       "  (4, 0.10654404404404405)],\n",
       " [(0, 0.23710183938986093),\n",
       "  (1, 0.25332735157768804),\n",
       "  (2, 0.0816509645580978),\n",
       "  (3, 0.3933004336772843),\n",
       "  (4, 0.03461941079706894)],\n",
       " [(0, 0.06928838951310862),\n",
       "  (1, 0.052507894543585225),\n",
       "  (2, 0.025482852316956746),\n",
       "  (3, 0.8311669236983182),\n",
       "  (4, 0.02155393992803114)],\n",
       " [(0, 0.37914267434420984),\n",
       "  (1, 0.036638942205161014),\n",
       "  (2, 0.0581787161441672),\n",
       "  (3, 0.023117935593943272),\n",
       "  (4, 0.5029217317125186)],\n",
       " [(0, 0.04761341689330338),\n",
       "  (1, 0.047849907374561504),\n",
       "  (2, 0.720980647195617),\n",
       "  (3, 0.08162863111426431),\n",
       "  (4, 0.10192739742225375)],\n",
       " [(0, 0.21689761354888376),\n",
       "  (1, 0.12637926610212985),\n",
       "  (2, 0.023351295868616884),\n",
       "  (3, 0.5915447780343854),\n",
       "  (4, 0.04182704644598409)],\n",
       " [(0, 0.2478423930233609),\n",
       "  (1, 0.12426008765984366),\n",
       "  (2, 0.014233428222854819),\n",
       "  (3, 0.12828159595138042),\n",
       "  (4, 0.4853824951425602)],\n",
       " [(0, 0.2686631944444445),\n",
       "  (1, 0.01965525793650794),\n",
       "  (2, 0.535280257936508),\n",
       "  (3, 0.05220734126984128),\n",
       "  (4, 0.12419394841269843)],\n",
       " [(0, 0.6227053140096618),\n",
       "  (1, 0.15706521739130436),\n",
       "  (2, 0.013707729468599035),\n",
       "  (3, 0.1252415458937198),\n",
       "  (4, 0.08128019323671498)],\n",
       " [(0, 0.023613963039014373),\n",
       "  (1, 0.10312571298197581),\n",
       "  (2, 0.00787132101300479),\n",
       "  (3, 0.849304129591604),\n",
       "  (4, 0.016084873374401096)],\n",
       " [(0, 0.06675874769797423),\n",
       "  (1, 0.1153314917127072),\n",
       "  (2, 0.09039287906691224),\n",
       "  (3, 0.6187845303867404),\n",
       "  (4, 0.10873235113566607)],\n",
       " [(0, 0.2080910398299256),\n",
       "  (1, 0.4637653973613456),\n",
       "  (2, 0.051209904333145755),\n",
       "  (3, 0.20852873131995248),\n",
       "  (4, 0.06840492715563058)],\n",
       " [(0, 0.05698005698005699),\n",
       "  (1, 0.5457175925925927),\n",
       "  (2, 0.035122863247863255),\n",
       "  (3, 0.34370548433048437),\n",
       "  (4, 0.018474002849002854)],\n",
       " [(0, 0.6910557666150743),\n",
       "  (1, 0.015243187467754795),\n",
       "  (2, 0.21898597626752966),\n",
       "  (3, 0.04554195394212279),\n",
       "  (4, 0.029173115707518408)],\n",
       " [(0, 0.6128707893413776),\n",
       "  (1, 0.011340148595050555),\n",
       "  (2, 0.30344673481928386),\n",
       "  (3, 0.057817998994469585),\n",
       "  (4, 0.014524328249818447)],\n",
       " [(0, 0.04765919181700669),\n",
       "  (1, 0.18720845276232226),\n",
       "  (2, 0.030854830551340414),\n",
       "  (3, 0.7093800932951161),\n",
       "  (4, 0.02489743157421458)],\n",
       " [(0, 0.41377132027795327),\n",
       "  (1, 0.06703165578718327),\n",
       "  (2, 0.12213097494209307),\n",
       "  (3, 0.27486488383519336),\n",
       "  (4, 0.12220116515757704)],\n",
       " [(0, 0.011278656368763025),\n",
       "  (1, 0.007846021821748192),\n",
       "  (2, 0.031833598953863756),\n",
       "  (3, 0.10306076580442156),\n",
       "  (4, 0.8459809570512035)],\n",
       " [(0, 0.02941015089163237),\n",
       "  (1, 0.8065843621399177),\n",
       "  (2, 0.04060356652949246),\n",
       "  (3, 0.1025514403292181),\n",
       "  (4, 0.020850480109739368)],\n",
       " [(0, 0.5899672269795962),\n",
       "  (1, 0.07918384607252353),\n",
       "  (2, 0.2527751347922614),\n",
       "  (3, 0.05449836134897981),\n",
       "  (4, 0.023575430806639184)],\n",
       " [(0, 0.014182499331014183),\n",
       "  (1, 0.9180864032349181),\n",
       "  (2, 0.00999018820800999),\n",
       "  (3, 0.039633693099039635),\n",
       "  (4, 0.018107216127018107)],\n",
       " [(0, 0.009178321678321678),\n",
       "  (1, 0.00810994560994561),\n",
       "  (2, 0.006847319347319347),\n",
       "  (3, 0.9689685314685315),\n",
       "  (4, 0.006895881895881896)],\n",
       " [(0, 0.07430555555555556),\n",
       "  (1, 0.2858630952380952),\n",
       "  (2, 0.023065476190476192),\n",
       "  (3, 0.025148809523809525),\n",
       "  (4, 0.5916170634920634)],\n",
       " [(0, 0.012245862884160756),\n",
       "  (1, 0.013191489361702127),\n",
       "  (2, 0.005626477541371158),\n",
       "  (3, 0.9516312056737589),\n",
       "  (4, 0.01730496453900709)],\n",
       " [(0, 0.040051228315287),\n",
       "  (1, 0.6331354057515428),\n",
       "  (2, 0.1584973027515815),\n",
       "  (3, 0.14728140645011065),\n",
       "  (4, 0.021034656731478252)],\n",
       " [(0, 0.8527473450027285),\n",
       "  (1, 0.019938714687486887),\n",
       "  (2, 0.08437224530915503),\n",
       "  (3, 0.018847332409856025),\n",
       "  (4, 0.024094362590773626)],\n",
       " [(0, 0.03869047619047619),\n",
       "  (1, 0.03464873603762492),\n",
       "  (2, 0.8160640799529688),\n",
       "  (3, 0.043944738389182836),\n",
       "  (4, 0.0666519694297472)],\n",
       " [(0, 0.02780656303972366),\n",
       "  (1, 0.7740356937248128),\n",
       "  (2, 0.012665515256188831),\n",
       "  (3, 0.1641335636154289),\n",
       "  (4, 0.02135866436384571)],\n",
       " [(0, 0.8114335398326269),\n",
       "  (1, 0.006521030322791001),\n",
       "  (2, 0.009238126290620584),\n",
       "  (3, 0.1640401405644314),\n",
       "  (4, 0.008767162989530123)],\n",
       " [(0, 0.7244982412580178),\n",
       "  (1, 0.03284709290295883),\n",
       "  (2, 0.032691909786881856),\n",
       "  (3, 0.18052969170287606),\n",
       "  (4, 0.029433064349265466)],\n",
       " [(0, 0.8886861313868614),\n",
       "  (1, 0.03602324952689916),\n",
       "  (2, 0.017977831846444987),\n",
       "  (3, 0.02142470938091376),\n",
       "  (4, 0.035888077858880776)],\n",
       " [(0, 0.4820886460230723),\n",
       "  (1, 0.01700060716454159),\n",
       "  (2, 0.4225313241706684),\n",
       "  (3, 0.01898769111883866),\n",
       "  (4, 0.05939173152287906)],\n",
       " [(0, 0.2231132806446614),\n",
       "  (1, 0.1460560979389431),\n",
       "  (2, 0.08228730822873083),\n",
       "  (3, 0.06276150627615062),\n",
       "  (4, 0.485781806911514)],\n",
       " [(0, 0.07152114999410863),\n",
       "  (1, 0.016967126193001062),\n",
       "  (2, 0.8355720513726876),\n",
       "  (3, 0.04830917874396135),\n",
       "  (4, 0.02763049369624131)],\n",
       " [(0, 0.6239903951102379),\n",
       "  (1, 0.10041475660336172),\n",
       "  (2, 0.09419340755293604),\n",
       "  (3, 0.08322418685876447),\n",
       "  (4, 0.09817725387469985)],\n",
       " [(0, 0.20850557434548414),\n",
       "  (1, 0.15758486784416886),\n",
       "  (2, 0.46912188400350746),\n",
       "  (3, 0.0604409369911061),\n",
       "  (4, 0.10434673681573343)],\n",
       " [(0, 0.02296166569371473),\n",
       "  (1, 0.016475319452552376),\n",
       "  (2, 0.8700136213271065),\n",
       "  (3, 0.008886294350392425),\n",
       "  (4, 0.08166309917623403)],\n",
       " [(0, 0.0657253319885695),\n",
       "  (1, 0.13909900823667842),\n",
       "  (2, 0.03370314338544293),\n",
       "  (3, 0.2571020339552866),\n",
       "  (4, 0.5043704824340225)],\n",
       " [(0, 0.17336175329288953),\n",
       "  (1, 0.5838115538066349),\n",
       "  (2, 0.020057932994479964),\n",
       "  (3, 0.18844619336503246),\n",
       "  (4, 0.034322566540962995)],\n",
       " [(0, 0.04054564334938167),\n",
       "  (1, 0.08090248277164165),\n",
       "  (2, 0.2661663362597942),\n",
       "  (3, 0.5935995468705749),\n",
       "  (4, 0.01878599074860757)],\n",
       " [(0, 0.020837232516064633),\n",
       "  (1, 0.25634786948655564),\n",
       "  (2, 0.02913469336827001),\n",
       "  (3, 0.675899931374384),\n",
       "  (4, 0.01778027325472581)],\n",
       " [(0, 0.02287926786342837),\n",
       "  (1, 0.08066408541593335),\n",
       "  (2, 0.061656693652469785),\n",
       "  (3, 0.08089874457350699),\n",
       "  (4, 0.7539012084946615)],\n",
       " [(0, 0.008819423162052644),\n",
       "  (1, 0.9374467940204992),\n",
       "  (2, 0.0165491878639289),\n",
       "  (3, 0.013552627098443832),\n",
       "  (4, 0.023631967855075424)],\n",
       " [(0, 0.015298290191096289),\n",
       "  (1, 0.14652480016939284),\n",
       "  (2, 0.010322375734476735),\n",
       "  (3, 0.03700174686358584),\n",
       "  (4, 0.7908527870414483)],\n",
       " [(0, 0.30708352922739385),\n",
       "  (1, 0.08501802225356528),\n",
       "  (2, 0.01151857075693465),\n",
       "  (3, 0.0640965365930105),\n",
       "  (4, 0.5322833411690957)],\n",
       " [(0, 0.2111574074074074),\n",
       "  (1, 0.058333333333333334),\n",
       "  (2, 0.6125),\n",
       "  (3, 0.04226851851851852),\n",
       "  (4, 0.07574074074074075)],\n",
       " [(0, 0.1984408920465759),\n",
       "  (1, 0.07144266824551017),\n",
       "  (2, 0.20021709098085652),\n",
       "  (3, 0.2910992697848826),\n",
       "  (4, 0.23880007894217486)],\n",
       " [(0, 0.03090507726269316),\n",
       "  (1, 0.020498265531378113),\n",
       "  (2, 0.03758015347419321),\n",
       "  (3, 0.0315357931251971),\n",
       "  (4, 0.8794807106065384)],\n",
       " [(0, 0.02881702247586839),\n",
       "  (1, 0.8795885295568239),\n",
       "  (2, 0.018389346861128728),\n",
       "  (3, 0.03628549284858734),\n",
       "  (4, 0.03691960825759178)],\n",
       " [(0, 0.0967098703888335),\n",
       "  (1, 0.058823529411764705),\n",
       "  (2, 0.021657250470809793),\n",
       "  (3, 0.05887891879915808),\n",
       "  (4, 0.7639304309294339)],\n",
       " [(0, 0.010791701553632896),\n",
       "  (1, 0.0056749465066517814),\n",
       "  (2, 0.0066362762427512635),\n",
       "  (3, 0.9658572890501442),\n",
       "  (4, 0.01103978664681986)],\n",
       " [(0, 0.34645139589640095),\n",
       "  (1, 0.06760847628657922),\n",
       "  (2, 0.03957842807489629),\n",
       "  (3, 0.3957842807489629),\n",
       "  (4, 0.15057741899316066)],\n",
       " [(0, 0.03609719903542942),\n",
       "  (1, 0.017325171582266742),\n",
       "  (2, 0.02429975885735485),\n",
       "  (3, 0.025301428306436655),\n",
       "  (4, 0.8969764422185124)],\n",
       " [(0, 0.8833300244854743),\n",
       "  (1, 0.025014889815366294),\n",
       "  (2, 0.016676593210244197),\n",
       "  (3, 0.03877969690953611),\n",
       "  (4, 0.036198795579379266)],\n",
       " [(0, 0.15011682242990654),\n",
       "  (1, 0.05101246105919003),\n",
       "  (2, 0.017847871235721704),\n",
       "  (3, 0.7078141225337488),\n",
       "  (4, 0.07320872274143302)],\n",
       " [(0, 0.02228228228228228),\n",
       "  (1, 0.017777777777777778),\n",
       "  (2, 0.012972972972972972),\n",
       "  (3, 0.5046846846846846),\n",
       "  (4, 0.4422822822822823)],\n",
       " [(0, 0.6470047386661009),\n",
       "  (1, 0.016267062734281065),\n",
       "  (2, 0.032321946389419334),\n",
       "  (3, 0.19845816535822902),\n",
       "  (4, 0.10594808685196973)],\n",
       " [(0, 0.5159722222222223),\n",
       "  (1, 0.05726495726495726),\n",
       "  (2, 0.20865384615384616),\n",
       "  (3, 0.016079059829059827),\n",
       "  (4, 0.20202991452991453)],\n",
       " [(0, 0.6884372647791095),\n",
       "  (1, 0.06218516588500955),\n",
       "  (2, 0.011116901163800591),\n",
       "  (3, 0.11962248856464594),\n",
       "  (4, 0.11863817960743443)],\n",
       " [(0, 0.11128691983122363),\n",
       "  (1, 0.18852555086732303),\n",
       "  (2, 0.017815283638068447),\n",
       "  (3, 0.5713783403656821),\n",
       "  (4, 0.11099390529770277)],\n",
       " [(0, 0.020140646976090013),\n",
       "  (1, 0.010295358649789029),\n",
       "  (2, 0.014289732770745429),\n",
       "  (3, 0.06222222222222222),\n",
       "  (4, 0.8930520393811533)],\n",
       " [(0, 0.18798011748757343),\n",
       "  (1, 0.026811266757041725),\n",
       "  (2, 0.028920018075011298),\n",
       "  (3, 0.07686900637646232),\n",
       "  (4, 0.6794195913039113)],\n",
       " [(0, 0.7760444444444444),\n",
       "  (1, 0.12635555555555555),\n",
       "  (2, 0.019066666666666666),\n",
       "  (3, 0.038266666666666664),\n",
       "  (4, 0.040266666666666666)],\n",
       " [(0, 0.02309236947791165),\n",
       "  (1, 0.8105756358768408),\n",
       "  (2, 0.023510709504685413),\n",
       "  (3, 0.06082663989290496),\n",
       "  (4, 0.08199464524765732)],\n",
       " [(0, 0.021027888952417254),\n",
       "  (1, 0.01905850962454736),\n",
       "  (2, 0.8717997585922114),\n",
       "  (3, 0.02922304809097262),\n",
       "  (4, 0.05889079473985134)],\n",
       " [(0, 0.05546026300743282),\n",
       "  (1, 0.07502699955530144),\n",
       "  (2, 0.023060796645702306),\n",
       "  (3, 0.8352709484784957),\n",
       "  (4, 0.011180992313067784)],\n",
       " [(0, 0.6880838964587719),\n",
       "  (1, 0.02311583767216868),\n",
       "  (2, 0.009379269598611573),\n",
       "  (3, 0.14726191794985413),\n",
       "  (4, 0.1321590783205938)],\n",
       " [(0, 0.0365339370533138),\n",
       "  (1, 0.6116216096240068),\n",
       "  (2, 0.21560793714209614),\n",
       "  (3, 0.10054601145292316),\n",
       "  (4, 0.03569050472766014)],\n",
       " [(0, 0.06692584232087619),\n",
       "  (1, 0.018602123568263525),\n",
       "  (2, 0.281790820165538),\n",
       "  (3, 0.05672602625198562),\n",
       "  (4, 0.5759551876933366)],\n",
       " [(0, 0.14804894179894182),\n",
       "  (1, 0.6311507936507937),\n",
       "  (2, 0.015277777777777779),\n",
       "  (3, 0.08098544973544974),\n",
       "  (4, 0.12453703703703706)],\n",
       " [(0, 0.021856104731791202),\n",
       "  (1, 0.012425805735840683),\n",
       "  (2, 0.9216175736395407),\n",
       "  (3, 0.016086980640150884),\n",
       "  (4, 0.028013535252676542)],\n",
       " [(0, 0.010054728267786688),\n",
       "  (1, 0.012600229095074456),\n",
       "  (2, 0.008081965126638667),\n",
       "  (3, 0.9542446226295024),\n",
       "  (4, 0.015018454880997836)],\n",
       " [(0, 0.027810342581737656),\n",
       "  (1, 0.1132603881724632),\n",
       "  (2, 0.03575615474794842),\n",
       "  (3, 0.4437280187573271),\n",
       "  (4, 0.37944509574052365)],\n",
       " [(0, 0.019694502597503296),\n",
       "  (1, 0.0222532371869427),\n",
       "  (2, 0.17841358455454756),\n",
       "  (3, 0.03248817554470032),\n",
       "  (4, 0.7471505001163061)],\n",
       " [(0, 0.795647268624668),\n",
       "  (1, 0.03894166029770353),\n",
       "  (2, 0.05939513047275904),\n",
       "  (3, 0.026822433307857482),\n",
       "  (4, 0.07919350729701205)],\n",
       " [(0, 0.020974706971005553),\n",
       "  (1, 0.8108849132908356),\n",
       "  (2, 0.059085612447734595),\n",
       "  (3, 0.035917472067996434),\n",
       "  (4, 0.07313729522242786)],\n",
       " [(0, 0.023709594286135043),\n",
       "  (1, 0.022163316397908844),\n",
       "  (2, 0.014726456078344746),\n",
       "  (3, 0.892865032030042),\n",
       "  (4, 0.046535601207569395)],\n",
       " [(0, 0.8273381294964028),\n",
       "  (1, 0.02305847629588637),\n",
       "  (2, 0.01955358789891164),\n",
       "  (3, 0.07108159626145237),\n",
       "  (4, 0.05896821004734674)],\n",
       " [(0, 0.020992111244818826),\n",
       "  (1, 0.8491776975531489),\n",
       "  (2, 0.013838748495788207),\n",
       "  (3, 0.028747158711057628),\n",
       "  (4, 0.08724428399518652)],\n",
       " [(0, 0.1602582225991895),\n",
       "  (1, 0.22759400621996043),\n",
       "  (2, 0.01653944020356234),\n",
       "  (3, 0.5846762793327679),\n",
       "  (4, 0.010932051644519838)],\n",
       " [(0, 0.12854609929078015),\n",
       "  (1, 0.025561465721040188),\n",
       "  (2, 0.4148197399527187),\n",
       "  (3, 0.0644208037825059),\n",
       "  (4, 0.3666518912529551)],\n",
       " [(0, 0.014007597340930674),\n",
       "  (1, 0.024691358024691357),\n",
       "  (2, 0.014185660018993353),\n",
       "  (3, 0.9250356125356125),\n",
       "  (4, 0.02207977207977208)],\n",
       " [(0, 0.10235300986746074),\n",
       "  (1, 0.03970339230454838),\n",
       "  (2, 0.06901383779996496),\n",
       "  (3, 0.03152916447714135),\n",
       "  (4, 0.7574005955508846)],\n",
       " [(0, 0.14617604617604618),\n",
       "  (1, 0.04448051948051948),\n",
       "  (2, 0.47748917748917746),\n",
       "  (3, 0.11829004329004329),\n",
       "  (4, 0.21356421356421357)],\n",
       " [(0, 0.08824993776450087),\n",
       "  (1, 0.04841921832213095),\n",
       "  (2, 0.2300638951124388),\n",
       "  (3, 0.05107459961828894),\n",
       "  (4, 0.5821923491826404)],\n",
       " [(0, 0.01799286489840236),\n",
       "  (1, 0.026679075539010395),\n",
       "  (2, 0.008375988832014892),\n",
       "  (3, 0.9298898712579495),\n",
       "  (4, 0.01706219947262293)],\n",
       " [(0, 0.12497235124972352),\n",
       "  (1, 0.022045270220452703),\n",
       "  (2, 0.08818108088181081),\n",
       "  (3, 0.230037602300376),\n",
       "  (4, 0.5347636953476369)],\n",
       " [(0, 0.7207325049051668),\n",
       "  (1, 0.025070852408981905),\n",
       "  (2, 0.1704091272436596),\n",
       "  (3, 0.0316110747765424),\n",
       "  (4, 0.0521764406656493)],\n",
       " [(0, 0.04382858609186554),\n",
       "  (1, 0.030433666923274313),\n",
       "  (2, 0.013754169874262253),\n",
       "  (3, 0.06620477290223249),\n",
       "  (4, 0.8457788042083654)],\n",
       " [(0, 0.34333489607751205),\n",
       "  (1, 0.038547689743189034),\n",
       "  (2, 0.22607699119654107),\n",
       "  (3, 0.36073344793457307),\n",
       "  (4, 0.0313069750481846)],\n",
       " [(0, 0.12970910824988077),\n",
       "  (1, 0.2312827849308536),\n",
       "  (2, 0.04938271604938271),\n",
       "  (3, 0.08891008318762253),\n",
       "  (4, 0.5007153075822603)],\n",
       " [(0, 0.009496962985335079),\n",
       "  (1, 0.006275378368401624),\n",
       "  (2, 0.009731870196986476),\n",
       "  (3, 0.9655693144065237),\n",
       "  (4, 0.008926474042753112)],\n",
       " [(0, 0.6226429795854761),\n",
       "  (1, 0.031790556334735855),\n",
       "  (2, 0.10604643914601838),\n",
       "  (3, 0.14297958547607917),\n",
       "  (4, 0.09654043945769052)],\n",
       " [(0, 0.06880695594013879),\n",
       "  (1, 0.7414095811387008),\n",
       "  (2, 0.01889474124237104),\n",
       "  (3, 0.1145389181506563),\n",
       "  (4, 0.0563498035281331)],\n",
       " [(0, 0.24549956036836504),\n",
       "  (1, 0.046369568235457445),\n",
       "  (2, 0.10898236845758712),\n",
       "  (3, 0.1418390485445879),\n",
       "  (4, 0.4573094543940025)],\n",
       " [(0, 0.09208321009563246),\n",
       "  (1, 0.5613230799566203),\n",
       "  (2, 0.018633540372670808),\n",
       "  (3, 0.31233362910381546),\n",
       "  (4, 0.01562654047126097)],\n",
       " [(0, 0.6823051059504263),\n",
       "  (1, 0.009578544061302683),\n",
       "  (2, 0.2692157322699195),\n",
       "  (3, 0.01794510907811401),\n",
       "  (4, 0.020955508640237708)],\n",
       " [(0, 0.026737293479631736),\n",
       "  (1, 0.0390969857485181),\n",
       "  (2, 0.8506116786480011),\n",
       "  (3, 0.03991676125614832),\n",
       "  (4, 0.04363728086770085)],\n",
       " [(0, 0.6257644349138274),\n",
       "  (1, 0.051703597807958065),\n",
       "  (2, 0.21904534985306964),\n",
       "  (3, 0.047414820109602095),\n",
       "  (4, 0.05607179731554285)],\n",
       " [(0, 0.02859889449651526),\n",
       "  (1, 0.03901305775855163),\n",
       "  (2, 0.022029960746615396),\n",
       "  (3, 0.019065929664343506),\n",
       "  (4, 0.8912921573339742)],\n",
       " [(0, 0.06293285155073773),\n",
       "  (1, 0.01771554752584563),\n",
       "  (2, 0.012245307638261567),\n",
       "  (3, 0.8943089430894309),\n",
       "  (4, 0.012797350195724179)],\n",
       " [(0, 0.7012227538543327),\n",
       "  (1, 0.01594896331738437),\n",
       "  (2, 0.01541733120680489),\n",
       "  (3, 0.11766790714159135),\n",
       "  (4, 0.1497430444798866)],\n",
       " [(0, 0.16978776529338327),\n",
       "  (1, 0.13137989278108247),\n",
       "  (2, 0.01424689726077697),\n",
       "  (3, 0.6501432033487552),\n",
       "  (4, 0.034442241316002056)],\n",
       " [(0, 0.008440934352931345),\n",
       "  (1, 0.008775228782750407),\n",
       "  (2, 0.09987046090844512),\n",
       "  (3, 0.8626885629518198),\n",
       "  (4, 0.02022481300405332)],\n",
       " [(0, 0.06862295583065872),\n",
       "  (1, 0.7147333027663152),\n",
       "  (2, 0.01184471954760813),\n",
       "  (3, 0.1686535228488461),\n",
       "  (4, 0.03614549900657191)],\n",
       " [(0, 0.03941472336534065),\n",
       "  (1, 0.8024691358024691),\n",
       "  (2, 0.059167809785093735),\n",
       "  (3, 0.042615454961133975),\n",
       "  (4, 0.056332876085962505)],\n",
       " [(0, 0.8433063171811233),\n",
       "  (1, 0.02230269586710353),\n",
       "  (2, 0.06259699948266945),\n",
       "  (3, 0.0421337012128528),\n",
       "  (4, 0.02966028625625108)],\n",
       " [(0, 0.05650301464254953),\n",
       "  (1, 0.038874533448176866),\n",
       "  (2, 0.01154177433247201),\n",
       "  (3, 0.8699971289118577),\n",
       "  (4, 0.02308354866494402)],\n",
       " [(0, 0.1644069664902998),\n",
       "  (1, 0.6074735449735449),\n",
       "  (2, 0.019345238095238092),\n",
       "  (3, 0.08735670194003525),\n",
       "  (4, 0.12141754850088181)],\n",
       " [(0, 0.43198153643257503),\n",
       "  (1, 0.026178700956149027),\n",
       "  (2, 0.12350807781074843),\n",
       "  (3, 0.3333333333333333),\n",
       "  (4, 0.0849983514671942)],\n",
       " [(0, 0.028915135608048995),\n",
       "  (1, 0.011986001749781277),\n",
       "  (2, 0.8346456692913385),\n",
       "  (3, 0.07939632545931759),\n",
       "  (4, 0.04505686789151356)],\n",
       " [(0, 0.6632793597617719),\n",
       "  (1, 0.05825423413363113),\n",
       "  (2, 0.01023636702028662),\n",
       "  (3, 0.11423785594639867),\n",
       "  (4, 0.1539921831379118)],\n",
       " [(0, 0.19750804408845074),\n",
       "  (1, 0.06079277058944342),\n",
       "  (2, 0.3810501814198672),\n",
       "  (3, 0.08776613952214692),\n",
       "  (4, 0.27288286438009174)],\n",
       " [(0, 0.0251358290047167),\n",
       "  (1, 0.01856827273270046),\n",
       "  (2, 0.9040539733715446),\n",
       "  (3, 0.02191175592572691),\n",
       "  (4, 0.030330168965311363)],\n",
       " [(0, 0.06027253668763104),\n",
       "  (1, 0.14255765199161427),\n",
       "  (2, 0.4479821802935011),\n",
       "  (3, 0.15756027253668767),\n",
       "  (4, 0.19162735849056606)],\n",
       " [(0, 0.012925819802610634),\n",
       "  (1, 0.01037886023559376),\n",
       "  (2, 0.010315186246418338),\n",
       "  (3, 0.9552371856096784),\n",
       "  (4, 0.011142948105698823)],\n",
       " [(0, 0.05486806783212205),\n",
       "  (1, 0.1619852026451909),\n",
       "  (2, 0.014142604596346496),\n",
       "  (3, 0.7165586328815557),\n",
       "  (4, 0.05244549204478492)],\n",
       " [(0, 0.028300899769826325),\n",
       "  (1, 0.0200355722954593),\n",
       "  (2, 0.8507009834693451),\n",
       "  (3, 0.06800585896631095),\n",
       "  (4, 0.03295668549905838)],\n",
       " [(0, 0.10885796333112438),\n",
       "  (1, 0.029511817981002875),\n",
       "  (2, 0.7756129887342612),\n",
       "  (3, 0.03764082173624918),\n",
       "  (4, 0.0483764082173625)],\n",
       " [(0, 0.1384096311632544),\n",
       "  (1, 0.08580630319760756),\n",
       "  (2, 0.5268000920174835),\n",
       "  (3, 0.1857219538378959),\n",
       "  (4, 0.06326201978375892)],\n",
       " [(0, 0.1780760019881614),\n",
       "  (1, 0.6293434548822918),\n",
       "  (2, 0.013374903980841353),\n",
       "  (3, 0.0883376259545434),\n",
       "  (4, 0.09086801319416203)],\n",
       " [(0, 0.7386149285906765),\n",
       "  (1, 0.1515314829785323),\n",
       "  (2, 0.04338453247103207),\n",
       "  (3, 0.03610886553489626),\n",
       "  (4, 0.030360190424863022)],\n",
       " [(0, 0.009534706331045004),\n",
       "  (1, 0.008560047461649293),\n",
       "  (2, 0.010975506398847359),\n",
       "  (3, 0.8870667005678448),\n",
       "  (4, 0.08386303924061361)],\n",
       " [(0, 0.295269770879527),\n",
       "  (1, 0.014720374476472037),\n",
       "  (2, 0.5502586844050259),\n",
       "  (3, 0.044592264104459226),\n",
       "  (4, 0.09515890613451589)],\n",
       " [(0, 0.03920753671377113),\n",
       "  (1, 0.14491548905513993),\n",
       "  (2, 0.03269603768356886),\n",
       "  (3, 0.03796065392075367),\n",
       "  (4, 0.7452202826267664)],\n",
       " [(0, 0.7925783953575041),\n",
       "  (1, 0.03790661768620561),\n",
       "  (2, 0.10072938295266996),\n",
       "  (3, 0.045626364265559285),\n",
       "  (4, 0.023159239738061013)],\n",
       " [(0, 0.7593610093610094),\n",
       "  (1, 0.02782865282865283),\n",
       "  (2, 0.03230565730565731),\n",
       "  (3, 0.10795685795685796),\n",
       "  (4, 0.07254782254782255)],\n",
       " [(0, 0.011579318330564366),\n",
       "  (1, 0.006544832099884207),\n",
       "  (2, 0.007300005034486231),\n",
       "  (3, 0.9658661833559885),\n",
       "  (4, 0.008709661179076676)],\n",
       " [(0, 0.8380750993207741),\n",
       "  (1, 0.08048186594899398),\n",
       "  (2, 0.020825323593489684),\n",
       "  (3, 0.019415609381007306),\n",
       "  (4, 0.04120210175573497)],\n",
       " [(0, 0.030774111675126906),\n",
       "  (1, 0.006274675690919346),\n",
       "  (2, 0.9250564015792443),\n",
       "  (3, 0.023723914269599553),\n",
       "  (4, 0.014170896785109985)],\n",
       " [(0, 0.208153890324433),\n",
       "  (1, 0.0652311225954637),\n",
       "  (2, 0.3517082974447316),\n",
       "  (3, 0.3551536032156188),\n",
       "  (4, 0.01975308641975309)],\n",
       " [(0, 0.10543348662919434),\n",
       "  (1, 0.16385624254811787),\n",
       "  (2, 0.06489524782830863),\n",
       "  (3, 0.08397206608754897),\n",
       "  (4, 0.5818429569068302)],\n",
       " [(0, 0.06575446598251615),\n",
       "  (1, 0.15355378183200305),\n",
       "  (2, 0.5024071962498416),\n",
       "  (3, 0.26099075129861904),\n",
       "  (4, 0.017293804637020144)],\n",
       " [(0, 0.07679863771817795),\n",
       "  (1, 0.17462750106428268),\n",
       "  (2, 0.621796509152831),\n",
       "  (3, 0.0389953171562367),\n",
       "  (4, 0.0877820349084717)],\n",
       " [(0, 0.4626440804517243),\n",
       "  (1, 0.08171169909935135),\n",
       "  (2, 0.012693079471743899),\n",
       "  (3, 0.2111157776844463),\n",
       "  (4, 0.23183536329273416)],\n",
       " [(0, 0.49270482603815935),\n",
       "  (1, 0.1578563411896745),\n",
       "  (2, 0.1856341189674523),\n",
       "  (3, 0.13148148148148148),\n",
       "  (4, 0.03232323232323232)],\n",
       " [(0, 0.011392729134664619),\n",
       "  (1, 0.3039554531490015),\n",
       "  (2, 0.009152585765488991),\n",
       "  (3, 0.6647465437788018),\n",
       "  (4, 0.010752688172043012)],\n",
       " [(0, 0.1057587830496197),\n",
       "  (1, 0.07223630729606825),\n",
       "  (2, 0.650730411686587),\n",
       "  (3, 0.062336512535715725),\n",
       "  (4, 0.10893798543200933)],\n",
       " [(0, 0.017777777777777778),\n",
       "  (1, 0.01772005772005772),\n",
       "  (2, 0.9286002886002886),\n",
       "  (3, 0.017777777777777778),\n",
       "  (4, 0.018124098124098126)],\n",
       " [(0, 0.020720276270350273),\n",
       "  (1, 0.6085073726908952),\n",
       "  (2, 0.011785342323082827),\n",
       "  (3, 0.10519103217672532),\n",
       "  (4, 0.2537959765389464)],\n",
       " [(0, 0.4218177221590191),\n",
       "  (1, 0.01672776303037964),\n",
       "  (2, 0.3862975603589938),\n",
       "  (3, 0.05073104959339317),\n",
       "  (4, 0.1244259048582143)],\n",
       " [(0, 0.6398983101540302),\n",
       "  (1, 0.016350132097103832),\n",
       "  (2, 0.02118538457704003),\n",
       "  (3, 0.30148048452220727),\n",
       "  (4, 0.021085688649618663)],\n",
       " [(0, 0.048641260652838074),\n",
       "  (1, 0.2971538832609744),\n",
       "  (2, 0.015114970252452163),\n",
       "  (3, 0.027978774722624215),\n",
       "  (4, 0.6111111111111112)],\n",
       " [(0, 0.08425487098472881),\n",
       "  (1, 0.01550523667427301),\n",
       "  (2, 0.8584050084839975),\n",
       "  (3, 0.01731905681352759),\n",
       "  (4, 0.024515827043473173)],\n",
       " [(0, 0.279499728113105),\n",
       "  (1, 0.09068938432723098),\n",
       "  (2, 0.2624614826898677),\n",
       "  (3, 0.06851549755301795),\n",
       "  (4, 0.29883390731677845)],\n",
       " [(0, 0.7164008611220948),\n",
       "  (1, 0.01568472386977725),\n",
       "  (2, 0.018408681516629322),\n",
       "  (3, 0.23469970563683493),\n",
       "  (4, 0.01480602785466368)],\n",
       " [(0, 0.02098108747044917),\n",
       "  (1, 0.015144799054373522),\n",
       "  (2, 0.013076241134751773),\n",
       "  (3, 0.9388297872340425),\n",
       "  (4, 0.011968085106382979)],\n",
       " [(0, 0.030278884462151396),\n",
       "  (1, 0.6599380256750774),\n",
       "  (2, 0.012970340858787075),\n",
       "  (3, 0.06525011066843736),\n",
       "  (4, 0.2315626383355467)],\n",
       " [(0, 0.060961456233583775),\n",
       "  (1, 0.12638201698124732),\n",
       "  (2, 0.04208661657809541),\n",
       "  (3, 0.19174149410543034),\n",
       "  (4, 0.5788284161016432)],\n",
       " [(0, 0.6479900391319816),\n",
       "  (1, 0.06272975216411718),\n",
       "  (2, 0.24279615795090717),\n",
       "  (3, 0.01915095458318511),\n",
       "  (4, 0.027333096169809088)],\n",
       " [(0, 0.018086268086268086),\n",
       "  (1, 0.016175266175266174),\n",
       "  (2, 0.010510510510510511),\n",
       "  (3, 0.9322276822276823),\n",
       "  (4, 0.023000273000273002)],\n",
       " [(0, 0.11894538606403013),\n",
       "  (1, 0.05694915254237288),\n",
       "  (2, 0.5910357815442561),\n",
       "  (3, 0.20662900188323918),\n",
       "  (4, 0.026440677966101694)],\n",
       " [(0, 0.2046753834833305),\n",
       "  (1, 0.026716477047602875),\n",
       "  (2, 0.6595913284655006),\n",
       "  (3, 0.024622176940057736),\n",
       "  (4, 0.08439463406350824)],\n",
       " [(0, 0.1877495889123796),\n",
       "  (1, 0.16625557904627672),\n",
       "  (2, 0.08074935400516796),\n",
       "  (3, 0.42576932111815835),\n",
       "  (4, 0.13947615691801737)],\n",
       " [(0, 0.1308562197092084),\n",
       "  (1, 0.11506013283073056),\n",
       "  (2, 0.028720157960868783),\n",
       "  (3, 0.037216538024292464),\n",
       "  (4, 0.6881469514748998)],\n",
       " [(0, 0.11054155394181024),\n",
       "  (1, 0.0710997199677251),\n",
       "  (2, 0.6743556884522284),\n",
       "  (3, 0.07969054060467987),\n",
       "  (4, 0.0643124970335564)],\n",
       " [(0, 0.06492089470812876),\n",
       "  (1, 0.23010244286840031),\n",
       "  (2, 0.011214160150330364),\n",
       "  (3, 0.6738194823301207),\n",
       "  (4, 0.019943019943019943)],\n",
       " [(0, 0.042100179990341983),\n",
       "  (1, 0.8044690284911541),\n",
       "  (2, 0.021511040870977653),\n",
       "  (3, 0.0178673339479345),\n",
       "  (4, 0.11405241669959174)],\n",
       " [(0, 0.006613235229711383),\n",
       "  (1, 0.007620549205097885),\n",
       "  (2, 0.005518328734726055),\n",
       "  (3, 0.9720579862479745),\n",
       "  (4, 0.008189900582490256)],\n",
       " [(0, 0.22469725095023424),\n",
       "  (1, 0.39193847785733227),\n",
       "  (2, 0.17979315831344472),\n",
       "  (3, 0.17130734553168922),\n",
       "  (4, 0.03226376734729957)],\n",
       " [(0, 0.027628274129888768),\n",
       "  (1, 0.8183231670852769),\n",
       "  (2, 0.01477096041143404),\n",
       "  (3, 0.12725750508312403),\n",
       "  (4, 0.012020093290276283)],\n",
       " [(0, 0.02210587550901687),\n",
       "  (1, 0.01648245103742486),\n",
       "  (2, 0.017581281106586518),\n",
       "  (3, 0.8823605455368108),\n",
       "  (4, 0.06146984681016095)],\n",
       " [(0, 0.6586113300061291),\n",
       "  (1, 0.0843183609141056),\n",
       "  (2, 0.026442518168286494),\n",
       "  (3, 0.08633219507924),\n",
       "  (4, 0.1442955958322389)],\n",
       " [(0, 0.017285930678071815),\n",
       "  (1, 0.014612848614452464),\n",
       "  (2, 0.014612848614452464),\n",
       "  (3, 0.9397665508331106),\n",
       "  (4, 0.013721821259912679)],\n",
       " [(0, 0.06141878555671659),\n",
       "  (1, 0.13003599210495762),\n",
       "  (2, 0.7286659700452804),\n",
       "  (3, 0.021363055845814466),\n",
       "  (4, 0.05851619644723093)],\n",
       " [(0, 0.6302315083510672),\n",
       "  (1, 0.03192106790481718),\n",
       "  (2, 0.2756819500870575),\n",
       "  (3, 0.028954665634874574),\n",
       "  (4, 0.03321080802218353)],\n",
       " [(0, 0.01702864337425244),\n",
       "  (1, 0.830028328611898),\n",
       "  (2, 0.008246773685867171),\n",
       "  (3, 0.015517784073024865),\n",
       "  (4, 0.12917847025495752)],\n",
       " [(0, 0.8365808397123637),\n",
       "  (1, 0.03119925771282765),\n",
       "  (2, 0.07156112270934818),\n",
       "  (3, 0.04256553003943401),\n",
       "  (4, 0.018093249826026444)],\n",
       " [(0, 0.07400881057268723),\n",
       "  (1, 0.22094958394517866),\n",
       "  (2, 0.013069016152716594),\n",
       "  (3, 0.05712187958883994),\n",
       "  (4, 0.6348507097405776)],\n",
       " [(0, 0.036564625850340135),\n",
       "  (1, 0.8060009718172984),\n",
       "  (2, 0.036321671525753156),\n",
       "  (3, 0.015913508260447035),\n",
       "  (4, 0.10519922254616132)],\n",
       " [(0, 0.014774272018315805),\n",
       "  (1, 0.7149602919081348),\n",
       "  (2, 0.007798526150103742),\n",
       "  (3, 0.013808399513486442),\n",
       "  (4, 0.24865851040995923)],\n",
       " [(0, 0.1515798284159866),\n",
       "  (1, 0.4215107763130362),\n",
       "  (2, 0.11358024691358025),\n",
       "  (3, 0.1572713956894748),\n",
       "  (4, 0.15605775266792216)],\n",
       " [(0, 0.14124293785310735),\n",
       "  (1, 0.19152823453354695),\n",
       "  (2, 0.5869803524749135),\n",
       "  (3, 0.05194367147314276),\n",
       "  (4, 0.028304803665289372)],\n",
       " [(0, 0.11129652806749178),\n",
       "  (1, 0.04913549344087517),\n",
       "  (2, 0.5568534742502202),\n",
       "  (3, 0.013720854772168917),\n",
       "  (4, 0.26899364946924403)],\n",
       " [(0, 0.2008607198748044),\n",
       "  (1, 0.1385758998435055),\n",
       "  (2, 0.017136150234741784),\n",
       "  (3, 0.36932707355242567),\n",
       "  (4, 0.2741001564945227)],\n",
       " [(0, 0.22266016016016016),\n",
       "  (1, 0.36261261261261263),\n",
       "  (2, 0.10754504504504504),\n",
       "  (3, 0.10497997997997997),\n",
       "  (4, 0.2022022022022022)],\n",
       " [(0, 0.6408554572271387),\n",
       "  (1, 0.0229228121927237),\n",
       "  (2, 0.03035889872173058),\n",
       "  (3, 0.26431907571288105),\n",
       "  (4, 0.04154375614552606)],\n",
       " [(0, 0.028784648187633266),\n",
       "  (1, 0.03755034352049278),\n",
       "  (2, 0.020137408197109692),\n",
       "  (3, 0.8957592987443734),\n",
       "  (4, 0.017768301350390907)],\n",
       " [(0, 0.031934939998016464),\n",
       "  (1, 0.6713940956725842),\n",
       "  (2, 0.005388607887864062),\n",
       "  (3, 0.08952362061555753),\n",
       "  (4, 0.20175873582597773)],\n",
       " [(0, 0.04113293481354968),\n",
       "  (1, 0.7370718284467218),\n",
       "  (2, 0.009915551760129046),\n",
       "  (3, 0.18222791536198882),\n",
       "  (4, 0.029651769617610783)],\n",
       " [(0, 0.025170351828674734),\n",
       "  (1, 0.01367450053307375),\n",
       "  (2, 0.029017753673573447),\n",
       "  (3, 0.0196078431372549),\n",
       "  (4, 0.9125295508274232)],\n",
       " [(0, 0.03470472991883571),\n",
       "  (1, 0.011914757506697054),\n",
       "  (2, 0.1268241973531646),\n",
       "  (3, 0.013634001039542602),\n",
       "  (4, 0.81292231418176)],\n",
       " [(0, 0.08319649698960044),\n",
       "  (1, 0.38076993249407043),\n",
       "  (2, 0.018883415435139574),\n",
       "  (3, 0.4992702061667579),\n",
       "  (4, 0.017879948914431672)],\n",
       " [(0, 0.029006352630948102),\n",
       "  (1, 0.023732470334412083),\n",
       "  (2, 0.30960086299892126),\n",
       "  (3, 0.06528427024651405),\n",
       "  (4, 0.5723760437892045)],\n",
       " [(0, 0.08641975308641975),\n",
       "  (1, 0.03993752788933512),\n",
       "  (2, 0.014799940502751747),\n",
       "  (3, 0.0722147850661907),\n",
       "  (4, 0.7866279934553027)],\n",
       " [(0, 0.8509681881051175),\n",
       "  (1, 0.015790686952512678),\n",
       "  (2, 0.011468418626094974),\n",
       "  (3, 0.10909405255878285),\n",
       "  (4, 0.012678653757491932)],\n",
       " [(0, 0.06280817655225238),\n",
       "  (1, 0.030350902388817082),\n",
       "  (2, 0.018909473885777205),\n",
       "  (3, 0.8351764086361243),\n",
       "  (4, 0.05275503853702906)],\n",
       " [(0, 0.0178056301601512),\n",
       "  (1, 0.01034517059584204),\n",
       "  (2, 0.9064458370635632),\n",
       "  (3, 0.024072416194170897),\n",
       "  (4, 0.04133094598627276)],\n",
       " [(0, 0.01929806326438693),\n",
       "  (1, 0.012645948099643142),\n",
       "  (2, 0.8892007067872363),\n",
       "  (3, 0.007760801025534421),\n",
       "  (4, 0.07109448082319925)],\n",
       " [(0, 0.061381381381381384),\n",
       "  (1, 0.02906906906906907),\n",
       "  (2, 0.8366366366366367),\n",
       "  (3, 0.015435435435435435),\n",
       "  (4, 0.05747747747747748)],\n",
       " [(0, 0.08154173929885726),\n",
       "  (1, 0.0249209116146943),\n",
       "  (2, 0.778229711408096),\n",
       "  (3, 0.09735941635999741),\n",
       "  (4, 0.01794822131835496)],\n",
       " [(0, 0.008210833002251358),\n",
       "  (1, 0.0069747936255683575),\n",
       "  (2, 0.0054297444047146076),\n",
       "  (3, 0.9706882090672317),\n",
       "  (4, 0.008696419900233964)],\n",
       " [(0, 0.019054424199118732),\n",
       "  (1, 0.017506252232940337),\n",
       "  (2, 0.9218768607836132),\n",
       "  (3, 0.01833988329165178),\n",
       "  (4, 0.023222579492675956)],\n",
       " [(0, 0.011978097193702944),\n",
       "  (1, 0.02931781884553959),\n",
       "  (2, 0.009012092174309835),\n",
       "  (3, 0.9387405886379193),\n",
       "  (4, 0.010951403148528407)],\n",
       " [(0, 0.5718766335598537),\n",
       "  (1, 0.09513852587558809),\n",
       "  (2, 0.13283382703142246),\n",
       "  (3, 0.03862461520590115),\n",
       "  (4, 0.16152639832723473)],\n",
       " [(0, 0.026004728132387706),\n",
       "  (1, 0.011495851295600983),\n",
       "  (2, 0.9246280072312613),\n",
       "  (3, 0.017197422704306308),\n",
       "  (4, 0.020673990636443703)],\n",
       " [(0, 0.19903884661593912),\n",
       "  (1, 0.4574823121078628),\n",
       "  (2, 0.09540337293641259),\n",
       "  (3, 0.06109553686646198),\n",
       "  (4, 0.18697993147332354)],\n",
       " [(0, 0.029865125240847785),\n",
       "  (1, 0.101530721472918),\n",
       "  (2, 0.01123956326268465),\n",
       "  (3, 0.8471419396274887),\n",
       "  (4, 0.0102226503960608)],\n",
       " [(0, 0.10577421985027055),\n",
       "  (1, 0.10266103328144689),\n",
       "  (2, 0.13490475131569193),\n",
       "  (3, 0.15640056333852198),\n",
       "  (4, 0.5002594322140687)],\n",
       " [(0, 0.2698412698412698),\n",
       "  (1, 0.14517195767195767),\n",
       "  (2, 0.038111772486772486),\n",
       "  (3, 0.517443783068783),\n",
       "  (4, 0.02943121693121693)],\n",
       " [(0, 0.07497879558948262),\n",
       "  (1, 0.15394402035623408),\n",
       "  (2, 0.19355385920271417),\n",
       "  (3, 0.03435114503816794),\n",
       "  (4, 0.5431721798134012)],\n",
       " [(0, 0.02875658161198866),\n",
       "  (1, 0.029566626164439044),\n",
       "  (2, 0.8079519373565546),\n",
       "  (3, 0.023086269744835967),\n",
       "  (4, 0.11063858512218172)],\n",
       " [(0, 0.011006617903169627),\n",
       "  (1, 0.01400208986415883),\n",
       "  (2, 0.009822361546499478),\n",
       "  (3, 0.9442006269592477),\n",
       "  (4, 0.020968303726924417)],\n",
       " [(0, 0.6091533474549072),\n",
       "  (1, 0.030682328775916298),\n",
       "  (2, 0.30791450028885037),\n",
       "  (3, 0.023428974902111818),\n",
       "  (4, 0.028820848578214264)],\n",
       " [(0, 0.10123111938754396),\n",
       "  (1, 0.13335402441547692),\n",
       "  (2, 0.20789364783778191),\n",
       "  (3, 0.11954272708462653),\n",
       "  (4, 0.4379784812745707)],\n",
       " [(0, 0.02851033499643621),\n",
       "  (1, 0.013304822998336897),\n",
       "  (2, 0.8324225865209471),\n",
       "  (3, 0.018531717747683536),\n",
       "  (4, 0.10723053773659619)],\n",
       " [(0, 0.009673518742442563),\n",
       "  (1, 0.010278113663845224),\n",
       "  (2, 0.009942227596399301),\n",
       "  (3, 0.9588203681311299),\n",
       "  (4, 0.01128577186618299)],\n",
       " [(0, 0.016875522138680033),\n",
       "  (1, 0.05338345864661654),\n",
       "  (2, 0.011695906432748537),\n",
       "  (3, 0.8957393483709273),\n",
       "  (4, 0.02230576441102757)],\n",
       " [(0, 0.01115522783134808),\n",
       "  (1, 0.010020797882397429),\n",
       "  (2, 0.011533371147664964),\n",
       "  (3, 0.9538665154093401),\n",
       "  (4, 0.013424087729249386)],\n",
       " [(0, 0.049303849303849304),\n",
       "  (1, 0.042205842205842205),\n",
       "  (2, 0.06917826917826918),\n",
       "  (3, 0.026153426153426154),\n",
       "  (4, 0.8131586131586132)],\n",
       " [(0, 0.03605888789416095),\n",
       "  (1, 0.5209390231771611),\n",
       "  (2, 0.17084452402267983),\n",
       "  (3, 0.25733611857157074),\n",
       "  (4, 0.014821446334427536)],\n",
       " [(0, 0.7580054894784996),\n",
       "  (1, 0.02475348175256684),\n",
       "  (2, 0.012757954660973875),\n",
       "  (3, 0.15350208396868964),\n",
       "  (4, 0.0509809901392701)],\n",
       " [(0, 0.1338930274723109),\n",
       "  (1, 0.08214102715699026),\n",
       "  (2, 0.015411296361988097),\n",
       "  (3, 0.0448543612786252),\n",
       "  (4, 0.7237002877300855)],\n",
       " [(0, 0.7790662003803914),\n",
       "  (1, 0.025584391680471195),\n",
       "  (2, 0.09221424627277747),\n",
       "  (3, 0.05478863733971409),\n",
       "  (4, 0.04834652432664581)],\n",
       " [(0, 0.019927052753313763),\n",
       "  (1, 0.016012810248198558),\n",
       "  (2, 0.5793968508139845),\n",
       "  (3, 0.3659816742282715),\n",
       "  (4, 0.018681611956231654)],\n",
       " [(0, 0.11966666666666667),\n",
       "  (1, 0.019925925925925927),\n",
       "  (2, 0.69),\n",
       "  (3, 0.10188888888888889),\n",
       "  (4, 0.06851851851851852)],\n",
       " [(0, 0.015065913370998118),\n",
       "  (1, 0.015363266924373082),\n",
       "  (2, 0.017742095351372784),\n",
       "  (3, 0.9322033898305085),\n",
       "  (4, 0.01962533452274755)],\n",
       " [(0, 0.10763371419109125),\n",
       "  (1, 0.12827730860517747),\n",
       "  (2, 0.19567257272175306),\n",
       "  (3, 0.5529613070596677),\n",
       "  (4, 0.015455097422310536)],\n",
       " [(0, 0.8331459972524041),\n",
       "  (1, 0.017484700886724117),\n",
       "  (2, 0.038466341950793057),\n",
       "  (3, 0.07968027975521419),\n",
       "  (4, 0.031222680154864492)],\n",
       " [(0, 0.019500304692260818),\n",
       "  (1, 0.018823210779335093),\n",
       "  (2, 0.9277879341864717),\n",
       "  (3, 0.009885571128715553),\n",
       "  (4, 0.024002979213216872)],\n",
       " [(0, 0.01615831517792302),\n",
       "  (1, 0.9032921810699589),\n",
       "  (2, 0.014342774146695715),\n",
       "  (3, 0.045449043815056885),\n",
       "  (4, 0.020757685790365528)],\n",
       " [(0, 0.009900327781122483),\n",
       "  (1, 0.022677102147300822),\n",
       "  (2, 0.010970633487189779),\n",
       "  (3, 0.9445447856043883),\n",
       "  (4, 0.011907150979998662)],\n",
       " [(0, 0.7527258566978193),\n",
       "  (1, 0.026739356178608516),\n",
       "  (2, 0.01791277258566978),\n",
       "  (3, 0.09055901696088611),\n",
       "  (4, 0.11206299757701627)],\n",
       " [(0, 0.027283511269276393),\n",
       "  (1, 0.651789244760775),\n",
       "  (2, 0.02224199288256228),\n",
       "  (3, 0.015816528272044286),\n",
       "  (4, 0.28286872281534203)],\n",
       " [(0, 0.04124763154059175),\n",
       "  (1, 0.007481902540931837),\n",
       "  (2, 0.01544964290919691),\n",
       "  (3, 0.14716027789923725),\n",
       "  (4, 0.7886605451100422)],\n",
       " [(0, 0.06865313250382013),\n",
       "  (1, 0.12153459943243833),\n",
       "  (2, 0.2743396638288583),\n",
       "  (3, 0.04016590264134468),\n",
       "  (4, 0.4953067015935385)],\n",
       " [(0, 0.7746074879227053),\n",
       "  (1, 0.026947463768115944),\n",
       "  (2, 0.07072765700483091),\n",
       "  (3, 0.0665006038647343),\n",
       "  (4, 0.06121678743961353)],\n",
       " [(0, 0.35928961748633886),\n",
       "  (1, 0.015892531876138438),\n",
       "  (2, 0.481648451730419),\n",
       "  (3, 0.07682149362477232),\n",
       "  (4, 0.06634790528233153)],\n",
       " [(0, 0.03409260086348072),\n",
       "  (1, 0.00833705523299092),\n",
       "  (2, 0.4545680115130763),\n",
       "  (3, 0.44136767406084065),\n",
       "  (4, 0.06163465832961143)],\n",
       " [(0, 0.07211631041787021),\n",
       "  (1, 0.04270171384556133),\n",
       "  (2, 0.12011361448103215),\n",
       "  (3, 0.7330059695744271),\n",
       "  (4, 0.032062391681109186)],\n",
       " [(0, 0.6981899192425509),\n",
       "  (1, 0.24917850181008078),\n",
       "  (2, 0.02160957950431635),\n",
       "  (3, 0.009468114731272627),\n",
       "  (4, 0.021553884711779454)],\n",
       " [(0, 0.21922992299229924),\n",
       "  (1, 0.05482948294829483),\n",
       "  (2, 0.0686028602860286),\n",
       "  (3, 0.05478547854785479),\n",
       "  (4, 0.6025522552255226)],\n",
       " [(0, 0.12517062517062516),\n",
       "  (1, 0.13636363636363635),\n",
       "  (2, 0.02265902265902266),\n",
       "  (3, 0.1532896532896533),\n",
       "  (4, 0.5625170625170626)],\n",
       " [(0, 0.04959649404093849),\n",
       "  (1, 0.028058361391694726),\n",
       "  (2, 0.015178237400459623),\n",
       "  (3, 0.06028539361872695),\n",
       "  (4, 0.8468815135481802)],\n",
       " [(0, 0.03170559094125973),\n",
       "  (1, 0.01429582448690729),\n",
       "  (2, 0.012314225053078557),\n",
       "  (3, 0.1789808917197452),\n",
       "  (4, 0.7627034677990092)],\n",
       " [(0, 0.04817980373535929),\n",
       "  (1, 0.3858816081038303),\n",
       "  (2, 0.032288698955365625),\n",
       "  (3, 0.06736308958531181),\n",
       "  (4, 0.466286799620133)],\n",
       " [(0, 0.010752688172043012),\n",
       "  (1, 0.2768817204301075),\n",
       "  (2, 0.017025089605734768),\n",
       "  (3, 0.022572111281788702),\n",
       "  (4, 0.672768390510326)],\n",
       " [(0, 0.03823311061618412),\n",
       "  (1, 0.03823311061618412),\n",
       "  (2, 0.7985028458302401),\n",
       "  (3, 0.023137837169017572),\n",
       "  (4, 0.10189309576837417)],\n",
       " [(0, 0.1097119341563786),\n",
       "  (1, 0.17666666666666667),\n",
       "  (2, 0.1608641975308642),\n",
       "  (3, 0.0834156378600823),\n",
       "  (4, 0.4693415637860082)],\n",
       " [(0, 0.11543494339193265),\n",
       "  (1, 0.023325937304431933),\n",
       "  (2, 0.761620299254708),\n",
       "  (3, 0.024236217784604885),\n",
       "  (4, 0.07538260226432271)],\n",
       " [(0, 0.03535735281767028),\n",
       "  (1, 0.022591752750482908),\n",
       "  (2, 0.044175694969345765),\n",
       "  (3, 0.14260519022423784),\n",
       "  (4, 0.7552700092382632)],\n",
       " [(0, 0.01755634638196916),\n",
       "  (1, 0.018821668643732706),\n",
       "  (2, 0.9235666271253461),\n",
       "  (3, 0.024238829576907872),\n",
       "  (4, 0.01581652827204429)],\n",
       " [(0, 0.808369644153958),\n",
       "  (1, 0.03249818445896878),\n",
       "  (2, 0.02723311546840959),\n",
       "  (3, 0.06481481481481483),\n",
       "  (4, 0.06708424110384896)],\n",
       " [(0, 0.027521682715290582),\n",
       "  (1, 0.017277880215802774),\n",
       "  (2, 0.9001570716383255),\n",
       "  (3, 0.022604657515536433),\n",
       "  (4, 0.03243870791504473)],\n",
       " [(0, 0.3106019276232042),\n",
       "  (1, 0.012426501788203917),\n",
       "  (2, 0.01157786264169243),\n",
       "  (3, 0.6516942474389283),\n",
       "  (4, 0.013699460507971146)],\n",
       " [(0, 0.07166864373270067),\n",
       "  (1, 0.04646105179913009),\n",
       "  (2, 0.8266113088177145),\n",
       "  (3, 0.019968366943455912),\n",
       "  (4, 0.03529062870699881)],\n",
       " [(0, 0.2819727438185289),\n",
       "  (1, 0.02596883739512585),\n",
       "  (2, 0.01207439960935766),\n",
       "  (3, 0.6671993607670795),\n",
       "  (4, 0.01278465840990811)],\n",
       " [(0, 0.015951868410884803),\n",
       "  (1, 0.01413037478611249),\n",
       "  (2, 0.008445106805762544),\n",
       "  (3, 0.9383452006402826),\n",
       "  (4, 0.023127449356957555)],\n",
       " [(0, 0.7506180469715699),\n",
       "  (1, 0.011468204916907018),\n",
       "  (2, 0.16790276060980636),\n",
       "  (3, 0.0434693036670787),\n",
       "  (4, 0.026541683834638098)],\n",
       " [(0, 0.04589371980676329),\n",
       "  (1, 0.017184265010351966),\n",
       "  (2, 0.012905452035886819),\n",
       "  (3, 0.053278122843340235),\n",
       "  (4, 0.8707384403036577)],\n",
       " [(0, 0.17264088813623107),\n",
       "  (1, 0.03725656223539373),\n",
       "  (2, 0.12075453946749459),\n",
       "  (3, 0.07065575312823408),\n",
       "  (4, 0.5986922570326465)],\n",
       " [(0, 0.07099985360854925),\n",
       "  (1, 0.1840872493046406),\n",
       "  (2, 0.03367003367003367),\n",
       "  (3, 0.21431708388230128),\n",
       "  (4, 0.4969257795344752)],\n",
       " [(0, 0.03199308257674016),\n",
       "  (1, 0.1756376999567661),\n",
       "  (2, 0.6010592304366623),\n",
       "  (3, 0.04755728491137051),\n",
       "  (4, 0.14375270211846086)],\n",
       " [(0, 0.19621122889674125),\n",
       "  (1, 0.017520612485276796),\n",
       "  (2, 0.7446014919513153),\n",
       "  (3, 0.013594424813506086),\n",
       "  (4, 0.02807224185316058)],\n",
       " [(0, 0.01003970929796958),\n",
       "  (1, 0.017344721660298195),\n",
       "  (2, 0.009852401288679103),\n",
       "  (3, 0.950550685547314),\n",
       "  (4, 0.012212482205739117)],\n",
       " [(0, 0.6318888204285616),\n",
       "  (1, 0.1252823988498665),\n",
       "  (2, 0.12884233586636545),\n",
       "  (3, 0.04422537139727528),\n",
       "  (4, 0.06976107345793113)],\n",
       " [(0, 0.013888888888888888),\n",
       "  (1, 0.01594650205761317),\n",
       "  (2, 0.009516460905349794),\n",
       "  (3, 0.9388717421124828),\n",
       "  (4, 0.021776406035665295)],\n",
       " [(0, 0.03210554411652559),\n",
       "  (1, 0.611149241211012),\n",
       "  (2, 0.022420498741706704),\n",
       "  (3, 0.29047510104476476),\n",
       "  (4, 0.043849614885991)],\n",
       " [(0, 0.28916757542711746),\n",
       "  (1, 0.16569732218587183),\n",
       "  (2, 0.2755967526959894),\n",
       "  (3, 0.16042651157154977),\n",
       "  (4, 0.10911183811947173)],\n",
       " [(0, 0.15830424665503198),\n",
       "  (1, 0.023414776032577078),\n",
       "  (2, 0.027050610820244327),\n",
       "  (3, 0.10027632344386271),\n",
       "  (4, 0.6909540430482839)],\n",
       " [(0, 0.08565970296419548),\n",
       "  (1, 0.07536821347137487),\n",
       "  (2, 0.030011708880261292),\n",
       "  (3, 0.13754853022739877),\n",
       "  (4, 0.6714118444567696)],\n",
       " [(0, 0.16106133402588407),\n",
       "  (1, 0.43072328269055965),\n",
       "  (2, 0.04735315759858027),\n",
       "  (3, 0.10267064883348483),\n",
       "  (4, 0.25819157685149113)],\n",
       " [(0, 0.1533847472150814),\n",
       "  (1, 0.07036084928115777),\n",
       "  (2, 0.023612301247262688),\n",
       "  (3, 0.13034371132057507),\n",
       "  (4, 0.6222983909359231)],\n",
       " [(0, 0.2610350076103501),\n",
       "  (1, 0.5028919330289193),\n",
       "  (2, 0.029071537290715373),\n",
       "  (3, 0.1537290715372907),\n",
       "  (4, 0.0532724505327245)],\n",
       " [(0, 0.03245942571785269),\n",
       "  (1, 0.06302122347066168),\n",
       "  (2, 0.7269413233458177),\n",
       "  (3, 0.04229712858926342),\n",
       "  (4, 0.1352808988764045)],\n",
       " [(0, 0.7487427963146497),\n",
       "  (1, 0.03758763719120508),\n",
       "  (2, 0.16529016628124657),\n",
       "  (3, 0.02327203318283596),\n",
       "  (4, 0.025107367030062767)],\n",
       " [(0, 0.015897594714565914),\n",
       "  (1, 0.08994872853652662),\n",
       "  (2, 0.016310519252606587),\n",
       "  (3, 0.8584012938302192),\n",
       "  (4, 0.01944186366608169)],\n",
       " [(0, 0.7267283291829717),\n",
       "  (1, 0.013992647930748251),\n",
       "  (2, 0.011739594450373533),\n",
       "  (3, 0.22453456658365945),\n",
       "  (4, 0.023004861852247125)],\n",
       " [(0, 0.07957803004242632),\n",
       "  (1, 0.031361082444673775),\n",
       "  (2, 0.5525742460726981),\n",
       "  (3, 0.02620112372434354),\n",
       "  (4, 0.31028551771585827)],\n",
       " [(0, 0.8300653594771242),\n",
       "  (1, 0.010415348935273034),\n",
       "  (2, 0.011047860004216741),\n",
       "  (3, 0.12848408180476492),\n",
       "  (4, 0.019987349778621127)],\n",
       " [(0, 0.013625565864371835),\n",
       "  (1, 0.8957913137017616),\n",
       "  (2, 0.009457218412442295),\n",
       "  (3, 0.0656178566626328),\n",
       "  (4, 0.01550804535879163)],\n",
       " [(0, 0.6860537836147592),\n",
       "  (1, 0.031061079841567647),\n",
       "  (2, 0.05904732124244319),\n",
       "  (3, 0.043777360850531584),\n",
       "  (4, 0.18006045445069835)],\n",
       " [(0, 0.011611197332873486),\n",
       "  (1, 0.006782778639995403),\n",
       "  (2, 0.012703339656262577),\n",
       "  (3, 0.919066505719377),\n",
       "  (4, 0.04983617865149164)],\n",
       " [(0, 0.23967280163599183),\n",
       "  (1, 0.2619534521375012),\n",
       "  (2, 0.029954231181225047),\n",
       "  (3, 0.16644269159606584),\n",
       "  (4, 0.3019768234492161)],\n",
       " [(0, 0.2738520104687128),\n",
       "  (1, 0.022008089459909588),\n",
       "  (2, 0.014235863272265841),\n",
       "  (3, 0.6754302482353874),\n",
       "  (4, 0.014473788563724324)],\n",
       " [(0, 0.023209409787859698),\n",
       "  (1, 0.04043268220961983),\n",
       "  (2, 0.6592102499474901),\n",
       "  (3, 0.1439823566477631),\n",
       "  (4, 0.1331653014072674)],\n",
       " [(0, 0.5023543990086741),\n",
       "  (1, 0.02643535729037588),\n",
       "  (2, 0.020817843866171002),\n",
       "  (3, 0.38166047087980176),\n",
       "  (4, 0.06873192895497728)],\n",
       " [(0, 0.43920145190562615),\n",
       "  (1, 0.04169187336156483),\n",
       "  (2, 0.2571587013510788),\n",
       "  (3, 0.03559185319620891),\n",
       "  (4, 0.2263561201855213)],\n",
       " [(0, 0.07714898904326217),\n",
       "  (1, 0.09469483037764977),\n",
       "  (2, 0.019014270115591703),\n",
       "  (3, 0.7992770812154072),\n",
       "  (4, 0.00986482924808916)],\n",
       " [(0, 0.15268557634278818),\n",
       "  (1, 0.15188090927378797),\n",
       "  (2, 0.4807885737276202),\n",
       "  (3, 0.05485147187018038),\n",
       "  (4, 0.15979346878562328)],\n",
       " [(0, 0.011673428713787458),\n",
       "  (1, 0.01786604028756495),\n",
       "  (2, 0.009395686525731368),\n",
       "  (3, 0.9469001352409424),\n",
       "  (4, 0.014164709231973806)],\n",
       " [(0, 0.08416983962155301),\n",
       "  (1, 0.5781123802930657),\n",
       "  (2, 0.06432444905965155),\n",
       "  (3, 0.06374754817122419),\n",
       "  (4, 0.2096457828545056)],\n",
       " [(0, 0.8794931471424877),\n",
       "  (1, 0.024135850357727782),\n",
       "  (2, 0.05167657960520645),\n",
       "  (3, 0.034091888630290494),\n",
       "  (4, 0.01060253426428756)],\n",
       " [(0, 0.24371822803195353),\n",
       "  (1, 0.398160251755023),\n",
       "  (2, 0.016557734204793027),\n",
       "  (3, 0.07358992979908012),\n",
       "  (4, 0.2679738562091503)],\n",
       " [(0, 0.8279539581865164),\n",
       "  (1, 0.026591496358938222),\n",
       "  (2, 0.025510923185341792),\n",
       "  (3, 0.05233732675593141),\n",
       "  (4, 0.06760629551327227)],\n",
       " [(0, 0.028080367949648997),\n",
       "  (1, 0.532639393205842),\n",
       "  (2, 0.03913499556201081),\n",
       "  (3, 0.04397643831195029),\n",
       "  (4, 0.35616880497054787)],\n",
       " [(0, 0.0759662216615859),\n",
       "  (1, 0.016643890816076246),\n",
       "  (2, 0.5200252286345002),\n",
       "  (3, 0.2999404323907635),\n",
       "  (4, 0.08742422649707418)],\n",
       " [(0, 0.433116413593637),\n",
       "  (1, 0.19908411665461556),\n",
       "  (2, 0.01524463726199084),\n",
       "  (3, 0.0675463967221017),\n",
       "  (4, 0.28500843576765483)],\n",
       " [(0, 0.007769071060210301),\n",
       "  (1, 0.007769071060210301),\n",
       "  (2, 0.007032348804500703),\n",
       "  (3, 0.9701292612685017),\n",
       "  (4, 0.007300247806576921)],\n",
       " [(0, 0.8488639900404606),\n",
       "  (1, 0.016184251478369126),\n",
       "  (2, 0.08141923436041083),\n",
       "  (3, 0.03398692810457516),\n",
       "  (4, 0.01954559601618425)],\n",
       " [(0, 0.6438080084858128),\n",
       "  (1, 0.021957040572792363),\n",
       "  (2, 0.016016971625563512),\n",
       "  (3, 0.15645717316361707),\n",
       "  (4, 0.16176080615221428)],\n",
       " [(0, 0.048180924287118974),\n",
       "  (1, 0.08725008194034743),\n",
       "  (2, 0.6841363487381187),\n",
       "  (3, 0.03477548344804982),\n",
       "  (4, 0.14565716158636513)],\n",
       " [(0, 0.15981912144702845),\n",
       "  (1, 0.02997416020671835),\n",
       "  (2, 0.04354005167958657),\n",
       "  (3, 0.717248062015504),\n",
       "  (4, 0.049418604651162795)],\n",
       " [(0, 0.6841670794122503),\n",
       "  (1, 0.024847284134059764),\n",
       "  (2, 0.052336140003301966),\n",
       "  (3, 0.05101535413571075),\n",
       "  (4, 0.18763414231467723)],\n",
       " [(0, 0.01269482151835093),\n",
       "  (1, 0.010390480978716274),\n",
       "  (2, 0.010139098374392492),\n",
       "  (3, 0.9500586559410089),\n",
       "  (4, 0.016716943187531422)],\n",
       " [(0, 0.24202736517063297),\n",
       "  (1, 0.6067188415056167),\n",
       "  (2, 0.09428738753127826),\n",
       "  (3, 0.008039184368844167),\n",
       "  (4, 0.04892722142362774)],\n",
       " [(0, 0.008274231678486997),\n",
       "  (1, 0.010820149118021458),\n",
       "  (2, 0.006364793598836152),\n",
       "  (3, 0.9660847426804874),\n",
       "  (4, 0.00845608292416803)],\n",
       " [(0, 0.06742151312403499),\n",
       "  (1, 0.023903471149997142),\n",
       "  (2, 0.7917881855092355),\n",
       "  (3, 0.04866472236518557),\n",
       "  (4, 0.06822210785154686)],\n",
       " [(0, 0.20158014799977744),\n",
       "  (1, 0.1868914482835364),\n",
       "  (2, 0.27346575418683583),\n",
       "  (3, 0.04367662605018639),\n",
       "  (4, 0.2943860234796639)],\n",
       " [(0, 0.07447473436222896),\n",
       "  (1, 0.019135535362277033),\n",
       "  (2, 0.20678878792249628),\n",
       "  (3, 0.016635415164190585),\n",
       "  (4, 0.6829655271888072)],\n",
       " [(0, 0.7892447607750099),\n",
       "  (1, 0.02402135231316726),\n",
       "  (2, 0.14269474100434953),\n",
       "  (3, 0.02056148675365757),\n",
       "  (4, 0.023477659153815736)],\n",
       " [(0, 0.3310950413223141),\n",
       "  (1, 0.01899678604224059),\n",
       "  (2, 0.4213154269972452),\n",
       "  (3, 0.030016069788797065),\n",
       "  (4, 0.19857667584940314)],\n",
       " [(0, 0.014974747474747474),\n",
       "  (1, 0.846919191919192),\n",
       "  (2, 0.004065656565656566),\n",
       "  (3, 0.10055555555555555),\n",
       "  (4, 0.033484848484848485)],\n",
       " [(0, 0.6750109793588054),\n",
       "  (1, 0.1397550383057629),\n",
       "  (2, 0.07436685697555262),\n",
       "  (3, 0.08105206655931292),\n",
       "  (4, 0.02981505880056604)],\n",
       " [(0, 0.049492499775442386),\n",
       "  (1, 0.7117578370609899),\n",
       "  (2, 0.14201023982753977),\n",
       "  (3, 0.040240725770232647),\n",
       "  (4, 0.05649869756579539)],\n",
       " [(0, 0.030595251926726307),\n",
       "  (1, 0.011424809263777546),\n",
       "  (2, 0.8789357499709539),\n",
       "  (3, 0.030595251926726307),\n",
       "  (4, 0.04844893691181596)],\n",
       " [(0, 0.02749308605823979),\n",
       "  (1, 0.07754460170272762),\n",
       "  (2, 0.057046797895992624),\n",
       "  (3, 0.45382571444064856),\n",
       "  (4, 0.3840897999023914)],\n",
       " [(0, 0.050077564475470245),\n",
       "  (1, 0.9000872600349041),\n",
       "  (2, 0.008919914679076985),\n",
       "  (3, 0.02695365522590654),\n",
       "  (4, 0.013961605584642236)],\n",
       " [(0, 0.25922470978441126),\n",
       "  (1, 0.0640547263681592),\n",
       "  (2, 0.09017412935323384),\n",
       "  (3, 0.3517827529021559),\n",
       "  (4, 0.2347636815920398)],\n",
       " [(0, 0.07565011820330969),\n",
       "  (1, 0.02251491613193741),\n",
       "  (2, 0.01677361251829337),\n",
       "  (3, 0.8318135764944276),\n",
       "  (4, 0.053247776652031974)],\n",
       " [(0, 0.15843023255813954),\n",
       "  (1, 0.03762919896640827),\n",
       "  (2, 0.29941860465116277),\n",
       "  (3, 0.034614556416882),\n",
       "  (4, 0.4699074074074074)],\n",
       " [(0, 0.057207442183968005),\n",
       "  (1, 0.046687532603025564),\n",
       "  (2, 0.07172665623369849),\n",
       "  (3, 0.019996522343940185),\n",
       "  (4, 0.8043818466353677)],\n",
       " [(0, 0.6598032157427407),\n",
       "  (1, 0.03326133909287258),\n",
       "  (2, 0.03369330453563715),\n",
       "  (3, 0.24670026397888173),\n",
       "  (4, 0.026541876649868013)],\n",
       " [(0, 0.12804183493838667),\n",
       "  (1, 0.11271616444030237),\n",
       "  (2, 0.3300196748472611),\n",
       "  (3, 0.10272341306824065),\n",
       "  (4, 0.3264989127058093)],\n",
       " [(0, 0.09265074380689568),\n",
       "  (1, 0.12601896786881087),\n",
       "  (2, 0.6565800742220953),\n",
       "  (3, 0.11383893170932852),\n",
       "  (4, 0.010911282392869604)],\n",
       " [(0, 0.035940803382663845),\n",
       "  (1, 0.012606687025291677),\n",
       "  (2, 0.02458695481951296),\n",
       "  (3, 0.6461514368491112),\n",
       "  (4, 0.28071411792342027)],\n",
       " [(0, 0.3302249350062168),\n",
       "  (1, 0.06572849553520968),\n",
       "  (2, 0.15095512603142308),\n",
       "  (3, 0.18243472363513055),\n",
       "  (4, 0.2706567197920199)],\n",
       " [(0, 0.07218802462227196),\n",
       "  (1, 0.013927749797923272),\n",
       "  (2, 0.6705216688428776),\n",
       "  (3, 0.027731144686936517),\n",
       "  (4, 0.21563141204999067)],\n",
       " [(0, 0.0524852950681921),\n",
       "  (1, 0.055717148212785214),\n",
       "  (2, 0.7650765949195268),\n",
       "  (3, 0.06011246848943184),\n",
       "  (4, 0.066608493310064)],\n",
       " [(0, 0.12844817110219955),\n",
       "  (1, 0.6783327257260907),\n",
       "  (2, 0.03763114189654475),\n",
       "  (3, 0.12897476404585412),\n",
       "  (4, 0.026613197229310978)],\n",
       " [(0, 0.03538028905892355),\n",
       "  (1, 0.023870250474135112),\n",
       "  (2, 0.025701392976260545),\n",
       "  (3, 0.559152442613302),\n",
       "  (4, 0.35589562487737886)],\n",
       " [(0, 0.03333890840162792),\n",
       "  (1, 0.01190277080894241),\n",
       "  (2, 0.005993198416680604),\n",
       "  (3, 0.919635390533534),\n",
       "  (4, 0.02912973183921503)],\n",
       " [(0, 0.27809415337889143),\n",
       "  (1, 0.055998481397114656),\n",
       "  (2, 0.5533409263477601),\n",
       "  (3, 0.09712730954188813),\n",
       "  (4, 0.015439129334345734)],\n",
       " [(0, 0.21896530126518854),\n",
       "  (1, 0.017098835024426908),\n",
       "  (2, 0.4828385318802455),\n",
       "  (3, 0.23694099962420143),\n",
       "  (4, 0.04415633220593762)],\n",
       " [(0, 0.1101722762052293),\n",
       "  (1, 0.6538515702013801),\n",
       "  (2, 0.06402854058113881),\n",
       "  (3, 0.13918227479697695),\n",
       "  (4, 0.032765338215274845)],\n",
       " [(0, 0.0050818260120585705),\n",
       "  (1, 0.011455641688199828),\n",
       "  (2, 0.004335343095033017),\n",
       "  (3, 0.9679299454493253),\n",
       "  (4, 0.01119724375538329)],\n",
       " [(0, 0.024724860545756067),\n",
       "  (1, 0.033418764762048345),\n",
       "  (2, 0.821850344238404),\n",
       "  (3, 0.025579174832906178),\n",
       "  (4, 0.09442685562088547)],\n",
       " [(0, 0.0770042194092827),\n",
       "  (1, 0.08136427566807314),\n",
       "  (2, 0.06160337552742616),\n",
       "  (3, 0.7575949367088608),\n",
       "  (4, 0.022433192686357243)],\n",
       " [(0, 0.0323056833021064),\n",
       "  (1, 0.026571282575370465),\n",
       "  (2, 0.8686765457332652),\n",
       "  (3, 0.02231306421393289),\n",
       "  (4, 0.05013342417532504)],\n",
       " [(0, 0.018099005154021335),\n",
       "  (1, 0.07407407407407407),\n",
       "  (2, 0.01861840265292261),\n",
       "  (3, 0.8732270566143274),\n",
       "  (4, 0.0159814615046546)],\n",
       " [(0, 0.2796934865900383),\n",
       "  (1, 0.028383767300023458),\n",
       "  (2, 0.17006802721088435),\n",
       "  (3, 0.18867776995855814),\n",
       "  (4, 0.33317694894049577)],\n",
       " [(0, 0.017200854700854702),\n",
       "  (1, 0.03445512820512821),\n",
       "  (2, 0.009615384615384616),\n",
       "  (3, 0.9213141025641025),\n",
       "  (4, 0.017414529914529915)],\n",
       " [(0, 0.030584591560201317),\n",
       "  (1, 0.210840108401084),\n",
       "  (2, 0.01494386372435153),\n",
       "  (3, 0.08966318234610918),\n",
       "  (4, 0.653968253968254)],\n",
       " [(0, 0.03243704652155356),\n",
       "  (1, 0.041020533978280455),\n",
       "  (2, 0.02793190117133779),\n",
       "  (3, 0.3383127045098876),\n",
       "  (4, 0.5602978138189406)],\n",
       " [(0, 0.02178481066941121),\n",
       "  (1, 0.03525022513829924),\n",
       "  (2, 0.150949869205369),\n",
       "  (3, 0.01659590891547665),\n",
       "  (4, 0.7754191860714439)],\n",
       " [(0, 0.03199276256271075),\n",
       "  (1, 0.012665515256188831),\n",
       "  (2, 0.07517065548153631),\n",
       "  (3, 0.037996545768566495),\n",
       "  (4, 0.8421745209309976)],\n",
       " [(0, 0.21772931495784745),\n",
       "  (1, 0.20965217830279165),\n",
       "  (2, 0.3681659851582614),\n",
       "  (3, 0.09505780200918774),\n",
       "  (4, 0.10939471957191176)],\n",
       " [(0, 0.18971109602474165),\n",
       "  (1, 0.05597043071584823),\n",
       "  (2, 0.3251866938221317),\n",
       "  (3, 0.08297503205853511),\n",
       "  (4, 0.3461567473787433)],\n",
       " [(0, 0.2353648135225246),\n",
       "  (1, 0.010816468946266573),\n",
       "  (2, 0.42471117314104057),\n",
       "  (3, 0.31484066061874855),\n",
       "  (4, 0.01426688377141971)],\n",
       " [(0, 0.26658916981497627),\n",
       "  (1, 0.046110626755788045),\n",
       "  (2, 0.03099874067616003),\n",
       "  (3, 0.3647195582679454),\n",
       "  (4, 0.29158190448513027)],\n",
       " [(0, 0.26352960499301964),\n",
       "  (1, 0.06339820973967315),\n",
       "  (2, 0.025827379485916072),\n",
       "  (3, 0.08643344009197668),\n",
       "  (4, 0.5608113656894145)],\n",
       " [(0, 0.05017367811655731),\n",
       "  (1, 0.14387409408636734),\n",
       "  (2, 0.7174835970667696),\n",
       "  (3, 0.02684506196663665),\n",
       "  (4, 0.06162356876366911)],\n",
       " [(0, 0.01687846203975236),\n",
       "  (1, 0.014467253176930596),\n",
       "  (2, 0.14128380579993483),\n",
       "  (3, 0.8160312805474096),\n",
       "  (4, 0.01133919843597263)],\n",
       " [(0, 0.1264684287812041),\n",
       "  (1, 0.12102300538423884),\n",
       "  (2, 0.024840920215369548),\n",
       "  (3, 0.7028267254038179),\n",
       "  (4, 0.024840920215369548)],\n",
       " [(0, 0.018661016319095943),\n",
       "  (1, 0.8736106464443701),\n",
       "  (2, 0.019701869818965836),\n",
       "  (3, 0.043641500315973386),\n",
       "  (4, 0.04438496710159474)],\n",
       " [(0, 0.23102061337355456),\n",
       "  (1, 0.0955253896430367),\n",
       "  (2, 0.45575666163901457),\n",
       "  (3, 0.04399195575666164),\n",
       "  (4, 0.17370537958773252)],\n",
       " [(0, 0.11010928961748634),\n",
       "  (1, 0.231511839708561),\n",
       "  (2, 0.4580145719489982),\n",
       "  (3, 0.08123861566484518),\n",
       "  (4, 0.11912568306010929)],\n",
       " [(0, 0.19337606837606838),\n",
       "  (1, 0.04291310541310541),\n",
       "  (2, 0.07086894586894586),\n",
       "  (3, 0.04309116809116809),\n",
       "  (4, 0.6497507122507122)],\n",
       " [(0, 0.04129793510324484),\n",
       "  (1, 0.3816930365602932),\n",
       "  (2, 0.01591132564583892),\n",
       "  (3, 0.5452757665147046),\n",
       "  (4, 0.015821936175918476)],\n",
       " [(0, 0.05228758169934641),\n",
       "  (1, 0.011319503646869376),\n",
       "  (2, 0.29790660225442833),\n",
       "  (3, 0.04788292128445581),\n",
       "  (4, 0.5906033911149)],\n",
       " [(0, 0.48135146994295747),\n",
       "  (1, 0.06869484666764177),\n",
       "  (2, 0.32475257179074646),\n",
       "  (3, 0.0921456779289162),\n",
       "  (4, 0.0330554336697382)],\n",
       " [(0, 0.01533816425120773),\n",
       "  (1, 0.034178743961352655),\n",
       "  (2, 0.014734299516908212),\n",
       "  (3, 0.9153381642512077),\n",
       "  (4, 0.02041062801932367)],\n",
       " [(0, 0.26298247984506595),\n",
       "  (1, 0.009728415079043371),\n",
       "  (2, 0.36855379903616625),\n",
       "  (3, 0.3279736972481196),\n",
       "  (4, 0.030761608791604732)],\n",
       " [(0, 0.22014826581943342),\n",
       "  (1, 0.6239078633836378),\n",
       "  (2, 0.014385314623598976),\n",
       "  (3, 0.10908128144029654),\n",
       "  (4, 0.03247727473303327)],\n",
       " [(0, 0.026455026455026454),\n",
       "  (1, 0.0974377266512098),\n",
       "  (2, 0.0495808810415552),\n",
       "  (3, 0.17032281077224898),\n",
       "  (4, 0.6562035550799595)],\n",
       " [(0, 0.3618190055498924),\n",
       "  (1, 0.013478310114395741),\n",
       "  (2, 0.30603692377392683),\n",
       "  (3, 0.030750934420659192),\n",
       "  (4, 0.28791482614112585)],\n",
       " [(0, 0.055105348460291734),\n",
       "  (1, 0.07302359085179183),\n",
       "  (2, 0.6483882585989555),\n",
       "  (3, 0.11696380334954079),\n",
       "  (4, 0.10651899873942014)],\n",
       " [(0, 0.29710806697108066),\n",
       "  (1, 0.05136986301369863),\n",
       "  (2, 0.018873668188736682),\n",
       "  (3, 0.6044901065449011),\n",
       "  (4, 0.02815829528158295)],\n",
       " [(0, 0.2851552476263793),\n",
       "  (1, 0.019630484988452657),\n",
       "  (2, 0.029445727482678985),\n",
       "  (3, 0.057865024377726455),\n",
       "  (4, 0.6079035155247626)],\n",
       " [(0, 0.010218600439787869),\n",
       "  (1, 0.01569439054887251),\n",
       "  (2, 0.034794981244340974),\n",
       "  (3, 0.9326089768464624),\n",
       "  (4, 0.00668305092053637)],\n",
       " [(0, 0.04072398190045249),\n",
       "  (1, 0.7447069996089604),\n",
       "  (2, 0.020948550360315066),\n",
       "  (3, 0.13680799955309758),\n",
       "  (4, 0.05681246857717446)],\n",
       " [(0, 0.0638559843756634),\n",
       "  (1, 0.04305184052986881),\n",
       "  (2, 0.1619326625058379),\n",
       "  (3, 0.032012907060671676),\n",
       "  (4, 0.6991466055279583)],\n",
       " [(0, 0.09701797385620915),\n",
       "  (1, 0.016748366013071895),\n",
       "  (2, 0.012152777777777778),\n",
       "  (3, 0.05361519607843137),\n",
       "  (4, 0.8204656862745098)],\n",
       " [(0, 0.8869135802469136),\n",
       "  (1, 0.016246913580246915),\n",
       "  (2, 0.01762962962962963),\n",
       "  (3, 0.02330864197530864),\n",
       "  (4, 0.055901234567901234)],\n",
       " [(0, 0.06875132563706329),\n",
       "  (1, 0.28224706844832287),\n",
       "  (2, 0.608368936157319),\n",
       "  (3, 0.01639244917128746),\n",
       "  (4, 0.024240220586007333)],\n",
       " [(0, 0.053032351085879066),\n",
       "  (1, 0.13445075245561863),\n",
       "  (2, 0.6767144273226998),\n",
       "  (3, 0.10885824997747139),\n",
       "  (4, 0.02694421915833108)],\n",
       " [(0, 0.058374983980520316),\n",
       "  (1, 0.02704088171216199),\n",
       "  (2, 0.2857875176214277),\n",
       "  (3, 0.5993848519800077),\n",
       "  (4, 0.029411764705882353)],\n",
       " [(0, 0.7387296799061505),\n",
       "  (1, 0.046421987598458184),\n",
       "  (2, 0.02807105748282219),\n",
       "  (3, 0.07331992626110273),\n",
       "  (4, 0.1134573487514664)],\n",
       " [(0, 0.0690414786159467),\n",
       "  (1, 0.028100150440575972),\n",
       "  (2, 0.03551472168493445),\n",
       "  (3, 0.06517300666236836),\n",
       "  (4, 0.8021706425961745)],\n",
       " [(0, 0.11022927689594356),\n",
       "  (1, 0.03772290809327846),\n",
       "  (2, 0.03292181069958848),\n",
       "  (3, 0.05055849500293945),\n",
       "  (4, 0.76856750930825)],\n",
       " [(0, 0.28186046511627905),\n",
       "  (1, 0.04403100775193798),\n",
       "  (2, 0.10335917312661498),\n",
       "  (3, 0.0815848406546081),\n",
       "  (4, 0.4891645133505599)],\n",
       " [(0, 0.13904641881957575),\n",
       "  (1, 0.17160260449485404),\n",
       "  (2, 0.4639781558496115),\n",
       "  (3, 0.2089897080445285),\n",
       "  (4, 0.016383112791430374)],\n",
       " [(0, 0.48888888888888893),\n",
       "  (1, 0.11363636363636365),\n",
       "  (2, 0.23044733044733048),\n",
       "  (3, 0.07922077922077923),\n",
       "  (4, 0.08780663780663782)],\n",
       " [(0, 0.7220952380952381),\n",
       "  (1, 0.018666666666666668),\n",
       "  (2, 0.05853968253968254),\n",
       "  (3, 0.1391111111111111),\n",
       "  (4, 0.061587301587301586)],\n",
       " [(0, 0.04403100775193798),\n",
       "  (1, 0.03028423772609819),\n",
       "  (2, 0.023979328165374677),\n",
       "  (3, 0.8797932816537468),\n",
       "  (4, 0.021912144702842376)],\n",
       " [(0, 0.09275206315034087),\n",
       "  (1, 0.12169596938165292),\n",
       "  (2, 0.5975959813419447),\n",
       "  (3, 0.04843918191603875),\n",
       "  (4, 0.13951680421002272)],\n",
       " [(0, 0.054493713897383625),\n",
       "  (1, 0.010236153584777438),\n",
       "  (2, 0.07016649677200136),\n",
       "  (3, 0.006328576282704723),\n",
       "  (4, 0.8587750594631328)],\n",
       " [(0, 0.02147326507394767),\n",
       "  (1, 0.2762135001896094),\n",
       "  (2, 0.008532423208191127),\n",
       "  (3, 0.5971748198710656),\n",
       "  (4, 0.0966059916571862)],\n",
       " [(0, 0.057868442964196486),\n",
       "  (1, 0.29401424738643717),\n",
       "  (2, 0.013969839948191318),\n",
       "  (3, 0.6163844944028124),\n",
       "  (4, 0.017762975298362474)],\n",
       " [(0, 0.6997597352554514),\n",
       "  (1, 0.08549798268280522),\n",
       "  (2, 0.009202593045922301),\n",
       "  (3, 0.15259077927376583),\n",
       "  (4, 0.052948909742055406)],\n",
       " [(0, 0.191247349142086),\n",
       "  (1, 0.08752650857914016),\n",
       "  (2, 0.023456076087655035),\n",
       "  (3, 0.36180194074930916),\n",
       "  (4, 0.33596812544180965)],\n",
       " [(0, 0.4695998860885661),\n",
       "  (1, 0.07651051307608335),\n",
       "  (2, 0.09098675779581375),\n",
       "  (3, 0.3350420048412359),\n",
       "  (4, 0.027860838198300823)],\n",
       " [(0, 0.010798623472172779),\n",
       "  (1, 0.006526640560104427),\n",
       "  (2, 0.255290534393418),\n",
       "  (3, 0.7176535738301492),\n",
       "  (4, 0.00973062774415569)],\n",
       " [(0, 0.037037037037037035),\n",
       "  (1, 0.016226442552572218),\n",
       "  (2, 0.03201629920686895),\n",
       "  (3, 0.722331368696791),\n",
       "  (4, 0.1923888525067307)],\n",
       " [(0, 0.039568618515986934),\n",
       "  (1, 0.08939014202172096),\n",
       "  (2, 0.14794562162983216),\n",
       "  (3, 0.03569529885319359),\n",
       "  (4, 0.6874003189792663)],\n",
       " [(0, 0.16273408239700374),\n",
       "  (1, 0.5935081148564295),\n",
       "  (2, 0.009176029962546817),\n",
       "  (3, 0.05867665418227216),\n",
       "  (4, 0.17590511860174782)],\n",
       " [(0, 0.6471174945142629),\n",
       "  (1, 0.03125207793071348),\n",
       "  (2, 0.17600904315446506),\n",
       "  (3, 0.07766473834696457),\n",
       "  (4, 0.06795664605359399)],\n",
       " [(0, 0.019456681350954477),\n",
       "  (1, 0.008994126284875184),\n",
       "  (2, 0.03132648066568772),\n",
       "  (3, 0.804637787567303),\n",
       "  (4, 0.13558492413117965)],\n",
       " [(0, 0.09812636165577343),\n",
       "  (1, 0.5291503267973857),\n",
       "  (2, 0.18413943355119827),\n",
       "  (3, 0.043747276688453164),\n",
       "  (4, 0.14483660130718956)],\n",
       " [(0, 0.7463266545233759),\n",
       "  (1, 0.00805504958510423),\n",
       "  (2, 0.022465088038858532),\n",
       "  (3, 0.188990082979154),\n",
       "  (4, 0.03416312487350739)],\n",
       " [(0, 0.038050149044362616),\n",
       "  (1, 0.7762581097667894),\n",
       "  (2, 0.12245017242387048),\n",
       "  (3, 0.04272605061663452),\n",
       "  (4, 0.020515518148342977)],\n",
       " [(0, 0.1279830873683783),\n",
       "  (1, 0.11810383380087004),\n",
       "  (2, 0.011627434239947963),\n",
       "  (3, 0.6902874334268407),\n",
       "  (4, 0.05199821116396309)],\n",
       " [(0, 0.16861019435276864),\n",
       "  (1, 0.0343967730106344),\n",
       "  (2, 0.11932526585991934),\n",
       "  (3, 0.6542720938760543),\n",
       "  (4, 0.023395672900623398)],\n",
       " [(0, 0.018187830687830687),\n",
       "  (1, 0.8448247354497355),\n",
       "  (2, 0.021164021164021163),\n",
       "  (3, 0.08606150793650794),\n",
       "  (4, 0.02976190476190476)],\n",
       " [(0, 0.07251211182590739),\n",
       "  (1, 0.5202128504487332),\n",
       "  (2, 0.04042570089746644),\n",
       "  (3, 0.292431101580494),\n",
       "  (4, 0.07441823524739893)],\n",
       " [(0, 0.09215885947046844),\n",
       "  (1, 0.12655578185109753),\n",
       "  (2, 0.38854944557592214),\n",
       "  (3, 0.017651052274270197),\n",
       "  (4, 0.37508486082824166)],\n",
       " [(0, 0.036854228364593615),\n",
       "  (1, 0.5145332894592519),\n",
       "  (2, 0.31013491280026323),\n",
       "  (3, 0.06948557639574421),\n",
       "  (4, 0.06899199298014698)],\n",
       " [(0, 0.04582977512097922),\n",
       "  (1, 0.0454502324698738),\n",
       "  (2, 0.025903785937944776),\n",
       "  (3, 0.845810797988424),\n",
       "  (4, 0.03700540848277825)],\n",
       " [(0, 0.10980545763154459),\n",
       "  (1, 0.04524089306698002),\n",
       "  (2, 0.47238542890716806),\n",
       "  (3, 0.3465204334769552),\n",
       "  (4, 0.026047786917352135)],\n",
       " [(0, 0.138538257752844),\n",
       "  (1, 0.10285179990649837),\n",
       "  (2, 0.2114695340501792),\n",
       "  (3, 0.4015116097865046),\n",
       "  (4, 0.1456287985039738)],\n",
       " [(0, 0.023523942352394234),\n",
       "  (1, 0.013621571362157137),\n",
       "  (2, 0.8557880055788005),\n",
       "  (3, 0.07271036727103673),\n",
       "  (4, 0.034356113435611346)],\n",
       " [(0, 0.03994126284875184),\n",
       "  (1, 0.02062326643824441),\n",
       "  (2, 0.8101158427149616),\n",
       "  (3, 0.09541523902757383),\n",
       "  (4, 0.03390438897046827)],\n",
       " [(0, 0.7348720224885338),\n",
       "  (1, 0.0230803373280071),\n",
       "  (2, 0.15327711199881638),\n",
       "  (3, 0.037801449918627016),\n",
       "  (4, 0.05096907826601568)],\n",
       " [(0, 0.2021553325901152),\n",
       "  (1, 0.02147900408769974),\n",
       "  (2, 0.12746191007060573),\n",
       "  (3, 0.015830546265328874),\n",
       "  (4, 0.6330732069862505)],\n",
       " [(0, 0.03264876250658241),\n",
       "  (1, 0.030241480478447304),\n",
       "  (2, 0.03829082976002407),\n",
       "  (3, 0.023922365154592642),\n",
       "  (4, 0.8748965621003536)],\n",
       " [(0, 0.659029317565903),\n",
       "  (1, 0.026915496427691553),\n",
       "  (2, 0.16869918699186995),\n",
       "  (3, 0.008622813500862284),\n",
       "  (4, 0.13673318551367333)],\n",
       " [(0, 0.8538044847568657),\n",
       "  (1, 0.05335097001763668),\n",
       "  (2, 0.051776266061980346),\n",
       "  (3, 0.017699672461577224),\n",
       "  (4, 0.023368606701940034)],\n",
       " [(0, 0.007279219887378108),\n",
       "  (1, 0.007856063727509957),\n",
       "  (2, 0.006455157258618322),\n",
       "  (3, 0.9672023073753605),\n",
       "  (4, 0.011207251751133086)],\n",
       " [(0, 0.6885466324410219),\n",
       "  (1, 0.25058061361691725),\n",
       "  (2, 0.03006967363403007),\n",
       "  (3, 0.014484781811514484),\n",
       "  (4, 0.01631829849651632)],\n",
       " [(0, 0.017798957315086347),\n",
       "  (1, 0.0952671880091235),\n",
       "  (2, 0.011567285760834147),\n",
       "  (3, 0.039263603779732814),\n",
       "  (4, 0.8361029651352232)],\n",
       " [(0, 0.13308104698833176),\n",
       "  (1, 0.09718280248081572),\n",
       "  (2, 0.03805319037107117),\n",
       "  (3, 0.28166719226321874),\n",
       "  (4, 0.4500157678965626)],\n",
       " [(0, 0.08906426155580609),\n",
       "  (1, 0.015115453672387156),\n",
       "  (2, 0.022046849555305024),\n",
       "  (3, 0.02434339638398263),\n",
       "  (4, 0.8494300388325191)],\n",
       " [(0, 0.17178975382568196),\n",
       "  (1, 0.03546240851630073),\n",
       "  (2, 0.16699933466400532),\n",
       "  (3, 0.1296074517631404),\n",
       "  (4, 0.4961410512308716)],\n",
       " [(0, 0.6616333207499686),\n",
       "  (1, 0.01503712092613565),\n",
       "  (2, 0.012583364791745315),\n",
       "  (3, 0.289480307034101),\n",
       "  (4, 0.021265886498049582)],\n",
       " [(0, 0.06516643225503985),\n",
       "  (1, 0.7423816221284576),\n",
       "  (2, 0.11808485700890764),\n",
       "  (3, 0.03662681669010783),\n",
       "  (4, 0.037740271917487106)],\n",
       " [(0, 0.05392512077294686),\n",
       "  (1, 0.08997584541062802),\n",
       "  (2, 0.44583333333333336),\n",
       "  (3, 0.36268115942028983),\n",
       "  (4, 0.04758454106280193)],\n",
       " [(0, 0.4279661016949153),\n",
       "  (1, 0.37372208770513854),\n",
       "  (2, 0.026937046004842615),\n",
       "  (3, 0.1617567931127253),\n",
       "  (4, 0.009617971482378262)],\n",
       " [(0, 0.009991673605328892),\n",
       "  (1, 0.007354981959478212),\n",
       "  (2, 0.06596354889443982),\n",
       "  (3, 0.8980016652789342),\n",
       "  (4, 0.018688130261818856)],\n",
       " [(0, 0.2088597722664171),\n",
       "  (1, 0.010710757552123955),\n",
       "  (2, 0.61269692715645),\n",
       "  (3, 0.15452607497530288),\n",
       "  (4, 0.013206468049706236)],\n",
       " [(0, 0.013379204892966361),\n",
       "  (1, 0.010448521916411825),\n",
       "  (2, 0.012487257900101937),\n",
       "  (3, 0.9485219164118247),\n",
       "  (4, 0.015163098878695208)],\n",
       " [(0, 0.06092848446350114),\n",
       "  (1, 0.3062304801228098),\n",
       "  (2, 0.021174104070721508),\n",
       "  (3, 0.587528452702345),\n",
       "  (4, 0.024138478640622518)],\n",
       " [(0, 0.8833941677459869),\n",
       "  (1, 0.010910946269476824),\n",
       "  (2, 0.01122493033478551),\n",
       "  (3, 0.0749244475842851),\n",
       "  (4, 0.019545508065465678)],\n",
       " [(0, 0.01338255652976465),\n",
       "  (1, 0.008562785212531406),\n",
       "  (2, 0.06470799364200379),\n",
       "  (3, 0.8425883197456802),\n",
       "  (4, 0.07075834487002)],\n",
       " [(0, 0.015627683324746695),\n",
       "  (1, 0.6430820310263896),\n",
       "  (2, 0.01328066861297155),\n",
       "  (3, 0.30934798786421663),\n",
       "  (4, 0.01866162917167554)],\n",
       " [(0, 0.025616577676074243),\n",
       "  (1, 0.019959318586320874),\n",
       "  (2, 0.16514111365369946),\n",
       "  (3, 0.774281718789728),\n",
       "  (4, 0.015001271294177472)],\n",
       " [(0, 0.055133808756997166),\n",
       "  (1, 0.7797714899164174),\n",
       "  (2, 0.01790506862970631),\n",
       "  (3, 0.06885975001917032),\n",
       "  (4, 0.07832988267770877)],\n",
       " [(0, 0.020702896735954584),\n",
       "  (1, 0.8982803239001584),\n",
       "  (2, 0.006845312630436595),\n",
       "  (3, 0.06440437432172968),\n",
       "  (4, 0.00976709241172051)],\n",
       " [(0, 0.09981515711645102),\n",
       "  (1, 0.1532142123639351),\n",
       "  (2, 0.20096529061408913),\n",
       "  (3, 0.5226945984801807),\n",
       "  (4, 0.023310741425344013)],\n",
       " [(0, 0.01783185271018411),\n",
       "  (1, 0.02864683919145101),\n",
       "  (2, 0.13396420754474056),\n",
       "  (3, 0.025878717651602934),\n",
       "  (4, 0.7936783829020214)],\n",
       " [(0, 0.03096008629989213),\n",
       "  (1, 0.09568500539374328),\n",
       "  (2, 0.6115965480043151),\n",
       "  (3, 0.16650485436893206),\n",
       "  (4, 0.0952535059331176)],\n",
       " [(0, 0.07663523424637031),\n",
       "  (1, 0.008831013321359079),\n",
       "  (2, 0.5601955795040663),\n",
       "  (3, 0.3378236790899566),\n",
       "  (4, 0.016514493838247768)],\n",
       " [(0, 0.15144372421921037),\n",
       "  (1, 0.7295226870948733),\n",
       "  (2, 0.01571400510705166),\n",
       "  (3, 0.04746939042755189),\n",
       "  (4, 0.055850193151312774)],\n",
       " [(0, 0.03188942938769208),\n",
       "  (1, 0.708362288626361),\n",
       "  (2, 0.06401050111960467),\n",
       "  (3, 0.023009806192572004),\n",
       "  (4, 0.17272797467377038)],\n",
       " [(0, 0.10146923783287419),\n",
       "  (1, 0.2558412406897255),\n",
       "  (2, 0.5627486991123355),\n",
       "  (3, 0.024028160391796755),\n",
       "  (4, 0.05591266197326803)],\n",
       " [(0, 0.6287436795021393),\n",
       "  (1, 0.07156748346946713),\n",
       "  (2, 0.040062232594321276),\n",
       "  (3, 0.16530532866588876),\n",
       "  (4, 0.09432127576818358)],\n",
       " [(0, 0.021332475746368654),\n",
       "  (1, 0.021064479819906737),\n",
       "  (2, 0.15731360883314574),\n",
       "  (3, 0.013828589805434957),\n",
       "  (4, 0.7864608457951439)],\n",
       " [(0, 0.07711651299245599),\n",
       "  (1, 0.6641209524696532),\n",
       "  (2, 0.01608146285430443),\n",
       "  (3, 0.04119710657849803),\n",
       "  (4, 0.20148396510508831)],\n",
       " [(0, 0.1913801949717804),\n",
       "  (1, 0.023886893563650875),\n",
       "  (2, 0.6952853315090359),\n",
       "  (3, 0.06960834616042415),\n",
       "  (4, 0.019839233795108602)],\n",
       " [(0, 0.10149028485191473),\n",
       "  (1, 0.05324467081682702),\n",
       "  (2, 0.6789756649688738),\n",
       "  (3, 0.15586681758158838),\n",
       "  (4, 0.010422561780796076)],\n",
       " [(0, 0.1365573466740782),\n",
       "  (1, 0.18423815700080295),\n",
       "  (2, 0.5134333889197703),\n",
       "  (3, 0.12006670372429129),\n",
       "  (4, 0.04570440368105738)],\n",
       " [(0, 0.06838379273987047),\n",
       "  (1, 0.052467741125671534),\n",
       "  (2, 0.7842546568258272),\n",
       "  (3, 0.05086107345483758),\n",
       "  (4, 0.04403273585379324)],\n",
       " [(0, 0.47984496124031006),\n",
       "  (1, 0.4089922480620155),\n",
       "  (2, 0.040671834625323),\n",
       "  (3, 0.013436692506459949),\n",
       "  (4, 0.05705426356589147)],\n",
       " [(0, 0.4563121198635217),\n",
       "  (1, 0.04306977204173466),\n",
       "  (2, 0.29654353953419377),\n",
       "  (3, 0.08257924145774613),\n",
       "  (4, 0.12149532710280374)],\n",
       " [(0, 0.015895278167367927),\n",
       "  (1, 0.02462209755337385),\n",
       "  (2, 0.9153031011376033),\n",
       "  (3, 0.027972572853358267),\n",
       "  (4, 0.01620695028829671)],\n",
       " [(0, 0.01833731970718272),\n",
       "  (1, 0.010871928680147858),\n",
       "  (2, 0.9328839602812206),\n",
       "  (3, 0.024787997390737115),\n",
       "  (4, 0.013118793940711748)],\n",
       " [(0, 0.06757054673721341),\n",
       "  (1, 0.05529835390946502),\n",
       "  (2, 0.6431143445032333),\n",
       "  (3, 0.07374338624338625),\n",
       "  (4, 0.16027336860670194)],\n",
       " [(0, 0.027440941345549266),\n",
       "  (1, 0.8064313302793497),\n",
       "  (2, 0.012350669181712027),\n",
       "  (3, 0.0479655079493398),\n",
       "  (4, 0.10581155124404923)],\n",
       " [(0, 0.10525261629362576),\n",
       "  (1, 0.6727254519052627),\n",
       "  (2, 0.0966401281858695),\n",
       "  (3, 0.08477292073506584),\n",
       "  (4, 0.04060888288017626)],\n",
       " [(0, 0.48743682824400314),\n",
       "  (1, 0.1542458537974233),\n",
       "  (2, 0.026336394049398533),\n",
       "  (3, 0.03445085059434835),\n",
       "  (4, 0.29753007331482667)],\n",
       " [(0, 0.8847826086956522),\n",
       "  (1, 0.017777777777777778),\n",
       "  (2, 0.04391304347826087),\n",
       "  (3, 0.03111111111111111),\n",
       "  (4, 0.02241545893719807)],\n",
       " [(0, 0.0216695753344968),\n",
       "  (1, 0.026977894124490982),\n",
       "  (2, 0.012434554973821989),\n",
       "  (3, 0.6181646305991856),\n",
       "  (4, 0.32075334496800467)],\n",
       " [(0, 0.05285827792414168),\n",
       "  (1, 0.11487475431773513),\n",
       "  (2, 0.026763685024881863),\n",
       "  (3, 0.24601681093965624),\n",
       "  (4, 0.5594864717935851)],\n",
       " [(0, 0.1877188487488257),\n",
       "  (1, 0.08719788197113332),\n",
       "  (2, 0.4514476044068665),\n",
       "  (3, 0.2229054573405073),\n",
       "  (4, 0.05073020753266718)],\n",
       " [(0, 0.02312111945139468),\n",
       "  (1, 0.02335279399499583),\n",
       "  (2, 0.009730330831248262),\n",
       "  (3, 0.0153831896951163),\n",
       "  (4, 0.928412566027245)],\n",
       " [(0, 0.23231738683127576),\n",
       "  (1, 0.5016718106995885),\n",
       "  (2, 0.01575360082304527),\n",
       "  (3, 0.16393389917695475),\n",
       "  (4, 0.08632330246913582)],\n",
       " [(0, 0.022301871764237355),\n",
       "  (1, 0.04828753484667463),\n",
       "  (2, 0.05246913580246913),\n",
       "  (3, 0.03136200716845878),\n",
       "  (4, 0.8455794504181601)],\n",
       " [(0, 0.06276106934001671),\n",
       "  (1, 0.050020885547201334),\n",
       "  (2, 0.7326649958228906),\n",
       "  (3, 0.029553049289891394),\n",
       "  (4, 0.125)],\n",
       " [(0, 0.6160489524030828),\n",
       "  (1, 0.019712637000890747),\n",
       "  (2, 0.3085860346229813),\n",
       "  (3, 0.01107625576081484),\n",
       "  (4, 0.044576120212230355)],\n",
       " [(0, 0.07086914995224451),\n",
       "  (1, 0.036867239732569244),\n",
       "  (2, 0.014390321553645335),\n",
       "  (3, 0.8258516396052212),\n",
       "  (4, 0.05202164915631964)]]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "doc_lda"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# article\n",
    "with open(\"./data/saved_results/data_words/article_sq.pickle\", \"rb\") as fp:\n",
    "    articles = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "def topics_df(model, bar=0.003, num_words=40):\n",
    "    \"\"\"\n",
    "    Use this method to display topics in one DataFrame\n",
    "\n",
    "    @param\n",
    "    model: input model \n",
    "    bar: threshold of word frequency \n",
    "    num_words: number of words for each topic\n",
    "    \"\"\"\n",
    "    topics = {}\n",
    "    for idx, item in model.print_topics(num_words=num_words):\n",
    "        idx += 1\n",
    "        topic = []\n",
    "        for num, j in zip(re.findall(r'\\s(0.0[0-9]{2})\\*', item),re.findall('\\\"([a-zA-Z]+)\\\"', item)):\n",
    "            \n",
    "            # set the threshold\n",
    "            if float(num) >= bar:\n",
    "                topic.append(j)\n",
    "        topics[f'Topic{idx}'] = ', '.join(topic)\n",
    "\n",
    "    # create the dataframe \n",
    "    df = pd.DataFrame(topics.values(), columns=['Terms per Topic'], index=topics.keys())\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "pd.set_option('display.max_colwidth', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([10., 10., 10., 10., 10.])"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lda_5.alpha"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([1., 1., 1., 1., 1.])"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lda_5.alpha"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                   Terms per Topic\n",
       "Topic1                                                                                                      mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary\n",
       "Topic2                                                                                  light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane\n",
       "Topic3  biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor\n",
       "Topic4                                         circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain\n",
       "Topic5                                                                                            yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([0.00000000e+00, 0.00000000e+00, 0.00000000e+00, ...,\n",
       "       1.53210683e-05, 0.00000000e+00, 6.12842732e-06])"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lda_5.get_topics()[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[(0,\n",
       "  '0.013*\"circuit\" + 0.007*\"input\" + 0.007*\"network\" + 0.006*\"output\" + 0.005*\"behavior\" + 0.005*\"gate\" + 0.004*\"strand\" + 0.004*\"simulation\" + 0.004*\"device\" + 0.004*\"module\"'),\n",
       " (1,\n",
       "  '0.006*\"c\" + 0.006*\"ion\" + 0.006*\"membrane\" + 0.005*\"residue\" + 0.005*\"peptide\" + 0.004*\"surface\" + 0.003*\"affinity\" + 0.003*\"fusion\" + 0.003*\"secretion\" + 0.003*\"E.\"'),\n",
       " (2,\n",
       "  '0.011*\"E.\" + 0.007*\"S.\" + 0.007*\"c\" + 0.005*\"biosynthesis\" + 0.005*\"titer\" + 0.004*\"carbon\" + 0.004*\"cluster\" + 0.004*\"metabolite\" + 0.004*\"fermentation\" + 0.004*\"flux\"'),\n",
       " (3,\n",
       "  '0.008*\"c\" + 0.007*\"E.\" + 0.005*\"terminator\" + 0.004*\"mrna\" + 0.004*\"bp\" + 0.004*\"GFP\" + 0.004*\"repression\" + 0.003*\"tRNA\" + 0.003*\"cassette\" + 0.003*\"rb\"'),\n",
       " (4,\n",
       "  '0.006*\"sensor\" + 0.005*\"light\" + 0.005*\"activation\" + 0.005*\"E.\" + 0.005*\"ligand\" + 0.005*\"GFP\" + 0.004*\"switch\" + 0.004*\"biosensor\" + 0.003*\"riboswitch\" + 0.003*\"receptor\"')]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lda_5.print_topics()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'examined',\n",
       " 'linker',\n",
       " 'MA',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'MA',\n",
       " 'linker',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'rigid',\n",
       " 'linker',\n",
       " 'supplementary',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'suitable',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'supplementary',\n",
       " 'display',\n",
       " 'MA',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'CFT53c',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'MA',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " '16-fold',\n",
       " 'CFT51b',\n",
       " 'express',\n",
       " 'unfused',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'MenF',\n",
       " 'fused',\n",
       " 'aroc',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'western',\n",
       " 'blotting',\n",
       " 'supplementary',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'placed',\n",
       " 'close',\n",
       " 'proximity',\n",
       " 'thereby',\n",
       " 'apparent',\n",
       " 'intermediate',\n",
       " 'chorismate',\n",
       " 'around',\n",
       " 'MenF',\n",
       " 'nonfusion',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'competing',\n",
       " 'branched',\n",
       " 'compound',\n",
       " 'channeling',\n",
       " 'fusion',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'fusion',\n",
       " 'improvement',\n",
       " 'MA',\n",
       " 'noticed',\n",
       " 'CFT53b',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'overexpressed',\n",
       " 'separately',\n",
       " 'MA',\n",
       " 'drastically',\n",
       " 'CFT53a',\n",
       " 'overexpressing',\n",
       " 'AroD',\n",
       " 'sds-page',\n",
       " 'CFT53b',\n",
       " 'AroD',\n",
       " 'overexpressed',\n",
       " 'supplementary',\n",
       " 's6',\n",
       " 'explanation',\n",
       " 'fact',\n",
       " 'overexpression',\n",
       " 'AroD',\n",
       " 'shifted',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'shikimate',\n",
       " 'effectively',\n",
       " 'PCA',\n",
       " 'AroZ',\n",
       " 'AroY',\n",
       " 'CatA',\n",
       " 'distinctively',\n",
       " 'CFT53b',\n",
       " 'imbalance',\n",
       " 'MA',\n",
       " 'hand',\n",
       " 'sds-page',\n",
       " 'CFT53c',\n",
       " 'distinctively',\n",
       " 'overexpressed',\n",
       " 'AroD',\n",
       " 'fused',\n",
       " 'arod',\n",
       " 'AroZ',\n",
       " 'supplementary',\n",
       " 's6',\n",
       " 'arod',\n",
       " 'AroZ',\n",
       " 'greatly',\n",
       " 'AroD',\n",
       " 'imbalance',\n",
       " 'eliminated',\n",
       " 'MA',\n",
       " 'restored',\n",
       " 'titer',\n",
       " 'CFT53c',\n",
       " 'CFT53a',\n",
       " 'seems',\n",
       " 'channeling',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'worked',\n",
       " 'inactivation',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'AroD',\n",
       " 'homodimer',\n",
       " 'E. coli',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " 'dimerize',\n",
       " 'properly',\n",
       " 'fused',\n",
       " 'AroZ',\n",
       " 'endogenous',\n",
       " 'AroD',\n",
       " '3-dehydroquinate dehydratase',\n",
       " 'CFT53a',\n",
       " 'hypothesis',\n",
       " 'explain',\n",
       " 'MA',\n",
       " 'nearly',\n",
       " 'CFT53a',\n",
       " 'CFT53c',\n",
       " 'difficult',\n",
       " 'preserve',\n",
       " 'dimeric',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'channeling',\n",
       " 'involves',\n",
       " 'protein-level',\n",
       " 'fusion',\n",
       " 'transpeptidases',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'MA',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'CFT51b',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'CFT51c',\n",
       " 'black',\n",
       " 'Pac',\n",
       " '-man',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'orange',\n",
       " 'purple',\n",
       " 'Pac',\n",
       " '-man',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'consuming',\n",
       " 'chorismate',\n",
       " 'yellow',\n",
       " 'band',\n",
       " 'flexible',\n",
       " 'gg',\n",
       " 'GGS',\n",
       " ') 3',\n",
       " 'linker',\n",
       " 'light',\n",
       " 'gray',\n",
       " 'circle',\n",
       " '5-O-(1-Carboxyvinyl)-3-phosphoshikimate',\n",
       " 'epsp',\n",
       " 'dark',\n",
       " 'gray',\n",
       " 'diamond',\n",
       " 'chorismate',\n",
       " 'square',\n",
       " 'isochorismate',\n",
       " 'yellow',\n",
       " 'star',\n",
       " 'MA',\n",
       " 'orange',\n",
       " 'pentagon',\n",
       " 'purple',\n",
       " 'triangle',\n",
       " 'byproduct',\n",
       " 'chorismate',\n",
       " 'fusion',\n",
       " 'compound',\n",
       " 'matsumoto',\n",
       " 'created',\n",
       " 'recombinant',\n",
       " 'E. coli',\n",
       " 'fusion',\n",
       " 'pyruvate-formate lyase',\n",
       " 'PFL',\n",
       " 'phosphate acetyltransferase',\n",
       " 'PTA',\n",
       " 'ligated',\n",
       " 'Staphylococcal',\n",
       " 'sortase A',\n",
       " 'acetate',\n",
       " 'PFL',\n",
       " 'PTA',\n",
       " 'separately',\n",
       " 'Sortase-A',\n",
       " '-mediated',\n",
       " 'fusion',\n",
       " 'channeling',\n",
       " 'hence',\n",
       " 'seems',\n",
       " 'AroZ',\n",
       " '-arod',\n",
       " 'fusion',\n",
       " 'ph-controlled',\n",
       " 'tube-scale',\n",
       " 'CaCO 3',\n",
       " 'attempted',\n",
       " 'MA',\n",
       " 'titer',\n",
       " 'controlling',\n",
       " 'culturing',\n",
       " 'bad',\n",
       " 'influence',\n",
       " 'bioproduction',\n",
       " 'E. coli',\n",
       " 'grows',\n",
       " 'environment',\n",
       " 'inside',\n",
       " 'external',\n",
       " 'ic',\n",
       " 'side',\n",
       " 'fall',\n",
       " 'intracellular',\n",
       " 'culturing',\n",
       " 'CFT51c',\n",
       " 'thompson',\n",
       " 'MA',\n",
       " 'E. coli',\n",
       " 'MA',\n",
       " 'titer',\n",
       " 'far',\n",
       " 'microbe',\n",
       " 'batch',\n",
       " 'ph-controlled',\n",
       " 'E. coli',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'MA',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'limit',\n",
       " 'non-ph-control',\n",
       " 'hypothesized',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'cultivation',\n",
       " 'illustrates',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'cultured',\n",
       " 'CaCO 3',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'meaning',\n",
       " '29-fold',\n",
       " 'CaCO 3',\n",
       " 'culturing',\n",
       " 'CFT51c',\n",
       " 'CaCO 3',\n",
       " 'maintained',\n",
       " 'profile',\n",
       " 'CFT51c',\n",
       " 'closed',\n",
       " 'circle',\n",
       " 'non-ph-controlled',\n",
       " 'cultivation',\n",
       " 'open',\n",
       " 'circle',\n",
       " 'ph-controlled',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'CFT51c',\n",
       " 'Glucose',\n",
       " 'consumption',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'arrow',\n",
       " 'CaCO 3',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'cultivationin',\n",
       " 'perform',\n",
       " 'cultivation',\n",
       " 'precisely',\n",
       " 'controlling',\n",
       " 'CFT51c',\n",
       " 'cultured',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'MA',\n",
       " 'consumption',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'cultivation',\n",
       " 'correspond',\n",
       " 'profile',\n",
       " 'ph-control',\n",
       " 'test tube',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'test tube',\n",
       " 'CaCO 3',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'adjustment',\n",
       " 'MA',\n",
       " 'stage',\n",
       " 'examine',\n",
       " 'dissolved',\n",
       " 'oxygen',\n",
       " 'handle',\n",
       " 'osmotic',\n",
       " 'pressure',\n",
       " 'MA',\n",
       " 'byproduct',\n",
       " 'accumulation',\n",
       " 'profile',\n",
       " 'CFT51c',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'CFT51c',\n",
       " 'Glucose',\n",
       " 'consumption',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'conclusionsin',\n",
       " 'summary',\n",
       " 'MA',\n",
       " 'aiming',\n",
       " 'select',\n",
       " 'suitable',\n",
       " 'MA',\n",
       " 'CFT5-derived strains',\n",
       " 'greatly',\n",
       " 'MA',\n",
       " 'compound',\n",
       " 'microorganism',\n",
       " 'optimal',\n",
       " 'perspective',\n",
       " 'industrial',\n",
       " 'compound',\n",
       " 'obtain',\n",
       " 'MA',\n",
       " 'fusion',\n",
       " 'hence',\n",
       " 'fusion',\n",
       " 'directed',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'shikimate',\n",
       " 'MA',\n",
       " 'MA',\n",
       " '44-fold',\n",
       " 'CFT51a',\n",
       " 'titer',\n",
       " 'MA',\n",
       " 'carbon',\n",
       " 'CFT51c',\n",
       " 'batch',\n",
       " 'cultures',\n",
       " 'flask',\n",
       " 'test tube',\n",
       " 'channeling',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'powerful',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'selectively',\n",
       " 'leading',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'difficult',\n",
       " 'apply',\n",
       " 'multimeric',\n",
       " 'titer',\n",
       " 'combine',\n",
       " 'disruption',\n",
       " 'metabolite',\n",
       " 'funneling',\n",
       " 'fusion',\n",
       " 'methodsstrains',\n",
       " 'constructionthe',\n",
       " 'E. coli',\n",
       " 'NovaBlue competent cells',\n",
       " 'novagen',\n",
       " 'cambridge',\n",
       " 'MA',\n",
       " 'usa',\n",
       " 'chain',\n",
       " 'quick',\n",
       " 'taq',\n",
       " 'toyobo',\n",
       " 'osaka',\n",
       " 'japan',\n",
       " 'custom',\n",
       " 'oligonucleotide',\n",
       " 'invitrogen',\n",
       " 'custom',\n",
       " 'oligos',\n",
       " 'thermo',\n",
       " 'fisher',\n",
       " 'tokyo',\n",
       " 'japan',\n",
       " 'supplementary',\n",
       " 'exogenous',\n",
       " 'pobA',\n",
       " 'P. putida',\n",
       " 'KT2440',\n",
       " 'aroY',\n",
       " 'K. pneumoniae',\n",
       " 'aroZ',\n",
       " 'Bacillus thuringiensis',\n",
       " 'invitrogen',\n",
       " 'geneart',\n",
       " 'service',\n",
       " 'thermo',\n",
       " 'fisher',\n",
       " 'tokyo',\n",
       " 'japan',\n",
       " 'exogenous',\n",
       " 'detailed',\n",
       " 'fusion',\n",
       " 'constructionthe',\n",
       " 'fusion',\n",
       " 'flexible',\n",
       " 'gg',\n",
       " 'GGS',\n",
       " ') 3',\n",
       " 'linker',\n",
       " 'inserted',\n",
       " 'c-terminal',\n",
       " 'AroC',\n",
       " 'AroD',\n",
       " 'n-terminal',\n",
       " 'MenF',\n",
       " 'AroZ',\n",
       " 'c-terminal',\n",
       " '-gg',\n",
       " 'GGS',\n",
       " 'gg',\n",
       " 'n-terminal',\n",
       " 'fusion',\n",
       " 'follows',\n",
       " 'linker',\n",
       " 'inserted',\n",
       " 'aroC',\n",
       " 'aroD',\n",
       " 'menF',\n",
       " 'aroZ',\n",
       " 'stop',\n",
       " 'deleted',\n",
       " '-ggtggtggtggtagtggtggcggtggttcaggcggtggtggttcc-',\n",
       " 'detailed',\n",
       " 'fusion',\n",
       " 'transformation',\n",
       " 'CFT5-Derived Strains',\n",
       " 'transformation',\n",
       " 'CFT5-derived strains',\n",
       " 'electroporation',\n",
       " 'kv',\n",
       " 'f',\n",
       " 'electric',\n",
       " 'pulse',\n",
       " 'cm',\n",
       " 'cuvette',\n",
       " 'pulser',\n",
       " 'bio-rad',\n",
       " 'laboratory',\n",
       " 'hercules',\n",
       " 'ca',\n",
       " 'usa',\n",
       " 'CFT5',\n",
       " 'harboring',\n",
       " 'pza23-',\n",
       " 'ncmI',\n",
       " 'pza23-',\n",
       " 'ncamI',\n",
       " 'pza23-',\n",
       " 'nca',\n",
       " 'mi',\n",
       " 'pza23-upyc',\n",
       " 'pza23-zyc',\n",
       " 'pza23-dzyc',\n",
       " 'pza23-d',\n",
       " 'zyc',\n",
       " 'designated',\n",
       " 'CFT51a',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'CFT52a',\n",
       " 'CFT53a',\n",
       " 'CFT53b',\n",
       " 'CFT53c',\n",
       " 'CFT51c',\n",
       " 'harboring',\n",
       " 'psak-',\n",
       " 'tktA',\n",
       " 'designated',\n",
       " 'CFT51c',\n",
       " 'mediam9yp',\n",
       " 'MA',\n",
       " 'tube-scale',\n",
       " 'cultures',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'comprised',\n",
       " 'M9',\n",
       " 'sodium pyruvate',\n",
       " 'M9',\n",
       " 'liter',\n",
       " 'NaCl',\n",
       " 'Na 2 HPO 4',\n",
       " '12h',\n",
       " 'KH 2 PO 4',\n",
       " 'NH 4 Cl',\n",
       " 'MgSO 4',\n",
       " '7h',\n",
       " 'CaCl 2',\n",
       " '2h',\n",
       " 'FeSO 4',\n",
       " '7h',\n",
       " 'thiamine hydrochloride',\n",
       " 'l-tyrosine',\n",
       " 'L-tryptophan',\n",
       " 'l-phenylalanine',\n",
       " 'Tyr',\n",
       " 'Trp',\n",
       " 'atcc31882',\n",
       " 'auxotrophic',\n",
       " 'amino acids',\n",
       " 'CFT5',\n",
       " 'derivative',\n",
       " 'auxotrophic',\n",
       " 'Phe)',\n",
       " 'kanamycin',\n",
       " 'ph-',\n",
       " 'controlled tube-scale cultures',\n",
       " 'autoclaved',\n",
       " 'CaCO 3',\n",
       " 'dispersed',\n",
       " 'distilled',\n",
       " 'cultures',\n",
       " 'MA',\n",
       " 'batch-',\n",
       " 'cultures',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'liter',\n",
       " 'tryptone',\n",
       " 'NaCl',\n",
       " 'kanamycin',\n",
       " 'conditionsfor',\n",
       " 'MA',\n",
       " 'fermentation',\n",
       " 'metabolically',\n",
       " 'precultured',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'c',\n",
       " 'shaking',\n",
       " 'test tube',\n",
       " 'preculture',\n",
       " 'seeded',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'isopropyl -d-1-thiogalactopyranoside',\n",
       " 'test tube',\n",
       " 'optical',\n",
       " 'tube-scale',\n",
       " 'cultures',\n",
       " 'c',\n",
       " 'shaking',\n",
       " 'ph-',\n",
       " 'controlled tube-scale cultures',\n",
       " 'CaCO 3',\n",
       " 'cultures',\n",
       " 'seeding',\n",
       " 'batch-',\n",
       " 'cultures',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'working',\n",
       " 'MA',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'inoculated',\n",
       " 'preculture',\n",
       " 'adjusting',\n",
       " 'maintain',\n",
       " 'cultivation',\n",
       " 'HCl',\n",
       " 'NH 4 OH',\n",
       " 'automatically',\n",
       " 'dissolved',\n",
       " 'oxygen',\n",
       " 'maintained',\n",
       " 'ppm',\n",
       " 'automatically',\n",
       " 'controlling',\n",
       " 'agitation',\n",
       " 'speed',\n",
       " 'supplementing',\n",
       " 'supplementing',\n",
       " 'air',\n",
       " 'iptg',\n",
       " 'initiation',\n",
       " 'cultivation',\n",
       " 'analytical',\n",
       " 'methodscell',\n",
       " 'measuring',\n",
       " 'uvmini-1240',\n",
       " 'spectrophotometer',\n",
       " 'shimadzu',\n",
       " 'corporation',\n",
       " 'kyoto',\n",
       " 'japan',\n",
       " 'Glucose',\n",
       " 'prominence',\n",
       " 'hplc',\n",
       " 'shimadzu',\n",
       " 'equipped',\n",
       " 'shodex',\n",
       " 'sugar',\n",
       " 'ks-801',\n",
       " 'shodex',\n",
       " 'Water',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'refractive',\n",
       " 'index',\n",
       " 'detector',\n",
       " 'MA',\n",
       " 'hplc',\n",
       " 'equipped',\n",
       " 'scr-102h',\n",
       " 'shimadzu',\n",
       " 'p-Toluenesulfonic acid',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'conductivity',\n",
       " 'detector',\n",
       " 'PHB',\n",
       " 'PCA',\n",
       " 'catechol',\n",
       " 'ware',\n",
       " 'hplc',\n",
       " 'equipped',\n",
       " 'PBr',\n",
       " 'nacalai',\n",
       " 'tesque',\n",
       " 'dual-solvent',\n",
       " 'solvent',\n",
       " 'phosphate',\n",
       " 'solvent',\n",
       " 'methanol',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'gradient',\n",
       " 'initiated',\n",
       " 'shifted',\n",
       " 'shifted',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'uv',\n",
       " 'vi',\n",
       " 'detector']"
      ]
     },
     "execution_count": 83,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_word[9][667:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['AroC',\n",
       " '-menf',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'examined',\n",
       " 'linker',\n",
       " 'MA',\n",
       " 'AroC',\n",
       " '-menf',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'MA',\n",
       " 'linker',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'rigid',\n",
       " 'linker',\n",
       " 'supplementary',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'suitable',\n",
       " 'AroC',\n",
       " '-menf',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'supplementary',\n",
       " 'display',\n",
       " 'MA',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'AroC',\n",
       " '-menf',\n",
       " 'CFT53c',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'MA',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " '16-fold',\n",
       " 'CFT51b',\n",
       " 'express',\n",
       " 'unfused',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'MenF',\n",
       " 'fused',\n",
       " 'AroC',\n",
       " 'menf',\n",
       " 'fusion',\n",
       " 'western',\n",
       " 'blotting',\n",
       " 'supplementary',\n",
       " 'AroC',\n",
       " '-menf',\n",
       " 'fusion',\n",
       " 'placed',\n",
       " 'close',\n",
       " 'proximity',\n",
       " 'thereby',\n",
       " 'apparent',\n",
       " 'intermediate',\n",
       " 'chorismate',\n",
       " 'around',\n",
       " 'MenF',\n",
       " 'nonfusion',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'competing',\n",
       " 'branched',\n",
       " 'compound',\n",
       " 'channeling',\n",
       " 'fusion',\n",
       " 'arod-',\n",
       " 'AroZ',\n",
       " 'fusion',\n",
       " 'improvement',\n",
       " 'MA',\n",
       " 'noticed',\n",
       " 'CFT53b',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'overexpressed',\n",
       " 'separately',\n",
       " 'MA',\n",
       " 'drastically',\n",
       " 'CFT53a',\n",
       " 'overexpressing',\n",
       " 'AroD',\n",
       " 'sds-page',\n",
       " 'CFT53b',\n",
       " 'AroD',\n",
       " 'overexpressed',\n",
       " 'supplementary',\n",
       " 's6',\n",
       " 'explanation',\n",
       " 'fact',\n",
       " 'overexpression',\n",
       " 'AroD',\n",
       " 'shifted',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'shikimate',\n",
       " 'effectively',\n",
       " 'PCA',\n",
       " 'AroZ',\n",
       " 'AroY',\n",
       " 'CatA',\n",
       " 'distinctively',\n",
       " 'CFT53b',\n",
       " 'imbalance',\n",
       " 'MA',\n",
       " 'hand',\n",
       " 'sds-page',\n",
       " 'CFT53c',\n",
       " 'distinctively',\n",
       " 'overexpressed',\n",
       " 'AroD',\n",
       " 'fused',\n",
       " 'arod',\n",
       " 'AroZ',\n",
       " 'supplementary',\n",
       " 's6',\n",
       " 'arod',\n",
       " 'AroZ',\n",
       " 'greatly',\n",
       " 'AroD',\n",
       " 'imbalance',\n",
       " 'eliminated',\n",
       " 'MA',\n",
       " 'restored',\n",
       " 'titer',\n",
       " 'CFT53c',\n",
       " 'CFT53a',\n",
       " 'seems',\n",
       " 'channeling',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'worked',\n",
       " 'inactivation',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'AroD',\n",
       " 'homodimer',\n",
       " 'E. coli',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " 'dimerize',\n",
       " 'properly',\n",
       " 'fused',\n",
       " 'AroZ',\n",
       " 'endogenous',\n",
       " 'AroD',\n",
       " '3-dehydroquinate dehydratase',\n",
       " 'CFT53a',\n",
       " 'hypothesis',\n",
       " 'explain',\n",
       " 'MA',\n",
       " 'nearly',\n",
       " 'CFT53a',\n",
       " 'CFT53c',\n",
       " 'difficult',\n",
       " 'preserve',\n",
       " 'dimeric',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'channeling',\n",
       " 'involves',\n",
       " 'protein-level',\n",
       " 'fusion',\n",
       " 'transpeptidases',\n",
       " 'AroC',\n",
       " '-menf',\n",
       " 'fusion',\n",
       " 'MA',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'CFT51b',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'CFT51c',\n",
       " 'black',\n",
       " 'pac-',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'orange',\n",
       " 'purple',\n",
       " 'pac-',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'consuming',\n",
       " 'chorismate',\n",
       " 'yellow',\n",
       " 'band',\n",
       " 'flexible',\n",
       " 'ggggs',\n",
       " ') 3',\n",
       " 'linker',\n",
       " 'light',\n",
       " 'gray',\n",
       " 'circle',\n",
       " '5-O-(1-Carboxyvinyl)-3-phosphoshikimate',\n",
       " 'epsp',\n",
       " 'dark',\n",
       " 'gray',\n",
       " 'diamond',\n",
       " 'chorismate',\n",
       " 'square',\n",
       " 'isochorismate',\n",
       " 'yellow',\n",
       " 'star',\n",
       " 'MA',\n",
       " 'orange',\n",
       " 'pentagon',\n",
       " 'purple',\n",
       " 'triangle',\n",
       " 'byproduct',\n",
       " 'chorismate',\n",
       " 'fusion',\n",
       " 'compound',\n",
       " 'matsumoto',\n",
       " 'created',\n",
       " 'recombinant',\n",
       " 'E. coli',\n",
       " 'fusion',\n",
       " 'pyruvate-formate lyase',\n",
       " 'PFL',\n",
       " 'phosphate acetyltransferase',\n",
       " 'PTA',\n",
       " 'ligated',\n",
       " 'Staphylococcal',\n",
       " 'sortase A',\n",
       " 'acetate',\n",
       " 'PFL',\n",
       " 'PTA',\n",
       " 'separately',\n",
       " 'Sortase-A',\n",
       " '-mediated',\n",
       " 'fusion',\n",
       " 'channeling',\n",
       " 'hence',\n",
       " 'seems',\n",
       " 'AroZ',\n",
       " '-arod',\n",
       " 'fusion',\n",
       " 'ph-controlled',\n",
       " 'tube-scale',\n",
       " 'CaCO 3',\n",
       " 'attempted',\n",
       " 'MA',\n",
       " 'titer',\n",
       " 'controlling',\n",
       " 'culturing',\n",
       " 'bad',\n",
       " 'influence',\n",
       " 'bioproduction',\n",
       " 'E. coli',\n",
       " 'grows',\n",
       " 'environment',\n",
       " 'inside',\n",
       " 'external',\n",
       " 'ic',\n",
       " 'side',\n",
       " 'fall',\n",
       " 'intracellular',\n",
       " 'culturing',\n",
       " 'CFT51c',\n",
       " 'thompson',\n",
       " 'MA',\n",
       " 'E. coli',\n",
       " 'MA',\n",
       " 'titer',\n",
       " 'far',\n",
       " 'microbe',\n",
       " 'batch',\n",
       " 'ph-controlled',\n",
       " 'E. coli',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'MA',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'limit',\n",
       " 'non-ph-control',\n",
       " 'hypothesized',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'cultivation',\n",
       " 'illustrates',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'cultured',\n",
       " 'CaCO 3',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'meaning',\n",
       " '29-fold',\n",
       " 'CaCO 3',\n",
       " 'culturing',\n",
       " 'CFT51c',\n",
       " 'CaCO 3',\n",
       " 'maintained',\n",
       " 'profile',\n",
       " 'CFT51c',\n",
       " 'closed',\n",
       " 'circle',\n",
       " 'non-ph-controlled',\n",
       " 'cultivation',\n",
       " 'open',\n",
       " 'circle',\n",
       " 'ph-controlled',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'CFT51c',\n",
       " 'Glucose',\n",
       " 'consumption',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'arrow',\n",
       " 'CaCO 3',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'cultivationin',\n",
       " 'perform',\n",
       " 'cultivation',\n",
       " 'precisely',\n",
       " 'controlling',\n",
       " 'CFT51c',\n",
       " 'cultured',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'MA',\n",
       " 'consumption',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'cultivation',\n",
       " 'correspond',\n",
       " 'profile',\n",
       " 'ph-control',\n",
       " 'test tube',\n",
       " 'cultivation',\n",
       " 'CaCO 3',\n",
       " 'test tube',\n",
       " 'CaCO 3',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'adjustment',\n",
       " 'MA',\n",
       " 'stage',\n",
       " 'examine',\n",
       " 'dissolved',\n",
       " 'oxygen',\n",
       " 'handle',\n",
       " 'osmotic',\n",
       " 'pressure',\n",
       " 'MA',\n",
       " 'byproduct',\n",
       " 'accumulation',\n",
       " 'profile',\n",
       " 'CFT51c',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'CFT51c',\n",
       " 'Glucose',\n",
       " 'consumption',\n",
       " 'CFT51c',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " 'conclusionsin',\n",
       " 'summary',\n",
       " 'MA',\n",
       " 'aiming',\n",
       " 'select',\n",
       " 'suitable',\n",
       " 'MA',\n",
       " 'CFT5-derived strains',\n",
       " 'greatly',\n",
       " 'MA',\n",
       " 'compound',\n",
       " 'microorganism',\n",
       " 'optimal',\n",
       " 'perspective',\n",
       " 'industrial',\n",
       " 'compound',\n",
       " 'obtain',\n",
       " 'MA',\n",
       " 'fusion',\n",
       " 'hence',\n",
       " 'fusion',\n",
       " 'directed',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'shikimate',\n",
       " 'MA',\n",
       " 'MA',\n",
       " '44-fold',\n",
       " 'CFT51a',\n",
       " 'titer',\n",
       " 'MA',\n",
       " 'carbon',\n",
       " 'CFT51c',\n",
       " 'batch',\n",
       " 'cultures',\n",
       " 'flask',\n",
       " 'test tube',\n",
       " 'channeling',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'powerful',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'selectively',\n",
       " 'leading',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'difficult',\n",
       " 'apply',\n",
       " 'multimeric',\n",
       " 'titer',\n",
       " 'combine',\n",
       " 'disruption',\n",
       " 'metabolite',\n",
       " 'funneling',\n",
       " 'fusion',\n",
       " 'methodsstrains',\n",
       " 'constructionthe',\n",
       " 'E. coli',\n",
       " 'NovaBlue competent cells',\n",
       " 'novagen',\n",
       " 'cambridge',\n",
       " 'MA',\n",
       " 'usa',\n",
       " 'chain',\n",
       " 'quick',\n",
       " 'taq',\n",
       " 'toyobo',\n",
       " 'osaka',\n",
       " 'japan',\n",
       " 'custom',\n",
       " 'oligonucleotide',\n",
       " 'invitrogen',\n",
       " 'custom',\n",
       " 'oligos',\n",
       " 'thermo',\n",
       " 'fisher',\n",
       " 'tokyo',\n",
       " 'japan',\n",
       " 'supplementary',\n",
       " 'exogenous',\n",
       " 'pobA',\n",
       " 'P. putida',\n",
       " 'KT2440',\n",
       " 'aroY',\n",
       " 'K. pneumoniae',\n",
       " 'aroZ',\n",
       " 'Bacillus thuringiensis',\n",
       " 'invitrogen',\n",
       " 'geneart',\n",
       " 'service',\n",
       " 'thermo',\n",
       " 'fisher',\n",
       " 'tokyo',\n",
       " 'japan',\n",
       " 'exogenous',\n",
       " 'detailed',\n",
       " 'fusion',\n",
       " 'constructionthe',\n",
       " 'fusion',\n",
       " 'flexible',\n",
       " 'ggggs',\n",
       " ') 3',\n",
       " 'linker',\n",
       " 'inserted',\n",
       " 'c-terminal',\n",
       " 'AroC',\n",
       " 'AroD',\n",
       " 'n-terminal',\n",
       " 'MenF',\n",
       " 'AroZ',\n",
       " 'c-terminal',\n",
       " '-gg',\n",
       " 'GGS',\n",
       " 'gg',\n",
       " 'n-terminal',\n",
       " 'fusion',\n",
       " 'follows',\n",
       " 'linker',\n",
       " 'inserted',\n",
       " 'aroC',\n",
       " 'aroD',\n",
       " 'menF',\n",
       " 'aroZ',\n",
       " 'stop',\n",
       " 'deleted',\n",
       " '-ggtggtggtggtagtggtggcggtggttcaggcggtggtggttcc-',\n",
       " 'detailed',\n",
       " 'fusion',\n",
       " 'transformation',\n",
       " 'CFT5-Derived Strains',\n",
       " 'transformation',\n",
       " 'CFT5-derived strains',\n",
       " 'electroporation',\n",
       " 'kv',\n",
       " 'f',\n",
       " 'electric',\n",
       " 'pulse',\n",
       " 'cm',\n",
       " 'cuvette',\n",
       " 'pulser',\n",
       " 'bio-rad',\n",
       " 'laboratory',\n",
       " 'hercules',\n",
       " 'ca',\n",
       " 'usa',\n",
       " 'CFT5',\n",
       " 'harboring',\n",
       " 'pza23-',\n",
       " 'ncmI',\n",
       " 'pza23-',\n",
       " 'ncamI',\n",
       " 'pza23-',\n",
       " 'nca',\n",
       " 'mi',\n",
       " 'pza23-upyc',\n",
       " 'pza23-zyc',\n",
       " 'pza23-dzyc',\n",
       " 'pza23-d',\n",
       " 'zyc',\n",
       " 'designated',\n",
       " 'CFT51a',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'CFT52a',\n",
       " 'CFT53a',\n",
       " 'CFT53b',\n",
       " 'CFT53c',\n",
       " 'CFT51c',\n",
       " 'harboring',\n",
       " 'psak-',\n",
       " 'tktA',\n",
       " 'designated',\n",
       " 'CFT51c',\n",
       " 'mediam9yp',\n",
       " 'MA',\n",
       " 'tube-scale',\n",
       " 'cultures',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'comprised',\n",
       " 'M9',\n",
       " 'sodium pyruvate',\n",
       " 'M9',\n",
       " 'liter',\n",
       " 'NaCl',\n",
       " 'Na 2 HPO 4',\n",
       " '12h',\n",
       " 'KH 2 PO 4',\n",
       " 'NH 4 Cl',\n",
       " 'MgSO 4',\n",
       " '7h',\n",
       " 'CaCl 2',\n",
       " '2h',\n",
       " 'FeSO 4',\n",
       " '7h',\n",
       " 'thiamine hydrochloride',\n",
       " 'l-tyrosine',\n",
       " 'L-tryptophan',\n",
       " 'l-phenylalanine',\n",
       " 'Tyr',\n",
       " 'Trp',\n",
       " 'atcc31882',\n",
       " 'auxotrophic',\n",
       " 'amino acids',\n",
       " 'CFT5',\n",
       " 'derivative',\n",
       " 'auxotrophic',\n",
       " 'Phe)',\n",
       " 'kanamycin',\n",
       " 'ph-',\n",
       " 'controlled tube-scale cultures',\n",
       " 'autoclaved',\n",
       " 'CaCO 3',\n",
       " 'dispersed',\n",
       " 'distilled',\n",
       " 'cultures',\n",
       " 'MA',\n",
       " 'batch-',\n",
       " 'cultures',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'liter',\n",
       " 'tryptone',\n",
       " 'NaCl',\n",
       " 'kanamycin',\n",
       " 'conditionsfor',\n",
       " 'MA',\n",
       " 'fermentation',\n",
       " 'metabolically',\n",
       " 'precultured',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'c',\n",
       " 'shaking',\n",
       " 'test tube',\n",
       " 'preculture',\n",
       " 'seeded',\n",
       " 'M9',\n",
       " 'yp',\n",
       " 'isopropyl -d-1-thiogalactopyranoside',\n",
       " 'test tube',\n",
       " 'optical',\n",
       " 'tube-scale',\n",
       " 'cultures',\n",
       " 'c',\n",
       " 'shaking',\n",
       " 'ph-',\n",
       " 'controlled tube-scale cultures',\n",
       " 'CaCO 3',\n",
       " 'cultures',\n",
       " 'seeding',\n",
       " 'batch-',\n",
       " 'cultures',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'working',\n",
       " 'MA',\n",
       " 'jar',\n",
       " 'fermentor',\n",
       " 'inoculated',\n",
       " 'preculture',\n",
       " 'adjusting',\n",
       " 'maintain',\n",
       " 'cultivation',\n",
       " 'HCl',\n",
       " 'NH 4 OH',\n",
       " 'automatically',\n",
       " 'dissolved',\n",
       " 'oxygen',\n",
       " 'maintained',\n",
       " 'ppm',\n",
       " 'automatically',\n",
       " 'controlling',\n",
       " 'agitation',\n",
       " 'speed',\n",
       " 'supplementing',\n",
       " 'supplementing',\n",
       " 'air',\n",
       " 'iptg',\n",
       " 'initiation',\n",
       " 'cultivation',\n",
       " 'analytical',\n",
       " 'methodscell',\n",
       " 'measuring',\n",
       " 'uvmini-1240',\n",
       " 'spectrophotometer',\n",
       " 'shimadzu',\n",
       " 'corporation',\n",
       " 'kyoto',\n",
       " 'japan',\n",
       " 'Glucose',\n",
       " 'prominence',\n",
       " 'hplc',\n",
       " 'shimadzu',\n",
       " 'equipped',\n",
       " 'shodex',\n",
       " 'sugar',\n",
       " 'ks-801',\n",
       " 'shodex',\n",
       " 'Water',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'refractive',\n",
       " 'index',\n",
       " 'detector',\n",
       " 'MA',\n",
       " 'hplc',\n",
       " 'equipped',\n",
       " 'scr-102h',\n",
       " 'shimadzu',\n",
       " 'p-Toluenesulfonic acid',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'conductivity',\n",
       " 'detector',\n",
       " 'PHB',\n",
       " 'PCA',\n",
       " 'catechol',\n",
       " 'ware',\n",
       " 'hplc',\n",
       " 'equipped',\n",
       " 'PBr',\n",
       " 'nacalai',\n",
       " 'tesque',\n",
       " 'dual-solvent',\n",
       " 'solvent',\n",
       " 'phosphate',\n",
       " 'solvent',\n",
       " 'methanol',\n",
       " 'mobile',\n",
       " 'maintained',\n",
       " 'c',\n",
       " 'gradient',\n",
       " 'initiated',\n",
       " 'shifted',\n",
       " 'shifted',\n",
       " 'hplc',\n",
       " 'profile',\n",
       " 'monitored',\n",
       " 'uv',\n",
       " 'vi',\n",
       " 'detector']"
      ]
     },
     "execution_count": 84,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_word_v2[9][667:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'The global market for adipic acid, an important aliphatic dicarboxylic acid, is estimated to reach 8 billion by 2022. More than half of the total adipic acid production is from cyclohexane, and the remainder is from cyclohexene, uncoupled cyclohexanol/cyclohexanone (KA oil), and phenol. The production of nylon-66 fibers and engineering resins accounted for approximately 57% of the total amount of adipic acid consumed in 2016. Recently, microbial production of adipic acid has attracted attention as a solution to the exhaustion of finite fossil resources along with the increasing global demand of adipic acid. The production of bioadipic acid, wherein carbons are derived from a renewable feedstock, may solve these problems. Yu et al. prepared 639  34 g/L of adipic acid from 10 g/L glucose via an artificial pathway with acetyl-CoA and succinyl-CoA as starting units (ACSC1 pathway) in Escherichia coli. Cheong et al. created an E. coli strain producing adipic acid at higher amount by introducing another artificial pathway with same starting units of ACSC1 pathway (ACSC2 pathway). Adipyl-CoA, an intermediate of these two artificial pathways, occurs via three steps from succinyl-CoA and acetyl-CoA in the same way in these two pathways. In ACSC2 pathway, adipyl-CoA was converted to adipic acid by acyl-CoA thioesterases, whereas by phosphate butyryltransferase and butyryl kinase in ACSC1 pathway. This strain produced 2.5 g/L adipic acid from glycerol at a molar yield of 4.1%. However, these findings showed that these pathways are not suitable for large-scale production of adipic acid from renewable resources, such as glucose.In this study, we investigated the bioproduction of muconic acid (cis, cis-muconic acid (MA) using the gene-level fusion proteins method. In nature, MA is an intermediate in the degradation pathway of various aromatic compounds in some bacteria, including Pseudomonas putida and Klebsiella pneumoniae. Normally, adipic acid is easily synthesized from MA through double-bond hydrogenation. MA is produced through microbial catalysis and is successfully converted to adipic acid by chemical hydrogenation in the presence of suitable catalysts. MA is also used as a precursor in the production of terephthalic acid (TPA), the main monomer in the production of polyethylene terephthalate, an important component of industrial resins and thermoplastic synthetic fibers (Figure). After MA isomerization from cis, cis-muconic acid to trans, trans-muconic acid, it follows a DielsAlder reaction with ethylene and is further dehydrogenated to produce TPA. Given its versatility as a precursor, several laboratories investigated the fermentative production of MA using microbes. When using E. coli as a host, 3-dehydroshikimate (DHS) and chorismate (intermediates or derivatives of the shikimate pathway) are used as starting materials for MA production (Figure A). Therefore, increasing the activity of the shikimate pathway is an important strategy to enhance MA yield. The first reaction involved in the shikimate pathway is the stereospecific condensation of erythrose-4-phosphate (E4P) and phosphoenolpyruvate (PEP) to generate 3-deoxy-d-heptulosonate-7-phosphate (DAHP), catalyzed by DAHP synthase. DHS is synthesized via a 2-step reaction from DAHP. Chorismate, a key intermediate in the shikimate pathway, is synthesized downstream of DHS and can be converted to various aromatic compounds, including aromatic amino acids.Synthetic metabolic pathway for the production of chorismate derivatives, including MA, in E. coli CFT5-derived strains. The red X indicates deletion of the ptsH, ptsI, pykA, pykF, pheA, and tyrA genes. The blue arrows indicate the GalP/Glk system (GGS) revised by introducing galP and glk genes. The green arrows indicate the MA synthesis pathways introduced. The orange arrow indicates the conversion of MA to various products by chemical catalysis. GalP, galactose permease; Glk, glucokinase; G6P, glucose 6-phosphate; ATP, adenosine triphosphate; ADP, adenosine diphosphate; G6P, glucose-6-phosphate; E4P, erythrose-4-phosphate; PEP, phosphoenolpyruvate; Acetyl-CoA, acetyl-coenzyme A; AA, adipic acid; TPA, terephthalic acid.(A) Diagram of MA synthesis pathways in CFT5-derived strains. DQA, dehydroquinate; DHS, dehydroshikimate; EPSP, 5-O-(1-Carboxyvinyl)-3-phosphoshikimate; PHB, p-hydroxbenzoate; ICA, isochorismate; PCA, protocatechuate; red arrows, Pathway 1; blue arrows, Pathway 2; and green arrows, Pathway 3. (B) Diagram of MA synthesis gene cassette in each plasmid.Several synthetic pathways for MA production have been reported using E. coli as a host strain (Figure A). Lin et al. created an MA synthetic pathway via salicylate, which is converted to MA via a sequential reaction pathway involving catechol transformation in reactions catalyzed by salicylate 1-monoxygenase and catechol 1,2-dioxygenase (Figure A, Pathway 1). Sengupta et al. used another pathway for MA production via 4-hydroxybenzoate (PHB) and protocatechuate (PCA) intermediates. After hydroxylation of 4-hydroxybenzoate, a derivative of chorismate, the PCA product, was converted into MA via catechol by both PCA decarboxylase and catechol 1,2-dioxygenase (Figure A, Pathway 2). Niu et al. engineered E. coli by expressing the DHS dehydratase, PCA decarboxylase, and catechol 1,2-dioxygenase, and creating a branching pathway from the intermediate DHS in the shikimate pathway (Figure A, Pathway 3).Previously, we engineered a metabolic pathway in E. coli to increase the activity of the shikimate pathway by accumulating intracellular PEP and to produce chorismate derivatives in high yields. To prevent PEP consumption during glucose uptake, we generated a recombinant strain by replacing the endogenous phosphotransferase system (PTS) with the galactose permease (GalP)/glucokinase (Glk) system (GGS) (PTS  GGS + strain). The PTS consumes 1 mol of PEP for 1 mol of glucose while the GGS needs ATP as the cofactor for glucose phosphorylation without PEP consumption. Two genes encoding pyruvate kinase (pykA, pykF) from the PTS  GGS + strain were disrupted to prevent the introduction of carbon-based compounds to the TCA cycle. As pyruvate kinase catalyzes the conversion of PEP to pyruvate in the glycolytic pathway, the disruption of the genes encoding these enzymes will deprive the engineered E. coli of pyruvate. In our platform strains, the lack of pyruvate was compensated by introducing an artificial pathway that produces chorismate derivatives, such as salicylate, p-hydroxybenzoate, 3-hydroxybenzoate, 2-aminobenzoate, p-aminobenzoate, phenol, and MA along with pyruvate. For example, using the salicylate biosynthesis pathway, the chorismate is converted to isochorismate by isochorismate synthase (MenF). Then, the isochorismate is cleaved by isochorismate pyruvate lyase to release pyruvate and salicylate (Figure).Fusion protein enzymes are attracting attention as an efficient strategy to selectively increase the carbon flux to the target pathway while decreasing that to the competitive pathway without completely inactivating the genes. One strategy used to express fusion proteins is to bind the enzymes after translation by transpeptidase. Matsumoto et al. increased acetate production in recombinant E. coli synthesizing a fusion protein of pyruvate-formate lyase and phosphate acetyltransferase ligated by Staphylococcal sortase A. Engineering of the fusion protein at the gene level is a simpler method of protein expression. Wang et al. increased the production of -farnesene in recombinant E. coli by expressing a fusion protein of farnesyl diphosphate synthase and -farnesene synthase. In these studies, the metabolic channeling effectively increased the target compound production. Fusion enzymes behave in the same way as the native multidomain enzymes (MDE). MDEs catalyze a series of reactions while preventing the diffusion of intermediate metabolites, thereby improving the efficiency of a metabolic reaction cascade by generating a substrate channeling from an active site to another. As the MDE, reaction intermediates produced by fusion enzymes do not diffuse, increasing the efficiency of the subsequent reaction. Hence, the developing of the artificial metabolic channeling by fusion proteins is a powerful tool for increasing the carbon flux to a specific pathway.This study investigates the effect of overexpressing fusion proteins in the shikimate pathway using the gene-level fusion method on MA yield. Hence, the MA synthesis pathway was introduced into a strain with increased shikimate pathway flux, as described above. After selecting the optimal MA synthesis pathways from three candidates, the production of MA by the selected pathways was increased by overexpressing chorismate synthase (AroC). Finally, the effect of the gene-level fusion method was investigated. The production of MA was further increased by overexpressing gene-level fusion proteins of AroC and MenF.Results and DiscussionSelection of the Candidate Pathways for the MA ProductionSeveral previous reports have described methods for increasing MA production, and among them is the introduction of several MA synthesis pathways. In this study, we tested three pathways for MA production using the CFT5 strain. Two of the three pathways start from chorismate, a key compound in the shikimate pathway. The Pathway 1 produces MA via salicylate, whereas the Pathway 2 produces MA via PHB (Figure A). The Pathway 3 starts with DHS, another intermediate in the shikimate pathway, and produces MA via PCA (Figure A). In all three pathways, the catechol is produced from salicylate or PCA, followed by cleavage of the catechol benzene ring to form MA. In Pathway 1, the order of the genes was tested, and choose suited to CFT5 (Supporting Information Table S3).We engineered three E. coli transformants, CFT51a, CFT52a, and CFT53a (carrying Pathway 1, Pathway 2, and Pathway 3, respectively). Figure shows the amount of MA produced by each transformant after 72 h cultivation. CFT51a and CFT53a produced 1.00  0.11 and 1.34  0.07 g/L of MA, respectively. Although CFT52a produced 0.40  0.02 g/L of MA after 24 h cultivation, the amount produced decreased to the detection limit after 72 h cultivation. No difference in cell growth was observed between strains, and all strains consumed glucose within 48 h of cultivation (Supplementary Figure S1, Figure S2). The intermediates of the MA synthesis pathway (salicylate [Pathway 1], PCA [Pathway 2], and catechol [Pathways 1, 2]) were not detected in the culture supernatants of CFT51a or CFT52a after 72 h of cultivation.MA production titer in CFT5-derived strains. Red, blue, and green bars indicate MA production via Pathway 1, Pathway 2, and Pathway3, respectively. n.c. is empty vector control (CFT5 containing pZA23MCS). The data are presented as the average of three independent experiments, and error bars indicate the standard error.Although high PHB production was previously observed in CFT5-derived strain (1820 mg/L after 72 h of cultivation), the CFT52a produced only a small amount of MA. CFT52a cultures contained no detectable PCA or catechol, and only a small amount of PHB was detected (0.108  0.00 g/L after 72 h of cultivation). The Pathways 2 and 3 both include the reactions catalyzed by AroY and CatA. Because CFT53a, carrying Pathway3, produced the highest amount of MA among the three strains, these reactions were considered as optimum in CFT5-derived strains. In the SDS-PAGE analysis of CFT52a strain, the significant overexpression of PobA and UbiC was not detected (Supplementary Figure S3). In order to produce MA at high yield in pathway 2, it would be necessary to solve these expression problems. Hence, we conclude that the Pathway 2 is not suitable for MA production in CFT5-derived strains.Optimizing the Production Pathway of the MATo further increase the MA production we approached the intracellular availability of the compound at the branch point between the endogenous shikimate and synthetic MA pathway. The branch point compounds are chorismate in CFT51a (Pathway 1) and DHS in CFT53a (Pathway 3). Since no accumulation of intermediates was detected in the culture supernatants of CFT51a or CFT53a, the Pathways 1 and 3 were not bottlenecks in these strains. Improved availability of chorismate or DHS is essential for further increasing MA production. Overexpression of AroC in CFT51a and AroD in CFT53a was used to increase the carbon flux to chorismate and DHS respectively. Figure shows the amount of produced MA by each transformant after 72 h cultivation. CFT51b produced 1.59  0.10 g/L of MA, which is 1.6-fold higher than that produced by CFT51a. However, the amount of MA produced by CFT53b was below the quantification limit. Lutke-Eversloh et al. investigated the key enzymes for tyrosine production, reporting that overexpression of AroC is an effective way to increase the production of compounds in the shikimate pathway. This report is consistent with our finding that showed that the MA production was effectively increased by the overexpression of AroC. However, contrary to our expectations, the overexpression of AroD had a negative effect on the MA production in CFT53b (Pathway 3). The overexpression of AroD might increase the carbon flux downstream of the shikimate pathway rather than that to PCA, the precursor of MA.Expression of Fusion Proteins To Further Increase MA YieldAs already stated, the metabolic channeling is a powerful strategy to enhance the carbon flux to specific compounds of interest, by promoting the efficient transfer of an intermediate product from an enzyme to another. Hence, this strategy was applied in our study for the MA biosynthesis aiming to increase its production (Figure). In Pathways 1 and 3, the two enzymes at the branch point between the shikimate and MA biosynthesis pathways were overexpressed as a fusion protein, AroC and MenF (AroC-MenF) in Pathway 1 and AroD and AroZ (AroD-AroZ) in Pathway 3 (Figure A). First we examined the effect of linker type on MA production. In AroC-MenF and AroD-AroZ, MA production was compared in the case of using without linker, flexible linker, and rigid linker (Supplementary Figure S4), respectively, and flexible linker was most suitable for AroC-MenF and AroD-AroZ (Supplementary Table S4). Figure displays the amount of MA produced by each transformant after 72 h cultivation. CFT51c (AroC-MenF) and CFT53c (AroD-AroZ) produced 3.45  0.04 and 1.20  0.10 g/L of MA, respectively. The production of MA in CFT51c was 2.16-fold higher than that produced by CFT51b, which expresses the unfused AroC and MenF proteins. We confirmed that CFT51b and CFT51c expressed MenF and fused AroC/MenF fusion protein respectively by Western blotting analysis (Supplementary Figure S5). In the AroC-MenF fusion protein, the two enzymes are placed in close proximity, thereby increasing the apparent concentration of the intermediate (chorismate) around MenF relative to that of the nonfusion enzyme, resulting in higher carbon flux to the MA biosynthesis pathway (Figure). If the competing pathway is branched from the synthetic pathway of the target compound, metabolic channeling by the fusion protein has the potential to significantly increase production. However, for the AroD-AroZ fusion protein, no improvement of the MA production was noticed. For CFT53b, in which AroD and AroZ were overexpressed separately, the MA production was drastically decreased than that of CFT53a (not overexpressing AroD) (Figure). In SDS-PAGE analysis of CFT53b, free AroD was significantly overexpressed in this strain (Supplementary Figure S6). One explanation could be related to the fact that the overexpression of AroD shifted the carbon flux in the shikimate pathway more effectively than to PCA. The expression of AroZ, AroY and CatA were not distinctively confirmed in CFT53b. The imbalance of expression levels of these enzymes might have a negative effect on MA production. On the other hand, in SDS-PAGE analysis of CFT53c, it was not distinctively confirmed to be overexpressed the free AroD and the fused AroD/AroZ (Supplementary Figure S6). Since the expression level of the AroD/AroZ was greatly reduced as compared with AroD, the imbalance would have been eliminated, and the production of MA was restored. However, the production titer of CFT53c was lower than CFT53a. It seems that metabolic channeling between AroD and AroZ would not have worked well because of the inactivation of the AroD domain in the fusion protein. The native AroD protein is expressed as a homodimer in E. coli. In CFT53c, the AroD may not dimerize properly, because it is fused to AroZ. In this case, only the endogenous AroD would function as a 3-dehydroquinate dehydratase, as in CFT53a. This hypothesis may explain why MA production was nearly the same in CFT53a and CFT53c. Thus, it is difficult to preserve the function of proteins with more complex structures, such as dimeric AroD, as fusion proteins. Another strategy for metabolic channeling involves the protein-level fusion after translation using transpeptidases.Effect of AroC-MenF fusion protein expression on MA production. (A) Diagram of the shikimate pathway in CFT51b. (B) Diagram of shikimate pathway in CFT51c. Black and red Pac-man shapes indicate the AroC and MenF, respectively. Orange and purple Pac-man shapes indicate the enzymes consuming chorismate as substrate. Yellow band indicates the flexible (GGGGS) 3 linker. Light gray circles, 5-O-(1-Carboxyvinyl)-3-phosphoshikimate (EPSP); dark gray diamonds, chorismate; red squares, isochorismate; and yellow stars, MA. Orange pentagons and purple triangles indicate the byproducts derived from chorismate.The protein fusion method has been used to increase the production of target compounds, as well. Matsumoto et al. created recombinant E. coli expressing a fusion protein of pyruvate-formate lyase (PFL) and phosphate acetyltransferase (PTA) ligated by Staphylococcal sortase A. This strain produced a significantly higher amount of acetate as compared to a control strain expressing PFL and PTA separately. The Sortase-A-mediated protein fusion method was effective for in vivo metabolic channeling. Hence, this approach seems to be effective to produce AroZ-AroD fusion proteins.pH-Controlled Tube-Scale Culture Using CaCO 3We attempted increasing the MA production titer by controlling pH. The decrease in pH during culturing often has a bad influence on bioproduction. Although E. coli grows at pH 58, the environment inside the bacteria also changes due to the change in external pH. On the acidic side, when the pH falls below 6, the intracellular pH decreases. After culturing CFT51c for 72 h, the pH in the medium decreased to 5.51  0.0. Thompson et al. introduced multiple MA synthesis pathways into E. coli that produced 3153  149 mg/L of MA. This is the highest titer so far using microbes in batch culture, without pH-controlled conditions. The E. coli transformant constructed in the present study also produced 3.45 g/L of MA. According to these results, it is possible that the MA produced in CFT51c reached the limit in non-pH-control. Thus, we hypothesized that further MA production can be increased by pH control. To control the pH during cultivation, CaCO 3 (10 g/L) was added to the culture medium after 24 h of cultivation. Figure C illustrates the amount of MA produced by CFT51c cultured in a medium containing CaCO 3. CFT51c produced 4.45  0.12 g/L of MA after 72 h of cultivation, meaning a 1.29-fold increase over that without CaCO 3, and the maximum production rate reached 2.77 g/L/day (24 h). After culturing CFT51c for 72 h with CaCO 3, the pH in the medium was maintained at 7.05  1.4.Culture profiles of CFT51c. Red closed circles, non-pH-controlled cultivation; red open circles, pH-controlled cultivation (CaCO 3 added). (A) Bacterial cell growth (CFT51c). (B) Glucose consumption by CFT51c. (C) MA production by CFT51c. Blue arrows indicate when CaCO 3 was added in to the culture medium. The data are presented as the average of three independent experiments, and error bars indicate the standard error.Jar Fermentor CultivationIn order to perform cultivation more precisely while controlling the pH, CFT51c was cultured with a jar fermentor. Figure indicates the cell growth, MA production, and glucose consumption in jar fermentor cultivation. CFT51c produced 4.55 g/L of MA after 72 h of cultivation, and maximum production rate was 2.20 g/L/day at 48 h cultivation. These results correspond with the culture profiles of pH-control test tube cultivation with CaCO 3. The results of test tube culture (with CaCO 3) and jar fermentor culture indicate that pH adjustment is important for MA production at this stage. In order to further increase the production, it will be necessary to examine the appropriate dissolved oxygen concentration and to handle the increase of osmotic pressure due to MA and byproducts accumulation.Culture profiles of CFT51c with jar fermentor. (A) Bacterial cell growth (CFT51c). (B) Glucose consumption by CFT51c. (C) MA production by CFT51c.ConclusionsIn summary, three MA synthesis pathways were compared, aiming to select the most suitable for MA production in CFT5-derived strains. The three initial strains were greatly different from each other in terms of MA production. Several possible pathways are available to produce a compound of interest by using microorganisms. However, as our results showed, the selection of the optimal pathway among the available ones is essential in the perspective of the industrial applications that need the optimization of the yield of a target compound.To obtain a high yield of MA, we applied the fusion protein method. Hence, the expressed fusion protein directed the carbon flux to the MA synthesis in the shikimate pathway, resulting in a production of MA of 4.45 g/L with a maximum production rate of 2.77 g/L/day. The MA yield from glucose was 207 mg/g, which is an increase of 4.44-fold as compared to that of CFT51a, used as a control strain. The production titer and rate of MA production using glucose as the carbon source in CFT51c were higher than those in all previous studies using batch cultures in flasks or test tubes. This result suggests that metabolic channeling using the gene-level fusion proteins is a powerful tool to direct the carbon flux to the specific pathways selectively leading to the desired product. However, the gene-level fusion method is difficult to apply to proteins with more complex structures, such as multimeric proteins. To further increase the production titer and yield would require to combine multiple approaches, such as gene disruption, metabolite funneling, and fusion proteins.MethodsStrains and Plasmid ConstructionThe strains and plasmids used in this study are listed in Table. E. coli NovaBlue competent cells (Novagen, Cambridge, MA, USA) were used for gene cloning. Polymerase chain reaction was performed using Quick Taq HS (TOYOBO, Osaka, Japan). Custom DNA oligonucleotide primers were synthesized by Invitrogen Custom DNA Oligos (Thermo Fisher Scientific, Tokyo, Japan) (Supplementary Table S2). Codon optimized exogenous genes fragments (pobA from P. putida KT2440, aroY from K. pneumoniae, and aroZ from Bacillus thuringiensis) were synthesized by the Invitrogen GeneArt Gene Synthesis service (Thermo Fisher Scientific, Tokyo, Japan). Sequences of primers and exogenous genes fragments and detailed plasmid construction methods are described in Supporting Information.Fusion Protein ConstructionThe two proteins were expressed as one fusion protein using the following method. A flexible (GGGGS) 3 linker was inserted between the C-terminal of the upstream protein (AroC or AroD) and the N-terminal of the downstream protein (MenF or AroZ), i.e., (upstream protein C-terminal)-GGGGS\\xadGGGGS\\xadGGGGS-(downstream protein N-terminal). A plasmid expressing the fusion protein was constructed as follows. The linker sequence was inserted between the 3 end of the upstream gene (aroC or aroD) and the 5 end of the downstream gene (menF or aroZ), and the stop codon of the upstream gene was deleted, i.e., (3end of upstream gene)-GGTGGT\\xadGGTGGT\\xadAGTGGT\\xadGGCGGT\\xadGGTTCA\\xadGGCGGT\\xadGGTG\\xadGTTCC-(5 end of downstream gene). The detailed method used to construct plasmids expressing the fusion proteins is described in the Supporting Information.Transformation of CFT5-Derived StrainsThe transformation of CFT5-derived strains was carried out by electroporation with a 1350 kV, 600 , 10 F electric pulse in a 0.1 cm cuvette using a Gene Pulser (Bio-Rad Laboratories, Hercules, CA, USA).CFT5 harboring pZA23-ncmI, pZA23-ncamI, pZA23-nca/mI, pZA23-UpYc, pZA23-ZYc, pZA23-DZYc, and pZA23-D/ZYc were designated CFT51a, CFT51b, CFT51c, CFT52a, CFT53a, CFT53b, and CFT53c, respectively. CFT51c harboring pSAK-tktA was designated CFT51ct.MediaM9YP medium was used for MA production in 5 mL test tube-scale cultures. M9YP medium comprised M9 minimal medium supplemented with 0.5% yeast extract and 10 mM sodium pyruvate. M9 minimal medium contains (per liter) 20 g glucose, 0.5 g NaCl, 17.1 g Na 2 HPO 4 12H 2 O, 3 g KH 2 PO 4, 1 g NH 4 Cl, 246 mg MgSO 4 7H 2 O, 14.7 mg CaCl 2 2H 2 O, 2.78 mg FeSO 4 7H 2 O, 10 mg thiamine hydrochloride, 40 mg l-tyrosine, 40 mg L-tryptophan, and 100 mg l-phenylalanine (Tyr and Trp were included because ATCC31882 is auxotrophic for these amino acids, and the CFT5 derivative strains are auxotrophic for Phe). When needed, kanamycin was added to the initial medium to final concentrations of 20 mg/L. In pH-controlled tube-scale cultures, autoclaved 100 g/L CaCO 3 (dispersed in distilled water) was added to the cultures to a final concentration of 10 g/L. LB medium was used for MA production in batch-scale cultures in a 2.0 L jar fermentor. LB medium contains (per liter) 20 g glucose, 10 g tryptone, 5 g yeast extract, 5 g NaCl, 20 mg kanamycin.Culture ConditionsFor MA fermentation from glucose, metabolically engineered strains were precultured in 4 mL M9YP medium for 1 day at 37 C with shaking at 220 rpm in a 15 mL test tube. Each preculture medium was seeded to 5 mL M9YP medium (supplemented with 0.1 mM isopropyl -d-1-thiogalactopyranoside) in a 15 mL test tube at an initial optical density (OD 600) of 0.1. The tube-scale cultures were incubated at 37 C with shaking at 220 rpm. In the pH-controlled tube-scale cultures, CaCO 3 was added to the cultures at 24 h after seeding. Batch-scale cultures were performed in a 2.0 L jar fermentor with a 400 mL working volume. LB medium was used for MA production at this scale.The 400 mL of culture medium in the jar fermentor was inoculated with preculture while adjusting the initial OD to 0.05. In order to maintain the pH at 7.0 during cultivation, 1 M HCl and 7% NH 4 OH was automatically added to the medium. The dissolved oxygen (DO) was maintained at over 0.69 ppm by automatically controlling the agitation speed from 200 to 600 rpm and supplementing and supplementing with air at 400 mL/min. In this experiment, 0.1 mM IPTG was added to the medium at 3 h after the initiation of cultivation.Analytical MethodsCell growth was determined by measuring the OD at 600 nm on a UVmini-1240 spectrophotometer (Shimadzu Corporation, Kyoto, Japan). Glucose was analyzed using a Prominence HPLC System (Shimadzu) equipped with a Shodex SUGAR KS-801 column (6 m, 300 mm  8.0 mm, L  I.D., Shodex). Water was used as the mobile phase with a flow rate of 0.8 mL/min, and the column was maintained at 50 C. The HPLC profile was monitored using a refractive index detector.MA was analyzed using an HPLC equipped with a SCR-102H column (7 m, 8.0 mm  300 mm, I.D.  L, respectively, Shimadzu). p-Toluenesulfonic acid (5 mM) was used as the mobile phase with a flow rate of 2.0 mL/min, and the column was maintained at 40 C. The HPLC profile was monitored using a conductivity detector.PHB, PCA, and catechol ware analyzed using an HPLC equipped with a PBr column (5 m, 4.6 mm  250 mm, I.D.  L, respectively, Nacalai Tesque). A dual-solvent system was used. Solvent A was 0.2% phosphate buffer and solvent B was methanol. The flow rate of the mobile phase was 1.0 mL/min, and the column was maintained at 40 C. The gradient was initiated as an 80:20 mixture of A and B (015 min), shifted to a 50:50 mixture of A and B (1520 min), and subsequently shifted to an 80:20 mixture of A and B (2025 min). The HPLC profile was monitored using a UVvis detector at 240 nm.'"
      ]
     },
     "execution_count": 85,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "jsonfiles[corpus[9]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{') 3',\n",
       " '. DHS',\n",
       " '2-aminobenzoate',\n",
       " '3-dehydroquinate dehydratase',\n",
       " '3-dehydroshikimate',\n",
       " '3-deoxy-d-heptulosonate-7-phosphate',\n",
       " '3-hydroxybenzoate',\n",
       " '4-hydroxybenzoate',\n",
       " '5-O-(1-Carboxyvinyl)-3-phosphoshikimate',\n",
       " 'AA',\n",
       " 'ACSC1',\n",
       " 'ACSC2',\n",
       " 'ADP',\n",
       " 'ATP',\n",
       " 'Acetyl-CoA',\n",
       " 'Adipyl-CoA',\n",
       " 'AroC',\n",
       " 'AroD',\n",
       " 'AroY',\n",
       " 'AroZ',\n",
       " 'Bacillus thuringiensis',\n",
       " 'CFT5',\n",
       " 'CFT5 strain',\n",
       " 'CFT5-Derived Strains',\n",
       " 'CFT5-derived',\n",
       " 'CFT5-derived strains',\n",
       " 'CFT51a',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'CFT52a',\n",
       " 'CFT52a cultures',\n",
       " 'CFT53a',\n",
       " 'CFT53b',\n",
       " 'CFT53c',\n",
       " 'CaCO',\n",
       " 'CaCO 3',\n",
       " 'CaCl 2',\n",
       " 'CatA',\n",
       " 'Chorismate',\n",
       " 'DAHP',\n",
       " 'DAHP synthase',\n",
       " 'DHS',\n",
       " 'DHS dehydratase',\n",
       " 'E.',\n",
       " 'E. coli',\n",
       " 'E4P',\n",
       " 'Escherichia coli',\n",
       " 'FeSO 4',\n",
       " 'G6P',\n",
       " 'GGS',\n",
       " 'GalP',\n",
       " 'Glk',\n",
       " 'Glucose',\n",
       " 'HCl',\n",
       " 'K. pneumoniae',\n",
       " 'KH 2 PO 4',\n",
       " 'KT2440',\n",
       " 'L-tryptophan',\n",
       " 'M9',\n",
       " 'MA',\n",
       " 'MenF',\n",
       " 'MgSO 4',\n",
       " 'NH 4 Cl',\n",
       " 'NH 4 OH',\n",
       " 'Na 2 HPO 4',\n",
       " 'NaCl',\n",
       " 'NovaBlue competent cells',\n",
       " 'P. putida',\n",
       " 'PBr',\n",
       " 'PCA',\n",
       " 'PCA decarboxylase',\n",
       " 'PEP',\n",
       " 'PFL',\n",
       " 'PHB',\n",
       " 'PTA',\n",
       " 'PTS',\n",
       " 'Pac',\n",
       " 'Pathway3',\n",
       " 'Phe)',\n",
       " 'PobA',\n",
       " 'Sortase-A',\n",
       " 'Staphylococcal',\n",
       " 'TPA',\n",
       " 'Trp',\n",
       " 'Tyr',\n",
       " 'UbiC',\n",
       " 'Water',\n",
       " 'acetate',\n",
       " 'acetyl-CoA',\n",
       " 'acetyl-coenzyme A',\n",
       " 'acid',\n",
       " 'acyl-CoA thioesterases',\n",
       " 'adenosine diphosphate',\n",
       " 'adenosine triphosphate',\n",
       " 'adipic acid',\n",
       " 'adipyl-CoA',\n",
       " 'aliphatic dicarboxylic acid',\n",
       " 'amino acids',\n",
       " 'aroC',\n",
       " 'aroD',\n",
       " 'aroY',\n",
       " 'aroZ',\n",
       " 'aromatic',\n",
       " 'aromatic amino acids.',\n",
       " 'aromatic compounds',\n",
       " 'butyryl kinase',\n",
       " 'carbon-based',\n",
       " 'catechol',\n",
       " 'catechol 1,2-dioxygenase',\n",
       " 'catechol benzene ring',\n",
       " 'chorismate',\n",
       " 'chorismate derivatives',\n",
       " 'chorismate synthase',\n",
       " 'cis, cis-muconic acid',\n",
       " 'coli',\n",
       " 'controlled tube-scale cultures',\n",
       " 'created',\n",
       " 'cultures',\n",
       " 'cyclohexane',\n",
       " 'cyclohexanol/cyclohexanone',\n",
       " 'cyclohexene',\n",
       " 'dehydroquinate',\n",
       " 'dehydroshikimate',\n",
       " 'engineered E. coli',\n",
       " 'erythrose-4-phosphate',\n",
       " 'ethylene',\n",
       " 'farnesyl diphosphate synthase',\n",
       " 'galP',\n",
       " 'galactose permease',\n",
       " 'glk',\n",
       " 'glucokinase',\n",
       " 'glucose',\n",
       " 'glucose 6-phosphate',\n",
       " 'glucose-6-phosphate',\n",
       " 'glycerol',\n",
       " 'isochorismate',\n",
       " 'isochorismate pyruvate lyase',\n",
       " 'isochorismate synthase',\n",
       " 'isopropyl -d-1-thiogalactopyranoside',\n",
       " 'kanamycin',\n",
       " 'l-phenylalanine',\n",
       " 'l-tyrosine',\n",
       " 'man',\n",
       " 'menF',\n",
       " 'methanol',\n",
       " 'muconic acid',\n",
       " 'nca',\n",
       " 'ncamI',\n",
       " 'ncmI',\n",
       " 'oligonucleotide',\n",
       " 'oxygen',\n",
       " 'p-Toluenesulfonic acid',\n",
       " 'p-aminobenzoate',\n",
       " 'p-hydroxbenzoate',\n",
       " 'p-hydroxybenzoate',\n",
       " 'pheA',\n",
       " 'phenol',\n",
       " 'phenol.',\n",
       " 'phosphate',\n",
       " 'phosphate acetyltransferase',\n",
       " 'phosphate butyryltransferase',\n",
       " 'phosphoenolpyruvate',\n",
       " 'phosphotransferase',\n",
       " 'pobA',\n",
       " 'polyethylene terephthalate',\n",
       " 'protocatechuate',\n",
       " 'ptsH',\n",
       " 'ptsI',\n",
       " 'putida',\n",
       " 'pykA',\n",
       " 'pykF',\n",
       " 'pyruvate',\n",
       " 'pyruvate kinase',\n",
       " 'pyruvate-formate lyase',\n",
       " 'salicylate',\n",
       " 'salicylate 1-monoxygenase',\n",
       " 'scale',\n",
       " 'shikimate',\n",
       " 'sodium pyruvate',\n",
       " 'sortase A',\n",
       " 'succinyl-CoA',\n",
       " 'terephthalic acid',\n",
       " 'test tube',\n",
       " 'thiamine hydrochloride',\n",
       " 'tktA',\n",
       " 'trans, trans-muconic acid',\n",
       " 'transpeptidase',\n",
       " 'transpeptidases',\n",
       " 'tryptone',\n",
       " 'tube',\n",
       " 'tube-scale',\n",
       " 'tyrA',\n",
       " 'tyrosine',\n",
       " '-farnesene',\n",
       " '-farnesene synthase'}"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "path2entity[corpus[9]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "metadata": {},
   "outputs": [],
   "source": [
    "from src.data.load_data import *\n",
    "from src.data import clean\n",
    "import pickle\n",
    "\n",
    "from sklearn.feature_extraction.text import TfidfTransformer\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.pipeline import Pipeline\n",
    "\n",
    "def load_json_ner(json_input_path, ner_input_path, contain_abstract):\n",
    "    json_files = load_json_files(json_input_path, contain_abstract)\n",
    "    path2entity = load_ner_entities(ner_input_path, json_files.keys())\n",
    "\n",
    "    return json_files, path2entity\n",
    "\n",
    "def create_data_words(json_files, path2entity, q):\n",
    "    article_sq = []\n",
    "    for k in json_files:\n",
    "        if k not in path2entity:\n",
    "            path2entity[k] = set()\n",
    "\n",
    "    data_words = []\n",
    "    for k in json_files.keys():\n",
    "        article_sq.append(k)\n",
    "        data_words.append(clean.NERCleanText(k, json_files[k], path2entity))\n",
    "\n",
    "    if q > 0:\n",
    "        corpus = [' '.join(d) for d in data_words]\n",
    "        tfidf = TfidfTransformer()\n",
    "        vocabulary = list(set(' '.join(corpus).split()))\n",
    "        pipe = Pipeline([('count', CountVectorizer(vocabulary=vocabulary)),('tfid', TfidfTransformer())]).fit(corpus)\n",
    "        ser = pd.Series(index = vocabulary, data = pipe['tfid'].idf_)\n",
    "\n",
    "        # create stop words list \n",
    "        stops = ser[ser<ser.quantile(q)].sort_values().index.tolist()\n",
    "\n",
    "        #update data_words \n",
    "        data_words_cleaned = []\n",
    "        for d in data_words:\n",
    "            data_words_cleaned.append([w for w in d if w not in stops])\n",
    "        data_words = data_words_cleaned\n",
    "\n",
    "    return data_words, article_sq\n",
    "    \n",
    "def save_data_words(json_input_path, ner_input_path, contain_abstract, q, output_path_data_words, output_path_a_sq):\n",
    "    json_files, path2entity = load_json_ner(json_input_path, ner_input_path, contain_abstract)\n",
    "    data_words, a_sq = create_data_words(json_files, path2entity, q)\n",
    "    return data_words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 158,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_params(fp):\n",
    "    \"\"\"\n",
    "    Load params from json file \n",
    "    \"\"\"\n",
    "    with open(fp) as fh:\n",
    "        param = json.load(fh)\n",
    "\n",
    "    return param"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 159,
   "metadata": {},
   "outputs": [],
   "source": [
    "params = load_params('config/data_words.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 160,
   "metadata": {},
   "outputs": [],
   "source": [
    "jsonfiles, path2entity = load_json_ner(params['json_input_path'], params['ner_input_path'], False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [],
   "source": [
    "params = load_params('config/data_words.json')\n",
    "data_words = save_data_words(**params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_words2 = save_data_words(**params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 74,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_words==data_words2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Topic Terms & Dominant topic counts by documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>circuit, input, output, network, gate, behavior, device, strand, simulation, module, algorithm, noise, distribution, population, degradation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>IPTG, GFP, rb, biosensor, mrna, repression, sensor, inducer, LB, strains, supplementary, strength, lac, riboswitch, regulator, RBS, cultures, inducible, constitutive, arabinose</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>cassette, bp, cluster, integration, deletion, terminator, recombination, CRISPR, chromosome, marker, clone, cerevisiae, locus, transformation, DNA, kb, assembled, ligation, homologous, sgrna, editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>light, ion, membrane, ligand, peptide, fusion, surface, activation, affinity, intensity, switch, receptor, ed, image, antibody, residue, scaffold</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>compound, biosynthesis, tRNA, flux, titer, carbon, metabolite, fermentation, fatty, residue, precursor, mass, synthase</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                Terms per Topic\n",
       "Topic1                                                             circuit, input, output, network, gate, behavior, device, strand, simulation, module, algorithm, noise, distribution, population, degradation\n",
       "Topic2                         IPTG, GFP, rb, biosensor, mrna, repression, sensor, inducer, LB, strains, supplementary, strength, lac, riboswitch, regulator, RBS, cultures, inducible, constitutive, arabinose\n",
       "Topic3  cassette, bp, cluster, integration, deletion, terminator, recombination, CRISPR, chromosome, marker, clone, cerevisiae, locus, transformation, DNA, kb, assembled, ligation, homologous, sgrna, editing\n",
       "Topic4                                                        light, ion, membrane, ligand, peptide, fusion, surface, activation, affinity, intensity, switch, receptor, ed, image, antibody, residue, scaffold\n",
       "Topic5                                                                                   compound, biosynthesis, tRNA, flux, titer, carbon, metabolite, fermentation, fatty, residue, precursor, mass, synthase"
      ]
     },
     "execution_count": 109,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5_v2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'lda_5_v2' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-1-758f0ea7d3d1>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mtm_results\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlda_5_v2\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mcorpus\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'lda_5_v2' is not defined"
     ]
    }
   ],
   "source": [
    "tm_results = lda_5_v2[corpus]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "corpus_topics = [sorted(topics, key=lambda record: -record[1])[0] for topics in tm_results]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "corpus_topic_df = pd.DataFrame({'Dominant Topic': [item[0]+1 for item in corpus_topics]})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Dominant Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>902</th>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>903</th>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>904</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>905</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>907 rows  1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Dominant Topic\n",
       "0                 2\n",
       "1                 1\n",
       "2                 2\n",
       "3                 5\n",
       "4                 4\n",
       "..              ...\n",
       "902               2\n",
       "903               4\n",
       "904               5\n",
       "905               5\n",
       "906               1\n",
       "\n",
       "[907 rows x 1 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corpus_topic_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "dominant_topic_df = corpus_topic_df.groupby('Dominant Topic').agg(\n",
    "                                  Doc_Count = ('Dominant Topic', np.size),\n",
    "                                  Total_Docs_Perc = ('Dominant Topic', np.size)).reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Dominant Topic</th>\n",
       "      <th>Doc_Count</th>\n",
       "      <th>Total_Docs_Perc</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>201</td>\n",
       "      <td>22.16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>168</td>\n",
       "      <td>18.52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3</td>\n",
       "      <td>185</td>\n",
       "      <td>20.40</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>4</td>\n",
       "      <td>170</td>\n",
       "      <td>18.74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5</td>\n",
       "      <td>183</td>\n",
       "      <td>20.18</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Dominant Topic  Doc_Count  Total_Docs_Perc\n",
       "0               1        201            22.16\n",
       "1               2        168            18.52\n",
       "2               3        185            20.40\n",
       "3               4        170            18.74\n",
       "4               5        183            20.18"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dominant_topic_df['Total_Docs_Perc'] = dominant_topic_df['Total_Docs_Perc'].apply(lambda row: round((row*100) / len(corpus), 2))\n",
    "\n",
    "dominant_topic_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import nltk\n",
    "import re\n",
    "import string\n",
    "import scipy\n",
    "import six\n",
    "import collections\n",
    "import unicodedata\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.stem.porter import PorterStemmer\n",
    "from nltk.stem import WordNetLemmatizer\n",
    "from sklearn.feature_extraction.text import TfidfTransformer\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "\n",
    "\n",
    "def convert_to_unicode(text):\n",
    "    \"\"\"Converts `text` to Unicode (if it's not already), assuming utf-8 input.\"\"\"\n",
    "    if six.PY3:\n",
    "        if isinstance(text, str):\n",
    "            return text\n",
    "        elif isinstance(text, bytes):\n",
    "            return text.decode(\"utf-8\", \"ignore\")\n",
    "        else:\n",
    "            raise ValueError(\"Unsupported string type: %s\" % (type(text)))\n",
    "    elif six.PY2:\n",
    "        if isinstance(text, str):\n",
    "            return text.decode(\"utf-8\", \"ignore\")\n",
    "        elif isinstance(text, unicode):\n",
    "            return text\n",
    "        else:\n",
    "            raise ValueError(\"Unsupported string type: %s\" % (type(text)))\n",
    "    else:\n",
    "        raise ValueError(\"Not running on Python2 or Python 3?\")\n",
    "\n",
    "def whitespace_tokenize(text):\n",
    "    \"\"\"Runs basic whitespace cleaning and splitting on a piece of text.\"\"\"\n",
    "    text = text.strip()\n",
    "    if not text:\n",
    "        return []\n",
    "    tokens = text.split()\n",
    "    return tokens  \n",
    "\n",
    "\n",
    "### Clean our Text and return a list of tokens ###\n",
    "class PlainTokenizer(object):\n",
    "    \"\"\"\n",
    "        Runs basic tokenization (punctuation splitting, lower casing, etc.).\n",
    "        Ner terms aren't used in this tokenizer. \n",
    "    \"\"\"\n",
    "\n",
    "    def __init__(self, do_lower_case=True):\n",
    "        \"\"\"Constructs a BasicTokenizer.\n",
    "        Args:\n",
    "            do_lower_case: Whether to lower case the input.\n",
    "        \"\"\"\n",
    "        self.do_lower_case = do_lower_case\n",
    "\n",
    "    def _is_whitespace(self, char):\n",
    "        \"\"\"Checks whether `chars` is a whitespace character.\"\"\"\n",
    "        # \\t, \\n, and \\r are technically contorl characters but we treat them\n",
    "        # as whitespace since they are generally considered as such.\n",
    "        if char == \" \" or char == \"\\t\" or char == \"\\n\" or char == \"\\r\":\n",
    "            return True\n",
    "        cat = unicodedata.category(char)\n",
    "        if cat == \"Zs\":\n",
    "            return True\n",
    "        return False\n",
    "\n",
    "    def _is_control(self, char):\n",
    "        \"\"\"Checks whether `chars` is a control character.\"\"\"\n",
    "        # These are technically control characters but we count them as whitespace\n",
    "        # characters.\n",
    "        if char == \"\\t\" or char == \"\\n\" or char == \"\\r\":\n",
    "            return False\n",
    "        cat = unicodedata.category(char)\n",
    "        if cat.startswith(\"C\"):\n",
    "            return True\n",
    "        return False\n",
    "\n",
    "    def _is_punctuation(self, char):\n",
    "        \"\"\"Checks whether `chars` is a punctuation character.\"\"\"\n",
    "        cp = ord(char)\n",
    "        # We treat all non-letter/number ASCII as punctuation.\n",
    "        # Characters such as \"^\", \"$\", and \"`\" are not in the Unicode\n",
    "        # Punctuation class but we treat them as punctuation anyways, for\n",
    "        # consistency.\n",
    "\n",
    "        # '-' not break \n",
    "        if cp == 45:\n",
    "            return False\n",
    "        if ((cp >= 33 and cp <= 47) or (cp >= 58 and cp <= 64) or\n",
    "            (cp >= 91 and cp <= 96) or (cp >= 123 and cp <= 126)):\n",
    "            return True\n",
    "        cat = unicodedata.category(char)\n",
    "        if cat.startswith(\"P\"):\n",
    "            return True\n",
    "        return False\n",
    "    \n",
    "    def tokenize(self, text):\n",
    "        \"\"\"Tokenizes a piece of text.\"\"\"\n",
    "\n",
    "        # convert 'text' to unicode \n",
    "        text = convert_to_unicode(text)\n",
    "\n",
    "        # remove invalide char and white space \n",
    "        text = self._clean_text(text)\n",
    "\n",
    "        # spliting on a piece of text \n",
    "        orig_tokens = whitespace_tokenize(text)\n",
    "        split_tokens = []\n",
    "        for token in orig_tokens:\n",
    "            if self.do_lower_case:\n",
    "                token = token.lower()\n",
    "\n",
    "                # remove accent (Nonspacing Mark from the tokens)\n",
    "                token = self._run_strip_accents(token)\n",
    "            split_tokens.extend(self._run_split_on_punc(token))\n",
    "\n",
    "        output_tokens = whitespace_tokenize(\" \".join(split_tokens))\n",
    "        return output_tokens\n",
    "\n",
    "    def _run_strip_accents(self, text):\n",
    "        \"\"\"Strips accents from a piece of text.\"\"\"\n",
    "        text = unicodedata.normalize(\"NFD\", text)\n",
    "        output = []\n",
    "        for char in text:\n",
    "            cat = unicodedata.category(char)\n",
    "            if cat == \"Mn\":\n",
    "                continue\n",
    "            output.append(char)\n",
    "        return \"\".join(output)\n",
    "\n",
    "    def _run_split_on_punc(self, text):\n",
    "        \"\"\"Splits punctuation on a piece of text.\"\"\"\n",
    "        chars = list(text)\n",
    "        i = 0\n",
    "        start_new_word = True\n",
    "        output = []\n",
    "        while i < len(chars):\n",
    "            char = chars[i]\n",
    "            if self._is_punctuation(char):\n",
    "                output.append([char])\n",
    "                start_new_word = True\n",
    "            else:\n",
    "                if start_new_word:\n",
    "                    output.append([])\n",
    "                start_new_word = False\n",
    "                output[-1].append(char)\n",
    "            i += 1\n",
    "\n",
    "        return [\"\".join(x) for x in output]\n",
    "\n",
    "    def _clean_text(self, text):\n",
    "        \"\"\"Performs invalid character removal and whitespace cleanup on text.\"\"\"\n",
    "        output = []\n",
    "        for char in text:\n",
    "            cp = ord(char)\n",
    "            if cp == 0 or cp == 0xfffd or self._is_control(char):\n",
    "                continue\n",
    "            if self._is_whitespace(char):\n",
    "                output.append(\" \")\n",
    "            else:\n",
    "                output.append(char)\n",
    "        return \"\".join(output)\n",
    "\n",
    "\n",
    "class NERTokenizer(PlainTokenizer):\n",
    "\n",
    "    \"\"\"\n",
    "        Subclass of PlainTokenizer and override 'tokenize' function \n",
    "        to keep the ner terms intact\n",
    "    \"\"\"\n",
    "\n",
    "    def __init__(self, do_lower_case=True):\n",
    "        super().__init__(do_lower_case=True)\n",
    "\n",
    "    def tokenize(self, text, terms):\n",
    "        \"\"\"Tokenizes a piece of text.\"\"\"\n",
    "        text = convert_to_unicode(text)\n",
    "        text = self._clean_text(text)\n",
    "        orig_tokens = whitespace_tokenize(text)\n",
    "        split_tokens = []\n",
    "\n",
    "        for token in orig_tokens:\n",
    "\n",
    "            # remain terms untouched\n",
    "            if token in terms:\n",
    "                split_tokens.extend([token])\n",
    "                continue\n",
    "\n",
    "            # if the token contains multiple (>=1) terms\n",
    "            elif sum([term in token for term in terms]) > 0:\n",
    "                for term in terms:\n",
    "                    if term in token:\n",
    "                        splited = token.split(term)\n",
    "\n",
    "                        # result: token_0 + term + token_1\n",
    "                        token = splited[0] #token_0\n",
    "                        if self.do_lower_case:\n",
    "                            token = token.lower()\n",
    "                            token = self._run_strip_accents(token)\n",
    "                        split_tokens.extend(self._run_split_on_punc(token))\n",
    "                        split_tokens.append(term) # term\n",
    "                        token = splited[1] # token_1\n",
    "                        if self.do_lower_case:\n",
    "                            token = token.lower()\n",
    "                            token = self._run_strip_accents(token)\n",
    "                        split_tokens.extend(self._run_split_on_punc(token))\n",
    "                continue\n",
    "\n",
    "            # if text is not in terms \n",
    "            else:\n",
    "                if self.do_lower_case:\n",
    "                    token = token.lower()\n",
    "                    token = self._run_strip_accents(token)\n",
    "                split_tokens.extend(self._run_split_on_punc(token))\n",
    "\n",
    "        output_tokens = whitespace_tokenize(\" \".join(split_tokens))\n",
    "        return output_tokens\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "from nltk.stem.porter import PorterStemmer\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.corpus import stopwords\n",
    "from nltk import pos_tag\n",
    "from nltk.stem import WordNetLemmatizer\n",
    "import string\n",
    "import re\n",
    "\n",
    "def NonNerCleanText(data):\n",
    "\n",
    "    # basic tokenizer \n",
    "    B_tokenizer = tokenizer.PlainTokenizer()\n",
    "\n",
    "    # set stop words\n",
    "    stop_words = set(stopwords.words('english'))\n",
    "    stop_words = stop_words.union({'et', 'al', 'use', 'using', 'used'})\n",
    "    # punctuations = string.punctuation.replace('-','')\n",
    "    lemmatizer = WordNetLemmatizer()\n",
    "\n",
    "    data = re.sub(r'\\([^()]*\\)', '', data)\n",
    "    for tag in ['REFEND', 'REF', 'EQL', 'FIG']:\n",
    "        data = data.replace(tag, '')\n",
    "    words = [s for s in B_tokenizer.tokenize(data) if re.match(\"^[A-Za-z0-9\\-]+$\", s)]\n",
    "    words = [w for w in words if not w in stop_words]\n",
    "    words = [lemmatizer.lemmatize(s) for s in words]\n",
    "    words = [s for s in words if not re.match(\"^[0-9]+$\", s)]\n",
    "    words = [s for s in words if not len(s) == 1]\n",
    "\n",
    "    # Only keeping the nouns \n",
    "    is_noun = lambda pos: pos[:2] == 'NN'\n",
    "    words = [word for (word, pos) in pos_tag(words) if is_noun(pos) or '-' in word] \n",
    "\n",
    "    return words\n",
    "\n",
    "def NERCleanText(key, data, path2entity):\n",
    "    \"\"\"\n",
    "        This method cleans the text and creates tokens \n",
    "        @param filepath: path to articles \n",
    "        @oaram path2entity: ner dictionary \n",
    "\n",
    "        @return words: nested list; \n",
    "                       each inner list contains the tokens of each article \n",
    "    \"\"\"\n",
    "    # using ner tokenizer which keep ner terms intact \n",
    "    B_tokenizer = NERTokenizer()\n",
    "    \n",
    "    # set stop words \n",
    "    stop_words = set(stopwords.words('english'))\n",
    "    stop_words = stop_words.union({'et', 'al', 'use', 'using', 'used'})\n",
    "    punctuations = string.punctuation\n",
    "    lemmatizer = WordNetLemmatizer()\n",
    "\n",
    "    # load NER entities\n",
    "    entities = list(path2entity[key])\n",
    "    entities.sort(key=len, reverse=True)    \n",
    "\n",
    "    # open file read data \n",
    "    \n",
    "\n",
    "    # non-words tag\n",
    "    for tag in ['REFEND', 'REF', 'EQL', 'FIG']:\n",
    "        data = data.replace(tag, '')\n",
    "    \n",
    "    # check NER terms\n",
    "    terms = []\n",
    "    for t in entities:\n",
    "\n",
    "        # replace \" \" with \"_\" for NER terms\n",
    "        if ' ' in t:\n",
    "            tnew = t.replace(' ', '_')\n",
    "            data = data.replace(t, tnew)\n",
    "            terms.append(tnew)\n",
    "        else:\n",
    "            terms.append(t)\n",
    "    terms.sort(key=len, reverse=True)\n",
    "    \n",
    "    # using NERTokenizer\n",
    "    words = [s for s in B_tokenizer.tokenize(data, terms) if not (len(s) == 1 and (s in punctuations))]\n",
    "  \n",
    "    # remove stop words \n",
    "    words = [w for w in words if not w in stop_words]\n",
    "    words_new = []\n",
    "    for w in words:\n",
    "\n",
    "        # if ner terms, keep intact \n",
    "        if w in terms:\n",
    "            words_new.append(w.replace('_', ' '))\n",
    "\n",
    "        # else lemmatize\n",
    "        else:\n",
    "            words_new.append(lemmatizer.lemmatize(w))\n",
    "    \n",
    "    words = words_new\n",
    "    words = [s for s in words if not re.match(\"^[0-9]+$\", s)]\n",
    "    words = [s for s in words if not len(s) == 1]\n",
    "   \n",
    "    return words\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "from src.data.load_data import *\n",
    "import pickle\n",
    "\n",
    "from sklearn.feature_extraction.text import TfidfTransformer\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.pipeline import Pipeline\n",
    "\n",
    "def load_json_ner(json_input_path, ner_input_path, contain_abstract):\n",
    "    json_files = load_json_files(json_input_path, contain_abstract)\n",
    "    path2entity = load_ner_entities(ner_input_path, json_files.keys())\n",
    "\n",
    "    return json_files, path2entity\n",
    "\n",
    "def create_data_words(json_files, path2entity, q):\n",
    "    article_sq = []\n",
    "    for k in json_files:\n",
    "        if k not in path2entity:\n",
    "            path2entity[k] = set()\n",
    "\n",
    "    data_words = []\n",
    "    for k in json_files.keys():\n",
    "        article_sq.append(k)\n",
    "        data_words.append(NERCleanText(k, json_files[k], path2entity))\n",
    "        \n",
    "\n",
    "#     if q > 0:\n",
    "#         corpus = [' '.join(d) for d in data_words]\n",
    "#         tfidf = TfidfTransformer()\n",
    "#         vocabulary = list(set(' '.join(corpus).split()))\n",
    "#         pipe = Pipeline([('count', CountVectorizer(vocabulary=vocabulary)),('tfid', TfidfTransformer())]).fit(corpus)\n",
    "#         ser = pd.Series(index = vocabulary, data = pipe['tfid'].idf_)\n",
    "\n",
    "#         # create stop words list \n",
    "#         stops = ser[ser<ser.quantile(q)].sort_values().index.tolist()\n",
    "\n",
    "#         #update data_words \n",
    "#         data_words_cleaned = []\n",
    "#         for d in data_words:\n",
    "#             data_words_cleaned.append([w for w in d if w not in stops])\n",
    "#         data_words = data_words_cleaned\n",
    "\n",
    "\n",
    "    return data_words, article_sq\n",
    "    \n",
    "def save_data_words(json_input_path, ner_input_path, contain_abstract, q, output_path_data_words, output_path_a_sq):\n",
    "    json_files, path2entity = load_json_ner(json_input_path, ner_input_path, contain_abstract)\n",
    "    data_words, a_sq = create_data_words(json_files, path2entity, q)\n",
    "    return data_words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "data_words = save_data_words(**params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['another',\n",
       " 'mde',\n",
       " 'reaction',\n",
       " 'intermediate',\n",
       " 'produced',\n",
       " 'fusion',\n",
       " 'enzyme',\n",
       " 'diffuse',\n",
       " 'increasing',\n",
       " 'efficiency',\n",
       " 'subsequent',\n",
       " 'reaction',\n",
       " 'hence',\n",
       " 'developing',\n",
       " 'artificial',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'powerful',\n",
       " 'tool',\n",
       " 'increasing',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'specific',\n",
       " 'pathway',\n",
       " 'study',\n",
       " 'investigates',\n",
       " 'effect',\n",
       " 'overexpressing',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'method',\n",
       " 'MA',\n",
       " 'yield',\n",
       " 'hence',\n",
       " 'MA',\n",
       " 'synthesis',\n",
       " 'pathway',\n",
       " 'introduced',\n",
       " 'strain',\n",
       " 'increased',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'flux',\n",
       " 'described',\n",
       " 'selecting',\n",
       " 'optimal',\n",
       " 'MA',\n",
       " 'synthesis',\n",
       " 'pathway',\n",
       " 'three',\n",
       " 'candidate',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'selected',\n",
       " 'pathway',\n",
       " 'increased',\n",
       " 'overexpressing',\n",
       " 'chorismate synthase',\n",
       " 'AroC',\n",
       " 'finally',\n",
       " 'effect',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'method',\n",
       " 'investigated',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'increased',\n",
       " 'overexpressing',\n",
       " 'gene-level',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'result',\n",
       " 'discussionselection',\n",
       " 'candidate',\n",
       " 'pathway',\n",
       " 'MA',\n",
       " 'productionseveral',\n",
       " 'previous',\n",
       " 'report',\n",
       " 'described',\n",
       " 'method',\n",
       " 'increasing',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'among',\n",
       " 'introduction',\n",
       " 'several',\n",
       " 'MA',\n",
       " 'synthesis',\n",
       " 'pathway',\n",
       " 'study',\n",
       " 'tested',\n",
       " 'three',\n",
       " 'pathway',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'CFT5 strain',\n",
       " 'two',\n",
       " 'three',\n",
       " 'pathway',\n",
       " 'start',\n",
       " 'chorismate',\n",
       " 'key',\n",
       " 'compound',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'pathway',\n",
       " 'produce',\n",
       " 'MA',\n",
       " 'via',\n",
       " 'salicylate',\n",
       " 'whereas',\n",
       " 'pathway',\n",
       " 'produce',\n",
       " 'MA',\n",
       " 'via',\n",
       " 'PHB',\n",
       " 'figure',\n",
       " 'pathway',\n",
       " 'start',\n",
       " 'DHS',\n",
       " 'another',\n",
       " 'intermediate',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'produce',\n",
       " 'MA',\n",
       " 'via',\n",
       " 'PCA',\n",
       " 'figure',\n",
       " 'three',\n",
       " 'pathway',\n",
       " 'catechol',\n",
       " 'produced',\n",
       " 'salicylate',\n",
       " 'PCA',\n",
       " 'followed',\n",
       " 'cleavage',\n",
       " 'catechol benzene ring',\n",
       " 'form',\n",
       " 'MA',\n",
       " 'pathway',\n",
       " 'order',\n",
       " 'gene',\n",
       " 'tested',\n",
       " 'choose',\n",
       " 'suited',\n",
       " 'CFT5',\n",
       " 'supporting',\n",
       " 'information',\n",
       " 'table',\n",
       " 's3',\n",
       " 'engineered',\n",
       " 'three',\n",
       " 'E. coli',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'CFT51a',\n",
       " 'CFT52a',\n",
       " 'CFT53a',\n",
       " 'carrying',\n",
       " 'pathway',\n",
       " 'pathway',\n",
       " 'pathway',\n",
       " 'respectively',\n",
       " 'figure',\n",
       " 'show',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'produced',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51a',\n",
       " 'CFT53a',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'respectively',\n",
       " 'although',\n",
       " 'CFT52a',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'cultivation',\n",
       " 'amount',\n",
       " 'produced',\n",
       " 'decreased',\n",
       " 'detection',\n",
       " 'limit',\n",
       " 'cultivation',\n",
       " 'difference',\n",
       " 'cell',\n",
       " 'growth',\n",
       " 'observed',\n",
       " 'strain',\n",
       " 'strain',\n",
       " 'consumed',\n",
       " 'glucose',\n",
       " 'within',\n",
       " 'cultivation',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's1',\n",
       " 'figure',\n",
       " 's2',\n",
       " 'intermediate',\n",
       " 'MA',\n",
       " 'synthesis',\n",
       " 'pathway',\n",
       " 'salicylate',\n",
       " 'pathway',\n",
       " 'PCA',\n",
       " 'pathway',\n",
       " 'catechol',\n",
       " 'pathway',\n",
       " 'detected',\n",
       " 'culture',\n",
       " 'supernatant',\n",
       " 'CFT51a',\n",
       " 'CFT52a',\n",
       " 'cultivation',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'titer',\n",
       " 'CFT5-derived strains',\n",
       " 'red',\n",
       " 'blue',\n",
       " 'green',\n",
       " 'bar',\n",
       " 'indicate',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'via',\n",
       " 'pathway',\n",
       " 'pathway',\n",
       " 'Pathway3',\n",
       " 'respectively',\n",
       " 'empty',\n",
       " 'vector',\n",
       " 'control',\n",
       " 'CFT5',\n",
       " 'containing',\n",
       " 'pza23mcs',\n",
       " 'data',\n",
       " 'presented',\n",
       " 'average',\n",
       " 'three',\n",
       " 'independent',\n",
       " 'experiment',\n",
       " 'error',\n",
       " 'bar',\n",
       " 'indicate',\n",
       " 'standard',\n",
       " 'error',\n",
       " 'although',\n",
       " 'high',\n",
       " 'PHB',\n",
       " 'production',\n",
       " 'previously',\n",
       " 'observed',\n",
       " 'CFT5-derived',\n",
       " 'strain',\n",
       " 'mg',\n",
       " 'cultivation',\n",
       " 'CFT52a',\n",
       " 'produced',\n",
       " 'small',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'CFT52a cultures',\n",
       " 'contained',\n",
       " 'detectable',\n",
       " 'PCA',\n",
       " 'catechol',\n",
       " 'small',\n",
       " 'amount',\n",
       " 'PHB',\n",
       " 'detected',\n",
       " 'cultivation',\n",
       " 'pathway',\n",
       " 'include',\n",
       " 'reaction',\n",
       " 'catalyzed',\n",
       " 'AroY',\n",
       " 'CatA',\n",
       " 'CFT53a',\n",
       " 'carrying',\n",
       " 'Pathway3',\n",
       " 'produced',\n",
       " 'highest',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'among',\n",
       " 'three',\n",
       " 'strain',\n",
       " 'reaction',\n",
       " 'considered',\n",
       " 'optimum',\n",
       " 'CFT5-derived strains',\n",
       " 'sds-page',\n",
       " 'analysis',\n",
       " 'CFT52a',\n",
       " 'strain',\n",
       " 'significant',\n",
       " 'overexpression',\n",
       " 'PobA',\n",
       " 'UbiC',\n",
       " 'detected',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's3',\n",
       " 'order',\n",
       " 'produce',\n",
       " 'MA',\n",
       " 'high',\n",
       " 'yield',\n",
       " 'pathway',\n",
       " 'would',\n",
       " 'necessary',\n",
       " 'solve',\n",
       " 'expression',\n",
       " 'problem',\n",
       " 'hence',\n",
       " 'conclude',\n",
       " 'pathway',\n",
       " 'suitable',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'CFT5-derived strains',\n",
       " 'optimizing',\n",
       " 'production',\n",
       " 'pathway',\n",
       " 'MA',\n",
       " 'increase',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'approached',\n",
       " 'intracellular',\n",
       " 'availability',\n",
       " 'compound',\n",
       " 'branch',\n",
       " 'point',\n",
       " 'endogenous',\n",
       " 'shikimate',\n",
       " 'synthetic',\n",
       " 'MA',\n",
       " 'pathway',\n",
       " 'branch',\n",
       " 'point',\n",
       " 'compound',\n",
       " 'chorismate',\n",
       " 'CFT51a',\n",
       " 'pathway',\n",
       " 'DHS',\n",
       " 'CFT53a',\n",
       " 'pathway',\n",
       " 'since',\n",
       " 'accumulation',\n",
       " 'intermediate',\n",
       " 'detected',\n",
       " 'culture',\n",
       " 'supernatant',\n",
       " 'CFT51a',\n",
       " 'CFT53a',\n",
       " 'pathway',\n",
       " 'bottleneck',\n",
       " 'strain',\n",
       " 'improved',\n",
       " 'availability',\n",
       " 'chorismate',\n",
       " 'DHS',\n",
       " 'essential',\n",
       " 'increasing',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'overexpression',\n",
       " 'AroC',\n",
       " 'CFT51a',\n",
       " 'AroD',\n",
       " 'CFT53a',\n",
       " 'increase',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'chorismate',\n",
       " 'DHS',\n",
       " 'respectively',\n",
       " 'figure',\n",
       " 'show',\n",
       " 'amount',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51b',\n",
       " 'produced',\n",
       " 'MA',\n",
       " '6-fold',\n",
       " 'higher',\n",
       " 'produced',\n",
       " 'CFT51a',\n",
       " 'however',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'produced',\n",
       " 'CFT53b',\n",
       " 'quantification',\n",
       " 'limit',\n",
       " 'lutke-eversloh',\n",
       " 'investigated',\n",
       " 'key',\n",
       " 'enzyme',\n",
       " 'tyrosine',\n",
       " 'production',\n",
       " 'reporting',\n",
       " 'overexpression',\n",
       " 'AroC',\n",
       " 'effective',\n",
       " 'way',\n",
       " 'increase',\n",
       " 'production',\n",
       " 'compound',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'report',\n",
       " 'consistent',\n",
       " 'finding',\n",
       " 'showed',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'effectively',\n",
       " 'increased',\n",
       " 'overexpression',\n",
       " 'AroC',\n",
       " 'however',\n",
       " 'contrary',\n",
       " 'expectation',\n",
       " 'overexpression',\n",
       " 'AroD',\n",
       " 'negative',\n",
       " 'effect',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'CFT53b',\n",
       " 'pathway',\n",
       " 'overexpression',\n",
       " 'AroD',\n",
       " 'might',\n",
       " 'increase',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'downstream',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'rather',\n",
       " 'PCA',\n",
       " 'precursor',\n",
       " 'MA',\n",
       " 'expression',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'increase',\n",
       " 'MA',\n",
       " 'yieldas',\n",
       " 'already',\n",
       " 'stated',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'powerful',\n",
       " 'strategy',\n",
       " 'enhance',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'specific',\n",
       " 'compound',\n",
       " 'interest',\n",
       " 'promoting',\n",
       " 'efficient',\n",
       " 'transfer',\n",
       " 'intermediate',\n",
       " 'product',\n",
       " 'enzyme',\n",
       " 'another',\n",
       " 'hence',\n",
       " 'strategy',\n",
       " 'applied',\n",
       " 'study',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'aiming',\n",
       " 'increase',\n",
       " 'production',\n",
       " 'figure',\n",
       " 'pathway',\n",
       " 'two',\n",
       " 'enzyme',\n",
       " 'branch',\n",
       " 'point',\n",
       " 'shikimate',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'pathway',\n",
       " 'overexpressed',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'pathway',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'pathway',\n",
       " 'figure',\n",
       " 'first',\n",
       " 'examined',\n",
       " 'effect',\n",
       " 'linker',\n",
       " 'type',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'compared',\n",
       " 'case',\n",
       " 'without',\n",
       " 'linker',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'rigid',\n",
       " 'linker',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's4',\n",
       " 'respectively',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'suitable',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'supplementary',\n",
       " 'table',\n",
       " 's4',\n",
       " 'figure',\n",
       " 'display',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'produced',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'respectively',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " '16-fold',\n",
       " 'higher',\n",
       " 'produced',\n",
       " 'CFT51b',\n",
       " 'express',\n",
       " 'unfused',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'protein',\n",
       " 'confirmed',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'expressed',\n",
       " 'MenF',\n",
       " 'fused',\n",
       " 'aroc',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'respectively',\n",
       " 'western',\n",
       " 'blotting',\n",
       " 'analysis',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's5',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'two',\n",
       " 'enzyme',\n",
       " 'placed',\n",
       " 'close',\n",
       " 'proximity',\n",
       " 'thereby',\n",
       " 'increasing',\n",
       " 'apparent',\n",
       " 'concentration',\n",
       " 'intermediate',\n",
       " 'chorismate',\n",
       " 'around',\n",
       " 'MenF',\n",
       " 'relative',\n",
       " 'nonfusion',\n",
       " 'enzyme',\n",
       " 'resulting',\n",
       " 'higher',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'pathway',\n",
       " 'figure',\n",
       " 'competing',\n",
       " 'pathway',\n",
       " 'branched',\n",
       " 'synthetic',\n",
       " 'pathway',\n",
       " 'target',\n",
       " 'compound',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'potential',\n",
       " 'significantly',\n",
       " 'increase',\n",
       " 'production',\n",
       " 'however',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'improvement',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'noticed',\n",
       " 'CFT53b',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'overexpressed',\n",
       " 'separately',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'drastically',\n",
       " 'decreased',\n",
       " 'CFT53a',\n",
       " 'overexpressing',\n",
       " 'AroD',\n",
       " 'figure',\n",
       " 'sds-page',\n",
       " 'analysis',\n",
       " 'CFT53b',\n",
       " 'free',\n",
       " 'AroD',\n",
       " 'significantly',\n",
       " 'overexpressed',\n",
       " 'strain',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's6',\n",
       " 'one',\n",
       " 'explanation',\n",
       " 'could',\n",
       " 'related',\n",
       " 'fact',\n",
       " 'overexpression',\n",
       " 'AroD',\n",
       " 'shifted',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'effectively',\n",
       " 'PCA',\n",
       " 'expression',\n",
       " 'AroZ',\n",
       " 'AroY',\n",
       " 'CatA',\n",
       " 'distinctively',\n",
       " 'confirmed',\n",
       " 'CFT53b',\n",
       " 'imbalance',\n",
       " 'expression',\n",
       " 'level',\n",
       " 'enzyme',\n",
       " 'might',\n",
       " 'negative',\n",
       " 'effect',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'hand',\n",
       " 'sds-page',\n",
       " 'analysis',\n",
       " 'CFT53c',\n",
       " 'distinctively',\n",
       " 'confirmed',\n",
       " 'overexpressed',\n",
       " 'free',\n",
       " 'AroD',\n",
       " 'fused',\n",
       " 'AroD',\n",
       " 'aroz',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's6',\n",
       " 'since',\n",
       " 'expression',\n",
       " 'level',\n",
       " 'AroD',\n",
       " 'aroz',\n",
       " 'greatly',\n",
       " 'reduced',\n",
       " 'compared',\n",
       " 'AroD',\n",
       " 'imbalance',\n",
       " 'would',\n",
       " 'eliminated',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'restored',\n",
       " 'however',\n",
       " 'production',\n",
       " 'titer',\n",
       " 'CFT53c',\n",
       " 'lower',\n",
       " 'CFT53a',\n",
       " 'seems',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'would',\n",
       " 'worked',\n",
       " 'well',\n",
       " 'inactivation',\n",
       " 'AroD',\n",
       " 'domain',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'native',\n",
       " 'AroD',\n",
       " 'protein',\n",
       " 'expressed',\n",
       " 'homodimer',\n",
       " 'E. coli',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " 'may',\n",
       " 'dimerize',\n",
       " 'properly',\n",
       " 'fused',\n",
       " 'AroZ',\n",
       " 'case',\n",
       " 'endogenous',\n",
       " 'AroD',\n",
       " 'would',\n",
       " 'function',\n",
       " '3-dehydroquinate dehydratase',\n",
       " 'CFT53a',\n",
       " 'hypothesis',\n",
       " 'may',\n",
       " 'explain',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'nearly',\n",
       " 'CFT53a',\n",
       " 'CFT53c',\n",
       " 'thus',\n",
       " 'difficult',\n",
       " 'preserve',\n",
       " 'function',\n",
       " 'protein',\n",
       " 'complex',\n",
       " 'structure',\n",
       " 'dimeric',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'another',\n",
       " 'strategy',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'involves',\n",
       " 'protein-level',\n",
       " 'fusion',\n",
       " 'translation',\n",
       " 'transpeptidases',\n",
       " 'effect',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'expression',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'CFT51b',\n",
       " 'diagram',\n",
       " 'shikimate',\n",
       " 'pathway',\n",
       " 'CFT51c',\n",
       " 'black',\n",
       " 'red',\n",
       " 'pac-',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'indicate',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'respectively',\n",
       " 'orange',\n",
       " 'purple',\n",
       " 'pac-',\n",
       " 'man',\n",
       " 'shape',\n",
       " 'indicate',\n",
       " 'enzyme',\n",
       " 'consuming',\n",
       " 'chorismate',\n",
       " 'substrate',\n",
       " 'yellow',\n",
       " 'band',\n",
       " 'indicates',\n",
       " 'flexible',\n",
       " 'ggggs',\n",
       " ') 3',\n",
       " 'linker',\n",
       " 'light',\n",
       " 'gray',\n",
       " 'circle',\n",
       " '5-O-(1-Carboxyvinyl)-3-phosphoshikimate',\n",
       " 'epsp',\n",
       " 'dark',\n",
       " 'gray',\n",
       " 'diamond',\n",
       " 'chorismate',\n",
       " 'red',\n",
       " 'square',\n",
       " 'isochorismate',\n",
       " 'yellow',\n",
       " 'star',\n",
       " 'MA',\n",
       " 'orange',\n",
       " 'pentagon',\n",
       " 'purple',\n",
       " 'triangle',\n",
       " 'indicate',\n",
       " 'byproduct',\n",
       " 'derived',\n",
       " 'chorismate',\n",
       " 'protein',\n",
       " 'fusion',\n",
       " 'method',\n",
       " 'increase',\n",
       " 'production',\n",
       " 'target',\n",
       " 'compound',\n",
       " 'well',\n",
       " 'matsumoto',\n",
       " 'created',\n",
       " 'recombinant',\n",
       " 'E. coli',\n",
       " 'expressing',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'pyruvate-formate lyase',\n",
       " 'PFL',\n",
       " 'phosphate acetyltransferase',\n",
       " 'PTA',\n",
       " 'ligated',\n",
       " 'Staphylococcal',\n",
       " 'sortase A',\n",
       " 'strain',\n",
       " 'produced',\n",
       " 'significantly',\n",
       " 'higher',\n",
       " 'amount',\n",
       " 'acetate',\n",
       " 'compared',\n",
       " 'control',\n",
       " 'strain',\n",
       " 'expressing',\n",
       " 'PFL',\n",
       " 'PTA',\n",
       " 'separately',\n",
       " 'Sortase-A',\n",
       " '-mediated',\n",
       " 'protein',\n",
       " 'fusion',\n",
       " 'method',\n",
       " 'effective',\n",
       " 'vivo',\n",
       " 'metabolic',\n",
       " 'channeling',\n",
       " 'hence',\n",
       " 'approach',\n",
       " 'seems',\n",
       " 'effective',\n",
       " 'produce',\n",
       " 'aroz-',\n",
       " 'AroD',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'ph-controlled',\n",
       " 'tube-scale',\n",
       " 'culture',\n",
       " 'CaCO 3',\n",
       " 'attempted',\n",
       " 'increasing',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'titer',\n",
       " 'controlling',\n",
       " 'ph',\n",
       " 'decrease',\n",
       " 'ph',\n",
       " 'culturing',\n",
       " 'often',\n",
       " 'bad',\n",
       " 'influence',\n",
       " 'bioproduction',\n",
       " 'although',\n",
       " 'E. coli',\n",
       " 'grows',\n",
       " 'ph',\n",
       " 'environment',\n",
       " 'inside',\n",
       " 'bacteria',\n",
       " 'also',\n",
       " 'change',\n",
       " 'due',\n",
       " 'change',\n",
       " 'external',\n",
       " 'ph',\n",
       " 'acid',\n",
       " 'ic',\n",
       " 'side',\n",
       " 'ph',\n",
       " 'fall',\n",
       " 'intracellular',\n",
       " 'ph',\n",
       " 'decrease',\n",
       " 'culturing',\n",
       " 'CFT51c',\n",
       " 'ph',\n",
       " 'medium',\n",
       " 'decreased',\n",
       " 'thompson',\n",
       " 'introduced',\n",
       " 'multiple',\n",
       " 'MA',\n",
       " 'synthesis',\n",
       " 'pathway',\n",
       " 'E. coli',\n",
       " 'produced',\n",
       " 'mg',\n",
       " 'MA',\n",
       " 'highest',\n",
       " 'titer',\n",
       " 'far',\n",
       " 'microbe',\n",
       " 'batch',\n",
       " 'culture',\n",
       " 'without',\n",
       " 'ph-controlled',\n",
       " 'condition',\n",
       " 'E. coli',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'constructed',\n",
       " 'present',\n",
       " 'study',\n",
       " 'also',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'according',\n",
       " 'result',\n",
       " ...]"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_words[9][667:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "from nltk.stem.porter import PorterStemmer\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.corpus import stopwords\n",
    "from nltk import pos_tag\n",
    "from nltk.stem import WordNetLemmatizer\n",
    "import string\n",
    "import re\n",
    "\n",
    "def NonNerCleanText(data):\n",
    "\n",
    "    # basic tokenizer \n",
    "    B_tokenizer = tokenizer.PlainTokenizer()\n",
    "\n",
    "    # set stop words\n",
    "    stop_words = set(stopwords.words('english'))\n",
    "    stop_words = stop_words.union({'et', 'al', 'use', 'using', 'used'})\n",
    "    # punctuations = string.punctuation.replace('-','')\n",
    "    lemmatizer = WordNetLemmatizer()\n",
    "\n",
    "    data = re.sub(r'\\([^()]*\\)', '', data)\n",
    "    for tag in ['REFEND', 'REF', 'EQL', 'FIG']:\n",
    "        data = data.replace(tag, '')\n",
    "    words = [s for s in B_tokenizer.tokenize(data) if re.match(\"^[A-Za-z0-9\\-]+$\", s)]\n",
    "    words = [w for w in words if not w in stop_words]\n",
    "    words = [lemmatizer.lemmatize(s) for s in words]\n",
    "    words = [s for s in words if not re.match(\"^[0-9]+$\", s)]\n",
    "    words = [s for s in words if not len(s) == 1]\n",
    "\n",
    "    # Only keeping the nouns \n",
    "    is_noun = lambda pos: pos[:2] == 'NN'\n",
    "    words = [word for (word, pos) in pos_tag(words) if is_noun(pos) or '-' in word] \n",
    "\n",
    "    return words\n",
    "\n",
    "def NERCleanText(key, data, path2entity):\n",
    "    \"\"\"\n",
    "        This method cleans the text and creates tokens \n",
    "        @param filepath: path to articles \n",
    "        @oaram path2entity: ner dictionary \n",
    "\n",
    "        @return words: nested list; \n",
    "                       each inner list contains the tokens of each article \n",
    "    \"\"\"\n",
    "    # using ner tokenizer which keep ner terms intact \n",
    "    B_tokenizer = NERTokenizer()\n",
    "    \n",
    "    # set stop words \n",
    "    stop_words = set(stopwords.words('english'))\n",
    "    stop_words = stop_words.union({'et', 'al', 'use', 'using', 'used'})\n",
    "    punctuations = string.punctuation\n",
    "    lemmatizer = WordNetLemmatizer()\n",
    "\n",
    "    # load NER entities\n",
    "    entities = list(path2entity[key])\n",
    "    entities.sort(key=len, reverse=True)    \n",
    "\n",
    "    # open file read data \n",
    "    \n",
    "\n",
    "    # non-words tag\n",
    "    for tag in ['REFEND', 'REF', 'EQL', 'FIG']:\n",
    "        data = data.replace(tag, '')\n",
    "    \n",
    "    # check NER terms\n",
    "    terms = []\n",
    "    for t in entities:\n",
    "\n",
    "        # replace \" \" with \"_\" for NER terms\n",
    "        if ' ' in t:\n",
    "            tnew = t.replace(' ', '_')\n",
    "            data = data.replace(t, tnew)\n",
    "            terms.append(tnew)\n",
    "        else:\n",
    "            terms.append(t)\n",
    "    terms.sort(key=len, reverse=True)\n",
    "    \n",
    "    # using NERTokenizer\n",
    "    words = [s for s in B_tokenizer.tokenize(data, terms) if not (len(s) == 1 and (s in punctuations))]\n",
    "  \n",
    "    # remove stop words \n",
    "    words = [w for w in words if not w in stop_words]\n",
    "    words_new = []\n",
    "    for w in words:\n",
    "\n",
    "        # if ner terms, keep intact \n",
    "        if w in terms:\n",
    "            words_new.append(w.replace('_', ' '))\n",
    "\n",
    "        # else lemmatize\n",
    "        else:\n",
    "            words_new.append(lemmatizer.lemmatize(w))\n",
    "    \n",
    "    words = words_new\n",
    "    words = [s for s in words if not re.match(\"^[0-9]+$\", s)]\n",
    "    words = [s for s in words if not len(s) == 1]\n",
    "   \n",
    "    return words\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "texts = 'AroC and MenF (AroC-MenF) in Pathway 1 and AroD and AroZ (AroD-AroZ) in Pathway 3 (Figure A). First we examined the effect of linker type on MA production. In AroC-MenF and AroD-AroZ, MA production was compared in the case of using without linker, flexible linker, and rigid linker (Supplementary Figure S4), respectively, and flexible linker was most suitable for AroC-MenF and AroD-AroZ (Supplementary Table S4). Figure displays the amount of MA produced by each transformant after 72 h cultivation. CFT51c (AroC-MenF) and CFT53c (AroD-AroZ) produced 3.45  0.04 and 1.20  0.10 g/L of MA, respectively. The production of MA in CFT51c was 2.16-fold higher than that produced by CFT51b, which expresses the unfused AroC and MenF proteins. We confirmed that CFT51b and CFT51c expressed MenF and fused AroC/MenF fusion protein respectively by Western blotting analysis (Supplementary Figure S5). In the AroC-MenF fusion protein, the two enzymes are placed in close proximity, thereby increasing the apparent concentration of the intermediate (chorismate) around MenF relative to that of the nonfusion enzyme, resulting in higher carbon flux to the MA biosynthesis pathway (Figure).'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [],
   "source": [
    "from src.data.load_data import *\n",
    "import pickle\n",
    "\n",
    "from sklearn.feature_extraction.text import TfidfTransformer\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.pipeline import Pipeline\n",
    "\n",
    "def load_json_ner(json_input_path, ner_input_path, contain_abstract):\n",
    "    json_files = load_json_files(json_input_path, contain_abstract)\n",
    "    path2entity = load_ner_entities(ner_input_path, json_files.keys())\n",
    "\n",
    "    return json_files, path2entity\n",
    "\n",
    "def create_data_words(json_files, path2entity, q):\n",
    "    article_sq = []\n",
    "    for k in json_files:\n",
    "        if k not in path2entity:\n",
    "            path2entity[k] = set()\n",
    "\n",
    "    data_words = []\n",
    "    for k in json_files.keys():\n",
    "        k_ = 'sb8b00380'\n",
    "        #article_sq.append(k)\n",
    "        data_words.append(NERCleanText(k_, texts, path2entity))\n",
    "        break\n",
    "\n",
    "    if q > 0:\n",
    "        corpus = [' '.join(d) for d in data_words]\n",
    "        tfidf = TfidfTransformer()\n",
    "        vocabulary = list(set(' '.join(corpus).split()))\n",
    "        pipe = Pipeline([('count', CountVectorizer(vocabulary=vocabulary)),('tfid', TfidfTransformer())]).fit(corpus)\n",
    "        ser = pd.Series(index = vocabulary, data = pipe['tfid'].idf_)\n",
    "\n",
    "        # create stop words list \n",
    "        stops = ser[ser<ser.quantile(q)].sort_values().index.tolist()\n",
    "\n",
    "        #update data_words \n",
    "        data_words_cleaned = []\n",
    "        for d in data_words:\n",
    "            data_words_cleaned.append([w for w in d if w not in stops])\n",
    "        data_words = data_words_cleaned\n",
    "      \n",
    "\n",
    "    return data_words, article_sq\n",
    "    \n",
    "def save_data_words(json_input_path, ner_input_path, contain_abstract, q, output_path_data_words, output_path_a_sq):\n",
    "    json_files, path2entity = load_json_ner(json_input_path, ner_input_path, contain_abstract)\n",
    "    data_words, a_sq = create_data_words(json_files, path2entity, q)\n",
    "    return data_words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_word_9 = save_data_words(**params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'sb8b00380'"
      ]
     },
     "execution_count": 81,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles[9]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['AroC',\n",
       " 'MenF',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'pathway',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'pathway',\n",
       " 'figure',\n",
       " 'first',\n",
       " 'examined',\n",
       " 'effect',\n",
       " 'linker',\n",
       " 'type',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'compared',\n",
       " 'case',\n",
       " 'without',\n",
       " 'linker',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'rigid',\n",
       " 'linker',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's4',\n",
       " 'respectively',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'suitable',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'supplementary',\n",
       " 'table',\n",
       " 's4',\n",
       " 'figure',\n",
       " 'display',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'produced',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'respectively',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " '16-fold',\n",
       " 'higher',\n",
       " 'produced',\n",
       " 'CFT51b',\n",
       " 'express',\n",
       " 'unfused',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'protein',\n",
       " 'confirmed',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'expressed',\n",
       " 'MenF',\n",
       " 'fused',\n",
       " 'aroc',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'respectively',\n",
       " 'western',\n",
       " 'blotting',\n",
       " 'analysis',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's5',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'two',\n",
       " 'enzyme',\n",
       " 'placed',\n",
       " 'close',\n",
       " 'proximity',\n",
       " 'thereby',\n",
       " 'increasing',\n",
       " 'apparent',\n",
       " 'concentration',\n",
       " 'intermediate',\n",
       " 'chorismate',\n",
       " 'around',\n",
       " 'MenF',\n",
       " 'relative',\n",
       " 'nonfusion',\n",
       " 'enzyme',\n",
       " 'resulting',\n",
       " 'higher',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'pathway',\n",
       " 'figure']"
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_word_9[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['AroC',\n",
       " 'MenF',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'pathway',\n",
       " 'AroD',\n",
       " 'AroZ',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'pathway',\n",
       " 'figure',\n",
       " 'first',\n",
       " 'examined',\n",
       " 'effect',\n",
       " 'linker',\n",
       " 'type',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'MA',\n",
       " 'production',\n",
       " 'compared',\n",
       " 'case',\n",
       " 'without',\n",
       " 'linker',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'rigid',\n",
       " 'linker',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's4',\n",
       " 'respectively',\n",
       " 'flexible',\n",
       " 'linker',\n",
       " 'suitable',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'supplementary',\n",
       " 'table',\n",
       " 's4',\n",
       " 'figure',\n",
       " 'display',\n",
       " 'amount',\n",
       " 'MA',\n",
       " 'produced',\n",
       " 'transfor',\n",
       " 'man',\n",
       " 'cultivation',\n",
       " 'CFT51c',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'CFT53c',\n",
       " 'AroD',\n",
       " '-aroz',\n",
       " 'produced',\n",
       " 'MA',\n",
       " 'respectively',\n",
       " 'production',\n",
       " 'MA',\n",
       " 'CFT51c',\n",
       " '16-fold',\n",
       " 'higher',\n",
       " 'produced',\n",
       " 'CFT51b',\n",
       " 'express',\n",
       " 'unfused',\n",
       " 'AroC',\n",
       " 'MenF',\n",
       " 'protein',\n",
       " 'confirmed',\n",
       " 'CFT51b',\n",
       " 'CFT51c',\n",
       " 'expressed',\n",
       " 'MenF',\n",
       " 'fused',\n",
       " 'aroc',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'respectively',\n",
       " 'western',\n",
       " 'blotting',\n",
       " 'analysis',\n",
       " 'supplementary',\n",
       " 'figure',\n",
       " 's5',\n",
       " 'aroc-',\n",
       " 'MenF',\n",
       " 'fusion',\n",
       " 'protein',\n",
       " 'two',\n",
       " 'enzyme',\n",
       " 'placed',\n",
       " 'close',\n",
       " 'proximity',\n",
       " 'thereby',\n",
       " 'increasing',\n",
       " 'apparent',\n",
       " 'concentration',\n",
       " 'intermediate',\n",
       " 'chorismate',\n",
       " 'around',\n",
       " 'MenF',\n",
       " 'relative',\n",
       " 'nonfusion',\n",
       " 'enzyme',\n",
       " 'resulting',\n",
       " 'higher',\n",
       " 'carbon',\n",
       " 'flux',\n",
       " 'MA',\n",
       " 'biosynthesis',\n",
       " 'pathway',\n",
       " 'figure']"
      ]
     },
     "execution_count": 85,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_word_9[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### NER Comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "with open(\"./data/saved_results/model/noner_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    noNer_abs = pickle.load(fp)\n",
    "\n",
    "    \n",
    "with open(\"./data/saved_results/model/noner_coherence.pickle\", \"rb\") as fp:\n",
    "    noNer = pickle.load(fp)\n",
    "\n",
    "with open(\"./data/saved_results/model/lda_k=5_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    ner = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.5773225598096631"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "noNer.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.517957605486875"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "noNer_abs.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.4248681962241365"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ner.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"./data/saved_results/model/lda_k=5_noner.pickle\", \"rb\") as fp:\n",
    "    noNerModel = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>yeast, codon, cassette, clone, trna, integration, bp, recombination, transformation, genomic, deletion, chromosome, sgrna, marker, locus, screening, crispr, replication, editing, template, round, assembled</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>circuit, input, network, output, specie, strand, simulation, device, gate, behavior, algorithm, module, noise, sbol, distribution, feature, population, domain, logic, term, feedback</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>gfp, circuit, induction, mrna, repression, translation, rb, sensor, iptg, regulatory, terminator, inducer, switch, biosensor, strength, activation, regulator, output, operator, repressor, upstream, ribosome, riboswitch, inducible, constitutive, degradation, atc, intensity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>domain, peptide, light, membrane, fusion, residue, surface, receptor, affinity, secretion, ligand, antibody, blue, activation, scaffold, image, supernatant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>glucose, mg, biosynthesis, titer, metabolite, carbon, compound, flux, fermentation, heterologous, biosynthetic, fatty, cluster, module, biomass, extract, operon, ethanol</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                         Terms per Topic\n",
       "Topic1                                                                     yeast, codon, cassette, clone, trna, integration, bp, recombination, transformation, genomic, deletion, chromosome, sgrna, marker, locus, screening, crispr, replication, editing, template, round, assembled\n",
       "Topic2                                                                                             circuit, input, network, output, specie, strand, simulation, device, gate, behavior, algorithm, module, noise, sbol, distribution, feature, population, domain, logic, term, feedback\n",
       "Topic3  gfp, circuit, induction, mrna, repression, translation, rb, sensor, iptg, regulatory, terminator, inducer, switch, biosensor, strength, activation, regulator, output, operator, repressor, upstream, ribosome, riboswitch, inducible, constitutive, degradation, atc, intensity\n",
       "Topic4                                                                                                                       domain, peptide, light, membrane, fusion, residue, surface, receptor, affinity, secretion, ligand, antibody, blue, activation, scaffold, image, supernatant\n",
       "Topic5                                                                                                         glucose, mg, biosynthesis, titer, metabolite, carbon, compound, flux, fermentation, heterologous, biosynthetic, fatty, cluster, module, biomass, extract, operon, ethanol"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(noNerModel)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>yeast, codon, cassette, clone, trna, integration, bp, recombination, transformation, genomic, deletion, chromosome, sgrna, marker, locus, screening, crispr, replication, editing, template, round, assembled</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>circuit, input, network, output, specie, strand, simulation, device, gate, behavior, algorithm, module, noise, sbol, distribution, feature, population, domain, logic, term, feedback</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>gfp, circuit, induction, mrna, repression, translation, rb, sensor, iptg, regulatory, terminator, inducer, switch, biosensor, strength, activation, regulator, output, operator, repressor, upstream, ribosome, riboswitch, inducible, constitutive, degradation, atc, intensity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>domain, peptide, light, membrane, fusion, residue, surface, receptor, affinity, secretion, ligand, antibody, blue, activation, scaffold, image, supernatant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>glucose, mg, biosynthesis, titer, metabolite, carbon, compound, flux, fermentation, heterologous, biosynthetic, fatty, cluster, module, biomass, extract, operon, ethanol</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                         Terms per Topic\n",
       "Topic1                                                                     yeast, codon, cassette, clone, trna, integration, bp, recombination, transformation, genomic, deletion, chromosome, sgrna, marker, locus, screening, crispr, replication, editing, template, round, assembled\n",
       "Topic2                                                                                             circuit, input, network, output, specie, strand, simulation, device, gate, behavior, algorithm, module, noise, sbol, distribution, feature, population, domain, logic, term, feedback\n",
       "Topic3  gfp, circuit, induction, mrna, repression, translation, rb, sensor, iptg, regulatory, terminator, inducer, switch, biosensor, strength, activation, regulator, output, operator, repressor, upstream, ribosome, riboswitch, inducible, constitutive, degradation, atc, intensity\n",
       "Topic4                                                                                                                       domain, peptide, light, membrane, fusion, residue, surface, receptor, affinity, secretion, ligand, antibody, blue, activation, scaffold, image, supernatant\n",
       "Topic5                                                                                                         glucose, mg, biosynthesis, titer, metabolite, carbon, compound, flux, fermentation, heterologous, biosynthetic, fatty, cluster, module, biomass, extract, operon, ethanol"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(noNerModel)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Terminal Testing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 187,
   "metadata": {},
   "outputs": [],
   "source": [
    "# model no abstract\n",
    "with open(\"../SBKS/data/saved_results/model/lda_k=5.pickle\", \"rb\") as fp:\n",
    "    v1 = pickle.load(fp)\n",
    "    \n",
    "# model 2 no abstract (for debugging)\n",
    "with open(\"../SBKS/data/saved_results/model/lda_k=5_v2.pickle\", \"rb\") as fp:\n",
    "    v2 = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 188,
   "metadata": {},
   "outputs": [],
   "source": [
    "# model with abstract\n",
    "with open(\"../SBKS/data/saved_results/model/lda_k=5_abstract.pickle\", \"rb\") as fp:\n",
    "    lda_abstract = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "metadata": {},
   "outputs": [],
   "source": [
    "# datawords\n",
    "with open(\"../SBKS/data/saved_results/data_words/ner=20210205_q=0.005.pickle\", \"rb\") as fp:\n",
    "    d1 = pickle.load(fp)\n",
    "    \n",
    "# datawords\n",
    "with open(\"../SBKS/data/saved_results/data_words/ner=20210205_q=0.005_v2.pickle\", \"rb\") as fp:\n",
    "    d2 = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 164,
   "metadata": {},
   "outputs": [],
   "source": [
    "# articles\n",
    "with open(\"../SBKS/data/saved_results/data_words/article_sq.pickle\", \"rb\") as fp:\n",
    "    articles = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>circuit, input, network, output, behavior, gate, strand, simulation, device, module, algorithm, noise, distribution, population, degradation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>ion, membrane, residue, peptide, surface, affinity, fusion, secretion, light</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, carbon, cluster, metabolite, fermentation, flux, deletion, biosynthetic, module, heterologous, compound, fatty, strains, cerevisiae, biomass, synthase, overexpression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>terminator, mrna, bp, GFP, repression, tRNA, cassette, rb, ribosome, strength, transformation, strains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>sensor, light, activation, ligand, GFP, switch, biosensor, riboswitch, receptor, mCherry, circuit, ed, intensity, output, supplementary, mammalian</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                    Terms per Topic\n",
       "Topic1                                                 circuit, input, network, output, behavior, gate, strand, simulation, device, module, algorithm, noise, distribution, population, degradation\n",
       "Topic2                                                                                                                 ion, membrane, residue, peptide, surface, affinity, fusion, secretion, light\n",
       "Topic3  biosynthesis, titer, carbon, cluster, metabolite, fermentation, flux, deletion, biosynthetic, module, heterologous, compound, fatty, strains, cerevisiae, biomass, synthase, overexpression\n",
       "Topic4                                                                                       terminator, mrna, bp, GFP, repression, tRNA, cassette, rb, ribosome, strength, transformation, strains\n",
       "Topic5                                           sensor, light, activation, ligand, GFP, switch, biosensor, riboswitch, receptor, mCherry, circuit, ed, intensity, output, supplementary, mammalian"
      ]
     },
     "execution_count": 167,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(v1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>circuit, input, network, output, behavior, gate, strand, simulation, device, module, algorithm, noise, distribution, population, degradation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>ion, membrane, residue, peptide, surface, affinity, fusion, secretion, light</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, carbon, cluster, metabolite, fermentation, flux, deletion, biosynthetic, module, heterologous, compound, fatty, strains, cerevisiae, biomass, synthase, overexpression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>terminator, mrna, bp, GFP, repression, tRNA, cassette, rb, ribosome, strength, transformation, strains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>sensor, light, activation, ligand, GFP, switch, biosensor, riboswitch, receptor, mCherry, circuit, ed, intensity, output, supplementary, mammalian</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                    Terms per Topic\n",
       "Topic1                                                 circuit, input, network, output, behavior, gate, strand, simulation, device, module, algorithm, noise, distribution, population, degradation\n",
       "Topic2                                                                                                                 ion, membrane, residue, peptide, surface, affinity, fusion, secretion, light\n",
       "Topic3  biosynthesis, titer, carbon, cluster, metabolite, fermentation, flux, deletion, biosynthetic, module, heterologous, compound, fatty, strains, cerevisiae, biomass, synthase, overexpression\n",
       "Topic4                                                                                       terminator, mrna, bp, GFP, repression, tRNA, cassette, rb, ribosome, strength, transformation, strains\n",
       "Topic5                                           sensor, light, activation, ligand, GFP, switch, biosensor, riboswitch, receptor, mCherry, circuit, ed, intensity, output, supplementary, mammalian"
      ]
     },
     "execution_count": 168,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(v2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 170,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                   Terms per Topic\n",
       "Topic1                                                                                                      mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary\n",
       "Topic2                                                                                  light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane\n",
       "Topic3  biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor\n",
       "Topic4                                         circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain\n",
       "Topic5                                                                                            yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation"
      ]
     },
     "execution_count": 170,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_abstract)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 189,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                   Terms per Topic\n",
       "Topic1                                                                                                      mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary\n",
       "Topic2                                                                                  light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane\n",
       "Topic3  biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor\n",
       "Topic4                                         circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain\n",
       "Topic5                                                                                            yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation"
      ]
     },
     "execution_count": 189,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_abstract)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 149,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d1 == d2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "491\n",
      "856\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(d1)):\n",
    "    if d1[i]!=d2[i]:\n",
    "        print(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "188\n",
      "189\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(d1[491])):\n",
    "    if d1[491][i]!=d2[491][i]:\n",
    "        print(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['tetra-', 'aspartic acid motif']"
      ]
     },
     "execution_count": 153,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d1[491][188:190]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['tetra-aspartic acid', 'motif']"
      ]
     },
     "execution_count": 154,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d2[491][188:190]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'sb8b00330'"
      ]
     },
     "execution_count": 156,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles[491]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 161,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Numerous clinical trials have shown that adeno-associated virus (AAV) is a promising gene therapy vector that has low immunogenicity and is nonpathogenic. Additionally, AAV can be engineered as a biocomputing nanoplatform capable of delivering genetic material in response to specific stimuli. Controllable and targeted gene delivery vectors are necessary to decrease negative off-target side effects and limit toxicity to healthy tissue. Such vectors also allow for systemically delivered therapeutics to act preferentially at specific locations in the body. To achieve more controlled and targeted gene delivery, we previously developed a protease-activatable AAV platform, called provector. The provector has switchable behavior and transduces cells only in the presence of matrix metalloproteinases (MMPs), which are overexpressed in many diseased states, such as various cancers, stroke, congestive heart failure, and atherosclerosis.The first generation provector was based on AAV serotype 2 (AAV2) and created by inserting a peptide lock into the capsid after location G586 near the heparan sulfate proteoglycan (HSPG) binding domain. The peptide lock is composed of MMP recognition sequences flanking an inactivating sequence (IS) (Figure). The IS serves to interfere with viruscell receptor interactions, thus greatly decreasing transduction when it is present on the capsid surface. Upon exposure to MMPs, the peptide lock is cleaved and IS removed from the capsid surface. The provector concept was then translated to AAV9 via incorporation of a peptide lock after G453 near the galactose binding domain. Cell surface glycans terminating in galactosyl residues is a major receptor for AAV9, and inhibition of virusgalactose interactions significantly reduces its ability to infect cells. The switchable behavior of the provector in response to MMP is robust, with the ON state being able to achieve up to 11.3-fold increased transduction compared to the OFF state for AAV2 provector and 5-fold increase for AAV9 provector. Further improvement of the ON/OFF ratio would allow for a more effective gene delivery vector with greater specificity to target tissues. In particular, the provectors currently transduce up to 10% of that of the wild-type capsid in the OFF state, which may lead to substantial off-target gene expression in vivo. Therefore, decreasing the OFF transduction level would further minimize nontarget delivery and increase the provector specificity.Provector overview. (a) Composition of peptide locks inserted into the AAV9 capsid after amino acid G453. The provector inactivating sequence (ProvIS) is in red and is flanked by glycine linkers and the MMP cleavage sequence, VPMSMRGG. The NgoMIV/KasI restriction sites (RE) make up the beginning and end of the peptide lock. (b) Overview of provector function. In the locked state, the peptide lock sequence blocks cell receptor binding and prevents the virus from transducing cells. When MMPs are prevalent, the lock sequences are cleaved and the virus is once again able to interact with and transduce cells.All provector platforms thus far have incorporated an IS that is four amino acids in length, with the tetra-aspartic acid motif (D4) being the most well-characterized. This motif was hypothesized to interfere with the electrostatic forces driving the interaction between AAV2 and HSPG. Further studies showed that steric hindrance and electrostatics both contribute to the disruption of virusreceptor interactions in the AAV2 provector. Additionally, the majority of IS variants besides the D4 motif tested thus far result in significant decreases in AAV2 provector production or capsid stability. For AAV9-based provectors, it is currently unclear how the IS affects binding between AAV9 and galactose. In order to further optimize the provector platform and to facilitate its translation to other AAV serotypes, the design rules for the peptide lock must be elucidated.In this study, we set out to define design rules regarding the IS for protease-activatable AAV9 vectors. We created a panel of AAV9 provectors with IS variants ranging from 0 (D0) to 10 (D10) aspartic acids in length. We characterized the physical properties (e.g., virus formation and stability) and the functional properties (e.g., protease cleavability and transduction activatability) of this panel of viral variants. In doing so, we were able to create an improved provector variant with an OFF transduction level of less than 1% of wild-type capsid and an ON/OFF transduction ratio of 27, the best achieved thus far.ResultsEffect of Inactivating Sequence Length on Virus Formation and Genome Protection AbilityProvIS (provector inactivating sequence) AAV mutants with inactivating sequences (IS) containing aspartic acid residues ranging from size 0 (D0, no IS) to size 10 (D10) were created to test the effect of IS length on provector formation and function (Table). The S1 mutant contains a scrambled MMP cleavage sequence that is not recognized by MMPs and thus acts as a negative control vector. Alteration of the IS changes not only the length but also the electrostatic charge of the inserted peptide lock sequence. The peptide lock sequences in the ProvIS mutants add from +2 (D0) to 8 (D10) charges to each viral protein (VP) subunit in the AAV capsid, which may affect virus formation and stability.All locks are flanked by NgoMIV/KasI restriction enzyme cut sites and are located after capsid location G453. MMP cleavage sequence is underlined, with arrow indicating cleavage location. Inactivating sequence is shown in bold. S1 contains the SMVGMRPG scrambled cleavage sequence, which is not cleavable by MMPs.To quantify the formation of the ProvIS mutants, cell lysates from ten 15 cm tissue culture plates were pooled, extracted using iodixanol gradient, and titered with qPCR. There are no statistically significant differences in vector production titers between wild-type (wt) AAV9 capsid vector and most of the ProvIS mutants (Figure a). The only exception is the mutant with no IS (i.e., D0), which has a significantly decreased virus titer. Next, an equal number of viral genomes of each ProvIS mutant was visualized on a Western blot with B1 antibody, which binds to a C-terminal epitope common to all three VPs (Figure b). Insertion of the peptide lock adds 2232 amino acids to all VP subunits in the viral capsid, with WT9 subunits having the smallest subunits and fastest migration on the protein gel, while mutant subunits from D0 to D10 undergo increasingly slower migration on the gel corresponding to increasingly larger molecular weight. S1 adds the same number of amino acids as D4 and both shift a similar distance. VP1/VP2/VP3 capsid protein ratios appear roughly the same as those of wt AAV9 capsid. No truncation products are seen, which suggests that the mutant viruses are stable at the level of each individual VP subunit.Structural characterization of ProvIS viruses. (a) Titer of 10 plate preparations of ProvIS viruses separated via iodixanol gradient ultracentrifugation and measured by qPCR. Standard error of mean (SEM) of two qPCR experiments are shown. (b) Visualization of ProvIS VP subunits via Western blotting. A total of 3  10 9 viral genomes was loaded on a 412% bis-Tris gel and probed with B1 antibody. (c) Genome protection assay of ProvIS viruses with plasmid control (Plas). Viruses were incubated with benzonase for 30 min. Genome protection is quantified as the ratio of intact viral genomes postincubation to sham-treated control. Two independent experiments were conducted, and error bars are SEM. Statistically significant difference from wt AAV9 capsid levels was calculated with one-way ANOVA and indicated with * for P  0.05.To test if the mutants are able to protect their encapsidated genomes from nuclease digestion, a benzonase assay was performed (Figure c). If the capsid is unstable, incubation with benzonase will digest ill-protected viral genomes. Genome protection was calculated as the ratio of viral titers remaining after benzonase treatment compared to no benzonase treatment. A plasmid-only control was included to ensure that the treatment conditions were adequate to digest all unprotected DNA. S1 shows a slight reduction in genome protection, whereas D0D10 mutants have protection levels similar to those of the wt AAV9 capsid. Taken together, these data suggest that completely abolishing the IS has a negative effect on virus formation but does not significantly affect genome protection, and extension of the IS length does not negatively affect virus formation or genome protection ability.Effect of Inactivating Sequence Length on Proteolytic SusceptibilityThe MMP recognition sequences in the peptide lock flank the IS (Figure), and we do not expect proteolytic cleavage to be affected by elongation of the central aspartic acid motif. To test this property, purified ProvIS mutants were incubated with MMP2, MMP7, and MMP9, and the cleavage results were visualized on a silver stain alongside the wt capsid (Figure). The expected VP fragments due to proteolysis are illustrated in Figure a. The control sham-treated samples were incubated with enzyme storage buffer containing no active enzyme, and thus we see all VP subunits intact (Figure b). Examination of the cleavage products shows that ProvIS mutants D0D10 are effectively cleaved by MMP2, MMP7, and MMP9. No cleavage is visible for the wt AAV9 capsid. Trace amounts of S1 subunits are cleaved by MMPs, with a substantial majority remaining uncleaved. Taken together, the results show that changing the IS length does not greatly affect the proteolytic susceptibility of the viruses.Proteolytic cleavage of ProvIS mutants. (a) Schematic of VPs illustrating the location of the peptide lock and expected cleavage fragments upon proteolysis. The peptide lock is inserted after amino acid G453 in the cap sequence. Thus, cleavage by MMP would result in distinct N-terminal cleavage fragments for each VP (VP1, VP2, VP3) and a common C-terminal cleavage product for all three VPs (C-term). Figure not drawn to scale. (b) Silver stain of sham (S)-treated and MMP2-, MMP7-, and MMP9-treated ProvIS viruses. Viruses were incubated with the indicated treatment for 20 h. Viral genomes (1.5  10 9) were denatured and run on a 412% bis-Tris gel.Protease-Activated Transduction Ability of ProvIS MutantsThe ability of ProvIS mutants to transduce cells was tested in vitro in the CHO-Lec2 cell line (Figure). Viruses carrying scGFP were treated with sham buffer or proteases and added to cells, and flow cytometry was used to quantify transduction efficiency. The transduction index (TI) is calculated as the percent of GFP positive cells multiplied by the geometric mean fluorescence intensity (TI = %GFP  gMFI) (Supplemental Figure S1). TI is the preferred measure of transduction efficiency because it scales linearly with transduction MOI (Supplemental Figure S2). This linear relationship allowed for retitering of proteolyzed virus samples to account for pipetting errors by using the titering results to normalize TI values (Figure a).Transduction efficiency of ProvIS viruses in OFF and ON states. (a) Transduction indices (TI, calculated as % GFP +  gMFI) normalized by the viral genomes added to cells, as quantified by titering sham-treated virus. Ratio of increase in TI for MMP9 unlocked virus compared to sham-treated locked virus is marked and indicated with * for P  0.05 by two-way ANOVA. Normalized TI of ProvIS viruses in (b) sham-treated OFF state and (c) MMP9-treated ON state. CHO-Lec2 cells were incubated with MOI (multiplicity of infection) 5000, and transduction efficiency was quantified with flow cytometry at 48 h post-transduction. Three independent experiments were conducted, and error bars are SEM; * indicates significance compared to wt AAV9 capsid, and + indicates significance compared to D0 by two-way ANOVA, P  0.05.Both wt AAV9 capsid and S1 control behave as expected and do not display protease-activatable behavior. D0 and D2 also do not display significant activatable behavior. The ratio of ON transduction versus OFF transduction (i.e., ON/OFF) increases as more aspartic acids are included in the IS from D4 to D10, ranging from 8.1-fold for D4 to 27-fold for D10. The D10 variant shows the greatest knock-down in transduction when locked, with a TI that is 0.9% of wt (Figure b). Although there is a significant decrease in TI from wt to locked D0, and from locked D0 to locked D2, the difference in transduction between locked D2D10 and S1 is not statistically significant. Upon unlocking via proteolytic cleavage, the ProvIS variants do not regain a wt level of transduction efficiency (Figure c). Unlocked D0 transduces best, at 37% of wt, though there are no statistically significant differences between the TIs of unlocked D0D10 variants. Additionally, there are no statistically significant differences between MMP2-, MMP7-, and MMP9-treated groups within each virus for the ProvIS mutants and S1. Taken together, these data show that the IS length does not impact virus unlocking but is necessary for locking the virus, and that increasing the number of aspartic acids in the IS increases the level of transduction knock-down in the OFF state.Cell Receptor Usage by ProvIS MutantsAAV9 binds to cell surface galactose in order to internalize into cells. As the peptide lock is located near the galactose binding site of the AAV9 capsid, we hypothesized that virus locking occurs primarily via interruption of galactose binding. Lectin competitively inhibits galactose binding by AAV, and thus the lectin competition assay was used to probe the galactose binding sensitivity of ProvIS mutants. As previously shown and reaffirmed here, the wt AAV9 capsids transduction decreases with increasing lectin concentration (Figure a). There is no statistically significant difference in lectin sensitivity between sham-and MMP-treated wt capsid virus, indicating that MMP treatment does not alter native viral galactose binding. In the ON state, the D0 and D4 ProvIS mutants respond as expected to lectin inhibition, with transduction efficiencies decreasing with increasing lectin concentration, leading to complete abolition of transduction at the highest lectin concentration (Figure b,c). Sham-treated D4 remains locked regardless of lectin concentration. These data suggest that the transduction efficiency restored by MMP cleavage in the D4 mutant is primarily due to restoration of galactose binding ability.Lectin competition transduction assay of D0 and D4 ProvIS mutants. Lectin inhibits transduction in (a) AAV9, (b) D4, and (c) D0. CHO-Lec2 cells were incubated with varying amounts of lectin to block galactose sites on the cell surfaces. Virus was incubated with sham buffer or MMP9 and then added to lectin-treated cells (MOI 5000) in a pulse transduction manner. Specifically, cells were incubated with the virus for 1 h on ice, after which cells were rinsed and then returned to 37 C. Flow cytometry was performed at 48 h to measure transgene expression. The * indicates significant difference from the corresponding 0 g/mL lectin condition as determined by two-way ANOVA, P  0.05. Unlabeled bars are not significantly different from 0 lectin. (d) Sham-and MMP-treated viruses were subjected to pulse transduction with no lectin present. Normalized transduction index (TI) was calculated relative to sham-or MMP9-treated wild-type AAV9. Two independent experiments were conducted, and error bars are SEM.Surprisingly, the sham-treated and MMP-treated D0 vectors transduce substantially differently in the lectin competition assay (Figure c) compared to the previous transduction assay (Figure). Under normal transduction conditions in which the viruses were incubated with cells for 24 h, sham-and MMP-treated D0 vectors transduce at above 30% of wt AAV9 (Figure). Under the lectin competition pulse-transduction protocol, which involves a 1 h viruscell incubation on ice followed by a PBS (phosphate-buffered saline) wash, sham-treated D0 transduction remains at above 30% of wt, whereas MMP-treated D0 transduction drops to less than 10% of wt (Figure c,d). It is expected that the overall transduction efficiency would be reduced with a shorter viruscell incubation time (24 h for regular transduction compared to 1 h for lectin competition). However, it is unexpected that this decrease in transduction is not proportional for the sham-versus MMP-treated D0 mutant. This difference suggests that the cellular transduction mechanism of D0 may differ between sham-and MMP-treated conditions.Transduction Kinetics of ProvIS MutantsTo further characterize the difference between ON and OFF ProvIS mutants, we performed a timed transduction study in which CHO-Lec2 cells were incubated with virus for various times, rinsed with PBS to remove unbound virus, and transduction quantified via flow cytometry at 48 h post-transduction (Figure). The TI for each incubation duration was also normalized to the maximum TI obtained for each virus at 24 h to find TI 50, the time at which the virus reaches 50% of the full transduction level (Figure d,e). Sham-treated D4 displays low transduction efficiencies at all incubation times tested, and the difference in TI at 1 versus 24 h of incubation is not statistically significant. Sham-treated D0 displays transduction kinetics similar to those of wt capsid but experiences an overall reduction in transduction efficiency. MMP-treated D0 and D4 have similar transduction profiles where the TI 50 occurs at a much later time (16 h) compared to sham-treated D0 and the wt AAV9 capsid (TI 50  8 h). In other words, the unlocked ProvIS vectors need to be in contact with cells for twice as long to achieve half of their maximum transduction levels compared to the wt capsid as well as to the locked D0 vector. This result helps to explain the previous discrepancies seen in TI values between transduction characterization (Figure) and lectin competition assay (Figure) and suggests that the cleavage scars left behind on unlocked ProvIS mutants cause a slowdown in transduction kinetics.Kinetics of sham-versus MMP9-treated provector transduction. (a) Schematic of experimental timeline. CHO-Lec2 cells were seeded, and MOI 5000 of virus was added for varying amounts of time followed by PBS wash and media change. Flow cytometry was performed at 48 h post-transduction to quantify (b,c) TI and (d,e) TI at each time point normalized by TI of each virus at 24 h. Data for the D4 sham is not shown in (d) because there is no statistically significant difference in TI after 1 h versus after 24 h incubation as measured by the student t test. Two independent experiments were conducted. Error bars are SEM and are only shown above each symbol. The * in (d,e) indicates significant difference compared to WT9 sham at each time point as determined by two-way ANOVA, P  0.05.DiscussionThe ProvIS mutants offer insights on the effect of IS length on the formation and switchability of protease-activatable AAV vectors. As lengthening the IS by increasing the number of aspartic acids results in incorporation of a greater number of charged residues into the capsid, we expected increased disturbance to the native virus capsid structure and thus decreased virus production and genome protection ability. Surprisingly, D0\\ue5f8the variant with no aspartic acids\\ue5f8forms at significantly lower titer, whereas the remaining mutants are similar to the wt AAV9 capsid (Figure a). This result demonstrates that a larger insert with more negative charge is not necessarily detrimental to virus formation. It is possible that elongation of the provector lock allows for more flexibility in the inserted peptide and thus puts less strain on the capsid attachment points and improves virus formation. None of the aspartic-acid-based ProvIS variants yields spontaneous VP truncation products (Figure b) as has been previously seen when four lysines were used instead of aspartic acids.Changing the IS length does not affect cleavage by MMP at the conditions tested in this study (Figure). This is expected because the MMP cleavage sequence is distinct from the IS and separated by glycine linkers. However, the conditions for the enzymatic reactions used in this study were carried out to completion based on previous provector testing. Thus, it is possible that the kinetics of MMP cleavage is affected by lengthening the distance between the two MMP cleavage motifs and/or by the increase in IS negative charge, and further enzymatic studies are needed to investigate these potential changes more closely.The IS appears to be essential for provector locking in the OFF state, with OFF state transduction decreasing with increasing IS size (Figure). In the panel tested here, no aspartic acids (D0) in the IS leads to least effective locking, with the locked and unlocked D0 achieving the same level of transduction (Figure). Two aspartic acid residues (D2) are also not enough to provide sufficient locking. Thus, the provector cannot be sufficiently locked without a dedicated IS that, if using consecutive aspartic acid motifs, must be four amino acids or longer.A limitation of the panel tested in this study is that changes in electric charge are coupled to changes in IS length. Therefore, it is difficult to conclude which property contributes more significantly to provector locking. Lectin competition assay with D0 and D4 suggests the restoration of transduction upon cleavage involves galactose binding (Figure). Although hydrophobic interactions have been shown to be important for galactose binding, galactose may also form hydrogen bonds with polar and charged amino acids. Aromatic residues that can contribute to hydrophobic interactions as well as polar/charged amino acids that can form hydrogen bonds can be found near the galactose binding site on AAV9. Thus, it is conceivable that disruption of either force can significantly affect galactose binding. Further studies on the IS would allow us to draw more concrete conclusions regarding which forces are dominant in inhibiting galactose binding in locked AAV9 provectors.Another aspect of provector function is restoration of transduction in the ON state. Interestingly, all cleaved ProvIS mutants achieve the same level of transduction in the ON state at around 3040% the level of wt capsid (Figure c). This decrease in transduction has been previously reported and is most likely due to the few amino acids of the lock that remain on the capsid after cleavage (cleavage scars) (Supplemental Figure S3). D0 is the only variant that has significant transduction in the OFF state, and timed transduction studies show that locked D0 has the same transduction kinetics profile as the wt capsid (Figure d). However, the transduction profiles of unlocked D0 and D4 have different transduction kinetics compared to wt or locked D0 capsid (Figure), with the unlocked viruses approaching their maximal transduction at a later time point. Therefore, locked and unlocked D0 reach similar transduction levels but have different kinetic profiles (Figure d), suggesting they operate through different mechanisms of transduction. Kinetic analysis of other members of the ProvIS panel besides D0 and D4 was not explored but may provide further insight on the effects of different IS motifs on transduction kinetics. It is possible that the cleavage scars confer the virus with the ability to transduce via an alternative unidentified pathway. It is also possible that the presence of the uncleaved peptide lock on sham-treated D0 provides an alternative transduction pathway. The existence of an alternate transduction pathway may also explain why provector transduction does not recover to the wt level after unlocking. Transduction of unlocked D0 and D4 mutants is still susceptible to lectin inhibition, however, suggesting that galactose binding is still a major component of their transduction pathway. The impact of the cleavage scar on avidity of AAV9 binding to galactose was not quantitatively explored in this study, and further examination of the cleavage scars is necessary to explain the altered provector transduction profile.Finally, the newly optimized D10 can be used as a basis for further improvement of the provector platform. Previous studies have shown that incorporation of wt subunits into a provector with a D4 IS can improve ON transduction levels. However, this incorporation also results in increased off-target transduction in the OFF state. The incorporation of wt subunits into D10 rather than D4 may result in an increase in ON transduction while maintaining an acceptable level of OFF transduction, further enhancing the ON/OFF switchability of the provector.ConclusionsThe IS of the provector peptide lock has been hypothesized to be necessary to block virus binding to cellular receptors, but its importance and impact had not been systematically investigated before. This study shows that the IS is necessary to block native receptor binding but does not affect transduction levels in the ON state. An aspartic acid motif of at least four residues in length is necessary for sufficient locking of the AAV capsid, with increasing length corresponding to better locking. The longest aspartic-acid-based motif tested thus far, D10, is optimal and yields a 27-fold ON/OFF transduction ratio, with OFF transduction at about 1% of that of the wt capsid. The remaining cleavage scars on unlocked provectors appear to negatively affect virus transduction and may cause the virus to transduce cells through an alternate pathway. In sum, this study outlines a basic design rule establishing the importance and impact of the IS in the provector peptide lock. More extensive studies would allow for further improvement of the protease-activatable AAV platform.Materials and MethodsPlasmid CloningProvIS plasmids were constructed based on the pAAV9453-L001 plasmid, which has AAV2 rep and AAV9 cap with a peptide lock inserted in the cap gene after G453. Forward primers attaching downstream of the inactivating sequence and reverse primers attaching upstream of the inactivating sequence (Supplemental Table S1) containing various numbers of aspartic acid codons were used in polymerase chain reaction (PCR) with Phusion HF polymerase (New England BioLabs) to generate ProvIS mutants containing 0 (D0) to 10 (D10) aspartic acids in the inactivating sequence (IS). The 5 ends of primers were phosphorylated using T4 polynucleotide kinase (PNK; New England BioLabs) under manufacturer recommended conditions. T4 PNK was inactivated by incubating at 65 C for 20 min. The optimized two-step PCR conditions for generating D0D8 were as follows: 1 Phusion HF buffer, 0.2 mM dNTPs, 0.5 M forward primer, 0.5 M reverse primer, 0.2 ng/L template DNA (pAAV9453-L001), 3% DMSO, and 0.4 units of Phusion polymerase, with thermocycler conditions (98 C for 30 s for initial denaturation, then 25 cycles of denaturation at 98 C for 10 s and annealing/extension at 72 C for 2 min 30 s, and final extension at 72 C for 10 min). D10 was generated using the same PCR conditions except pAAV9453-L001-D8 was used as the template. PCR products were run on 0.7% agarose gel with GelStar stain (Lonza) and extracted using Zymoclean gel DNA recovery kit (Zymo Research). Blunt-end ligation was performed using T4 DNA ligase (New England BioLabs) overnight at 16 C, and the ligated product was transformed into NEB10--competent. All plasmids were sequence-verified by an external vendor (GeneWiz).Virus ProductionVirus was produced using triple-plasmid transfection in 1.6:2:2 molar ratio of (1) pXX680, helper plasmid encoding adenoviral genes, (2) self-complementary GFP (scGFP) transgene cassette containing ITRs, and (3) plasmid containing rep and cap including the appropriate ProvIS mutation. HEK293T cells were seeded on poly-l-lysine-coated 15 cm plates and grown using DMEM medium with 10% fetal bovine serum (FBS; Gemini Bio-Products) and 1% penicillinstreptomycin (P/S, Thermo Fischer Scientific). At 8090% confluency, 40 g of total DNA was added with 7.5 M polyethylenimine (Polysciences). Medium was changed at 24 h post-transfection, and cells were harvested at 48 h post-transfection. Protease inhibitor cocktail (Sigma-Aldrich) was added to collected cells; cells were lysed with three freezethaw cycles, and virus was extracted using iodixanol gradient and ultracentrifugation as previously described. Virus was diluted in 1 gradient buffer (GB, 10 mM Tris, 10 mM MgCl 2, 150 mM NaCl) with 0.001% Pluronic F-68 (Sigma-Aldrich) and buffer-exchanged and concentrated using Amicon Ultra-15 100 kDa centrifugation units (Millipore).Virus Titer QuantificationThe genomic titer of viruses was measured using quantitative polymerase chain reaction (qPCR) as described previously. Briefly, a denatured and diluted virus sample was mixed with SYBR green (Applied Biosystems) and primers against the CMV promoter (forward: 5-TCACGG\\xadGGATTT\\xadCCAAGTCTC-3 and reverse: 5-AATGGGG\\xadCGGAGTT\\xadGTTACGA-3), and the reaction quantified using the Bio-Rad C1000 thermal cycler.Cellular Transduction AssayTo measure the cellular transduction efficiency of OFF versus ON ProvIS viruses in vitro, CHO-Lec2 cells were seeded on poly-l-lysine-coated 48-well plates at 5  10 4 cells/well. At 7080% confluency, viruses were added at multiplicity of infection (MOI) 5000 in serum-free MEM- medium with 1% P/S. After 24 h of incubation at 37 C, the medium was replaced with MEM- containing 10% FBS and 1% P/S. The cells were harvested at 48 h for flow cytometry with FACSCanto II flow cytometer (BD Biosciences).Nuclease Protection AssayTo estimate the degree of viral genome protection against nuclease digestion, virus was first incubated in 1 GB for 20 h at 37 C, and then capsid protection of its genome was measured using the benzonase protection assay. Virus in endo buffer (15 mM MgCl 2, 5 mg/mL BSA, 500 mM Tris pH 8.0) was treated with benzonase (250 units/L, Sigma) or sham buffer (50% glycerol, 20 mM Tris pH 8.0, 20 mM NaCl, 2 mM MgCl 2) for 30 min at 37 C. The reaction was terminated with 0.5 M EDTA, and remaining viral genomes were quantified via qPCR as described above.Western Blot and Silver StainWestern blotting was used to visualize viral capsid proteins. A total of 3  10 9 viral genomes of iodixanol-extracted virus was heated at 75 C for 15 min with LDS and NuPAGE sample reducing agent (Thermo Fisher Scientific). The sample was separated via electrophoresis on a NuPAGE 412% bis-Tris gel (Life Technologies) and transferred onto a nitrocellulose membrane. The common C-terminal of capsid proteins was probed using B1 primary antibody (ARP American Research Products) and goat anti-mouse IgG-HRP secondary antibody (Santa Cruz Biotechnology). Viral capsid proteins and proteolyzed VP fragments were visualized using silver staining. A total of 1.5  10 9 viral genomes was denatured for 75 C for 15 min with LDS, separated via electrophoresis on NuPAGE 412% bis-Tris gel, and stained using the SilverQuest silver staining kit (Life Technologies).ProteolysisViruses were treated with matrix metalloproteinases (MMPs) to probe for cleavage susceptibility. Concentrated viruses were diluted 1:1 in MMP buffer (50 mM Tris-Cl pH 7.4, 150 mM NaCl, 5 mM CaCl 2) and treated with 0.317 M of recombinant human MMP2, MMP7, or MMP9 (catalytic domains, Enzo Life Sciences) or sham buffer (50 mM Tris-HCl pH 7.5, 1 mM CaCl 2, 300 mM NaCl, 5 M ZnCl 2, 0.1% Brig-35, 15% glycerol) for 20 h at 37 C. The reaction was terminated with MMP stop buffer (1 GB with 0.001% Pluronic F-68 and 20 mM EDTA).Lectin Competition AssayThe galactose binding ability of ProvIS viruses was tested using a lectin competition assay. CHO-Lec2 cells were seeded on poly-l-lysine-coated 48-well plates at 5  10 4 cells/well. At 7080% confluency, serum-free MEM- medium with 1% P/S and 050 g/mL of unconjugated lectin (Vector Laboratories) dissolved in PBS was added and incubated with the cells on ice for 30 min. Virus was added at MOI 5000, and the cells were incubated for another 1 h on ice. The cells were then rinsed with sterile PBS and incubated at 37 C for 24 h with serum-free medium. At 24 h, serum-free medium was replaced with serum-containing medium. At 48 h post-transduction, cells were harvested for flow cytometry (FACSCanto II) to measure GFP expression.Statistical AnalysisData presented in Figure and Figure S1 were obtained in three independent experiments, with duplicates in each experiment. All other data were obtained in two independent experiments with duplicates in each experiment. The mean and standard error of the mean are plotted. Statistical analysis was done using Graph Pad Prism 7. As indicated in the figure legends, statistically significant differences were calculated using either one-way analysis of variance (ANOVA) or two-way ANOVA with Bonferrini post hoc test with a significance level of 0.05.'"
      ]
     },
     "execution_count": 161,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "jsonfiles[articles[491]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 183,
   "metadata": {},
   "outputs": [],
   "source": [
    "# coherence model with abstract\n",
    "with open(\"../SBKS/data/saved_results/model/lda_k=5_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model = pickle.load(fp)\n",
    "    \n",
    "# coherence model w/o abstract\n",
    "with open(\"../SBKS/data/saved_results/model/lda_k=5_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model_noabs = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.4248681962241365"
      ]
     },
     "execution_count": 180,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 184,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.45293548917502635"
      ]
     },
     "execution_count": 184,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model_noabs.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load Model\n",
    "with open(\"./data/saved_results/model/lda_k=5_abstract.pickle\", \"rb\") as fp:\n",
    "    lda_5 = pickle.load(fp)\n",
    "    \n",
    "# coherence model with abstract\n",
    "with open(\"./data/saved_results/model/lda_k=5_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model = pickle.load(fp)\n",
    "    \n",
    "# Load Model\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=5_abstract.pickle\", \"rb\") as fp:\n",
    "    lda_5_a_5 = pickle.load(fp)\n",
    "    \n",
    "# coherence model with abstract\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=5_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model_a_5 = pickle.load(fp)\n",
    "    \n",
    "# Load Model\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=30_abstract.pickle\", \"rb\") as fp:\n",
    "    lda_5_a_30 = pickle.load(fp)\n",
    "    \n",
    "# coherence model with abstract\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=30_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model_a_30 = pickle.load(fp)\n",
    "    \n",
    "# Load Model\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=100_abstract.pickle\", \"rb\") as fp:\n",
    "    lda_5_a_100 = pickle.load(fp)\n",
    "\n",
    "# coherence model with abstract\n",
    "with open(\"./data/saved_results/model/lda_k=5_a=100_abstract_coherence.pickle\", \"rb\") as fp:\n",
    "    coh_model_a_100 = pickle.load(fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                   Terms per Topic\n",
       "Topic1                                                                                                      mrna, terminator, repression, GFP, strength, ribosome, rb, membrane, IPTG, strains, RBS, supplementary\n",
       "Topic2                                                                                  light, domain, activation, ligand, ion, fusion, switch, sensor, receptor, ed, GFP, mCherry, intensity, mammalian, membrane\n",
       "Topic3  biosynthesis, titer, metabolite, carbon, fermentation, flux, compound, deletion, biosynthetic, biosensor, strains, fatty, heterologous, intracellular, recombinant, overexpression, supernatant, precursor\n",
       "Topic4                                         circuit, network, output, input, behavior, gate, strand, device, simulation, module, algorithm, noise, distribution, degradation, population, logic, switch, domain\n",
       "Topic5                                                                                            yeast, cassette, tRNA, residue, bp, CRISPR, recombination, cluster, DNA, incorporation, transformation, ligation"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.4248681962241365"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>module, algorithm, base, SBOL, RBS, class, predicted, nucleotide, DNA, user</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>yeast, cassette, bp, integration, CRISPR, recombination, GFP, ed, marker, mCherry, chromosome, locus, light, targeting, activation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>domain, ion, membrane, peptide, light, fusion, tRNA, residue, surface, affinity, ligand, secretion, incorporation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, output, network, input, behavior, GFP, mrna, gate, device, simulation, degradation, noise, repression, switch, strand, domain, population, inducer, IPTG, regulator, repressor, activation, operator, distribution, feedback</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>biosynthesis, carbon, titer, metabolite, flux, fermentation, strains, biosynthetic, heterologous, cluster, yeast, fatty, compound</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                              Terms per Topic\n",
       "Topic1                                                                                                                                                            module, algorithm, base, SBOL, RBS, class, predicted, nucleotide, DNA, user\n",
       "Topic2                                                                                                     yeast, cassette, bp, integration, CRISPR, recombination, GFP, ed, marker, mCherry, chromosome, locus, light, targeting, activation\n",
       "Topic3                                                                                                                      domain, ion, membrane, peptide, light, fusion, tRNA, residue, surface, affinity, ligand, secretion, incorporation\n",
       "Topic4  circuit, output, network, input, behavior, GFP, mrna, gate, device, simulation, degradation, noise, repression, switch, strand, domain, population, inducer, IPTG, regulator, repressor, activation, operator, distribution, feedback\n",
       "Topic5                                                                                                      biosynthesis, carbon, titer, metabolite, flux, fermentation, strains, biosynthetic, heterologous, cluster, yeast, fatty, compound"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5_a_5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.40679661251908356"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model_a_5.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>algorithm, module, SBOL, base, RBS, class, predicted, core, input, compound, file, user</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>light, domain, ion, membrane, fusion, peptide, surface, affinity, activation, ed, antibody, GFP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>carbon, titer, biosynthesis, flux, fermentation, metabolite, strains, compound, fatty, intracellular, heterologous, synthase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, output, network, input, GFP, mrna, device, behavior, gate, repression, noise, degradation, strand, population, switch, domain, simulation, repressor, IPTG, inducer, regulator, activation, operator, ligand, feedback</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>yeast, cassette, bp, terminator, tRNA, cluster, integration, cerevisiae, CRISPR, recombination, deletion, chromosome, transformation, marker, clone, strains, locus, GFP</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                        Terms per Topic\n",
       "Topic1                                                                                                                                          algorithm, module, SBOL, base, RBS, class, predicted, core, input, compound, file, user\n",
       "Topic2                                                                                                                                  light, domain, ion, membrane, fusion, peptide, surface, affinity, activation, ed, antibody, GFP\n",
       "Topic3                                                                                                     carbon, titer, biosynthesis, flux, fermentation, metabolite, strains, compound, fatty, intracellular, heterologous, synthase\n",
       "Topic4  circuit, output, network, input, GFP, mrna, device, behavior, gate, repression, noise, degradation, strand, population, switch, domain, simulation, repressor, IPTG, inducer, regulator, activation, operator, ligand, feedback\n",
       "Topic5                                                         yeast, cassette, bp, terminator, tRNA, cluster, integration, cerevisiae, CRISPR, recombination, deletion, chromosome, transformation, marker, clone, strains, locus, GFP"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5_a_30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.44419961366656563"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model_a_30.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Terms per Topic</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Topic1</th>\n",
       "      <td>module, algorithm, base, SBOL, predicted, RBS, strength, class, nucleotide, DNA, droplet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic2</th>\n",
       "      <td>yeast, cassette, bp, GFP, terminator, integration, CRISPR, sensor, recombination, ed, riboswitch, chromosome, marker, mCherry, day, targeting, activation, locus, clone</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic3</th>\n",
       "      <td>domain, light, membrane, ion, fusion, peptide, tRNA, surface, affinity, residue, secretion, ligand, antibody, intensity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic4</th>\n",
       "      <td>circuit, output, network, input, behavior, strand, gate, simulation, mrna, device, noise, degradation, GFP, domain, switch, population, repression, distribution, regulator, repressor, feedback, operator, logic, inducer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Topic5</th>\n",
       "      <td>biosynthesis, titer, carbon, metabolite, strains, flux, fermentation, biosynthetic, cluster, heterologous, yeast, compound, fatty, cerevisiae, biomass, precursor</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                   Terms per Topic\n",
       "Topic1                                                                                                                                    module, algorithm, base, SBOL, predicted, RBS, strength, class, nucleotide, DNA, droplet\n",
       "Topic2                                                     yeast, cassette, bp, GFP, terminator, integration, CRISPR, sensor, recombination, ed, riboswitch, chromosome, marker, mCherry, day, targeting, activation, locus, clone\n",
       "Topic3                                                                                                     domain, light, membrane, ion, fusion, peptide, tRNA, surface, affinity, residue, secretion, ligand, antibody, intensity\n",
       "Topic4  circuit, output, network, input, behavior, strand, gate, simulation, mrna, device, noise, degradation, GFP, domain, switch, population, repression, distribution, regulator, repressor, feedback, operator, logic, inducer\n",
       "Topic5                                                           biosynthesis, titer, carbon, metabolite, strains, flux, fermentation, biosynthetic, cluster, heterologous, yeast, compound, fatty, cerevisiae, biomass, precursor"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "topics_df(lda_5_a_100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.4214844247835856"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coh_model_a_100.get_coherence()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "Entrez.email = 'dxiang@ucsd.edu'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [],
   "source": [
    "search_query = 'ethics[Abstract] AND open access[filter] AND ((\"synthetic biology\"[MeSH Terms] OR (\"synthetic\"[All Fields] AND \"biology\"[All Fields]) OR \"synthetic biology\"[All Fields]) AND (\"ethics\"[Subheading] OR \"ethics\"[All Fields] OR \"ethics\"[MeSH Terms]))'\n",
    "handle = Entrez.esearch(db='pmc',\n",
    "                        sort='relevance',\n",
    "                        retmax='100000',\n",
    "                        term=search_query,\n",
    "                        usehistory='y')\n",
    "results = Entrez.read(handle)\n",
    "ids = results['IdList']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Count': '33', 'RetMax': '33', 'RetStart': '0', 'QueryKey': '1', 'WebEnv': 'MCID_6152917d8e61fe7a8d705dca', 'IdList': ['7123342', '7089176', '7226902', '7149545', '6630032', '7880687', '7014608', '5801888', '7863570', '5680775', '8316367', '8268180', '7052494', '6267627', '4681174', '6950820', '4071700', '3901125', '5147464', '6824826', '7805885', '6889464', '8435764', '7953679', '5561630', '8278188', '7683513', '4030158', '8011122', '5402672', '7853504', '8300107', '7695393'], 'TranslationSet': [], 'TranslationStack': [{'Term': 'dilemma[Abstract]', 'Field': 'Abstract', 'Count': '5243', 'Explode': 'N'}, {'Term': 'open access[filter]', 'Field': 'filter', 'Count': '3852273', 'Explode': 'N'}, 'AND', {'Term': '\"synthetic biology\"[MeSH Terms]', 'Field': 'MeSH Terms', 'Count': '1294', 'Explode': 'Y'}, {'Term': '\"synthetic\"[All Fields]', 'Field': 'All Fields', 'Count': '1000963', 'Explode': 'N'}, {'Term': '\"biology\"[All Fields]', 'Field': 'All Fields', 'Count': '1746653', 'Explode': 'N'}, 'AND', 'GROUP', 'OR', {'Term': '\"synthetic biology\"[All Fields]', 'Field': 'All Fields', 'Count': '20979', 'Explode': 'N'}, 'OR', 'GROUP', {'Term': '\"ethics\"[Subheading]', 'Field': 'Subheading', 'Count': '11390', 'Explode': 'Y'}, {'Term': '\"ethics\"[All Fields]', 'Field': 'All Fields', 'Count': '1269437', 'Explode': 'N'}, 'OR', {'Term': '\"ethics\"[MeSH Terms]', 'Field': 'MeSH Terms', 'Count': '15458', 'Explode': 'Y'}, 'OR', 'GROUP', 'AND', 'GROUP', 'AND'], 'QueryTranslation': 'dilemma[Abstract] AND open access[filter] AND ((\"synthetic biology\"[MeSH Terms] OR (\"synthetic\"[All Fields] AND \"biology\"[All Fields]) OR \"synthetic biology\"[All Fields]) AND (\"ethics\"[Subheading] OR \"ethics\"[All Fields] OR \"ethics\"[MeSH Terms]))'}"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 215,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(db='pmc', rettype='full', retstart=0, retmax=1, retmode='xml', webenv=results['WebEnv'], query_key=results['QueryKey'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(db='pmc', id='1470055', rettype='full',retmode='xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "parsed = Entrez.read(handle, validate=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['sub-article', 'response', 'front', 'body'])"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['list', 'def-list', 'notes', 'journal-meta', 'article-meta'])"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0]['front'].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['supplementary-material', 'chem-struct', 'table-wrap', 'preformat', 'related-article', 'disp-quote', 'graphic', 'sec', 'ack', 'fig-group', 'array', 'list', 'boxed-text', 'statement', 'verse-group', 'chem-struct-wrapper', 'p', 'speech', 'fig', 'mml:math', 'media', 'disp-formula', 'tex-math', 'def-list', 'table-wrap-group'])"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed[0]['body'].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pprint"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'abstract': [{'ack': [],\n",
      "               'array': [],\n",
      "               'boxed-text': [],\n",
      "               'chem-struct': [],\n",
      "               'chem-struct-wrapper': [],\n",
      "               'def-list': [],\n",
      "               'disp-formula': [],\n",
      "               'disp-quote': [],\n",
      "               'fig': [],\n",
      "               'fig-group': [],\n",
      "               'fn-group': [],\n",
      "               'glossary': [],\n",
      "               'graphic': [],\n",
      "               'list': [],\n",
      "               'media': [],\n",
      "               'mml:math': [],\n",
      "               'notes': [],\n",
      "               'object-id': [],\n",
      "               'p': ['The 12th meeting of the Scientific Group on Methodologies for the Safety Evaluation of Chemicals (SGOMSEC) considered the topic of methodologies for determining human and ecosystem susceptibility to environmental hazards. The report prepared at the meeting describes measurement of susceptibility through the use of biological markers of exposure, biological markers of effect, and biomarkers directly indicative of susceptibility of humans or of ecosystems. The utility and validity of these biological markers for the study of susceptibility are evaluated, as are opportunities for developing newer approaches for the study of humans or of ecosystems. For the first time a SGOMSEC workshop also formally considered the issue of ethics in relation to methodology, an issue of particular concern for studies of susceptibility.'],\n",
      "               'preformat': [],\n",
      "               'ref-list': [],\n",
      "               'related-article': [],\n",
      "               'sec': [],\n",
      "               'speech': [],\n",
      "               'statement': [],\n",
      "               'supplementary-material': [],\n",
      "               'table-wrap': [],\n",
      "               'table-wrap-group': [],\n",
      "               'tex-math': [],\n",
      "               'verse-group': []}],\n",
      " 'aff': ['National Institute of Environmental Health Sciences, Research Triangle Park, North Carolina, USA.'],\n",
      " 'article-categories': {'series-title': [],\n",
      "                        'subj-group': [{'subj-group': [],\n",
      "                                        'subject': ['Research Article']}]},\n",
      " 'article-id': [StringElement('9255554', attributes={'pub-id-type': 'pmid'}),\n",
      "                StringElement('1470055', attributes={'pub-id-type': 'pmc'})],\n",
      " 'conference': [],\n",
      " 'contract-num': [],\n",
      " 'contract-sponsor': [],\n",
      " 'contrib-group': [{'address': [],\n",
      "                    'aff': [],\n",
      "                    'author-comment': [],\n",
      "                    'bio': [],\n",
      "                    'contrib': [{'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'J C',\n",
      "                                           'surname': 'Barrett'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'H',\n",
      "                                           'surname': 'Vainio'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'D',\n",
      "                                           'surname': 'Peakall'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []},\n",
      "                                {'address': [],\n",
      "                                 'aff': [],\n",
      "                                 'author-comment': [],\n",
      "                                 'bio': [],\n",
      "                                 'collab': [],\n",
      "                                 'degrees': [],\n",
      "                                 'email': [],\n",
      "                                 'etal': [],\n",
      "                                 'ext-link': [],\n",
      "                                 'fn': [],\n",
      "                                 'name': [{'given-names': 'B D',\n",
      "                                           'surname': 'Goldstein'}],\n",
      "                                 'on-behalf-of': [],\n",
      "                                 'role': [],\n",
      "                                 'string-name': [],\n",
      "                                 'uri': [],\n",
      "                                 'x': [],\n",
      "                                 'xref': []}],\n",
      "                    'email': [],\n",
      "                    'etal': [],\n",
      "                    'ext-link': [],\n",
      "                    'fn': [],\n",
      "                    'on-behalf-of': [],\n",
      "                    'role': [],\n",
      "                    'uri': [],\n",
      "                    'x': [],\n",
      "                    'xref': []}],\n",
      " 'email': [],\n",
      " 'ext-link': [],\n",
      " 'fpage': '699',\n",
      " 'issue': 'Suppl 4',\n",
      " 'issue-id': [],\n",
      " 'issue-title': [],\n",
      " 'kwd-group': [],\n",
      " 'lpage': '737',\n",
      " 'product': [],\n",
      " 'pub-date': [{'month': '6',\n",
      "               'year': '1997'}],\n",
      " 'related-article': [],\n",
      " 'self-uri': [],\n",
      " 'supplementary-material': [],\n",
      " 'title-group': {'alt-title': [],\n",
      "                 'article-title': '12th meeting of the Scientific Group on Methodologies for the Safety Evaluation of Chemicals: susceptibility to environmental hazards.',\n",
      "                 'subtitle': [],\n",
      "                 'trans-title': []},\n",
      " 'trans-abstract': [],\n",
      " 'uri': [],\n",
      " 'volume': '105',\n",
      " 'volume-id': [],\n",
      " 'x': []}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(parsed[0]['front']['article-meta'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'bytes' object has no attribute 'to_string'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-82-9ff7387e49ca>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m     \u001b[0mhandle\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mEntrez\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mefetch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdb\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'pmc'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mid\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mids\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrettype\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'full'\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mretmode\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'xml'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0;32mwith\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'../PMC data/ethics/pmc{}.xml'\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mformat\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mids\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'w'\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m         \u001b[0mf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwrite\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mhandle\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mto_string\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'bytes' object has no attribute 'to_string'"
     ]
    }
   ],
   "source": [
    "for ids in results['IdList']:\n",
    "    handle = Entrez.efetch(db='pmc', id=ids, rettype='full',retmode='xml')\n",
    "    with open('../PMC data/ethics/pmc{}.xml'.format(ids), 'w') as f:\n",
    "        f.write(handle.read())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "with open('../PMC data/ethics/6692427.xml', 'w') as f:\n",
    "    f.write(str(handle.read()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<generator object DataHandler.parse at 0x7fc69b07a250>"
      ]
     },
     "execution_count": 197,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "article"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "# def search(query):\n",
    "#     Entrez.email = 'dxiang@ucsd.edu'\n",
    "#     handle = Entrez.esearch(db='pmc', \n",
    "#                             sort='relevance', \n",
    "#                             retmax='20',\n",
    "#                             retmode='txt', \n",
    "#                             term=query)\n",
    "#     results = Entrez.read(handle)\n",
    "#     return results\n",
    "\n",
    "# def fetch_details(id_list):\n",
    "#     ids = ','.join(id_list)\n",
    "#     Entrez.email = 'dxiang@ucsd.edu'\n",
    "#     handle = Entrez.efetch(db='pmc',\n",
    "#                            retmode='txt',\n",
    "#                            id=ids)\n",
    "#     results = Entrez.read(handle)\n",
    "#     return results\n",
    "\n",
    "# if __name__ == '__main__':\n",
    "#     results = search('synthetic biology ethics')\n",
    "#     id_list = results['IdList']\n",
    "#     papers = fetch_details(id_list)\n",
    "#     for i, paper in enumerate(papers['PubmedArticle']):\n",
    "#         print(\"{}) {}\".format(i+1, paper['MedlineCitation']['Article']['ArticleTitle']))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
