{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import re\n",
    "import pickle\n",
    "import numpy as np\n",
    "from pprint import pprint\n",
    "from lxml import etree\n",
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "input_path = os.path.abspath(\"../acs-data/unzip-files/\")\n",
    "output_path = os.path.abspath(\"../acs-data/processed-files/\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# function to collect matching files and dirs\n",
    "def collect_files(root, res, pattern=\"\", collect_dirs=True):\n",
    "    \n",
    "    # go through all item in the dir\n",
    "    for item in os.listdir(root):\n",
    "        \n",
    "        # process item\n",
    "        item_path = os.path.join(root, item)\n",
    "        item_is_dir = os.path.isdir(item_path)\n",
    "        \n",
    "        # pull valid file in res\n",
    "        if re.match(pattern, item_path):\n",
    "            if not item_is_dir or collect_dirs:\n",
    "                res.append(item_path)\n",
    "        \n",
    "        # recursively collect all files\n",
    "        if item_is_dir:\n",
    "            collect_files(item_path, res, pattern, collect_dirs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total xml files: 1545\n"
     ]
    }
   ],
   "source": [
    "# collect all xml files\n",
    "xml_paths = []\n",
    "collect_files(input_path, xml_paths, pattern=\".*\\.xml$\", collect_dirs=False)\n",
    "print(f\"total xml files: %d\" % len(xml_paths))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# helps to extract text from paragraph\n",
    "def p_helper(node):\n",
    "    \n",
    "    # <p/> does not have text\n",
    "    if node.text is None:\n",
    "        return \"\"\n",
    "    \n",
    "    # each paragarph is put into a line\n",
    "    line_list = [node.text]\n",
    "    for child in node:\n",
    "\n",
    "        # get the text inside the child if the tag isn't \n",
    "        # named-content and inline-formula\n",
    "        # and the text following the child\n",
    "        if not child.tag in (\"named-content\", \"inline-formula\"):\n",
    "            line_list.append(\" \".join(child.xpath(\".//text()\")))\n",
    "        line_list.append(child.tail)\n",
    "\n",
    "    # there might be none in line_list\n",
    "        \n",
    "    # re dark magic\n",
    "    line = \" \".join(line_list)\n",
    "    line = line.strip()\n",
    "    line = line.replace(\"\\n\", \" \")\n",
    "\n",
    "    # clean up consecutive spaces\n",
    "    line = re.sub(\"\\s+\", \" \", line)\n",
    "\n",
    "    # fix the space around punctuation\n",
    "    line = re.sub(\"\\s([.,\\):;])\", r\"\\1\", line)\n",
    "    line = re.sub(\"\\(\\s\", r\"(\", line)\n",
    "    line = re.sub(\"\\s*([-/])\\s*\", r\"\\1\", line)\n",
    "    return line"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def kwd_helper(node):\n",
    "    \n",
    "    # return a keyword string\n",
    "    kwd_tokens = node.xpath(\".//text()\")\n",
    "    kwd = \" \".join(kwd_tokens).replace(\"\\n\", \" \").strip()\n",
    "    kwd = re.sub(\"\\s+\", \" \", kwd)\n",
    "    return kwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# this returns interesting titles\n",
    "# for example: intro, method, and results\n",
    "# return None for non interesting titles\n",
    "def title_helper(node):\n",
    "    \n",
    "    # extract text from title node\n",
    "    title = \" \".join(node.xpath(\".//text()\"))\n",
    "    title = title.replace(\"\\n\", \" \")\n",
    "    title = re.sub(\"\\s+\", \" \", title)\n",
    "    title = title.strip()\n",
    "    title = title.lower()\n",
    "    \n",
    "    print(title)\n",
    "    \n",
    "    # categorize title\n",
    "    res = []\n",
    "    if \"intro\" in title:\n",
    "        res.append(\"introduction\")\n",
    "    if \"result\" in title:\n",
    "        res.append(\"result\")\n",
    "    if \"discuss\" in title:\n",
    "        res.append(\"discussion\")\n",
    "    if \"material\" in title:\n",
    "        res.append(\"materials\")\n",
    "    if \"method\" in title or \"procedure\" in title:\n",
    "        res.append(\"method\")\n",
    "    if \"summary\" in title:\n",
    "        res.append(\"summary\")\n",
    "    return res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_body(root):\n",
    "    \n",
    "    # we are interested in the text in the body section\n",
    "    curr_title = []\n",
    "    text = []\n",
    "    text_nodes = root.xpath(\"/article/body//*[self::p or (self::title and not(ancestor::caption))]\")\n",
    "    for text_node in text_nodes:\n",
    "        \n",
    "        # handle title\n",
    "        if text_node.tag == \"title\":\n",
    "            tmp_title = title_helper(text_node)\n",
    "            if (tmp_title):\n",
    "                curr_title = tmp_title\n",
    "        \n",
    "        # handle paragraph\n",
    "        elif text_node.tag == \"p\":\n",
    "            text.append({\n",
    "                \"text\": p_helper(text_node),\n",
    "                \"section\": curr_title\n",
    "            })\n",
    "    return text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_abstract(root):\n",
    "    \n",
    "    # get the abstract paragraph\n",
    "    abstract = []\n",
    "    abstract_nodes = root.xpath(\"//abstract/p\")\n",
    "    if abstract_nodes:\n",
    "        abstract.append(p_helper(abstract_nodes[0]))\n",
    "    return abstract"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_keywords(root):\n",
    "    \n",
    "    # get the keywords\n",
    "    keywords = []\n",
    "    kwd_nodes = root.xpath(\"//kwd-group/kwd\")\n",
    "    for kwd_node in kwd_nodes:\n",
    "        keywords.append(kwd_helper(kwd_node))\n",
    "    return keywords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "068e980a8c034c8c96b5ee0d435ac566",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=1545.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00050/sb7b00050.xml\n",
      "results\n",
      "purification and characterization of acecas9 ribonucleoprotein particle\n",
      "acecas9 has a double cytosine pam\n",
      "dna unwinding limits acecas9 cleavage efficiency\n",
      "acecas9 cleaves negatively supercoiled dna efficiently\n",
      "acecas9 has low off-target cleavage activity for relaxed dna\n",
      "acecas9 cleaves dna efficiently in escherichia coli\n",
      "discussion\n",
      "experimental procedures\n",
      "cloning of acecas9 and acecas9 mutant\n",
      "protein expression and purification\n",
      "in vitro transcription and purification of sgrna\n",
      "pam library construction\n",
      "oligonucleotide dna cleavage assay\n",
      "plasmid dna cleavage assay\n",
      "generation of negatively supercoiled dna by dna gyrase\n",
      "single-turnover kinetic assay\n",
      "bacteria-based positive selection assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00301/sb6b00301.xml\n",
      "introduction\n",
      "results and discussion\n",
      "analysis of existing rna thermometers\n",
      "computational analysis of a thermometer library suggests a diversity of responses\n",
      "experimental characterization of a thermometer library\n",
      "assessment of match between computations and experiments\n",
      "summary and future work\n",
      "materials and methods\n",
      "computational analysis\n",
      "plasmids and bacterial strains\n",
      "measurements in a cell-free biomolecular breadboard\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300068g/sb300068g.xml\n",
      "results and discussion\n",
      "one-step metabolon purification and immobilization\n",
      "purification of cbm3-free synthetic metabolon\n",
      "substrate channeling in synthetic metabolons\n",
      "methods\n",
      "chemicals\n",
      "strains and medium\n",
      "construction of plasmids\n",
      "recombinant protein expression and purification\n",
      "one-step metabolon purification and immobilization\n",
      "purification of cbm3-free metabolon through intein self-cleavage\n",
      "enzymatic activity assays\n",
      "other assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00050/sb6b00050.xml\n",
      "results\n",
      "design and characterization of single target gene perturbation devices\n",
      "perturbation of gene expression influences bacterial growth characteristics during stress exposure\n",
      "competition assay corroborates fitness impacts of gene perturbation\n",
      "phenotypic reversibility and evidence of nongenetic impact of transcriptional perturbation\n",
      "design and characterization of strains perturbing multiple targets simultaneously\n",
      "simultaneous gene perturbation predominantly yields negative epistatic interactions\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, media and culture conditions\n",
      "plasmid assembly\n",
      "rt-qpcr analysis\n",
      "mic determination\n",
      "stress growth conditions\n",
      "competitive fitness assays\n",
      "flow cytometry\n",
      "sequencing of experimental strains\n",
      "mutation fluctuation assay\n",
      "determination of epistasis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00034/sb6b00034.xml\n",
      "methods\n",
      "library construction\n",
      "comparison of monovalent and bivalent libraries\n",
      "library screening\n",
      "generation of yeast displaying ntl-1 and ctl-1 using a yeast dual-display system\n",
      "recombinant expression and purification of bvl-1\n",
      "estimation of k d\n",
      "size exclusion chromatography (sec)\n",
      "measurement of t m\n",
      "expression and purification of gfp1–9, ntl-gfp11 and gfp10-ctl-1\n",
      "in vitro “mix and read” fluorescence reconstitution assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00187/sb5b00187.xml\n",
      "methods\n",
      "strains and culture media\n",
      "plasmids and cloning methods\n",
      "lambda red recombineering and selections\n",
      "integration efficiency and rescue ability calculations\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00088/sb7b00088.xml\n",
      "results and discussion\n",
      "biodisplay programming, culturing, sampling and readout\n",
      "multiplexed characterization of bacterial strains\n",
      "biodisplay\n",
      "spores enable long-term storage\n",
      "summary\n",
      "methods\n",
      "materials\n",
      "device fabrication\n",
      "cell culture, transformation and spore formation\n",
      "cell arraying\n",
      "cell-display culturing\n",
      "imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00471/sb8b00471.xml\n",
      "results\n",
      "engineering of a third generation photosensitive degron module\n",
      "combining chemical control of protein biosynthesis with light control of stability\n",
      "engineering of a photosensitive transcription factor\n",
      "synergistic optogenetic multistep control (somco) by light-regulated modules\n",
      "discussion\n",
      "methods\n",
      "yeast strains, growth conditions, and plasmids\n",
      "microscopy\n",
      "fluorescence measurement\n",
      "immunoblotting, cycloheximide-chase assay, quantification, and fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400072v/sb400072v.xml\n",
      "polyketide precursor pathway engineering\n",
      "engineered streptomyces for secondary metabolite biosynthesis\n",
      "designing new-to-nature antimicrobial peptides\n",
      "synthetic biology of antimicrobial discovery\n",
      "modeling secondary metabolite production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300069k/sb300069k.xml\n",
      "results and discussion\n",
      "design and representation of libraries\n",
      "psbd 5′ utr element identification\n",
      "identification of regulatory elements within the psaa 5′ utr\n",
      "constructing a synthetic utr\n",
      "methods\n",
      "design and synthesis of utr libraries\n",
      "cloning and vector creation\n",
      "algal transformation, strain isolation, and expression analysis\n",
      "next-generation sequencing and data analysis\n",
      "synthesizing and cloning the synthetic utr\n",
      "xylanase assays\n",
      "quantitative pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00194/sb8b00194.xml\n",
      "results and discussion\n",
      "construction and verification of urease expression operons in e. coli\n",
      "effect of genes outside the urease gene cluster on caco 3 production in engineered e. coli\n",
      "effect of engineered e. coli strains on caco 3 phase and morphology\n",
      "caco 3 production using strains with different urease expression levels\n",
      "methods\n",
      "plasmid construction\n",
      "strain construction\n",
      "culture conditions\n",
      "caco 3 precipitation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500331x/sb500331x.xml\n",
      "results and discussion\n",
      "design of standardized and inducible gene expression tools in s. oneidensis\n",
      "enhanced production of flavins by a synthetic flavin pathway at optimized expression level\n",
      "enhanced current output in mfc\n",
      "eet kinetics revealed by bioelectrochemistry analysis\n",
      "mfc performance\n",
      "enhanced inward current\n",
      "methods\n",
      "in vitro gene synthesis\n",
      "plasmid construction and transformation\n",
      "culture condition for flavin production and flavin measurements\n",
      "bes setup\n",
      "electrochemical analysis\n",
      "confocal image\n",
      "electrode attached biomass measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00016/sb7b00016.xml\n",
      "results\n",
      "design of emma\n",
      "optimized overhangs to improve assembly efficiency\n",
      "assembly connectors\n",
      "benchmarking the mammalian assembly kit\n",
      "one-step combinatorial assembly for the generation of a pooled library of expression vectors\n",
      "hierarchical assembly of complex expression vectors\n",
      "discussion\n",
      "methods\n",
      "bacterial cell culture\n",
      "domestication of parts\n",
      "part entry vectors and receiver plasmid construction\n",
      "golden gate assembly protocol\n",
      "cell cultures and transfections\n",
      "transient transfection\n",
      "stable transfections\n",
      "bxb1-mediated integration\n",
      "flow cytometry and data analysis\n",
      "microscopy measurement and image processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00059/sb8b00059.xml\n",
      "letícia magalhães arruda\n",
      "nils j. h. averesch\n",
      "edward ding\n",
      "minji lee\n",
      "zilong li\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00040/sb8b00040.xml\n",
      "results\n",
      "closed loop rna-based controller designs\n",
      "mathematical models\n",
      "reference tracking with open and closed loop control\n",
      "large variations in the reference signal can be tracked accurately\n",
      "closed loop control enables disturbance rejection\n",
      "direct and indirect controllers demonstrate similar parameter sensitivity\n",
      "quantifying controller performance with control-theory metrics\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00020/sb9b00020.xml\n",
      "results and discussion\n",
      "cycle 1 design, build, and test: relationships among fatty alcohol titer, engineered pathway proteins, and rbs strength\n",
      "cycle 1 learn: machine-learning and model building based on dodecanol and pathway protein concentrations\n",
      "cycle 2 design and build: strategies for optimization of maqu_2507- and maqu_2220-expressing strains from cycle 1\n",
      "cycle 2 test and learn: strain performance and predictability\n",
      "conclusions\n",
      "methods\n",
      "plasmid and strain construction\n",
      "level 0 constructs and single-orf expression constructs\n",
      "level 1 constructs for dodecanol production\n",
      "production runs in a biolector microbioreactor\n",
      "fatty alcohols by gas chromatography–mass spectrometry (gc–ms)\n",
      "glucose and short-chain organic acids by high-performance liquid chromatography (hplc)\n",
      "targeted proteomics by lc–ms/ms\n",
      "proteomic sample preparation\n",
      "liquid chromatography–mass spectrometry\n",
      "in silico selected reaction monitoring (srm) methods selection\n",
      "fatty aldehyde analysis by nanostructure-initiator mass spectrometry (nims)\n",
      "fatty acid and acyl-coa analysis by lc–ms\n",
      "ensemble model construction\n",
      "cross validation of machine-learning models\n",
      "calculation of success probability of designed strains\n",
      "partial correlation analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003295/sb5003295.xml\n",
      "richard t. moore\n",
      "corina oßwald\n",
      "shingo suzuki\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00202/sb5b00202.xml\n",
      "results and discussion\n",
      "comparison of peak expression timing\n",
      "candidates for controlling nif cluster expression\n",
      "candidates for regulating other pertinent metabolic processes\n",
      "materials and methods\n",
      "cycling and homologous gene identification\n",
      "cluster generation\n",
      "network elucidation using optimization formulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001003/sb3001003.xml\n",
      "results and discussion\n",
      "properties of s. avermitilis as a versatile host\n",
      "heterologous expression of biosynthetic gene clusters of secondary metabolites\n",
      "expression of the biosynthetic gene clusters for compounds derived from the sugar pathway\n",
      "expression of the biosynthetic gene clusters for compounds derived from the polyketide pathway\n",
      "expression of the biosynthetic gene clusters for compounds derived from the amino acid pathway\n",
      "expression of the biosynthetic gene clusters for compounds derived from the shikimate pathway\n",
      "expression of the biosynthetic gene clusters for compounds derived from the mva or mep pathway\n",
      "productivity of secondary metabolites in a heterologous host\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "cultivation for secondary metabolite production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500241e/sb500241e.xml\n",
      "methods and materials\n",
      "strains and media\n",
      "vector construction and assembly preparation\n",
      "primer design and dna fragments preparation\n",
      "yeast transformation\n",
      "yeast whole plate cell population collection and plasmid recovery\n",
      "blue/white screening and pcr analysis of e. coli colonies\n",
      "dna sequences of the synthesized fragments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000367/sb4000367.xml\n",
      "results and discussion\n",
      "screening of amps with activity against gram-negative bacteria\n",
      "construction of recombinant l. lactis strains for amp production\n",
      "peptide production and secretion\n",
      "e. coli growth inhibition by alyteserin produced by l. lactis\n",
      "salmonella growth inhibition by alyteserin-1a and a3apo produced by l. lactis\n",
      "improving active peptide production\n",
      "summary\n",
      "methods\n",
      "synthetic peptide synthesis\n",
      "bacterial strains and growth conditions\n",
      "molecular biology\n",
      "construction of expression vectors\n",
      "protein production\n",
      "peptide transcript quantification by qpcr\n",
      "production and immunodetection of his-tagged proteins\n",
      "bioassays for antimicrobial activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00208/sb9b00208.xml\n",
      "results and discussion\n",
      "construction of allosteric cooperative dna i/o controllers with nonlinear regulation\n",
      "fine-tuning of the nonlinearities of the dna i/o controller\n",
      "reversible dynamic regulation of the nonlinearity of the dna i/o controller\n",
      "higher-order nonlinearities of a dna crn\n",
      "conclusion\n",
      "materials and methods\n",
      "dna oligonucleotides\n",
      "buffer conditions\n",
      "annealing\n",
      "design rationale for the reporter system\n",
      "characterization of cooperative allosteric chemical reaction networks using spectrofluorimetry\n",
      "dynamic reversible regulation of nonlinear behavior by introduction of cys\n",
      "construction of a nonlinear i/o chemical reaction network\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00057/sb6b00057.xml\n",
      "introduction\n",
      "results and discussion\n",
      "materials and methods\n",
      "strain construction\n",
      "transfer functions in liquid culture\n",
      "spatial patterning of gene expression on plates\n",
      "automated imaging acquisition\n",
      "mathematical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500368w/sb500368w.xml\n",
      "results and discussion\n",
      "deletion of the hydroxymandelic acid oxidase-encoding gene, hmo\n",
      "phylogenetic analysis reveals several orthologs of hmo and mdlb genes\n",
      "hmo and mdlb orthologs complement the s. coelicolor hmo mutant\n",
      "expression and purification of the hmo and mdlb variants\n",
      "enzyme activity of the hmo orthologs\n",
      "catalytic activity of the mdlb orthologs\n",
      "future perspectives\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "identification of hmo and mdlb orthologs\n",
      "construction of the s. coelicolor l -hpg pathway mutant\n",
      "cloning of the orthologous genes into pij10257 and pet28­(+)\n",
      "complementation of s. coelicolor lw107 with hmo and mdlb orthologous genes\n",
      "purification of the n-terminally his-tagged proteins\n",
      "western blot analysis\n",
      "enzyme activity determinations\n",
      "uhplc-ms analysis of cda production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00493/sb8b00493.xml\n",
      "results and discussion\n",
      "methods\n",
      "cell-free experiment preparation and execution\n",
      "pcr product preparation and plasmid dna assembly\n",
      "in vivo experiment\n",
      "microfluidic device experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00218/sb8b00218.xml\n",
      "results and discussion\n",
      "optimizing s. oneidensis control over nanoparticle biosynthesis and cellular viability\n",
      "loss of outer membrane cytochromes decreases pd nanoparticle size and alters cellular localization\n",
      "loss of outer membrane cytochromes attenuates pd­(ii) reduction\n",
      "mtrc expression controls extracellular pd nanoparticle formation\n",
      "flavins decrease outer membrane-nanoparticle size\n",
      "implications\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "bacterial strains and culture\n",
      "nanoparticle biosynthesis\n",
      "transmission electron microscopy\n",
      "nanoparticle size and number analysis\n",
      "pd­(ii) quantification\n",
      "powder x-ray diffraction\n",
      "flow cytomtery analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5001565/sb5001565.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and culture conditions\n",
      "standard oligonucleotides\n",
      "oligo library synthesis\n",
      "oligo library processing\n",
      "oligos for t7 cell library generation\n",
      "mage\n",
      "enrichment for t7 promoter containing sequences\n",
      "illumina dna sequencing and analysis\n",
      "pcr and freq-seq of 12 loci\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00213/sb8b00213.xml\n",
      "results and discussion\n",
      "construction of rna–protein hybrid input parts\n",
      "tuning of rna–protein hybrid input parts\n",
      "response of the hybrid input to an endogenous signal\n",
      "methods\n",
      "bacterial strains, plasmids, and oligonucleotides\n",
      "genetic circuit construction\n",
      "fluorescence measurement with coenzyme b 12 at various concentrations\n",
      "fitting of dose–response curves and calculation of ec 50\n",
      "validation of the circuit response to endogenously produced coenzyme b 12\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00077/sb5b00077.xml\n",
      "introduction\n",
      "results and discussion\n",
      "model derivation\n",
      "parameter estimation through sensitivity analysis-based experimental design\n",
      "model predictions\n",
      "understanding tx-tl batch variation using model parameterization\n",
      "conclusion\n",
      "methods\n",
      "sensitivity analysis to design parameterization experiments\n",
      "parameter estimation\n",
      "plasmid construction and purification\n",
      "tx-tl extract and buffer preparation\n",
      "extract preparation\n",
      "buffer preparation\n",
      "tx-tl experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00009/sb5b00009.xml\n",
      "timeline\n",
      "current applications of recombineering for genome engineering\n",
      "the recombineering mechanism\n",
      "complications in measuring efficiency\n",
      "increasing efficiency\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300025d/sb300025d.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00195/sb9b00195.xml\n",
      "results and discussion\n",
      "development of a tool for directed integration at 16s rdna sites\n",
      "improving mrna translation for enhanced protein expression\n",
      "expression analysis of different rrn promoters\n",
      "multiple rrn integration and stability analysis\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains, plasmids, and cultivation conditions\n",
      "plasmid construction\n",
      "genome integration\n",
      "growth and fluorescence analysis\n",
      "rna quantification via rt-qpcr\n",
      "confirmation of ribozyme autocatalytic activity\n",
      "expression stability test\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00407/sb7b00407.xml\n",
      "methods\n",
      "materials and general considerations\n",
      "structural comparisons of the pnuc and pnut proteins\n",
      "error-prone pcr mutagenesis\n",
      "construction of s. pneumoniae pnuc combinatorial variants by multiple site-directed mutagenesis\n",
      "site-saturation mutagenesis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400058n/sb400058n.xml\n",
      "results and discussion\n",
      "methods\n",
      "materials and reagents\n",
      "streptomycete cultivation, rna extraction, and real-time pcr analysis\n",
      "promoter cloning\n",
      "pathway refactoring and yeast transformation\n",
      "restriction digestion analysis\n",
      "heterologous expression in s. lividans\n",
      "liquid chromatography–mass spectrometry analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000194/sb3000194.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains, media, and growth conditions\n",
      "preparation of s. elongatus genomic dna for tar cloning\n",
      "assembly vector preparation\n",
      "assembly of ∼100-kb dna pieces in yeast\n",
      "isolation of yac dna from yeast\n",
      "isolation of bac dna from e. coli\n",
      "preparation of s. elongatus dna fragments propagated in e.coli for assembly\n",
      "lambda red insertion of ars sequence\n",
      "estimation of 454-kb s. elongatus clone stability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00154/sb7b00154.xml\n",
      "results and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00291/sb9b00291.xml\n",
      "results and discussion\n",
      "construction and characterization of initial stress sensors\n",
      "a repressor-based hsr of stress sensors\n",
      "sensing the nanomaterial triggered toxicity\n",
      "conclusion and future perspectives\n",
      "materials and methods\n",
      "media and strains\n",
      "plasmid construction\n",
      "heat shock experiments and toxicity assay\n",
      "fluorescence measurement and data analysis\n",
      "hspr expression and western blot analysis\n",
      "rna purification and cdna synthesis\n",
      "qpcr and data analysis\n",
      "time resolved fluorescence spectroscopy\n",
      "electron mobility shift assay (emsa)\n",
      "microscopy\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00479/sb8b00479.xml\n",
      "results\n",
      "evaluation of atp-fcs in c. glabrata\n",
      "atp-fcs enhances pyruvate synthesis as a final compound in glycolysis\n",
      "optimizing atp-fcs enhances the pyruvate accumulation\n",
      "coordination of the atp-fcs and other metabolic pathways\n",
      "discussion\n",
      "material and methods\n",
      "strains and plasmids\n",
      "plasmid construction and transformation\n",
      "media and culture conditions\n",
      "analysis methods\n",
      "real-time quantitative pcr (rt-qpcr)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00104/sb5b00104.xml\n",
      "a function-based, mutation-resistant, synthetic signaling device\n",
      "accurate predictions of genetic circuit behavior\n",
      "biofuel production by an engineered yeast platform\n",
      "high-efficiency multiplex genome editing of streptomyces using engineered crispr/cas\n",
      "web-based software tool for constraint-based design specification of synthetic biological systems\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00199/sb6b00199.xml\n",
      "results and discussion\n",
      "construction of the riboswitch-based dual-color reporter system\n",
      "characterization of theophylline riboswitch-based sensor\n",
      "time course study\n",
      "verification of fime-catalyzed inversion\n",
      "dose–response study\n",
      "riboswitch activation in a system with a degradation-tagged reporter\n",
      "stability of the riboswitch-based system\n",
      "creation of the reversible switch\n",
      "testing synthetic riboswitches with fime recombinase reporter system\n",
      "characterization of the riboswitch-recombinase systems\n",
      "time course study\n",
      "verification of fime-catalyzed inversion\n",
      "dose–response study\n",
      "stability of tnt riboswitch-based system\n",
      "creation of the reversible tnt riboswitch-based sensor\n",
      "conclusions\n",
      "methods\n",
      "reagents, bacterial strains, and plasmids\n",
      "time course experiments\n",
      "dose–response experiments\n",
      "fluorescence microscopy\n",
      "culture pcr\n",
      "maintenance of stable output memory in riboswitch-based system\n",
      "testing of the reversible switch\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00046/sb5b00046.xml\n",
      "geneguard: a modular plasmid system for biosafety\n",
      "mapping of amino acid substitutions conferring herbicide resistance\n",
      "speeding up directed evolution\n",
      "adaptive evolution of an artificial rna genome to a reduced ribosome environment\n",
      "using dynamic noise propagation to infer causal regulatory relationships\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500053j/sb500053j.xml\n",
      "results\n",
      "required and optional fields\n",
      "automated population of fields from registry pages\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003136/sb5003136.xml\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "construction of plasmids used for heterologous expression of antimycins\n",
      "biosynthesis of antimycins in e. coli\n",
      "hplc and lc–ms analysis of antimycin production\n",
      "enzymatic synthesis of antimycin a2\n",
      "formylation studies with e. coli overexpressing anto\n",
      "construction of s. albus s4 strains\n",
      "analysis of s. albus s4 strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00087/sb7b00087.xml\n",
      "results and discussion\n",
      "qs and negative feedback attenuate gene expression noise\n",
      "experimental results confirm computational predictions\n",
      "feedback pays off when extrinsic noise dominates\n",
      "quorum sensing helps feedback to cope with intrinsic noise\n",
      "tuning luxi expression allows noise strength to be minimized\n",
      "fast luxr turnover reduces luxi noise strength\n",
      "conclusions\n",
      "methods\n",
      "circuit description\n",
      "mathematical model\n",
      "reduced deterministic model\n",
      "stochastic model\n",
      "computational analysis\n",
      "strains and plasmids\n",
      "experimental protocol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00016/sb8b00016.xml\n",
      "results\n",
      "a hierarchical composition framework for multicellular boolean logic using integrase switches\n",
      "an automated design platform for recombinase logic\n",
      "discussion\n",
      "methods\n",
      "equations for determining of numbers of functions/strains/devices\n",
      "automated generation of genetic designs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00077/sb7b00077.xml\n",
      "results and discussion\n",
      "a toggle switch with diffusion can generate prolonged spatially cued patterns in a one-dimensional space\n",
      "a representative bistable model highlights conditions for nonhomogeneous steady state solutions\n",
      "eigenvalue analysis explains the prolonged transient behavior of nonhomogeneous steady state solutions\n",
      "an unbiased tug-of-war leads to spontaneous patterns in two-dimensional space\n",
      "a toggle switch with quorum sensing molecules produces patterns in two-dimensional space\n",
      "leakiness and crosstalk can improve patterning\n",
      "patterning is maintained with unequal diffusion coefficients\n",
      "prepatterned spatial profiles persist despite unequal production rates\n",
      "discussion\n",
      "methods\n",
      "simulations\n",
      "calculation of average dominant modes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00219/sb6b00219.xml\n",
      "introduction\n",
      "results\n",
      "expression and processing of siclopps constructs in hek-293 cells\n",
      "genetically encoded p1 inhibits hif-1 activity\n",
      "engineering physiological control of peptide expression\n",
      "endogenous p1 expression alters transcriptional response to hypoxia\n",
      "inhibition of hif-1 dimerization confers synthetic lethality to glycolysis inhibitor\n",
      "discussion\n",
      "methods\n",
      "cell culture\n",
      "transfection and selection of stable clones\n",
      "quantitative pcr\n",
      "western immunoblotting\n",
      "detection of peptide in cell lysates by hplc and ms\n",
      "hre luciferase reporter assay\n",
      "duolink proximity ligation assay\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hif-2α knockdown\n",
      "cell viability assays\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00429/sb8b00429.xml\n",
      "results\n",
      "initial design of crispri-mediated repression system for isopentenol production\n",
      "optimization of a tunable crispri system\n",
      "crispri negatively affects gene expression in the same operon\n",
      "efficient multiplex repression of crispri system reduced acetate production\n",
      "redirecting metabolic flux for isopentenol production\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "construction of plasmids\n",
      "strains, medium, and growth\n",
      "analysis and detection of isopentenol\n",
      "analysis and detection of rfp and gfp fluorescence\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300044r/sb300044r.xml\n",
      "motif control of input/output dynamics\n",
      "an incoherent feedforward loop can generate a biphasic response\n",
      "bistability arises via positive feedback\n",
      "increasing complexity in motif design can lead to autonomous dynamics\n",
      "cellular adaptation occurs via negative feedback and incoherent feed-forward loops\n",
      "network motifs underlying sustained, tunable, and synchronized oscillations\n",
      "synthetic circuits for studying cellular pattern formation\n",
      "role of noise in generating unique network dynamics\n",
      "concluding remarks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00488/sb8b00488.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400126a/sb400126a.xml\n",
      "results and discussion\n",
      "stochastic model for an enzymatic reaction under transcriptional feedback repression\n",
      "fast simulations of the stochastic model\n",
      "analytic characterization of metabolic noise\n",
      "discussion\n",
      "methods\n",
      "linear noise approximation\n",
      "derivation of the metabolic noise\n",
      "attenuation and amplification of metabolic noise\n",
      "theoretical limit of the metabolic noise for switch-like promoters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00359/sb6b00359.xml\n",
      "results and discussion\n",
      "experimental procedures\n",
      "dna constructs\n",
      "biochemical characterization of the kinase activity\n",
      "imaging of mda-mb-231 cells\n",
      "imaging of mouse hippocampal cells\n",
      "molecular modeling, discrete molecular dynamics, and correlative motion analysis\n",
      "sequence conservation and coevolution analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00010/sb9b00010.xml\n",
      "introduction\n",
      "results\n",
      "stochastic modeling of dna devices\n",
      "programmable dna device parameters\n",
      "tuning the lengths of binding sequence to create diversity\n",
      "tuning the number of domains to create diversity\n",
      "tuning the order by domain sequestering to create diversity\n",
      "tuning the dark-time with a competing secondary structure\n",
      "enhanced tunability with nanostructures\n",
      "using supervised machine learning for barcode classification\n",
      "discussion\n",
      "scaling the number of unique barcodes\n",
      "error-correcting optical barcodes using dna nanostructure\n",
      "insights for experimental demonstration\n",
      "cost-effectiveness of temporal dna barcodes\n",
      "methods\n",
      "generating temporal barcode for individual devices\n",
      "generating temporal barcode for nanostructures\n",
      "training svm model for supervised learning\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00100/sb6b00100.xml\n",
      "methods\n",
      "chemicals\n",
      "culture conditions\n",
      "ethoxyresorufin o-deethylation (erod) assay for cyp1a1 activity\n",
      "biophysical measurements\n",
      "molecular cloning, immunoblotting and quantification of cyp1a1, thylakoid preparation, dry cell weight determination, chlorophyll- a measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00007/sb8b00007.xml\n",
      "results and discussion\n",
      "design of de novo extender pn–blocks (epn–blocks) with various linkers to construct self-assembling chain-like nanostructures\n",
      "self-assembling homooligomers of epn–blocks\n",
      "reconstruction of heterooligomeric extender and stopper pn–block complexes by denaturation and refolding\n",
      "oligomeric state analyses of espn–block heterocomplexes\n",
      "shape analyses of espn–block heterocomplexes\n",
      "atomic force microscopy (afm) observations of self-assembling supramolecular nanostructures of espn–block complexes\n",
      "conclusions\n",
      "methods\n",
      "construction of protein expression plasmids\n",
      "protein expression and purification\n",
      "denaturation, refolding, and further purification of pn–block proteins\n",
      "size exclusion chromatography–multiangle light scattering (sec–mals)\n",
      "small-angle x-ray scattering (saxs)\n",
      "modeling analyses\n",
      "transmission electron microscopy (tem) imaging\n",
      "atomic force microscopy (afm) imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00288/sb6b00288.xml\n",
      "results and discussion\n",
      "recreation of two tar-envz chimeras (taz1 and tez1a1)\n",
      "modulation of signal output from taz1 and tez1a1 in response to osmolarity and aspartate\n",
      "aromatic tuning of taz1 and tez1a1\n",
      "response of the aromatically tuned taz1 and tez1a1 chimeras to aspartate\n",
      "creation and analysis of tuned narx-envz (navz1) chimeras\n",
      "summary and concluding remarks\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "population-based analysis of signal output\n",
      "single-cell analysis of signal output\n",
      "assessing expression of the v5-tagged chimeras\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00354/sb8b00354.xml\n",
      "results and discussion\n",
      "retrosynthetic analysis and design of csr for 5-ala biosynthesis\n",
      "metabolic flux balance analysis of 5-ala synthesis pathways\n",
      "proof-of-concept of the unnatural pathway in vitro\n",
      "in vivo production of 5-ala from glucose via the unnatural glyoxylate pathway\n",
      "conclusion\n",
      "materials and methods\n",
      "analytical methods\n",
      "enzyme assay and in vitro 5-ala production\n",
      "cultivation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00036/sb8b00036.xml\n",
      "results and discussion\n",
      "common yeast vectors produce a range of membrane protein yields and homogeneity\n",
      "integrating and modified high-copy vectors preclude plasmid loss associated with commercial, nonintegrating plasmids\n",
      "concurrent neo expression improves homogeneity of ste2p gpcr production in the pyes-kanmx backbone compared to an unmodified episomal vector\n",
      "facs screen enables rapid isolation of strains producing a 2 agh in high yields and homogeneity\n",
      "high gene dosage results in significantly improved functional a 2 agh yield\n",
      "full-length, functional a 2 agh is produced at the yeast plasma membrane\n",
      "materials and methods\n",
      "plasmid construction\n",
      "yeast strains and culturing conditions\n",
      "fluorescence-activated cell sorting\n",
      "ligand binding assays\n",
      "isolation of yeast strains with high a 2 agh gene dosage\n",
      "quantitative pcr\n",
      "plasmid loss assay\n",
      "sds-page, in-gel fluorescence, and western blotting\n",
      "confocal microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00463/sb7b00463.xml\n",
      "results\n",
      "the use of hexose transporter promoters serves to create cells able to detect and respond to different extracellular glucose levels\n",
      "implementation of different minimal circuits to produce hormones in a glucose-dependent manner\n",
      "the affinity and sensitivity of circuits can be tuned by modulating extracellular wiring molecule levels\n",
      "implementation of a time-dependent single pulse behavior by multicellular consortia\n",
      "discussion\n",
      "methods\n",
      "engineered yeast cell library and cell growth conditions\n",
      "protease purification\n",
      "synthetic pheromones\n",
      "fluorescence measurements using flow cytometry\n",
      "insulin and glucagon production and measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00164/sb8b00164.xml\n",
      "results and discussion\n",
      "design and construction of isoprene biosensor\n",
      "characterization of the isoprene biosensor\n",
      "improvement of the isoprene biosensor using a t7rnap-based amplification cascade\n",
      "chromosomal t7ispgess biosensor\n",
      "monitoring enhanced production of bioisoprene\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and reagents\n",
      "plasmid construction\n",
      "ispgess hce , ispgess ace , and ispgess trc\n",
      "t7ispgess trc , t7ispgess hce , t7ispgess trcsf , t7ispgess j23105sf , and t7ispgess j23114sf\n",
      "pseva221-ispgess trcsf and pseva331-ispgess trcsf\n",
      "bl21­(de3) p tbua1 :t7rnap strain construction\n",
      "bl21­(de3) p tbua1 :t7rnap t7ispgess strain construction\n",
      "culture conditions and isoprene supplementation\n",
      "isoprene determination\n",
      "fluorescence assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00275/sb9b00275.xml\n",
      "results\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00208/sb7b00208.xml\n",
      "results and discussion\n",
      "split-intein mediated protein ligation of sil-6r and il-6 generates active hyper-il-6\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "split 2 -cytokine protein-ligation: combination of split-il-6 variants and split-inteins generates i-h-il-6 ab‑cd -fc\n",
      "combination of split-il-11 variants and split-inteins generates h-il-11\n",
      "methods\n",
      "cells and reagents\n",
      "cloning\n",
      "structural analysis\n",
      "western blotting\n",
      "cytokine stimulation of cells for subsequent stat3 analysis\n",
      "enzyme-linked-immuno-sorbent-assay (elisa)\n",
      "proliferation assays\n",
      "affinity chromatography\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00362/sb7b00362.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and growth conditions\n",
      "crispr plasmid construction\n",
      "n. oceanica transformation\n",
      "nanoluciferase luminescence assays\n",
      "n. oceanica colony pcr\n",
      "episomal dna isolation from n. oceanica\n",
      "episome rescue\n",
      "episome curing\n",
      "segregation efficiency\n",
      "immunoblotting\n",
      "confocal microscopy\n",
      "southern-blot analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00201/sb6b00201.xml\n",
      "results and discussion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "measurement of gfp fluorescence\n",
      "total protein extraction\n",
      "western blotting analysis\n",
      "fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00120/sb8b00120.xml\n",
      "publication search\n",
      "optogenetic switches\n",
      "tools\n",
      "find the expert\n",
      "find the switch\n",
      "find the application\n",
      "materials\n",
      "illumination units converter\n",
      "trends in optogenetics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00057/sb5b00057.xml\n",
      "methods\n",
      "reagents\n",
      "culture conditions\n",
      "plasmid construction\n",
      "transformation of s. elongatus\n",
      "quantification of sfgfp expression\n",
      "acetoin quantification\n",
      "quantification of 23bd\n",
      "enzyme assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500286r/sb500286r.xml\n",
      "what is a microbial biosensor?\n",
      "whole cell microbial biosensors outputs\n",
      "whole cell multiple input microbial biosensors\n",
      "specificity of heavy metal biosensors\n",
      "overcoming the toxic nature of heavy metals\n",
      "expansion of microbial chassis for heavy metal biosensors\n",
      "microbial biosensors in action: from lab to field\n",
      "framework for future biosensor design\n",
      "risk and regulation of genetically modified microbes (gmms)\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300032y/sb300032y.xml\n",
      "results\n",
      "feature matching\n",
      "signal matching\n",
      "part matching\n",
      "discussion and future improvements\n",
      "methods\n",
      "feature matching\n",
      "extension to handle small molecule interactions and other features\n",
      "experimental setup\n",
      "signal matching\n",
      "extension to handle small molecule interactions and other features\n",
      "part matching\n",
      "algorithms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00109/sb6b00109.xml\n",
      "materials and methods\n",
      "plasmid and strain construction\n",
      "growth media\n",
      "shake flask cultivations\n",
      "fed-batch cultivations\n",
      "measurement of cell growth\n",
      "quantification of farnesene\n",
      "fret analysis\n",
      "production and quantification of phb\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00141/sb9b00141.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003407/sb5003407.xml\n",
      "flux-balance analysis with continuous gene expression\n",
      "pathway-based bayesian factor modeling\n",
      "results and discussion\n",
      "sensitivity and mean-square error analysis\n",
      "bacterial flux responses and pathway cross-correlations\n",
      "flux rate progression and temporal pathway activation\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500039z/sb500039z.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and fluorescent labeling\n",
      "tdmh and llo expression\n",
      "culture conditions and killing assay\n",
      "ph dependence of tdmh activity\n",
      "macrophage experiments and microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400068z/sb400068z.xml\n",
      "results and discussion\n",
      "lsrr can be modified through directed evolution\n",
      "reverse function lsrr\n",
      "enhanced lsrr\n",
      "docking of phospho-ai-2 into putative binding site in lsrr and mutants\n",
      "qs-mediated switch\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "lsrr directed evolution and reporter plasmid construction\n",
      "lsr operon expression\n",
      "lsrr sequencing and predicted structure modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400066m/sb400066m.xml\n",
      "results and discussion\n",
      "model annotation\n",
      "dna component assembly\n",
      "graph construction\n",
      "graph traversal\n",
      "discussion\n",
      "methods\n",
      "model annotation\n",
      "dna component assembly\n",
      "graph construction\n",
      "dfa construction\n",
      "graph traversal\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400174s/sb400174s.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "plasmids and strains\n",
      "cloning\n",
      "protein purification and cytotoxicity assay\n",
      "light-induction time courses\n",
      "blood agar assays\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00069/sb5b00069.xml\n",
      "results and discussion\n",
      "design and characterization of the malonyl-coa sensor in s. cerevisiae\n",
      "screening and isolation of malonyl-coa overproducing strains from a genome-wide library\n",
      "overexpression of tpi1 and pmp1 increased the production of 3-hydroxypropionic acid\n",
      "conclusions\n",
      "methods\n",
      "strains, media, and cultivation conditions\n",
      "dna manipulation\n",
      "sensor activity assays\n",
      "library creation and high-throughput screening\n",
      "3-hp production and quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400006v/sb400006v.xml\n",
      "timothy fenton\n",
      "aunica kane\n",
      "sabrina kille\n",
      "takafumi miyamoto\n",
      "keegan owsley\n",
      "andrew yao\n",
      "chun you\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00021/sb9b00021.xml\n",
      "results and discussion\n",
      "methods\n",
      "bacterial strains\n",
      "plasmid construction\n",
      "culture growth conditions for assay measurements\n",
      "fluorescence measurements\n",
      "fitness analysis\n",
      "nanoluciferase assay\n",
      "nanoluc expression and purification\n",
      "sample preparation for mass spectrometry proteome analysis\n",
      "swath based differential protein expression analysis\n",
      "gene ontology analysis\n",
      "generation of in-house off-target database\n",
      "targeted protein mass spectrometry of n-terminal peptide and off-target proteins\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00140/sb5b00140.xml\n",
      "generating systems biology markup language models\n",
      "fast and accurate circuit design automation through hierarchical model switching\n",
      "a hybrid of metabolic flux analysis and bayesian factor modeling for multiomic temporal pathway activation\n",
      "probabilistic analysis of localized dna hybridization circuits\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00465/sb8b00465.xml\n",
      "results\n",
      "double mutant of acinetobacter baylyi adp1 pobr and activity in pseudomonas putida kt2440 for 4hb biosensing\n",
      "intracellular 4hb production and sensing\n",
      "mutational library of ubic to reduce product inhibition\n",
      "benzoate as a product surrogate molecule for 4hb sensor-mediated high-throughput screening of ubic library for reduced product inhibition\n",
      "kinetic analysis of the ubic variants\n",
      "ubic as a metabolic shunt for cis,cis -muconic acid production in p. putida\n",
      "genomic integration of e. coli ubic gene in p. putida\n",
      "discussion\n",
      "materials and methods\n",
      "growth medium, conditions, and small molecules\n",
      "plasmids and strains\n",
      "vector construction for genomic integration\n",
      "genomic integration\n",
      "whole cell biosensing of 4hb and benzoate\n",
      "selecting mutagenesis sites in ubic and library creation\n",
      "selection of ubic variants with relieved product inhibition\n",
      "enzyme expression and purification\n",
      "determining enzyme kinetic parameters\n",
      "shake flask experiments for muconate production\n",
      "quantification of glucose and muconate concentrations using high-performance liquid chromatography (hplc)\n",
      "computational modeling of ubic-wt and mutants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000222/sb3000222.xml\n",
      "a protein interaction toolbox for synthetic peptides\n",
      "a sequence based server for protein engineering\n",
      "assembly of synthetic transmembrane proteins\n",
      "de novo proteins bind to small molecules\n",
      "evolutionary synthetic biology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00367/sb6b00367.xml\n",
      "definition of the target function and network coarse graining\n",
      "enumeration results for ultrasensitive networks\n",
      "experimental accessibility\n",
      "parameter optimization\n",
      "engineering the transcriptional parts for parameter-tuning\n",
      "experimental construction of the genetic circuit\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00182/sb9b00182.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400114p/sb400114p.xml\n",
      "methods\n",
      "plasmid construction\n",
      "luciferase assays\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00511/sb8b00511.xml\n",
      "methods\n",
      "tna oligonucleotide synthesis and preparation\n",
      "broad screen for tna ligase activity\n",
      "optimized condition for tna-tna ligation\n",
      "ligation junction assay\n",
      "rate determination assay\n",
      "mass spectrometry validation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000725/sb5000725.xml\n",
      "results and discussion\n",
      "methods\n",
      "synthesis of the aptamer and fluorophore\n",
      "nucleic acid assembly experiments\n",
      "fluorimeter studies\n",
      "dicer experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200019x/sb200019x.xml\n",
      "results and discussion\n",
      "d -amino acid oxidase\n",
      "map 2.0 3d analysis\n",
      "rgdaao directed evolution\n",
      "phytase\n",
      "map 2.0 3d analysis\n",
      "phytase directed evolution\n",
      "n -acetylneuraminic acid aldolase\n",
      "map 2.0 3d analysis\n",
      "neu5ac aldolase directed evolution\n",
      "conclusions\n",
      "methods\n",
      "mutational probability and statistics\n",
      "map indicators\n",
      "local chemical diversity and protein structure components\n",
      "map 2.0 3d server description\n",
      "map 2.0 3d output\n",
      "model proteins\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00326/sb8b00326.xml\n",
      "results\n",
      "ligand specificity mapping of the natural and synthetic biosensor circuits\n",
      "strategy 1: customized ligand specificity profile through chimeric effector modules\n",
      "strategy 2: customized ligand specificity profiles through chimeric detector modules\n",
      "material and methods\n",
      "strains and growth conditions\n",
      "plasmid construction\n",
      "in vivo fluorescence experiments\n",
      "data processing and statistical analysis\n",
      "sequence alignments\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400091p/sb400091p.xml\n",
      "construction of synthetic core promoter pcore11\n",
      "identification of variable sequence stretches of pcore aox1\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000723/sb3000723.xml\n",
      "results and discussion\n",
      "generation of the mouse artificial chromosome (mac) vector\n",
      "targeted integration of egfp and loxp sites into the mac\n",
      "transfer of the mac vectors from dt40 cells to chinese hamster ovary cells\n",
      "site-specific insertion of an exogenous gene into the mac2 vector\n",
      "transfer of the mac1 vector from cho cells to mouse embryonic stem cells\n",
      "retention rate of the mac1 vector in mouse tissues\n",
      "construction of mac2 with multiple integration sites (mi-mac)\n",
      "summary and conclusion\n",
      "methods\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cell culture\n",
      "construction of targeting vectors\n",
      "mmct\n",
      "transfection of dt40 and cho cells\n",
      "genomic pcr analysis\n",
      "fish analysis\n",
      "mfish analysis\n",
      "generation of chimeric mice\n",
      "fcm analyses of mouse es cells\n",
      "fcm analyses for tissues and blood cells\n",
      "insertion of multiple integration sites into mac2\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400069w/sb400069w.xml\n",
      "introduction\n",
      "results and discussion\n",
      "sensing potential as a reliable predictor of temporal qs dynamics\n",
      "experimental methodology to quantify qs-mediated activation\n",
      "materials and methods\n",
      "mathematical modeling of diverse qs systems\n",
      "strains, media, and growth conditions\n",
      "processing plate reader data\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00045/sb5b00045.xml\n",
      "andrew e. blanchard\n",
      "mihails delmans\n",
      "bin jia\n",
      "hanson lee\n",
      "jiazhang lian\n",
      "joanna lipinski-kruszka\n",
      "juergen mairhofer\n",
      "junwen mao\n",
      "ryo mizuuchi\n",
      "morten nørholm\n",
      "tong si\n",
      "gerald striedner\n",
      "maulik thaker\n",
      "nanxi wang\n",
      "weston whitaker\n",
      "oliver wright\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00305/sb8b00305.xml\n",
      "results and discussion\n",
      "computational approach to protein engineering\n",
      "selection of relevant amino acid residues\n",
      "ncaa-compatible ef-tu variants\n",
      "mass spectrometry confirms substrate-promiscuous ef variants\n",
      "ef-tu variant supports organismal fitness\n",
      "conclusion\n",
      "materials and methods\n",
      "reap library\n",
      "in vivo assay\n",
      "calculate ic 50\n",
      "protein purification\n",
      "in-gel digestion and mass spectrometry\n",
      "growth curves\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400187a/sb400187a.xml\n",
      "in situ overlap and sequence synthesis during dna assembly\n",
      "e. coli with two linear chromosomes\n",
      "yeast oligo-mediated genome engineering\n",
      "artificial conversion of mating-type without autopolyploidization\n",
      "genetically programmable pathogen sense and destroy\n",
      "controlling mammalian gene expression\n",
      "noncontiguous recombination to identify stabilizing mutations\n",
      "fractal patterning of cell layers by cell polarity-driven instability\n",
      "schema computational design of virus capsid chimeras\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00333/sb7b00333.xml\n",
      "results\n",
      "selection strategy\n",
      "selection cycle\n",
      "cloning assay\n",
      "effect on the n-16s rrna, including the native anti-sd\n",
      "translation of other proteins\n",
      "reconstitution with 50s subunit instead of tp50 and 23s rrna\n",
      "reconstitution–translation assay from rrna\n",
      "the effect of u1495c in vivo\n",
      "discussion\n",
      "methods\n",
      "plasmids\n",
      "dna and rna fragments for reconstitution\n",
      "ribosomal subunits, ribosomal proteins, and rrnas\n",
      "translation system\n",
      "liposome\n",
      "selection cycle\n",
      "reconstitution–translation assay in test tubes\n",
      "in vivo experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00147/sb7b00147.xml\n",
      "results\n",
      "design of the optogenetic switch\n",
      "characterization of the optogenetic switch\n",
      "light-inducible myogenic differentiation\n",
      "patterned myogenic differentiation\n",
      "light-inducible vegf 165 and ang1 flox vector\n",
      "characterization of the stable flox vpa polyclonal cell line\n",
      "light-inducible angiogenesis in vivo\n",
      "discussion\n",
      "methods\n",
      "plasmid construction and cell culture\n",
      "illumination\n",
      "western blot\n",
      "immunofluorescence staining\n",
      "elisa\n",
      "cell/hydrogel constructs\n",
      "mouse dorsal window chamber installation\n",
      "live intravital imaging of blood vessels\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00134/sb6b00134.xml\n",
      "methods\n",
      "strains, plasmids, and primers\n",
      "genome editing experiment\n",
      "plasmid construction for butanol production\n",
      "media and culture conditions\n",
      "citrate synthase activity assay\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00134/sb8b00134.xml\n",
      "results and discussion\n",
      "lactose transport and gdp- l -fucose accumulation in engineered s. cerevisiae\n",
      "production of 2-fl in engineered s. cerevisiae\n",
      "l -fucose production in engineered s. cerevisiae\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and media\n",
      "plasmids and strains construction\n",
      "yeast culture, fermentation, and metabolite analysis\n",
      "confirmation of α- l -fucosidase enzymatic activity in vitro\n",
      "identification of 2-fl and l -fucose using gc/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00431/sb7b00431.xml\n",
      "results and discussion\n",
      "plasmid construct and transgene organization\n",
      "homoplasmy of the transformants\n",
      "transformant protein analysis\n",
      "partial cmr and npti sequences as leader nucleotides in fusion constructs\n",
      "biomass accumulation and β-phellandrene productivity\n",
      "conclusions\n",
      "materials and methods\n",
      "synechocystis strains, recombinant constructs, and culturing conditions\n",
      "genomic dna pcr analysis of synechocystis transformants\n",
      "protein analysis\n",
      "quantification of β-phellandrene synthesis by synechocystis transformants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00532/sb8b00532.xml\n",
      "results and discussion\n",
      "encoding sbol entities\n",
      "encoding abstract entities\n",
      "representing relationships between design components\n",
      "the sbol vocabulary\n",
      "adding metadata classes\n",
      "querying using sbol-owl\n",
      "semantic reasoning for genetic circuit designs\n",
      "validating genetic circuits\n",
      "semanticsbol\n",
      "discussion\n",
      "methods\n",
      "constructing the ontology\n",
      "testing the ontology with standard definition of genetic circuits designs\n",
      "automated reasoning and querying\n",
      "availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00135/sb8b00135.xml\n",
      "results\n",
      "azide-containing proteins are successfully secreted and assembled extracellularly from engineered e. coli nissle 1917\n",
      "engineered curli fibers containing nonstandard amino acids can perform click chemistry in vitro\n",
      "click labeling is specific to mutant curli fibers\n",
      "labeling efficiency of engineered bacteria\n",
      "engineered probiotic bacteria with mutant curli fibers are able to perform click chemistry in vivo in murine model\n",
      "discussion\n",
      "experimental procedures\n",
      "cell strains and plasmids\n",
      "mice\n",
      "in vitro expression of curli fibers\n",
      "quantitative congo red binding assay\n",
      "whole-cell filtration elisa\n",
      "electron microscopy\n",
      "growth curves and doubling times\n",
      "in vitro labeling of click microbes\n",
      "labeling of simulated curli fiber mats\n",
      "purification and identification of proteins labeled in vitro\n",
      "flow cytometry and confocal microscopy\n",
      "in vivo tracking of click microbes\n",
      "whole gi tract ex vivo fluorescence imaging\n",
      "whole cecum fluorescence and protein analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00336/sb6b00336.xml\n",
      "results\n",
      "the de novo protein synglta rescues an auxotroph deleted for citrate synthase\n",
      "purified synglta protein has no detectable citrate synthase activity\n",
      "synglta alters gene expression in e. coli\n",
      "synglta alters the metabolome of e. coli\n",
      "methylcitrate synthase encoded by prpc has promiscuous catalytic activity\n",
      "e. coli prpc is required for the rescue of δ glta cells by synglta\n",
      "rescue of δ glta by synglta requires native regulation of prpc\n",
      "e. coli zrar is not required for rescue of δ glta cells by synglta\n",
      "discussion\n",
      "methods\n",
      "reagents\n",
      "strains\n",
      "dna methods\n",
      "protein expression and purification\n",
      "enzymatic assay\n",
      "rnaseq and differential expression analysis\n",
      "rt-qpcr\n",
      "lc/ms metabolomic analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00459/sb7b00459.xml\n",
      "results and discussion\n",
      "genetic circuit construction\n",
      "deriving dynamic models\n",
      "workflow example\n",
      "data integration\n",
      "enriching sbol designs\n",
      "discussion\n",
      "methods\n",
      "standards\n",
      "data curation\n",
      "data repositories\n",
      "data conversion\n",
      "cello tool for construction of genetic boolean logic gates\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500345t/sb500345t.xml\n",
      "results and discussion\n",
      "identification of the optimal minimal set of enzymes for 1,2-propanediol production\n",
      "disruption of lactate producing pathways\n",
      "carbon flux redistribution\n",
      "increasing nadh availability as a driving force\n",
      "cell adaptation studies\n",
      "1-propanol production\n",
      "dual strain strategies for improvement of 1-propanol production\n",
      "discussion\n",
      "conclusion\n",
      "methods\n",
      "chemicals and reagents\n",
      "bacterial strains and plasmids\n",
      "dna manipulations\n",
      "formate dehydrogenase enzyme assay\n",
      "culture medium\n",
      "whole cell conversion studies\n",
      "shake flask fermentations\n",
      "cell adaptation\n",
      "dual strain strategies for 1-propanol production\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400137b/sb400137b.xml\n",
      "crystal structure\n",
      "assembly of tale proteins\n",
      "functional domain based repression and activation\n",
      "nucleases\n",
      "recombinases\n",
      "methylated dna targets\n",
      "regulation of tale function\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500273n/sb500273n.xml\n",
      "results and discussion\n",
      "refactoring cph8-ompr\n",
      "identification of a novel cph8 mutation and ompf output promoter variant that improve sensor performance\n",
      "optimizing ompr expression level\n",
      "refactoring and optimization of ccas-ccar\n",
      "characterizing the v2.0 light sensors\n",
      "conclusions\n",
      "methods\n",
      "growth conditions, strains, and plasmids\n",
      "light exposure experiments\n",
      "flow cytometry and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00080/sb9b00080.xml\n",
      "results and discussion\n",
      "assessing the tolerance of procm toward mutations in proca2.8\n",
      "protease susceptibility and epitope grafting\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and materials\n",
      "mutagenesis\n",
      "small scale expression and isolation of proca2.8 variants\n",
      "ms analysis, nem labeling, and tandem ms\n",
      "hydrolysis, derivatization, and gc-ms analysis of pcn2.8­(c3s/s9c)\n",
      "large scale isolation and purification of proca2.8 variants\n",
      "fluorescence polarization competition assays\n",
      "protease stability assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400205x/sb400205x.xml\n",
      "results and discussion\n",
      "effect of mutations on photoreceptor activity\n",
      "effect of mutations on dark recovery and absolute light responsiveness\n",
      "effect of mutations on the flavin chromophore environment\n",
      "correlated effects on dark recovery and flavin chromophore environment\n",
      "implications for photoreceptor applications in biotechnology and synthetic biology\n",
      "methods\n",
      "molecular biology and protein expression\n",
      "yf1 in vivo activity assays\n",
      "absorption spectroscopy\n",
      "electron-paramagnetic resonance spectroscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00166/sb6b00166.xml\n",
      "results and discussion\n",
      "methods\n",
      "genetic engineering of plasmids and bacterial cell strains\n",
      "induction of curli synthesis\n",
      "assaying amyloid formation efficiency of csga with peptide insertions\n",
      "gold-nanoparticle formation\n",
      "gold-nanoparticle assembly and gold enhancement\n",
      "transmission electron microscopy (tem) imaging\n",
      "scanning electron microscopy (sem) imaging\n",
      "quartz crystal microbalance\n",
      "circular dichroism spectroscopy\n",
      "conductance measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00484/sb8b00484.xml\n",
      "introduction\n",
      "results and discussion\n",
      "identification of the u-responsive urprs tcs\n",
      "urprs exhibits improved metal selectivity compared to uzcrs\n",
      "construction of a u-responsive and gate pathway in c. crescentus\n",
      "u-sensing and gate exhibits improved selectivity relative to uzcrs alone\n",
      "integration of uzcy signal amplifier improves u sensitivity and selectivity\n",
      "whole-cell u sensor detects as low as 1.0 μm in contaminated groundwater\n",
      "methods\n",
      "bacterial strains, media, and materials\n",
      "clean deletions and site-directed mutagenesis\n",
      "transposon screen for regulators of p phyt\n",
      "construction of promoter- gfp transcriptional fusions\n",
      "tripartite gfp and gate sensor construction\n",
      "metal induction experiments\n",
      "cloning, overexpression, and purification of strep-urpr\n",
      "electrophoretic mobility shift assays\n",
      "site 300 sample collection\n",
      "inductively coupled plasma mass spectrometry/optical emission spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00206/sb6b00206.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00214/sb7b00214.xml\n",
      "results\n",
      "synechocystis transformations\n",
      "protein expression in wild type and isps strains\n",
      "pigment analysis\n",
      "photosynthetic activity\n",
      "rate of cell growth and isoprene accumulation\n",
      "enzymatic properties of cpcb*isps, cpcb*l7*isps, and cpcb*l16*isps in vitro\n",
      "modeling\n",
      "discussion\n",
      "materials and methods\n",
      "strains and culturing conditions\n",
      "isps-containing constructs\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pigment analysis\n",
      "protein analysis\n",
      "isoprene and biomass accumulation\n",
      "photosynthetic activity\n",
      "in vitro assay enzymatic activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500252a/sb500252a.xml\n",
      "results and discussion\n",
      "structure of synthetic adhesins and targeted adhesion of e. coli to antigenic surfaces\n",
      "constitutive, stable, and nontoxic expression of sas from the chromosome of e. coli.\n",
      "adhesion of engineered e. coli to antigens expressed on the surface of mammalian cells\n",
      "colonization of tumors in vivo by engineered e. coli with synthetic adhesins\n",
      "lower retention of engineered e. coli in liver and spleen\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmids, dna constructs, and oligonucleotides\n",
      "e. coli genome modification and strain construction\n",
      "in vitro cell culture and plasmid transfection\n",
      "flow cytometry analysis\n",
      "protein extract preparation, sds-page, and western blots\n",
      "adhesion assays to antigens immobilized on a plastic surface\n",
      "adhesion assays to in vitro cultured hela cells and immunofluorescence microscopy\n",
      "infection of tumor-bearing mice and recovery of bacteria from tissues\n",
      "bioluminescence imaging\n",
      "statistics\n",
      "histology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500035y/sb500035y.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00136/sb8b00136.xml\n",
      "results and discussion\n",
      "defining a promoter and quantifying its activity\n",
      "defining the boundaries and architectures of four parental promoters\n",
      "narrow range of promoter activities between the four parental promoters\n",
      "expanding the range of promoter activities through rational engineering\n",
      "promoter nomenclature\n",
      "mutations in −35 box tuned transcriptional activity down\n",
      "a minimal p phac1 promoter with enhanced activity\n",
      "synthetic rbs and rbs repeat increased transcriptional activity\n",
      "repeat of −35 and −10 boxes increased transcriptional activity\n",
      "operator insertion reduced transcriptional activity drastically\n",
      "divergent promoters, arranged in back-to-back, increased transcriptional activity\n",
      "promoter characterization using p bad as reference scale\n",
      "summary of rational promoter engineering for c. necator h16\n",
      "the use of engineered constitutive promoters in c. necator h16\n",
      "conclusion\n",
      "materials and methods\n",
      "materials\n",
      "strains\n",
      "promoter engineering and sequences\n",
      "bacterial cultivation and transformation\n",
      "promoter activity quantification using fluorescence assay\n",
      "fold change and relative promoter activity change\n",
      "effects of engineered constitutive promoters on bacterial growth and protein excretion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00202/sb8b00202.xml\n",
      "results\n",
      "sensor design and construction\n",
      "the genetic sensor specifically recognizes wtp53 in p53 knockout cells\n",
      "the genetic circuit functions in p53 -knockout cells\n",
      "the genetic sensor and genetic circuit detects endogenous p53 in cancer cell lines\n",
      "discussion\n",
      "methods\n",
      "cell lines and cell culture\n",
      "plasmid construction\n",
      "cell transfection\n",
      "real-time quantitative pcr assay\n",
      "establishment of the p53 -knockout cell line\n",
      "caspase-3 elisa assay\n",
      "flow cytometry assay\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00003/sb7b00003.xml\n",
      "results\n",
      "cellengine\n",
      "cellpro\n",
      "prospec\n",
      "other libraries: cellnutrient, cellsignals and bacterial conjugation\n",
      "cellnutrient\n",
      "cellsignals\n",
      "bacterial conjugation\n",
      "systems biology studies and synthetic biology examples\n",
      "discussion\n",
      "future work\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00060/sb8b00060.xml\n",
      "results\n",
      "materials and methods\n",
      "3g assembly\n",
      "functional validation of 3g constructs\n",
      "cell-lysis circuit characterization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00262/sb6b00262.xml\n",
      "methods\n",
      "culture media\n",
      "plasmid construction\n",
      "yeast strain and transformants\n",
      "culture conditions\n",
      "gfp measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00001/sb5b00001.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00482/sb8b00482.xml\n",
      "results and discussion\n",
      "overview\n",
      "analysis of genes from purified clones\n",
      "analysis of genes from cell lysates\n",
      "comparison with available assemblers\n",
      "indel detection\n",
      "methods\n",
      "tagmentation-based library construction\n",
      "analysis of mixed samples and pcr amplicons\n",
      "comparison with available assemblers\n",
      "indel detection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00063/sb9b00063.xml\n",
      "results and discussion\n",
      "asymmetric pcr for the generation of or-type libraries\n",
      "or-type libraries for large-scale mutagenesis of monoamine oxidase-n\n",
      "mao-n improved variants to non-native amine substrates\n",
      "mao-n activity to native primary amine substrates\n",
      "mao-n activity to novel substrates\n",
      "active site mutagenesis using combinatorial or-type mutations\n",
      "conclusions\n",
      "materials and methods\n",
      "design of oligonucleotide primers for or-type libraries\n",
      "synthesis of or-type libraries\n",
      "screening for mao-n activity\n",
      "expression and purification of mao-n\n",
      "liquid phase kinetic assay\n",
      "data availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00174/sb8b00174.xml\n",
      "methods\n",
      "cell culture\n",
      "mitochondrial generation of h 2 o 2 using d -amino acid oxidase\n",
      "redox western blotting\n",
      "protein s -glutathionylation immunofluorescence\n",
      "cell viability after exposure to h 2 o 2 produced by daao-mito\n",
      "mitochondrial depolarization assay\n",
      "fluorescence microscopy\n",
      "quantification and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00212/sb7b00212.xml\n",
      "results and discussion\n",
      "identifying the need for a new lims\n",
      "understanding how leaf lims operates\n",
      "managing projects and their outputs\n",
      "managing laboratory processes through workflows\n",
      "sample and consumable tracking\n",
      "controlling access\n",
      "using equipment to automate processes\n",
      "expanding the scope beyond synthetic biology\n",
      "future direction\n",
      "conclusion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00070/sb9b00070.xml\n",
      "results\n",
      "symbolic heterologous reaction repositories\n",
      "target molecules synthesized\n",
      "astaxanthin biosynthesis\n",
      "1-octanol biosynthesis\n",
      "apigenin biosynthesis\n",
      "cytisine biosynthesis\n",
      "identification of precursor molecules and universal reactions\n",
      "discussion\n",
      "methods\n",
      "candidate hosts from the bigg database\n",
      "construction of symbolic heterologous reaction repositories\n",
      "pathway-searching algorithm\n",
      "identification of heterologous pathway\n",
      "construction of host-based metabolic network\n",
      "thermodynamic feasibility and web server\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00096/sb8b00096.xml\n",
      "results and discussion\n",
      "decoding a uaga codon in mammalian cells\n",
      "construction of a genomically recoded hiv-1 mutant\n",
      "controlling hiv-1 replication through quadruplet codon decoding\n",
      "methods\n",
      "materials and general methods\n",
      "cell transfection, fluorescence imaging and flow cytometry assay\n",
      "reconstitution and infectivity assay of hiv-1 viruses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00105/sb5b00105.xml\n",
      "ryan e. cobb\n",
      "tom de greef\n",
      "yi-ling du\n",
      "rachit jain\n",
      "ami kabadi\n",
      "andicus lai\n",
      "andreja majerle\n",
      "lenny meijer\n",
      "ernst oberortner\n",
      "eun joong oh\n",
      "sergio peisajovich\n",
      "yannick rondelez\n",
      "paloma m. sato\n",
      "hendrik w. h. van roekel\n",
      "xiaoli xue\n",
      "li-bang zhou\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500036q/sb500036q.xml\n",
      "methods and results\n",
      "design of conjugative crispri system\n",
      "assay for conjugative transfer of crispri system\n",
      "conjugated crispri system can specifically repress the target mrfp gene\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00294/sb5b00294.xml\n",
      "results and discussion\n",
      "using score variance to search for enzymes catalyzing reactions\n",
      "pathway ranking\n",
      "predicting k m\n",
      "data set 1\n",
      "data set 2\n",
      "data set 3\n",
      "conclusion\n",
      "methods\n",
      "data representation\n",
      "training data\n",
      "semisupervised learning\n",
      "null-category noise model gaussian process\n",
      "clustering data\n",
      "feature selection\n",
      "covariance function\n",
      "regression\n",
      "support vector machines\n",
      "construction of expression plasmids and e. coli culturing\n",
      "extraction of metabolites\n",
      "mass spectrometry analysis\n",
      "reagents\n",
      "liquid chromatography\n",
      "mass spectrometry\n",
      "quantification by the standards addition method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00137/sb7b00137.xml\n",
      "methods\n",
      "cloning\n",
      "cell culture and metal exposure\n",
      "quantitation of protein expression\n",
      "transmission electron microscopy\n",
      "scanning electron microscopy\n",
      "quantitation of metal binding\n",
      "cell flocculation studies\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4002085/sb4002085.xml\n",
      "product sensing to remove flux control\n",
      "enhancing bacterial tolerance to short-chain alcohols\n",
      "new tool for metabolic engineering of whole-cell biocatalysis\n",
      "soxr as a single-cell biosensor\n",
      "redirecting photosynthesis toward the synthesis of alternative products\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200008j/sb200008j.xml\n",
      "design of a photoactivated caspase-7\n",
      "caspase-7 activity is suppressed when fused with lov2\n",
      "l57v mediates blue light-induced cell death with spatiotemporal precision\n",
      "l57v-mediated cell death is dependent on caspase-7 and lov2 domains\n",
      "l57v-mediated cell death is apoptosis\n",
      "methods\n",
      "plasmids\n",
      "cell culture and transfection\n",
      "illumination and imaging\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00303/sb7b00303.xml\n",
      "results\n",
      "setting up the in vivo model\n",
      "delivery platform calibration\n",
      "identification of selective mirna\n",
      "the construction and testing of aav bidirectional mirna activity reporters\n",
      "bidirectional reporters in vivo\n",
      "evaluation of the cytotoxic output hsv-tk/gcv\n",
      "discussion\n",
      "methods\n",
      "plasmids construction\n",
      "cell culture and transfection\n",
      "construction of luciferase and mcitrine-expressing cell line\n",
      "production, purification and quantification of single-stranded aav vectors\n",
      "in vitro infection\n",
      "cell preparation for in vivo inoculation\n",
      "animal surgeries for cell inoculation\n",
      "in vivo aav fluorescent reporter delivery regimen\n",
      "in vivo aav cytotoxic agent delivery regimen\n",
      "tissue processing and analysis\n",
      "liver section and histology\n",
      "fluorescent microscopy\n",
      "flow cytometry\n",
      "data analysis\n",
      "small rna sequencing with illumina\n",
      "small rna extraction\n",
      "library preparation and deep sequencing of mirna\n",
      "deep sequencing analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00060/sb9b00060.xml\n",
      "results and discussion\n",
      "design and combinatorial assembly of a 15-gene nitrogenase biosynthetic pathway\n",
      "the recombinant s. cerevisiae strains\n",
      "nif protein expression in the recombinant s. cerevisiae nif-c\n",
      "yeast nifh exhibiting active fe protein\n",
      "nifdk complex formation\n",
      "methods\n",
      "strains, plasmids, and methods\n",
      "construction of assembled dna fragments for the nitrogenase biosynthetic pathway and generation of s. cerevisiae strains\n",
      "pcr and rt-pcr analysis\n",
      "immunoblot analysis of nif protein expression\n",
      "growth of yeast strains and purification of yeast nifh and nifdk proteins under anaerobic conditions\n",
      "determination of fe protein or mofe protein activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00176/sb9b00176.xml\n",
      "results and discussion\n",
      "v. natriegens growth characteristics\n",
      "plasmid origins, maintenance, and copy number\n",
      "inducible promoter function\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "function of constitutive promoters, ribosomal binding sites, and riboj insulation\n",
      "v. natriegens native p1 promoter\n",
      "uber system\n",
      "reporter proteins\n",
      "terminators and degradation tags\n",
      "and gate\n",
      "conclusions\n",
      "methods\n",
      "media components\n",
      "molecular biology reagents\n",
      "bacterial strains and growth\n",
      "electrocompetent cell preparation and electroporation\n",
      "chemically competent cell preparation and chemical transformation\n",
      "dna transformation through conjugation\n",
      "generation time\n",
      "plasmid stability testing\n",
      "plasmid copy number determination\n",
      "dna sequences of parts, primers, and plasmidsconstruction and cloning\n",
      "flow cytometry\n",
      "plate reader\n",
      "light plate apparatus (lpa)\n",
      "parts testing\n",
      "inducible promoter testing\n",
      "and gate testing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500134w/sb500134w.xml\n",
      "infobiotics workbench\n",
      "related work\n",
      "model checking\n",
      "qualitative model checking\n",
      "quantitative model checking\n",
      "biological models\n",
      "quorum sensing\n",
      "stochastic model\n",
      "nondeterministic model\n",
      "pulse-generator model\n",
      "stochastic model\n",
      "nondeterministic model\n",
      "experiments\n",
      "quorum sensing model\n",
      "qualitative model checking\n",
      "quantitative model checking\n",
      "prism results\n",
      "mc2 results\n",
      "pulse-generator model\n",
      "qualitative model checking\n",
      "quantitative model checking\n",
      "conclusion and future work\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00349/sb7b00349.xml\n",
      "methods\n",
      "strains, plasmids, primers, enzymes, chemicals and general methods\n",
      "genome sequencing of jv178\n",
      "streptomycete conjugations\n",
      "cfta markerless gene disruption\n",
      "cfta heterologous expression\n",
      "ptm detection by hplc–ms/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00058/sb7b00058.xml\n",
      "results\n",
      "killing and lysis protein characterization\n",
      "characterization of the system integrated with the sense, lyse, and killing devices\n",
      "in silico mechanistic approach to study the positive feedback mechanism\n",
      "our system delivers a killing payload upon detection of v. cholerae\n",
      "discussion\n",
      "materials and methods\n",
      "strains, dna sequences, and plasmids\n",
      "consumables and services\n",
      "microplate readings\n",
      "lysis-and-killing protein study\n",
      "lysis circuit study\n",
      "killing study\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00299/sb5b00299.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid dna construction\n",
      "cell culturing and transfections\n",
      "luciferase assay\n",
      "surveyor assay\n",
      "statistical analyses\n",
      "mutant clone library\n",
      "cross-linked chromatin immunoprecipitation (chip)\n",
      "real-time quantitative pcr of chip-enriched dna\n",
      "statistical analyses\n",
      "sirna transfections\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00226/sb5b00226.xml\n",
      "background\n",
      "site-specific recombinases\n",
      "invertases\n",
      "prior work\n",
      "results and discussion\n",
      "permuton architectures\n",
      "pancake permuton\n",
      "linksort permuton\n",
      "key length and complexity\n",
      "feasible designs\n",
      "summary and future work\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000965/sb4000965.xml\n",
      "results\n",
      "and-gate promoter design and characterization\n",
      "quorum sensing-dependent gene expression of the esa -based and gates\n",
      "and gate-based cell–cell communication\n",
      "methods\n",
      "plasmid construction\n",
      "quantitative characterization\n",
      "shake flask assays\n",
      "agar plate luminescence assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00230/sb8b00230.xml\n",
      "bottom up or de novo construction of human artificial chromosomes\n",
      "construction of human artificial chromosomes from natural alphoid dna\n",
      "construction of human artificial chromosomes from alphoid synthetic repeats\n",
      "construction of synthetic human artificial chromosome with a conditional centromere\n",
      "human artificial chromosome with regulated centromere for gene functional analyses\n",
      "conversion of alphoid teto -hac into a gene delivery vector\n",
      "re-engineering the alphoid teto -hac vector to allow a unique control for gene phenotypes by a simple doxycycline manipulation\n",
      "pericentromeric gamma-satellite dna and tdna prevent heterochromatin spreading and protect gene expression from alphoid teto -hac\n",
      "an improved microcell-mediated chromosome transfer (mmct) technique for hac transfer to recipient cells\n",
      "expression of a genomic copy of human genes from alphoid teto -hac\n",
      "construction of alphoid teto -hac with multi-integration site to assemble large entire genomic loci and engineer synthetic chromosomes with a predetermined set of genes\n",
      "use of the alphoid teto -hac for epigenetic engineering of chromatin within a single functional centromere\n",
      "human centromeres: what we know and what we do not\n",
      "centromere chromatin state and kinetochore function are interconnected\n",
      "human kinetochores have a remarkable plasticity that tolerates profound changes to their chromatin environment but critically sensitive to the level of centromeric transcription\n",
      "histone h3k9 acetyl/methyl balances regulate cenp-a assembly\n",
      "centromeric transcription and h3k9 acetylation keep heterochromatin from invading centrochromatin\n",
      "cenp-c and cenp-i are key connecting factors for kinetochore and cenp-a assembly\n",
      "future direction: next-generation human artificial chromosomes containing alphoid arrays with different targeted sequences\n",
      "human artificial chromosomes for measuring chromosome instability (cin) in cancer cells\n",
      "chromosome instability (cin) as a driver of tumorigenesis\n",
      "hac-based “loss of signal” assay for measuring chromosome instability and identification of drugs that elevate cin in cancer cells\n",
      "hac-based “gain of signal” high-throughput screening assay for analysis of chemical libraries\n",
      "human artificial chromosome to identify conserved dosage chromosome instability genes in human cancer\n",
      "conclusions and prospects\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00284/sb5b00284.xml\n",
      "results and discussion\n",
      "flowcal description\n",
      "downloading and installing flowcal\n",
      "using flowcal\n",
      "validating flowcal by calibrating gene expression data collected at different gain settings\n",
      "using flowcal calibration to reduce day-to-day variability in gene expression data\n",
      "using flowcal to calibrate s. cerevisiae gene expression data\n",
      "cross-instrument calibration of mef units\n",
      "outlook\n",
      "methods\n",
      "e. coli p tac -gfp experimental protocol for figure a and b\n",
      "e. coli ccas/ccar v2.0 experimental protocol\n",
      "s. cerevisiae phyb/pif3/venus experimental protocol\n",
      "e. coli p tac -gfp experimental protocol for figure f\n",
      "flow cytometry data acquisition and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00398/sb6b00398.xml\n",
      "meirong gao\n",
      "anthony goering\n",
      "masahiro kanno\n",
      "xiaoping olson\n",
      "sayed ahmad salehi\n",
      "zengyi shao\n",
      "yunan zheng\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00094/sb5b00094.xml\n",
      "materials and methods\n",
      "design and fabrication\n",
      "cell culture and device loading\n",
      "image capture\n",
      "single cell tracking and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00271/sb7b00271.xml\n",
      "methods\n",
      "media, strains, and plasmids\n",
      "dna manipulations\n",
      "de novo biosynthesis of glutaric acid\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00106/sb9b00106.xml\n",
      "results and discussion\n",
      "modeling of conserved residues for interaction of bacterial mlps and fungal nrps\n",
      "copurification of bacterial mlps with nrps enzymes in vitro\n",
      "bacterial mlps increase titers of fungal nrps-derived products in p. chrysogenum\n",
      "comb mlp performance for production of lld-acv and htd in p. chrysogenum\n",
      "strain lineage and secondary metabolite fingerprint impact effect of comb mlp\n",
      "conclusions\n",
      "methods\n",
      "modeling and molecular docking\n",
      "protein expression analysis\n",
      "strains\n",
      "vector construction\n",
      "transformation of mlp into p. chrysogenum\n",
      "cultivation of p. chrysogenum\n",
      "qpcr analysis of gene copy number and nrps expression in mlp strains\n",
      "lc–ms analysis and evaluation of metabolite profiles\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00074/sb6b00074.xml\n",
      "challenges of synthetic biology in mammalian systems\n",
      "scalability\n",
      "orthogonality, characterization of genetic parts and predictability\n",
      "episomes and mammalian artificial chromosome (mac) as orthogonal vectors\n",
      "generation of macs\n",
      "top-down approach\n",
      "bottom-up approach\n",
      "loading of genes into hacs\n",
      "rapid and efficient prototyping\n",
      "design and implementation of new regulatory layers for gene expression\n",
      "limitations of macs\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00297/sb7b00297.xml\n",
      "results and discussion\n",
      "characterization of a promoter library\n",
      "design and characterization of an rbs library\n",
      "translation efficiencies predicted by software vs experimental measurements\n",
      "quantification of efe in total soluble proteins in synechocystis\n",
      "efe expression level no longer limits ethylene productivity in synechocystis\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "construction of recombinant plasmids\n",
      "construction of the promoter library\n",
      "construction of the rbs library\n",
      "modification of the synechocystis genome\n",
      "sds-page and western blotting\n",
      "quantification of efe in the total soluble protein of synechocystis\n",
      "in vitro efe activity assay\n",
      "ethylene production from synechocystis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00163/sb9b00163.xml\n",
      "results\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "creating b. subtilis strains with riboswitch regulation of protein expression\n",
      "establishing stability of theta-replicating, pht, plasmids during vegetative growth, and sporulation in b. subtilis\n",
      "quantifying heterogeneity of protein expression in b. subtilis mutants\n",
      "characterizing expression heterogeneity for different plasmid and cell types\n",
      "evaluating copy number and polymerase accessibility in plasmids and chromosomes\n",
      "characterizing expression heterogeneity after sporulation using pht plasmid\n",
      "comparing postsporulation behavior of b. subtilis cells to uv damage behavior\n",
      "discussion\n",
      "methods\n",
      "lkp kinetic model of protein expression\n",
      "dna sequences and mutants\n",
      "cell growth and protein expression\n",
      "rfp standard curve\n",
      "stability of theta-replicating plasmids\n",
      "plasmid retention\n",
      "qpcr and rt-qpcr\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4002079/sb4002079.xml\n",
      "sergey boyarskiy\n",
      "michael fisher\n",
      "lærke münter lassen\n",
      "solvej siedler\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00345/sb8b00345.xml\n",
      "results and discussion\n",
      "rationale of fenix, a synthetic post-translational control system for pathway engineering\n",
      "the fenix system enables precise control of protein accumulation in recombinant e. coli strains\n",
      "the fenix system enables a precise and concerted temporal switch of protein accumulation\n",
      "establishing a fenix-based metabolic switch for biopolymer accumulation in recombinant e. coli strains\n",
      "the phaa activity can be tightly regulated by means of the fenix system\n",
      "the fenix system enables efficient decoupling of phb biosynthesis and bacterial growth and leads to high rates of biopolymer accumulation\n",
      "enhanced phb accumulation mediated by phaa* stems from flux rewiring around the acetyl-coa node\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and cultivation conditions\n",
      "general molecular biology techniques\n",
      "design and construction of pfenix plasmids carrying proteolizable versions of gfp and mcherry\n",
      "construction of plasmid pfenix·pha* for post-translational control of phb accumulation in recombinant e. coli strains\n",
      "flow cytometry evaluation of the fenix system\n",
      "in vitro quantification of the phaa activity\n",
      "phb quantification\n",
      "other analytical techniques\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00114/sb6b00114.xml\n",
      "methods\n",
      "synthesis of crrnas\n",
      "purification of cpf1 proteins\n",
      "cpf1 in vitro cleavage assay\n",
      "construction and assembly of c-brick parts\n",
      "comparison of the c-brick assembly efficiency with the type ii re-based method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00188/sb9b00188.xml\n",
      "results and discussion\n",
      "lethality of dsb induced by active cas9 in shewanella\n",
      "recombinase-independent ssdna genome-editing coupled to crispr/cas9 counter-selection\n",
      "ssdna recombineering coupled to crispr/cas9 counter-selection gives high efficiency genome-editing\n",
      "system optimization\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, oligonucleotides, and growth conditions\n",
      "plasmid construction\n",
      "electroporation\n",
      "crspr/cas9 activity assessment\n",
      "recombineering in s. amazonensis\n",
      "ssdna genome-editing combined with crispr/cas9 counter-selection\n",
      "screening for edited cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00113/sb8b00113.xml\n",
      "results\n",
      "analysis and optimizations of sensor-effector cells\n",
      "validation of the sensor\n",
      "therapeutic in vitro validation of the cytokine neutralizer cells\n",
      "controllability of the cytokine neutralizer cells\n",
      "discussion\n",
      "materials and methods\n",
      "dna synthesis and vector design\n",
      "cultivation and general handling of different cell lines\n",
      "transfection\n",
      "quantification of target gene expression\n",
      "detection of fluorescent proteins\n",
      "nf-κb profiling\n",
      "alginate encapsulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300102k/sb300102k.xml\n",
      "results\n",
      "photosynthetic carbon metabolism\n",
      "identifiability analysis for carbon metabolism model\n",
      "chlamydomonas reinhardtii\n",
      "rhodobacter spheroides\n",
      "discussion\n",
      "role of organelles in the photosynthesis\n",
      "methods\n",
      "in silico design of metabolic pathways\n",
      "sensitivity analysis\n",
      "identifiability analysis\n",
      "robustness analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00092/sb6b00092.xml\n",
      "results and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500058e/sb500058e.xml\n",
      "background\n",
      "2013 findings\n",
      "chassis\n",
      "parts\n",
      "future considerations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00066/sb7b00066.xml\n",
      "metabolic engineering of a complex trait\n",
      "a novel approach to synthesize polyphenols\n",
      "synthetic core promoters for fine-tuning expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00195/sb8b00195.xml\n",
      "results and discussion\n",
      "strategy for characterizing orthogonality\n",
      "integration of heterologous genes onto p2 using crispr/cas9\n",
      "replication of p2 by tp-dnap2 in trans\n",
      "discovery of error-prone tp-dnap2 variants through library screening\n",
      "orthogonality between p2 replication and genomic replication\n",
      "mutual orthogonality between p1 and p2 replication\n",
      "conclusion\n",
      "methods\n",
      "dna cloning\n",
      "yeast strains\n",
      "yeast transformation\n",
      "linear plasmid dna extraction\n",
      "curing of cytoplasmic plasmids using cas9\n",
      "tp-dnap2 library generation\n",
      "p2 fluctuation tests using leu2*\n",
      "genomic fluctuation tests using ura3\n",
      "calibration curve relating p2 copy number to mkate2 fluorescence\n",
      "qpcr\n",
      "primer melting curve\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400021j/sb400021j.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00060/sb5b00060.xml\n",
      "results and discussion\n",
      "construction of micodes\n",
      "development of a two-hybrid fluorescence localization assay for measuring protein–protein interactions in the yeast cytosol\n",
      "synzips and design of new specific coiled-coils\n",
      "application of micoding toward the screening of a library of coiled-coil zippers\n",
      "scaling the size of a micoded library\n",
      "summary\n",
      "methods\n",
      "micode design and construction\n",
      "synzip design\n",
      "strains and media\n",
      "i-scei integration\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00167/sb8b00167.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "chemicals, media and cultivation conditions\n",
      "strain construction\n",
      "continuous cultures\n",
      "turbidostat regime\n",
      "medium swap regime\n",
      "carbon labeling analysis\n",
      "lc–ms analysis\n",
      "whole genome sequencing and mutation analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00522/sb8b00522.xml\n",
      "results and discussion\n",
      "models\n",
      "well-mixed compartments\n",
      "basic effects of compartmental organization are revealed through the study of simple motifs\n",
      "the presence of a trade-off associated with changing compartment spacing\n",
      "consequences for feedback\n",
      "adaptation to spatial separation\n",
      "compartmental organization of bistable motifs\n",
      "the consequence of distributing a bistable motif and the effect of spatial separation of compartments\n",
      "the qualitative effect of spatial separation depends on the distribution of circuit nodes across compartments\n",
      "modular augmentation of a bistable circuit\n",
      "compartmental organization of oscillators\n",
      "different ways of distributing an oscillatory circuit has qualitatively different implications\n",
      "the role of topology of spatial organization\n",
      "the roles of leak and separation in determining oscillatory behavior\n",
      "spatial organization and competition for resources\n",
      "uniform channel\n",
      "basic effects of compartmental organization\n",
      "multiple spatial configurations that preserve interaction strengths\n",
      "effects of diffusivity of species\n",
      "spatial organization of degradation\n",
      "localized degrading enzyme\n",
      "degrading enzyme distributed between sender and receiver\n",
      "uniformly distributed degrading enzyme\n",
      "effect of localization/nonlocalization of degrading enzyme on global features of output\n",
      "adaptation to spatial separation\n",
      "localized bistable and oscillatory circuits in a uniform channel\n",
      "bistable behavior can depend on location relative to boundaries\n",
      "the effect of diffusivity on circuits\n",
      "applications\n",
      "adaptation to template concentration\n",
      "communication with a target\n",
      "single node circuit\n",
      "two node circuit\n",
      "discussion\n",
      "basic effects of distributing circuits\n",
      "well-mixed compartments vs uniform channel\n",
      "adaptation to spatial separation\n",
      "spatial organization of degradation\n",
      "systems insights applied to an exemplar circuit\n",
      "bistable circuit\n",
      "methods\n",
      "models\n",
      "spatial organization\n",
      "well-mixed configuration\n",
      "uniform 1-d channel\n",
      "boundary conditions and spatial domain\n",
      "parameters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00210/sb5b00210.xml\n",
      "results\n",
      "sbol stack design, architecture and implementation\n",
      "sbol stack api and client library\n",
      "sbol stack data integration, semantics, and the example data set\n",
      "sbol stack data federation\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "synbiohub functionality and use cases\n",
      "uploading and sharing synthetic biology designs\n",
      "browsing data from the sbol stack\n",
      "hosting a synbiohub instance\n",
      "discussion\n",
      "methods\n",
      "software availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00391/sb6b00391.xml\n",
      "methods\n",
      "construction of plasmids\n",
      "cell-free protein synthesis experiments and gfp assays\n",
      "michaelis–menten plots\n",
      "mrna analysis and enzyme assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00399/sb8b00399.xml\n",
      "results and discussion\n",
      "construction of constitutive expression bglbrick plasmids\n",
      "benchmarking of a compatible set of constitutive bglbrick expression plasmids\n",
      "development of selprom: a predictive selection tool providing a database of inducible and constitutive expression plasmids\n",
      "validation of the selprom selection tool\n",
      "methods\n",
      "bacterial strains and media\n",
      "construction of constitutive bglbrick expression vectors\n",
      "benchmarking of constitutive plasmids\n",
      "database implementation\n",
      "response characterization\n",
      "predictive models of relative strength for inducible and constitutive promoters\n",
      "selprom: an online tool for promoter selection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00346/sb6b00346.xml\n",
      "results\n",
      "effect of transcriptional activators ino2p or ino4p deletion and repressor opi1p overexpression on 3-hp production\n",
      "effect of inositol or choline concentration on cell growth and 3-hp production in δino2 strain\n",
      "ino2p variants partly recovered cell growth deficiency of δino2 strain and increased 3-hp production\n",
      "discussion\n",
      "materials and methods\n",
      "medium and growth conditions\n",
      "plasmid construction\n",
      "strain construction\n",
      "real-time qpcr analysis\n",
      "fermentation and metabolite analysis\n",
      "biomass determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00013/sb5b00013.xml\n",
      "noise propagation in synthetic gene circuits for metabolic control\n",
      "negative feedback regulation of fatty acid production\n",
      "trade-offs in engineering sugar utilization pathways\n",
      "designing rna-based genetic control systems\n",
      "rational design of allosteric regulation of homoserine dehydrogenase\n",
      "design and implementation of a biomolecular concentration tracker\n",
      "synthetic anhydrotetracycline-controllable gene expression system\n",
      "recognition sequences and substrate evolution in cyanobactin biosynthesis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000228/sb4000228.xml\n",
      "computational models for synthetic biology\n",
      "dynamic modeling under uncertainty\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00058/sb5b00058.xml\n",
      "results and discussion\n",
      "development of a novel promoter probe vector with minimal transcriptional noise\n",
      "genome integrated controls exhibit increased expression with reduced transcription variability\n",
      "plasmid-based screening of a synthetic promoter library in e. coli\n",
      "assessment of a genome integrated and calibrated synthetic promoter library in p. putida kt2440\n",
      "sequence analysis of synthetic promoters\n",
      "conclusions\n",
      "methods\n",
      "dna techniques\n",
      "genome integration\n",
      "bacterial strains, plasmids, and cultivation conditions\n",
      "fluorescence measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00159/sb6b00159.xml\n",
      "results and discussion\n",
      "stabilization of dehydroshikimate dehydratase through rational design\n",
      "dehydroshikimate dehydratase library design\n",
      "enzyme thermostability, heat challenge and high-throughput screen\n",
      "steady-state kinetics and half-life determination of wtasbf and mut1\n",
      "in vivo dhsase overexpression analysis: wtasbf versus mut1\n",
      "commodity chemical production\n",
      "methods\n",
      "structure based protein design for stability\n",
      "library construction and screening\n",
      "expression and purification\n",
      "high-throughput screening (hts)\n",
      "medium-throughput screening (mts)\n",
      "steady-state measurements\n",
      "functional protein in vivo\n",
      "long-term stability at 37 °c\n",
      "heterologous production of dhb and β-carboxy- cis , cis -muconic acid\n",
      "circular dichroism\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00327/sb7b00327.xml\n",
      "a shortage of biological parts for the solventogenic clostridia\n",
      "reporters of gene expression\n",
      "fluorescent reporter proteins\n",
      "gfp-family fluorescent proteins\n",
      "flavin-binding fluorescent proteins\n",
      "enzymatic reporter proteins\n",
      "chloramphenicol acetyltransferase\n",
      "carbohydrate hydrolases: β-glucuronidase, β-galactosidase, amylase, endoglucanase\n",
      "lipase and alkaline phosphatase\n",
      "luciferase\n",
      "transcription\n",
      "transcription initiation\n",
      "transcription termination\n",
      "translation\n",
      "translation initiation\n",
      "codon usage during translation\n",
      "post-transcriptional control of gene expression\n",
      "controlling mrna stability\n",
      "antisense rnas and riboswitches\n",
      "replication\n",
      "plasmid origins of replication\n",
      "chromosomal integration\n",
      "insulators\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500257f/sb500257f.xml\n",
      "thrombin-mediated transcriptional regulation\n",
      "an optimum compartment volume for high-order reactions\n",
      "control of cell-free operon expression\n",
      "dependence of membrane protein integration on vesicle volume\n",
      "annexation of a high-activity enzyme in a synthetic three-enzyme complex\n",
      "direct polymerization of proteins\n",
      "cell-free protein synthesis from a release factor 1 deficient e. coli\n",
      "linear dna for rapid prototyping of synthetic biological circuits\n",
      "design and selection of a synthetic operon\n",
      "gene circuit performance characterization and resource usage\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500362n/sb500362n.xml\n",
      "results and discussion\n",
      "reducing tagmentation reaction volume\n",
      "barcoding pcr\n",
      "source of dna for nextera library preparation\n",
      "increasing the number of samples receiving sufficient sequence data\n",
      "analyzing the ngs qc data\n",
      "applications of this process\n",
      "methods\n",
      "instrumentation\n",
      "dna assembly and quantitation\n",
      "preparing nextera libraries for sequencing\n",
      "sequence data processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200003v/sb200003v.xml\n",
      "cell-free gene expression and polymerization of yfp-mreb in synthetic liposomes\n",
      "design and polymerization of mreb in artificial vesicles\n",
      "methods\n",
      "cell-free gene expression\n",
      "plasmids construction and protein purification\n",
      "liposome preparation and mreb polymerization inside liposome\n",
      "data acquisition and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00145/sb6b00145.xml\n",
      "results and discussion\n",
      "selection strategy\n",
      "development of a negative selection method\n",
      "selection for improved tyrosine suppressor trnas\n",
      "second-generation tyrosine suppressor trna selection\n",
      "selecting for increased 3-halo-tyrosine incorporation activity\n",
      "selected suppressor trnas improve organismal fitness\n",
      "conclusion\n",
      "materials and methods\n",
      "trna and trna-synthetase expression plasmids\n",
      "assembly of trna libraries and variants\n",
      "reporter and positive selection plasmid construction\n",
      "construction of pselt vector\n",
      "tyrosine suppressor trna selection by compartmentalized partnered replication (cpr)\n",
      "adapted phes negative selection\n",
      "design and selection of first-generation trna libraries: ac-lib and t-lib\n",
      "selection of second-generation trna library: act5s -lib\n",
      "design and selection of third-generation trna libraries: s7-ta\n",
      "orthogonality assay protocol\n",
      "tyrosine suppression efficiency assay protocol\n",
      "noncanonical incorporating suppression efficiency assay protocol\n",
      "fitness/growth assay protocol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00179/sb5b00179.xml\n",
      "results and discussion\n",
      "the most robust two-node oscillators combine positive and negative autoregulation\n",
      "different activation reactions increase the robustness of the ring oscillator\n",
      "robustness of three-node oscillators is achieved through combinations of oscillating motifs\n",
      "both the robustness and the phenotype of the oscillators depend on degradation rates\n",
      "increasing the number of regulatory interactions increases robustness\n",
      "conclusion\n",
      "methods\n",
      "defining and calculating robustness\n",
      "representation of network topologies\n",
      "defining the objective for stochastic oscillations\n",
      "maximizing robustness simultaneously in model-parameter space\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00041/sb9b00041.xml\n",
      "results\n",
      "materials and methods\n",
      "assembly and validation of grna arrays\n",
      "construction of single grna vectors for quantification of repression\n",
      "strains and cell cultures\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00124/sb7b00124.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids, primers, and enzymes for preparing dna templates\n",
      "preparation of dna templates\n",
      "preparation of mrna templates\n",
      "cell-free translation of mrna templates\n",
      "evaluation of translation efficiency\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00111/sb5b00111.xml\n",
      "results and discussion\n",
      "a secondary downstream arsr binding site reduces background expression but retains induction from p ars\n",
      "decreased background expression from upstream-placed arsr binding sites\n",
      "heterologous application of the arsr operator as a transcriptional roadblock\n",
      "background reduction is dependent on the sequence and orientation of the arsr operator\n",
      "secondary operators downstream reduce background expression by acting as a transcription roadblock\n",
      "increased local repressor concentration at dual upstream operators\n",
      "conclusions\n",
      "methods\n",
      "strains and culture conditions\n",
      "arsr operator designs in homologous ars setting\n",
      "design of reporter circuits with the arsr operator in heterologous environment\n",
      "bioreporter assays\n",
      "culture preparation\n",
      "bioreporter assay preparation and readout\n",
      "northern blotting\n",
      "chemicals\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300098w/sb300098w.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "gene circuit to study retroactivity\n",
      "effects of retroactivity on the response of the gene circuit\n",
      "dynamic effects of retroactivity: induction\n",
      "dynamic effects of retroactivity: de-induction\n",
      "retroactivity impact at the single cell level\n",
      "effects of retroactivity on the steady-state transfer curve\n",
      "parametric study of retroactivity effects on a simple model\n",
      "steady-state effects of retroactivity\n",
      "dynamic effects of retroactivity\n",
      "conclusions and discussion\n",
      "methods\n",
      "plasmid construction\n",
      "strain and growth conditions\n",
      "steady state and dynamic experiments in plate reader\n",
      "dynamic experiments in microscope\n",
      "quantitation of plasmid copy number\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00110/sb9b00110.xml\n",
      "results and discussion\n",
      "construction of an antibiotic-mediated predator–prey ecosystem\n",
      "validation of the synthetic consortium\n",
      "predatory dynamics in an antibiotic dose-dependent manner\n",
      "context dependence enriches community dynamics under varied initial conditions\n",
      "mathematical modeling of the synthetic consortium\n",
      "invasion test over the predatory ecosystem\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and growth conditions\n",
      "plasmid construction\n",
      "growth and relative abundance measurements for the predator–prey consortium\n",
      "determination of chloramphenicol concentration for the survival of the prey and predator strains\n",
      "antibiotic-varying experiments\n",
      "predator–prey ratio-varying experiment\n",
      "mathematical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00296/sb5b00296.xml\n",
      "results and discussion\n",
      "overview of the platform\n",
      "cell-free synthesis: strength and repertoire\n",
      "semicontinuous cell-free tx-tl\n",
      "adjustable mrna degradation\n",
      "control of protein degradation\n",
      "examples of circuits\n",
      "cell-free synthesis of phages\n",
      "minimal cell system\n",
      "conclusion\n",
      "methods\n",
      "cell-free reaction\n",
      "dna part list and plasmid preparation\n",
      "liposome preparation and observations\n",
      "measurements of batch mode cell-free tx-tl reactions\n",
      "protein purification\n",
      "bacteriophage titration and imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200002m/sb200002m.xml\n",
      "results and discussion\n",
      "construction of bak bh3 peptide libraries for affinity maturation\n",
      "affinity maturation of the bak bh3 peptide by hydrophobic face engineering\n",
      "affinity maturation of the bak bh3 peptide by helical propensity engineering\n",
      "specificity of affinity matured bak bh3 peptides\n",
      "solution binding assays\n",
      "qsar analysis of affinity matured bak bh3 peptides\n",
      "biological activity of affinity matured bak bh3 peptides\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, plasmid and library construction\n",
      "pro-survival bcl-2 protein expression and purification\n",
      "binding assays, library screening, and determination of apparent k d values\n",
      "hela culture and killing assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500296p/sb500296p.xml\n",
      "results and discussion\n",
      "design of expression strategy\n",
      "knockout of the eqx locus\n",
      "eqxs promoter drives expression of sgfp in recombinant strain\n",
      "expression of the cpas gene from aspergillus flavus produces expected metabolite\n",
      "use of the divergent promoter for simultaneous dual gene introduction\n",
      "dieckmann cyclase depends upon unmodified c-terminal r domain\n",
      "production of multiple gene products leads to synthesis of lovastatin precursors in f. heterosporum\n",
      "eqxs requires cognate trans -er\n",
      "resurrection of antituberculosis agent from a silenced biosynthetic pathway\n",
      "conclusion\n",
      "methods\n",
      "cloning of vectors and expression plasmids\n",
      "fungal mutagenesis\n",
      "generating the uracil auxotroph\n",
      "fluorescence microscopy\n",
      "genome sequencing and analysis\n",
      "chemical analysis\n",
      "purification of caatrp 1\n",
      "purification of 4\n",
      "purification of dihydromonacolin l 9\n",
      "purification of 10 – 13\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00164/sb9b00164.xml\n",
      "methods\n",
      "general concept methodology\n",
      "results\n",
      "results: encryption by the sender\n",
      "results: decryption by the receiver\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00501/sb8b00501.xml\n",
      "results and discussion\n",
      "protein colocalization in vivo by de novo designed ppis\n",
      "transcription activation in vivo by de novo designed ppis\n",
      "transcription repression in vivo by de novo designed ppis\n",
      "oligomerization of tal-based repressors by de novo designed ppis\n",
      "conclusion\n",
      "materials and methods\n",
      "plasmids\n",
      "bacterial two-hybrid assay utilizing adenylate cyclase reconstitution\n",
      "bacterial two-hybrid assay utilizing transcription activation\n",
      "gfp assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00051/sb5b00051.xml\n",
      "results and discussion\n",
      "experimental approach\n",
      "data analysis and modeling\n",
      "influence of template concentration and droplet size\n",
      "conclusion\n",
      "materials and methods\n",
      "chemicals\n",
      "droplet production\n",
      "microscopy\n",
      "spectrometry\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400142b/sb400142b.xml\n",
      "results and discussion\n",
      "flash-tetracysteine assay for extracellular protein expression\n",
      "isolation of highly fluorescent transposon mutants\n",
      "mutants have stable membranes and secrete more yebf into culture medium\n",
      "enhanced extracellular expression of yebf-cellulase fusions\n",
      "universal reporter of diverse secretory mechanisms\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "plasmid construction\n",
      "protein analysis\n",
      "protein purification\n",
      "flash labeling of extracellular proteins\n",
      "transposon insertion library generation\n",
      "screening of transposon libraries\n",
      "characterization of outer membrane permeability\n",
      "cmc assay for endoglucanase activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00074/sb9b00074.xml\n",
      "results\n",
      "design of the hys-deg circuit\n",
      "predictive modeling and design rules of the hys-deg circuit\n",
      "implementation of the hys-deg circuit as a sensor of proteasomal degradation\n",
      "modulating the hysteretic response of the hys-deg\n",
      "discussion\n",
      "materials and methods\n",
      "plasmids\n",
      "cell cultures and transfection\n",
      "lentiviral vector production and transduction\n",
      "stable cell lines\n",
      "fluorescence-activated cell sorting (facs) and flow cytometry analyses\n",
      "quantitative rt-pcr\n",
      "seap activity assay\n",
      "western blot analyses\n",
      "mathematical model\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00152/sb9b00152.xml\n",
      "introduction\n",
      "results and discussion\n",
      "biochemical characterization of aa pyks- aa cyp82m3- aa tri/ aa trii\n",
      "reconstitution of tropinone biosynthetic pathway in s. cerevisiae\n",
      "biosynthesis of tropine and pseudotropine in s. cerevisiae\n",
      "conclusions\n",
      "methods\n",
      "plasmids construction\n",
      "enzymatic assay\n",
      "pathway construction in yeast\n",
      "shake flask cultivation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00021/sb5b00021.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00240/sb6b00240.xml\n",
      "methods\n",
      "bacterial strains and plasmid construction\n",
      "library construction\n",
      "cell lysis for high throughput screening\n",
      "kiv absorbance-based kivd assay\n",
      "high throughput screening\n",
      "computational simulations\n",
      "protein purification\n",
      "heat incubation\n",
      "nadh-coupled kivd enzyme assay\n",
      "recombination\n",
      "t 50 determination\n",
      "half-life determination\n",
      "cd spectroscopy\n",
      "kinetic parameters and substrate specificity determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00262/sb9b00262.xml\n",
      "results\n",
      "engineering pancreatic β-cells with stable expression of bpac\n",
      "optogenetic control of camp and gsis in min6-bpac cells\n",
      "optical enhancement of gsis in min6-bpac pis leaves oxygen consumption rate (ocr) unchanged\n",
      "improvement of glucose tolerance after transplantation of min6-bpac cells\n",
      "assessment of function and viability of retrieved min6-bpac cells\n",
      "discussion\n",
      "materials and methods\n",
      "cell culture\n",
      "recombinant lentivirus generation and cell transduction\n",
      "genomic dna isolation and pcr\n",
      "immunocytochemistry\n",
      "western blotting\n",
      "pseudoislet gel encapsulation\n",
      "streptozotocin-induced diabetes, cell transplantation, and retrieval\n",
      "glucose tolerance test (gtt), illumination regimen, and blood sampling\n",
      "determination of [camp] i and secreted insulin\n",
      "cell apoptosis\n",
      "oxygen consumption\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00076/sb7b00076.xml\n",
      "results and discussion\n",
      "a chimeric csf1r (named csf1rchi) triggered a ca 2+ signal upon stimulation with csf1\n",
      "cells coexpressing csf1rchi and carq dynamically blebbed upon csf1 stimulus\n",
      "csf1 seeking cells autonomously migrate to csf1 source cells\n",
      "csf1 fusing cell underwent membrane fusion and death with csf1 source cells\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture, transfection and stable cell line generation\n",
      "illumination and imaging\n",
      "cell stimulation\n",
      "cell assays\n",
      "data quantification\n",
      "statistical analysis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00358/sb6b00358.xml\n",
      "results and discussion\n",
      "design of the crispr-mmej mediated chip-seq (cmchip-seq) method\n",
      "analyses for sp1, myc and tcf7l2 binding sites in hct116 cells\n",
      "chip-seq analyses of tf binding sites in pooled cell populations\n",
      "methods and materials\n",
      "construction of crisprexp plasmids\n",
      "construction of donor vectors\n",
      "cell culture and transfection\n",
      "genomic pcr and dna sequencing\n",
      "chip-seq analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500370m/sb500370m.xml\n",
      "mads t. bonde\n",
      "yoichiro ito\n",
      "ernst oberortner\n",
      "nicholas roehner\n",
      "daven sanassy\n",
      "chris takahashi\n",
      "lev tsimring\n",
      "paweł widera\n",
      "jifeng yuan\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200018e/sb200018e.xml\n",
      "results and discussion\n",
      "small-molecule microarrays enable high-throughput screening for binding to de novo proteins\n",
      "estimation of binding affinities\n",
      "binding embelin induces a conformational change in protein s824\n",
      "similar de novo proteins have different binding specificities\n",
      "implications for protein evolution in natural biology and for protein design in synthetic biology\n",
      "methods\n",
      "protein expression and purification\n",
      "smm screen\n",
      "calculation of correlation coefficients\n",
      "circular dichroism (cd) spectroscopy\n",
      "surface plasmon resonance (spr)\n",
      "heteronuclear single quantum coherence (hsqc) nmr\n",
      "pfgse nmr and estimation of k d\n",
      "mass spectrometry\n",
      "isothermal titration calorimetry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500261t/sb500261t.xml\n",
      "results and discussion\n",
      "measurement of steady-state signal output from the envz/ompr osmosensing circuit\n",
      "aromatic tuning modulates steady-state signal output from the envz/ompr circuit\n",
      "correlation between the surface of tm2 the aromatic residues reside upon and signal output\n",
      "advantages of employing aromatic tuning to modulate shk signal output\n",
      "using aromatic tuning to study receptor concentration-dependent robustness\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "analysis of steady-state signal output from osmosensing circuits\n",
      "protein quantification of envz-v5\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00254/sb7b00254.xml\n",
      "results\n",
      "design of r-m system-mediated genome editing (rmge) for bacteria with or without type iv reases\n",
      "construction of an rmge system based on the inducible mtase bcesiim (rmge- bcesiim )\n",
      "scarless single gene deletion in e. coli\n",
      "precise point mutation and gene replacement in e. coli\n",
      "construction of a rmge system based on the inducible rease mcra (rmge- mcra )\n",
      "genetic manipulation in b. subtilis using the rmge- mcra system\n",
      "genome editing in saccharomyces cerevisiae using rmge- mcra system\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and primers\n",
      "media and culture conditions\n",
      "dna manipulation\n",
      "counter-selection efficiencies\n",
      "flow cytometry analysis\n",
      "fermentation in shake flasks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00287/sb6b00287.xml\n",
      "results and discussion\n",
      "a toxin counter-selectable cassette regulated by an antitoxin switch (tccras) designed by reprogramming of the ta systems\n",
      "scheme for genome modifications using the tccras system\n",
      "scarless deletion of a gene and a large operon or chromosomal region\n",
      "sequence-specific integration of various genes\n",
      "sequence-specific gene replacement and precise point mutation\n",
      "genome editing in c. glutamicum using the tccras system\n",
      "regulating mechanism of the tccras system\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, culture conditions, and dna manipulation methods\n",
      "construction of the tccras system and recombinant plasmids\n",
      "counter-selectable and mutation efficiencies\n",
      "microscopy\n",
      "flow cytometry analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00260/sb8b00260.xml\n",
      "results and discussion\n",
      "yeast-displayed reporter of stop codon suppression\n",
      "comparison of yeast-displayed and fluorescent protein readthrough reporters\n",
      "evaluation of factors known to affect stop codon readthrough efficiency\n",
      "methods\n",
      "materials\n",
      "media preparation and yeast strain construction\n",
      "reporter plasmid construction\n",
      "suppressor plasmid construction\n",
      "cloning acfrs into “high copy” vector prs425\n",
      "preparing noncanonical amino acid liquid stocks\n",
      "yeast transformations, propagation, and induction\n",
      "flow cytometry data collection and analysis\n",
      "plate reader data collection\n",
      "calculating rre and mmf\n",
      "click chemistry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00038/sb8b00038.xml\n",
      "results and discussion\n",
      "methods\n",
      "assembly of pcap01-based plasmids\n",
      "liquid chromatography–mass spectrometry (lc–ms)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500272d/sb500272d.xml\n",
      "stefanie frank\n",
      "wentao kong\n",
      "andrew d. lawrence\n",
      "yuheng lin\n",
      "sergios a. nicolaou\n",
      "stephen sarria\n",
      "tsutomu tanaka\n",
      "yongbo yuan\n",
      "kyle zingaro\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500351f/sb500351f.xml\n",
      "methods\n",
      "strains, media, and reagents\n",
      "plasmid construction\n",
      "transformation\n",
      "screening of streptomyces strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00019/sb9b00019.xml\n",
      "results\n",
      "design of fusion proteins\n",
      "oligomer formation of fusion proteins\n",
      "structures of the tetramer of fusion protein 1 and hexamer of fusion protein 2\n",
      "conformational dynamics of the tetramer of fusion protein 1 and hexamer of fusion protein 2\n",
      "discussion\n",
      "conclusions\n",
      "methods\n",
      "construction of expression systems for fusion proteins 1 and 2\n",
      "preparation of the tetramer of fusion protein 1 and hexamer of fusion protein 2\n",
      "small angle x-ray scattering measurement\n",
      "size-exclusion chromatography-multiangle static light scattering\n",
      "high-speed atomic force microscopy observation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00249/sb5b00249.xml\n",
      "results and discussion\n",
      "rna-seq analysis of cho-k1 cell adaptation to suspension culture\n",
      "screening genetic targets for accelerating adaptation to a suspension culture\n",
      "functional deletion of target genes by dna-free rna-guided cas9 nuclease\n",
      "effect of genome engineering on the adaptability of cho-k1 cell line in suspension culture\n",
      "deep sequencing analysis of dna-free rna-guided cas9 nuclease mediated cleavage at target genomic loci\n",
      "conclusion\n",
      "materials and methods\n",
      "cell culture and adaptation to serum-free growth environments\n",
      "strand specific rna sequencing\n",
      "in vitro dna-free rna-guided cas9 nuclease assay\n",
      "t7e1 assay\n",
      "amplicon sequencing\n",
      "data analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00085/sb5b00085.xml\n",
      "methods\n",
      "materials and equipment\n",
      "plasmid library fabrication\n",
      "genome library fabrication with crispr-cas9\n",
      "genome library fabrication with lambda-red recombination\n",
      "positive selection\n",
      "negative selection\n",
      "phenotypic screening\n",
      "preliminary characterization\n",
      "combining multiple slide modules into bl21­(de3)\n",
      "characterization of escape frequency\n",
      "liquid growth assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400073z/sb400073z.xml\n",
      "anna lechner\n",
      "manuel montabán-lópez\n",
      "dongdong mu\n",
      "auke j. van heel\n",
      "bijan zakeri\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000904/sb3000904.xml\n",
      "results and discussion\n",
      "methods\n",
      "general\n",
      "in vivo mutagenesis protocol\n",
      "sporulation and mating\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00226/sb9b00226.xml\n",
      "results and discussion\n",
      "strain design and construction\n",
      "muconate production is costly but provides a competitive advantage against sensitive e. coli\n",
      "selection of muconate producers from mixed cultures requires spatial structure\n",
      "materials and methods\n",
      "media and chemicals\n",
      "genetic manipulation\n",
      "growth curves\n",
      "bulk competition assays\n",
      "droplet competition assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00529/sb8b00529.xml\n",
      "results\n",
      "cell density measurement by scattered light\n",
      "feedback control for continuous culture\n",
      "cellular physiology in dynamic environments\n",
      "pooled selective growth of diverse libraries\n",
      "phenotypic characterization of diverse libraries in continuous culture\n",
      "fitness effects of rpl28 mutations\n",
      "cycloheximide resistance alleles delineate its binding site on the ribosome\n",
      "discussion\n",
      "methods\n",
      "turbidostat construction\n",
      "calibrating turbidity and cell density\n",
      "bacterial turbidostatic growth\n",
      "yeast media\n",
      "yeast turbidostatic growth in limiting nitrogen\n",
      "estimation of fixed nitrogen content\n",
      "constructing an expression library of fusion proteins\n",
      "genomic fragment library generation\n",
      "expression library vector preparation\n",
      "recombinational cloning and selection of fragment plasmid library\n",
      "recombinant plasmid recovery\n",
      "sequencing library construction\n",
      "deep sequencing and analysis\n",
      "mutational analysis of rpl28\n",
      "genomic rpl28 deletion\n",
      "verification of rpl28δ::klleu2\n",
      "mutagenesis of rpl28 exon 2\n",
      "adding barcodes to mutagenized rpl28 exon 2\n",
      "rpl28 plasmid vector preparation\n",
      "construction of a plasmid library of barcoded rpl28 variants\n",
      "construction of a deep sequencing library linking barcodes to rpl28 variants\n",
      "analysis of deep sequencing data to associate rpl28 variant sequences with barcodes\n",
      "transformation of barcoded rpl28 variants into yeast and pooled growth and selection of the transformant library\n",
      "analysis of plasmid distribution in turbidostat cultures\n",
      "recovery of rpl28 variant plasmids from turbidostat culture\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "construction of a deep sequencing library to quantify barcode abundance\n",
      "analysis of barcode deep sequencing libraries\n",
      "analysis of rpl28 variant phenotypes from barcode abundance data\n",
      "cycloheximide sensitivity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb2000116/sb2000116.xml\n",
      "results and discussion\n",
      "j5 dna assembly design automation software\n",
      "slic/gibson/cpec assembly design with j5 and plasmid construction\n",
      "combinatorial golden gate assembly design with j5 and plasmid library construction\n",
      "linear gene deletion cassette assembly design with j5 and clpx protease markerless deletion\n",
      "experimental characterization of gfpuv variant library\n",
      "summary and conclusion\n",
      "methods\n",
      "j5 software license and availability\n",
      "j5 software implementation\n",
      "strain and sequence availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00450/sb8b00450.xml\n",
      "results and discussion\n",
      "characterization and optimization of persia\n",
      "persia for gene expression analysis: “test-driving” genetic code designs\n",
      "persia with additional enzymatic assay\n",
      "persia with enzymatic assay for probing sequence/function relationships\n",
      "persia with enzymatic assay to evaluate drug resistance with hiv protease\n",
      "integration of persia into a microfluidic/multi-gene assay\n",
      "conclusions\n",
      "methods\n",
      "synthetic dna design\n",
      "protein standard\n",
      "rna standard\n",
      "purexpress master mix solution\n",
      "persia plate reader assay\n",
      "determination of fluorescent substrate binding kinetics\n",
      "analysis of ribosomal gene sequences and rplx variant synthetic dna\n",
      "persia analysis of hivpr variants and alanine scan mutagenesis\n",
      "microfluidic persia\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00385/sb6b00385.xml\n",
      "virus-enabled approach for unnatural amino acid mutagenesis in mammalian cells and tissues\n",
      "evolving orthogonal suppressor trnas to incorporate modified amino acids\n",
      "impacting bacterial adaptation using crispr-cas9 technology\n",
      "engineered polyketide synthases as a synthetic biology tool for chemical production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001728/sb4001728.xml\n",
      "results and discussion\n",
      "development of three new pr-pr automation units\n",
      "tecan automation unit\n",
      "microfluidic device automation unit\n",
      "human language automation unit\n",
      "scandrop microscope automation unit\n",
      "modified combinatorial golden gate dna assembly\n",
      "kunkel mutagenesis\n",
      "gibson dna assembly\n",
      "summary and conclusion\n",
      "methods\n",
      "modified golden gate dna assembly experiments\n",
      "pcr amplification\n",
      "dpni digest and purification\n",
      "bsai digest and gel purification\n",
      "robotic dna assembly\n",
      "microfluidic dna assembly\n",
      "manual dna assembly\n",
      "transformation\n",
      "colony pcr and sequencing\n",
      "kunkel mutagenesis\n",
      "manual protocol\n",
      "microfluidic protocol\n",
      "gibson dna assembly experiments\n",
      "pcr amplification\n",
      "two fragment dna assembly\n",
      "dpni digest and purification\n",
      "microfluidic dna assembly\n",
      "two fragment dna assembly\n",
      "manual dna assembly\n",
      "transformation\n",
      "ncoi digest screen\n",
      "pr-pr software implementation, license, and availability\n",
      "dna sequence, design file, and sequencing trace availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00083/sb7b00083.xml\n",
      "methods\n",
      "chemicals and reagents\n",
      "plasmid and strain construction\n",
      "cyanobacteria growth condition\n",
      "gas chromatography–mass spectrometry (gc–ms) analysis for squalene quantification\n",
      "squalene production in a scalable photobioreactor\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00363/sb7b00363.xml\n",
      "results and discussion\n",
      "midas design overview\n",
      "level-1: module cloning\n",
      "level-2: tu assembly\n",
      "level-3: assembly of multigene constructs\n",
      "metabolic pathway engineering\n",
      "level-1 assemblies\n",
      "level-2 assemblies\n",
      "level-3 assemblies\n",
      "fungal transformations and analysis of indole diterpene (idt) phenotypes\n",
      "complementation of single gene knockout mutations\n",
      "restoration of paspaline production in a δpax deletion mutant\n",
      "restoration of paxilline production in a δpax deletion mutant\n",
      "discussion\n",
      "materials and methods\n",
      "molecular biology\n",
      "bacterial and fungal strains\n",
      "golden gate assembly reactions\n",
      "fungal protocols\n",
      "indole diterpene production and extraction\n",
      "normal-phase thin-layer chromatography (tlc)\n",
      "reverse-phase liquid chromatography–mass spectrometry (lc-ms)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00297/sb6b00297.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00189/sb6b00189.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "device fabrication\n",
      "experimental procedure (chambers)\n",
      "experimental procedure (vesicles)\n",
      "resource sharing model\n",
      "data acquisition and analysis\n",
      "image processing (chambers)\n",
      "image processing (vesicles)\n",
      "noise extraction\n",
      "chamber combination analysis\n",
      "calculating mrna contributions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00157/sb6b00157.xml\n",
      "introduction\n",
      "results and discussion\n",
      "choice of a pulsatile drive signal\n",
      "theoretical investigation\n",
      "experimental entrainment of the oscillator in a microfluidic device\n",
      "conclusion\n",
      "methods\n",
      "model details and simulation\n",
      "reagents\n",
      "strains and plasmids\n",
      "microfluidics and microscopy\n",
      "microfluidics devices\n",
      "chirp signal generation\n",
      "analysis of mean fluorescence\n",
      "image analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00348/sb8b00348.xml\n",
      "results and discussion\n",
      "wild-type gt citap is a high-affinity citrate binder\n",
      "ligand binding of gt citap can be coupled to an optical output\n",
      "the specificity of gt citap can be altered by binding pocket grafting\n",
      "rationally designed binding motifs enable novel ligand specificities of gt citap\n",
      "the specificity of gt citap can be adapted to man-made ligands\n",
      "fluorescently labeled variants of gt citap are active on solid supports\n",
      "gt citap variants enable selective molecular recognition in complex sample matrices\n",
      "conclusions\n",
      "methods\n",
      "recombinant dna work\n",
      "protein expression and purification\n",
      "ligand removal and refolding\n",
      "protein modification with fluorescent dyes\n",
      "isothermal titration calorimetry\n",
      "analysis of ligand binding by fluorescence\n",
      "immobilization of gt citap variants on dnasee7-coated microplates\n",
      "quantification of ethylmalonate in synthetic urine\n",
      "homology modeling and ligand docking\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00196/sb8b00196.xml\n",
      "results\n",
      "effect of glms ribozyme deletion on glcnac production\n",
      "independently dynamic engineering of pathways for improved glcnac production\n",
      "multiple target dynamic engineering of metabolic pathways for improved glcnac production\n",
      "analysis and verification of function of glms ribozyme dynamic control systems\n",
      "discussion\n",
      "materials and methods\n",
      "microorganisms, plasmids and cultivation conditions\n",
      "gene knockout and strain construction\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00372/sb8b00372.xml\n",
      "results and discussion\n",
      "upregulation of erythromycin biosynthetic genes induced by nitrogen-limitation depended on (p)­ppgpp\n",
      "the expression of erythromycin biosynthetic genes and concentrations of (p)­ppgpp revealed synchronization during culture process\n",
      "induction of c-terminally truncated rela increased the intracellular (p)­ppgpp concentration and erythromycin production in s. erythraea\n",
      "engineering of industrial high-producing strain s. erythraea e3 based on induction of (p)­ppgpp synthesis could not enhance erythromycin production\n",
      "endogenous nitrogen limitation led to high concentration of (p)­ppgpp and erythromycin in e3 strains\n",
      "c-di-gmp activates binding of bldd to the ery promoters\n",
      "effect of cdgb overexpression on erythromycin biosynthesis and morphogenesis\n",
      "overexpression of cdgb in the industrial strain e3 enhanced erythromycin yield\n",
      "conclusions\n",
      "material and methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "construction of gene deficient mutant, complementation, and overexpression strains\n",
      "rna preparation and qrt-pcr\n",
      "(p)­ppgpp determination\n",
      "fermentation and erythromycin determination\n",
      "transcriptome assay\n",
      "scanning electron microscopy\n",
      "2-oxoglutarate (2og) determination\n",
      "overexpression and purification of protein\n",
      "electrophoretic mobility shift assay\n",
      "c-di-gmp determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00083/sb8b00083.xml\n",
      "results and discussion\n",
      "design and characterization of a synthetic hes1 promoter with and logic\n",
      "single cell monitoring of endogenous hes1 oscillations with the synthetic sp3 promoter\n",
      "construction, characterization and modeling of the synthetic hes1 oscillator (sho)\n",
      "methods\n",
      "cell culture\n",
      "molecular cloning\n",
      "cytofluorimetry and real-time pcr of transfected cells\n",
      "stable cell line production\n",
      "real time bioluminescence imaging\n",
      "image processing and time series analysis\n",
      "mathematical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00527/sb8b00527.xml\n",
      "results\n",
      "modular assembly of crispr expression vectors used in the comparative study\n",
      "prevalence of pam sequences for orthogonal grna-guided nucleases\n",
      "comparative study of different crispra systems\n",
      "comparative study of different crispri systems\n",
      "testing the orthogonal rna scaffold recruitment system for simultaneous up- and down-regulation of target genes\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial cell culture\n",
      "standardization of genetic parts for golden gate cloning\n",
      "golden gate assembly protocol\n",
      "bioinformatic assessment of pam sequences in the promoter regions of human genes\n",
      "cell culture and transfections\n",
      "rna extraction and qpcr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00412/sb7b00412.xml\n",
      "methods\n",
      "whole-cell biosensor construction\n",
      "substrate screening and sensitivity measurements of the pet15 p emrr -gfp biosensor\n",
      "p emrr -gfp biosensor performance in coculture with fosmid clones in lignin-amended media\n",
      "high-throughput fosmid library screening\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500372z/sb500372z.xml\n",
      "results and discussion\n",
      "construction of transcription units (tus) for expression in s. cerevisiae\n",
      "acceptor vectors\n",
      "designing and constructing pro, cds, and ter parts\n",
      "efficiency of ygg assembly\n",
      "c-terminally tagging tus generated by ygg\n",
      "perspective for further applications using ygg\n",
      "methods\n",
      "one-pot ygg assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00209/sb9b00209.xml\n",
      "results and discussion\n",
      "characterization of the trans -cleavage activity of cas12b on collateral ssdna\n",
      "target dna detection by holmesv2\n",
      "target rna detection by holmesv2\n",
      "target dna quantitation by one-step holmesv2\n",
      "quantitation of target dna methylation degree by holmesv2 combined with bisulfite treatment\n",
      "methods\n",
      "generation of dna targets\n",
      "transcription of sgrnas\n",
      "purification of cas12b protein\n",
      "holmesv2 method\n",
      "asymmetric pcr and lamp\n",
      "preparation of sgrna and cas12b-mediated trans -cleavage reaction\n",
      "determination of the sensitivity for target rna\n",
      "detection of urine spike-ins of target dna or target rna with holmesv2\n",
      "one-step holmesv2\n",
      "dna methylation detection\n",
      "bisulfite treatment, bsp-clone sequencing, bsp-direct sequencing and ngs high-throughput sequencing\n",
      "methylation degree quantitation with holmesv2\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00190/sb5b00190.xml\n",
      "roadmap\n",
      "moving forward\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00088/sb8b00088.xml\n",
      "results and discussion\n",
      "cz preferentially interacts with mixed hybrid-type h-telo g-quadruplex\n",
      "cz induces a conformational switch\n",
      "cz binds to nucleases and exhibits distinct fluorescence signal\n",
      "sensor design and detection of dnase i activity\n",
      "enzyme-regulated logic devices\n",
      "reusable logic devices with multireset function\n",
      "enzyme regulated combinatorial logic gates\n",
      "logic system for the identification of nonzero square numbers\n",
      "conclusion\n",
      "materials and methods\n",
      "general materials\n",
      "fluorescence binding titration with dna targets\n",
      "fret melting analysis\n",
      "cd spectroscopy\n",
      "fluorescence binding titration with nucleases\n",
      "fluorescence titration of cz in different solvent environments\n",
      "fluorescence binding assay to monitor the activity of nucleases\n",
      "reusable logic device with multireset function\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00037/sb8b00037.xml\n",
      "results\n",
      "inferring the regulatory subnetwork containing a positive regulator of defense\n",
      "che is a positive regulator of defense against b. cinerea\n",
      "validating edges in the 9grn model\n",
      "a validated dynamic model of the che regulatory subnetwork\n",
      "design of a feedback controller for perturbation mitigation\n",
      "controller implementation using regulatory network rewiring\n",
      "discussion\n",
      "methods\n",
      "transgenic arabidopsis line\n",
      "infection assay\n",
      "yeast-1-hybrid assay\n",
      "accession numbers\n",
      "generating the tf network\n",
      "generating a dynamic model of the che regulatory subnetwork\n",
      "parameter estimation\n",
      "performance and robustness analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400044g/sb400044g.xml\n",
      "results and discussion\n",
      "low sat leads to a stable off state\n",
      "attenuating noise at both on and off\n",
      "sensitivity, isat, and noise amplification rate at the on state\n",
      "method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00012/sb5b00012.xml\n",
      "results and discussion\n",
      "crispathbrick assembly strategy\n",
      "design of pcrispathbrick and modular assembly of type ii-a crispr arrays\n",
      "repression of plasmid-borne and chromosomally integrated fluorescent reporter in divergent strains\n",
      "attenuation of capsular polysaccharide in virulent and probiotic e. coli strains\n",
      "combinatorial repression of growth by targeting amino acid biosynthesis\n",
      "design of pcrispreporter and quantification of transcriptional repression of endogenous genes\n",
      "dcas9-mediated repression for metabolic engineering of e. coli\n",
      "methods\n",
      "strain and plasmid construction\n",
      "construction of spacer–repeat bricks\n",
      "growth conditions\n",
      "fluorescence assays\n",
      "growth assays\n",
      "metabolite production and quantification\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00094/sb7b00094.xml\n",
      "results\n",
      "plant transcription factors for establishing orthogonal transcription units in yeast\n",
      "test for activation of plant cis -regulatory elements by endogenous yeast tfs\n",
      "transcriptional competence of plant-derived tfs in yeast\n",
      "functionality of native plant tfs in yeast\n",
      "dna-binding domains of plant tfs allow tunable transcriptional output in yeast\n",
      "plant edll acts as a strong activation domain in yeast\n",
      "a wide range of transcriptional outputs from plant-derived atfs\n",
      "transcriptional output of rav1-based atf implemented in centromeric plasmids\n",
      "expression of two plant-derived atfs from plasmids\n",
      "growth effects\n",
      "discussion\n",
      "conclusions\n",
      "materials and methods\n",
      "general\n",
      "bacterial and yeast strains\n",
      "construction of synthetic promoters\n",
      "construction of plant-derived atf expression clones\n",
      "construction of yeast plasmids for expression of two plant-derived atfs\n",
      "induction experiments, flow cytometry and data analysis\n",
      "search for transcription factor binding sites in the yeast genome\n",
      "growth assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300048s/sb300048s.xml\n",
      "a designed variant of human arginase\n",
      "coiled-coil peptide oligomers for protein design and synthetic biology\n",
      "robustness of metabolic networks\n",
      "engineered networks to control cell migration\n",
      "evolutionary synthetic biology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500338g/sb500338g.xml\n",
      "methods\n",
      "strains and recombination\n",
      "recombination\n",
      "recovery time assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00011/sb9b00011.xml\n",
      "results and discussion\n",
      "design of an engineered bmc shell protein\n",
      "structural characterization of bmc-h 2\n",
      "in vivo assembly of capped bmc-h 2 shells\n",
      "conclusions\n",
      "materials and methods\n",
      "cloning and protein expression\n",
      "protein purification of bmc-h 2\n",
      "crystallization and structure determination\n",
      "dynamic light scattering\n",
      "transmission electron microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500085b/sb500085b.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002517/sb5002517.xml\n",
      "materials and methods\n",
      "inactivation of genes involved in terpene biosynthesis\n",
      "generation of bisabolene and farnesyl pyrophosphate synthase expression vectors\n",
      "generation of bisabolene and farnesyl pyrophosphate synthase expression strains\n",
      "bisabolene production in streptomyces venezuelae\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200010w/sb200010w.xml\n",
      "results and discussion\n",
      "methods\n",
      "protein construction and purification\n",
      "sh3 (activator) and pdz domain\n",
      "synthetic regulatory systems\n",
      "peptide labeling\n",
      "fluorescence anisotropy\n",
      "echinoid plasmid construction and echinoid cell-adhesion assays\n",
      "immunostaining, immunofluorescence microscopy, and data analysis\n",
      "analytical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300119v/sb300119v.xml\n",
      "results and discussion\n",
      "design of strains with variable levels of transcription of the ccm and mtr operons\n",
      "high transcriptional levels of ccm and mtr decrease cytochrome c expression and cell growth\n",
      "increasing the ccm promoter activity results in decreased levels of ccme and cytochrome c maturation\n",
      "efficiency of mtrc and mtra synthesis and cell growth decrease strongly with increasing mtr promoter activity\n",
      "anode-reduction correlates with minimal alterations in cell morphology and moderate mtrc and mtra expression\n",
      "transcriptional balance is critical for optimal in vivo synthesis and function of electron conduits\n",
      "conclusions\n",
      "methods\n",
      "plasmids, strains, and growth conditions\n",
      "analysis of cell pellets for assessment of cytochrome c expression and cell growth\n",
      "quantitative assessment of cytochrome c per cell density by diffuse reflectance spectroscopy\n",
      "immunoblotting of ccme\n",
      "electrochemistry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00117/sb7b00117.xml\n",
      "results and discussion\n",
      "model validation\n",
      "model based identification of limiting factors\n",
      "increasing ternary complex concentrations in cfps reactions\n",
      "translational control\n",
      "estimation of cfps potential\n",
      "methods\n",
      "purification of eftu\n",
      "lysate preparation\n",
      "ribosome quantification\n",
      "reaction setup\n",
      "mechanistic model\n",
      "transcription\n",
      "translation\n",
      "initiation\n",
      "elongation\n",
      "termination\n",
      "numerical integration\n",
      "dilution factor\n",
      "translational control analysis\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00374/sb6b00374.xml\n",
      "results and discussion\n",
      "establishment of a regulatable crispri system in s. oneidensis mr-1\n",
      "regulation of the eet efficiency of s. oneidensis by crispri\n",
      "establishment of a regulatable hfq-dependent srna system in s. oneidensis mr-1\n",
      "coordinated transcriptional–translational regulation of eet efficiency in s. oneidensis mr-1\n",
      "materials and methods\n",
      "genes, strains, and growth conditions\n",
      "plasmid construction and transformation and the design of sgrnas and srnas\n",
      "fluorescence assay\n",
      "mfc setup and operation\n",
      "electrochemical analysis\n",
      "electrode attached biomass measurement\n",
      "quantitative real-time reverse transcription polymerase chain reaction (qrt-pcr)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00111/sb7b00111.xml\n",
      "results and discussion\n",
      "method overview\n",
      "growth of bacteria in gel beads\n",
      "determination of enrichment rates\n",
      "ability to find averages of bimodal signals using population measurements\n",
      "sorting a pbad library\n",
      "methods\n",
      "gel-bead formation\n",
      "cell encapsulation\n",
      "cell growth\n",
      "flow cytometry data analysis\n",
      "plasmids\n",
      "time-course experiment ( figure )\n",
      "enrichment experiments ( table )\n",
      "averaging the signal ( figure )\n",
      "pbad library generation\n",
      "sorting of the pbad library\n",
      "measurements of the average fluorescence\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00266/sb6b00266.xml\n",
      "seva linkers: a versatile and automatable dna backbone exchange standard\n",
      "dna replication in engineered e. coli genomes with extra replication origins\n",
      "expanding biosensing abilities through computer-aided design of metabolic pathways\n",
      "benchmarking of tale- and crispr/dcas9-based transcriptional regulators in mammalian cells\n",
      "mammalian synthetic biology: time for big macs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00248/sb8b00248.xml\n",
      "results and discussion\n",
      "turnover dependent phenotypic simulation\n",
      "validation using a toy model\n",
      "tdps validation using experimental results\n",
      "case study 1: phb production in s. cerevisiae\n",
      "case study 2: santalene production in s. cerevisiae\n",
      "case study 3: 3-hydroxypropionic acid production in s. cerevisiae\n",
      "methods\n",
      "implementation of the tdps algorithm\n",
      "precalculations\n",
      "splitting reversible reactions in two half-reactions\n",
      "tdps objective function and formulation\n",
      "tdps_fba objective function\n",
      "mathematical solver\n",
      "toy model simulations\n",
      "genome scale models and reference flux distributions\n",
      "parametrization of the tdps simulation of mutant phenotypes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00187/sb8b00187.xml\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "petal-specific promoters and terminators\n",
      "binary plasmid construction\n",
      "preparation of agrobacterium-competent cells\n",
      "agrobacterium transformation\n",
      "preparation of agrobacterium for infiltration\n",
      "petal injection\n",
      "preparation of crude petal extract\n",
      "visible light spectrometry\n",
      "mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300030d/sb300030d.xml\n",
      "related work\n",
      "results and discussion\n",
      "validation\n",
      "program and platform specification\n",
      "compilation\n",
      "part assignment\n",
      "assembly\n",
      "validation of tasbe products in vivo\n",
      "validation of mammalian construct\n",
      "validation of e. coli construct\n",
      "discussion and conclusions\n",
      "methods\n",
      "program and platform specification\n",
      "specification of programs\n",
      "specification of network motifs\n",
      "specification of cellular platforms\n",
      "compilation\n",
      "interpreting proto into a dataflow graph\n",
      "compilation of dataflow graph to agrn\n",
      "part assignment\n",
      "from agrn to grn: feature assignment and signal matching\n",
      "from grn to part sequence\n",
      "assembly\n",
      "assembly with puppeteer\n",
      "assembly with biocad\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00423/sb8b00423.xml\n",
      "bacterial biofilm amyloids and curli fibers\n",
      "structure and properties of bacterial biofilm amyloids\n",
      "e. coli curli biofilms as biological elms\n",
      "the future prospects of biofilm biological elms\n",
      "bacterial cellulose\n",
      "the structure and properties of bacterial cellulose\n",
      "bacterial cellulose as a biological elm\n",
      "coculture approaches for bacterial cellulose biological elms\n",
      "genetically programming biological elm morphology\n",
      "the french flag model\n",
      "turing patterns\n",
      "phase separation\n",
      "artificial colony symmetry breaking\n",
      "biological elm systems of the future\n",
      "engineering complex natural biological materials\n",
      "building novel biological elms from the bottom up\n",
      "outlook\n",
      "summary\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00244/sb5b00244.xml\n",
      "results and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500173f/sb500173f.xml\n",
      "results\n",
      "single replicon transfection\n",
      "multi-replicon transfection\n",
      "engineering multi-replicon systems\n",
      "discussion\n",
      "methods\n",
      "cloning/rna generation\n",
      "transfection\n",
      "flow cytometry\n",
      "statistical analysis and modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00019/sb5b00019.xml\n",
      "taliman afroz\n",
      "zhen chen\n",
      "victoria hsiao\n",
      "jean-baptiste lugagne\n",
      "diego oyarzún\n",
      "debosmita sardar\n",
      "jason stevens\n",
      "di liu\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500095n/sb500095n.xml\n",
      "artificial genetic circuit based high-throughput enzyme screening system\n",
      "computer-assisted engineering of a synthetic pathway for biodegradation\n",
      "exploring the protein expression characteristics of e. coli\n",
      "multiple amino acid-excluded genetic codes\n",
      "improving the gene-regulation ability of small rnas\n",
      "rewiring the wax ester production pathway\n",
      "spinach rna aptamer as a characterization tool\n",
      "synthetic core promoters for pichia pastoris\n",
      "database construction for promotercad\n",
      "designing synthetic biology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00118/sb9b00118.xml\n",
      "results\n",
      "implementation and optimization of a fatty acyl-coa sensor\n",
      "screening for candidate genes enhancing fatty acyl-coa levels\n",
      "evaluation of candidate genes through fatty acid analysis\n",
      "evaluation of candidate genes through fatty alcohol analysis\n",
      "discussion\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "strains\n",
      "media\n",
      "culture conditions\n",
      "plasmid and strain construction\n",
      "flow cytometry measurements and fluorescence-activated cell sorting\n",
      "library sequencing\n",
      "quantification of fatty acids\n",
      "quantification of fatty alcohols\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500031h/sb500031h.xml\n",
      "results and discussion\n",
      "efficiency of laci-lva functionality\n",
      "natural laci is superior to laci-lva\n",
      "laci response to different iptg concentrations\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00073/sb5b00073.xml\n",
      "introduction\n",
      "results and discussion\n",
      "electrochemical characterization of lacz expression\n",
      "characterization of β-gal produced by biosensor cells\n",
      "real-time electrochemical vs spectrophotometric measurement of β-gal activity\n",
      "electrochemical response of ai-2 biosensor cells\n",
      "two-compartment mathematical model\n",
      "conclusion\n",
      "methods\n",
      "chemical reagents and biocomponents\n",
      "electrochemical pap detection\n",
      "bacterial strains, plasmids and culture conditions\n",
      "miller assay (colorimetric β-gal activity detection)\n",
      "ai-2 detection with bacterial biosensor\n",
      "cell lysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00126/sb8b00126.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "plasmid construction\n",
      "trka-mch-cry2 (opto-full-trka)\n",
      "cry2-mch-itrka (opto-itrka)\n",
      "lyn-itrka-mch-cry2 (opto-lyn-itrka)\n",
      "hsyn-cib1-gfp-caax and hsyn-gfp\n",
      "cell culture and transfection\n",
      "programmable led light box\n",
      "optical activation in culture\n",
      "immunoblotting\n",
      "cell fixation and immunocytochemistry staining\n",
      "fluorescence imaging\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00480/sb8b00480.xml\n",
      "results and discussion\n",
      "ngs-based linker optimization\n",
      "analyzing the trends in ngs sequencing data\n",
      "experimentally assessing the predictive power of the matrix\n",
      "efficient single-pot scarless golden gate assembly scheme\n",
      "finding the optimum set of linkers for dna assembly\n",
      "scarless golden gate assembly\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, media, and cultivation conditions\n",
      "dna manipulation and library construction\n",
      "next generation sequencing library construction and analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb2000275/sb2000275.xml\n",
      "results and discussion\n",
      "identification of potential rna transcriptional activators\n",
      "selection of strong transcriptional activators\n",
      "identification of interacting proteins\n",
      "mutagenesis of the strongest rna transcriptional activator\n",
      "methods\n",
      "synthesis of the target and bait vectors\n",
      "expression of ptrg-var. and pbt-ms2\n",
      "identification of potential rna transcriptional activators\n",
      "selection of strong transcriptional activators\n",
      "identification of interacting proteins by emsa\n",
      "immunoprecipitation of rna transcriptional activator using rna polymerase subunits\n",
      "mutagenesis of the strongest transcriptional activator\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400039g/sb400039g.xml\n",
      "milestones and highlights\n",
      "the coming revolution\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00196/sb5b00196.xml\n",
      "results and discussion\n",
      "construction of a crispr/cas9 workshop for mycoplasma genome engineering\n",
      "customization of a plasmid for expression of grna and integration of glpo target sequence\n",
      "deletion of the glpo gene in the mmc genome resident in yeast\n",
      "evaluation of mycoplasma genome integrity and back transplantation\n",
      "conclusion\n",
      "materials and methods\n",
      "yeast and bacterial strains\n",
      "plasmids\n",
      "yeast transformation\n",
      "screening for mycoplasma genome modification, verification of mycoplasma genome integrity and back transplantation\n",
      "quantification of h 2 o 2 production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00121/sb8b00121.xml\n",
      "results\n",
      "an artificial genetic circuit for the life-death selection of tetracyclines in yeast\n",
      "tetracycline selection in yeast can detect antimicrobial resistance\n",
      "investigation of tetx structure and function using an artificial genetic circuit\n",
      "library construction and differential substrate deep mutational scanning\n",
      "fad positioning is not favorable for atc destruction\n",
      "tetx tolerates mutation differently depending on the tetracycline substrate\n",
      "positioning of atc and dox in tetx is similar\n",
      "dox destruction is more sensitive to a loss of hydrophobic shielding than for atc\n",
      "deep mutational scanning results are consistent with individually isolated mutants\n",
      "discussion\n",
      "conclusion\n",
      "methods\n",
      "microbial culture\n",
      "plasmids\n",
      "survival assays\n",
      "quantitative growth assay\n",
      "detection of anhydrotetracycline in yeast by mass spectrometry\n",
      "ep-pcr library generation\n",
      "tetx library screening\n",
      "next generation sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00284/sb8b00284.xml\n",
      "results and discussion\n",
      "optimization of the bistable circuit in oxygen-limiting conditions\n",
      "inducer concentration\n",
      "plasmid burden\n",
      "metabolic valves to decouple growth from lactate production\n",
      "design, construction, and metabolic optimization of the phenotypic state-controller for metabolic engineering\n",
      "toggle switch driven by state b is limited by background expression\n",
      "toggle switch driven by state a for tighter expression control\n",
      "degradation tags improve switching rate and reduce leaky expression\n",
      "choice of evolved mutant influences phenotype\n",
      "phenotypic stability for scaled production\n",
      "improved production rates using two-stage production\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "media\n",
      "automated 96 well plate preparation and evaluation\n",
      "automated preparation of samples for end-point analyte and reporter measurement\n",
      "inducer toxicity and effectiveness\n",
      "500 ml bioreactor trials\n",
      "switch stability\n",
      "analytical methods\n",
      "analyte data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00457/sb8b00457.xml\n",
      "results\n",
      "system design\n",
      "engineered e. coli can sense a molecule produced by c. albicans\n",
      "engineered e. coli can inhibit filamentation\n",
      "protection of epithelial cells by engineered bacteria\n",
      "sensing and hypha inhibition by engineered e. coli\n",
      "discussion\n",
      "methods\n",
      "bacterial and fungal strains and growth conditions\n",
      "mammalian cell culture\n",
      "plasmid and strain construction\n",
      "tyrosol and hpa measurement by hplc-ms\n",
      "bdsf measurement by hplc\n",
      "dose–response analysis\n",
      "bacterial–fungal coculture\n",
      "hypha inhibition assay\n",
      "analysis of virulence gene expression\n",
      "fungal–mammalian coculture and cytotoxicity assay\n",
      "transwell culture conditions and epithelial barrier measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00100/sb5b00100.xml\n",
      "results and discussion\n",
      "oleic acid inducible enhancers are upstream in the native pox2 promoter\n",
      "uas1b sequences act in a distance-dependent manner\n",
      "core promoter sequence modulates expression level\n",
      "uas1b from xpr2 confers substrate-specific responsiveness\n",
      "tata box sequence has a large influence on expression level\n",
      "tata box engineering modulates expression level of rna polymerase ii promoters\n",
      "proximal promoter sequences can enhance expression level\n",
      "tef proximal promoter sequence enhances expression of engineered promoters in y. lipolytica\n",
      "utility of the tata box mutations in different episomal and genomic contexts\n",
      "conclusions\n",
      "methods\n",
      "chemicals and enzymes\n",
      "strains and cultures\n",
      "general plasmid construction\n",
      "construction of pox2 promoter truncation plasmids\n",
      "construction of hybrid core promoter plasmids\n",
      "construction of hybrid tata box promoter plasmids\n",
      "construction of hybrid proximal promoter plasmids\n",
      "construction of uas1b8 hybrid promoters for genome integration\n",
      "construction of vectors containing three tandem uas­(tef) elements\n",
      "flow cytometry\n",
      "quantitative reverse transcription pcr (qrt-pcr)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300082b/sb300082b.xml\n",
      "spatial modeling\n",
      "dynamic modeling\n",
      "sbml modeling\n",
      "multicellular simulation\n",
      "visualization\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00029/sb8b00029.xml\n",
      "results and discussion\n",
      "mechanistic model of the resource allocation controller\n",
      "model analysis reveals a trade off between gene expression and level of decoupling\n",
      "designing system response times by tuning controller parameters\n",
      "potential biological implementations of the controller designs\n",
      "dynamic resource allocation controller restores modularity in a range of more complex gene circuits\n",
      "conclusions\n",
      "methods\n",
      "numerical simulations\n",
      "assessment of controller function\n",
      "optimization\n",
      "selection of controller parameters for design guidelines\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00245/sb8b00245.xml\n",
      "results and discussion\n",
      "single site-directed mutagenesis by icm system\n",
      "double site-directed mutagenesis by icm system\n",
      "site-directed mutagenesis for creating multiple mutants with the same plasmid backbone\n",
      "site-directed mutagenesis on a large plasmid by icm system\n",
      "comparison of icm system with traditional pcr-based approach for nnk-based single-site saturation mutagenesis\n",
      "comparison of icm system with traditional pcr-based approach for nnn-based double-site saturation mutagenesis\n",
      "discussion\n",
      "material and methods\n",
      "plasmids and bacterial strains\n",
      "expression and purification of cas9\n",
      "sgrna transcription in vitro\n",
      "in vitro cas9-mediated digestion of target plasmid\n",
      "t5 exo-mediated assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00163/sb7b00163.xml\n",
      "results and discussion\n",
      "a single dcas9-vpr activator can predictably repress genes based on sgrna target location\n",
      "streamlining and enhancing sgrna expression using a pol ii-ribozyme system\n",
      "establishing an inducible sgrna set for tunable gene modulation\n",
      "a cistron of up to 4 sgrnas can be used to direct simultaneous dcas9-vpr repression and activation\n",
      "discussion and conclusion\n",
      "methods\n",
      "methods, strains, and media\n",
      "transformations\n",
      "cloning procedures\n",
      "strain construction\n",
      "flow cytometry characterization\n",
      "galactose timecourse experiment\n",
      "quantitative pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400002n/sb400002n.xml\n",
      "results and discussion\n",
      "translational coupling depends on the upstream gene specific sequence and not only on expression level\n",
      "translational coupling in our synthetic operon shows similar coupling enhancement values between the second and third gene location\n",
      "a cascading translational coupling effect is evident between the first and third genes\n",
      "translational coupling depends on the distance between the stop codon and downstream start codon\n",
      "conclusions and comparison to previous studies on translational coupling\n",
      "methods\n",
      "bacterial strains and reagents\n",
      "choosing a compact set of rbs sequences to span expression space\n",
      "flanking “insulator” sequences\n",
      "plasmids\n",
      "operon assembly process\n",
      "fluorescence measurements of reporter proteins\n",
      "could od 600 measurements be influenced by the absorbance of rfp?\n",
      "data analysis\n",
      "rt-qpcr measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002399/sb5002399.xml\n",
      "results and discussion\n",
      "library design\n",
      "library construction and quick quality control (qqc)\n",
      "library quality and diversity\n",
      "library screening effort\n",
      "hit analysis\n",
      "conclusion and outlook\n",
      "methods\n",
      "figure preparation\n",
      "library construction, quality control, screening, and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00150/sb5b00150.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500310v/sb500310v.xml\n",
      "maría-eugenia guazzaroni\n",
      "kana ishimatsu\n",
      "jason g. lomnitz\n",
      "david l. shis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00233/sb6b00233.xml\n",
      "results and discussion\n",
      "characterization of the promoters under aerobic conditions\n",
      "characterization of the promoters under oxygen limitation at low otr max\n",
      "characterization of the promoters under oxygen limitation at high otr max\n",
      "characterization of selected promoters under oxygen low otr max and 10 g l –1 glucose\n",
      "oxygen-limited expression of fbfp using selected promoters and engineered e. coli\n",
      "methods\n",
      "strains\n",
      "parts synthesis and assembly\n",
      "culture media\n",
      "culture conditions\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500237x/sb500237x.xml\n",
      "materials and methods\n",
      "genome mining and screening lov domains for fluorescence\n",
      "characterization of creilov and vaflov\n",
      "gene expression experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00068/sb9b00068.xml\n",
      "results and discussion\n",
      "mutagenesis of mcyh, mcyi, and mcyj in e. coli expression strains\n",
      "export of microcystins from e. coli expression strains\n",
      "inhibition of pp2a by heterologously produced microcystins\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains and culture conditions\n",
      "insertional mutagenesis of mcy genes\n",
      "analysis of mcy transcripts in e. coli\n",
      "heterologous production of microcystins\n",
      "extraction and structural analysis of microcystins\n",
      "protein phosphatase 2a (pp2a) inhibition assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00370/sb6b00370.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna cloning\n",
      "protein expression, purification, and assembly\n",
      "protease assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00397/sb7b00397.xml\n",
      "results and discussion\n",
      "the shinorine biosynthesis in fischerella\n",
      "heterologous production of shinorine in synechocystis\n",
      "production improvement of shinorine using different promoters\n",
      "overproduction of shinorine by tuning the expression of individual genes\n",
      "no effect of exogenous l -serine on the overproduction of shinorine\n",
      "protection of sh-tp560 from uv damages by the expressed shinorine\n",
      "conclusions\n",
      "materials and methods\n",
      "reagents, strains and culture conditions\n",
      "triparental mating\n",
      "extraction of shinorine from cyanobacterial biomass\n",
      "quantitative rt-pcr analysis of shinorine biosynthetic genes\n",
      "hplc and lc–ms analysis of extracted shinorine\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00345/sb7b00345.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids expressing t7 polymerase, cas9, single-guide rna (sgrna), and single-stranded oligodeoxynucleotides (ssodns)\n",
      "sgr-dna library design\n",
      "sgr-dna library construction for e. coli\n",
      "construction of sgr-dna targeting galk under the control of the lac promoter\n",
      "sgr-dna library construction for human cells\n",
      "e. coli cell culture and electroporation\n",
      "functional evaluation of sgr-dna with cas9\n",
      "human cell culture and transfection\n",
      "fluorescence-activated cell sorting\n",
      "clonal expansion and genotyping\n",
      "genomic dna isolation\n",
      "confirmation of target modification by sequencing in the e. coli experiment\n",
      "confirmation of target modification by sequencing in the human cell experiment\n",
      "cpf1 experiments\n",
      "data processing and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400090s/sb400090s.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids\n",
      "antibodies\n",
      "cell culture\n",
      "cell illumination, lysis, and western blotting\n",
      "seap reporter assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00011/sb8b00011.xml\n",
      "results\n",
      "mathematical model of the neurospora cellulase regulation\n",
      "cellulase model suggested clr-2 is the key transcription factor to increase expression level of cellulases\n",
      "synthetic positive feedback gene circuits increase the expression of cellulases\n",
      "transcriptome analysis of the transgenic strains\n",
      "expression of hemicellulase, galacturonase, and laccase genes in the transgenic strains\n",
      "amplified cellulase, hemicellulase, and galacturonase activities in the transgenic strains\n",
      "discussion\n",
      "methods\n",
      "mathematical model\n",
      "strains\n",
      "construction of pcbh-1-clr-1 and pcbh-1-clr-2 donor plasmid vectors\n",
      "transformation of n. crassa with crispr-cas9\n",
      "rna extraction, quantitative rt-pcr, and rna sequencing\n",
      "data analysis for rna sequencing result\n",
      "sugars in culturing media and enzymatic assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400064y/sb400064y.xml\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "construction of the receiver strains\n",
      "promoter activity assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00183/sb9b00183.xml\n",
      "material and methods\n",
      "strains and culture\n",
      "crispr interference\n",
      "sampling for measurement of intracellular and extracellular metabolites\n",
      "lc–ms/ms for metabolite measurements\n",
      "proteomics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001905/sb4001905.xml\n",
      "eran bram\n",
      "bernhard paetzold\n",
      "matthew smith\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001432/sb4001432.xml\n",
      "results and discussion\n",
      "increased hairpin expression level improves rnai efficiency\n",
      "increased hairpin length improves rnai efficiency\n",
      "decreasing hairpin-containing plasmid copy number improves rnai efficiency\n",
      "implementation of rnai in alternate yeast strains\n",
      "rapid prototyping of itaconic acid production in yeasts through rna interference\n",
      "conclusions\n",
      "methods\n",
      "strains and media\n",
      "cloning procedures\n",
      "flow cytometry\n",
      "rt-pcr assay\n",
      "itaconic acid production\n",
      "growth rate analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400077j/sb400077j.xml\n",
      "results and discussion\n",
      "directed chemotactic motility of e. coli toward pseudomonas aeruginosa pao1\n",
      "extracellular secretion and characterization of microcin s against pao1\n",
      "cellular ‘sense and killing’ against pao1 cells\n",
      "antibiofilm activity of dnasei against pao1 biofilm\n",
      "p. aeruginosa targeting motile e. coli with ‘2-hit killing’ systemseek and kill\n",
      "discussion\n",
      "methods\n",
      "strains and growth conditions\n",
      "plasmid construction\n",
      "motility assays\n",
      "protein secretion, purification, and concentration measurement\n",
      "protein electrophoresis and western blotting\n",
      "cell killing assay\n",
      "coculture assay\n",
      "detachment of mature p. aeruginosa biofilm by the secreted biofilm enzyme\n",
      "biofilm cell viability assay\n",
      "confocal microscopy of biofilm\n",
      "transwell assay to assess the final p. aeruginosa -specific ‘seek and kill’ system\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00392/sb8b00392.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "plasmid design and construction\n",
      "yeast strain, transformation, and cultivation\n",
      "fermentation\n",
      "product extraction and uplc–ms analysis\n",
      "rna isolation and confirmation of gene expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00033/sb7b00033.xml\n",
      "a general tool for engineering the nad/nadp cofactor preference of oxidoreductases\n",
      "synthetic biogenesis of bacterial amyloid nanomaterials\n",
      "maintenance and function of a plant chromosome in human cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00338/sb7b00338.xml\n",
      "results and discussion\n",
      "enzyme selection for 1-alkene production\n",
      "cofactor engineering for enhanced 1-alkene biosynthesis\n",
      "dynamic regulation of undb to improve 1-alkene production with relieved metabolic burden\n",
      "expressing a transporter for improved 1-alkene production and secretion\n",
      "methods\n",
      "yeast strains, plasmids, and reagents\n",
      "strain cultivation\n",
      "genetic engineering\n",
      "product extraction and quantification\n",
      "fluorescence microscopy analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00015/sb8b00015.xml\n",
      "results\n",
      "model derivation: dynamic growth\n",
      "model verification: fitting to growth curves\n",
      "flat response\n",
      "flat response following a delay\n",
      "bang–bang response following a delay\n",
      "behavior linked before and after environmental change (dilution into fresh media)\n",
      "discussion\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, growth and transformation\n",
      "growth assays to determine relationship between pre- and postdilution behavior ( figure )\n",
      "growth assays to validate the dynamic growth model ( figures , and )\n",
      "key sequences\n",
      "extrapolation\n",
      "data fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00025/sb6b00025.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "construction of the mineralization pathways of mp and γ-hch in p. putida kt2440\n",
      "confirmation of transcription of the inserted genes in p. putida ktu-9\n",
      "evaluating the capability of p. putida ktu-9 for complete mineralization of mp and γ-hch\n",
      "functional expression of vhb and gfp\n",
      "genetic stability of engineered strain\n",
      "simultaneous mineralization of mp and γ-hch in soil by p. putida ktu-9\n",
      "potential of synthetic biology to create novel strains for bioremediation\n",
      "materials and methods\n",
      "chemicals and culture conditions\n",
      "construction of p. putida kt2440 mutants by scarless gene replacement\n",
      "detection of transcription of the inserted genes by pcr\n",
      "mineralization of mp and γ-hch by p. putida ktu-9\n",
      "co–difference spectral analysis\n",
      "imaging bacteria\n",
      "soil remediation experiments\n",
      "gc–ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00116/sb5b00116.xml\n",
      "intrinsic promoter characteristics\n",
      "results\n",
      "simultaneous measurement of two promoters in a single replicon\n",
      "calculating fluorescent protein synthesis rate from time-course data\n",
      "fluorescent protein synthesis rate is proportional to growth rate in exponential phase\n",
      "fluorescent protein synthesis rate varies significantly under different growth conditions\n",
      "mathematical derivation of promoter characteristics from fluorescent protein synthesis rates\n",
      "ratiometric measurement approximates intrinsic characteristics of promoters\n",
      "discussion\n",
      "methods\n",
      "protocol\n",
      "microbial strains and growth conditions\n",
      "plasmids\n",
      "plate fluorometry assays\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00130/sb5b00130.xml\n",
      "results and discussion\n",
      "design and performance of a csy4-based translational activator\n",
      "optimize tracr system by redesigning rbs\n",
      "optimize translational activator by refining rbs-crrna duplex\n",
      "expanding the translational controller tool box with diverse and orthogonal cas endoribonucleases\n",
      "materials and methods\n",
      "plasmid circuit construction\n",
      "strains, media, and growth condition\n",
      "assay of gene expression by flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400169s/sb400169s.xml\n",
      "results\n",
      "dna strand displacement models\n",
      "dna enzyme models\n",
      "rna enzyme models\n",
      "chemical reaction network design\n",
      "dna strand displacement implementation\n",
      "annihilation\n",
      "catalysis\n",
      "degradation\n",
      "dna enzyme implementation\n",
      "annihilation\n",
      "catalysis\n",
      "degradation\n",
      "rna enzyme implementation\n",
      "annihilation\n",
      "catalysis\n",
      "degradation\n",
      "comparison of implementation strategies\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00070/sb5b00070.xml\n",
      "methods\n",
      "construction of the peerm-series of vectors\n",
      "construction of transgenic synechocystis pcc 6803 strains\n",
      "growth conditions\n",
      "rt-pcr\n",
      "extraction of 13r-mo\n",
      "gc–ms and gc-fid analysis for identification 13r-manoyl oxide\n",
      "pigment quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300071a/sb300071a.xml\n",
      "results and discussion\n",
      "construction of engineered e. coli strains for the production of l -isoleucine\n",
      "engineering of l -isoleucine export system\n",
      "increasing l -isoleucine biosynthesis by the amplification of the ilvced genes\n",
      "engineering of a global regulator to enhance l -isoleucine production\n",
      "fed-batch culture for enhanced l -isoleucine production\n",
      "methods\n",
      "bacterial strains, plasmids and oligonucleotide primers\n",
      "site-directed mutagenesis\n",
      "chromosomal promoter replacement\n",
      "construction of plasmids\n",
      "dl -4-thiaisoleucine tolerance test\n",
      "cell growth inhibition test\n",
      "fermentation\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00155/sb8b00155.xml\n",
      "results and discussion\n",
      "overview of the approach\n",
      "preparation of the initial data of gfp variants for machine learning\n",
      "machine learning with the initial library\n",
      "construction of a machine-learning-guided mutagenesis library\n",
      "screening the gfp variants in the second-round library\n",
      "roles of mutations in yellow fluorescence\n",
      "combination of molecular evolution techniques and machine learning\n",
      "methods\n",
      "preparation of gene fragments of gfp variants\n",
      "preparation of gfp variants\n",
      "gene sequence analysis and fluorescence measurement for gfp variants\n",
      "sandwich elisa\n",
      "machine learning model\n",
      "fluorescence performance score\n",
      "feature vector\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00155/sb9b00155.xml\n",
      "results\n",
      "shell proteins are widely distributed in parageobacillus spp\n",
      "shell proteins are clustered within the pdu operon in p. thermoglucosidasius\n",
      "shell proteins of the pdu bmc from p . thermoglucosidasius are expressed in b. subtilis\n",
      "b. subtilis bmcs can incorporate msfgfp tagged to 24 amino acids from the n-terminal region of the pdup cargo protein\n",
      "assembly of bmcs occurs after 20 min of shell operon induction\n",
      "transmission electron microscopy (tem) analysis confirms the presence of large tubular bmcs in b. subtilis cells\n",
      "bmcs are produced in p. thermoglucosidasius at 60 °c\n",
      "discussion\n",
      "methods\n",
      "genomic data analysis\n",
      "general methods and strain construction\n",
      "fluorescence microscopy\n",
      "fluorescence time-lapse imaging\n",
      "thin sections of cells and transmission electron microscopy (tem)\n",
      "sds-page and western blot\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400025p/sb400025p.xml\n",
      "known knowns\n",
      "known unknowns\n",
      "unknown unknowns\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200016s/sb200016s.xml\n",
      "results and discussion\n",
      "repertoire of cell-free transcriptional activation units\n",
      "crosstalks between transcriptional activation units\n",
      "transcription co-activation by σ 54 and ntrc: an and gate system\n",
      "competition-induced transcription regulation\n",
      "a five-stage transcriptional activation cascade: an example of series circuit\n",
      "parallel circuits\n",
      "transcriptional repression units\n",
      "long-lived cell-free expression\n",
      "conclusion\n",
      "methods\n",
      "crude extract preparation and batch mode cell-free reaction\n",
      "dna part list and plasmid preparation\n",
      "long-lived cell-free reaction\n",
      "phospholipid vesicle preparation\n",
      "measurements\n",
      "mrna turnover and protein degradation\n",
      "gene circuits and regime of plasmid concentrations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00181/sb5b00181.xml\n",
      "a versatile microfluidic device for automating synthetic biology\n",
      "probing yeast polarity with acute, reversible, optogenetic inhibition of protein function\n",
      "talens-assisted multiplex editing for accelerated genome evolution\n",
      "enzymatic menthol production: one-pot approach using engineered escherichia coli\n",
      "partitioning variability of a compartmentalized in vitro transcriptional thresholding circuit\n",
      "sequence design for a test tube of interacting nucleic acid strands\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00191/sb7b00191.xml\n",
      "results and discussion\n",
      "methods\n",
      "microbial strains and growth conditions\n",
      "phaeodactylum tricornutum media\n",
      "transfer of dna to p. tricornutum via conjugation from e. coli\n",
      "plasmid dna isolation\n",
      "plasmid construction\n",
      "western blot\n",
      "rna isolation and cdna synthesis\n",
      "quantitative pcr experiments\n",
      "generation of urease knockouts using cas9 and tevcas9\n",
      "whole plasmid sequencing and genotyping\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300086d/sb300086d.xml\n",
      "combinatorial gene libraries\n",
      "results and discussion\n",
      "problem formulation and notation\n",
      "green fluorescent protein coding libraries\n",
      "polio virus capsid protein coding libraries\n",
      "conclusions\n",
      "methods\n",
      "algorithm description\n",
      "runtime analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000417/sb4000417.xml\n",
      "results and discussion\n",
      "specific detection and destruction of pao1\n",
      "export of copy into the extracellular space of e. coli.\n",
      "characterization of purified copy and nonsecreted and secreted flgm-copy\n",
      "sense and destroy system\n",
      "discussion\n",
      "methods\n",
      "strains and growth conditions\n",
      "plasmids\n",
      "cell density measurements\n",
      "detection of exogenous and endogenous 3oc 12 hsl produced by pao1\n",
      "protein secretion, purification, and concentration measurement\n",
      "western blot\n",
      "experimental setup to calculate the number of sentinels needed to kill one pao1\n",
      "microscopy cell imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00229/sb8b00229.xml\n",
      "results\n",
      "logic inference system\n",
      "nucleic acid equational theory\n",
      "abstraction hierarchy\n",
      "kinetic rate hypotheses\n",
      "enzyme interactions\n",
      "localized interactions\n",
      "complex nucleic acid topologies\n",
      "discussion\n",
      "methods\n",
      "sldnf resolution\n",
      "language semantics\n",
      "process canonical form\n",
      "compilation performance\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00105/sb8b00105.xml\n",
      "results\n",
      "methods\n",
      "high-complexity barcode-grna library construction\n",
      "mock barcode-grna construction\n",
      "lentiviral assembly\n",
      "barcoding cell lines\n",
      "barcode amplification\n",
      "barcode sequencing analysis\n",
      "recall plasmid assembly\n",
      "mock recall screens\n",
      "lineage isolation\n",
      "annexin v red assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00336/sb8b00336.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00256/sb7b00256.xml\n",
      "results and discussion\n",
      "selection and expression of enzymes in the side chain-elongation pathway\n",
      "selection and expression of enzymes in the core structure-formation pathway\n",
      "selection and expression of enzymes in secondary modification\n",
      "functional verification of core formation and side chain-modification pathways in vitro\n",
      "assembling the entire glucoraphanin-biosynthesis pathway in e. coli\n",
      "in vivo production of gra in e. coli harboring the modified synthesis pathway\n",
      "methods\n",
      "functional expression of individual enzymes in e. coli\n",
      "reconstitution of the core structure formation and side-chain modification pathways in vitro\n",
      "integration of the glucoraphanin-synthesis pathway into e. coli\n",
      "verification of gene expression by rt-pcr analysis\n",
      "transforming methionine to gra by resting cell reactions\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00273/sb8b00273.xml\n",
      "results\n",
      "promoter optimization\n",
      "mismatch repair and exonuclease knockouts improve editing frequency\n",
      "retrons can overwrite multiple positions within a 30 base segment\n",
      "retron-mediated mutagenesis\n",
      "discussion\n",
      "materials and methods\n",
      "strains and sequences\n",
      "retron plasmids\n",
      "t7 plasmids\n",
      "induction and plating assays\n",
      "growth assays\n",
      "fluctuation analysis\n",
      "materials\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300028q/sb300028q.xml\n",
      "results and discussion\n",
      "rational peptide design\n",
      "solution-phase biophysical characterization\n",
      "x-ray crystal structures\n",
      "negative design: specifying dimer in cc-pil\n",
      "revisiting gcn4-pil\n",
      "negative design: modulating the stabilities of cc-tri and cc-di\n",
      "conclusions, implications, and applications\n",
      "toward a registry of peptide components ( pcomp ) for synthetic biology\n",
      "methods\n",
      "general\n",
      "peptide synthesis\n",
      "circular dichroism spectroscopy\n",
      "analytical ultracentrifugation\n",
      "dynamic light scattering\n",
      "x-ray crystallography: crystallization\n",
      "x-ray crystallography: data collection and processing\n",
      "x-ray crystallography: structure solution, model building, and refinement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00342/sb7b00342.xml\n",
      "results and discussion\n",
      "long rise-times of metabolite levels in unregulated pathways\n",
      "genetic negative feedback circuits accelerate metabolite dynamics\n",
      "model analysis elucidates metabolite dynamics for the three feedback architectures\n",
      "discussion\n",
      "methods\n",
      "materials\n",
      "plasmids and strains\n",
      "cell growth and fluorescence assay\n",
      "protein and metabolite production dynamics\n",
      "mathematical model and fitting to experimental data\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00124/sb8b00124.xml\n",
      "results\n",
      "indole rescue of fitc-e2 v h w118g activity\n",
      "indole enhances binding affinity of fitc-e2 v h w118g for fluorescein\n",
      "indole stabilizes active fitc-e2 v h w118g\n",
      "indole rescues fitc-e2 v h w118g structure and dynamics\n",
      "structure–activity relationship of rescuing ligands\n",
      "discussion\n",
      "materials and methods\n",
      "protein expression and purification\n",
      "fluorescence quenching assay\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "thermal shift assay\n",
      "synthesis of compound 1\n",
      "spr inhibition in solution assay (isa)\n",
      "size-exclusion chromatography multiangle light scattering (sec-mals)\n",
      "hydrogen exchange mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00287/sb5b00287.xml\n",
      "results and discussion\n",
      "compatible biological components for the screening system\n",
      "phenol-mediated remote enzyme activity detection\n",
      "screening of microbes with phosphatase enzyme activity\n",
      "perspectives on mp-gess applications\n",
      "methods\n",
      "pgess-daat construction\n",
      "construction of pgess- cm /tc/km and evaluation of compatibility\n",
      "verification of mp-gess performance\n",
      "screening of soil microorganisms having phosphatase activity\n",
      "phosphatase enzyme assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5001406/sb5001406.xml\n",
      "results and discussion\n",
      "stochastic simulation\n",
      "performance prediction\n",
      "performance estimation\n",
      "prediction of the fastest ssa\n",
      "prediction inaccuracies\n",
      "large scale model experiments\n",
      "web application\n",
      "discussion\n",
      "methods\n",
      "simulation algorithms\n",
      "biochemical models\n",
      "topological analysis\n",
      "algorithm performance\n",
      "prediction experiment classifiers\n",
      "web application architecture\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00051/sb9b00051.xml\n",
      "results and discussion\n",
      "research design\n",
      "directed evolution of acphers\n",
      "characterization of acphers mutants\n",
      "encoding ncaas with two orthogonal quadruplet codons\n",
      "methods\n",
      "reagents\n",
      "construction of plasmids\n",
      "library construction\n",
      "positive selection\n",
      "negative selection\n",
      "hit validation\n",
      "fluorescence assay\n",
      "protein expression and purification\n",
      "mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00344/sb7b00344.xml\n",
      "results\n",
      "crp strongly activates synthetic promoters with cis -elements immediately upstream of a core promoter\n",
      "cis -element for ihf enhances promoter activity in the complex promoters in the absence of this tf\n",
      "rise of emergent properties in complex promoters with crp and ihf cis -elements\n",
      "ihf cis -elements generates fine-tuning for crp activated promoters\n",
      "emergent properties are consequence of the cis -elements\n",
      "discussion\n",
      "materials and methods\n",
      "plasmids, bacterial strains, and growth conditions\n",
      "design of the minimal promoter scaffold and ligation reactions\n",
      "gfp­(lva) fluorescence assay and data processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00273/sb6b00273.xml\n",
      "existing tools\n",
      "implementation\n",
      "default view\n",
      "abstract view\n",
      "cartoon view\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300125z/sb300125z.xml\n",
      "methods\n",
      "liposome preparation\n",
      "dna constructs\n",
      "dna loading with yoyo-1 dyes\n",
      "spectrofluorometry\n",
      "fluorescence imaging\n",
      "image analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300115n/sb300115n.xml\n",
      "a single-input synthetic gene circuit for reporting sam levels in budding yeast\n",
      "a dual-input tunable synthetic gene circuit tto enrich sam overproducers\n",
      "tuning of the dual-input gene circuit for efficient screening\n",
      "identification of gal11 as a multicopy enhancer of sam level\n",
      "elusive mechanisms for gal11 -induced sam elevation\n",
      "summary and conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00335/sb6b00335.xml\n",
      "results and discussion\n",
      "knocking out sipd and exogenously adding sipd increase secretion titer\n",
      "exogenous sipd addition increases secretion in a concentration-dependent manner\n",
      "exogenous sipd addition acts additively with hila overexpression\n",
      "exogenous sipd addition increases the secretion of total expressed protein\n",
      "t3ss promoters in the hila -controlled secretion pathway have increased activity in δsipd strains and in response to exogenous sipd addition\n",
      "disabling sipd interaction with prgi increases protein secretion\n",
      "the n-terminal domain of sipd is required to increase secretion titer when sipd is added exogenously\n",
      "increase in secretion derived from exogenous sipd addition is independent of transcription and translation\n",
      "methods\n",
      "cloning\n",
      "strains and growth conditions\n",
      "recombinant sipd purification\n",
      "sds-page and western blotting\n",
      "liquid chromatography–mass spectrometry\n",
      "densitometry analysis\n",
      "transcriptional and translational arrest\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00375/sb6b00375.xml\n",
      "results and discussion\n",
      "methods\n",
      "rpod p.putida plasmid\n",
      "plasmid library\n",
      "fosmid library\n",
      "selection experiment\n",
      "tolerance testing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00264/sb6b00264.xml\n",
      "results and discussion\n",
      "high-resolution qtl mapping of hydrolysate tolerance identifies 17 qtls\n",
      "creating a “genetic blueprint” for strain engineering using the qtl map and brha\n",
      "efficient allele replacements in large regions of highly homologous dna using cas9\n",
      "defining a functional part for engineering hydrolysate tolerance: 04a-07\n",
      "cas9-mediated partial allele replacements facilitate ultrafine mapping of qtl parts\n",
      "a 5 kb region at qtl 10a is identified as a part for hydrolysate tolerance engineering\n",
      "allele replacements at qtl 12c indicate hap1 is a part for hydrolysate tolerance engineering\n",
      "qtl-guided metabolic engineering results in a superior phenotype\n",
      "conclusions\n",
      "future considerations\n",
      "methods\n",
      "yeast media\n",
      "yeast strains\n",
      "yeast transformations\n",
      "whole genome sequencing and read mapping\n",
      "variant detection\n",
      "bioreactor fermentations\n",
      "generation and growth of the segregant pool\n",
      "snp allele frequency calculations\n",
      "qtl detection\n",
      "cloning of protospacers into sgrna/cas9 vectors\n",
      "microplate reader growth assays for hydrolysate tolerance screening\n",
      "growth curve data analysis\n",
      "characterizing hydrolysate tolerance of engineered strains in a bioreactor\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00447/sb7b00447.xml\n",
      "results\n",
      "system design\n",
      "fabrication via iterative stepwise additions\n",
      "activation\n",
      "stepwise addition\n",
      "further stepwise additions\n",
      "protein ligation optimization\n",
      "rapid, specific production and use of the n-terminal cysteine\n",
      "optimum temperature, ph and protein concentration\n",
      "identity and buffer conditions of the c-terminal thioester\n",
      "sequential ligations and nanostructure formation\n",
      "functionalizing ligated nanostructures\n",
      "discussion\n",
      "methods\n",
      "construction of fusion protein genes and vectors\n",
      "protein production and purification\n",
      "activation for native chemical ligation\n",
      "n-terminal activation\n",
      "c-terminal activation\n",
      "protein ligation reactions\n",
      "reaction yields\n",
      "accuracy and reproducibility of the ligation reactions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400007z/sb400007z.xml\n",
      "scientific meeting overview\n",
      "reprogramming cells\n",
      "engineering new strains\n",
      "molecular approaches to human health\n",
      "synthetic biology for regenerative medicine and beyond\n",
      "in conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00127/sb5b00127.xml\n",
      "results and discussion\n",
      "in vivo experiments show that synthetic gene oscillators are entrained by aperiodic signals\n",
      "stochastic simulations suggest that both periodic and aperiodic signals can entrain a synthetic gene oscillator\n",
      "d. discoideum car1 receptor entrainment: a computational example of the importance of aperiodic signals to natural biological systems\n",
      "predator–prey entrainment: a computational example of the importance of aperiodic signals to natural biological systems\n",
      "conclusion\n",
      "methods\n",
      "drive signal choice and generation\n",
      "reagents\n",
      "strains and plasmids\n",
      "microfluidics and microscopy\n",
      "analysis of mean gfp fluorescence\n",
      "stochastic simulations of a synthetic gene oscillator with periodic and aperiodic signals\n",
      "d. discoideum car1 receptor entrainment: a computational example of the importance of aperiodic signals to natural biological systems\n",
      "predator–prey entrainment: a computational example of the importance of aperiodic signals to natural biological systems\n",
      "image analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00214/sb5b00214.xml\n",
      "genome engineering in cyanobacteria\n",
      "cas9-facilitated multiloci genomic integration of in vivo assembled dna parts in yeast\n",
      "genome engineering of the 2,3-butanediol biosynthetic pathway\n",
      "bacterial recombineering: genome engineering via phage-based homologous recombination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00046/sb6b00046.xml\n",
      "in vivo real-time control of gene expression\n",
      "slowing translation between protein domains improves solubility\n",
      "development of an unnatural amino acid incorporation system\n",
      "a toolbox of diverse promoters related to methanol utilization\n",
      "efficient reassignment of a frequent serine codon in wild-type e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00267/sb5b00267.xml\n",
      "methods\n",
      "microbial strains and culture conditions\n",
      "plasmid and recombinant strain construction\n",
      "analysis of geraniol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00291/sb6b00291.xml\n",
      "methods\n",
      "bacterial strains, plasmids, media, and growth conditions\n",
      "construction of plasmids and strains\n",
      "metabolite extraction\n",
      "lc–ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00286/sb5b00286.xml\n",
      "results and discussion\n",
      "screening lux-like qs systems\n",
      "rational design of qs systems\n",
      "quorum sensing circuit multiplexing\n",
      "toward microbial consortia\n",
      "methods summary\n",
      "plasmids and strains\n",
      "fluorescence expression measurements\n",
      "dose–response fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00117/sb5b00117.xml\n",
      "results and discussion\n",
      "design of a new flexible and versatile strategy that combines lcr and dna assembler\n",
      "evaluation of the assembly fidelity of lcr\n",
      "evaluation of the unique linkers\n",
      "construction of a 44 kb plasmid using the combined lcr and dna assembler strategy\n",
      "functional analysis of the constructed 44 kb plasmid\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "summary\n",
      "methods\n",
      "strains, media, and cell cultivation\n",
      "ligase cycling reaction (lcr)\n",
      "design of unique linkers (uls)\n",
      "hierarchical dna assembly by combining lcr and dna assembler\n",
      "fermentation for zeaxanthin production\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500309r/sb500309r.xml\n",
      "results and discussion\n",
      "relaxed substrate specificity of yphers (t415g) variant\n",
      "construction of yphers library\n",
      "high-throughput screening of yphers library\n",
      "characterization of the selective yphers variant\n",
      "amino acid activation analysis by yphers_naph\n",
      "elimination of trp misincorporation at uuu codons by yphers_naph in vivo\n",
      "site-specific incorporation of 2nal into a single site of mdhfr in vivo\n",
      "site-specific incorporation of 2nal into a single site of gfp6 in vivo\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "preparation of e. coli hosts\n",
      "plasmid construction for reporter gene expression\n",
      "construction of a homology model of yphers\n",
      "construction of yphers library\n",
      "construction of expression library\n",
      "screening of yphers library\n",
      "characterization of the isolated clones\n",
      "amino acid activation assay\n",
      "construction of plasmids and expression hosts for incorporation of 2nal at phe codons\n",
      "site-specific incorporation of 2nal into a single site of mdhfr and gfp6\n",
      "expression of mdhfr variants and gfp variants in vivo\n",
      "quantitative analysis of codon occupancy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300122q/sb300122q.xml\n",
      "results\n",
      "solid-phase assembly\n",
      "comparison of the three assembly methods\n",
      "creation of multipart combinatorial libraries by ssodn ligation in the presence or absence of linker odn\n",
      "assembly of a protein expression device containing a small rbs library\n",
      "materials and methods\n",
      "materials\n",
      "synthetic oligodeoxynucleotides\n",
      "methods\n",
      "solid support\n",
      "solid-phase assembly of single-stranded oligodeoxynucleotides using rna ligase\n",
      "solid-phase assembly of single-stranded oligodeoxynucleotides in the presence of linker oligodeoxynucleotides using dna ligase\n",
      "solid-phase assembly of double-stranded oligodeoxynucleotides using dna ligase\n",
      "ligation efficiency determination of the three different solid-phase assembly methods by sec\n",
      "purification and pcr amplification of ligation products\n",
      "creation of multipart combinatorial libraries by assembly of ssodn in the presence of linker odn\n",
      "assembly of a protein expression construct containing a small rbs library\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00217/sb5b00217.xml\n",
      "results and discussion\n",
      "agent-based simulation generates a heterogeneous multiphasic growth (hmg) simulator\n",
      "parameter optimization allows matching of the hmg simulator to measured dna distributions from shake flask experiments\n",
      "the hmg simulator can guide construct design and evaluation\n",
      "conclusions\n",
      "materials and methods\n",
      "culture conditions and strains\n",
      "growth assays\n",
      "dna content analysis\n",
      "simulations and optimization tools\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500278z/sb500278z.xml\n",
      "results and discussion\n",
      "conclusions\n",
      "methods\n",
      "mutagenesis\n",
      "protein expression and purification\n",
      "odn–protein conjugation and purification\n",
      "activity assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000468/sb3000468.xml\n",
      "materials and methods\n",
      "plasmids\n",
      "cell culture and transfection\n",
      "induction of membrane fusion\n",
      "illumination and imaging\n",
      "quantification of membrane fusion in varying ph conditions\n",
      "induction of photoactivated apoptosis\n",
      "propidium iodide staining and imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00007/sb5b00007.xml\n",
      "results and discussion\n",
      "rationale and design of casemblr\n",
      "casemblr for multiloci integrative pathway building\n",
      "casemblr for strain building without obvious phenotypes\n",
      "benchmarking casemblr\n",
      "methods\n",
      "strains, plasmids, media and primers\n",
      "dna preparation\n",
      "grna selection and plasmid construction\n",
      "strain construction\n",
      "strain genotyping\n",
      "hplc\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500343g/sb500343g.xml\n",
      "methods\n",
      "construction of arginine biosensor plasmid parg-limex\n",
      "expression strains and media\n",
      "characterization of parg-limex\n",
      "optimization of limex and identification of sorting parameters for flow cytometry\n",
      "protein expression and preparation of crude cell extracts in 96-well microtiter plate (mtp)\n",
      "citrulline-detection system in mtp format\n",
      "ammonia-detection system in mtp format\n",
      "ppadi error-prone pcr (eppcr) library generation and cloning\n",
      "site-directed mutagenesis for recombination of selected amino acid positions\n",
      "expression and purification of ppadi wt, parent m6 and the “best” variant m31\n",
      "kinetic characterization of the purified ppadi variants\n",
      "cultivation conditions for melanoma cell line cultures\n",
      "cell proliferation assay\n",
      "gene synthesis, expression and activity analysis of human arginase i\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500235p/sb500235p.xml\n",
      "results and discussion\n",
      "learning by examples\n",
      "distributed classification with randomized synthetic gene sensors\n",
      "classifier training algorithm\n",
      "classification problems\n",
      "discrimination of two unimodal classes\n",
      "discrimination of a bimodal and a unimodal class\n",
      "analytical description of soft-learning performance\n",
      "summary and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00169/sb9b00169.xml\n",
      "results and discussion\n",
      "construction and expression of a synthetic seracb operon\n",
      "choice of assays for selecting well-expressed operons in expression libraries and for subsequent screening of serine production\n",
      "antibiotic-based selection of a high-expressing sera mut variant from a tir library\n",
      "experimental optimization of downstream tir sequences in the sera mut cb operon enhances serine production\n",
      "selection of alternatively designed rbs variants of the sera mut cb operon for improved l -serine production\n",
      "fed-batch fermentation experiments of selected optimized operon variants show industrially relevant titers\n",
      "performance of a low copy plasmid design is similar when the pathway is integrated into the genome\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, media, and growth conditions\n",
      "plasmid and strain construction\n",
      "tir library design and construction\n",
      "redlibs library design and construction\n",
      "selection experiments\n",
      "mic determination\n",
      "production screening\n",
      "genome integration\n",
      "batch and fed-batch fermentation\n",
      "analysis by lc–ms\n",
      "analysis by hplc\n",
      "sample preparation for proteomic analysis\n",
      "proteomics data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00339/sb6b00339.xml\n",
      "results and discussion\n",
      "in vivo expression validation of spcas9 at different temperatures\n",
      "in vivo validation of spcas9 activity at different temperatures\n",
      "efficient gene deletion using a hr-crispr–cas9 counterselection system\n",
      "expanding the toolbox: knocking out the type i rm system\n",
      "metabolic engineering using spcas9: knock-in of the ldhl gene\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "genomic dna isolation, transformations, colony pcr, sequencing, and phenotypic verification\n",
      "plasmid construction\n",
      "rna isolation and rt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300020f/sb300020f.xml\n",
      "results and discussion\n",
      "zero dimensional projection of landscape and robustness of system\n",
      "one dimensional projection of landscape: u versus q and u versus rmsd\n",
      "projected landscape in two- and three-dimensional space and entropy production rate\n",
      "sensitivity analysis of parameters and control of flux\n",
      "input and output\n",
      "conclusion\n",
      "methods\n",
      "metabolic network ,\n",
      "mean field self-consistent approximation\n",
      "gaussian approximation\n",
      "entropy production rate\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000806/sb4000806.xml\n",
      "results and discussion\n",
      "identification and characterization of the “black mutant” generated by transposon random mutagenesis\n",
      "construction of broadly adaptable site-specific gene integration vectors\n",
      "production of the anti-mrsa wap-8294a2 in lysobacter enzymogenes expressing a tonb-dependent receptor\n",
      "wap-8294a yield in strains expressing phmga-p1-wo8 with a putative terminator\n",
      "hsaf production in the orf8-expressing strains\n",
      "summary and concluding remarks\n",
      "methods\n",
      "bacterial strains, plasmids, and general dna manipulations\n",
      "construction of hmga site-specific integration vectors and screening for hmga mutant\n",
      "construction of promoter-containing phmga-p vectors\n",
      "construction of wap-orf8 expression vectors\n",
      "extraction and analysis of wap-8294a\n",
      "extraction and analysis of hsaf\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00419/sb7b00419.xml\n",
      "results\n",
      "pnatsens: characterization of the native biosensor circuit\n",
      "psynsens: decoupling of the detector and effector module\n",
      "psynsens1.x: engineering the biosensor detector module\n",
      "psynsens2.x: engineering the biosensor effector module\n",
      "discussion\n",
      "methods\n",
      "strains and growth conditions\n",
      "plasmid construction\n",
      "in vivo fluorescence experiments\n",
      "data processing and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00215/sb5b00215.xml\n",
      "overview of sbol 2.0\n",
      "results\n",
      "design of a 4-input and sensor\n",
      "discussion\n",
      "methods\n",
      "dnaplotlib\n",
      "sbol designer\n",
      "cello\n",
      "ibiosim\n",
      "sbol stack\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00144/sb5b00144.xml\n",
      "methods\n",
      "chemicals, strains, and plasmid construction\n",
      "indirect immunofluorescence\n",
      "microscopy, live cell imaging, and image analysis\n",
      "cell culture studies\n",
      "yeast studies\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00076/sb6b00076.xml\n",
      "improving the predictability of part assembly\n",
      "engineering promoter architecture in oleaginous yeast\n",
      "identifying regulatory changes to facilitate nitrogen fixation\n",
      "antigen binding and site-directed labeling of biosilica-immobilized fusion proteins expressed in diatoms\n",
      "flux control at the malonyl-coa node through hierarchical dynamic pathway regulation in yeast\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00427/sb8b00427.xml\n",
      "results and discussion\n",
      "methods\n",
      "escherichia coli strains, plasmids, and linear dna templates\n",
      "buffers used for protein and ribosome purification\n",
      "onepot protein preparation\n",
      "homemade pure protein preparation\n",
      "ni-nta resin preparation and regeneration\n",
      "od600 measurement\n",
      "ribosome purification\n",
      "sds-page gels\n",
      "mass spectrometry\n",
      "energy solution preparation\n",
      "in vitro protein expression and functional assays\n",
      "cost calculations\n",
      "important details and tips\n",
      "data and material availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00467/sb7b00467.xml\n",
      "results and discussion\n",
      "design of second generation opto-crac and carrot constructs\n",
      "chemical and photoinducible nuclear translocation of dcas9\n",
      "chemical and photoactivatable transcriptional modulation\n",
      "methods\n",
      "cloning and plasmid construction\n",
      "cell culture and transfections\n",
      "fluorescence imaging and statistical analysis\n",
      "flow cytometry analysis\n",
      "endogenous genes expression assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001062/sb3001062.xml\n",
      "identification of the isobutyrylmalonyl-coa biosynthetic gene cassette\n",
      "biosynthesis of 36-methyl-fk506 through a genetic approach\n",
      "establishing a tool box for future engineering efforts\n",
      "methods\n",
      "dna sequencing and computer-assisted sequence analysis\n",
      "accession codes\n",
      "construction of expression vectors pld06 and pmw1\n",
      "generation of exconjugants\n",
      "analysis of fk506 congeners\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400115e/sb400115e.xml\n",
      "results and discussion\n",
      "the sequential mechanism of fpp synthesis by erg20p hinders efficient monoterpene synthesis\n",
      "conversion of erg20p into a gpp synthase\n",
      "construction of a dominant negative synthase\n",
      "erg20p fusions\n",
      "screening heterozygous deletion strains\n",
      "heterozygous deletion of the phosphatase genes dpp1 and lpp1\n",
      "deletion of one erg20 allele\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals and enzymes\n",
      "yeast media\n",
      "gene cloning and expression in yeast\n",
      "yeast two-hybrid\n",
      "mutagenesis\n",
      "development of yeast strains\n",
      "terpene quantification and extraction from yeast cells\n",
      "determination of fpp\n",
      "protein expression in bacteria\n",
      "determination of kinetic parameters\n",
      "construction of the erg20p model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400147n/sb400147n.xml\n",
      "results and discussion\n",
      "toxicity of tcp and intermediate metabolites for e . coli\n",
      "construction of tcp degraders by the assembly of an engineered pathway in e . coli\n",
      "experimental characterization of the tcp degraders\n",
      "identification of pathway bottlenecks\n",
      "conclusions\n",
      "methods\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chemicals, media, strains and plasmids\n",
      "molecular techniques and culture conditions\n",
      "analytical techniques\n",
      "determination of toxicity of the substrate and intermediate metabolites\n",
      "determination of expression levels of the enzymes of the tcp pathway\n",
      "verification of expression levels by activity measurements\n",
      "mathematical modeling and optimization of the tcp pathway\n",
      "characterization of the degraders in minimal medium with tcp\n",
      "recovery of degraders in lb medium\n",
      "degradation of tcp in buffer by preinduced resting cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300047u/sb300047u.xml\n",
      "results and discussion\n",
      "design strategy of adaptive assembly\n",
      "functional display of the anchoring scaffoldin on the yeast surface\n",
      "expression of the adapter scaffoldins and their functional docking onto the anchoring scaffoldin\n",
      "functional assembly of the complex cellulosome\n",
      "synergistic effects of the complex cellulosome\n",
      "direct fermentation of amorphous cellulose to ethanol\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "display of anchor protein on yeast surface\n",
      "expression of adaptor scaffoldins and dockerin-tagged cellulases\n",
      "assembly of complex cellulosome of the yeast surface\n",
      "immunofluorescence microscopy\n",
      "enzyme assays\n",
      "fermentation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00078/sb6b00078.xml\n",
      "results\n",
      "features of datel\n",
      "optimization of key parameters to improve assembly efficiency\n",
      "model prediction of the optimum melting temperature\n",
      "characterization of datel in the assembly of multiple fragments\n",
      "rapid construction and optimization of a β-carotene biosynthetic pathway\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00318/sb7b00318.xml\n",
      "results and discussion\n",
      "fine-tuning the expression of secondary metabolites bgcs in streptomyces\n",
      "a novel autoregulated strategy for fine-tuning the expression of bgcs\n",
      "determination of the optimal induction condition\n",
      "selection of qualified native temporal promoters\n",
      "an elaborate design for promoter replacement\n",
      "performance of the selected native temporal promoters\n",
      "fine-tune the expression of otc bgc in a heterologous host\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "construction of plasmids and engineered strains\n",
      "total rna isolation\n",
      "microarray experiments\n",
      "selection of the candidate native temporal promoters\n",
      "real-time qrt-pcr\n",
      "detection of act, jdb and otc\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400089c/sb400089c.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400176e/sb400176e.xml\n",
      "point errors from sequencing or manual data entry\n",
      "real world example\n",
      "gene structure misannotation\n",
      "real world example\n",
      "unintended orfs\n",
      "real world example\n",
      "results and discussion\n",
      "performance\n",
      "sequence inspector reports\n",
      "three illustrative examples\n",
      "methods\n",
      "bioinformatics workflow\n",
      "workflow technical details\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00507/sb8b00507.xml\n",
      "results and discussion\n",
      "mining transcriptional terminators in metagenomic libraries\n",
      "the novel transcriptional terminators display broad host-range activity\n",
      "the novel terminators are functional at single cell level\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "construction of the metagenomic library\n",
      "analysis of transcriptional terminator activity\n",
      "analysis of terminator sequences\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00129/sb5b00129.xml\n",
      "results and discussion\n",
      "methods\n",
      "bacterial strains, media, and chemicals\n",
      "tolerance to 6-aca\n",
      "sequence design and plasmid construction\n",
      "in vitro enzymatic conversions\n",
      "lab-scale fermentation\n",
      "lab-scale fed-batch fermentation\n",
      "lc-ms/ms analysis method for the determination of α-keto acids, 6-aca, aap, adipate semialdehyde, adipate, and homocitrate\n",
      "apex proteomics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00498/sb8b00498.xml\n",
      "results and discussion\n",
      "development of modified s. oneidensis strains\n",
      "cathodic current drives acetoin reduction and hydrogen production\n",
      "light and current drive a target reduction reaction in modified s. oneidensis\n",
      "evaluation of cell density and total protein in bioelectrochemical systems\n",
      "comparison with previous work\n",
      "ongoing and future work\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "design and assembly of bdh and bdh-pr plasmids\n",
      "verifying expression by western blot\n",
      "bioelectrochemical system construction and operation\n",
      "hplc analysis\n",
      "expected 2,3-butanediol production based on charge transfer\n",
      "total protein analysis\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00033/sb6b00033.xml\n",
      "results and discussion\n",
      "selection of 54 candidate class i and ii lantibiotic gene clusters for heterologous expression using the nisin modification machinery\n",
      "five new class i lantibiotics showed antimicrobial activity\n",
      "purified flavucin, agalacticin and bagelicin are potent antimicrobials against gram-positive bacteria\n",
      "ms/ms results confirm the homology-based lanthionine ring pattern of flavucin and agalacticin\n",
      "specific antimicrobial activity of flavucin depends on its dehydration degree and its integrity\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and media\n",
      "cloning and expression of hybrid genes\n",
      "in silico prediction and selection\n",
      "liter scale lantibiotic production and purification\n",
      "antimicrobial activity assays\n",
      "maldi-tof\n",
      "ms/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00116/sb8b00116.xml\n",
      "materials and methods\n",
      "culture growth and experimental conditions\n",
      "intracellular metabolite extraction and analysis\n",
      "carbon labeling experiment\n",
      "isotopically nonstationary metabolic flux analysis (inst-mfa)\n",
      "results\n",
      "requirement for stable cultures for inst-mfa analysis\n",
      "how n nutrient availability changes carbon metabolite pool sizes\n",
      "how n nutrient availability changes 13 c-labeling kinetics of carbon metabolites\n",
      "quantitative analysis of carbon flux distribution\n",
      "discussion\n",
      "changes of carbon flux distributions at metabolic branches under −n\n",
      "the opp pathway\n",
      "metabolic channeling\n",
      "the hgpp pathway induced by nitrogen deprivation\n",
      "the malic cyclic route\n",
      "the ssa route\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00333/sb6b00333.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna constructs\n",
      "cell culture and fluorescence microscopy\n",
      "illumination schemes\n",
      "analysis of endosome distribution\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00281/sb8b00281.xml\n",
      "results and discussion\n",
      "comparison of different pdc biosynthetic pathways via in silico flux response analysis\n",
      "biosynthesis of pdc from glucose\n",
      "redirecting flux into the truncated shk pathway for improved pdc production\n",
      "importer engineering for reduced byproduct dhs accumulation\n",
      "effect of genetic manipulations of the soluble transhydrogenase (stha) on pdc production\n",
      "bioreactor fermentations\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and media\n",
      "plasmid construction\n",
      "genome manipulation\n",
      "in silico flux simulation experiments\n",
      "cultivation\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00163/sb8b00163.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001118/sb4001118.xml\n",
      "results and discussion\n",
      "pdup interacts with pduk\n",
      "solution structure of the p18 peptide\n",
      "saturation transfer difference (std) nmr\n",
      "transferred noe\n",
      "fluorescein-labeled p18 as a reporter\n",
      "localization of a reporter protein using p18\n",
      "construction of an ethanol bioreactor\n",
      "purification of recombinant bioreactors\n",
      "a functional ethanol bioreactor\n",
      "methods\n",
      "bacterial strains and media\n",
      "pull-down assays\n",
      "nmr\n",
      "localization of apex using p18\n",
      "construction of plasmids for ethanol production\n",
      "solubility study of the shell proteins\n",
      "microcompartment purification\n",
      "em and afm\n",
      "activity assays\n",
      "analysis of ethanol production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00290/sb7b00290.xml\n",
      "results and discussion\n",
      "iptg-induced production of sec-containing hgpx4 using trna utu in amber-less e. coli\n",
      "tuning the acceptor stem and t-stem and the t-loop of trna utu results in variants that mediate the expression of hgpx4 with different yields and activities\n",
      "expression and characterization of hgpx1\n",
      "evaluated the sec incorporation ratio by sds-page analysis of purified hgpx2\n",
      "potential mechanisms for the improvement of trna utut6 for hgpx1 expression\n",
      "methods\n",
      "strains and plasmids\n",
      "construction of the hgpx4 and hgpx1 expression plasmids\n",
      "construction of variant chimeric trna ututx and expression plasmids\n",
      "protein expression and purification\n",
      "sds-page analysis\n",
      "gpx activity assays\n",
      "steady-state kinetics of hgpx1\n",
      "electrospray ionization mass spectrometry\n",
      "electrospray ionization tandem mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400053b/sb400053b.xml\n",
      "methods\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cell culture\n",
      "localized cyst transfection in three-dimensional collagen gel culture\n",
      "stable cell lines\n",
      "production and quantification of nk4\n",
      "microscopy quantification\n",
      "qrt-pcr\n",
      "wide-field fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00232/sb7b00232.xml\n",
      "max adrian\n",
      "hasan baig\n",
      "julio r. banga\n",
      "attila becskei\n",
      "ilaria benedetti\n",
      "sun young choi\n",
      "george m. church\n",
      "onur dagliyan\n",
      "roger r. draheim\n",
      "z. hugh fan\n",
      "mumun gencoglu\n",
      "jan b. heidelberger\n",
      "maciej b. holowko\n",
      "hsin-ho huang\n",
      "premkumar jayaraman\n",
      "ryuji kawano\n",
      "seong keun kim\n",
      "jason r. king\n",
      "erkin kuru\n",
      "dae-hee lee\n",
      "benjamin a. e. lehner\n",
      "christina e. lehning\n",
      "marie-laurence lemay\n",
      "jun li\n",
      "sierin lim\n",
      "chueh loo poh\n",
      "natalia i. majewska\n",
      "andrea martella\n",
      "wilco nijenhuis\n",
      "irene otero-muras\n",
      "yili qian\n",
      "john reinhard\n",
      "alexander schmidt\n",
      "gregory stephanopoulos\n",
      "han min woo\n",
      "benjamin m. woolston\n",
      "jing wui yeoh\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00113/sb5b00113.xml\n",
      "results\n",
      "effect of using assembly junctions with different melting temperatures\n",
      "optimizing the annealing step in uracil excision cloning\n",
      "assembling dna fragments directly from the pcr\n",
      "accuracy of uracil excision cloning for assembly of complex metabolic pathways\n",
      "uracil excision cloning for genome engineering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002888/sb5002888.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400051t/sb400051t.xml\n",
      "results and discussion\n",
      "characterization and optimization of oligo meditated plasmid recombineering\n",
      "construction and testing of mipe\n",
      "mipe for combinatorial optimization of metabolic pathways\n",
      "mipe for diversification of protein coding sequence\n",
      "unique features of mipe\n",
      "methods\n",
      "strains, media, and reagents\n",
      "primers\n",
      "recombineering methodology\n",
      "calculation of cos factors with plx15\n",
      "performing mipe\n",
      "screening and characterizing riboflavin overproduction strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00446/sb7b00446.xml\n",
      "methods\n",
      "preparation of dumbbell probe\n",
      "rca reaction\n",
      "sgrna production\n",
      "in vitro dcas9 binding and hrp activity assessment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500347b/sb500347b.xml\n",
      "near-infrared light responsive synthetic c-di-gmp module\n",
      "investigation of the chromophore-binding pocket of light-oxygen-voltage photoreceptors\n",
      "manipulation of endogenous kinase activity in living cells\n",
      "refactoring and optimization of light-switchable bacterial two-component systems\n",
      "benchmarking of optical dimerizer systems\n",
      "orthogonal optogenetic triple-gene control in mammalian cells\n",
      "a golden gate modular cloning toolbox for plants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00275/sb5b00275.xml\n",
      "results and discussion\n",
      "the propositional calculus of dna localized computation circuits\n",
      "directed graph abstraction\n",
      "localized dna circuits as distributed systems\n",
      "example\n",
      "dna mechanisms for track blockage\n",
      "destroying the track\n",
      "removing the walker\n",
      "trapping the walker\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00046/sb7b00046.xml\n",
      "results and discussion\n",
      "instability of the s. stipitis vector lacking a cen\n",
      "gc 3 chromosome scanning for cen prediction\n",
      "stepwise identification of the minimal cen5 for s. stipitis\n",
      "comparison with the ars vector and additional evidence to confirm the cen identity\n",
      "site-specific gene knockout facilitated by the crispr technology\n",
      "materials and methods\n",
      "gc 3 chromosome scanning\n",
      "plasmid construction and yeast transformation\n",
      "flow cytometry analysis and cell sorting\n",
      "step-wise determination of the minimal cen5\n",
      "crispr-based gene knockout\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500280q/sb500280q.xml\n",
      "methods\n",
      "construction of pacr01 and tar capture of alterochromide cluster to generate pacr02\n",
      "construction of pacr03\n",
      "construction of pacr04, pacr05, and pacr06\n",
      "lambda red transfer of gene cluster to generate pacr07, pacr08, and pacr09 (whole cluster and δ alta and δ altn )\n",
      "construction of pacr10\n",
      "analysis of alterochromide production by p. piscicida jcm 20779\n",
      "heterologous expression of alterochromides in e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000756/sb4000756.xml\n",
      "materials and methods\n",
      "plasmid construction\n",
      "single-stranded dna template preparation\n",
      "cell-free protein synthesis experiments (cfps)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00143/sb6b00143.xml\n",
      "results and discussion\n",
      "functional expression of cyp79a2 by protein engineering in e. coli\n",
      "functional expression of cyp83b1 in e. coli\n",
      "selection and expression of the post-aldoxime enzyme\n",
      "selection and detection of suitable myrosinase\n",
      "completion of the bitc biosynthesis pathway in vitro\n",
      "methods\n",
      "strains, plasmids, and chemicals\n",
      "protein modification and construction of expression plasmids\n",
      "protein expression and purification procedures\n",
      "myrosinase activity assay\n",
      "in vitro reconstitution of the bitc biosynthesis pathway\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400070q/sb400070q.xml\n",
      "results and discussion\n",
      "active xylosidase (xyl) expression using e. coli as a host and growth on xylooligosaccharides as a carbon source\n",
      "cellooligosaccharide and xylooligosaccharide-coutilizing e. coli by display of both bgl and xyl\n",
      "methods\n",
      "strains and media\n",
      "plasmids construction\n",
      "evaluation of xyl or bgl activity\n",
      "growth analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00117/sb6b00117.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "reagents, instruments and general methods\n",
      "homology modeling\n",
      "cloning techniques\n",
      "silver sensitivity tests\n",
      "expression and purification of protein compartments\n",
      "icp-ms analysis\n",
      "silver nanoparticle synthesis, kinetics and long-term stability tests\n",
      "em analysis\n",
      "dls analysis\n",
      "disk diffusion assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00294/sb7b00294.xml\n",
      "methods\n",
      "thermometer design and in silico validation\n",
      "plasmid construction\n",
      "purexpress reporter assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500083t/sb500083t.xml\n",
      "results and discussion\n",
      "general strategy for assaying egfr tyrosine kinase activity\n",
      "investigation of acceptable distance between gγ cyto and plasma membrane for g-protein signaling\n",
      "relocalization of autophosphorylated tyrosine residues of egfr cyto to the plasma membrane using a c-terminal lipidation motif\n",
      "integration of genes encoding grb2-gγ cyto and egfr (lr) cyto -fp into yeast chromosomal dna to improve gene stability\n",
      "evaluation of egfr (lr) cyto inhibitory activity of ag1478\n",
      "conclusion\n",
      "methods\n",
      "strains and media\n",
      "construction of plasmids\n",
      "construction of yeast strains\n",
      "fluorescent reporter assay\n",
      "diploid growth assay\n",
      "evaluation of tyrosine kinase inhibition\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00259/sb5b00259.xml\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture and transfection\n",
      "fluorescence spectroscopy\n",
      "luciferase assays\n",
      "sds-page and western blotting\n",
      "flow cytometry\n",
      "confocal microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00041/sb7b00041.xml\n",
      "results\n",
      "identification of a yeast fermentation lipid droplet-targeting domain\n",
      "functional localization of enzymes to lipid droplet membranes\n",
      "spatial organization of a multienzyme pathway colocalized to lds\n",
      "enzyme colocalization enhances acetyl-coa and ethyl acetate biosynthesis rates\n",
      "optimizing pathway architecture and expression enhances ester biosynthesis\n",
      "discussion\n",
      "methods\n",
      "strains and culture conditions\n",
      "plasmid and yeast strain construction\n",
      "fluorescence microscopy and fret analysis\n",
      "preparation of cell lysates\n",
      "enzyme and pathway activity assays\n",
      "protein expression analysis\n",
      "uv–vis ester detection assay and library screening\n",
      "ethyl acetate fermentations\n",
      "gas chromatography analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000977/sb4000977.xml\n",
      "results and discussion\n",
      "real-time in vitro transcription can be monitored with spinach\n",
      "spinach and fluorescent proteins allow for the simultaneous monitoring of in vitro transcription and translation in real-time\n",
      "t7 transcriptional promoter strength is more effective in controlling rna synthesis in vitro than protein synthesis\n",
      "gene position within multigene operons strongly affects protein expression\n",
      "methods\n",
      "genetic constructs\n",
      "in vitro transcription\n",
      "rt-qpcr\n",
      "cell-free transcription–translation\n",
      "proteins and rna standard curves\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001867/sb4001867.xml\n",
      "results and discussion\n",
      "adaptation in a single-gene network\n",
      "gene expression buffering\n",
      "noise suppression\n",
      "tuning network parameters by varying the number of binding sites, interaction type, and mirna number\n",
      "methods\n",
      "plasmids\n",
      "cell culture\n",
      "selection of stable cell lines\n",
      "time-course experiments\n",
      "mrna and mirna quantitative rt-pcr\n",
      "single-molecule fish\n",
      "lna transfections\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500282v/sb500282v.xml\n",
      "results and discussion\n",
      "considerations for creating a compressed codon collection\n",
      "using dynamcc_0 for compressing nonredundant codons\n",
      "using dynamcc_r for exploring the redundant space\n",
      "experimental demonstration for eliminating redundancy\n",
      "conclusion\n",
      "methods\n",
      "algorithm compression logic\n",
      "dynamcc _r\n",
      "dynamcc _0\n",
      "library coverage calculations\n",
      "construction of editing cassettes\n",
      "plasmids\n",
      "creating a muts negative strain\n",
      "λ-red recombineering and crispr-based selection\n",
      "library cells genomic amplification and high throughput sequencing\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00053/sb5b00053.xml\n",
      "materials and methods\n",
      "yeast strain construction and growth conditions\n",
      "microscopy and optogenetic control\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00390/sb7b00390.xml\n",
      "results and discussion\n",
      "optimization of the glycolysis module\n",
      "optimization of the c1 metabolic module\n",
      "optimization of the pyruvate fermentation module\n",
      "optimization of the tca cycle module\n",
      "modular assembly of multiple essential genes identified from metabolic modules to enhance nadh regeneration and promote eet\n",
      "bioelectrochemistry analyses of the eet efficiency of engineered s. oneidensis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "conclusions\n",
      "materials and methods\n",
      "in vitro gene synthesis\n",
      "plasmid construction, transformation and culture conditions\n",
      "bes setup\n",
      "electrochemical analysis\n",
      "riboflavin and metabolite quantification\n",
      "real-time quantitative pcr\n",
      "quantification of intracellular nad­(h/ + )\n",
      "measurement of electrode-attached biomass\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00485/sb8b00485.xml\n",
      "results\n",
      "structure-guided analysis of rex operator sites\n",
      "comparing the performance of promoter designs under aerobic vs anaerobic growth\n",
      "redox state in respiratory chain mutants\n",
      "redox state in e . coli grown under different carbon sources\n",
      "redox biosensor-reporter system as a tool for high-throughput screening\n",
      "discussion\n",
      "materials and methods\n",
      "materials\n",
      "structural comparison of rex operators\n",
      "strains and growth conditions\n",
      "plasmids construction\n",
      "fluorescence normalization\n",
      "quantification of nad + and nadh in different strains\n",
      "fluorescence activated cell sorting (facs)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00051/sb8b00051.xml\n",
      "methods\n",
      "media, strains and plasmids\n",
      "chemical synthesis of gentisyl alcohol\n",
      "dna manipulations\n",
      "feeding experiments\n",
      "de novo biosynthesis of salicyl alcohol and gentisyl alcohol\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00023/sb9b00023.xml\n",
      "results and discussion\n",
      "identification and characterization of the argyrin biosynthetic pathway\n",
      "establishment of a heterologous expression system based on synthetic dna\n",
      "approaches to engineer the argyrin production profile\n",
      "approaches to further improve the heterologous expression system\n",
      "conclusion\n",
      "materials and methods\n",
      "identification and sequence analysis of the arg bgc from cystobacter sp. sbcb004\n",
      "strains, transformation, and cultivation conditions\n",
      "general methods for dna manipulation, analysis, and pcr\n",
      "studies with the native argyrin producer strains\n",
      "sequence design and assembly of synthetic arg bgcs\n",
      "heterologous expression in m. xanthus and argyrin production analysis\n",
      "nrps a1 domain engineering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400157z/sb400157z.xml\n",
      "results\n",
      "negative feedback can modulate activity to meet downstream demand\n",
      "matching output fluxes of interconnected devices\n",
      "experimental results: negative autoregulation balances rna transcription rates in a two-gene artificial network\n",
      "mathematical modeling\n",
      "scalability and alternative architectures\n",
      "scalability\n",
      "alternative positive feedback architecture\n",
      "discussion\n",
      "methods\n",
      "dna oligonucleotides and enzymes\n",
      "oligonucleotide sequences\n",
      "transcription\n",
      "data acquisition\n",
      "numerical simulations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00033/sb9b00033.xml\n",
      "results and discussion\n",
      "cas12a protein screening for c. ljungdahlii\n",
      "improved transformation of c. ljungdahlii using a modified electroporation protocol\n",
      "gene deletion in c. ljungdahlii by using crispr-cas12a\n",
      "gene regulation in c. ljungdahlii by crispr interference (crispri) using the dnase-deactivated cas12a (ddcas12a)\n",
      "adhe1 downregulation in an artificial butyric acid (ba)-producing c. ljungdahlii strain improves ba yield in gas fermentation\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, media, and reagents\n",
      "plasmid construction\n",
      "β-galactosidase assays\n",
      "rna preparation and real-time qrt-pcr\n",
      "modified electrotransformation of c. ljungdahlii with pmtl83151-derived plasmids\n",
      "fermentation and analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003482/sb5003482.xml\n",
      "results and discussion\n",
      "selection of alternative domain-shuffled scaffolds capable of rescuing mating pathway activity in a ste5δ strain\n",
      "identity of membrane recruitment interactions differentiates inducible from constitutive scaffolds\n",
      "synthetic scaffolds can mediate cell–cell fusion\n",
      "synthetic scaffolds can catalytically unlock the mapk fus3\n",
      "specific domain-mediated interactions are required for the proper function of the synthetic scaffolds\n",
      "kinetics of the mating pathway response depends on the domain architecture of the signaling scaffold\n",
      "sensitivity of the mating pathway response does not depend on the domain architecture of the signaling scaffold\n",
      "synthetic scaffolds do not significantly alter growth rate\n",
      "methods\n",
      "library construction and strains\n",
      "fluorescence-activated cell sorting\n",
      "flow cytometry\n",
      "quantitative mating assays\n",
      "site directed mutagenesis\n",
      "western blots\n",
      "growth rates measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00049/sb8b00049.xml\n",
      "results and discussion\n",
      "generated metabolic network around methyl ethyl ketone\n",
      "pathway reconstruction toward five target compounds\n",
      "evaluation of reconstructed pathways\n",
      "flux balance analysis\n",
      "thermodynamics-based flux analysis\n",
      "yield analysis\n",
      "analysis of alternative assumptions on reaction directionalities\n",
      "bridgit analysis\n",
      "identification and analysis of anabolic subnetworks capable of synthesizing target molecules\n",
      "origins of diversity of alternative anabolic subnetworks\n",
      "core precursors of five target compounds\n",
      "clustering of feasible pathways\n",
      "clustering based on core precursors and byproducts of lumped reactions\n",
      "clustering based on involved enzymes\n",
      "ranking of biosynthetic pathways and recommendations\n",
      "further experimental implementation and pathway optimization\n",
      "methods\n",
      "metabolic network generation\n",
      "pathway reconstruction\n",
      "pathway evaluation\n",
      "flux balance and thermodynamic-based flux balance analysis\n",
      "bridgit analysis\n",
      "subnetwork reconstruction analysis\n",
      "identification and lumping of active anabolic subnetworks\n",
      "clustering of subnetworks\n",
      "ranking and visualization of in silico pathways\n",
      "experimental implementation and pathway optimization\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00149/sb7b00149.xml\n",
      "methods\n",
      "lacq gene circuit construction\n",
      "cell culture and transfections\n",
      "inducing cells with iptg\n",
      "flow cytometry\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00065/sb8b00065.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "genetic constructs\n",
      "cell culture\n",
      "transient transfection\n",
      "linker analysis\n",
      "fluorescence microscopy and hardware\n",
      "membrane localization kinetics\n",
      "calcium suppression assays in transfected cells\n",
      "correlation analysis and machine learning prediction\n",
      "lentivirus production and cell line generation\n",
      "calcium dynamics of stable cell lines\n",
      "quantitative pcr (qpcr)\n",
      "stochasticity analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00393/sb8b00393.xml\n",
      "results\n",
      "design of a mirna-mediated synthetic gene circuit in chlamydomonas\n",
      "a fluorescent protein reporter can be engineered as proxy of mirna abundance in vivo (module 1)\n",
      "construction of a reporter system to measure mirna-mediated gene repression activity of a mirna (module 2)\n",
      "generation of synthetic mirnas and target 3′utrs\n",
      "characterization of the repressive activity of a synthetic mirna\n",
      "modulation of the mirna repressive activity\n",
      "a mirna decoy partially inactivates a synthetic mirna\n",
      "construction of self-regulatory loops using intronic mirnas\n",
      "discussion\n",
      "methods\n",
      "strains, growth conditions, and transformation\n",
      "plasmid construction\n",
      "rna extraction\n",
      "northern blots and qpcr\n",
      "mirna cleavage site determination by 5′race\n",
      "fluorescence measurements in plate reader\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00434/sb8b00434.xml\n",
      "results and discussion\n",
      "generation of hrp operon expression constructs by golden gate cloning\n",
      "complementation studies with hrp operon expression constructs\n",
      "modular assembly of the hrp gene cluster using the moclo system\n",
      "modular cloning of accessory and regulatory genes of the hrp gene cluster\n",
      "effector protein translocation by the modular t3s system\n",
      "the orientation of hrp operons within the hrp gene cluster is flexible\n",
      "the modular t3s system facilitates localization studies with fluorescent reporter fusions\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "plant material and plant infections\n",
      "generation of x. euvesicatoria mutant strains\n",
      "generation of hrcd expression constructs\n",
      "generation of expression vectors for golden gate cloning of hrp operons\n",
      "cloning of hrp operons using the golden gate method\n",
      "cloning of the hrp gene cluster, associated and regulatory genes using the moclo system\n",
      "generation of level −2 and level −1 modules\n",
      "generation of level 0 and level 1 modules\n",
      "generation of modules containing reporter genes\n",
      "generation of level m and level p modules\n",
      "generation of expression constructs deleted in single hrp operons\n",
      "fluorescence microscopy\n",
      "protein analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00170/sb6b00170.xml\n",
      "production\n",
      "delay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00144/sb6b00144.xml\n",
      "overview of analog computation\n",
      "potential applications of analog dna circuits\n",
      "prior molecular-scale analog computation\n",
      "our contribution\n",
      "results\n",
      "preliminaries of analog dna gates\n",
      "toehold-mediated strand displacement\n",
      "related concepts\n",
      "input range\n",
      "valid output range\n",
      "abstractions of the gates\n",
      "addition gate\n",
      "the chemical reaction network to inspire our addition gate\n",
      "dna implementation of our addition gate\n",
      "subtraction gate\n",
      "the chemical reaction network to inspire our subtraction gate\n",
      "dna implementation of our subtraction gate\n",
      "multiplication gate\n",
      "the chemical reaction network to inspire our multiplication gate\n",
      "dna implementation of our multiplication gate\n",
      "related work on dna multiplication gate\n",
      "leak reactions\n",
      "simulation results of the gates\n",
      "strategy to construct analog dna circuits\n",
      "an analog dna circuit to compute f ( x ) = 1 + x + x 2 /2! + x 3 /3! for 0 < x < 1\n",
      "simulation of the circuit to compute f ( x )\n",
      "an analog dna circuit to compute g ( x , y ) = 2 y – y 2 x for x , y ∈ (0,2) and g ( x , y ) > 0\n",
      "analog dna circuits for nonpolynomial functions\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500359q/sb500359q.xml\n",
      "systematic transfer of prokaryotic sensors and circuits to mammalian cells\n",
      "modular extracellular sensor architecture for engineering mammalian cell-based devices\n",
      "a novel and stable mouse artificial chromosome vector\n",
      "a theoretical exploration of medea- dependent population suppression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00398/sb7b00398.xml\n",
      "results and discussion\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "evaluation of oxygen-independent reporters favors glucuronidase\n",
      "conventional randomization yields promoters that function in e. coli but not in clostridium\n",
      "manually tuning mutation rates yields functional promoters with diverse strengths in clostridium\n",
      "expression of acetolactate decarboxylase in c. acetobutylicum using synthetic promoters\n",
      "synthetic promoters are portable between clostridium spp\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, growth and plasmid transfer\n",
      "plasmid construction\n",
      "fluorescent reporters\n",
      "glucuronidase assay\n",
      "acetoin fermentation and quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00063/sb8b00063.xml\n",
      "results and discussion\n",
      "circuit design and sensor response to aip-i\n",
      "aip biosensor kinetics\n",
      "specificity and real-time dynamic reporting\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "aip stock solutions, bacterial cells, and antagonist used\n",
      "plasmid constructions\n",
      "transformation of l. reuteri\n",
      "gusa assay\n",
      "kinetic assay\n",
      "measurement of aip-i levels in various microbial cultures\n",
      "perturbation of aip-i levels in staphylococcus aureus by subinhibitory concentrations of erythromycin and ambuic acid\n",
      "mass spectrometry quantification of aip-i\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00132/sb5b00132.xml\n",
      "methods\n",
      "strains and media\n",
      "strain construction\n",
      "construction of cas9 mutations\n",
      "design, cloning and multiplexing of sgrnas\n",
      "transformation of sgrnas\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "remodeling of 36 kilobase histidine genomic region\n",
      "remodeling of 97 kilobase tryptophan genomic region\n",
      "pcr genotyping\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500307w/sb500307w.xml\n",
      "results and discussion\n",
      "methods\n",
      "bacterial growth conditions\n",
      "plasmids and strains\n",
      "growth rate and toxicity calculations\n",
      "mathematical model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00191/sb8b00191.xml\n",
      "results and discussion\n",
      "benchmarking an mrna stability parametrization system\n",
      "appraisal of mrna stability determinants for heterologous gene expression\n",
      "the effect of synthetic stem-loops and adjacent single stranded sequences on mrna decay\n",
      "physiological burden of altered mrna stability\n",
      "conclusion\n",
      "the choice of parts for manipulating mrna stability\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "bacterial growth\n",
      "monitoring gfp expression\n",
      "rna extraction, northern blot analysis and half-life determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300034m/sb300034m.xml\n",
      "results and discussion\n",
      "simulation\n",
      "guarded-command programming\n",
      "examples\n",
      "refinement\n",
      "conclusion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00055/sb8b00055.xml\n",
      "results and discussion\n",
      "operating principle of hites and instability of the first-generation hites\n",
      "construction of a stable inducible hites\n",
      "stability of the inducible hites for long-period cultivation\n",
      "the cross-species application of hites\n",
      "the regulation performance of the hites\n",
      "construction of stable constitutive hitess in sinorhizobium th572\n",
      "constitutive expression of d -hase and d -case in sinorhizobium th572\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "plasmid construction\n",
      "induction of protein expression\n",
      "fluorescence measurements\n",
      "growth measurements\n",
      "western blot\n",
      "serial subculture experiments\n",
      "enzyme activity measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300050j/sb300050j.xml\n",
      "methods\n",
      "molecular biology and mutagenesis\n",
      "wavelength selection and library screening with tandem heterodimers\n",
      "protein purification and characterization\n",
      "live cell imaging\n",
      "cell culture experiments\n",
      "fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500226j/sb500226j.xml\n",
      "results and discussion\n",
      "conserved shell protein features in new high resolution crystal structures of bmc-h proteins from both α- and β-carboxysome\n",
      "design of chimeric carboxysome shell and pore mimics\n",
      "in vivo testing of native or mutant bmc-h proteins for incorporation in β-carboxysomes\n",
      "native and pore mutant α- and β-type bmc-h proteins can structurally complement a ccmk2 deletion (carboxysome-minus) mutant\n",
      "conclusions and outlook\n",
      "methods\n",
      "protein purification\n",
      "crystallization\n",
      "structure determination and analysis\n",
      "chimeric shell protein design, pore mutant strains construction, and growth conditions\n",
      "fluorescence microscopy\n",
      "transmission electron microscopy\n",
      "phylogenetic analysis for bmc-h proteins\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00128/sb7b00128.xml\n",
      "results and discussion\n",
      "selection of rna aptamers for naringenin\n",
      "in vivo selection of naringenin riboswitches\n",
      "dose–responses differed depending on the selection route\n",
      "response of the naringenin riboswitches to other flavonoids\n",
      "methods\n",
      "preparation of a naringenin-coupled matrix\n",
      "in vitro selection of the rna aptamers\n",
      "construction of the riboswitch library\n",
      "in vivo selection of the naringenin riboswitches\n",
      "fitting of the dose–response curves and evaluation of the ec 50 s\n",
      "sgfp expression assay to determine the specificity of the riboswitches\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200021s/sb200021s.xml\n",
      "results and discussion\n",
      "definition of the gene circuits\n",
      "silencing of protein expression by trans–cis annealing\n",
      "tuning of post-transcriptional control by iptg\n",
      "test of the modularity of the parts\n",
      "methods\n",
      "cis and trans sequence design\n",
      "plasmid construction\n",
      "fluorescence measurements\n",
      "purification of laci repressor\n",
      "dna preparation\n",
      "gel shift assay\n",
      "statistical analysis\n",
      "mathematical model and parameter definition\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001084/sb3001084.xml\n",
      "results and discussion\n",
      "functional expression of gdmd\n",
      "functional expression of ltnj\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "molecular cloning\n",
      "construction of expression vectors\n",
      "protein expression and purification\n",
      "mass spectrometry\n",
      "maldi-tof\n",
      "esi-ms\n",
      "antimicrobial activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00030/sb9b00030.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids and cell strains\n",
      "flow cytometry and data analysis\n",
      "e. coli experiments\n",
      "s. cerevisiae experiments\n",
      "modeling\n",
      "parameter searching\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00002/sb8b00002.xml\n",
      "results and discussion\n",
      "c. pinatubonensis jmp134 as a host for synthetic qs networks and beneficial plant–bacteria interactions\n",
      "characterization of a lux-like qs system in c. pinatubonensis jmp134 to create an autoinducible regulation of gene expression\n",
      "qs-dependent iaa biosynthesis in c. pinatubonensis jmp134 creates a synthetic beneficial interaction with a. thaliana\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "plate reader fluorescence and flow cytometry analysis\n",
      "sds-page\n",
      "measurement of iaa synthesis\n",
      "plants construction, inoculation, and growth parameters\n",
      "confocal microscopy analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00017/sb7b00017.xml\n",
      "results and discussion\n",
      "protected oligonucleotide duplex assisted cloning\n",
      "case study 1: iterative assembly of a biosynthetic pathway\n",
      "case study 2: cloning of constructs containing sequence repeats\n",
      "conclusion\n",
      "methods\n",
      "chemicals, oligonucleotides and molecular reagents\n",
      "plasmids and genes\n",
      "podac\n",
      "strains\n",
      "media and culture conditions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00414/sb7b00414.xml\n",
      "results\n",
      "the design of synthetic high-order oligomerization domains and cognate regulated promoters\n",
      "the oligomeric domains increased the ultrasensitivity of all tested transcriptional repressors\n",
      "the oligomeric domains did not affect the allosteric capability of the ligand-responsive transcription factors\n",
      "ultrasensitive chimeric transcription factors predictably tuned and improved the gene circuit functions\n",
      "discussion\n",
      "materials and methods\n",
      "media\n",
      "molecular biology reagents\n",
      "staining and plasmids\n",
      "cell growth conditions\n",
      "fluorescence measurements\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00428/sb7b00428.xml\n",
      "results and discussion\n",
      "selection of targets and design and assembly of the lac ocd\n",
      "effect of laccase ocd activation on growth and protein profiles of five e. coli strains\n",
      "in vitro performance of the laccase ocd in e. coli dh5α\n",
      "in vitro performance of the laccase ocd in other genetic backgrounds\n",
      "in vitro performance of ocds based on laccase variants\n",
      "in vivo performance of ocds\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals\n",
      "strains and media\n",
      "selection of protein targets, device design, and dna synthesis\n",
      "assembly of ocds\n",
      "effect of lac ocd induction on e. coli growth\n",
      "sample collection and preparation of cell extracts\n",
      "confirmation of protein expression\n",
      "characterization of ocds\n",
      "in vitro assays\n",
      "in vivo assays\n",
      "online tools and statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400063n/sb400063n.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500291r/sb500291r.xml\n",
      "results and discussion\n",
      "analysis of optical dimerizers using a yeast transcriptional system\n",
      "generation of an optimized blue light regulated transcription system\n",
      "light-inducible activation of a map kinase pathway\n",
      "discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "β-galactosidase assays\n",
      "yeast growth assays\n",
      "immunoblotting\n",
      "map kinase experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400177x/sb400177x.xml\n",
      "results and discussion\n",
      "synthetic biology platform, screening, and compound isolation\n",
      "synthetic biology compound properties\n",
      "synthetic biology compound novelty analysis\n",
      "synthetic biology compound diversity analysis\n",
      "chemical space classification by molecular quantum numbers\n",
      "chemical space analysis using chemgps-np\n",
      "activity of synthetic biology compounds\n",
      "utility of the approach\n",
      "methods\n",
      "yeast strains\n",
      "primary bmv assay screening\n",
      "confirmation of primary hit clones\n",
      "secondary bmv assay screening\n",
      "purification of compounds for ms and nmr characterization\n",
      "nmr analysis of compounds\n",
      "calculation of physicochemical properties\n",
      "diversity analysis using maccs keys\n",
      "chemgps analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000466/sb5000466.xml\n",
      "methods\n",
      "organism, cultures, and plating\n",
      "plasmid library assembly\n",
      "solution analysis\n",
      "results and discussion\n",
      "four-city tsp\n",
      "translating tsp solutions into plasmids\n",
      "plasmid assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00385/sb8b00385.xml\n",
      "results\n",
      "discovery of c 60 -phytoene synthase activity\n",
      "identification of additional size-shifting substitutions\n",
      "further attempts to increase the production of larger carotenoids\n",
      "biosynthesis of the c 55 carotenoid pigments\n",
      "discussion\n",
      "methods\n",
      "strains and reagents\n",
      "plasmid construction\n",
      "culture conditions\n",
      "product extraction and purification\n",
      "hplc–ms analysis for compound identification and quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00119/sb5b00119.xml\n",
      "results and discussion\n",
      "biochemical comparison of the switches\n",
      "dark and lit state binding affinities\n",
      "light-dependent cry2 homo-oligomerization\n",
      "reversion kinetics\n",
      "comparison of switch behavior in the cytoplasm of live cells\n",
      "controlling subcellular localization\n",
      "mitochondrial relocalization assay\n",
      "functional comparison of the switches\n",
      "light-controlled transcription in yeast\n",
      "manipulation of lamellipodial protrusion\n",
      "correlating in vitro binding measurements with in vivo activities\n",
      "correlating in vitro and in vivo kinetics of activation and reversion\n",
      "practical considerations\n",
      "methods\n",
      "cloning\n",
      "expression and purification\n",
      "fluorescent probe generation\n",
      "fluorescent polarization binding assay\n",
      "multiangle light scattering\n",
      "dynamic light scattering\n",
      "absorption recovery after activation\n",
      "yeast plasmids generation\n",
      "yeast transformation and mating\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "β-galactose assay\n",
      "yeast growth assays\n",
      "mammalian cell culture and transfection\n",
      "mammalian cell localization/gef microscopy\n",
      "image analysis and quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00375/sb7b00375.xml\n",
      "results\n",
      "target deletion and nhej mediated targeted integration via crispr/cas9\n",
      "establishment of cho-k1 cell lines modified by hsqsox1b and survivin and its er location identification\n",
      "gsh/gssg ratio in the cells that expressing hsqsox1b\n",
      "enhanced resistance to apoptosis\n",
      "effects of overexpression of kdel-tagged hsqsox1b on the folding of gaussia luciferase\n",
      "cell viability during the serum deprivation\n",
      "discussion\n",
      "necessity and significance of improving er lumen microenvironment and reinforcing antiapoptotic ability of cho-k1 cells\n",
      "biological activities of human derived protein qsox1 and survivin and their functions in modified cho cells\n",
      "advantages of precise editing of the genome of cho-k1 cell with dual-genes by crispr/cas9 technology and further improvements\n",
      "material and methods\n",
      "sgrna target design and sgrna-cas9 plasmid construction\n",
      "expression cassette construction\n",
      "cell culture, transfection, and isolation of clones by facs\n",
      "genomic dna extraction, pcr amplification, and t7e1 assay\n",
      "ta cloning and sanger sequence\n",
      "western blot and live cell imaging\n",
      "measurement of gsh and gssg\n",
      "facs analysis to detect apoptotic cells\n",
      "methylthiazoletetrazolium assay\n",
      "gaussia luciferase assay\n",
      "statistic analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400110j/sb400110j.xml\n",
      "results and discussion\n",
      "soxr as an nadph sensor\n",
      "correlation between fluorescence and lb adh activity\n",
      "application of soxr for high-throughput screening of nadph-dependent enzymes\n",
      "methods\n",
      "bacterial strains, media, and growth conditions\n",
      "recombinant dna work\n",
      "whole-cell biotransformation in microtiter plates\n",
      "ninety-six-well screening system\n",
      "fluorescence microscopy\n",
      "fluorescence-activated cell sorting\n",
      "determination of alcohol dehydrogenase activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00426/sb7b00426.xml\n",
      "results and discussion\n",
      "cloning, heterologous co-overexpression, and copurification of a domains used in this study with their mbth-like protein (mlp) partners\n",
      "characterization of the adenylating activity of ecm6­(a 1 t 1 ), ecm6­(a 1a m h a 1b t 1 ), and ecm6­(a 1a m 3s a 1b t 1 )\n",
      "evaluation of the methylating activity of ecm6­(a 1a m h a 1b t 1 ) and ecm6­(a 1a m 3s a 1b t 1 )\n",
      "elucidation of m domain methylation specificity\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00235/sb7b00235.xml\n",
      "methods\n",
      "database organization\n",
      "literature curation\n",
      "software\n",
      "compatibility\n",
      "data accession\n",
      "results and discussion\n",
      "database description\n",
      "genes and functional classes over-represented in e. coli exposed to stress\n",
      "enriched interactions in regulatory networks\n",
      "combining genome and transcriptome data to ascertain targets not yet mutationally exploited\n",
      "estimating promiscuity in differential expression\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00241/sb9b00241.xml\n",
      "results\n",
      "ssdna configuration\n",
      "short ssdna with dsdna handles\n",
      "dsdna with ssdna handles\n",
      "replication-fork-shaped dna\n",
      "dna holiday junction\n",
      "three-site multiple-labeled and nicked dna\n",
      "rna structures with rdh handles\n",
      "torsion-constrained rna\n",
      "a few of the new na configurations\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500096y/sb500096y.xml\n",
      "methods\n",
      "host strain and media\n",
      "modular construction of plasmids\n",
      "yeast transformation\n",
      "flow cytometry and data processing\n",
      "secreted cellulase activity\n",
      "fermentation of avicel to ethanol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00433/sb7b00433.xml\n",
      "methods\n",
      "construct\n",
      "synthesis of precharged trnas\n",
      "cell-free protein synthesis and incorporation of fluorescent uaas\n",
      "samples for fluorescence microscopy\n",
      "slide preparation\n",
      "smfret measurements and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00056/sb5b00056.xml\n",
      "fei cai\n",
      "jay h. choi\n",
      "jan claesen\n",
      "roger r. draheim\n",
      "lauren e. s. gulland\n",
      "yoshiki higuchi\n",
      "yukiko kamiya\n",
      "kento koketsu\n",
      "carlos piñero lambea\n",
      "yi liu\n",
      "arnab mukherjee\n",
      "wei niu\n",
      "avena c. ross\n",
      "jennifer a. schoborg\n",
      "yasuhito shomura\n",
      "markus sutter\n",
      "lennart schada von borzyskowski\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400018u/sb400018u.xml\n",
      "timothy harvey-samuel\n",
      "heather jensen\n",
      "li jin\n",
      "tae seok moon\n",
      "kevin solomon\n",
      "claudia vickers\n",
      "adam s. walker\n",
      "thomas c. williams\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00090/sb5b00090.xml\n",
      "results and discussion\n",
      "computationally aided selection of qacr mutant sequences\n",
      "in vitro screening of generated sequences\n",
      "further in vitro testing of qacr2 and qacr5\n",
      "in vivo testing of qacr2 and qacr5\n",
      "framework enables engineering of sensors through rational reduction of design space\n",
      "methods\n",
      "computationally aided selection of mutant sequences\n",
      "cell free in vitro transcription–translation system and reactions\n",
      "cell strain and media\n",
      "genes and plasmids\n",
      "in vivo experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00406/sb7b00406.xml\n",
      "results and discussion\n",
      "a microfluidic assay to study the emergent behavior of bacterial populations in real time and at single-cell resolution\n",
      "a simple mathematical model of chemotaxis accurately recapitulates emergent chemotactic behavior\n",
      "quantifying qs responses in chemotactic bacterial populations\n",
      "the role of spatial heterogeneity in the timeline of the qs response\n",
      "quantitative tuning of synthetic genetic circuits using the predictive computational framework\n",
      "methods\n",
      "strains, media, and growth conditions\n",
      "modeling bacterial motility and chemotaxis\n",
      "modeling physical interactions between cells\n",
      "modeling quorum sensing\n",
      "accounting for the qs initial condition\n",
      "modeling synthetic rbss\n",
      "modeling bacterial growth\n",
      "numerical method\n",
      "plasmid construction and transformation\n",
      "microfluidic assay device\n",
      "chemotaxis experiments\n",
      "qs experiments\n",
      "quantification and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00296/sb7b00296.xml\n",
      "results and discussion\n",
      "identification of the 32 longest contiguous deletions for e. coli using mingenome\n",
      "e. coli minimized genome using gene essentiality from both e. coli and buchnera spp. along with transcription factor information\n",
      "identification of multiple longest contiguous deletions for e. coli using mingenome while allowing for one gene reinsertion (per deletion) to the genome\n",
      "conclusion\n",
      "methods\n",
      "mingenome: optimization-driven workflow for genome minimization\n",
      "sets\n",
      "parameters\n",
      "variables\n",
      "milp representation of mingenome algorithm\n",
      "gpr constraints\n",
      "identifying multiple simultaneous deletions with mingenome\n",
      "genome minimization algorithm for larger dispensable stretches on the chromosome while allowing for gene reinsertions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500019b/sb500019b.xml\n",
      "results and discussion\n",
      "expression and purification of diverse cyanobactin enzymes and substrates\n",
      "biochemical analysis of thc pathway enzymes\n",
      "efficient heterocyclase activity is driven by rsi\n",
      "leader peptide coevolves with cyanobactin modification enzymes\n",
      "noncognate leaders do not hinder heterocyclase reactivity\n",
      "importance of rsii and insignificance of the native leader peptide for n-terminal proteolysis\n",
      "portability of rss to produce cyanobactins in vivo\n",
      "conclusions\n",
      "methods\n",
      "genes and cloning\n",
      "protein expression, purification, and synthesis\n",
      "thce4 ptm assay conditions\n",
      "precursor peptide assay conditions and product characterization\n",
      "synthetic substrate assay conditions and product characterization\n",
      "competition reaction assay conditions and product characterization\n",
      "sds-page band-shift assay\n",
      "mass spectrometric methods\n",
      "construction of phylogenetic trees\n",
      "expression of hybrid precursor peptides in e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00232/sb5b00232.xml\n",
      "methods\n",
      "overview of double dutch workflow\n",
      "dna component categorization\n",
      "problem formulation\n",
      "module clustering\n",
      "module assignment\n",
      "results\n",
      "cost function\n",
      "performance\n",
      "discussion\n",
      "access\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00091/sb8b00091.xml\n",
      "results and discussion\n",
      "in silico analysis of pptases and cps\n",
      "phosphopantetheinylation of noncognate carrier proteins in vitro\n",
      "phosphopantetheinylation of noncognate carrier proteins in vivo\n",
      "in vivo activation of a multidomain nrps-pks gene cluster\n",
      "conclusions\n",
      "methods\n",
      "in silico analysis of proteins used in this study\n",
      "construction of expression plasmid\n",
      "protein expression and purification\n",
      "in vitro phosphopantetheinylation assay and detection of phosphopantetheinylation by high performance liquid chromatography (hplc)\n",
      "detection of in vivo phosphopantetheinylation by maldi-tof/tof mass spectrometry\n",
      "in vivo activation of multidomains nrps-pks gene cluster\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00320/sb7b00320.xml\n",
      "beyza bulutoglu\n",
      "farzaneh moghadam\n",
      "gita naseri\n",
      "michael pineda\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200007r/sb200007r.xml\n",
      "methods\n",
      "plasmids\n",
      "cell culture and transfection\n",
      "illumination and imaging\n",
      "co-localization analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00277/sb7b00277.xml\n",
      "results\n",
      "control of reporter fluorescence using strand displacement\n",
      "regulating rna polymerase activity via dna kleptamer displacement of inhibiting aptamers\n",
      "dynamic control of transcription using aptamers and rna kleptamers\n",
      "discussion and conclusions\n",
      "materials and methods\n",
      "oligonucleotides and enzymes\n",
      "oligonucleotide sequences\n",
      "strand displacement and transcription\n",
      "fluorescence data acquisition\n",
      "numerical simulations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00075/sb6b00075.xml\n",
      "results and discussion\n",
      "processing and statistical modeling of gene expression data\n",
      "elucidating key tfbss for representative strong promoters\n",
      "development and experimental testing of synthetic mammalian promoters using enriched tfbss\n",
      "conclusion\n",
      "methods\n",
      "processing of gene expression data\n",
      "annotation and analysis of candidate promoter sequences\n",
      "plasmid construction and preparation for cell culture\n",
      "cell culture and transient transfections\n",
      "expression quantification using flow cytometry and chemiluminescent reporter assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00073/sb9b00073.xml\n",
      "methods\n",
      "construction of plasmids\n",
      "puc19-acrlla4\n",
      "phl-ef1α-acrlla4-ip-a\n",
      "phl-ef1α-acrlla4_e70r-ip-a\n",
      "phl-ef1α-mir_acrlla4-ip-a (mir = t21, t302a, 4× t21)\n",
      "phl-ef1α-m9_acrlla4-ip-a and phl-ef1α-t21_m9_acrlla4-ip-a\n",
      "phl-grna­[egfp]-mef1α-mrfp-rih\n",
      "phl-grna­[tre]-mef1α-mrfp-rih\n",
      "phl-grna­[target]-mef1α-irfp670-rih (target = tre and egfp)\n",
      "ptre-tight-hmag1\n",
      "pb-tre_hmag1\n",
      "template dna for in vitro transcription (ivt)\n",
      "cas9 and acrlla4 mrnas and sgrna synthesis and purification\n",
      "generation of hela-tre hmag1 cell line\n",
      "cell culture\n",
      "mirna inhibitors and mimics\n",
      "cas9 activity assay (egfp knockout assay)\n",
      "dcas9-vpr activity assay (hmag1 activation assay)\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00509/sb8b00509.xml\n",
      "results and discussion\n",
      "departure point: parental laccases\n",
      "schema-raspp library design and recombination in vivo\n",
      "sequence analysis and thermostability of laccase chimeras\n",
      "activity pattern of thermostable laccase chimeras\n",
      "ph stability of thermostable laccase chimeras\n",
      "conclusion\n",
      "material and methods\n",
      "materials\n",
      "culture media\n",
      "protein modeling\n",
      "schema-raspp library design\n",
      "comparison of chimera assembly methods on schema disruption energy and mutation rate\n",
      "chimeric library creation in s. cerevisiae\n",
      "thermostability screening assay\n",
      "production of chimeras\n",
      "kinetic thermostability\n",
      "ph stability measurements\n",
      "activity assays with different substrates\n",
      "dna sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300074k/sb300074k.xml\n",
      "fuel types\n",
      "alcohol fuels\n",
      "fermentative short-chain alcohol fuels\n",
      "non-fermentative alcohol fuels\n",
      "fatty acid-derived fuels\n",
      "overproduction of biodiesels\n",
      "exploiting the β-oxidation pathway\n",
      "production of alkanes and alkenes\n",
      "isoprenoid-derived fuels\n",
      "strategies in strain development for biofuel production\n",
      "“omics” analysis and metabolic models\n",
      "overcoming toxicity\n",
      "modulating protein expression\n",
      "promoter engineering\n",
      "ribosomal binding site (rbs) engineering\n",
      "rna devices\n",
      "dynamic biosensor-regulators\n",
      "modulating protein activity\n",
      "protein engineering\n",
      "enhancing protein activity through colocalization\n",
      "conclusion and future directions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00172/sb7b00172.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "design constraints in dose–response curves of metabolite biosensors\n",
      "phenomenological theory for biosensor tuning\n",
      "orthogonal control of dynamic range and threshold\n",
      "discussion\n",
      "methods\n",
      "mathematical modeling and curve fitting\n",
      "construction and characterization of promoters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400032x/sb400032x.xml\n",
      "complex dna nanostructures from oligonucleotides\n",
      "programming membrane fusion and apoptosis\n",
      "photoswitching nucleic acid catalytic activity\n",
      "linking genotype and phenotype in protein synthesizing liposomes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00025/sb8b00025.xml\n",
      "materials and methods\n",
      "reagents and materials\n",
      "plasmid preparation and transformation\n",
      "conventional benchtop culture, induction, and expression\n",
      "microfluidic device fabrication\n",
      "automated induction microfluidics system (aims)\n",
      "microfluidic automated culture, induction, expression\n",
      "results and discussion\n",
      "characterization of the aims\n",
      "inducer concentration optimization: monitoring gene expression\n",
      "expression optimization: screening active bgl enzymes\n",
      "future work\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500366v/sb500366v.xml\n",
      "results and discussion\n",
      "definition of an assembly standard for yeast\n",
      "a toolkit of yeast parts\n",
      "characterization of promoters\n",
      "characterization of terminators\n",
      "protein degradation tags\n",
      "copy number, gene expression, and single-cell variability\n",
      "high-efficiency integrations into the chromosome\n",
      "multiplex, markerless genome editing using crispr/cas9\n",
      "summary\n",
      "methods\n",
      "strains and growth media\n",
      "yeast transformations\n",
      "golden gate assembly protocol\n",
      "cloning of parts\n",
      "promoter, terminator, and degradation tag characterization\n",
      "copy number characterization\n",
      "high-efficiency integrations\n",
      "multiplexed knockouts\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000433/sb4000433.xml\n",
      "methods\n",
      "plasmid construction\n",
      "quantitative characterization\n",
      "shake flask assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00101/sb9b00101.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00317/sb6b00317.xml\n",
      "highly efficient construction and expression of multicomponent protein complexes\n",
      "a cell–cell communication-based screening system for novel microbes\n",
      "optimum rebalancing of the 3-hydroxypropionic acid production pathway from glycerol\n",
      "construction and characterization of a synthetic microrna cluster for multiplex rnai in mammalian cells\n",
      "accelerated adaptation of cho cells to suspension culture\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00105/sb7b00105.xml\n",
      "results and discussion\n",
      "inhibition of nucleases using competitor molecules\n",
      "design of dna circuit components with increased nuclease resistance\n",
      "operation of simple dna strand-displacement circuits\n",
      "modeling dna strand-displacement circuits under nuclease-degrading conditions\n",
      "predicting and verifying the operation of complex dna strand-displacement circuits in medium\n",
      "conclusion\n",
      "materials and methods\n",
      "reagents\n",
      "preparation of dna complexes\n",
      "qpcr measurements of dna strand-displacement reactions in cell medium\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00112/sb8b00112.xml\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "generation of the combinatorial deletion library\n",
      "miniature chemostat assembly\n",
      "design of experiments and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00436/sb8b00436.xml\n",
      "results and discussion\n",
      "in vitro enzymatic reactions of various sugar acceptors by α1,2-fucosyltransferase\n",
      "identification of enzymatic reaction products by lc–ms–it–tof analysis\n",
      "production of 2′-fg by metabolically engineered e. coli\n",
      "identification of α1,2-glycosidic linkage of 2′-fg\n",
      "in vitro prebiotic activity of 2′-fg\n",
      "conclusions\n",
      "methods\n",
      "strains and plasmids\n",
      "expression of fuct2 and α1,2-fucosidase in e. coli bl21­(de3)\n",
      "production of agarobiose by acid hydrolysis of agarose\n",
      "in vitro enzymatic reactions of various sugar acceptors by fuct2\n",
      "analyses of reaction products using lc–ms–it–tof and hplc\n",
      "gc–tof–ms analysis\n",
      "2′-fg production by an engineered e. coli bl21­(de3)\n",
      "purification of 2′-fg by gel-permeation chromatography\n",
      "identification of α1,2-glycosidic linkage of 2′-fg by α1,2-fucosidase reaction\n",
      "stability test of 2′-fg on simulated gastric fluid\n",
      "growth test of probiotic and pathogenic bacteria under 2′-fg carbon source\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00461/sb7b00461.xml\n",
      "results\n",
      "t7 polymerase can transcribe functional sgrna in s. cerevisiae\n",
      "mutants of t7 polymerase enhance sgrna expression and improve dcas9 targeting\n",
      "guide rna expression and performance can be improved through guanine additions to the t7 promoter sequence\n",
      "t7-expressed sgrna can enable efficient cas9-mediated genome editing in s. cerevisiae\n",
      "t7-based sgrna expression enables genome editing in kluyveromyces lactis and yarrowia lipolytica\n",
      "discussion\n",
      "methods\n",
      "media and strains construction\n",
      "plasmid construction\n",
      "flow cytometry for dcas9-vpr activation\n",
      "quantitative pcr\n",
      "can1 mutagenesis in saccharomyces cerevisiae\n",
      "can1 editing in yarrowia lipolytica\n",
      "ndt80 editing in kluyveromyces lactis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500299c/sb500299c.xml\n",
      "materials and methods\n",
      "library design and selection\n",
      "in vivo gfp assay\n",
      "t7 rna polymerase purification\n",
      "i n vitro transcription assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00305/sb5b00305.xml\n",
      "results and discussion\n",
      "ak tolerance to circular permutation\n",
      "permuted protein expression variability\n",
      "mapping permuted sequences onto ak structure\n",
      "evaluating trends using rationally designed variants\n",
      "biophysical implications\n",
      "materials and methods\n",
      "permuteposon design\n",
      "constructing permute libraries\n",
      "bacterial complementation\n",
      "translation initiation calculations\n",
      "vectors for expressing permuted variants\n",
      "structural calculations\n",
      "mutational tolerance calculations\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00099/sb7b00099.xml\n",
      "results and discussion\n",
      "precursor acceptance of prodigiosin production strain\n",
      "construction of knockout strain and development of mutasynthesis\n",
      "in vivo exploration of substrate acceptance in mutasynthesis\n",
      "in vitro substrate acceptance of condensation enzyme pigc\n",
      "preparative scale production of prodiginines in mutasynthesis\n",
      "biological evaluation of prodiginines\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "dna manipulation equipment\n",
      "construction of p. putida pig-r2 δ pigd mutant strain\n",
      "mutasynthesis experiments\n",
      "analytical scale\n",
      "preparative scale\n",
      "plasmid construction for pigc expression\n",
      "expression of the pigc gene\n",
      "pigc in vitro assay\n",
      "quantification of prodiginines\n",
      "evaluation of biological activity of prodiginines\n",
      "antibodies\n",
      "western blotting\n",
      "confluence and apoptosis measurement using incucyte zoom\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300084h/sb300084h.xml\n",
      "results and discussion\n",
      "assembly of part models\n",
      "degenerate sequences\n",
      "fbda i: optimization scheme\n",
      "fbda ii: convergence\n",
      "computational realization\n",
      "application\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002906/sb5002906.xml\n",
      "pr–pr: cross-platform laboratory automation system\n",
      "r2odna designer: computational design of biologically neutral synthetic dna sequences\n",
      "heuristic for maximizing dna reuse in synthetic dna library assembly\n",
      "directed acyclic graph-based technology mapping\n",
      "optimal part and module selection for circuit design automation\n",
      "retropath: automated pipeline for embedded metabolic circuits\n",
      "computational method for automated characterization of genetic components\n",
      "negative autoregulation matches production and demand in synthetic transcriptional networks\n",
      "computational design of nucleic acid feedback control circuits\n",
      "load capacity improvements in nucleic acid based systems\n",
      "biological 2-input decoder circuit in human cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00225/sb5b00225.xml\n",
      "results\n",
      "the scope of detectable molecules is significantly expanded in silico via enzymatic pathways\n",
      "implementation of semps in vivo expands e. coli sensing abilities\n",
      "discussion\n",
      "methods\n",
      "data source\n",
      "molecular and reaction signatures\n",
      "compounds and reactions preprocessing\n",
      "semp prediction\n",
      "chemicals and reagents\n",
      "genetic constructs\n",
      "biosensor dose–response characterization\n",
      "characterization of the transcriptional response to nitroglycerin of different e. coli strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00062/sb8b00062.xml\n",
      "results and discussion\n",
      "design and validation of reporter construct\n",
      "effect of core promoter mutations on promoter activity\n",
      "effect of mutations in the rbs on translation initiation\n",
      "regulatory effect of full length mcrb 5′ utr\n",
      "effects of regulatory sequences surrounding the atph promoter\n",
      "methods\n",
      "strains and cell culture\n",
      "plasmid construction\n",
      "β-galactosidase assay\n",
      "rna extraction and reverse transcription\n",
      "qpcr\n",
      "promoter alignments\n",
      "rna structures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00177/sb6b00177.xml\n",
      "results and discussion\n",
      "construction of a series of tunable qs systems in response to different cell densities\n",
      "characterization of different cell density sensing systems in the and gate\n",
      "selection of a suitable stationary phase sensing system in response to the cell physiological state\n",
      "construction of an autoinduced and gate to dynamically control metabolic pathways\n",
      "an example for application of the autoinduced and gate\n",
      "conclusion\n",
      "methods and materials\n",
      "strains, plasmids, primers, and culture medium\n",
      "plasmid construction\n",
      "construction and selection of luxr mutant library\n",
      "quorum sensing system characterization\n",
      "and gate characterization\n",
      "polyhydroxybutyrate fermentation analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400117c/sb400117c.xml\n",
      "results and discussion\n",
      "mismatch repair knockout\n",
      "dna recombinase overexpression\n",
      "external factors affecting oligonucleotide recombination\n",
      "locus variability\n",
      "multiplex modifications and cycling\n",
      "future considerations\n",
      "methods\n",
      "strains and media\n",
      "strain modification\n",
      "gene cassette transformation\n",
      "oligonucleotide electroporation optimization protocol\n",
      "oligonucleotide electroporation cycling protocol\n",
      "oligonucleotide design\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00224/sb8b00224.xml\n",
      "results and discussion\n",
      "design strategy\n",
      "choice of building blocks\n",
      "biophysical characterization\n",
      "crystal structure of bric1\n",
      "nmr structure of bric1\n",
      "crystal structure of bric2\n",
      "architectural uniqueness and interface design precision\n",
      "conclusions\n",
      "materials and methods\n",
      "computational design\n",
      "expression and purification\n",
      "biophysical characterization\n",
      "x-ray crystallography\n",
      "nmr structure determination\n",
      "structural analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00203/sb6b00203.xml\n",
      "methods\n",
      "strains and cultivation\n",
      "molecular techniques\n",
      "extraction and analysis of trypacidin\n",
      "expression analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00154/sb6b00154.xml\n",
      "results and discussion\n",
      "construction of a cell-free platform for mevalonate production\n",
      "evaluation of enzyme variants for mevalonate production\n",
      "testing of enzyme ratios and guided construction of an all-in-one source strain\n",
      "optimization of cofactors\n",
      "source strain genome engineering to redirect metabolic flux: knockout of ldha , acs , and acka-pta\n",
      "evaluation of freeze-dried extracts for distributed biomanufacturing\n",
      "summary\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "cell growth and extract preparation\n",
      "cell free biosynthesis of mevalonate\n",
      "lyophilization of cfme reactions\n",
      "quantification of mevalonate, nad + , and organic acids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00164/sb6b00164.xml\n",
      "chris p. barnes\n",
      "ioana grigoras\n",
      "miriam leon\n",
      "ruben perez-carrasco\n",
      "nicholas roehner\n",
      "mae l. woods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00001/sb7b00001.xml\n",
      "results and discussion\n",
      "optimizing cho-based cell-free reaction parameters\n",
      "enhancing disulfide bond formation by tuning the oxidizing environment\n",
      "utilizing protein isomerases to encourage proper disulfide bond formation\n",
      "delayed addition of hc plasmid reduces aggregation products\n",
      "determining percent mab generated in optimized conditions\n",
      "ranking and functional analysis of known high and low in vivo mab expressors\n",
      "summary\n",
      "methods\n",
      "cell-free protein synthesis reaction\n",
      "isomerase purification\n",
      "quantification of active gfp\n",
      "protein gel and western blot analysis\n",
      "autoradiogram analysis\n",
      "purified mab control\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300129j/sb300129j.xml\n",
      "results and discussion\n",
      "strain design and static strategy\n",
      "ideal dynamic strategy\n",
      "dynamic strategy\n",
      "global sensitivity analysis\n",
      "effect of α c and γ c\n",
      "effect of α c and luxr\n",
      "effect of γ c and luxr\n",
      "summary of the effects of changing two parameters at a time\n",
      "effect of all three parameters\n",
      "preliminary design considerations\n",
      "conclusions\n",
      "methods\n",
      "dynamic flux balance analysis\n",
      "global sensitivity analysis\n",
      "analysis of the most sensitive parameters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00135/sb6b00135.xml\n",
      "results\n",
      "characterization and optimization of sensor-reporter performance\n",
      "yap1p can sense changes in nadph regeneration\n",
      "model-guided identification of gene targets to improve nadph availability\n",
      "construction of a novel sensor-selector combination\n",
      "discussion\n",
      "methods\n",
      "plasmids and strains\n",
      "media and cell cultures\n",
      "fluorescence measurements\n",
      "flow cytometry analysis\n",
      "cofactor quantification by calorimetric assay\n",
      "constraint-based metabolic modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00116/sb7b00116.xml\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "generation of mutant strains and constructs\n",
      "natural transformation/mugent assays\n",
      "alcian blue stained gels\n",
      "whole genome sequencing\n",
      "qualitative and quantitative assessment of phb production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00222/sb8b00222.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00342/sb6b00342.xml\n",
      "results\n",
      "individual expression of 30s and 50s r-proteins in batch format pure system\n",
      "individual expression of 30s and 50s r-proteins in ce format pure system\n",
      "coexpression of 30s and 50s r-proteins in batch and ce format pure system\n",
      "30s subunit reconstitution with pure system synthesized 30s r-proteins\n",
      "discussion\n",
      "methods\n",
      "media, chemicals, and reagents\n",
      "pure system reconstitution\n",
      "device fabrication\n",
      "optimized feeding solution\n",
      "molecular cloning\n",
      "individual expression of r-proteins in batch format pure system\n",
      "individual expression of r-proteins in ce format pure system\n",
      "coexpression of 30s and 50s r-proteins\n",
      "mass spectrometry sample preparation\n",
      "mass spectrometry data analysis\n",
      "isolation of tightly coupled 70s ribosomes and ribosomal subunits\n",
      "isolation of native rrna and r-proteins\n",
      "preparation of pure system synthesized 30s r-proteins\n",
      "integrated ribosome assembly, in vitro transcription and translation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00289/sb7b00289.xml\n",
      "results and discussion\n",
      "engineering ppsr2-repressible e. coli promoters\n",
      "engineering bphp1–ppsr2 to function as an e. coli photoreceptor\n",
      "transfer functions\n",
      "input/output dynamics\n",
      "action spectra\n",
      "conclusions\n",
      "methods\n",
      "plasmids, strains, and media\n",
      "optical hardware\n",
      "experimental protocols\n",
      "flow cytometry and data analysis\n",
      "model fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00320/sb6b00320.xml\n",
      "results and discussion\n",
      "autonomous plasmid replication mediated by heterologous repabc regions in s. meliloti\n",
      "pabc vector design\n",
      "parts characterization\n",
      "orivsm module\n",
      "synter-mcs module\n",
      "orivec and orit modules\n",
      "ar module\n",
      "characterization of pabc shuttle vectors\n",
      "repabc mediated in vivo cloning in s. meliloti (abc cloning)\n",
      "applicability of repabc- based plasmids in the rhizobiales\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "dna manipulation and plasmid extraction\n",
      "plasmid and strain construction\n",
      "dna transfer\n",
      "plasmid stability assays\n",
      "qpcr-based copy number determination\n",
      "microscopy\n",
      "bioinformatics tools\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000029/sb3000029.xml\n",
      "methods\n",
      "media, chemicals, and reagents\n",
      "strains and mage cycling protocol\n",
      "oligonucleotides (oligos)\n",
      "multiplex allele-specific colony pcr (masc-pcr)\n",
      "purification of his-tagged ribosomes\n",
      "purification of his-tagged factors from epure strains\n",
      "overexpression and purification of individual pure factors\n",
      "in vitro translation with pure and epure factors\n",
      "mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00122/sb6b00122.xml\n",
      "materials and methods\n",
      "chemicals, oligonucleotides and molecular biology\n",
      "strains\n",
      "media and culture conditions\n",
      "plasmids and plasmid construction\n",
      "library construction\n",
      "pathway assembly\n",
      "fluorescence measurements/evaluation of ssa methods\n",
      "lycopene assay\n",
      "statistical calculations\n",
      "results and discussion\n",
      "step 1: introducing sequence variability\n",
      "step 2: directed multigene pathway assembly\n",
      "workflow and parts validation: test case lycopene\n",
      "creation of individual gene libraries\n",
      "combinatorial construction of the lycopene pathway and assaying thereof\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500070c/sb500070c.xml\n",
      "results and discussion\n",
      "catf-mediated intracellular calcium sensing\n",
      "stability of catf-mediated reporter gene expression\n",
      "detection of egf-induced intracellular calcium increase\n",
      "engineering of the intracellular calcium-dependent transcription factor\n",
      "methods\n",
      "cell culture and transfection\n",
      "luciferase assay\n",
      "fluorescence microscopy\n",
      "ratiometric intracellular calcium measurement\n",
      "design of a calcium-dependent transcription factor\n",
      "gene sequence\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00397/sb8b00397.xml\n",
      "results and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00350/sb6b00350.xml\n",
      "results\n",
      "integration of the tetracycline-resistant “landing-pad” into 4 genomic loci\n",
      "integration of the fluorescent reporter protein mcherry into various genomic loci\n",
      "expression of chromosomally integrated mcherry\n",
      "multicopy expression levels of mcherry\n",
      "genomic integration of the violacein pathway\n",
      "vector construction and genomic integration of the tal pathway\n",
      "violacein production\n",
      "production of trans -cinnamic acid from cells containing an integrated copy of tal\n",
      "discussion\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, dna vectors, and media\n",
      "construction of epathbrick vectors containing multiple copies of genes\n",
      "construction of integration vectors\n",
      "integration of large constructs into the lacz , atpi-gidb , reca , and ybbd genomic loci on the mg1655­(de3) genome\n",
      "screening integration colonies\n",
      "mcherry expression measurements\n",
      "construction of violacein and tal pathway integration vectors\n",
      "sequence verification of the integrated violacein pathway\n",
      "violacein production\n",
      "trans -cinnamic acid production\n",
      "hplc analysis of violacein and trans -cinnamic acid production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00289/sb9b00289.xml\n",
      "results\n",
      "day-time production of fumarate\n",
      "fumarate production rate is proportional to growth rate\n",
      "genetic stability of fumarate production by the δfumc strain\n",
      "night-time production of fumarate\n",
      "conclusions\n",
      "methods\n",
      "strains\n",
      "plasmid and strain construction\n",
      "strains cultivated in multi-cultivator\n",
      "batch cultivation, day–night batch cultivation, prolonged cultivation, fumarate and lactate quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00026/sb6b00026.xml\n",
      "working with the web site\n",
      "dynamcc_0\n",
      "dynamcc_r\n",
      "codon exploder\n",
      "differences between the previous version to the current one\n",
      "discussion\n",
      "material and methods\n",
      "web interface design\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400162z/sb400162z.xml\n",
      "results and discussion\n",
      "simple mathematical model of sugar utilization\n",
      "experimentally probing l -arabinose utilization to explore model predictions\n",
      "constitutively expressing the high-affinity transporter more readily generates a uniform response\n",
      "trade-offs when deleting one transporter\n",
      "trade-offs when eliminating sugar catabolism\n",
      "breakdown of the sugar can help linearize the response\n",
      "similar trends when linearizing the response to d -xylose\n",
      "design rules to engineer sugar utilization pathways for titratable control\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "growth conditions and media\n",
      "flow cytometry analysis\n",
      "curve fitting to extract performance metrics\n",
      "mathematical modeling\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00257/sb6b00257.xml\n",
      "materials and methods\n",
      "strains and media\n",
      "cloning\n",
      "culture conditions\n",
      "crp-promoter characterization\n",
      "toggle switch stability trials\n",
      "phb production\n",
      "assays\n",
      "optical density and fluorescence\n",
      "glucose and organic acid assays\n",
      "phb analysis\n",
      "results and discussion\n",
      "selecting glucose responsive promoters for evaluation\n",
      "different glucose-responsive promoters have differing activatation characteristics\n",
      "select crp promoters enable toggle behavior\n",
      "the original genetic switch efficiently produces phb\n",
      "a glucose induced toggle switch enables autonomous phb production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300045j/sb300045j.xml\n",
      "results and discussion\n",
      "construction of synthetic promoters\n",
      "fidelity of promoter assembly\n",
      "e. coli library quality\n",
      "yeast library quality\n",
      "in vivo functionality of synthetic promoters\n",
      "methods\n",
      "synthetic promoter design\n",
      "promoter assembly and amplification\n",
      "plasmid assembly and cloning\n",
      "library transformation into yeast\n",
      "predicting pho4 affinity to engineered regulatory elements\n",
      "kinetic induction measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00251/sb6b00251.xml\n",
      "results and discussion\n",
      "a 2d-parallel circuit is a quantitative tool for simultaneously perturbing two cellular variables\n",
      "a 2d-series circuit is a versatile noise rheostat\n",
      "conclusion\n",
      "methods\n",
      "plasmids and strains\n",
      "media and growth conditions\n",
      "in parallel circuit automated flow cytometry measurements\n",
      "noise measurements and analysis\n",
      "gene expression, cell density and morphology measurements for noise rheostat-driven expression of sic1\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00162/sb7b00162.xml\n",
      "methods\n",
      "cell culture and transfection\n",
      "dna plasmid construction\n",
      "preparation of folate-conjugated, fluorescent-labeling microbubbles (fmb)\n",
      "setup of focused ultrasound sonication\n",
      "live cell imaging\n",
      "cytotoxicity measurements\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00174/sb9b00174.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna fragment preparation\n",
      "dna assembly and transformation\n",
      "colony pcr\n",
      "fluorescence measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00400/sb7b00400.xml\n",
      "introduction\n",
      "results and discussion\n",
      "selection of candidate genes for deletion via transcriptomic analysis\n",
      "crispr/cas9-based iterative deletion in e. coli\n",
      "a fluorescence detection method for evaluating extracellular expression systems\n",
      "construction and growth characteristics of single-gene deletion mutants\n",
      "extracellular expression systems in single-gene deletion mutants\n",
      "construction and growth characteristics of multigene deletion mutants\n",
      "extracellular expression systems in multigene deletion mutants\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "transcriptomic analysis\n",
      "plasmid construction\n",
      "gene deletion via crispr-cas9\n",
      "plasmid curing\n",
      "integrity assay of cell membranes\n",
      "cell viability assays of deletion mutants\n",
      "proteins expression and flash labeling assay\n",
      "real-time pcr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00524/sb8b00524.xml\n",
      "results\n",
      "a functional bistable switch was constructed using a positive feedback loop on a novel transcriptional activator\n",
      "a model describes the behavior of the switch and our controls\n",
      "the switch was used to drive antibiotic resistance to construct a growth differentiation circuit\n",
      "minimal changes to the bistable switch model describe the growth differentiation circuit\n",
      "discussion\n",
      "methods\n",
      "plasmid construction\n",
      "strain construction\n",
      "media preparation\n",
      "time-course cytometry assays\n",
      "flow cytometry\n",
      "time-course plate reader assays\n",
      "plate reader measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00180/sb8b00180.xml\n",
      "results and discussion\n",
      "design, optimization, and validation of the single-component light-switchable gene expression system\n",
      "characterization of the ylighton system\n",
      "optical control of cell growth and cell cycle\n",
      "spatiotemporal control of protein stability\n",
      "conclusion\n",
      "methods and materials\n",
      "molecular biology\n",
      "cell growth and blue light irradiation\n",
      "optical control of cell growth and cell cycle\n",
      "western blot\n",
      "imaging and flow cytometry\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400139h/sb400139h.xml\n",
      "results\n",
      "part and module library\n",
      "benchmarking and performance\n",
      "discussion\n",
      "methods\n",
      "definitions\n",
      "abstraction levels of a circuit specification\n",
      "circuit graph\n",
      "part and module library\n",
      "cost function\n",
      "part and module selection problem\n",
      "expansion from functional behaviors to biological processes\n",
      "dp-based algorithm for part and module selection\n",
      "algorithmic extensions for increased performance\n",
      "detect symmetric motif inputs to eliminate duplicated subsolutions\n",
      "utilize a connection look-ahead scheme to prune subsolution branches\n",
      "apply branch and bound to eliminate subsolutions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00264/sb5b00264.xml\n",
      "methods\n",
      "assembly of crispri constructs\n",
      "cyanobacteria transformation and culturing methods\n",
      "measurement of gfp in plate and flow cytometer\n",
      "measurement of phb by nile red staining\n",
      "measurement of glycogen\n",
      "quantification of mrna\n",
      "western blotting of dcas9\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00097/sb6b00097.xml\n",
      "results\n",
      "structure–activity relationship of alternate effector ligands\n",
      "direct determination of effector binding\n",
      "effect on rescue of both the substrate and the effector ligand\n",
      "separating effects of binding affinity from effects of active site geometry\n",
      "designing a better activating ligand for β-gly w33g\n",
      "designing activating ligands for β-gly w33a\n",
      "designing activating ligands for β-gly w33g_v37a\n",
      "discussion\n",
      "methods\n",
      "19 f nmr competition assay\n",
      "β-glycosidase enzyme assay\n",
      "rate equation\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00440/sb8b00440.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00118/sb7b00118.xml\n",
      "results\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00101/sb7b00101.xml\n",
      "results and discussion\n",
      "principle of the and operation using biological nanopores\n",
      "design of the dna for the and operation, using a thermodynamic simulation\n",
      "verification of the and operation by gel electrophoresis\n",
      "asymmetric solution conditions for nanopore measurement in a two-droplet system\n",
      "and operation with nanopore measurements in four operation devices\n",
      "conclusions\n",
      "methods\n",
      "reagents and chemicals\n",
      "design and formation of input and template dna\n",
      "preparation of the multichannel device\n",
      "gel electrophoresis and analysis of fluorescence intensity\n",
      "enzymatic reaction in droplets, preparation of bilayer lipid membranes, and reconstitution of α-hemolysin\n",
      "channel current measurements and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00228/sb8b00228.xml\n",
      "results and discussion\n",
      "effects of altering 483 kinetic parameters on mgg peptide synthesis\n",
      "robustness of mgg peptide synthesis\n",
      "reaction network analysis of mgg peptide synthesis\n",
      "analysis of the susceptible and robust reactions\n",
      "comparison with experimental results\n",
      "conclusion\n",
      "methods\n",
      "computational model\n",
      "simulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00252/sb8b00252.xml\n",
      "results and discussion\n",
      "identifying extract preparation conditions for v. natriegens cfps\n",
      "identification of optimal procedures for preparation of v. natriegens lysates\n",
      "optimization of reagent concentrations and reaction conditions in vibrio cfps\n",
      "assessing the capabilities of v. natriegens cfps\n",
      "increasing system productivity via genome engineering\n",
      "conclusions\n",
      "methods\n",
      "strains and plasmids\n",
      "cell culture\n",
      "extract preparation\n",
      "cfps reaction\n",
      "quantification of active sfgfp\n",
      "cfps lyophilization\n",
      "quantification of antimicrobial peptide yield in cfps\n",
      "generation of negative effector knockout library\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00029/sb7b00029.xml\n",
      "results and discussion\n",
      "design of the hob cycle\n",
      "selection for hob utilization as a c1-moiety source\n",
      "elucidation of the genetic basis for the resurgence of prototrophy\n",
      "conclusions\n",
      "methods\n",
      "chemicals, media and cultivation conditions\n",
      "construction of bacterial strains and plasmids\n",
      "continuous cultures\n",
      "turbidostat regime\n",
      "medium-swap regime\n",
      "bacterial growth assays\n",
      "biochemical assays\n",
      "alac enzyme characterization\n",
      "panb enzyme characterization\n",
      "modeling of panb bsu enzyme three-dimensional structure\n",
      "chemical synthesis of c 4 -[ 14 c]- d -homoserine\n",
      "chemical synthesis of radiolabeled potassium cyanate k 14 cno\n",
      "radio-labeling, extraction and digestion of total rna and genomic dna\n",
      "hplc analysis of radio-labeled nucleosides\n",
      "preparation and sequencing of genomic dna libraries\n",
      "mutation analysis\n",
      "bioinformatic analysis of alac mutations\n",
      "quantitative pcr analysis of panb gene expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400161v/sb400161v.xml\n",
      "problem solved?\n",
      "results and discussion\n",
      "computational hardness of bdmsp\n",
      "new greedy algorithm for dna library assembly\n",
      "computational validation\n",
      "efficacy evaluation\n",
      "equivalent-efficacy efficiency evaluation\n",
      "efficiency evaluation\n",
      "discussion\n",
      "methods\n",
      "running time measurements\n",
      "runtime complexity analysis using recurrences\n",
      "data set curation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00194/sb6b00194.xml\n",
      "results and discussion\n",
      "srta-mediated metabolic enzyme ligation in vivo\n",
      "the effect on acetate production by srta-mediated metabolic enzyme ligation\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction and gene disruption\n",
      "sds-page and western blotting analyses\n",
      "metabolite analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00077/sb6b00077.xml\n",
      "mark a. blenner\n",
      "christopher h. bowen\n",
      "olena r. bracho\n",
      "nicole r. ford\n",
      "karen a. hecht\n",
      "zhen kang\n",
      "cyril manchery\n",
      "mariko f. matsuura\n",
      "thomas j. mueller\n",
      "qi ouyang\n",
      "ryan w. shaw\n",
      "haoqian m. zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00418/sb8b00418.xml\n",
      "results and discussion\n",
      "scaffolding in chloroplast thylakoid membranes: curt1a fusion proteins retain native curt1a scaffold-forming properties\n",
      "expression and membrane integration\n",
      "ctp-curt_fluob oligomerization in thylakoid membranes\n",
      "scaffolding in higher plants without organelle targeting: curt1a c-terminal fusion proteins target the endoplasmic reticulum\n",
      "expression and membrane integration\n",
      "curt_fluob oligomerization in n. benthamiana er\n",
      "saccharomyces cerevisiae : curt1a c-terminal fusion proteins scaffold in clusters in the peripheral endoplasmic reticulum\n",
      "expression and membrane integration\n",
      "curt_fluob oligomerization in s. cerevisiae er\n",
      "escherichia coli : curt1a c-terminal fusion proteins scaffold in clusters in the inner membrane\n",
      "expression and membrane integration in e. coli\n",
      "inner membrane protein scaffolding in e. coli\n",
      "implications for designing membrane bound protein scaffolds\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00098/sb6b00098.xml\n",
      "nicholas j. bolten\n",
      "emmanuel de los santos\n",
      "jonathan garamella\n",
      "arjun khakhar\n",
      "ryan marshall\n",
      "jessica polka\n",
      "mark rustad\n",
      "cory schwartz\n",
      "sung sun yim\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000492/sb4000492.xml\n",
      "claudio angione\n",
      "changhao bi\n",
      "linh huynh\n",
      "hua jiang\n",
      "taek kang\n",
      "gregory linshiz\n",
      "dimitris papamichail\n",
      "sean poust\n",
      "guillermo rodrigo\n",
      "jason stevens\n",
      "athanasios tsoukalas\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00160/sb8b00160.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00139/sb8b00139.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00293/sb6b00293.xml\n",
      "results and discussion\n",
      "design of a single-transcript talen synthesis scheme\n",
      "one-pot assembly of talens\n",
      "single-transcript talen functionality in hek293t and hesc cells\n",
      "full automation of single-transcript talen assembly on a biological foundry\n",
      "high-throughput synthesis of 192 single-transcript talens\n",
      "methods\n",
      "ibiofab\n",
      "plasmids\n",
      "golden gate assembly and verification\n",
      "mammalian gene knockout and verification\n",
      "oct4 down-regulation assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000657/sb3000657.xml\n",
      "results and discussion\n",
      "functional elements and implementation of the trex system\n",
      "trex-mediated functional expression of a recombinant carotenoid biosynthesis pathway\n",
      "broad host range applicability of the trex system\n",
      "trex-mediated expression of the prodigiosin gene cluster\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "general dna manipulations\n",
      "implementation of trex for crt gene expression in p. putida\n",
      "implementation of trex for crt gene expression in r. capsulatus\n",
      "implementation of trex for crt gene expression in e. coli\n",
      "transcript analysis of crt genes using qpcr\n",
      "carotenoid analysis\n",
      "trex-dependent pig gene expression in p. putida\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00234/sb8b00234.xml\n",
      "results and discussion\n",
      "ligiv model of p. tricornutum\n",
      "dna ligation assay using recombinant ligiv_his\n",
      "generation and characterization of ligiv knock-down mutants\n",
      "integration of dna in wild type\n",
      "integration of hrl in ligiv knock-down lines\n",
      "integration of hrt in nas3 and nas4\n",
      "phenotypical analysis of tgl1 mutants\n",
      "outlook\n",
      "conclusion\n",
      "methods\n",
      "cell culture\n",
      "ligiv homologue in p. tricornutum\n",
      "structural model\n",
      "purification of recombinant ligiv_his and dna ligation assay\n",
      "isolation of nucleic acids\n",
      "isolation of genomic dna from p. tricornutum\n",
      "isolation of rna from p. tricornutum\n",
      "plasmid isolation from e. coli\n",
      "concentration of nucleic acids\n",
      "amplifications, restriction, cloning, and sequencing\n",
      "southern blot\n",
      "generation of ligiv knock-down mutants\n",
      "characterization of ligiv knock-down mutants\n",
      "construction and transformation of hr construct hrl\n",
      "plasmid dna recovery from p. tricornutum\n",
      "construction and transformation of hr construct hrt\n",
      "pcr analysis of tgl1 mutants\n",
      "phenotypical analysis of tgl1 mutants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00150/sb6b00150.xml\n",
      "methods and materials\n",
      "database sources and implementation\n",
      "chemgen integration\n",
      "input standardization\n",
      "network analysis\n",
      "statistical analysis\n",
      "results and discussion\n",
      "database summary\n",
      "classifying resistance using machine learning\n",
      "drivers of resistance and sensitivity\n",
      "adaptation via regulatory mutations\n",
      "walls, pumps, and locks: specific adaptation mechanisms\n",
      "growth versus resistance\n",
      "limitations of random mutation-selection\n",
      "detecting phenotype antagonism\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00361/sb6b00361.xml\n",
      "results\n",
      "surprising biphasic response of a genetic activation cascade\n",
      "a cascade model taking into account resource competition predicts nonregulatory interactions\n",
      "model-guided design recovers monotonically increasing response of the cascade\n",
      "general rules to draw effective interactions in genetic circuits\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00219/sb5b00219.xml\n",
      "methods\n",
      "oligo design\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "oligo efficiency prediction\n",
      "population diversity prediction from ares\n",
      "masc-pcr primer optimization\n",
      "masc-pcr primer pool generation\n",
      "validation methods\n",
      "infrastructure\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500233u/sb500233u.xml\n",
      "results and discussion\n",
      "two-morphogens turing model\n",
      "the dimensionless problem\n",
      "the dispersion relation\n",
      "parameter region for turing patterns\n",
      "turing patterns and cooperativity\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00052/sb5b00052.xml\n",
      "synthetic microbes as drug delivery systems\n",
      "engineering terpene biosynthesis in streptomyces for production of bisabolene\n",
      "programming controlled adhesion of e. coli\n",
      "characterizing and alleviating substrate limitations for improved in vitro ribosome construction\n",
      "analysis of rna containing artificial genetic components\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00146/sb8b00146.xml\n",
      "materials and methods\n",
      "strains and growth conditions\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "mammalian cell culture and inflammation model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500240p/sb500240p.xml\n",
      "methods\n",
      "molecular biology and microbiology\n",
      "syntheses of coa derivatives\n",
      "enzymology\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400082j/sb400082j.xml\n",
      "methods\n",
      "strains, plasmids, and culture conditions\n",
      "pcr amplification and cloning of nrps expression plasmids\n",
      "construction of an e. coli strain with genomically expressed sfp\n",
      "lc–ms analysis and quantitation of valinomycin\n",
      "atp-ppi exchange assay\n",
      "valinomycin susceptibility assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00203/sb9b00203.xml\n",
      "introduction\n",
      "results and discussion\n",
      "development of a versatile bstxi-based shotgun (bsg) cloning system for multiplex sirna expression in a single vector\n",
      "shotgun cloning of three sirnas targeting mouse β-catenin (simbc) using the bsg vector system\n",
      "silencing mouse β-catenin by pseb361-simbc effectively suppresses wnt3a-induced osteogenic differentiation of mesenchymal stem cells (mscs)\n",
      "silencing mmouse smad4 (simsmad4) using the bsg vector system effectively inhibits bmp9-induced osteogenic differentiation of mscs\n",
      "simultaneous silencing of β-catenin and smad4 (simbc/smad4) effectively diminishes wnt3a and bmp9-induced osteogenic differentiation of mscs in vitro and in vivo\n",
      "the bstxi-based modular sirna expression system provides a flexible and versatile platform for efficient multiplex gene silencing\n",
      "materials and methods\n",
      "cell culture and chemicals\n",
      "construction of bstxi-based shotgun (bsg) multiplex sirna expression system\n",
      "establishment of stable sirna expressing cell lines of mesenchymal stem cells imefs\n",
      "generation and amplification of recombinant adenoviruses ad-wnt3a, ad-bmp9, and ad-gfp\n",
      "total rna isolation and touchdown-qpcr (tqrcr) analysis\n",
      "bacterial colony pcr screening assay\n",
      "alkaline phosphatase (alp) assays\n",
      "transfection and gaussia and firefly luciferase reporter assays\n",
      "subcutaneous stem cell implantation and ectopic bone formation\n",
      "microcomputed tomographic (μct) analysis\n",
      "histologic staining and masson’s trichrome staining\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00115/sb6b00115.xml\n",
      "results\n",
      "expression of the acetone carboxylase\n",
      "expression of candidate alcohol dehydrogenases\n",
      "purification and in vitro activity determination of the recombinant acx complex\n",
      "growth of engineered strains on isopropanol\n",
      "further analysis of pathway limitations\n",
      "purification of recombinant acx from cells grown in supplemented minimal medium\n",
      "discussion\n",
      "materials and methods\n",
      "commercial materials\n",
      "bacterial strains and genomic dna template\n",
      "plasmid construction\n",
      "acx next-generation plasmids\n",
      "protein expression studies\n",
      "alcohol dehydrogenase expression for lysate analysis\n",
      "acetone carboxylase expression for lysate activity analysis\n",
      "acetone carboxylase expression comparison\n",
      "acetone carboxylase expression and purification for activity studies\n",
      "isopropanol growth studies\n",
      "growth studies\n",
      "hplc analysis of growth medium\n",
      "solubility analysis of acxnt\n",
      "hplc analysis of acx reaction with acetone\n",
      "spectrophotometric activity assays\n",
      "alcohol dehydrogenase activity in isopropanol oxidation\n",
      "acetone carboxylase kinetic activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00122/sb5b00122.xml\n",
      "hannes beyer\n",
      "jef boeke\n",
      "kathleen a. curran\n",
      "dawn eriksen\n",
      "kathrin herbst\n",
      "victor holmes\n",
      "samuel juillot\n",
      "cyrus modavi\n",
      "christopher reeves\n",
      "thomas steele reynolds\n",
      "elaine shapland\n",
      "david j. sukovich\n",
      "yun yang\n",
      "matias zurbriggen\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00038/sb5b00038.xml\n",
      "results and discussion\n",
      "design of a one plasmid-based crispr-cas9 system\n",
      "actinorhodin biosynthesis was successfully interrupted by targeting genes within the pathway with the crispr-cas9 system\n",
      "random sized deletions caused by imperfect nhej in actinomycetes\n",
      "optimizing the intrinsic nhej pathway by coexpression of a ligd ligase\n",
      "precise gene deletion in actinomycetes using hdr\n",
      "an effective and reversible crispri system for gene expression controls in s. coelicolor\n",
      "discussion\n",
      "methods\n",
      "strains, plasmids, media and growth condition\n",
      "construction of the crispr-cas9 vectors\n",
      "construction of homologous recombination templates\n",
      "dna manipulation\n",
      "transfer of the plasmids from e. coli to s. coelicolor by conjugation\n",
      "genomic dna extraction, miseq library construction, sequencing and sequence analysis\n",
      "colony pcr for actinomycetes\n",
      "detection of actinorhodin\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002455/sb5002455.xml\n",
      "optogenetic control of protein kinase activity\n",
      "a genetically encoded sender–receiver system\n",
      "engineering monoterpene production in yeast\n",
      "optimization of a yeast rnai system for controlling gene expression\n",
      "microrna-based single-gene circuits buffer protein synthesis rates against perturbations\n",
      "generation of novel chemical structures as scaffolds for drug discovery\n",
      "a heat-inducible gene expression system\n",
      "using a stochastic bistable switch to coordinate responses of autonomous bacteria\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300057c/sb300057c.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00314/sb9b00314.xml\n",
      "results and discussion\n",
      "efficient production of the key precursor gdp- l -fucose via the salvage pathway\n",
      "production of 2′-fl by introducing fucosyltransferases\n",
      "improving lactose utilization by introducing heterologous lactose permeases and knocking-out yesz\n",
      "enhancing 2′-fl and gdp- l -fucose production by engineering the cofactor gtp regeneration module\n",
      "production of 2′-fucosyllactose by b. subtilis in a 3 l bioreactor\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and culture conditions\n",
      "sequence and statistical analysis\n",
      "plasmid construction and genome manipulation\n",
      "analytical methods\n",
      "fucosyltransferases enzymatic assay\n",
      "identification of gdp- l -fucose and 2′-fl by lc/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00314/sb6b00314.xml\n",
      "guo-qiang (george) chen\n",
      "huiqing chong\n",
      "ichiro hirao\n",
      "maciej b. holowko\n",
      "premkumar jayaraman\n",
      "haseong kim\n",
      "michiko kimoto\n",
      "teng li\n",
      "tian li\n",
      "yuya miyatake\n",
      "itaru okamoto\n",
      "chueh loo poh\n",
      "yiran qin\n",
      "kazuya saeki\n",
      "wonjae seong\n",
      "chan woo song\n",
      "masahiro tominaga\n",
      "huijuan wang\n",
      "tingting wang\n",
      "hui wu\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00449/sb8b00449.xml\n",
      "concluding remarks\n",
      "methods\n",
      "strains used, cultured conditions, preparation methods\n",
      "tem protocol\n",
      "creation of deletion strains glpo , glpk , glpf\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00291/sb8b00291.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00466/sb8b00466.xml\n",
      "results and discussion\n",
      "generating a soluble rtxa variant with sole aminotransferase activity\n",
      "creating 2-apd producing strains with engineered rtxa variants in e. coli\n",
      "optimization of amino donor supplementation for enhancing 2-apd production\n",
      "identification of a promiscuous phosphatase for the efficient conversion of 2-apd phosphate to 2-apd\n",
      "producing 2-apd in a fed-batch fermentation\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "bacterial strains and plasmids\n",
      "dna manipulations\n",
      "culture conditions\n",
      "protein expression and purification\n",
      "n-terminal sequencing\n",
      "enzyme activity assays\n",
      "analytical procedures\n",
      "computational methods for predicting domain boundary and structure of rtxa\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00246/sb7b00246.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "cell culture\n",
      "establishment of stable cell lines\n",
      "reporter gene activation\n",
      "endogenous gene activation\n",
      "real-time reverse transcription pcr\n",
      "adipogenic induction using gene activation\n",
      "cell staining\n",
      "quantification and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300013k/sb300013k.xml\n",
      "visualization of gene silencing\n",
      "increasing target-peptide affinity\n",
      "synthetic inhibition of protein expression\n",
      "a photo-activated inducer of cell death\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00037/sb7b00037.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "conclusions\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00192/sb7b00192.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500229s/sb500229s.xml\n",
      "results and discussion\n",
      "vector construction and assembly\n",
      "functional component assembly\n",
      "construction and characterization of a standard promoter set\n",
      "concluding remarks\n",
      "methods\n",
      "strains, transformations, and growth conditions\n",
      "vector and functional unit assembly; restriction site removal\n",
      "microtiter plate reader assays\n",
      "microscopy and microbetracker analysis\n",
      "mrna extraction, reverse transcription, and qpcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00304/sb6b00304.xml\n",
      "what are bioblocks?\n",
      "customization of blocks\n",
      "bioblocks output\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00126/sb9b00126.xml\n",
      "results and discussion\n",
      "development of a rapid and high-throughput screening method based on the detection of blue coloration\n",
      "directed evolution of dszc based on random mutation\n",
      "further saturation mutagenesis for the directed evolution of dszc\n",
      "further enhancement of the desulfurization efficiency using a combination of desensitization engineering and the overexpression of dszc\n",
      "conclusions\n",
      "methods\n",
      "media and culture conditions\n",
      "construction of plasmids and strains\n",
      "protein expression and purification\n",
      "error-prone pcr and construction of the e. coli mutant library\n",
      "high-throughput screening method and resting cell assays\n",
      "discovery studio 3.0\n",
      "dszc enzyme assays\n",
      "iterative saturation mutagenesis for dszc\n",
      "analytical methods\n",
      "lc-ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200013j/sb200013j.xml\n",
      "results and discussion\n",
      "initial attempts to clone a whole a. laidlawii genome in yeast\n",
      "cloning the a. laidlawii genome in fragments\n",
      "identification of a. laidlawii toxic gene\n",
      "demonstration that a a. laidlawii endonuclease gene is also toxic in escherichia coli\n",
      "demonstration that the a. laidlawii endonuclease gene must be expressed to cause toxicity in yeast\n",
      "new strategy to clone the a. laidlawii apai fragment 1\n",
      "other strategies to clone the whole a. laidlawii genome in yeast\n",
      "summary and conclusions\n",
      "methods\n",
      "strains used and culture conditions\n",
      "genomic dna isolation\n",
      "yeast transformation/colony selection\n",
      "pcr analysis of clones\n",
      "sizing of a. laidlawii genomic fragments or full genomes cloned in yeast\n",
      "preparation of ysvs\n",
      "preparation of yrbvs\n",
      "deletion analysis of the endonuclease gene\n",
      "expression of the toxic region of the a. laidlawii endonuclease gene under galactose induction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00464/sb8b00464.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "reagents\n",
      "yeast strains, media, and cultivation\n",
      "plasmid and genomic dna extraction from yeast\n",
      "growth conditions and determination of growth rate\n",
      "yeast transformation\n",
      "plasmid construction\n",
      "carbon labeling\n",
      "determination of formate concentration in media\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00018/sb8b00018.xml\n",
      "results and discussion\n",
      "construction of the hybrid teto/lacogal4 synthetic arrays by rca-tar cloning\n",
      "hac formation using the hybrid α-satellite array in human ht1080 cells\n",
      "cenp-a overexpression significantly increases the efficiency of hac formation\n",
      "the alphoid hybrid -hac contains both alphoid dna arrays\n",
      "the alphoid hybrid -hac segregates accurately during cell division\n",
      "structural characterization of the alphoid hybrid -hac shows contiguous tandem copies of the input dna\n",
      "epigenetic engineering showed a functional two-domain kinetochore in alphoid hybrid -hac that can be manipulated with different targeting systems\n",
      "conclusions\n",
      "materials and methods\n",
      "construction of α21-i-teto and α21-ii-lacogal4 alphoid arrays by rolling-circle amplification (rca) followed by transformation-associated recombination (tar) in yeast\n",
      "construction of the hybrid teto-cenpb+/lacogal4-cenpb– array by recombinational cloning in yeast\n",
      "southern blot hybridization analysis\n",
      "cell culture, transfections, and hac formation\n",
      "fluorescent in situ hybridization (fish)\n",
      "indirect immunofluorescence staining and microscopy\n",
      "chromatin immunoprecipitation and quantitative pcr (chip-qpcr)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300040v/sb300040v.xml\n",
      "the model system: t7\n",
      "results\n",
      "molecular evolution of design elements\n",
      "fitness evolution\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "growth of phage for adaptation\n",
      "fitness assays\n",
      "sequences\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400030p/sb400030p.xml\n",
      "results and discussion\n",
      "rod-shaped e. coli form fractal domains\n",
      "computational modeling of biophysics of rod-shaped bacterial populations\n",
      "marking and measurement of lineage domains within colonies\n",
      "effect of cell-shape on domain boundary geometry\n",
      "methods\n",
      "microbial strains and growth conditions\n",
      "plasmids\n",
      "microscopy\n",
      "image analysis\n",
      "plate fluorometry\n",
      "computational modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00241/sb8b00241.xml\n",
      "results and discussion\n",
      "methods and materials\n",
      "protein design and production\n",
      "protein modeling\n",
      "protein refolding and filament assembly\n",
      "imaging and quantification of filaments\n",
      "circular dichroism\n",
      "fret filament assembly assay\n",
      "chaperone assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00204/sb9b00204.xml\n",
      "results and discussion\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "dna constructs\n",
      "txtl reactions\n",
      "bulk expression assay\n",
      "liposome preparation\n",
      "imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00303/sb5b00303.xml\n",
      "results and discussion\n",
      "pathway optimization strategy\n",
      "pathway amplification by strong promoter and 5′ utrs\n",
      "optimization of 3-hp production pathway by fine-tuning the expression of glycerol dehydratase\n",
      "fed-batch cultivation of the engineered strain\n",
      "discussion\n",
      "methods\n",
      "reagents, plasmids and strains\n",
      "media and culture conditions\n",
      "analysis of glycerol, 3-hp, and other metabolites\n",
      "preparation of crude cell lysate\n",
      "sds-page for expression analysis\n",
      "enzyme assay for glycerol dehydratase and aldehyde dehydrogenase\n",
      "quantification of the amount of dhab1 transcript\n",
      "bioreactor study\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00125/sb6b00125.xml\n",
      "results and discussion\n",
      "promoter target site identification\n",
      "test for activity: mscv 5′ltr and cmv promoters\n",
      "external control for off-target effects\n",
      "rnaa internal control platform\n",
      "qpcr validation of platform\n",
      "significance and future work\n",
      "materials and methods\n",
      "calculations and software\n",
      "plasmid cloning\n",
      "lentiviral production and infection\n",
      "sarna transfections and cell growth\n",
      "flow cytometry and cell sorting\n",
      "rna isolation and qpcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00304/sb8b00304.xml\n",
      "preliminaries\n",
      "probability representation with molecules\n",
      "simulation model\n",
      "ldpc code\n",
      "message-passing algorithm\n",
      "molecular ldpc decoder\n",
      "rationale\n",
      "decoding schedule\n",
      "q ij initialization\n",
      "r ji computing\n",
      "problem analysis\n",
      "our strategy\n",
      "q ij computing\n",
      "problem analysis\n",
      "our strategy\n",
      "y i computing\n",
      "the whole crn system\n",
      "simulation results\n",
      "case study one: an (8, 4) molecular ldpc decoder\n",
      "case study two: a (20, 4) molecular ldpc decoder\n",
      "complexity analysis\n",
      "conclusion and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00337/sb6b00337.xml\n",
      "results and discussion\n",
      "a toolkit for the high-throughput construction of p. pastoris expression vectors\n",
      "characterization of promoter strengths\n",
      "transformation of pichia pastoris\n",
      "design of synthetic secretion signal peptides\n",
      "characterization of secretion signal peptides\n",
      "effect of methanol on protein secretion\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, media and growth conditions\n",
      "plasmids and library construction\n",
      "transformation\n",
      "characterization\n",
      "data analysis and plotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500332c/sb500332c.xml\n",
      "results and discussion\n",
      "proof-of-concept of metabolic flux control by riboswitches in e. coli\n",
      "functionality examination of riboswitches in c. glutamicum\n",
      "exploring riboswitches for metabolic pathway control in c. glutamicum\n",
      "fermentation with the mutants for enhanced lysine production\n",
      "methods\n",
      "bacterial strains, plasmids, media, and growth conditions\n",
      "dna manipulation\n",
      "construction of riboswitch- lacz fusions\n",
      "construction of chromosomal riboswitch- glta mutant in e. coli and c. glutamicum\n",
      "fermentation\n",
      "analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00151/sb8b00151.xml\n",
      "results and discussion\n",
      "cloning strategy and dipac of the hapalosin biosynthetic gene cluster\n",
      "heterologous expression of the hapalosin gene cluster\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and genomic dna extraction\n",
      "bioinformatic analysis and pcr\n",
      "dipac strategy and construction of expression vectors\n",
      "heterologous expression of the hapalosin gene cluster\n",
      "extraction of expression cultures and lc–ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00263/sb6b00263.xml\n",
      "results and discussion\n",
      "promoter and 5′ utr target selections based on multiomic data\n",
      "construction and validation of promoters and 5′ utrs\n",
      "replacements of 5′ utr sequences from the previously reported promoters, p sco5768 , p sco4658 , and p sco3410\n",
      "applications of promoter and 5′ utr combinations for actinorhodin productions\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, plasmids, and culture conditions\n",
      "strain construction\n",
      "rna-seq, ribo-seq, and tss-seq data\n",
      "gusa activity measurements of the promoter/5′ utr combinations\n",
      "actinorhodin measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb2000066/sb2000066.xml\n",
      "methods\n",
      "strains\n",
      "genetic engineering\n",
      "cultivations and luminescence determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00204/sb7b00204.xml\n",
      "methods\n",
      "experiment description terms (edd ontology)\n",
      "key capabilities\n",
      "data input\n",
      "visualization\n",
      "data standardization\n",
      "data output\n",
      "read/edit permissions\n",
      "implementation\n",
      "templates and views\n",
      "front-end and visualization\n",
      "database\n",
      "importers and exporters\n",
      "services\n",
      "results and discussion\n",
      "using metabolite concentration data to derive internal metabolic fluxes through flux balance analysis (fba)\n",
      "using targeted proteomics data to improve biofuel production through principal component analysis (pcap)\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00220/sb6b00220.xml\n",
      "results and discussion\n",
      "multicellular feedback control design\n",
      "model derivation\n",
      "controller population\n",
      "target population\n",
      "communication between the two populations\n",
      "multicellular feedback control objectives and simulation scenarios\n",
      "aggregate populations scenario\n",
      "control results\n",
      "robustness analysis\n",
      "modularity\n",
      "agent-based scenario\n",
      "control results\n",
      "robustness analysis\n",
      "modularity\n",
      "conclusions\n",
      "methods\n",
      "model parametrization\n",
      "aggregate populations scenario\n",
      "simulations\n",
      "modularity analysis\n",
      "agent-based scenario\n",
      "simulations without cell growth\n",
      "simulations with cell growth\n",
      "grns and signaling\n",
      "modularity analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00306/sb7b00306.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "methods\n",
      "preparation of protocells\n",
      "lipid vesicles\n",
      "proteinosomes\n",
      "encapsulation of genetic machinery within lipid vesicles\n",
      "determination of activity of esar switch\n",
      "cell-free gene expression of gfp-tagged α-hemolysin within lipid vesicles\n",
      "glucose-based signaling from lipid vesicles\n",
      "lipid vesicle to proteinosome chemical signaling cascade\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00302/sb7b00302.xml\n",
      "results\n",
      "a ratiometric quantification method for dd characterization\n",
      "a double dd architecture reduces the background levels and improves the fold-induction\n",
      "the least-background dd architecture tightly regulates a lethal phenotype in drosophila\n",
      "discussion\n",
      "methods\n",
      "dna constructs\n",
      "cell culture, transfection and antibiotic treatment\n",
      "drug treatment and flow cytometry\n",
      "western blot and quantitative analysis\n",
      "transgenesis\n",
      "protein induction in flies\n",
      "imaging and quantitative analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00214/sb8b00214.xml\n",
      "results and discussion\n",
      "the master regulator of the anthocyanin pathway is inactive when it is translationally fused to selected proteins or protein fragments\n",
      "niapro can control the activity of rosea1 at the post-translational level\n",
      "genetic circuits able of basic boolean computation in plants with a visual output signal\n",
      "perspectives on synthetic genetic circuits deployed in plant tissues through agrobacteria\n",
      "materials and methods\n",
      "plasmid construction\n",
      "plant inoculation for viral infection\n",
      "analysis of viral infection by rt-pcr\n",
      "plant inoculation for transient expression of synthetic circuits\n",
      "analysis of anthocyanins\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00107/sb5b00107.xml\n",
      "results\n",
      "construction of p t7/ beto hybrid promoter\n",
      "effect of plasmid (over-)­expression of beti on behavior of p t7/ beto\n",
      "screening for choline-responsive mutants\n",
      "choline-inducible beti mutants: bet-on system\n",
      "choline-suppressed beti mutants (beti rev ): bet-off system\n",
      "on the difference between p t7/ beto with p t7/ laco\n",
      "overexpression of recombinant proteins using bet-on/-off systems\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains, media, and chemicals\n",
      "plasmids construction\n",
      "screening plasmid\n",
      "reporter plasmids\n",
      "beti-expressing plasmid.\n",
      "library construction\n",
      "screening of gfp expressing e. coli colonies\n",
      "characterization of beti mutants\n",
      "analysis of cell growth rate\n",
      "determination of the expression level of sfgfp\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00087/sb8b00087.xml\n",
      "results and discussion\n",
      "expression and functionality of cpf1 in c. difficile\n",
      "achieve genome editing in c. difficile through inducible expression of cpf1\n",
      "application of crispr-cpf1 for the deletion of larger gene fragments\n",
      "application of crispr-cpf1 for the deletion of extremely large gene fragment\n",
      "application of crispr-cpf1 for multiplex genome editing in a single step\n",
      "application by crispr-cpf1 for multigene editing in consecutive steps\n",
      "characterization of the antibiotic susceptibility in the relevant mutants\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmids construction\n",
      "a python-based algorithm to design primers for one-step-assembly (osa)\n",
      "conjugation of c. difficile and mutant screening\n",
      "plasmid curing\n",
      "antibiotics susceptibility assay\n",
      "statistical analysis\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00069/sb7b00069.xml\n",
      "results and discussion\n",
      "construction of a genetically encoded ammonium biosensor using glnra from lactococcus lactis\n",
      "characterization and optimization of the ammonium biosensor in e. coli\n",
      "responses of the ammonium glnr2–1 biosensor to various nitrogen sources\n",
      "testing the ammonium biosensor in pseudomonas and synechocystis\n",
      "using the ammonium biosensor to control cyanophycin biosynthesis\n",
      "methods\n",
      "materials, reagents, and medium\n",
      "plasmid construction\n",
      "cell culture and fluorescence assay\n",
      "cyanophycin production and measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00256/sb6b00256.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00235/sb9b00235.xml\n",
      "results and discussion\n",
      "synthesis of modified curli fibers\n",
      "characterization of curli fiber-based nanomaterials\n",
      "material precursor-inducible whole-cell sensor systems\n",
      "production of functionalized curli nanofibers upon induction by material precursors\n",
      "conclusions\n",
      "materials and methods\n",
      "genetic designs and cloning of fusion peptide constructs\n",
      "congo red staining\n",
      "crystal violet staining\n",
      "material synthesis by modified curli fibers\n",
      "microscopy imaging\n",
      "induction of precursor-induced material sensitive systems\n",
      "cross-reactivity of precursor ions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00372/sb6b00372.xml\n",
      "results\n",
      "orf26 enzymology\n",
      "valerolactam and caprolactam production in vivo\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "orf26 protein expression and purification\n",
      "atp-ppi release assays for orf26\n",
      "4-guanidinobutyryl coa product identification\n",
      "product analysis of in vitro orf26 lactam formation\n",
      "in vivo lactam production\n",
      "in vivo valerolactam biosynthesis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00276/sb8b00276.xml\n",
      "results\n",
      "design of the experimental space\n",
      "improving isat protein yield through iterated edoe\n",
      "dependence of response on individual components\n",
      "composition of isat reaction conditions achieving highest protein expression and their kinetic profiles\n",
      "variation of magnesium and dna templates concentrations\n",
      "high-dimensional intercomponent synergy\n",
      "discussion\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "s150 cell extract and component preparation\n",
      "isat reaction setup\n",
      "robotic liquid-handling reaction setup and exploration of the experimental space using pivots\n",
      "gfp quantification\n",
      "rna quantification\n",
      "modeling and design of experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00229/sb7b00229.xml\n",
      "results and discussion\n",
      "redesigning central carbon metabolism in m. extorquens am1: engineering a heterologous glyoxylate shunt to drain intermediates from the emcp\n",
      "the heterologous glyoxylate shunt enzymes are active in m. extorquens am1\n",
      "the heterologous glyoxylate shunt does not restore methylotrophic growth of m. extorquens am1 emcp disruption strains\n",
      "key enzymes in methanol oxidation are downregulated in the engineered strains\n",
      "the heterologous glyoxylate shunt enables wt-like growth of m. extorquens am1 emcp disruption strains on acetate\n",
      "specific production of crotonic acid in glyoxylate shunt-engineered m. extorquens am1 from acetate\n",
      "conclusions\n",
      "material and methods\n",
      "strains, media and growth conditions\n",
      "chemicals\n",
      "construction of plasmids\n",
      "construction of emcp disruption strains\n",
      "spectrophotometric enzyme assays of isocitrate lyase and malate synthase\n",
      "spectrophotometric enzyme assays of methanol dehydrogenase and nad-linked formate dehydrogenase\n",
      "growth assays in liquid media\n",
      "crotonic acid production and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00106/sb7b00106.xml\n",
      "methods\n",
      "construction of (nnk) 2–6 vcsm13-pviii randomized libraries\n",
      "preparation of bacteriophages\n",
      "preparation of phagemid dna\n",
      "enzyme-linked immunosorbent assay (elisa)\n",
      "synthesis of gold nanocubes\n",
      "biopanning\n",
      "scanning electron microscopy\n",
      "surface-enhanced raman spectroscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00140/sb8b00140.xml\n",
      "results\n",
      "stable mcls generated using crispr/cas9 and a preselected target locus\n",
      "isogenic subclones expressing recombinant proteins show similar growth and low transcriptional variation\n",
      "isogenic subclones show high stability in growth and transcriptome during three months of cultivation\n",
      "epo expression levels were doubled when expressed from a composite promoter\n",
      "increase in epo expression leads to large transcriptional changes in isogenic cell lines\n",
      "large transcriptional variation was observed between subclones derived from different mcls\n",
      "discussion\n",
      "methods\n",
      "plasmids and plasmid construction\n",
      "generation of rmce mcls using crispr/cas9\n",
      "fluorescent level analysis\n",
      "pcr amplification of target regions\n",
      "quantitative real-time pcr (qrt-pcr) for copy number analysis\n",
      "quantitative real-time pcr (qrt-pcr) for relative rna expression levels analysis\n",
      "rmce subclone generation\n",
      "batch cultivation\n",
      "long-term cultivation\n",
      "rna extraction for rna-seq\n",
      "library preparation and rna-seq\n",
      "rna-seq analysis and differential gene expression analysis\n",
      "canonical pathway and network analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300101g/sb300101g.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "small molecule antibiotics\n",
      "secondary metabolic pathways\n",
      "type i pks assembly: erythromycin\n",
      "type ii pks assembly: oxytetracycline\n",
      "synthetic biology: tools of the trade\n",
      "antimicrobial peptides\n",
      "nonribosomal peptides\n",
      "antimicrobial peptides\n",
      "non-traditional therapeutics\n",
      "phage engineering\n",
      "other non-traditional therapeutics\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00248/sb7b00248.xml\n",
      "methods\n",
      "plasmid dna preparation\n",
      "cell culture and transfection\n",
      "live cell fluorescent imaging\n",
      "statistic analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00460/sb7b00460.xml\n",
      "results and discussion\n",
      "vesicle formation and osmotic deflation\n",
      "shape transformation induced by peg\n",
      "morphological analysis\n",
      "depletion volume effect\n",
      "shape transformation induced by protein\n",
      "shape transformation induced by dna\n",
      "shape transformation induced by internally amplified dna\n",
      "discussion\n",
      "methods\n",
      "formation of peg- and bsa-containing guvs\n",
      "formation of λdna-containing guv\n",
      "formation of guv containing wga reaction mixture\n",
      "experimental procedure\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00192/sb5b00192.xml\n",
      "results and discussion\n",
      "characterization of the toxin and attraction modules\n",
      "combining the toxin and attraction module results in an unexpected decrease in intoxication efficiency\n",
      "mathematical modeling reveals that subsequent activation of our circuit modules can increase intoxication efficacy\n",
      "methods\n",
      "strains and growth conditions\n",
      "intoxication assays\n",
      "microscopy\n",
      "attraction assays\n",
      "learning assay\n",
      "feeding rate analysis\n",
      "mathematical modeling\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00415/sb8b00415.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid design and cloning\n",
      "protein concentration measurements\n",
      "labeling and determination of degree of labeling\n",
      "cleaning of coverslips\n",
      "assay chamber assembly\n",
      "lipid preparation\n",
      "slb formation\n",
      "pdms microstructures\n",
      "microscopy\n",
      "image manipulations (fiji)\n",
      "spatiotemporal regulation of peripheral and lipid-anchored proteins by minde\n",
      "atpase assay, nadh coupled\n",
      "bacterial transformation and agar pads for microscopy, imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002483/sb5002483.xml\n",
      "methods\n",
      "dna constructs\n",
      "luciferase reporter assay\n",
      "confocal microscopy\n",
      "quantification of ifn-β1 levels\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00029/sb6b00029.xml\n",
      "introduction\n",
      "results and discussion\n",
      "description\n",
      "implementation\n",
      "measurements\n",
      "implementation\n",
      "single-cell measurements\n",
      "spatial measurements\n",
      "case study\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "sbol–seva description\n",
      "mathematical modeling and sbml-to-python conversion\n",
      "two-dimensional in vivo setup\n",
      "two-dimensional in silico setup\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00369/sb8b00369.xml\n",
      "results\n",
      "transcriptional activators switch from specific short-range to long-range activation during differentiation\n",
      "formation of an activity gradient around transcriptional activators\n",
      "both activators and repressors have asymmetric upstream and downstream effects\n",
      "the transcriptional activity gradient is mirrored by decreased cpg methylation\n",
      "discussion\n",
      "methods\n",
      "construction of the cells containing the tale-r and the expression cascade (rtta and tale-a-α11)\n",
      "sequence alignment\n",
      "quantification of gene expression\n",
      "memory experiment\n",
      "titration of tale-a-α11 expression\n",
      "models of the activity gradient flanking an activator\n",
      "quantification of the variance of downstream and upstream genes by activators or repressors\n",
      "oxidative bisulfite sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00396/sb7b00396.xml\n",
      "results and discussion\n",
      "construction of the hsa-amp fusion protein expressing p. pastoris strain\n",
      "expression and purification of apidaecin amp in shake flasks\n",
      "antibacterial assays\n",
      "scale-up production of biologically active amp apidaecin in bioreactors\n",
      "materials and methods\n",
      "construct design and p. pastoris strain construction\n",
      "protein expression and production in shake flasks and sds page analysis\n",
      "peptide purification\n",
      "bacterial killing assays\n",
      "p. pastoris fermentation\n",
      "cell growth stage\n",
      "protein production stage\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00171/sb9b00171.xml\n",
      "results and discussion\n",
      "candidate gene screening and phylogenetic analysis\n",
      "substrates screening of the o -glycosyltransferases\n",
      "glycosylation of liquiritigenin ( 1 ) and 7,4′-dihydroxyflavone ( 2 )\n",
      "glycosylation of isoliquiritigenin ( 3 )\n",
      "glycosylation of kaempferol ( 4 )\n",
      "glycosylation of formononetin ( 5 ) and daidzein ( 6 )\n",
      "glycosylation of glycyrrhetinic acid ( 7 ) and glycyrrhizic acid ( 8 )\n",
      "substrate promiscuity and regiospecificity of gugt26\n",
      "substrate promiscuity of gugt10 and gugt14\n",
      "potential biosynthetic roles of gugts in licorice\n",
      "potential roles of gugts as biocatalysts\n",
      "conclusion\n",
      "methods\n",
      "chemicals\n",
      "plant materials\n",
      "transcriptome analysis and phylogenetic analysis\n",
      "molecular cloning, heterologous expression, and protein purification\n",
      "in vitro enzyme catalytic activity assay\n",
      "hplc and lc/ms analysis\n",
      "preparative-scale purification of glycosylated products\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00250/sb8b00250.xml\n",
      "results and discussion\n",
      "blue light-dependent self-assembly based on photoswitchable proteins\n",
      "reversibility of blue light-dependent self-assembly\n",
      "self-assembly and disassembly kinetics\n",
      "light-dependent self-assembly is based on specific heterodimerization\n",
      "stoichiometry controlled self-assembly\n",
      "social self-sorting in a four component mixture\n",
      "conclusions\n",
      "experimental methods\n",
      "plasmids\n",
      "protein expression and purification\n",
      "protein immobilization on the beads\n",
      "bead aggregation assay\n",
      "light sources and light intensity dependence\n",
      "imaging\n",
      "data analysis\n",
      "stoichiometry experiments\n",
      "social self-sorting experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00391/sb7b00391.xml\n",
      "methods\n",
      "strains and growth conditions\n",
      "ade2 , ura3 , and can1 targeting using gap repaired grna\n",
      "the design of grna\n",
      "insertion of wickets\n",
      "preassembled integration\n",
      "cocktail integration method\n",
      "determining the copy number of β-carotene genes\n",
      "quantification of carotenoid yield by hplc\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500242x/sb500242x.xml\n",
      "methods\n",
      "molecular biology\n",
      "biochemistry\n",
      "computational modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00407/sb8b00407.xml\n",
      "data sources\n",
      "database content\n",
      "annotation of sequences\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300016b/sb300016b.xml\n",
      "results and discussion\n",
      "design and characterization of epathbrick vectors\n",
      "tunable gene expression with expandable regulatory elements\n",
      "multigene pathway can be assembled in three configurations\n",
      "combinatorial generation of pathway diversities\n",
      "modular assembly of multigene pathways on a single epathbrick vector\n",
      "summary and conclusions\n",
      "methods\n",
      "epathbrick vectors construction\n",
      "reporter plasmid construction\n",
      "assay of gene expression\n",
      "flavanone pathway assembly\n",
      "flavanone fermentation and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00298/sb5b00298.xml\n",
      "materials and methods\n",
      "mutagenic pcr\n",
      "estimation of transformation efficiency\n",
      "site directed mutagenesis\n",
      "parts characterization with sfgfp\n",
      "plasmid copy number estimation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00383/sb8b00383.xml\n",
      "methods\n",
      "general\n",
      "lycopene quantification\n",
      "in vitro fgapt2 assay\n",
      "in vivo fgapt2 assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500221v/sb500221v.xml\n",
      "results and discussion\n",
      "construction of methylobrick vectors for the convenient assembly of genetic parts\n",
      "identification and molecular characterization of a basic methylobrick promoter set\n",
      "quantification of methylobrick promoter activities and intrapopulation variability in different growth phases on the single-cell level\n",
      "methylobrick promoter-driven expression of crotonyl-coa carboxylase/reductase\n",
      "expression levels and application of polycistronic, synthetic operons\n",
      "genomic integration and expression of methylobrick constructs\n",
      "application of brick vectors and tn5 delivery in other alphaproteobacteria\n",
      "conclusions and outlook\n",
      "materials and methods\n",
      "bacterial strains, culture conditions, and plasmid delivery\n",
      "plasmid construction\n",
      "two-parental mating for mini-tn5 transposon delivery\n",
      "fluorescence microscopy and image analysis\n",
      "5′ rapid amplification of cdna ends (5′ race)\n",
      "preparation of cell extracts and enzyme activity assays of ccr, gcl, garr, and glxk\n",
      "quantification of organic acids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00343/sb6b00343.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00261/sb9b00261.xml\n",
      "results\n",
      "discussion\n",
      "materials and methods\n",
      "strains and chemicals\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00157/sb7b00157.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "reagents\n",
      "culture conditions\n",
      "plasmid and strain construction\n",
      "transformation of 7002\n",
      "acetoin quantification\n",
      "quantification of 23bd\n",
      "enzyme assays\n",
      "o 2 evolution\n",
      "chlorophyll content\n",
      "glycerol quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00078/sb9b00078.xml\n",
      "results and discussion\n",
      "refactoring the fapr operon for an efficient prototype malonyl-coa biosensor\n",
      "in vivo optimization of the prototype malonyl-coa biosensor\n",
      "fapr-based detection of malonyl-coa across various e. coli strains\n",
      "probing the effector promiscuity of fapr by cell-free transcription–translation\n",
      "fapr-based detection of methylmalonyl-coa in vivo\n",
      "discussion\n",
      "materials and methods\n",
      "general\n",
      "construction of plasmids\n",
      "rbs library construction and screening\n",
      "expression and purification of mutant matb\n",
      "synthesis of acyl-coas by matb\n",
      "synthesis of mmsnac\n",
      "cell-free characterization of fapr promiscuity\n",
      "fapr-bioassay of methylmalonyl-coa production in e. coli k207-3\n",
      "purification of fapr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000436/sb3000436.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "alphoid teto -hac contains a ∼1.1 mb alphoid dna array that is stable through hac transfer into different cells\n",
      "alphoid dna array in the alphoid teto -hac has an irregular structure\n",
      "alphoid teto -hac consists of direct and inverted alphoid dna repeats\n",
      "integrity of functional domains in the alphoid teto -hac after multiple rounds of mmct\n",
      "summary and conclusions\n",
      "methods\n",
      "cell cultures\n",
      "southern blot hybridization analysis\n",
      "microcell-mediated chromosome transfer\n",
      "pcr analysis\n",
      "fish with pna probes\n",
      "dna fiber-fish by dna combing\n",
      "rescue of the hac fragments by in vivo recombination in yeast\n",
      "characterization of the tar-rescued molecules\n",
      "chip assay and real-time pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00131/sb7b00131.xml\n",
      "results\n",
      "assembly of the integrase/excisionase system\n",
      "system modeling and simulation\n",
      "introduction of excisionase reduces the system’s expression level and output variation\n",
      "cell-to-cell noise is reduced by the synthetic feedback loop\n",
      "downstream process is easily tuned to a predictable threshold\n",
      "discussion\n",
      "methods\n",
      "plasmids\n",
      "cell culture and experimental conditions\n",
      "measurement and data analysis\n",
      "mathematical modeling\n",
      "simulation setup and parameters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00243/sb8b00243.xml\n",
      "results and discussion\n",
      "piace strategy\n",
      "rfp and gfp chromosomal integration and expansion\n",
      "stability of rfp and gfp copy numbers post expansion\n",
      "isopentenol pathway integration\n",
      "machine learning for the prediction of isopentenol titers from gene copy numbers\n",
      "stability of isopentenol titers and goi1–3 copy numbers across expression configurations\n",
      "discussion\n",
      "methods\n",
      "materials\n",
      "dna design/visualization/protocol generation software\n",
      "plasmid and strain construction and availability\n",
      "chromosomal integration, gene copy number expansion, and curation of helper plasmids\n",
      "growth conditions and isopentenol production\n",
      "fluorescence and gene copy number measurement\n",
      "stability tests\n",
      "machine learning\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00469/sb8b00469.xml\n",
      "methods\n",
      "dna and strain construction\n",
      "codon optimization of sfgfp\n",
      "media and experimental protocols\n",
      "flow cytometry analysis\n",
      "statistical analysis\n",
      "growth model fitting\n",
      "transfer function fitting\n",
      "kinetic model fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300012g/sb300012g.xml\n",
      "factors controlling the function of regulatory signaling networks\n",
      "evolution of cellular regulatory networks architecture: lessons from synthetic biology\n",
      "evolution of network architecture through rewiring of protein–protein interactions\n",
      "network architecture rewiring by replacement of specificity-determining amino acids often proceeds through intermediates with expanded specificities\n",
      "short linear docking motifs provide fast routes for network architecture rewiring\n",
      "recombination of modular domains enables network rewiring without promiscuous intermediates\n",
      "domain recombination can evolve novel signal processing capabilities\n",
      "domain recombination can alter network function by changing the time or the subcellular localization of an interaction\n",
      "scaffold proteins are highly evolvable network hubs\n",
      "evolution of network architecture through rewiring of transcriptional connections\n",
      "combinatorial recombination of cres and orfs leads to transcriptional networks with diverse behaviors\n",
      "novel phenotypes can evolve by rewiring of master transcriptional regulators\n",
      "parameter tuning by base substitutions could be needed for proper function of a rewired network\n",
      "synthetic biology approaches to investigate signaling network evolution at the genome-wide level\n",
      "final remarks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00258/sb8b00258.xml\n",
      "results and discussion\n",
      "enhancing metabolic flux of the mk-7 pathway\n",
      "engineering sa pathway to improve the supply of chorismate\n",
      "engineering mep pathway to improve the supply of heptaprenyl-pp\n",
      "engineering glycerol metabolism pathway to increase glycerol utilization\n",
      "reducing the consumption of pyruvate, chorismate, and isochorismate\n",
      "fermentation in 2 l baffled flasks\n",
      "conclusions\n",
      "materials and methods\n",
      "microorganisms, plasmids, and cultivation conditions\n",
      "dna manipulation techniques\n",
      "construction of the starting strain bs168nu\n",
      "in situ replacement of the promoter in the b. subtilis chromosome\n",
      "gene overexpression\n",
      "gene knockout\n",
      "mk-7 extraction and hplc analysis\n",
      "quantitative rt-pcr analysis\n",
      "microbial fermentation in 2 l flasks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00145/sb8b00145.xml\n",
      "results and discussion\n",
      "construction of the reporter system and verification of grnas\n",
      "creation of ert2-cas9 fusion proteins\n",
      "substitution of the 4-hydroxytamoxifen-sensitive ligand binding domain with a mifepristone-sensitive one\n",
      "adding drug control to the alternative crispr effector, cpf1\n",
      "targeting an endogenous gene\n",
      "conclusions\n",
      "materials and methods\n",
      "plasmid construction\n",
      "ptrex-mcherry\n",
      "fusion proteins\n",
      "grna-carrying plasmids\n",
      "mammalian cell culture and transfection\n",
      "mcherry imaging/fluorescence microscopy\n",
      "flow cytometric analysis\n",
      "t7 endonuclease i assay\n",
      "next generation sequencing (ngs)\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300079h/sb300079h.xml\n",
      "results and discussion\n",
      "engineering novel synthetic medea elements\n",
      "medea dah and medea o‑fut1 show maternal-effect lethal and zygotic rescue behavior\n",
      "medea dah and medea o‑fut1 drive population replacement\n",
      "molecular characterization of medea -dependent gene silencing\n",
      "using medea to bring about environmental cue-dependent population suppression\n",
      "methods\n",
      "construction of mirnas\n",
      "construction of the antidotes and final medea constructs\n",
      "transgenesis and population cage experiments\n",
      "embryo and adult viability determination\n",
      "total rna isolation\n",
      "small rna extraction, cloning, and sequencing\n",
      "poly(a+) read alignment and quantification\n",
      "small rna read alignment and quantification\n",
      "fitness cost calculation\n",
      "modeling a medea -dependent population crash\n",
      "population frequency model\n",
      "discrete population model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00120/sb5b00120.xml\n",
      "methods\n",
      "genome sequencing and bioinformatics analyses\n",
      "synthetic dkp standards\n",
      "preparation of constructs for expression of noza and ncda in e. coli\n",
      "evaluation of metabolite production by e. coli expressing noza or ncda\n",
      "site-directed mutagenesis of noza and ncda\n",
      "purification of noza\n",
      "purification of ncda\n",
      "characterization of noza and ncda function and substrate specificity in vitro\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00267/sb9b00267.xml\n",
      "results and discussion\n",
      "design of phytochrome production construct\n",
      "production and characterization of phyb*\n",
      "production and characterization of cph1*\n",
      "conclusion\n",
      "methods\n",
      "synthesis of expression constructs\n",
      "protein production by fermentation\n",
      "protein production in flask culture\n",
      "protein purification\n",
      "protein analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00075/sb9b00075.xml\n",
      "results\n",
      "riboswitches demonstrate theophylline dependency in clostridium\n",
      "riboswitches can be adjusted to the desired regulatory window\n",
      "gene expression and transcript abundance\n",
      "functional characterization of theophylline-riboswitches in clostridium\n",
      "ribocas: a highly efficient crispr/cas9-based tool for clostridium\n",
      "application of ribocas for the deletion and integration of larger fragments\n",
      "ribocas demonstrates universality in clostridium\n",
      "discussion\n",
      "materials and methods\n",
      "strains, media, and growth conditions\n",
      "reagents\n",
      "plasmid design, construction, and transformation\n",
      "growth and cat activity assays\n",
      "hplc–ms analysis\n",
      "conjugation/transformation efficiency determination and mutant screening\n",
      "dna sequencing and bioinformatics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00157/sb8b00157.xml\n",
      "results\n",
      "synthesis of mtg variants with 3-chlorotyrosines in place of tyrosines\n",
      "evaluation of the thermostability of the chlorinated mtg variants\n",
      "incorporation of an α-hydroxy amino acid to achieve the autonomous release of the n-terminal inhibitory peptide from the mtg pro-enzyme\n",
      "engineering of a thermostabilized automaturing mtg variant\n",
      "discussion\n",
      "methods\n",
      "non-natural amino acids\n",
      "biosynthesis of halogenated mtg\n",
      "biosynthesis of mtg with two different synthetic components\n",
      "mass spectrometry\n",
      "engineering and analysis of alockohrs\n",
      "alkaline cleavage of the n-terminal peptide from the mtg pro-enzyme\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00165/sb7b00165.xml\n",
      "results and discussion\n",
      "thermal stabilities of dsdna containing cognate and noncognate ubps in various sequence contexts\n",
      "ds base-mediated thermal stability control of dsdnas\n",
      "noncognate natural base pair recognition by neighboring g–ds pair in dsdna\n",
      "conclusion\n",
      "methods\n",
      "oligonucleotides\n",
      "measurement of melting curves\n",
      "determination of thermodynamic parameters\n",
      "prediction of melting temperatures\n",
      "pcr using ds-containing and fluorescently labeled primers\n",
      "real-time pcr using ds-containing primers\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00116/sb6b00116.xml\n",
      "methods\n",
      "reaction components\n",
      "chemiluminescence assay\n",
      "mass spectrometry assay\n",
      "data processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500090b/sb500090b.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "function and optimization of the detection system in lab\n",
      "detection of e. faecalis in coculture\n",
      "ccf10-inducible bacteriocin production by l. lactis\n",
      "coculture detection and inhibition\n",
      "methods\n",
      "bacterial strains, growth conditions, and plasmids\n",
      "construction of plasmids and l. lactis transformation\n",
      "β-galactosidase assays\n",
      "expression of lacz by l. lactis in coculture with e. faecalis\n",
      "antimicrobial activity assays\n",
      "e. faecalis growth-inhibition in coculture with recombinant l. lactis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00237/sb7b00237.xml\n",
      "results\n",
      "mga fluorescence is enhanced by aptamer scaffolds\n",
      "robust mga fluorescence can be seen in the presence of fluorescent proteins\n",
      "robust mga fluorescence can be seen in the presence of other aptamers\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "fluorescence measurement using plate reader\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00059/sb7b00059.xml\n",
      "results\n",
      "designs for stabilized broccoli aptamers and the split-broccoli system\n",
      "split-broccoli assembles in vitro to reconstitute fluorescence activation\n",
      "transcription terminator structures enhance activation ratio\n",
      "split-broccoli assembly in vitro approximately follows second-order kinetics\n",
      "cotranscriptional assembly of split-broccoli improves signal\n",
      "split-broccoli functions in vivo\n",
      "split-broccoli is functionally modular and can be used to monitor rna–rna assembly in vivo\n",
      "discussion\n",
      "methods\n",
      "broccoli designs and nupack analysis\n",
      "dna templates and rna transcription\n",
      "in vitro assembly and fluorescence assays\n",
      "dual staining native gel shift assay\n",
      "split-broccoli assembly kinetics\n",
      "cotranscription fluorescence assay\n",
      "in vivo fluorescence\n",
      "split-broccoli toehold switch assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00084/sb8b00084.xml\n",
      "results\n",
      "identification of cryptic antimicrobial peptides via computational analysis\n",
      "recombinant production of pepsinogen a3-derived peptides\n",
      "human pepsinogen a3-derived peptides exhibit antimicrobial properties\n",
      "direct killing activity of (p)­pap-a3\n",
      "effect of pap-a3 and its fragments on biofilms\n",
      "hemolytic activity and cytotoxicity\n",
      "in vivo activity of pap-a3 and its fragments\n",
      "discussion\n",
      "experimental procedures\n",
      "recombinant vectors\n",
      "protein expression and purification of recombinant peptides\n",
      "mic assays\n",
      "time-kill assays\n",
      "biofilm cultivation in static conditions and confocal microscopy\n",
      "molecular docking\n",
      "hemolytic assays\n",
      "cytotoxicity toward mammalian cells\n",
      "scarification skin infection mouse model\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00202/sb6b00202.xml\n",
      "results and discussion\n",
      "identification of an in vivo reporter for streptomyces\n",
      "development of a set of orthogonal integration vectors for streptomyces\n",
      "analysis of natural and synthetic promoters at multiple attb loci\n",
      "combinatorial improvement of bisabolene production using chromosomally integrated genes\n",
      "conclusion\n",
      "materials and methods\n",
      "measurement of s. venezuelae fluorescence\n",
      "generation of vectors expressing fluorescent proteins\n",
      "exchange of promoters in the integrating vectors\n",
      "conjugal transfer of vectors to s. venezuelae\n",
      "small scale culture and analysis of bisabolene production\n",
      "increased scale culture and analysis of bisabolene production\n",
      "s. venezuelae microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00452/sb8b00452.xml\n",
      "results\n",
      "development of artificial caprolactam riboswitch through coupled in vitro – in vivo selection\n",
      "characterization of caprolactam riboswitch\n",
      "caprolactam-dependent control of cell growth using the caprolactam riboswitch\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, plasmids, and reagents\n",
      "in vitro selection of rna aptamers for caprolactam\n",
      "in vivo selection of caprolactam riboswitch\n",
      "characterization of caprolactam riboswitch\n",
      "intracellular metabolite extraction\n",
      "intracellular metabolite sample preparation and derivatization\n",
      "gc-ms analysis of intracellular metabolites\n",
      "tetracycline resistance assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300077b/sb300077b.xml\n",
      "iván lópez-montero\n",
      "tomoaki matsuura\n",
      "lara hernández moleiro\n",
      "jonghyeon shin\n",
      "ming wang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400128g/sb400128g.xml\n",
      "results and discussion\n",
      "characterization of mesa design space using model receptors\n",
      "mesa performance optimization through design-based tuning\n",
      "ligand-inducible mesa signaling\n",
      "additional mesa output modality: enzyme reconstitution\n",
      "ligand-inducible enzyme reconstitution\n",
      "conclusions and future prospects\n",
      "methods\n",
      "dna constructs\n",
      "cell culture and transfection\n",
      "flow cytometry\n",
      "immunolabeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00145/sb5b00145.xml\n",
      "claudio angione\n",
      "harish chandran\n",
      "neil dalchau\n",
      "chelsea hu\n",
      "linh huynh\n",
      "benjamin jack\n",
      "sean leonard\n",
      "jens nielsen\n",
      "john reif\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00252/sb6b00252.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400042a/sb400042a.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00160/sb6b00160.xml\n",
      "results and discussion\n",
      "methods\n",
      "preparation of s30 cell extracts\n",
      "cell-free protein synthesis (cfps) reactions\n",
      "fluorescence labeling of grsa and grsb1\n",
      "dkp production in vitro\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00161/sb5b00161.xml\n",
      "results\n",
      "establishing a malonyl-coa sensor in s. cerevisiae\n",
      "increasing the malonyl-coa sensor range\n",
      "sensitivity assessment of the malonyl-coa sensor\n",
      "metabolic pathway control for 3-hp production\n",
      "discussion\n",
      "methods\n",
      "media and culture conditions\n",
      "plasmid construction\n",
      "strain construction\n",
      "characterization of malonyl-coa sensor in s. cerevisiae\n",
      "characterization of malonyl-coa sensor for 3-hp production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00138/sb6b00138.xml\n",
      "results and discussion\n",
      "ncaa incorporation\n",
      "ncaa incorporation efficiency into laci is position-dependent\n",
      "protein truncation is more pronounced in the presence of rf1\n",
      "ncaa-independent read-through of the uag codon is increased in recoded e. coli\n",
      "dye labeling\n",
      "cell membrane affects the efficiency of intracellular protein labeling\n",
      "dye labeling does not interfere with the tetramerization of laci\n",
      "presence of additional fluorescently labeled proteins\n",
      "presence of endogenous tag stop codons does not increase spurious dye incorporation\n",
      "extensive nonspecific dye retention precludes a microscopy analysis of the labeled laci in cells\n",
      "search for the factors contributing to the nonspecific fluorescence incorporation\n",
      "labeling of extracellular proteins\n",
      "fluorescently labeled ompc can be specifically detected in live cells\n",
      "dye-labeled ompc is localized on the membrane\n",
      "dye labeling preserves ompc folding\n",
      "optimizing tco*-ak washout\n",
      "ncaa incorporation at endogenous tag codons is negligible\n",
      "protein expression from the tag-containing ompc construct is toxic in bw25993\n",
      "an ompc fluorescent protein fusion is toxic to the cells\n",
      "ompc diffusion is restricted in e.coli outer membrane\n",
      "conclusions\n",
      "methods\n",
      "optical setup\n",
      "plasmids and strains\n",
      "chemicals\n",
      "western blot analysis\n",
      "ncaa incorporation analysis into laci\n",
      "in-gel fluorescence analysis of laci and ompc labeling\n",
      "laci tetramerization analysis\n",
      "fluorescence labeling of laci for microscopy\n",
      "fluorescence labeling of ompc for microscopy\n",
      "fluorescence imaging of dye treated cells on agarose pads\n",
      "sample preparation and setup for ompc imaging in microfluidic chips\n",
      "cell geometry determination and analysis of fluorescence images of cells in microfluidic devices\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00253/sb5b00253.xml\n",
      "results and discussion\n",
      "synthesis of modified dpxtps\n",
      "assessment methods for the amplification efficiency and fidelity of ds–px pairing in pcr\n",
      "amplification efficiency and selectivity of ds–px pairing\n",
      "misincorporation rate of the unnatural base substrates opposite the natural bases\n",
      "gel shift assay of amplified dnas containing biotin-px\n",
      "copper-free click reaction of the amplified dnas using ds–n 3 -px pair\n",
      "methods\n",
      "general information for chemical synthesis\n",
      "1-(2-deoxy-β- d -ribofuranosyl)-4-(1,7-octadiyn-1-yl)-2-nitropyrrole ( 2 )\n",
      "1-(2-deoxy-β- d -ribofuranosyl)-4-(1,7-octadiyn-1-yl)-2-nitropyrrole 5′-triphosphate ( 3 )\n",
      "1-[2-deoxy-3,5- o -bis­( tert -butyldimethylsilyl)-β- d -ribofuranosyl]-4-iodo-2-nitropyrrole ( 4 )\n",
      "1-[2-deoxy-3,5- o -bis­( tert -butyldimethylsilyl)-β- d -ribofuranosyl]-4-(4-hydroxypentyn-1-yl)-2-nitropyrrole ( 5 )\n",
      "1-[2-deoxy-3,5- o -bis­( tert -butyldimethylsilyl)-β- d -ribofuranosyl]-4-(4-azidopentyn-1-yl)-2-nitropyrrole ( 6 )\n",
      "1-(2-deoxy-β- d -ribofuranosyl)-4-(4-azidopentyn-1-yl)-2-nitropyrrole ( 7 )\n",
      "1-(2-deoxy-β- d -ribofuranosyl)-4-(4-azidopentyn-1-yl)-2-nitropyrrole 5′-triphosphate ( 8 )\n",
      "biotin-dpxtp ( 10 )\n",
      "general information for biological experiments\n",
      "pcr involving ds–px pairing\n",
      "dna sequencing\n",
      "analysis of misincorporation rates of unnatural bases into natural bases\n",
      "analysis of the biotinylated 75-mer amplicons\n",
      "post modification of the amplicons containing n 3 -px moiety by copper-free click reaction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500243c/sb500243c.xml\n",
      "results and discussion\n",
      "cloning of functional β-oxidation enzymes expressed in the cytosol of s. cerevisiae\n",
      "n -butanol biosynthesis as proof-of-concept\n",
      "medium-chain fatty acids (mcfas) production\n",
      "increased production of medium-chain fatty acid ethyl esters (mcfaees)\n",
      "conclusions\n",
      "methods\n",
      "strains, media, and cultivation conditions\n",
      "dna manipulation\n",
      "enzyme activity assays\n",
      "n -butanol fermentation and detection\n",
      "mcfas fermentation and detection\n",
      "mcfaees fermentation and detection\n",
      "prediction of the mitochondrial and peroxisomal targeting sequences\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00229/sb5b00229.xml\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00109/sb7b00109.xml\n",
      "materials and methods\n",
      "plasmids and strains\n",
      "reagents and media\n",
      "96-well fluorescence assays\n",
      "dynamical fluorescence quantification assays\n",
      "statistical analysis\n",
      "mathematical model\n",
      "in silico experiments of multicellular control strategy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00319/sb8b00319.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00188/sb7b00188.xml\n",
      "results\n",
      "biofuel production model\n",
      "the control problem\n",
      "proposed integral controller\n",
      "stability, tracking, and perfect adaptation\n",
      "stability of the equilibrium point\n",
      "tracking and perfect adaptation\n",
      "maximizing the biofuel production while limiting the metabolic load\n",
      "productivity and toxicity measures\n",
      "a productivity–toxicity conservation law\n",
      "optimal biofuel production\n",
      "metabolic cost\n",
      "robustness results\n",
      "robustness and maximum worst-case equilibrium productivity\n",
      "sensitivity analysis with respect to set-point implement errors\n",
      "a simple robust suboptimal set-point\n",
      "comparison with static (open-loop) control\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00172/sb9b00172.xml\n",
      "material and methods\n",
      "oligonucleotides, plasmids, cell lines, and transfection conditions\n",
      "quantification of hsa-mir373 expression\n",
      "analysis of fkis and f′kid18 expression\n",
      "analysis of cell proliferation and cell death\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500176h/sb500176h.xml\n",
      "sbol version 1.1\n",
      "results and discussion\n",
      "structural representation\n",
      "functional representation\n",
      "composition of structure and function\n",
      "examples\n",
      "rna replicons\n",
      "crispr cascade\n",
      "discussion\n",
      "methods\n",
      "identified, documented, and collection\n",
      "components\n",
      "structural instantiation\n",
      "modules and module instantiations\n",
      "ports and port maps\n",
      "interactions\n",
      "models\n",
      "summary of proposed data model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00255/sb7b00255.xml\n",
      "results\n",
      "reconfigurable inversion of gene expression in a minimal synthetic network\n",
      "mathematical description of the gene network\n",
      "modeling-guided perturbation of the gene network\n",
      "experimental validation of the model predictions\n",
      "dynamical and noise characteristics of synthetic transfer-function inversion\n",
      "generalizing reconfigurability as a function of network topology\n",
      "transfer-function inversion topologies in natural networks\n",
      "discussion\n",
      "methods\n",
      "experimental model and subject details\n",
      "characterization of biochemical circuits in vivo\n",
      "dynamic transfer-function inversion\n",
      "characterizing fold-change of induction and repression\n",
      "expression noise measurement\n",
      "method details\n",
      "plasmid construction and expression control\n",
      "magnitude analysis\n",
      "generality analysis\n",
      "quantification and statistical analysis\n",
      "classification of transfer functions\n",
      "motif incidence and significance\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00045/sb9b00045.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00150/sb7b00150.xml\n",
      "results and discussion\n",
      "solution properties of gatckx\n",
      "structure of the host–guest complex\n",
      "helical properties of duplex dna containing x : k pairs\n",
      "local properties of the x : k nucleobase pair\n",
      "the bigger picture. why terran dna uses the standard nucleobase pairs\n",
      "conclusion\n",
      "methods\n",
      "synthesis and purification of kx containing oligonucleotides\n",
      "circular dichroism analyses\n",
      "crystallization and data collection\n",
      "quantum mechanical calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00227/sb6b00227.xml\n",
      "sc2.0 consortium updates\n",
      "mammalian synthetic biology\n",
      "network of foundries\n",
      "hgp-write\n",
      "panel discussion: sc2.0 in context: postcards from a different species\n",
      "summary and conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00388/sb8b00388.xml\n",
      "results and discussion\n",
      "construction of shinorine biosynthetic pathway in s. cerevisiae\n",
      "generation of shinorine-producing s. cerevisiae strains by random multicopy delta-integration of the biosynthetic genes\n",
      "using xylose as a carbon source for shinorine production by introducing xylose assimilation pathway\n",
      "construction of xylose-fermenting yeast strains by random integration of xylose assimilation genes into nts-sites\n",
      "disruption of a competing pathway to enhance shinorine production\n",
      "enhancing carbon flux to the pentose phosphate pathway by overexpressing transcriptional factor stb5 and transketolase tkl1\n",
      "optimizing the ratio of xylose to glucose in the medium for shinorine production\n",
      "materials and methods\n",
      "strains and culture conditions\n",
      "plasmid construction\n",
      "delta-integration and rdna-integration\n",
      "quantitative pcr (qpcr)\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00077/sb8b00077.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500258b/sb500258b.xml\n",
      "detection of molecular inputs\n",
      "intracellular signal integration\n",
      "production and delivery of a small molecule therapeutic\n",
      "containment and safety concerns\n",
      "choosing an appropriate chassis based on the target body niche\n",
      "outlook\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00525/sb8b00525.xml\n",
      "results and discussion\n",
      "identification and characterization of an ipp methyltransferase from streptomyces monomycini\n",
      "in vivo production of noncanonical prenyl pyrophosphates by the ipp methyltransferase\n",
      "in vivo incorporation of noncanonical prenyl pyrophosphates enables biosynthesis of new-to-nature carotenoids\n",
      "materials and methods\n",
      "bioinformatic analyses\n",
      "protein model generation\n",
      "chemicals, media, and bacterial strains\n",
      "plasmid construction\n",
      "production of alkylated isoprenoids\n",
      "enzyme expression and purification\n",
      "in vitro incubations\n",
      "hs-spme-gc/ms analysis\n",
      "sbse gc-ms analysis\n",
      "carotenoid analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400015r/sb400015r.xml\n",
      "engineered female-specific lethality\n",
      "tuning primary metabolism for pathway productivity\n",
      "engineered quorum sensing in yeast\n",
      "tuning promoter strengths for improved electron conduits\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00210/sb6b00210.xml\n",
      "toward the first data acquisition standard in synthetic biology\n",
      "survival of phenotypic information during cellular growth transitions\n",
      "permutation machines\n",
      "supervised learning in adaptive dna strand displacement networks\n",
      "the formal language and design principles of autonomous dna walker circuits\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00074/sb5b00074.xml\n",
      "results and discussion\n",
      "evaluation of the efficiency of talens-mediated targeted mutagenesis in saccharomyces cerevisiae\n",
      "genome-wide computational analysis and selection of multiple talens modification sites\n",
      "design of talens-assisted multiplex editing\n",
      "accelerated genome evolution to improve phenotypes via tame\n",
      "future considerations\n",
      "methods\n",
      "computational analysis of potential tale binding sites on the s. cerevisiae s288c genome\n",
      "talen plasmid construction\n",
      "editing the ura3 gene by different talens in yeast\n",
      "adaptive evolution of yeast cells via talens-assisted multiplex editing (tame)\n",
      "fluorescence assays\n",
      "genome resequencing and data analysis\n",
      "semiquantitative analysis of mcherry protein expression in yeast\n",
      "microarray analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00122/sb9b00122.xml\n",
      "results and discussion\n",
      "generality of orthorep across strains\n",
      "compatibility of orthorep with ρ + strains\n",
      "expedient crispr/cas9 manipulation of p1\n",
      "exploring the possible size of recombinant orthogonal p1 plasmids\n",
      "conclusion\n",
      "methods\n",
      "general dna cloning\n",
      "yeast strains and media\n",
      "yeast transformation and integration\n",
      "crispr/cas9-based p1 manipulation\n",
      "yeast protoplast fusion\n",
      "linear plasmid dna extraction\n",
      "p1 fluctuation test\n",
      "genomic orthogonality measurements\n",
      "determination of copy number via quantitative pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00028/sb7b00028.xml\n",
      "results and discussion\n",
      "construction of a synthetic acmnpv genome\n",
      "rescue of the synthetic baculovirus in cell culture\n",
      "acmnpv-wiv-syn1 retains the properties of the parental virus\n",
      "conclusion\n",
      "methods\n",
      "cells and viruses\n",
      "synthesis and construction of the acmnpv-wiv-syn1 genome\n",
      "transfection and infection assays\n",
      "one-step growth curve\n",
      "bioassays\n",
      "electron microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00192/sb9b00192.xml\n",
      "materials production and processing\n",
      "environmental detoxification\n",
      "fundamental research\n",
      "development of biofilm model systems\n",
      "conclusions and outlook\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00442/sb8b00442.xml\n",
      "results and discussion\n",
      "deletion and truncation of the pro-peptide area\n",
      "amino acid substitutions at the cleavage site of the p1 position\n",
      "site-directed mutagenesis of the pro-peptide\n",
      "multisite saturation mutagenesis\n",
      "fermentation of recombinant keratinase in a 15 l fermenter\n",
      "materials and methods\n",
      "materials\n",
      "expression of kerbp in the bacillus subtilis system\n",
      "culture conditions\n",
      "keratinase activity assay\n",
      "sds-page analysis\n",
      "verification of the key role of the pro-peptide\n",
      "truncation of the pro-peptide\n",
      "amino acid substitutions at the cleavage site of the p1 position\n",
      "site-directed mutagenesis of pro-peptide area\n",
      "multisite saturation mutagenesis\n",
      "investigation of the keratinase expression level in the 15 l fermenter\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002246/sb5002246.xml\n",
      "an evolved quorum sensing regulator for altered switching functions\n",
      "reprogramming microbes to seek and destroy pathogens\n",
      "dynamic multilevel, multiobjective metabolic modeling of microbial communities\n",
      "control of gene expression by quorum sensing-modulated and-gate promoters\n",
      "a generic metric to quantify quorum sensing activation dynamics\n",
      "engineering multicellular logic in bacteria with metabolic wires\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00047/sb9b00047.xml\n",
      "results and discussion\n",
      "methods\n",
      "cloning and protein expression\n",
      "size-exclusion chromatography docking assays\n",
      "fret measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500059x/sb500059x.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid assembly\n",
      "additional methods\n",
      "computerized luminous system\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00399/sb7b00399.xml\n",
      "results and discussion\n",
      "design of the bee microbiome toolkit (btk)\n",
      "btk plasmids function in diverse bacterial species found in the bee gut\n",
      "identifying functional promoters in bgm species\n",
      "inducible gene expression in bgm species\n",
      "crispri repression of chromosomal gene expression in bartonella apis\n",
      "cas9-assisted gene disruption in the bgm\n",
      "engineered strains colonize bees and can be directly visualized in the ileum\n",
      "snodgrassella staa contributes to gut colonization in vivo\n",
      "summary\n",
      "methods\n",
      "bacterial culture\n",
      "biparental conjugation\n",
      "btk construction\n",
      "measuring bgm gfp in vitro\n",
      "flow cytometry analysis of gfp expression\n",
      "tn7-transposition in b. apis\n",
      "crispri gene repression\n",
      "chromosomal disruption using cas9 and homologous recombination\n",
      "efficiency of chromosomal disruption with and without cas9\n",
      "laboratory care of honey bees\n",
      "mono- and co-inoculation of engineered bgm into honey bees\n",
      "in vivo imaging of bacterial burden using e2-crimson\n",
      "confocal fluorescence microscopy\n",
      "qpcr to assess colonization of staa mutant\n",
      "quantification and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00223/sb5b00223.xml\n",
      "methods\n",
      "plasmid assembly\n",
      "bacterial strains, media and culture conditions\n",
      "fluorescence measurements: flow cytometry\n",
      "microplate reader\n",
      "culture spotting on solid medium\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00027/sb6b00027.xml\n",
      "results\n",
      "design of vividzn\n",
      "characterization of vividzn-1\n",
      "increasing the dynamic range of vividzn\n",
      "tuning zn 2+ affinities toward physiologically relevant concentrations\n",
      "direct monitoring of light-induced zn 2+ release\n",
      "discussion\n",
      "materials and methods\n",
      "construction of expression plasmids\n",
      "expression and purification of vividzn switch proteins\n",
      "absorption measurements\n",
      "fluorescence measurements and zn 2+ titrations\n",
      "in vitro monitoring of zn 2+ release using red-colored fret sensor\n",
      "mammalian cell culture and imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00323/sb6b00323.xml\n",
      "results and discussion\n",
      "cipa and cipb as scaffolds in the native host p. luminescens\n",
      "cipa and cipb serve as scaffolds in e. coli\n",
      "isolation and characterization of cipa-gfp and cipb-gfp pcis from e. coli\n",
      "cip scaffolds organize lacz into enzymatically active pcis\n",
      "cip scaffolds co-organize gfp and lacz into bioactive pcis\n",
      "cipa scaffold brings together multiple enzymes involved in violacein biosynthetic pathway\n",
      "methods\n",
      "strains and growth conditions\n",
      "plasmid construction\n",
      "generation of the strains e. coli vio1 to vio6\n",
      "generation of p. luminescens δ cipa and/or δ cipb and cipa-gfp or cipb-gfp production strains\n",
      "isolation and characterization of pcis\n",
      "microscopy\n",
      "overproduction, purification, and assay of β-galactosidase (lacz)\n",
      "violacein/deoxyviolacein production and quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00271/sb8b00271.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "engineering the information transfer system in e. coli and s. typhimurium\n",
      "the e. coli and the s. typhimurium responders switch on in response to 3oc6hsl\n",
      "the e. coli and the s. typhimurium signalers sense atc and efficiently produce 3oc6hsl\n",
      "model describes the dynamic changes of the environment shaped by the signaler\n",
      "the intraspecies information transfer system in the mouse gut\n",
      "the interspecies information transfer system in the mouse gut using the s. typhimurium signaler and the e. coli responder\n",
      "discussion\n",
      "methods\n",
      "engineering the e. coli signaler and the responder\n",
      "engineering the s. typhimurium signaler and the responder\n",
      "time course dosage responses of the e. coli and the s. typhimurium responders\n",
      "quantification of the 3oc6hsl production by the e. coli and the s. typhimurium signalers\n",
      "in vitro cocultures of the s. typhimurium signaler and the e. coli responder\n",
      "intraspecies information transfer in the mouse gut\n",
      "interspecies information transfer in the mouse gut\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400045k/sb400045k.xml\n",
      "scientific meeting overview\n",
      "tools for genome engineering\n",
      "applications of precision genome engineering\n",
      "engineering biomolecules, circuits, modules, systems, and pathways\n",
      "in conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00380/sb8b00380.xml\n",
      "results and discussion\n",
      "selection of the candidate pathways for the ma production\n",
      "optimizing the production pathway of the ma\n",
      "expression of fusion proteins to further increase ma yield\n",
      "ph-controlled tube-scale culture using caco 3\n",
      "jar fermentor cultivation\n",
      "conclusions\n",
      "methods\n",
      "strains and plasmid construction\n",
      "fusion protein construction\n",
      "transformation of cft5-derived strains\n",
      "media\n",
      "culture conditions\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00378/sb8b00378.xml\n",
      "results and discussion\n",
      "rbps repress translation when bound within δ < 15 from the aug\n",
      "in vitro shape-seq reveal an extended protected region by pcp\n",
      "in vivo shape-seq measurements are consistent with in vitro measurements\n",
      "effective dissociation constant of rbps is insensitive to binding-site position\n",
      "discussion\n",
      "materials and methods\n",
      "design and construction of binding-site plasmids\n",
      "design and construction of fusion-rbp plasmids\n",
      "transformation of binding-site plasmids\n",
      "rna extraction and reverse-transcription for qpcr measurements\n",
      "qpcr measurements\n",
      "in vivo shape-seq\n",
      "rbp protection assay using in vitro shape-seq\n",
      "shape-seq library preparation and sequencing\n",
      "analysis routines and models\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00166/sb8b00166.xml\n",
      "results and discussion\n",
      "experimental data\n",
      "protein concentration vs plasmid copy number\n",
      "bin-dependent ode model\n",
      "modular composition\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00093/sb8b00093.xml\n",
      "results\n",
      "sarcosine as an effective source of glycine and formaldehyde\n",
      "minimal selection for the activity of the rump shunt\n",
      "medium selection for the activity of the rump shunt\n",
      "strong selection for the activity of the rump shunt\n",
      "strong selection for the activity of the rump shunt: changing the metabolic context\n",
      "discussion\n",
      "methods\n",
      "strains and genomic modifications\n",
      "synthetic-operon construction\n",
      "media and growth conditions\n",
      "formaldehyde concentration assay\n",
      "isotopic-labeling experiments\n",
      "flux balance analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500339k/sb500339k.xml\n",
      "results and discussion\n",
      "methods\n",
      "definitions\n",
      "circuit graph\n",
      "logical model\n",
      "circuit graph matching\n",
      "fan-in node\n",
      "part and module\n",
      "abstract subsolution and subsolution\n",
      "solution and cost\n",
      "part and module selection problem\n",
      "a two-step design workflow\n",
      "bound estimation on solution cost\n",
      "search for a list of solutions\n",
      "extension 1 (interaction look-ahead)\n",
      "extension 2 (branching extension)\n",
      "rank solutions by their predicted behavior\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300109d/sb300109d.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00064/sb5b00064.xml\n",
      "results\n",
      "synthetic, scalable, auxin-modulated transcription factors\n",
      "engineered adctf variants exhibit a broad range of auxin sensitivities and degradation kinetics\n",
      "yeast produce tunable levels of auxin via expression of iaah from agrobacterium tumefaciens\n",
      "sender cells produce a tunable auxin response in receiver cells\n",
      "discussion\n",
      "methods\n",
      "strain construction\n",
      "cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00236/sb5b00236.xml\n",
      "methods\n",
      "strains\n",
      "culture media and growth conditions\n",
      "lc/ms exometabolomics analysis\n",
      "genome-wide z. mobilis mutant fitness assays\n",
      "identification of lethal auxotroph and confirmation of rescue by released exometabolites\n",
      "flow cytometry analysis of coculture ratios\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00072/sb6b00072.xml\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00285/sb7b00285.xml\n",
      "b . subtilis endospores as a platform for protein display\n",
      "the advantages of the crust as a platform for protein display\n",
      "the goal of this study\n",
      "results and discussion\n",
      "p cotyz is the strongest crust promoter and therefore most suitable for surface display\n",
      "twelve sporovectors allow n- and c-terminal fusions to all crust protein genes\n",
      "gfp fusions to crust proteins reveal a general preference for the n-terminal fusion site\n",
      "proof of concept: laccase fusion spores exhibit enzymatic activity\n",
      "spore display increases the stability and allows reuse of laccases\n",
      "conclusion\n",
      "methods\n",
      "bacterial growth conditions\n",
      "transformation\n",
      "dna manipulation\n",
      "cloning of the sporovectors\n",
      "general cloning\n",
      "n-terminal sporovectors\n",
      "c-terminal sporovectors\n",
      "cloning of the gene of interest\n",
      "bioluminescence assays for determing promoter strengths\n",
      "spore preparation\n",
      "fluorescence microscopy and gfp quantification\n",
      "protein purification of the laccases\n",
      "laccase activity assay with abts and subsequent quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00148/sb7b00148.xml\n",
      "results and discussion\n",
      "design of the activation circuit\n",
      "characterization of the activation circuit\n",
      "encryption for activation circuit\n",
      "design of the degradation circuit\n",
      "integration of the activation circuit and the degradation circuit\n",
      "methods and materials\n",
      "strains and media\n",
      "plasmid circuit construction\n",
      "pathway encryption in plasmid\n",
      "pathway encryption in genome\n",
      "relative fluorescence measurements\n",
      "quantitative real-time pcr\n",
      "fluorescent-based plasmids degradation assay\n",
      "o-rrna plasmid degradation assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500084f/sb500084f.xml\n",
      "results and discussion\n",
      "plasticity of tf binding sites in the genome of e. coli\n",
      "in silico optimization of overlapping binding sites\n",
      "construction and in vivo experimental validation of synthetic lac promoters\n",
      "conclusions\n",
      "methods\n",
      "cis -regulatory data set\n",
      "analysis of binding site arrangements\n",
      "analysis of cis -regulatory elements in random sequences\n",
      "in silico evolution of cis -regulatory elements\n",
      "bacterial strains and growth conditions\n",
      "construction of gfp probe vector and promoter cloning\n",
      "in vivo promoter assay\n",
      "confocal fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00277/sb6b00277.xml\n",
      "results and discussion\n",
      "libsbolj library\n",
      "online web-based tool\n",
      "restful api\n",
      "discussion\n",
      "software compliance testing\n",
      "support for conversion between more formats\n",
      "integration with existing libraries\n",
      "additional features\n",
      "methods\n",
      "sbol validation\n",
      "conversion\n",
      "online web interface implementation\n",
      "testing of the sbol validator/converter\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300080t/sb300080t.xml\n",
      "results and discussion\n",
      "construction of a functional epothilone polyketide pathway by synthetic genes\n",
      "functional sequence requirements for m. xanthus as heterologous expression host\n",
      "constructive sequence requirements for pathway assembly and modification\n",
      "design of an artificial epothilone pathway sequence\n",
      "construction of an artificial epothilone pathway\n",
      "heterologous expression in m. xanthus\n",
      "outlook\n",
      "methods\n",
      "sequence analysis and design\n",
      "source of coding sequences\n",
      "analysis of codon adaptation\n",
      "analysis of slow codon clusters\n",
      "strains and culture conditions\n",
      "dna manipulation, analysis, and pcr\n",
      "heterologous expression of the artificial epothilone biosynthetic pathway\n",
      "epothilone production analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00163/sb6b00163.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00282/sb6b00282.xml\n",
      "results\n",
      "design of the subsystems\n",
      "weak homodimerization of gal4p amplifies the glucose signal quadratically\n",
      "quantification of interactions between gal80p, gal1p, gal3p and galactose\n",
      "validation of the parameters by prediction the behavior of a feedback loop\n",
      "discussion\n",
      "methods\n",
      "selection of proteotypic peptides by label-free quantification and directed lc–ms\n",
      "determination of absolute molecule numbers\n",
      "determination of the normalization factor\n",
      "construction of deletion strains\n",
      "construction of strains for controllable protein expression\n",
      "growth conditions\n",
      "memory experiments\n",
      "shutting off gene expression to vary the protein concentration\n",
      "measurement of intracellular galactose concentration\n",
      "data fitting\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400074w/sb400074w.xml\n",
      "results and discussion\n",
      "chemistry\n",
      "primer extension with d s ntps using a series of dna polymerases\n",
      "kinetic studies of the incorporation of d s ntps via single nucleotide insertion\n",
      "optimization of the conditions for the pcr with d s ntps\n",
      "conclusion\n",
      "methods\n",
      "synthesis of d s ntps\n",
      "2′-deoxy-4′-thioadenosine 5′-triphosphate (d s atp)\n",
      "2′-deoxy-4′-thioguanosine 5′-triphosphate (d s gtp)\n",
      "primer extension\n",
      "single nucleotide insertion and steady-state kinetics\n",
      "pcr with d s ntp\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300105b/sb300105b.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00258/sb7b00258.xml\n",
      "results\n",
      "design and characterization of the broad-spectrum promoter p bs.\n",
      "construction of shuttle vectors with the broad-spectrum promoter\n",
      "construction of broad-spectrum promoters with different strengths\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00238/sb7b00238.xml\n",
      "repurposing a two-component system-based biosensor for the killing of vibrio cholerae\n",
      "a straightforward approach for 3d bacterial printing\n",
      "designing uniquely addressable bio-orthogonal synthetic scaffolds for dna and rna origami\n",
      "cogenerating synthetic parts toward a self-replicating system\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00197/sb7b00197.xml\n",
      "methods\n",
      "plasmids, bacterial strains and other materials\n",
      "cloning of pmag variants into pb33ecpx plasmid\n",
      "expression and purification of nmaghigh\n",
      "functionalization of glass surfaces with nmaghigh\n",
      "bacterial adhesion and detachment assays\n",
      "bacterial patterning\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00280/sb8b00280.xml\n",
      "methods\n",
      "strains, reagents, and plasmids\n",
      "illumination setup and microplate readings\n",
      "dual-input promoter design and characterization\n",
      "reprogrammable logic circuit design and characterization\n",
      "spatial pattern formation\n",
      "mathematical modeling to understand dual-input promoter dynamics\n",
      "data analysis\n",
      "results\n",
      "ligand and blue light-based dual-input promoter system\n",
      "temporal control dynamics of dual-input promoter system\n",
      "reprogrammable logic gene circuit based on layered dual-mode promoters\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200017p/sb200017p.xml\n",
      "results and discussion\n",
      "discussion\n",
      "methods\n",
      "medium\n",
      "gene construction\n",
      "strain construction\n",
      "cell culture and induction\n",
      "fermentation test\n",
      "enzyme activity test\n",
      "fluorescence microscopy\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400202s/sb400202s.xml\n",
      "results and discussion\n",
      "expression of scve\n",
      "growth inhibition curves\n",
      "npn permeability\n",
      "enhanced ca activity\n",
      "increased functional activity\n",
      "diffusion/reaction quantification\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "cloning\n",
      "protein expression\n",
      "npn assay\n",
      "kinetic assays\n",
      "caco 3 precipitation\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00120/sb9b00120.xml\n",
      "results and discussion\n",
      "characteristics of the e. coli strains suptoxd and suptoxr\n",
      "optimization of expression conditions for recombinant mp production in suptoxd and suptoxr\n",
      "suptoxd and suptoxr achieve enhanced production of detergent-extractable and functional recombinant mp\n",
      "suptoxd and suptoxr achieve greatly enhanced production of purified recombinant mps\n",
      "methods\n",
      "plasmid construction\n",
      "mp overexpression in liquid cultures\n",
      "membrane isolation\n",
      "western blot and in-gel fluorescence analyses\n",
      "bulk fluorescence measurements\n",
      "fluorescence analysis by flow cytometry\n",
      "rhodopsin extraction and quantification\n",
      "mscl and br2 purification\n",
      "peldor measurements, data analysis, and in silico spin labeling and distance modeling\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00091/sb7b00091.xml\n",
      "experimental setup\n",
      "results and discussion\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00426/sb8b00426.xml\n",
      "results\n",
      "crispr interference from a broad-host-range vector\n",
      "crispri-me reduces mutation rates in e. coli\n",
      "crispri-me stabilizes burdensome protein expression in e. coli\n",
      "crispri-me reduces mutation rates in a. baylyi adp1\n",
      "discussion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "broad-host-range crispri platform\n",
      "gfp repression assays\n",
      "mutation rate measurements\n",
      "doubling time measurements\n",
      "monitoring decay of plasmid function\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300014t/sb300014t.xml\n",
      "results and discussion\n",
      "schema library design\n",
      "rational generation of an informative set of chimeras\n",
      "regression model for long-term stability\n",
      "metal dependence of stability\n",
      "summary and conclusions\n",
      "methods\n",
      "active learning algorithm\n",
      "gene synthesis and cloning\n",
      "expression and purification\n",
      "enzyme kinetics\n",
      "long-term stability\n",
      "thermal stability\n",
      "regression analysis\n",
      "immunogenicity calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00178/sb9b00178.xml\n",
      "results and discussion\n",
      "protocol standardization\n",
      "establishing key sources of variability via material exchange\n",
      "small contribution of extract to interlaboratory variability\n",
      "baseline analysis of cfps reaction variability when using shared reagents\n",
      "further investigation of factors that contribute to site/operator variability\n",
      "overall assessment of sources of interlaboratory variability\n",
      "materials and methods\n",
      "strains, protein purification, and plasmids\n",
      "preparation of cell extract\n",
      "cell-free mrna and protein synthesis\n",
      "cfps reactions using an acoustic liquid handler\n",
      "gfp standard curves\n",
      "data management and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000302/sb5000302.xml\n",
      "results and discussion\n",
      "choice of an appropriate membrane anchor template from p. patens\n",
      "identification and experimental validation of the signal peptide\n",
      "bioinformatic analysis of the transmembrane domain\n",
      "assembly and characterization of hybrid membrane proteins\n",
      "proteolytic release of the extracellular functional domain\n",
      "discussion and conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500356d/sb500356d.xml\n",
      "methods\n",
      "basic assembly protocol\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00368/sb7b00368.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids construction\n",
      "plant material and transformation\n",
      "confocal microscopy\n",
      "squalene analysis\n",
      "droplet isolation\n",
      "treatment in darkness\n",
      "photosynthesis and growth measurement\n",
      "stimulated raman scattering microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00285/sb6b00285.xml\n",
      "methods\n",
      "strains and media\n",
      "cloning procedures\n",
      "genomic integrations\n",
      "flow cytometry\n",
      "lycopene quantification\n",
      "quantitative pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002714/sb5002714.xml\n",
      "creation of cellobiose and xylooligosaccharides- coutilizing e. coli\n",
      "reconstituted biosynthesis of the antibiotic valinomycin\n",
      "building stepwise, complex, multicomponent tolerance to toxic chemicals\n",
      "solution structure of a bacterial microcompartment targeting peptide\n",
      "engineered platform for bioethylene production\n",
      "microbial synthesis of pinene\n",
      "cloning and optimization of a nisin biosynthesis pathway\n",
      "engineering bacteria for the microbial synthesis of 5-hydroxytryptophan\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00002/sb6b00002.xml\n",
      "results and discussion\n",
      "methods\n",
      "genetically engineered bacteria\n",
      "microfluidic device\n",
      "live cell lithography (lcl)\n",
      "prepolymer solution\n",
      "fluorescence imaging\n",
      "simulation of the dynamic response of transmitters/receivers to an inducer\n",
      "finite element simulation\n",
      "simulation of dynamic gene expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00383/sb7b00383.xml\n",
      "results and discussion\n",
      "construction of a plasmid toolkit for cfps-i: a versatile and modular genetic system\n",
      "cell-free synthesis and immobilization of his-sgfp in one pot\n",
      "on-demand synthesis of different biomaterials functionalized with sgfp tagged with different tags\n",
      "incorporation of non-natural amino acids through cfps to enable bio-orthogonal immobilization mediated by copper-free click chemistry\n",
      "in vitro synthesis and selective co-immobilization of two proteins fused to different tags on the same carrier\n",
      "two-pot fabrication of immunoconjugates by cfps-i\n",
      "conclusion\n",
      "methods\n",
      "reagents and materials\n",
      "genetic engineering of a toolbox of plasmids\n",
      "in vitro cell-free protein synthesis and immobilization\n",
      "determination of his-sgfp loading on ag-co\n",
      "enzymatic activity assays\n",
      "fluorescence microscopy imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00207/sb6b00207.xml\n",
      "michael a. boemo\n",
      "hieu bui\n",
      "sudhanshu garg\n",
      "matthew lakin\n",
      "curtis madsen\n",
      "james a. mclaughlin\n",
      "göksel misirli\n",
      "reem mokhtar\n",
      "matthew pocock\n",
      "j. christian j. ray\n",
      "tianqi song\n",
      "darko stefanovic\n",
      "anil wipat\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00351/sb6b00351.xml\n",
      "kim c. anderson\n",
      "adokiye berepiki\n",
      "antoni e. bordoy\n",
      "jimmy budiardjo\n",
      "michael b. cory\n",
      "quentin m. dudley\n",
      "indro neil ghosh\n",
      "tobias w. giessen\n",
      "allison hoynes-o’connor\n",
      "beijing k. huang\n",
      "he huang\n",
      "linh huynh\n",
      "shi-yuan li\n",
      "duo liu\n",
      "audrey morin\n",
      "cameron myhrvold\n",
      "ana m. oliveira paiva\n",
      "daniel schindler\n",
      "wiep klaas smits\n",
      "kassi t. stein\n",
      "james winkler\n",
      "wang xi\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00089/sb7b00089.xml\n",
      "results and discussion\n",
      "mutation positions and library generation\n",
      "library selection using ribosome display\n",
      "determination of binding parameters via isothermal titration calorimetry (itc)\n",
      "characterization of identified rtx block v mutants by circular dichroism (cd) spectroscopy\n",
      "combined binding interfaces for improved apparent affinity\n",
      "in vitro demonstration of molecular recognition regulation via affinity chromatography\n",
      "materials and methods\n",
      "plasmids and chemicals\n",
      "library construction\n",
      "selection of mutants via ribosome display\n",
      "cloning of mutants into expression plasmids and construction of concatemers\n",
      "construction of double face mutants\n",
      "expression and purification of wt and mutant rtx peptides\n",
      "circular dichroism (cd) spectroscopy\n",
      "isothermal titration calorimetry (itc)\n",
      "affinity chromatography experiments\n",
      "activity assay of lysozyme\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00379/sb6b00379.xml\n",
      "results\n",
      "cloning of m. hominis pg21 into yeast\n",
      "analysis of yeast transformants by multiplex pcr and pfge\n",
      "sequencing of the arg-trna region\n",
      "stability of the m. hominis pg21 genome during yeast propagation\n",
      "subcloning and analyses of partial genomes\n",
      "m. hominis pg21 genome modification using crispr/cas9 technology\n",
      "discussion\n",
      "methods\n",
      "yeast and m. hominis culture conditions\n",
      "isolation of m. hominis pg21 gdna\n",
      "construction of the tar vector\n",
      "yeast transformation with m. hominis pg21 gdna\n",
      "screening of yeast transformants\n",
      "sequence analysis at the insertion site\n",
      "yeast clone stability\n",
      "genome modification using crispr/cas9 technology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00441/sb7b00441.xml\n",
      "meike baumgart\n",
      "nicholas c. o. lee\n",
      "nicholas j. morse\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00016/sb5b00016.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "screening of strong constitutive promoters in the s. albus transcriptome via rna-seq\n",
      "qpcr analysis of the selected promoters in the native host\n",
      "cloning of the constitutive promoters\n",
      "characterization of the chosen promoters using the xyle reporter gene\n",
      "analysis of the identified promoters using qpcr\n",
      "comparison of polycyclic tetramate macrolactam (ptm) production using newly identified strong promoters versus previously identified promoters\n",
      "conclusions\n",
      "methods\n",
      "strains and reagents\n",
      "streptomyces albus cultivation and rna extraction\n",
      "rna-seq analysis\n",
      "expression analysis\n",
      "promoter cloning\n",
      "verification of the assembled gene clusters\n",
      "heterologous expression in s. albus\n",
      "promoter characterization via the xyle assay\n",
      "promoter characterization via qpcr analysis\n",
      "ptm gene cluster reconstruction, heterologous expression, and hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500303q/sb500303q.xml\n",
      "methods\n",
      "strains, plasmids and medium\n",
      "in vivo enzyme assays\n",
      "microbial production of 5-haa\n",
      "microbial production of 5-htp\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00170/sb5b00170.xml\n",
      "results\n",
      "a resettable memory switch based on fime and hbif\n",
      "evolutionary stability of the memory switch\n",
      "model-based design of a rewritable not gate\n",
      "construction of a rewritable not gate using fime and hbif\n",
      "evolutionary stability of the not gate\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500254g/sb500254g.xml\n",
      "results and discussion\n",
      "rationale for the design of new gating behavior into protein switch rg13\n",
      "identification of the locations for the engineered disulfide bonds in rg13\n",
      "switching cellular phenotype shows redox-dependent, maltose-controlled switching behavior\n",
      "rg13-and2 and rg13-orn2 exhibit gating behavior in vitro\n",
      "rg13-yes: inverting a yes gate\n",
      "conformational changes as monitored by intrinsic protein fluorescence\n",
      "perspective\n",
      "methods\n",
      "materials\n",
      "identifying target residues for engineering disulfide bonds and molecular modeling of disulfide bonds-engineered rg13 variants\n",
      "mic amp assay\n",
      "expression and purification of rg13 variants\n",
      "nitrocefin hydrolysis enzymatic assay\n",
      "intrinsic protein fluorescence\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00099/sb5b00099.xml\n",
      "results and discussion\n",
      "qs peptide production in b. megaterium\n",
      "density-dependent gene regulation in b. megaterium in complex and minimal media\n",
      "synthetic communication between b. megaterium sender and receiver cells lines grown in co-culture\n",
      "methods\n",
      "bacterial strains, media, and growth conditions\n",
      "plasmid construction\n",
      "cell supernatant assays\n",
      "shake flask assays\n",
      "b. megaterium cell–cell communication assays in liquid media\n",
      "b. megaterium cell–cell communication assays on semi-solid agar\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00025/sb7b00025.xml\n",
      "methods\n",
      "cell culture and transfection\n",
      "rna-fish and immunofluorescence\n",
      "image acquisition of hela cells\n",
      "quantification of mrna transport in hela cells\n",
      "image acquisition of hippocampal neurons\n",
      "quantification of mrna transport in hippocampal neurons\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00048/sb6b00048.xml\n",
      "results and discussion\n",
      "methods\n",
      "engineering alternate cooperative communication\n",
      "fluorescent reporter system\n",
      "growth and flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00339/sb8b00339.xml\n",
      "results and discussion\n",
      "design of a synthetic landing pad system for multicopy gene integration in s. cerevisiae\n",
      "construction and performance of the lp system\n",
      "improving efficiency of multicopy gene integration\n",
      "application to synthesis of bias\n",
      "materials and methods\n",
      "strains, cultivation, and dna manipulation\n",
      "metabolite analysis and analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000244/sb3000244.xml\n",
      "results and discussion\n",
      "gene structure of benm and catm regulated biosensors\n",
      "biosensors induced by catechol\n",
      "biosensors induced by salicylate\n",
      "biosensors induced by benzoate\n",
      "cross regulation in acinetobacter baylyi adp1\n",
      "traditional gene transcription and translation model\n",
      "new mathematical gene regulation model developed in this study\n",
      "theoretical and mathematical analysis of cross gene regulation\n",
      "application of new model to predict performance of adpwh_catm_δ s alr\n",
      "materials and methods\n",
      "bacteria strains, plasmids, culture media, and chemicals\n",
      "biosensor construction\n",
      "cell culture and induction\n",
      "bioluminescence detection and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00188/sb6b00188.xml\n",
      "results and discussion\n",
      "approach\n",
      "experimental validation\n",
      "comparison to previous studies\n",
      "conclusion and outlook\n",
      "methods\n",
      "csr-salad development\n",
      "structure analysis\n",
      "cloning and library creation\n",
      "library screening\n",
      "enzyme expression, purification, and kinetic measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00260/sb7b00260.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "measuring pinene sensor\n",
      "selection experiment\n",
      "toxicity measurements\n",
      "next generation sequencing\n",
      "bioinformatics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300015f/sb300015f.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003218/sb5003218.xml\n",
      "methods\n",
      "combinatorial engineering\n",
      "production and analysis of metabolites from engineered streptomyces strains\n",
      "isolation, purification, and structural elucidation of bisindoles from streptomyces mutants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00013/sb6b00013.xml\n",
      "results and discussion\n",
      "part implementation and characterization\n",
      "a bistable and hysteretic feedback loop system\n",
      "“all-or-none” state switching\n",
      "summary and conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "strain construction\n",
      "media\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "mathematical modeling\n",
      "parameter estimation for open-loop data\n",
      "prediction of closed-loop response\n",
      "stochastic simulations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00289/sb8b00289.xml\n",
      "results and discussion\n",
      "saturation mutagenesis at f340 and a570 of isoprene synthase increased isoprene production\n",
      "combination of isoprene synthase engineering with pathway compartmentation\n",
      "balance reconstruction by regulation of mvd1 and idi1 expression\n",
      "fermentation of diploid s. cerevisiae in 5 l bioreactor\n",
      "methods\n",
      "construction of plasmids and strains\n",
      "molecular docking analysis of isps\n",
      "media and culture conditions in sealed vials\n",
      "gas chromatography (gc) analysis of isoprene\n",
      "high performance liquid chromatography (hplc) analysis of sugars, ethanol and squalene\n",
      "batch and fed-batch fermentation in 5 l bioreactor\n",
      "fed-batch fermentation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00517/sb8b00517.xml\n",
      "results and discussions\n",
      "positive feedback in ligand production\n",
      "positive feedback in both ligand and receptor\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400144h/sb400144h.xml\n",
      "methods\n",
      "dna constructs\n",
      "preparation of trna transcripts\n",
      "cell-free protein expression\n",
      "detection of radiolabeled products\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500112f/sb500112f.xml\n",
      "christina agapakis\n",
      "kazuaki amikura\n",
      "jaroslav bendl\n",
      "francesca ceroni\n",
      "pavel dvorak\n",
      "anton glieder\n",
      "thomas gorochowski\n",
      "haseong kim\n",
      "nagendra prasad kurumbang\n",
      "kilkoang kwon\n",
      "julia pitzer\n",
      "georgios pothoulakis\n",
      "yuta sakai\n",
      "suvi santala\n",
      "thomas vogl\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00225/sb8b00225.xml\n",
      "results and discussion\n",
      "different αhbs have binding abilities\n",
      "αhb receptors induce chirality in the encapsulated dyes\n",
      "computational modeling sheds light on the modes of binding of dph to αhbs\n",
      "αhbs bind biological lipophilic molecules\n",
      "a cc-hex2:farnesol complex elucidated by x-ray crystallography\n",
      "the lumens of αhbs can be engineered to accept charged molecules\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00242/sb8b00242.xml\n",
      "results\n",
      "dimeric biosensors are stabilized by ligand binding\n",
      "biosensor activity is improved by computational redesign of the ligand-binding domain dimer interface\n",
      "selection of mutations that impart improved heterodimerization or ligand selectivity\n",
      "optimizing biosensor activity\n",
      "discussion\n",
      "materials and methods\n",
      "yeast methods\n",
      "yeast strains\n",
      "plasmid constructs\n",
      "mutagenesis and selection\n",
      "luciferase assays\n",
      "computational redesign\n",
      "sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00424/sb7b00424.xml\n",
      "results and discussion\n",
      "the biolinker, a cost-effective 3d printer for bacteria\n",
      "bacteria can survive for extended periods of time in alginate gels\n",
      "bacteria can be 3d printed in discrete layers\n",
      "csga expression is rate-limiting for curli formation\n",
      "biofilm formation can be controlled by exogenous expression of csga\n",
      "csga-expressing bacteria allow the production of patterned biofilms\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains, plasmids, and culture\n",
      "the biolinker\n",
      "bioink preparation\n",
      "printing substrates\n",
      "characterization of line widths\n",
      "confocal microscopy\n",
      "print and microscopy of layered bioink structures\n",
      "biofilm formation upon expression of csga in microtiter plates\n",
      "alginate gel dissolution\n",
      "bacterial survival in printed bioink\n",
      "biofilm formation in alginate gels\n",
      "release of bacteria into citrate solution\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00392/sb6b00392.xml\n",
      "results and discussion\n",
      "engineering protease-responsive grb switches\n",
      "sumo-grb selectively triggers apoptosis of senp1-overexpressing cells\n",
      "the covert architecture is modular and compatible with diverse proteases\n",
      "grb-based cytotoxic switches are mechanistically compatible with adoptive t-cell therapy\n",
      "conclusion\n",
      "methods\n",
      "cell lines\n",
      "dna constructs\n",
      "cell transfection\n",
      "retrovirus production\n",
      "primary human t-cell isolation and retroviral transduction\n",
      "ni 2+ -affinity protein purification\n",
      "western blot\n",
      "ac-iepd-pna activity assay\n",
      "flow cytometry\n",
      "live-cell confocal microscopy\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00430/sb7b00430.xml\n",
      "results and discussion\n",
      "constructing an e. coli – c. testosteroni shuttle system\n",
      "evaluating an inducible promoter set\n",
      "redesigning the hybrid iptg inducible promoters\n",
      "establishing an orthogonal gene expression system\n",
      "constructing an orthogonal t7 promoter library\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, plasmids, and cultivation conditions\n",
      "genetic manipulation and transformation\n",
      "plasmid construction\n",
      "plasmid transformation efficiency and stability assessment\n",
      "t7 rnap genome integration\n",
      "growth and promoter strength measurements\n",
      "facs screening\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00036/sb9b00036.xml\n",
      "results and discussion\n",
      "5-fc toxicity assay in e. coli confirms ycd activity\n",
      "accumulation of hif-1α in ht1080 cells in mock-hypoxia using cobalt chloride\n",
      "generation of ht1080 stable cell lines by lentiviral transduction\n",
      "mock-hypoxia increases 5-fc toxicity in ht1080 fibrosarcoma cells expressing hope fusions\n",
      "hope fusions accumulate in mock-hypoxia\n",
      "conclusions\n",
      "methods\n",
      "construction of hope fusions\n",
      "5-fc toxicity assay in e. coli\n",
      "construction of phiv-egfp vectors containing the cag-hope fusion cassette\n",
      "lentiviral transduction to create stable lines in ht1080 cells\n",
      "accumulation of hif-1α by addition of cobalt chloride\n",
      "prodrug toxicity assay on ht1080 stable lines in normoxia and mock-hypoxia\n",
      "cocl 2 -induced accumulation of hope fusions\n",
      "western blots\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00169/sb7b00169.xml\n",
      "results and discussion\n",
      "design of light-inducible t7 rna polymerases (opto-t7rnaps)\n",
      "dynamic light-inducible t7rnap\n",
      "expression level set points and light sensitivity\n",
      "high expression strength opto-t7rnaps\n",
      "fast-reverting light-inducible t7 rna polymerase allows for fast dynamic regulation\n",
      "comparison of opto-t7rnaps to pat7p-1\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and media\n",
      "plasmids and genetic parts\n",
      "integration of lacya177c into e. coli bw25113\n",
      "t7rnap-induced gene expression during steady-state growth\n",
      "mcherry maturation assay\n",
      "formaldehyde cell fixation assay\n",
      "flow cytometry measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00534/sb8b00534.xml\n",
      "results and discussions\n",
      "design of a biosynthetic pathway for nitrotrp production in e. coli\n",
      "detection of recombinant biosynthetic enzymes and nitrotrp production in e. coli\n",
      "optimization of the nitrotrp biosynthetic pathway\n",
      "improved production of nitrotrp via screening fermentation media and temperature\n",
      "dependence of all three enzymes for the production of nitrotrp\n",
      "inactivation of competitive l -trp metabolic pathways in e. coli to improve nitrotrp production\n",
      "production of nitrotrp analogues in e. coli\n",
      "conclusions\n",
      "methods and materials\n",
      "general chemicals, dna subcloning, and bacterial strains\n",
      "creation of nitrotrp biosynthetic pathways\n",
      "whole-cell biotransformation\n",
      "inactivation of genes in e. coli\n",
      "tnaa assay\n",
      "hplc and lc–ms methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000143/sb5000143.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "protoplasting and transformation of cochliobolus heterostrophus:\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00238/sb9b00238.xml\n",
      "results and discussion\n",
      "characterization of rna activators\n",
      "mathematical modeling\n",
      "experimental evaluation of prototype gate-model prediction\n",
      "additional parts characterization\n",
      "cross-design of single components into logic devices\n",
      "comparison of dose–response matrices\n",
      "s6t3 peak production almost matches unregulated plasmid\n",
      "in vivo implementation\n",
      "conclusion\n",
      "materials and methods\n",
      "cell-free reactions\n",
      "preparation of cell-free reactions\n",
      "incubation and kinetic measurements\n",
      "end-point measurements\n",
      "assembly of expression constructs\n",
      "flow cytometer analysis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "parallel tempering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00535/sb8b00535.xml\n",
      "results and discussion\n",
      "developed random integration method\n",
      "genomic integration of the flavonoid pathway\n",
      "probing the genomic integration site with the aro1 gene\n",
      "iterative integration of flavonoids precursor pathway\n",
      "functional expression of plant cytochrome c p450 enzyme to produce hydroxylated flavonoids\n",
      "conclusions\n",
      "materials and methods\n",
      "plasmids and strains\n",
      "26s rdna and loxp pasmid prdnaloxp construction\n",
      "flavonoid pathway construction\n",
      "26s rdna random integration and marker curation\n",
      "iterative chromosomal integration and marker curation\n",
      "analytical method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400201u/sb400201u.xml\n",
      "methods\n",
      "modeling\n",
      "simulation\n",
      "yield calculations\n",
      "parameter sampling\n",
      "statistical analysis\n",
      "experimental measurements of toxicity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500319p/sb500319p.xml\n",
      "results and discussion\n",
      "design of an orthogonal fatty acid biosynthesis route\n",
      "construction of a heterologous faee pathway\n",
      "cloning and characterization of four type-i fas genes\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "media and culture conditions\n",
      "free fatty acid extraction\n",
      "faee extraction and thin-layer chromatography\n",
      "gc-ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400062d/sb400062d.xml\n",
      "swapnil bhatia\n",
      "sander a. a. kooijmans\n",
      "agnieszka janina zygadlo nielsen\n",
      "markus de raad\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00242/sb5b00242.xml\n",
      "results and discussion\n",
      "discussion\n",
      "methods\n",
      "extending sbml to include arrays\n",
      "analysis of arrayed models\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00381/sb8b00381.xml\n",
      "results and discussion\n",
      "module i: investigating mechanisms of antibiotic resistance\n",
      "module ii: fundamentals of crispr-cas9 gene editing\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains and plasmids\n",
      "construction of rpsl r86s mutant strain\n",
      "cell-free extract preparation\n",
      "cell-free protein synthesis (cfps)\n",
      "lyophilization of cell-free reactions\n",
      "quantification of in vitro synthesized protein\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00156/sb5b00156.xml\n",
      "methods\n",
      "bi-plateau gene expression and cell growth\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00068/sb6b00068.xml\n",
      "results and discussion\n",
      "transporting uirs–uirr into e. coli\n",
      "characterizing p csir1 functional elements\n",
      "uirs signals to uirr via phosphorylation\n",
      "hardwiring uirs and uirr expression for optogenetics\n",
      "characterizing the uirs–uirr action spectrum\n",
      "characterizing and modeling the uirs–uirr steady-state transfer function\n",
      "characterizing and modeling uirs–uirr response dynamics\n",
      "predictive control of uirs–uirr gene expression dynamics\n",
      "conclusions\n",
      "methods\n",
      "plasmids, strains, and media\n",
      "phos-tag sds page and western blot\n",
      "optical hardware\n",
      "optogenetic experiments\n",
      "flow cytometry and data analysis\n",
      "hill function model fitting\n",
      "dynamical model fitting\n",
      "gene expression programming\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00139/sb9b00139.xml\n",
      "results\n",
      "glyph specifications\n",
      "diagram language\n",
      "expanded glyph collection\n",
      "examples of usage\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00175/sb8b00175.xml\n",
      "results and discussion\n",
      "construction of the rs assay system\n",
      "validation of the rs assay system\n",
      "design of degenerate streptomyces rss\n",
      "construction of the library of synthetic streptomyces rss\n",
      "construction of synthetic streptomyces rs cassettes\n",
      "refactoring of the act gene cluster using synthetic rs cassettes\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and culture conditions\n",
      "construction of the indigoidine synthetase gene-based rs assay vector\n",
      "construction of the rs library\n",
      "construction of rs cassettes\n",
      "refactoring of the act biosynthetic gene cluster\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001504/sb4001504.xml\n",
      "results and discussion\n",
      "golden gate assembly standard for plant biology\n",
      "the golden gate moclo plant tool kit\n",
      "the golden gate moclo plant parts kit\n",
      "conclusion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001326/sb3001326.xml\n",
      "results and discussion\n",
      "biological model\n",
      "assessment of the mutation frequency generated by erasc\n",
      "functional assessment\n",
      "fluorescence characterization of mutants\n",
      "conclusion\n",
      "methods\n",
      "synthesis of degenerate oligonucleotides\n",
      "erasc approach\n",
      "mkate-s158a gene ( emkate )\n",
      "gene libraries\n",
      "mkate gene\n",
      "protein purification\n",
      "spectral measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00287/sb8b00287.xml\n",
      "results and discussion\n",
      "methods\n",
      "a. gossypii strains and growth conditions\n",
      "golden gate assembly method\n",
      "lipid extraction and analysis\n",
      "quantitative real-time pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00037/sb6b00037.xml\n",
      "methods\n",
      "indirect control scheme surface preparation\n",
      "direct control scheme surface preparation\n",
      "induced biotin production\n",
      "modeling and simulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00004/sb6b00004.xml\n",
      "in-yeast engineering of a bacterial genome\n",
      "a microfluidic platform for long-term monitoring of algae\n",
      "engineering translational activators with crispr-cas\n",
      "metabolic engineering toward sustainable production of nylon-6\n",
      "orthogonal modular gene repression in e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001382/sb4001382.xml\n",
      "results and discussion\n",
      "identifying geranyl diphosphate synthases\n",
      "identifying pinene synthases\n",
      "microbial synthesis of pinene via coexpression\n",
      "geraniol toxicity\n",
      "pinene toxicity\n",
      "overcoming gpps inhibition by gpp via protein fusions\n",
      "pinene microbial production using protein fusions\n",
      "rationalization of operon vs protein fusion performance\n",
      "ratios of α- and β-pinene\n",
      "methods\n",
      "construction of pps-gpps (co-expressions)\n",
      "construction of pgpps-(gsg) 2 -ps (protein fusions)\n",
      "pinene production and quantification\n",
      "pinene and geraniol toxicity measurements\n",
      "pinene conversion using cell lysate\n",
      "flux balance analysis yield calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00014/sb8b00014.xml\n",
      "results\n",
      "noise distributions and quantification\n",
      "a biological implementation: fret reporter pair\n",
      "parameter selection\n",
      "deterministic analysis\n",
      "parameter dependence and noise sensitivity\n",
      "discussion\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00104/sb9b00104.xml\n",
      "results\n",
      "library design and assembly\n",
      "high-throughput functional screening\n",
      "functional characterization\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00223/sb7b00223.xml\n",
      "results and discussion\n",
      "development of a model for dynamic promoter activity in cell culture\n",
      "model implementation\n",
      "model reconstructs both shape and magnitude of true promoter activity\n",
      "an evaluation of robustness to inaccuracies in maturation and degradation rates\n",
      "comparison to existing models\n",
      "case study 1: gfp driven by a xylose-inducible promoter\n",
      "case study 2: gfp driven by the e. coli fis promoter\n",
      "future considerations: use of fluorescent rna input\n",
      "in summary\n",
      "methods\n",
      "simulation of biomass and fluorescence data\n",
      "analysis of parameter space\n",
      "varying temporal resolution\n",
      "strain and growth conditions\n",
      "fluorescence and biomass measurement\n",
      "analysis of pxyla promoter activity\n",
      "analysis of fis promoter activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00192/sb6b00192.xml\n",
      "results and discussion\n",
      "assessing orthogonal translation system performance\n",
      "comparing orthogonal translation systems\n",
      "effect of varying nonstandard amino acid concentration\n",
      "characterizing orthogonal translation system polyspecificity\n",
      "amberless e. coli evolved for more rapid growth\n",
      "future improvements to the kit\n",
      "conclusions\n",
      "methods\n",
      "media and chemicals\n",
      "kit plasmid construction\n",
      "ots plasmids\n",
      "incorporation assays\n",
      "calculation of rre and mmf values\n",
      "protein purification and mass spectrometry\n",
      "amberless e. coli evolution experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00283/sb8b00283.xml\n",
      "results and discussion\n",
      "addressing cotranslational repa-wh1 aggregation in vitro\n",
      "dnak (hsp70) chaperone modulates the size of repa-wh1 aggregates\n",
      "the cochaperones dnaj (hsp40) and grpe (nef) enhance remodeling of repa-wh1 aggregates\n",
      "materials and methods\n",
      "plasmid construction\n",
      "in vitro transcription and translation system\n",
      "expression and purification of dnak\n",
      "protein detection and quantitation by western blot\n",
      "dot-blot assays\n",
      "giant vesicles preparation\n",
      "visualizing guvs with fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300023h/sb300023h.xml\n",
      "methods\n",
      "materials\n",
      "synthesis of lipidoids\n",
      "cell culture\n",
      "in vitro gene transfection\n",
      "cytotoxicity of unsaturated lipidoids (mtt assay)\n",
      "transmission electron microscopy (tem)\n",
      "gel retardation and heparin competition assay\n",
      "ethidium bromide (eb) exclusion assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00123/sb8b00123.xml\n",
      "results\n",
      "library design and screening of protein binders to light-switchable domains\n",
      "characterization of light-dependent protein–protein interactions with purified proteins\n",
      "characterization of light-switchable binding in live cells\n",
      "discussion\n",
      "methods\n",
      "phagemid vectors\n",
      "construction of libraries\n",
      "ga domain\n",
      "expression and purification of cpyp and aslov2 proteins\n",
      "phage panning of naïve libraries\n",
      "conversion of phage pool to piii fusion display and panning\n",
      "expression and purification of protein binders\n",
      "uv–vis measurements\n",
      "nmr measurements\n",
      "modeling of interactions between cpyp-m121e and bopd\n",
      "mammalian cell expression, imaging, and photoactivation\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00081/sb7b00081.xml\n",
      "quantitative tracking of combinatorially engineered populations with multiplexed binary assemblies\n",
      "controlling multicycle replication of live-attenuated hiv-1 using an unnatural genetic switch\n",
      "cell-free translation is more variable than transcription\n",
      "in vitro reconstitution and optimization of the entire pathway to convert glucose into fatty acids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00255/sb6b00255.xml\n",
      "methods\n",
      "reverse-phase hplc analysis\n",
      "aba-efs stability test\n",
      "cloning and plasmid construction\n",
      "cell culture and transfection\n",
      "flow cytometry analysis\n",
      "fluorescence microscopy and statistical analysis\n",
      "chemical synthesis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300128r/sb300128r.xml\n",
      "results and discussion\n",
      "targeting novel biosynthetic enzymes to the chloroplast\n",
      "demonstration of light-dependent enzyme p450 activity\n",
      "expression of the entire dhurrin pathway in the chloroplast\n",
      "in vivo production of dhurrin\n",
      "electron donors to the p450s in the chloroplast\n",
      "methods\n",
      "vector construction\n",
      "vector construction\n",
      "agrobacterium infiltration of n. benthamiana\n",
      "thylakoid membrane isolation\n",
      "chloroplast isolation and fractionation\n",
      "immunoblotting\n",
      "enzyme activity assays in the thylakoid membrane\n",
      "cyp79a1 activity assay\n",
      "cyp71e1 activity assay\n",
      "enzyme activity assays in the chloroplast\n",
      "ugt85b1 activity assay\n",
      "lc–ms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400160m/sb400160m.xml\n",
      "results and discussion\n",
      "operon design and function\n",
      "mock operon selection\n",
      "ligand-specific operon selection\n",
      "methods\n",
      "synthesis of operons and conjugates\n",
      "operon synthesis\n",
      "cross-linking bap with operon dna\n",
      "in vitro transcription and translation (ivtt)\n",
      "in vitro compartmentalization selection scheme\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400087e/sb400087e.xml\n",
      "methods\n",
      "in vitro transcription and translation systems\n",
      "microchamber fabrication\n",
      "observations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400016j/sb400016j.xml\n",
      "results and disscussion\n",
      "general strategy\n",
      "gfp reporter expression as an indicator of mat a or mat α mating-type\n",
      "blocking the mating abilities of haploids via constitutive expression of the a1 or α2 gene\n",
      "specific expression of the α2 gene in mat a cells to avoid undesirable repression of reporter gene expression\n",
      "artificial regulation network designed for conversion of mating-type without autopolyploidization\n",
      "conversion of mating-type from mat α into mat a\n",
      "conversion of mating-type from mat a into mat α\n",
      "conclusion\n",
      "methods\n",
      "strains and media\n",
      "fluorescent reporter assay\n",
      "diploid growth assay\n",
      "conversion of mating-type\n",
      "ploidy analysis using real-time pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000542/sb4000542.xml\n",
      "results and discussion\n",
      "generation of randomized one-gene circuits\n",
      "optimizing one-gene and three-gene assembly reactions\n",
      "generation of three-gene randomized circuits\n",
      "generation of randomized lycopene biosynthesis pathways\n",
      "general conclusions and future directions\n",
      "methods\n",
      "parts, vectors, and plasmid engineering\n",
      "pcr and randomized assembly reactions\n",
      "transformations\n",
      "functional screening of positive clones and assembly success rates\n",
      "circuit and pathway characterization\n",
      "cmy color visualization and inducer experiments\n",
      "lycopene extractions\n",
      "strain availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500055u/sb500055u.xml\n",
      "results and discussion\n",
      "design principles for cloning pipeline\n",
      "application of the standardized cloning pipeline to protoporphyrin ix production\n",
      "standardized assembly of multiple fragments\n",
      "assembly of tetrapyrrole core biosynthesis genes\n",
      "chemical characterization of stage-one clones\n",
      "quantitative analysis of stage-one clones\n",
      "building on stage-one clones for increased porphyrin ix production\n",
      "concluding remarks\n",
      "methods\n",
      "strains and media\n",
      "pcr and uracil-excision cloning\n",
      "plasmid construction\n",
      "35 s-labeling methionine labeling of proteins\n",
      "screening for porphyrin production\n",
      "assay conditions and extraction procedure\n",
      "analytical method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00212/sb9b00212.xml\n",
      "results and discussion\n",
      "selection of naringenin biosynthesis genes\n",
      "modular assembly of naringenin biosynthetic libraries\n",
      "selection of platform strains\n",
      "design of the naringenin sensor in s. cerevisiae\n",
      "characterization of the naringenin sensor in s. cerevisiae\n",
      "in vivo screening application of naringenin biosensor in s. cerevisiae\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "chemicals and cultivation conditions\n",
      "dna manipulation\n",
      "characterization of naringenin sensor in s. cerevisiae\n",
      "library construction and high-throughput screening\n",
      "metabolite quantification using hplc\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000035/sb3000035.xml\n",
      "results\n",
      "regulating transcription by steric hindrance from a fluorescent protein\n",
      "the sensor is activated upon mirna induction\n",
      "a sensor designed to detect mir-34 is activated in human osteosarcoma cells over a short time scale\n",
      "conclusion\n",
      "methods\n",
      "constructs\n",
      "cell culture and transfections\n",
      "flow cytometry\n",
      "quantitative real-time reverse transcription pcr of mir-luc and yfp detection\n",
      "single cell fluorescence assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00134/sb7b00134.xml\n",
      "results and discussion\n",
      "characterization of the insulation effect on different promoters\n",
      "insulation effect on performance of synthetic gene circuits\n",
      "methods\n",
      "reagents and enzymes\n",
      "plasmid dna constructs\n",
      "cell culture and transfection\n",
      "facs measurement\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00191/sb9b00191.xml\n",
      "results\n",
      "analysis of conserved elements in benr and xyls close homologues and target promoters\n",
      "single amino acid position is critical for aromatic recognition in benr and xyls\n",
      "transcriptional factors responsive to a new set of aromatic compounds and aspirin\n",
      "promoter engineering further enhances x-benr-a111v response to nonoptimal inducers\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "gfp fluorescence assay and data processing\n",
      "flow cytometry analysis\n",
      "3d structure model construction and docking analysis\n",
      "molecular docking\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00321/sb6b00321.xml\n",
      "results and discussion\n",
      "the tunability of p rhabad -mediated protein production in e. coli depends on l -rhamnose consumption\n",
      "precisely setting p rhabad -based protein production rates in e. coli\n",
      "optimizing p rhabad -based production yields of the membrane protein gltp using e. coli rhab′δrhat\n",
      "optimizing p rhabad -based production of a single-chain variable fragment using e. coli rhab′δrhat\n",
      "an e. coli rhab′δrhat -based membrane and secretory protein production screen\n",
      "summary\n",
      "methods\n",
      "strains, plasmids and culture conditions\n",
      "whole cell fluorescence measurements and flow cytometry\n",
      "sds-page, in-gel fluorescence and immunoblotting\n",
      "bl1 activity measurement\n",
      "isolation of gltp-gfp and gltp activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00285/sb8b00285.xml\n",
      "results and discussion\n",
      "rational design and evaluation of selected endogenous terminators\n",
      "synthetic terminator design and evaluation\n",
      "terminators exhibit functionality across different genomic contexts\n",
      "terminators exhibit functionality in a different host cell line\n",
      "extracting key composition features leading to mammalian terminator function\n",
      "terminators function through increased mrna stability\n",
      "designed promoter-terminator elements can outperform conventional, viral-derived sequences\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction and preparation for cell culture\n",
      "cell culture and transient transfections\n",
      "expression quantification using flow cytometry and chemiluminescent reporter assay\n",
      "experimental conditions for half-life determination\n",
      "software for data analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00034/sb7b00034.xml\n",
      "results and discussion\n",
      "3d modeling of the ketoacyl synthase domain from yarrowia lipolytica fasi\n",
      "design of talen targeting position 1220\n",
      "site-directed mutagenesis by homologous recombination\n",
      "fatty acid specificity of i1220 mutants\n",
      "discussion\n",
      "theoretical and experimental procedures\n",
      "computational methods\n",
      "3d protein modeling\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sequence analysis\n",
      "experimental methods\n",
      "media\n",
      "talen design and plasmids\n",
      "matrix design\n",
      "strain and transformation\n",
      "pcr screening on genomic dna\n",
      "cultures, lipids extraction and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400033c/sb400033c.xml\n",
      "divita mathur\n",
      "seema nagaraj\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00230/sb6b00230.xml\n",
      "results and discussion\n",
      "functional elements, layout, and design of glycolytic modules i and ii\n",
      "the activities encoded in the glucobricks restore or enhance the growth of glycolytic escherichia coli mutants\n",
      "the activities encoded by glucobricks are instrumental for engineering glycolysis in two pseudomonas species\n",
      "impact of the glucobrick platform in the intracellular metabolome of p. putida kt2440\n",
      "enhancing heterologous phb production by boosting glycolytic activity in recombinant e. coli\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and culture conditions\n",
      "dna manipulation and sequencing, and construction of mutant strains\n",
      "design and assembly of the glucobrick modules\n",
      "preparation of cell-free extracts and in vitro enzymatic assays\n",
      "analytical determinations\n",
      "determination of intracellular metabolite concentrations\n",
      "statistical analysis\n",
      "nucleotide sequence accession numbers\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00322/sb7b00322.xml\n",
      "results\n",
      "star-mediated activation and asrna-mediated deactivation of gene expression\n",
      "influences of genetic parts on deactivation efficiency\n",
      "improved deactivation efficiency by integrating the two types of asrnas\n",
      "real-time activation and deactivation of gene expression\n",
      "regulation of a gene in the genome\n",
      "multilevel expression control of multiple genes\n",
      "discussion\n",
      "methods\n",
      "strains and culture media\n",
      "plasmid construction and genetic circuit design\n",
      "integration of genetic parts into the e. coli genome\n",
      "fluorimetry\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00295/sb6b00295.xml\n",
      "results and discussion\n",
      "csy4 represses transgene expression through transcript cleavage\n",
      "csy4 allows synchronized repression of transgenes\n",
      "csy4 allows conditional repression of transgene\n",
      "csy4 can be used for cell-type-specific repression of transgene\n",
      "csy4 can repress transgene expression in rice\n",
      "csy4 expression does not affect plant growth\n",
      "csy4 orthologs showed transgene repression activity in tobacco leaves\n",
      "conclusion\n",
      "methods\n",
      "construction of multigene plasmids\n",
      "agrobacterium-mediated transient leaf transformations\n",
      "whole leaf imaging for fluorescent signals\n",
      "rt-pcr and real-time pcr analysis\n",
      "activity detection of firefly luciferase\n",
      "arabidopsis plant growth and transformation\n",
      "identification of csy4 orthologs\n",
      "transcript blast in plant gdb\n",
      "rice protoplast isolation and transformation\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00123/sb6b00123.xml\n",
      "results and discussion\n",
      "the effect of the homologous end’s length on multigene assembly\n",
      "tuning single gene expression with regulatory linkers\n",
      "engineering violacein synthesis pathway with regulatory linkers\n",
      "methods\n",
      "strain and media\n",
      "preparation of dna materials\n",
      "yeast transformation\n",
      "linker synthesis and dna verification of assembled results\n",
      "measurement of fluorescence\n",
      "hplc detection of violacein products\n",
      "determination of transcriptional levels by qpcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00048/sb5b00048.xml\n",
      "resutls and discussion\n",
      "membranous cry2 oligomerizes much more readily than its cytosolic form\n",
      "the oligomerization of membranous cry2 is highly dynamic and reversible\n",
      "membranous cry2 oligomerization is concentration dependent and occurs readily at low level of light\n",
      "cry2 oligomerization and cry2-cib1 heterodimerization coexist under blue light activation\n",
      "cry2 oligomerization can be modulated by cry2-cib1 heterodimerization\n",
      "cry2 oligomerization can be enhanced through cry2-cib1 interaction\n",
      "conclusions and discussion\n",
      "materials and methods\n",
      "plasmids\n",
      "cell cultures and transfection\n",
      "live-cell imaging\n",
      "image processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500033d/sb500033d.xml\n",
      "results and discussion\n",
      "methods\n",
      "design of an srna to target antibiotic resistance\n",
      "construction of phage and antibiotic resistant strains\n",
      "antibiotic sensitivity assay\n",
      "phage infection assay\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00261/sb7b00261.xml\n",
      "results and discussion\n",
      "combinatorial deletion of irrelevant gene clusters\n",
      "phenotypic characterization of prechassis (pc)\n",
      "building and testing of c. glutamicum chassis (c)\n",
      "genome resequencing of pc1, pc2 and c1\n",
      "transcriptome analysis of c1 against mb001\n",
      "reversion of the mutation in the rama promoter in chassis c1\n",
      "revised c. glutamicum genome annotation linked to literature\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids and deletion of gene clusters\n",
      "growth experiments\n",
      "genome resequencing\n",
      "transcriptome analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000047/sb3000047.xml\n",
      "computer-aided dna-assembly design\n",
      "a new toolbox for cell-free synthetic biology\n",
      "bacterial genomes as yeast centromeric plasmids\n",
      "artificial small rnas for conditional gene silencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00421/sb8b00421.xml\n",
      "results and discussion\n",
      "incorporation efficiency of cr-oas using different synthetase variants\n",
      "modification of different proteins via staudinger ligation with dylight650\n",
      "chemoselective fluorescence labeling of synthesized proteins using further cr-oas\n",
      "conclusion\n",
      "methods\n",
      "non-canonical amino acids and reactive dyes\n",
      "protein expression vectors\n",
      "in vitro transcription of trna pyl cua\n",
      "cell-free expression and incorporation of cr-oas\n",
      "quantification of synthesized proteins by trichloroacetic acid (tca) precipitation\n",
      "chemoselective protein labeling\n",
      "protein analysis\n",
      "microscale thermophoresis (mst) measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000059/sb5000059.xml\n",
      "results and discussion\n",
      "using the chemical master equation to derive moment equations\n",
      "dynamic noise propagation equations\n",
      "strategy for using dynamic noise equations to predict causal relationships in a circuit\n",
      "estimation of necessary parameters\n",
      "test using in silico data\n",
      "in vivo test using synthetic circuits\n",
      "conclusion\n",
      "methods\n",
      "plasmids and strain construction\n",
      "growth and fluorescence measurements by flow cytometry\n",
      "flow cytometry data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400094c/sb400094c.xml\n",
      "results and discussion\n",
      "emre synthesis inside cell-sized vesicles\n",
      "etbr transport by emre-displaying vesicles\n",
      "quantification of intravesicular emre synthesis\n",
      "dependence of emre integration on vesicle volume\n",
      "modeling the emre membrane integration\n",
      "conclusions\n",
      "methods\n",
      "plasmid construction\n",
      "preparation of cell-sized vesicles\n",
      "facs analysis\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00161/sb6b00161.xml\n",
      "designs for the most robust stochastic gene oscillators\n",
      "genotype specification language\n",
      "merlin: computer-aided oligonucleotide design for large scale genome engineering with mage\n",
      "semisupervised gaussian process for automated enzyme search\n",
      "sharing structure and function in biological design with sbol 2.0\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00104/sb8b00104.xml\n",
      "insertion of the channel protein ompf\n",
      "enzyme encapsulation inside the cavities of guvs\n",
      "enzyme reaction inside guv cavities\n",
      "guv size distribution\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00099/sb8b00099.xml\n",
      "results and discussion\n",
      "methods\n",
      "reagents\n",
      "bacterial strains and growth conditions\n",
      "library construction and plasmid cloning\n",
      "dual genetic selection and fluorescence-activated cell sorting\n",
      "fluorescence assays\n",
      "beta-galactosidase assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00061/sb6b00061.xml\n",
      "results and discussion\n",
      "enhancing the performance of the biosensor through random mutagenesis of dmpr\n",
      "dose–response analysis of dmpr mutants dm01 and dm12\n",
      "construction of op sensor plasmid pek-dtor and performance of dm01 and dm12 in response to op pesticides\n",
      "performance of dmpr mutants in membrane-defective e. coli mutant strains toward 4-nitrophenol and op pesticides\n",
      "performance of agar-immobilized dmpr mutants in response to fenitrothion\n",
      "discussion\n",
      "methods\n",
      "strains, media, and chemicals\n",
      "construction of plasmids\n",
      "random mutagenesis of dmpr and construction of mutant libraries\n",
      "selection for dmpr mutants forming red colonies in the presence of 4-nitrophenol\n",
      "gene deletion of rfad from e. coli mg1655\n",
      "dose–response analysis of dmpr mutants for 4-nitrophenol\n",
      "exposure of a bacterial culture of dmpr mutants to op pesticides\n",
      "exposure of agar-immobilized dmpr mutants to op pesticides\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000113/sb3000113.xml\n",
      "results and discussion\n",
      "in vitro autogene schema\n",
      "implementation and optimization to achieve self-amplification\n",
      "adaptation to emulsion\n",
      "selection\n",
      "methods\n",
      "template preparation\n",
      "in vitro transcription and translation reactions in solution\n",
      "in vitro transcription and translation reactions in emulsion\n",
      "mrna purification\n",
      "mrna quantitation\n",
      "template regeneration\n",
      "cloning and sequencing\n",
      "t7 rna polymerase activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400004q/sb400004q.xml\n",
      "promoters from biobrick fusions\n",
      "analysis of engineered shewanella oneidensis to bind gold electrodes\n",
      "self-assembly of synthetic metabolons\n",
      "reducing codon redundancy and screening efforts\n",
      "synthesizing biomolecule-based boolean logic gates\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00217/sb9b00217.xml\n",
      "results and discussion\n",
      "construction of an engineered s. cerevisiae capable of producing vitamin a from xylose\n",
      "comparison of vitamin a production patterns on glucose and xylose\n",
      "improved vitamin a production by two-phase in situ extraction\n",
      "fed-batch fermentation for the production of vitamin a by engineered yeast\n",
      "conclusions\n",
      "materials and methods\n",
      "plasmids and strain construction\n",
      "batch and fed-batch culture for vitamin a production\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00132/sb6b00132.xml\n",
      "results and discussion\n",
      "shikimate pathway and rate-limiting precursor in yeasts\n",
      "identification of strong constitutive promoters and terminators via transcriptome analysis\n",
      "promoter characterization and length optimization\n",
      "terminator discovery and characterization\n",
      "shikimate production in s. stipitis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "materials and methods\n",
      "s. stipitis cultivation and rna isolation\n",
      "plasmid construction and transformation\n",
      "terminator characterization via qpcr\n",
      "promoter and terminator characterization using egfp assay\n",
      "shikimate production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00139/sb7b00139.xml\n",
      "application of an acyl-coa ligase from streptomyces aizunensis for lactam biosynthesis\n",
      "an engineered survival-selection assay for extracellular protein expression uncovers hypersecretory phenotypes in escherichia coli\n",
      "refactoring the embden–meyerhof–parnas pathway as a whole of portable glucobricks for implantation of glycolytic modules in gram-negative bacteria\n",
      "biophysics of artificially expanded genetic information systems: thermodynamics of dna duplexes containing matches and mismatches\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00265/sb7b00265.xml\n",
      "results and discussion\n",
      "phospholipid biosynthesis pathway design\n",
      "formation of phosphatidic acid in detergent solution\n",
      "reconstitution of phosphatidic acid formation\n",
      "reconstitution of polar headgroup attachment\n",
      "reconstitution of a mixed phospholipid membrane\n",
      "membrane expansion by phospholipid biosynthesis\n",
      "conclusions\n",
      "methods\n",
      "bacterial strain and cloning procedures\n",
      "expression and purification of phospholipid synthesizing enzymes\n",
      "liposomes preparation\n",
      "in vitro assays for phospholipid production\n",
      "fluorescent assays for membrane expansion\n",
      "cryotransmission electron microscopy\n",
      "lc–ms analysis of lipids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00456/sb7b00456.xml\n",
      "methods\n",
      "strains and plasmids\n",
      "construction of dna fragments and plasmids\n",
      "dna transformation\n",
      "quantitative reverse transcription pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00087/sb9b00087.xml\n",
      "material and methods\n",
      "reagents and materials\n",
      "replacement pcr for the conversion from ds to natural bases\n",
      "deep sequencing\n",
      "sequence data analysis of ndsn2–49 and ndsn3–49\n",
      "sequence data analysis using enriched libraries obtained by exselex\n",
      "receiver operating characteristic (roc) curve analysis\n",
      "results and discussion\n",
      "making an encyclopedia of natural-base composition rates by replacement pcr for all of the sequence contexts around ds\n",
      "intermediate ub substrates for replacement pcr\n",
      "preparation of two sets of encyclopedias of replacement pcr for each sequence context (enbre)\n",
      "evaluation of the sequencing method using ub-dna aptamer sequences from enriched libraries obtained by exselex\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00398/sb8b00398.xml\n",
      "results and discussion\n",
      "development of an automated screening pipeline\n",
      "translational tuning of gpps and lims\n",
      "modeling of the sequence–phenotype relationship\n",
      "translational tuning of an mva pathway\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and media\n",
      "plasmid design\n",
      "rbs variant library design\n",
      "introduction of rbs libraries\n",
      "limonene production\n",
      "limonene quantification\n",
      "machine learning\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00089/sb9b00089.xml\n",
      "results\n",
      "evaluation\n",
      "usage\n",
      "distributed search\n",
      "system overview\n",
      "methods\n",
      "discussion\n",
      "infrastructure\n",
      "data augmentation\n",
      "data visualization\n",
      "better metrics\n",
      "integration of more data repositories\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00281/sb6b00281.xml\n",
      "results and discussion\n",
      "cell growth and metabolites of the pdc1 / adh1 -deleted strain yph499/dpdaw\n",
      "global metabolic engineered yeast strain with high glucose consumption rate\n",
      "cell growth, metabolites, and copy numbers and transcription levels of glycolysis-related genes of the engineered yeast strain\n",
      "conclusions\n",
      "methods\n",
      "strains and media\n",
      "gene deletion in yeast\n",
      "plasmid construction and yeast transformation\n",
      "construction of yeast strains with high glucose consumption rate using global metabolic engineering\n",
      "analysis of growth and metabolites\n",
      "real-time pcr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00448/sb7b00448.xml\n",
      "results and discussion\n",
      "construction of cas9-meditated plasmid for genome editing in s. erythraea\n",
      "temperature-sensitive psg5 replicon controls egfp expression\n",
      "targeting sgrna determination\n",
      "target-specific cleavage of sace_1765 by cas9 and sgrna\n",
      "crispr-cas9-mediated deletion of inda by dual sgrna targeting\n",
      "crispr-cas9-mediated knock-in of perme- egfp at the sace_0712 locus\n",
      "methods\n",
      "microorganisms and culture medium\n",
      "construction of flag- egfp fusion temperature-sensitive vector\n",
      "fluorescence microscopy\n",
      "predictive model of sgrna activity and crispr target design\n",
      "construction of an effective crispr-cas9 platform for s. erythraea\n",
      "calculation of deletion efficiencies\n",
      "mutations of genomic analysis and sequencing\n",
      "quantitative real-time pcr analysis\n",
      "western blot analysis\n",
      "growth analysis and erythromycin yield evaluation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400206c/sb400206c.xml\n",
      "results and discussion\n",
      "rna transcriptional cascades function in tx-tl\n",
      "ideal tx-tl batch characteristics for circuit testing\n",
      "characterizing the dynamics of rna circuitry with tx-tl\n",
      "cascade response time can be tuned by using tandem attenuators\n",
      "theophylline responsive antisense provides a bridge to move rna circuitry in vivo\n",
      "an rna sim functions in vivo\n",
      "conclusions\n",
      "methods\n",
      "plasmid construction and purification\n",
      "tx-tl extract and buffer preparation\n",
      "extract preparation\n",
      "buffer preparation\n",
      "tx-tl experiment\n",
      "strains, growth media and in vivo gene expression\n",
      "response time calculation\n",
      "tx-tl experiments\n",
      "in vivo experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00100/sb8b00100.xml\n",
      "results and discussion\n",
      "plasmid construction\n",
      "amber suppression toxicity\n",
      "amber suppression screening and verification\n",
      "amber suppression efficiency comparison\n",
      "methods\n",
      "plasmid construction and cell strains\n",
      "amber suppression screening and efficiency\n",
      "mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00107/sb7b00107.xml\n",
      "zehua bao\n",
      "ran chao\n",
      "fabio chizzolini\n",
      "brady f. cress\n",
      "xuefeng cui\n",
      "jacob a. englaender\n",
      "michele forlin\n",
      "xin gao\n",
      "murtaza shabbir hussain\n",
      "j. andrew jones\n",
      "myeong-gyun kang\n",
      "mattheos koffas\n",
      "hiroyuki kuwahara\n",
      "ashwin lahiry\n",
      "richard a. lease\n",
      "jing liang\n",
      "youyun liang\n",
      "zihe liu\n",
      "david r. mcmillen\n",
      "samuel d. stimple\n",
      "ramzan umarov\n",
      "nanxi wang\n",
      "zhe yuan\n",
      "yuchen zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00422/sb7b00422.xml\n",
      "methods\n",
      "plasmid construction and purification\n",
      "el222 purification\n",
      "illumination setup\n",
      "cell-free characterization experiments\n",
      "model development and parametrization\n",
      "root-mean-square deviation (rmsd)\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003554/sb5003554.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500260k/sb500260k.xml\n",
      "results and discussion\n",
      "promoter library based on the pcc 6803 cpcb promoter\n",
      "development of a synthetic promoter library in pcc 7002\n",
      "redesigning the p cpt promoter for iptg induction\n",
      "increasing the intrinsic strength of p cpcb -based induction system\n",
      "comparison with trc promoters\n",
      "design and construction of a pcc 7002 rbs library\n",
      "conclusions\n",
      "methods\n",
      "chemicals, reagents, and media\n",
      "strain construction\n",
      "measurement of fluorescence\n",
      "construction of a promoter library based on the pcc 6803 p cpcb promoter\n",
      "construction of a synthetic promoter library\n",
      "construction of the iptg inducible pcc 7002 strains\n",
      "design and construction of the rbs library\n",
      "qpcr analysis of the rbs library in pcc 7002\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00052/sb8b00052.xml\n",
      "results and discussion\n",
      "laci-based biosensor\n",
      "a single mutation in biosensor l1.0 is sufficient for biosensor activity\n",
      "transcriptional activation of the l1.7 biosensor\n",
      "tuning the biosensor response\n",
      "a mutation altering laci ligand-specificity results in improved response to iptg and d -fucose\n",
      "ligand-dependent stabilization is a general method for biosensor engineering\n",
      "concluding comments\n",
      "materials and methods\n",
      "enzymes and reagents\n",
      "strains and media\n",
      "preparation of library\n",
      "e. coli transformation with library and positive selection for his3 expression\n",
      "replica-plating to identify variants with ligand-dependent growth\n",
      "measurement of biosensor growth rates\n",
      "analysis of growth rates\n",
      "assessing biosensor transcriptional activation\n",
      "assessing biosensor level by western blot\n",
      "positive and negative selection to identify mphr biosensors\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00197/sb5b00197.xml\n",
      "results and discussion\n",
      "ifrs recognizes trna pyl variants independent of their anticodons\n",
      "intracellular aminoacylation of trna pyl variants\n",
      "reassignment of targeted ser agu codons in sfgfp with 3-i-phe\n",
      "quantification of amino acid incorporation at a single ser agu site\n",
      "reassignment of the most frequent codon in the ser ucn box\n",
      "reassignment of codons with the highest codon usage\n",
      "conclusions\n",
      "materials and methods\n",
      "purification of proteins\n",
      "preparation of radiolabeled m. mazei trna pyl anticodon variants\n",
      "biochemical characterization of the ifrs/trna pyl pair\n",
      "northern blot analyses of intracellular aminoacyl-trna\n",
      "reporter protein sfgfp overexpression and purification\n",
      "asp-n digestion\n",
      "capillary lc mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00067/sb7b00067.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "kirill alexandrov\n",
      "andreas angermayr\n",
      "filipe branco dos santos\n",
      "zhenling cui\n",
      "wei du\n",
      "shuichi hoshika\n",
      "nicolai kallscheuer\n",
      "myong-jung kim\n",
      "claudia kniely\n",
      "quanfeng liang\n",
      "ishna mistry\n",
      "ewa maria musiol-kroll\n",
      "ernst oberortner\n",
      "rui miguel correia portela\n",
      "anam qudrat\n",
      "brandon razooky\n",
      "jeong sang yi\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400050k/sb400050k.xml\n",
      "results and discussion\n",
      "rationale for creating logic gates based on xylr\n",
      "genetic grafting of protease-cleaving sites through the xylr structure\n",
      "analysis of nia-tagged xylr variants\n",
      "novel boolean logic of xylr d210 and xylr e499\n",
      "pu promoter anti-activation: engineering a cleavable variant of xylrδa\n",
      "conclusion\n",
      "methods\n",
      "strains, plasmids, media, and growth conditions\n",
      "dna constructs\n",
      "in vitro transposition and construction of knock-in-leave-behind (kilb) insertion libraries\n",
      "monitoring promoter activity in vivo\n",
      "western blot analyses of xylr expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00483/sb8b00483.xml\n",
      "results and discussion\n",
      "selection of appropriate plant enzymes for n -methylpyrrolinium biosynthesis\n",
      "de novo production of n -methylpyrrolinium in e. coli\n",
      "de novo production of n -methylpyrrolinium in s. cerevisiae\n",
      "optimization of n -methylpyrrolinium production in s. cerevisiae\n",
      "conclusion\n",
      "methods\n",
      "protein purification\n",
      "in vitro enzymatic assays\n",
      "de novo production of n -methylpyrrolinium in e. coli\n",
      "de novo production of n -methylpyrrolinium in s. cerevisiae\n",
      "lc–hrms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300104f/sb300104f.xml\n",
      "methods\n",
      "dna linear template preparation and sequences\n",
      "reaction setup and rna and egfp measurements\n",
      "preparation of pure mrna and qrt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00265/sb6b00265.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500029n/sb500029n.xml\n",
      "results\n",
      "synthetic tandem promoter has higher expression strength\n",
      "crispr/dcas9 repression system widens the range of expression levels\n",
      "discussion\n",
      "methods\n",
      "plasmids and strains\n",
      "grna repression\n",
      "fluorescence measurement and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300036z/sb300036z.xml\n",
      "the evolution of the center\n",
      "dcsb milestones\n",
      "developing collaborations that work\n",
      "fostering scientific exchange at duke and beyond\n",
      "training\n",
      "ongoing challenges\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00082/sb6b00082.xml\n",
      "results and discussion\n",
      "cas9-sgrna rnps for efficient hdr based gene replacements\n",
      "dsbs by cas9 allows for efficient hdr using 60 bp oligonucleotide recombination flanks\n",
      "marker-free genome editing with crispr/cas9\n",
      "ama1 based expression of cas9 and sgrna\n",
      "ama1 based expression of cas9 and in vitro synthesized sgrna\n",
      "deletion of an integrated amds marker with transiently expressed cas9\n",
      "marker-free deletions with rnp’s\n",
      "conclusions\n",
      "methods\n",
      "fungal strains and genetic targets\n",
      "selection of protospacers for sgrna and construction of sgrna templates\n",
      "construction of deletion plasmids and ddna\n",
      "construction of ama1 based vectors\n",
      "construction of cas9 and sgrna expression vectors\n",
      "cas9 purification\n",
      "in vitro sgrna synthesis and in vitro activity of cas9\n",
      "fungal transformations and analysis of transformants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00359/sb7b00359.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00213/sb5b00213.xml\n",
      "results\n",
      "a synthetic molecular signal buffer to effectively relay the riboswitch signal\n",
      "mathematical model of a synthetic signal buffer system\n",
      "molecular construction of the synthetic signal buffer system\n",
      "modular inversion of sensor signals\n",
      "using the synthetic buffer network for modular tuning of input sensitivity\n",
      "discussion\n",
      "methods\n",
      "colony formation response assays\n",
      "gfp response assays\n",
      "direct riboswitch response (gfp-tagging)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500352b/sb500352b.xml\n",
      "the motivation behind minieugene\n",
      "the minieugene web application\n",
      "step i“specify design length”\n",
      "step ii“specify design constraints”\n",
      "step iii“design generation”\n",
      "step iv“select design representation”\n",
      "the biological constraints provided by minieugene\n",
      "the programmatic integration of minieugene\n",
      "minieugene in use and future work\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00230/sb9b00230.xml\n",
      "results and discussion\n",
      "lepidopteran mevalonate pathway enzyme bioprospecting and plasmid design\n",
      "heterologous pathway expression in e. coli and small molecule analysis\n",
      "discussion\n",
      "methods\n",
      "chemicals\n",
      "dna synthesis and sources\n",
      "pcr and plasmid assemblies\n",
      "plasmid confirmation via miseq sequencing\n",
      "e. coli kcm competent cell preparation\n",
      "e. coli pathway expressions\n",
      "small scale samples\n",
      "large scale enrichment\n",
      "gc–ms analysis method\n",
      "gc–qtof analysis method\n",
      "proteomics\n",
      "method for novel terpene titer estimation\n",
      "method for novel homosesquiterpene to total terpene fraction estimation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00271/sb6b00271.xml\n",
      "synthetic scaffold design\n",
      "graph construction\n",
      "graph filtering\n",
      "sequence construction\n",
      "sequence optimization\n",
      "laboratory validation\n",
      "further characterization\n",
      "discussion\n",
      "methods\n",
      "software\n",
      "dna origami and rna–dna hybrid origami designs\n",
      "dna origami folding\n",
      "rna–dna hybrid origami folding\n",
      "atomic force microscope (afm)\n",
      "alexa 488 labeled dbs scaffold preparation\n",
      "electroporation and plug preparation\n",
      "lysis of cells in plugs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00321/sb9b00321.xml\n",
      "results\n",
      "creation of and logic using a downstream laci roadblock site\n",
      "point mutations in the laco site tune the extent of roadblock repression caused by laci, changing logic behavior\n",
      "and behavior is improved through tuning inducer concentrations\n",
      "mathematical modeling of rnap roadblock and development of transfer functions\n",
      "addition of transcriptional activity associated with a weak roadblock to the downstream tim creates or logic\n",
      "tuning inducer concentrations improves or gate behavior\n",
      "mathematical modeling of or gate behavior\n",
      "discussion\n",
      "methods\n",
      "strains, plasmids, and cell culture\n",
      "gfp induction assays\n",
      "flow cytometry\n",
      "mathematical characterization of logic behavior\n",
      "triangular and gate plots\n",
      "triangular or gate plots\n",
      "transfer function modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00365/sb8b00365.xml\n",
      "results\n",
      "isolation of single b cells from a vaccinated pig and mab generation\n",
      "mabs hk24 and hk44 exhibit high specificity and sensitivity to the csfv e2 protein\n",
      "evaluation of the binding affinity of mabs hk44 and hk24 to the e2 protein by surface plasmon resonance (spr)\n",
      "mabs hk24 and hk44 can bind to a specific epitope of the e2 protein\n",
      "mabs hk24 and hk44 significantly neutralize csfv in vitro\n",
      "discussion\n",
      "materials and methods\n",
      "cell lines and viruses\n",
      "single b cell sorting\n",
      "isolation and expression of the ig gene\n",
      "sds-page and western blotting analysis\n",
      "expression of the e2 protein\n",
      "detecting the affinity of mabs hk24 and hk44 for the csfv e2 protein via indirect elisa\n",
      "immunofluorescence to detect the binding specificity of mabs hk44 and hk24 to epitope 303\n",
      "competition binding assay\n",
      "surface plasmon resonance analysis\n",
      "neutralization assays\n",
      "quantification of csfv rna\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00054/sb6b00054.xml\n",
      "results and discussion\n",
      "atlas of biochemistry: known and novel reactions\n",
      "network generation and exact reconstruction of kegg reactions\n",
      "pathway search and biotransformation reconstruction\n",
      "example 1: one-step reconstructed biotransformation\n",
      "example 2: multistep reconstructed biotransformation\n",
      "validation of the generalized reaction rules\n",
      "integration of kegg compounds in de novo reactions\n",
      "reconsideration of biochemistry in the context of chemical knowledge\n",
      "ec number analysis\n",
      "thermodynamic analysis\n",
      "bridgit analysis of novel reactions\n",
      "description of the online database\n",
      "methods\n",
      "preprocessing of the kegg compound and reaction databases\n",
      "bnice.ch\n",
      "bnice.ch: generalized enzymatic reaction rules\n",
      "bnice.ch: network generation algorithm\n",
      "known and novel reactions: biological and chemical databases integrated\n",
      "reconstruction of kegg reactions\n",
      "exact reconstruction of kegg reactions\n",
      "biotransformation reconstruction\n",
      "bridgit analysis\n",
      "reaction characterization\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00295/sb5b00295.xml\n",
      "results\n",
      "sybiont ontology\n",
      "development of the sybiont knowledge base (sybiontkb)\n",
      "testing the competency of sybiont\n",
      "mining sybiontkb for biological parts\n",
      "mapping sybiont to the sbol data model\n",
      "discussion\n",
      "methods\n",
      "rdf graph representation of biological data\n",
      "building the ontological representation in owl\n",
      "mining sybiontkb\n",
      "constructing sbol parts\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500365m/sb500365m.xml\n",
      "methods\n",
      "biosensing protocol\n",
      "statistical analysis\n",
      "biosensor performance calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00038/sb6b00038.xml\n",
      "results and discussion\n",
      "heterologous expression of ltxa–c in anabaena 7120 results in ltxa production\n",
      "construction of a tar-capable vector (ppjav550) and cloning of a large (>40 kb) insert using λ/red recombination\n",
      "anabaena 7120 can produce ltxa at levels comparable to those of moorea producens\n",
      "nitrogen limitation effects ltxa production and/or accumulation\n",
      "promoter exchange results in increased ltxa production\n",
      "anabaena 7120 recognizes a variety of promoters from marine cyanobacterial bgcs\n",
      "ltxd does not form ltxb and ltxc\n",
      "conclusions\n",
      "methods\n",
      "microbiological techniques\n",
      "production, purification, and analysis of ltxa\n",
      "quantification by lc-ms/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001112/sb3001112.xml\n",
      "nucleic acid-based computation in cell-free systems\n",
      "nucleic acid enzyme-based computation systems\n",
      "nucleic acid enzyme-free computation systems\n",
      "protein-based computation in cell-free systems\n",
      "nucleic acid-based computation within cells\n",
      "network plasmid\n",
      "rna aptamer\n",
      "riboswitch\n",
      "ribozyme\n",
      "rna interference\n",
      "nanorobot\n",
      "amber suppressor trna\n",
      "orthogonal ribosome and orthogonal mrna pair\n",
      "intercellular networks\n",
      "protein-based computation within a cell\n",
      "transactivator-based gene circuits\n",
      "cid-based gene circuits\n",
      "cid-based post-translational circuits\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400166j/sb400166j.xml\n",
      "results and discussion\n",
      "rationale for the bi-hac vector system\n",
      "construction of the tet-o hac2 vector carrying a 3′neo-loxp cassette\n",
      "loading of the dsred transgene into the tet-o hac2 in cho cells\n",
      "mitotic stability of 21hac2 and tet-o hac2 vectors propagated in the same human host cells\n",
      "summary and conclusion\n",
      "methods\n",
      "construction of targeting vectors\n",
      "cell culture\n",
      "fluorescence in situ hybridization (fish)\n",
      "microcell-mediated chromosome transfer (mmct)\n",
      "real-time pcr analysis\n",
      "facs analysis\n",
      "genomic pcr analysis\n",
      "elimination assay of the tet-o hac\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00142/sb7b00142.xml\n",
      "jesús f. barajas\n",
      "yves berset\n",
      "elleke f. bosma\n",
      "raj chari\n",
      "alejandro chavez\n",
      "george m. church\n",
      "behnam enghiad\n",
      "jin hou\n",
      "jason d. kahn\n",
      "yan liang\n",
      "ioannis mougiakos\n",
      "alberto sánchez-pascuala\n",
      "richard van kranenburg\n",
      "jing wang\n",
      "xiaoyu wang\n",
      "nan cher yeo\n",
      "jingwei zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00160/sb5b00160.xml\n",
      "crispr-cas9 based engineering of actinomycetal genomes\n",
      "regulation of bacterial gene expression by protease-alleviated spatial sequestration\n",
      "crispathbrick: modular combinatorial assembly of type ii-a crispr arrays for transcriptional repression in e. coli\n",
      "a highly characterized yeast toolkit for modular, multipart assembly\n",
      "influence of electrostatics on small molecule flux through a protein nanoreactor\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00224/sb6b00224.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results\n",
      "discussion\n",
      "materials and methods\n",
      "design of the oligonucleotides studied\n",
      "oligonucleotide synthesis\n",
      "uv absorbance melting curves and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00304/sb7b00304.xml\n",
      "results and discussion\n",
      "comparison of different product export scenarios\n",
      "stoichiometric limitations of the different pathways\n",
      "pathway limitations and impact of compartment ph on gibbs energies of reactions\n",
      "thermodynamic feasibility of elementary flux modes and impact on maximum carbon yields\n",
      "pathway robustness and consequences for commercial application\n",
      "methods\n",
      "assessment of cellular transport processes\n",
      "biochemical pathways to adipic acid\n",
      "muconic acid production via the shikimate pathway\n",
      "glucaric acid production via inositol\n",
      "production via the 3-oxoadipate pathway\n",
      "production via the 2-oxoadipate pathway\n",
      "production via lysine degradation\n",
      "production via fatty acid metabolism\n",
      "stoichiometric networks of the central carbon metabolism\n",
      "determination of feasibility limits in dependence on ph\n",
      "network-embedded thermodynamic analysis of elementary flux modes\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00338/sb8b00338.xml\n",
      "results and discussion\n",
      "availability\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00324/sb8b00324.xml\n",
      "results and discussion\n",
      "benchmarking alternate dna binding under wild-type (i + ) allosteric control\n",
      "engineering non-natural operator dna binding with alternate allosteric phenotypes (i a adr )\n",
      "assessment of orthogonal gene suppression imposed by engineering antilacs with adr\n",
      "assessment of increased signal concentration on i a adr suppression\n",
      "effect of operator number and symmetry on i a adr performance\n",
      "assessment of full orthogonality: engineering single-signal toggle switches\n",
      "conclusion\n",
      "materials and methods\n",
      "construction of laci mutants and operator variants\n",
      "microplate assay for transcription factor operator unit pair phenotyping\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00044/sb9b00044.xml\n",
      "results and discussion\n",
      "conclusions\n",
      "experimental section\n",
      "materials\n",
      "in vitro transcription\n",
      "preparation of tna templates by primer extension\n",
      "rna-catalyzed polymerization\n",
      "analysis of polymerization products\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00212/sb5b00212.xml\n",
      "standards\n",
      "annotations\n",
      "results and discussion\n",
      "four-input and gate\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00222/sb5b00222.xml\n",
      "standard workflows\n",
      "information and data standards\n",
      "sequence description (description of genetic objects of interest)\n",
      "modeling (representing the actual or desired behavior of genetic objects)\n",
      "raw data acquisition (learning about the genetic objects)\n",
      "results and discussion\n",
      "motivation: data acquisition is the missing standard\n",
      "dicom and dicom-sb\n",
      "dicom for synthetic biology\n",
      "encoding cell population measurements\n",
      "encoding single cell measurements\n",
      "supporting communication\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400203p/sb400203p.xml\n",
      "results and discussion\n",
      "combined transcription–translation reporter\n",
      "constitutive gene expression under standard conditions\n",
      "performance as a function of promoter strength and dna concentration\n",
      "role of ntps\n",
      "performance of a simple transcription–translation cascade\n",
      "resource utilization and crosstalk between orthogonal genes\n",
      "relevance for in vivo\n",
      "on biocircuit prototyping\n",
      "methods\n",
      "cell-free system and reactions\n",
      "reporters\n",
      "plasmids and bacterial strains\n",
      "preparation of pure mrna and qrt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00328/sb7b00328.xml\n",
      "results\n",
      "orthogonal and gate circuit design and rna-seq assays\n",
      "circuit metabolic load increases with its copy number in the host\n",
      "plasmid copy number outweighs circuit composition for effect on host gene expression\n",
      "copy number variation caused contrary changes in circuit components\n",
      "discussion\n",
      "materials and methods\n",
      "plasmid circuit construction\n",
      "strains, media and growth conditions\n",
      "assay of gene expression\n",
      "rna-seq sample preparation and sequencing\n",
      "cell growth rate modeling and metabolic load calculation\n",
      "gene expression clustering and differential expression analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00083/sb6b00083.xml\n",
      "results and discussion\n",
      "strategy for phb pathway optimization combining rbs library design and high-throughput screening\n",
      "in-silico design of rbs libraries\n",
      "generation of the library of phb pathways\n",
      "visual selection and high-throughput screening of recombinants with phb pathways\n",
      "characterization of phb pathway activities\n",
      "correlation between fi and phb contents\n",
      "quantitative analysis of phb contents in recombinants with selected pathways\n",
      "phb accumulation patterns of cells with 0–90% phb contents\n",
      "molecular weights of phb produced in different pathways\n",
      "discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "culture media and cell cultivation\n",
      "olma based one-step library construction\n",
      "cells staining with bodipy fluorescent dye and flow cytometric analysis\n",
      "cell dry weight, true cell mass, and phb content\n",
      "transmission electron microscopy\n",
      "phb molecular weight studies\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00130/sb6b00130.xml\n",
      "metabolic engineering of corynebacterium glutamicum with crispri\n",
      "metabolic engineering of pseudomonas putida kt2440\n",
      "precise engineering of regulated gene expression in mammalian cells\n",
      "structure of a thermophilic kinase shapes fitness upon random circular permutation\n",
      "engineering of a nepetalactol-producing platform strain of yeast\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00409/sb7b00409.xml\n",
      "methods\n",
      "bacterium strain, medium, and production condition\n",
      "plasmid construction\n",
      "enzyme assays with crude cell extract\n",
      "sbm1 protein purification\n",
      "enzyme assay with purified protein\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00119/sb8b00119.xml\n",
      "results and discussion\n",
      "molecular design for patterning cell surface with defined pmhc valency and density\n",
      "protein scaffold-directed assembly of pmhc on a cell surface\n",
      "t cell activation by multivalent pmhc assembly\n",
      "facs-facilitated tuning of the global pmhc surface density\n",
      "relationship between pmhc surface density and protein scaffold spacing\n",
      "minimum pmhc requirement for t cell activation\n",
      "protein scaffold-directed coassembly of pmhc and icam-1\n",
      "methods\n",
      "design of pmhc, icam-1, and cohesin scaffolds\n",
      "baculovirus transfection and pmhc, icam-1 purification\n",
      "scaffold purification and scaffold-directed protein complex assembly\n",
      "fluorescent-activated cell sorting\n",
      "t cell activation\n",
      "regression equations for t cell activation\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00172/sb8b00172.xml\n",
      "results\n",
      "design of hairpin promoter switches\n",
      "performance and modeling of hairpin promoter switches\n",
      "regulation of hairpin promoter switches\n",
      "buffering of inputs for hairpin promoter switches\n",
      "construction of larger circuit cascades, a double inverter\n",
      "construction of nand gates\n",
      "discussion\n",
      "methods\n",
      "oligonucleotides\n",
      "purification of wt t7 rnap\n",
      "in vitro transcription assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00140/sb6b00140.xml\n",
      "results and discussion\n",
      "e. coli production of pinene using a wild-type pinene synthase\n",
      "screening for mono-tps activity\n",
      "screening for ps mutants with higher cellular activity\n",
      "characterization of the ps mutant\n",
      "e. coli production of pinene using ps mutants\n",
      "cyanobacterial pinene production using ps variants\n",
      "conclusions\n",
      "materials and methods\n",
      "strain and plasmids\n",
      "construction of the ps-variant library\n",
      "screening for ps variants with elevated substrate consumption\n",
      "first screening\n",
      "second screening\n",
      "determination of the production level of pinene\n",
      "western blot analysis\n",
      "purification of pss\n",
      "in vitro reaction of ps variants\n",
      "construction of the ps-expressing strains of cyanobacteria\n",
      "culturing, gas-stripping, and gc analysis for the quantification of pinene production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00111/sb9b00111.xml\n",
      "results and discussion\n",
      "characterizing the effects of m2 expression on influenza vlp production\n",
      "inhibiting m2 ion channel activity alleviates cytopathic effects in sf9 cells\n",
      "amantadine improves cellular and surface expression of influenza proteins\n",
      "amantadine improves both yield and quality of influenza vlps\n",
      "experimental methods\n",
      "strains, media, and reagents\n",
      "recombinant baculovirus generation\n",
      "baculovirus titer determination\n",
      "sf9 cell infection characterization and influenza protein quantification\n",
      "virus-like particle (vlp) production and characterization\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00007/sb7b00007.xml\n",
      "experimental methods\n",
      "cell growth and media exchange\n",
      "ep/ewd device design\n",
      "ep/ewd chip staging\n",
      "on-chip transformation\n",
      "off-chip sample processing\n",
      "cuvette-based transformation\n",
      "results and discussion\n",
      "cell survival\n",
      "transformation\n",
      "plasmid transport\n",
      "nonideal device behavior\n",
      "performance comparison\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500003x/sb500003x.xml\n",
      "results and discussions\n",
      "internal ribosome binding sites are selected against in prokaryotic cdss but not in eukaryotic cdss\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "internal ribosome binding site produces truncated protein\n",
      "balancing transcription and translation rates to minimize expression noise and protein truncation when low protein expression is desired\n",
      "summary\n",
      "methods\n",
      "plasmids and integration\n",
      "culture and induction conditions\n",
      "irbs strength calculations\n",
      "genomic shuffling, recoding and irbs frequency calculation\n",
      "fluorescent assay\n",
      "western blot\n",
      "microscopy and image analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500165g/sb500165g.xml\n",
      "results and discussion\n",
      "theory of operation\n",
      "specifications and characterization\n",
      "modeling and programming\n",
      "gene network parameter dependence on cell density\n",
      "fluorescence detecting turbidostat\n",
      "conclusion\n",
      "methods\n",
      "yeast methods\n",
      "fabrication\n",
      "fluorescence detection\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00530/sb8b00530.xml\n",
      "results and discussions\n",
      "design of synthetic mirna toehold switch for mirna detection\n",
      "characterization of mirna toehold switch\n",
      "toehold switch for exogenously expressed mirna-155 detection in mammalian cells\n",
      "toehold switch for endogenous mirna-155 detection in mammalian cells\n",
      "multiplex detection of mirnas in mammalian cells using synthetic toehold switches\n",
      "conclusion\n",
      "materials and methods\n",
      "plasmid construction\n",
      "design of synthetic mirna switch sensor\n",
      "cell culture\n",
      "transfections\n",
      "reverse transcription and rt-pcr\n",
      "flow cytometry\n",
      "imaging\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00268/sb7b00268.xml\n",
      "results\n",
      "plasmid-independent hierarchical assembly of multiple sgrna expression cassettes\n",
      "establishment of a stable cell line expressing the suntag system\n",
      "single nonrepetitive sequence labeling by multiplexed sgrna expression\n",
      "synergistic regulation of endogenous genes by multiplexed sgrna expression\n",
      "coordinated activation and repression of endogenous genes by multiplexed sgrna\n",
      "discussion\n",
      "methods\n",
      "hierarchical assembly of multiple sgrna expression cassettes\n",
      "construction of other plasmids used in this work\n",
      "cell culture, transfection, stable cell line construction\n",
      "fish and immunofluorescence\n",
      "qrt-pcr\n",
      "image acquisition and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00047/sb8b00047.xml\n",
      "results and discussion\n",
      "characterization of the β-ketoadipate pathway in p. chlororaphis ht66\n",
      "genetic engineering to screen suitable locus for optimized expression of genes\n",
      "construction of a synthetic pathway for 4-hba in p . chlororaphis ht66\n",
      "creation of a biosynthesis pathway for muconic acid from glycerol\n",
      "modular optimization of the synthetic pathway of ma\n",
      "improvement of a rate-limiting step in ma production\n",
      "improved ma production by feeding glycerol\n",
      "material and methods\n",
      "strains, plasmids, medium, and culture conditions\n",
      "strain construction\n",
      "protein expression, purification, and in vitro poba enzyme assay\n",
      "identification of the catechol and protocatechuate branches of the β-ketoadipate pathway in p . chlororaphis ht66\n",
      "hplc and uplc–q/tof ms analyses\n",
      "rt-pcr and quantitative real-time pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001189/sb4001189.xml\n",
      "methods\n",
      "chemicals and reagents\n",
      "strains and culture condition\n",
      "characterization of the induction profile of the controllable gene expression system\n",
      "high-throughput screening of the promoter library\n",
      "alss toxicity test\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500222z/sb500222z.xml\n",
      "results and discussions\n",
      "methods\n",
      "dna constructs\n",
      "cell culture and transfection\n",
      "live cell imaging\n",
      "image analysis, fluorescence intensity measurement, and ruffling quantification\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000595/sb3000595.xml\n",
      "construction and characterization of a fime recombinase-based dna switch\n",
      "nitric oxide sensing as the input for the recombinase-based genetic switch\n",
      "strain ea3020 responds to biologically produced nitric oxide\n",
      "methods\n",
      "bacterial strains\n",
      "fluorescent time lapse microscopy\n",
      "dose response and time course experiments\n",
      "mouse ileum explant culture\n",
      "pcr detection of switch activation\n",
      "qrt-pcr\n",
      "nitric oxide measurement from ileum explants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000993/sb4000993.xml\n",
      "results and discussion\n",
      "discovery of a highly active ald\n",
      "construction of synthetic enzyme complex containing high-activity ald\n",
      "substrate channeling of two enzyme complexes\n",
      "substrate channeling of the different ratio enzyme complexes\n",
      "methods\n",
      "chemicals\n",
      "strains and medium\n",
      "construction of plasmids\n",
      "recombinant protein expression and purification\n",
      "one-step metabolon purification and immobilization\n",
      "enzymatic activity assays\n",
      "other assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00218/sb5b00218.xml\n",
      "an infinite loop\n",
      "a better-informed design\n",
      "a ribosome with tethered subunits, ribo-t\n",
      "breaking the infinite loop\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00367/sb7b00367.xml\n",
      "results and discussion\n",
      "methods\n",
      "library preparation\n",
      "in vitro transcription\n",
      "reverse transcription and template switching reactions\n",
      "sequencing and data analysis\n",
      "ribozyme assay by page\n",
      "ribozyme assay in mammalian cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500023f/sb500023f.xml\n",
      "results and discussion\n",
      "design and construction of biosensors\n",
      "detection spectrum and dynamic properties\n",
      "three types of interference caused by noninducer compounds\n",
      "expanding the detection spectrum\n",
      "conclusion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001883/sb4001883.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00531/sb8b00531.xml\n",
      "results\n",
      "mechanistic host-circuit models\n",
      "limits on expression of heterologous proteins\n",
      "critical cellular capacity\n",
      "loss-of-function in gene circuits\n",
      "genetic toggle switch\n",
      "genetic oscillator\n",
      "genetic logic gates\n",
      "discussion\n",
      "methods\n",
      "model for a bacterial host\n",
      "proteome components\n",
      "shared cellular resources\n",
      "integrated host-circuit models\n",
      "model parameters\n",
      "model for host\n",
      "models for gene circuits\n",
      "circuit simulations\n",
      "inducible gene\n",
      "genetic toggle switch\n",
      "genetic oscillator\n",
      "genetic logic gates\n",
      "relation between energy and free ribosomes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00005/sb6b00005.xml\n",
      "william e. bentley\n",
      "pei du\n",
      "jim haseloff\n",
      "chunbo lou\n",
      "tanya tschirhart\n",
      "lev tsimring\n",
      "jan roelof van der meer\n",
      "yongbo yuan\n",
      "xinyi zhou\n",
      "bartlomiej borek\n",
      "andriy didovyk\n",
      "carole lartigue\n",
      "pascal sirand-pugnet\n",
      "iason tsarmpopoulos\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500263b/sb500263b.xml\n",
      "results and discussion\n",
      "methods\n",
      "culture conditions\n",
      "transfection\n",
      "flow cytometry\n",
      "cloning\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00158/sb5b00158.xml\n",
      "results and discussion\n",
      "thermotolerant parts customized for the desired strain\n",
      "thermotolerant system for hierarchal heat resistance\n",
      "quorum regulating system for altruistic cell death\n",
      "intelligent microbial heat regulating engine (imhere)\n",
      "application of imhere to high-temperature fermentation\n",
      "perspective for further applications using imhere\n",
      "methods\n",
      "plasmid construction\n",
      "strains and growth conditions\n",
      "fluorescence assay\n",
      "lysine fermentation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00131/sb8b00131.xml\n",
      "results and discussion\n",
      "methods\n",
      "reagents\n",
      "synthetic-operon construction\n",
      "bacterial strains\n",
      "growth conditions\n",
      "carbon labeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001273/sb4001273.xml\n",
      "results and discussion\n",
      "metabolic space\n",
      "design space\n",
      "bioproduction circuits\n",
      "biosensor circuits\n",
      "bioregulation circuits\n",
      "bioprocessing circuits\n",
      "retropath: an automated design software for metabolic circuits\n",
      "methods\n",
      "definitions\n",
      "circuit specifications\n",
      "circuit scope\n",
      "circuit enumeration\n",
      "disease-biomarker biosensor detection\n",
      "data sets\n",
      "experimental procedure for e. coli metabolite identification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00180/sb5b00180.xml\n",
      "results\n",
      "characterization of the synthetic mirna precursor\n",
      "hierarchical assembly of the synthetic mirna cluster\n",
      "characterization of the synthetic mirna cluster\n",
      "genome integration of the synthetic mirna cluster\n",
      "discussion\n",
      "methods\n",
      "reagents and enzymes\n",
      "cell culture and transfection\n",
      "fluorescence-activated cell sorting measurement\n",
      "construction of stable cell lines and genomic dna extraction\n",
      "rna extraction and quantitative rt-pcr\n",
      "rna library preparation, sequencing, and data analysis\n",
      "plasmid dna constructs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00437/sb7b00437.xml\n",
      "methods\n",
      "materials and animals\n",
      "structure refinement and model building\n",
      "growth inhibition assay\n",
      "cell culture assay\n",
      "germline transformation\n",
      "production of azphe-containing silk fibroin\n",
      "urea degumming\n",
      "in-gel digestion and mass analysis\n",
      "click reaction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500279w/sb500279w.xml\n",
      "methods\n",
      "oligonucleotide synthesis\n",
      "hcr with caged initiator\n",
      "assembly of substrate complex, reporter gate, and translator gate\n",
      "fuel–catalyst cycle with caged oligonucleotides gel analysis\n",
      "fluorescence quantification\n",
      "fuel–catalyst spatial control\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00339/sb7b00339.xml\n",
      "results and discussion\n",
      "design and construction of an sa biosensor based on a lysr-type transcriptional regulator\n",
      "the sa biosensor responded to intracellular sa\n",
      "the sa biosensor is applicable for monitoring sa production in c. glutamicum\n",
      "application of the sa biosensor in single-cell sorting and high-throughput screening of an rbs library for a high-yield c. glutamicum strain\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "construction of a whole-cell biosensor for extracellular sa detection by engineering the in vivo sa biosensor\n",
      "methods\n",
      "bacterial strains, media and growth conditions\n",
      "dna manipulations\n",
      "confocal laser scanning microscopy\n",
      "measurement of fluorescence intensity of c. glutamicum cells\n",
      "characterization of the intracellular biosensor specificity\n",
      "measurement of intracellular sa concentration in c. glutamicum\n",
      "monitoring sa production in c. glutamicum cultures\n",
      "construction of an rbs library for optimization of tkta translation during sa production\n",
      "single-cell sorting with flow cytometry\n",
      "extracellular sa supplementation experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00313/sb6b00313.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture, transfections, and induction\n",
      "flow cytometry\n",
      "quantitative real-time pcr\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00275/sb6b00275.xml\n",
      "results and discussion\n",
      "genetic toggle switch example\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00082/sb5b00082.xml\n",
      "ribozyme-based aminoglycoside switches of gene expression\n",
      "conditional control of mammalian gene expression by tetracycline-dependent hammerhead ribozymes\n",
      "cell-free tx-tl systems for rapid characterization of fast dynamics of rna genetic circuitry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300118m/sb300118m.xml\n",
      "e. coli for l -isoleucine production\n",
      "modification-free photocontrol of enzyme synthesis\n",
      "genetic circuit performance under industrial bioreactor conditions\n",
      "engineering the β-oxidation cycle\n",
      "advances in biofuel production\n",
      "synthetic biology and metabolic engineering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00060/sb6b00060.xml\n",
      "results and discussion\n",
      "establish grna expression in c. beijerinckii\n",
      "lethality of double-strand breakages (dsbs) induced by active cas9\n",
      "achieve genome editing using crispr-cas9 and plasmid-borne editing template\n",
      "achieve reliable genome editing by expressing cas9 with an inducible promoter\n",
      "large gene fragment deletion\n",
      "markerless chromosomal gene integration\n",
      "single nucleotide modification (snm)\n",
      "vector integrating problem\n",
      "alternative approach for snm\n",
      "using crispr-cas9 as a tool for positive mutant selection\n",
      "conclusions\n",
      "methods\n",
      "strains, culture conditions, plasmids, and oligonucleotides\n",
      "plasmid construction\n",
      "rna isolation, cdna synthesis, and semiquantitative reverse transcription pcr (sqrt-pcr)\n",
      "subcultivation procedure for isolating desirable mutants when the lactose inducible promoter was used for cas9 expression\n",
      "confirmation of snm using mutation detection assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00271/sb9b00271.xml\n",
      "results and discussion\n",
      "selective enrichment enables isolation of a stable native consortium\n",
      "a natural gut consortium accelerates biogas production relative to a fungal monoculture\n",
      "metagenomic analysis suggests that syntrophy and compartmentalized metabolism drive stability of an enriched consortium\n",
      "synthetic biomass-degrading co-cultures produce methane but lack the stability and productivity of natural consortia\n",
      "conclusions\n",
      "methods\n",
      "enrichment and assembly of microbial consortia\n",
      "microbial cultivation and characterization\n",
      "determination and quantification of culture metabolites\n",
      "quantification of cellulose degradation rates in monocultures and synthetic co-culturess\n",
      "metagenomic library preparation and sequencing\n",
      "metagenomic binning and analysis\n",
      "16s and its profiling\n",
      "data availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00358/sb8b00358.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00243/sb5b00243.xml\n",
      "introduction\n",
      "results and discussion\n",
      "the spatiotemporal model with control\n",
      "open loop control\n",
      "closed loop control\n",
      "discussion\n",
      "methods\n",
      "numerical methods in bsim\n",
      "extending bsim to include open and closed loop control\n",
      "assessing periodic behavior with poincare stroboscopic sections\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00481/sb8b00481.xml\n",
      "results\n",
      "design and characterization of mdh pance\n",
      "multiplexed plate-based pance of bm mdh2\n",
      "pace of bm mdh2 pance variants\n",
      "evolution at low methanol concentrations leads to frmr-binding mdh variants\n",
      "evolved bm mdh2 variants show improved activity in vitro\n",
      "homology models reveal a common region for key bm mdh2 mutations\n",
      "evolved bm mdh2 variants outperform a state-of-the-art variant for in vivo methanol assimilation\n",
      "discussion\n",
      "materials and methods\n",
      "reagents\n",
      "strains and plasmids\n",
      "cloning\n",
      "luciferase reporter assays\n",
      "plaque assays for phage titer quantification\n",
      "pace experiments\n",
      "pance experiments\n",
      "crude lysate mdh assays\n",
      "mdh purification\n",
      "mdh kinetics\n",
      "13 c labeling and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400028c/sb400028c.xml\n",
      "introduction\n",
      "results and discussion\n",
      "materials and methods\n",
      "device fabrication\n",
      "cell-free experiments\n",
      "imaging\n",
      "image processing\n",
      "noise analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400158w/sb400158w.xml\n",
      "results and discussion\n",
      "methods\n",
      "materials\n",
      "plasmids and strains\n",
      "cell growth and fluorescence assay\n",
      "malonyl-coa quantification\n",
      "fatty acid production analysis\n",
      "fatty acid quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400059y/sb400059y.xml\n",
      "results and discussion\n",
      "monitoring cytosolic l -arginine in single cells\n",
      "generation and facs screening of argb muteins\n",
      "kinetic and physical characterization of argb muteins\n",
      "generation of lysc and hisg muteins\n",
      "methods\n",
      "strains, plasmids, and growth conditions\n",
      "microfluidic chip cultivations and live cell imaging\n",
      "error prone pcr and plasmid library construction\n",
      "facs and library screening\n",
      "high-throughput cultivation and analyses\n",
      "protein work\n",
      "structure modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00164/sb5b00164.xml\n",
      "methods\n",
      "strain and plasmid construction\n",
      "culture conditions\n",
      "hexokinase activity assay\n",
      "metabolite analysis\n",
      "quantification of mrna levels\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000687/sb5000687.xml\n",
      "annotating systems biology markup language models\n",
      "fairytale: a high-throughput tal effector synthesis platform\n",
      "the engineered dna sequence syntax inspector\n",
      "bi-hac vector system toward gene and cell therapy\n",
      "rapid and reliable dna assembly\n",
      "universal assay for engineering extracellular protein expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00198/sb6b00198.xml\n",
      "methods\n",
      "literature search\n",
      "evaporation profiles\n",
      "plate reader parameters\n",
      "modulation of optical measurements by cover method\n",
      "growth over time experiments\n",
      "maximum od600 calculation\n",
      "maximum growth rate calculation\n",
      "coefficient of variance calculation\n",
      "luminescence vs od600 experiments\n",
      "maximum luminescence per od600 calculation\n",
      "drop-off od600 calculation\n",
      "linearity before drop-off calculation\n",
      "preparation of biofilm\n",
      "biofilm quantification assay\n",
      "experiments using synthetic promoters\n",
      "experiments using antibiotics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00141/sb6b00141.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and culturing\n",
      "particle purification\n",
      "time lapse microscopy\n",
      "induction and particle stability testing\n",
      "growth curves\n",
      "co-localization analysis\n",
      "electron microscopy\n",
      "western blotting\n",
      "confocal microscopy\n",
      "indigo production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00194/sb5b00194.xml\n",
      "results and discussion\n",
      "overview\n",
      "language elements\n",
      "roughage\n",
      "compiler directives\n",
      "rewriting designs enables users to innovate\n",
      "target platforms\n",
      "example applications\n",
      "promoter titration\n",
      "allele swaps\n",
      "enzyme saturation mutagenesis\n",
      "artemisinin design\n",
      "issues\n",
      "ongoing work\n",
      "methods\n",
      "compiler architecture\n",
      "reference data\n",
      "output targets\n",
      "implementation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500267d/sb500267d.xml\n",
      "results and discussion\n",
      "library strategy\n",
      "double mutant libraries\n",
      "quadruple mutant library\n",
      "sequence selectivity rules\n",
      "beyond the rules: relative sequence preferences\n",
      "library quality and statistics\n",
      "outlook for library design\n",
      "methods\n",
      "library design and expression\n",
      "chemical analysis\n",
      "determination of selectivity rules\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00166/sb7b00166.xml\n",
      "methods\n",
      "strains and chemicals\n",
      "gc 3 chromosome scanning\n",
      "plasmid construction and yeast transformation\n",
      "flow cytometry analysis\n",
      "library-based screening of s. stipitis cens\n",
      "cen alignment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00454/sb7b00454.xml\n",
      "results\n",
      "establishment of an expression system mimicking that of bacteria\n",
      "concurrent single-channel recordings during in vitro protein synthesis\n",
      "single-channel features of the nascent channel\n",
      "orientation of expressed channels\n",
      "channel synthesis, localization and oligomeric state\n",
      "acceleration of the first appearance by the membrane potential\n",
      "discussion\n",
      "methods\n",
      "kcsa channel\n",
      "the dib method\n",
      "channel synthesis and its localization and oligomeric state\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00004/sb5b00004.xml\n",
      "methods\n",
      "plasmids\n",
      "cell culture and transfection\n",
      "light-mediated nuclear transport\n",
      "analytics\n",
      "zebrafish experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003357/sb5003357.xml\n",
      "results and discussion\n",
      "developing a panel of synthetic terminators composed of minimal sequence elements\n",
      "evaluating heterologous protein expression from a panel of short synthetic terminators\n",
      "transcription termination efficiency of synthetic terminators\n",
      "short synthetic terminators also function in an alternative yeast host\n",
      "altering the expression of a metabolic pathway using synthetic terminators\n",
      "conclusions\n",
      "methods\n",
      "strains\n",
      "plasmid construction\n",
      "flow cytometry\n",
      "quantitative pcr\n",
      "itaconic acid quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00348/sb6b00348.xml\n",
      "results and discussion\n",
      "establishment of an in vitro reconstituted system of the e. coli biosynthetic pathway to convert glucose to fatty acid\n",
      "optimization and titration of components in the glycolytic pathway\n",
      "comparison of pdcs from e. coli and e. faecalis and effects of protein ratios\n",
      "optimization of reducing equivalents\n",
      "step-wise optimization of the entire pathway\n",
      "conclusions\n",
      "methods\n",
      "plasmid construction\n",
      "protein purification\n",
      "reconstitution and optimization of fatty acid synthetic components\n",
      "analysis of fatty acids produced from glucose in the reconstituted system\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00103/sb5b00103.xml\n",
      "results\n",
      "stoichiometric evaluation of the synthetic pathway for conversion of ( d )-xylose into selected value-added chemicals\n",
      "identification and characterization of candidate enzymes possessing ( d )-xylulose-1-kinase and ( d )-xylulose-1p aldolase activity\n",
      "simultaneous expression of ( d )-xylose-1-kinase and ( d )-xylulose-1p aldolase confers growth of a ( d )-xylulose-5-kinase deleted mutant on ( d )-xylose\n",
      "the metabolic fate and regulatory role of glycolaldehyde\n",
      "strain engineering for complete assimilation of glycolaldehyde and increasing glycolic acid yield\n",
      "discussion\n",
      "materials and methods\n",
      "media and cultivation conditions\n",
      "quantification of extracellular metabolites by hplc analyses\n",
      "plasmid and strain constructions\n",
      "expression and purification of enzymes\n",
      "preparation of protein extracts and enzymatic assays\n",
      "rna extraction and microarray analysis\n",
      "data treatment and statistical analyses\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00291/sb5b00291.xml\n",
      "results and discussion\n",
      "overall design of ecoexpress for coexpression of multicomponent protein complexes\n",
      "a versatile set of vectors\n",
      "the efficiency of assembly into individual vectors and application of individual vector for protein expression\n",
      "purification of protein complex using ecoexpress\n",
      "improving the assembly efficiency of coexpression vector with an oligonucleotides blocking cloning method\n",
      "additional applications for ecoexpress\n",
      "methods\n",
      "construction of ecoexpress vectors\n",
      "construction of individual expression vector for each subunit\n",
      "coexpression vector assembly\n",
      "oligonucleotides blocking\n",
      "test of protein expression\n",
      "copurification of protein complexes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00155/sb7b00155.xml\n",
      "methods\n",
      "strains, media, and reagents\n",
      "library construction\n",
      "plasmid construction\n",
      "consensus sequence identification of promoters in c. acetobutylicum\n",
      "promoter strength distinguishing\n",
      "β-galactosidase assays\n",
      "fermentation\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00124/sb5b00124.xml\n",
      "methods\n",
      "moclo cloning protocols\n",
      "multiplex moclo cloning protocol\n",
      "statistical analysis methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00234/sb7b00234.xml\n",
      "results and discussion\n",
      "engineering of pichia pastoris for production of zmapp antibodies\n",
      "immunofluorescence assay\n",
      "materials and methods\n",
      "construct design and mab sequences\n",
      "strain construction\n",
      "transformation efficiency comparison\n",
      "pichia pastoris cultivation\n",
      "quantification via spr\n",
      "quantification via elisa\n",
      "antibody purification\n",
      "immunofluorescence assay\n",
      "lds-page\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00079/sb6b00079.xml\n",
      "results\n",
      "pqrr4 can be made selective toward phosphorylated luxo by truncation\n",
      "design and characterization of the crispri based inverter system\n",
      "establishing the optimal sensor protein expression using in silico model\n",
      "full system integration\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, dna sequences and plasmids\n",
      "consumables and services\n",
      "microplate readings\n",
      "qrr4 promoter study\n",
      "crispr matrix study\n",
      "full system integration study\n",
      "in silico model\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00385/sb7b00385.xml\n",
      "results and discussion\n",
      "design an artificial pathway for ht biosynthesis\n",
      "tyrosol production from tyrosine\n",
      "tyrosol production from simple carbon sources\n",
      "ht production from tyrosine\n",
      "ht production from simple carbon sources\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains and culture media\n",
      "dna manipulation\n",
      "feeding experiments\n",
      "de novo production of tyrosol and ht\n",
      "toxicity test and estimation of the partition coefficient\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00072/sb7b00072.xml\n",
      "results and discussion\n",
      "retro-biosynthetic design of a dxp pathway\n",
      "a combinatorial approach enables ha production\n",
      "in vivo coupling of ha and ga to form dxp\n",
      "in vivo utility of a dxs bypass\n",
      "methods\n",
      "general\n",
      "strains and plasmids\n",
      "media preparation, cell culture, and protein expression and analysis\n",
      "ha production and analysis in e. coli cultures\n",
      "dxp synthesis, metabolite extraction, and labeling analysis in engineered cells\n",
      "bap inhibition of e. coli growth and rescue of dxp strains\n",
      "analysis of cell lycopene content\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00136/sb6b00136.xml\n",
      "results and discussion\n",
      "lactam analogue search\n",
      "design and confirmation of a biplasmid lactam biosensor system\n",
      "characterization of improved cyclohexanone inducible chnr/pb promoter pair lactam biosensor\n",
      "methods\n",
      "analogue generation toward catabolizable chemicals\n",
      "biosensing protocol\n",
      "biosensor performance calculations\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00077/sb9b00077.xml\n",
      "results and discussion\n",
      "increasing mk-7 production by overexpression of mep pathway genes\n",
      "increasing mk-7 production by overexpression of mep pathway genes and mena\n",
      "enhanced mk-7 production induces pellicle formation in b. subtilis\n",
      "isoprene production as a measurement of mep pathway carbon flux\n",
      "intermediate metabolite profiling to identify mep pathway bottlenecks\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "bacterial growth media and conditions\n",
      "construction of e. coli / bacillus integrative shuttle vector pus258 containing mep pathway genes\n",
      "integration of synthetic gene clusters into bacillus subtilis\n",
      "mk-7 production, extraction, and hplc analysis\n",
      "headspace gc–ms analysis of isoprene production\n",
      "lc–ms/ms analysis of mep pathway intracellular metabolites\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000146/sb4000146.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid assembly\n",
      "genetic selection for caffeine demethylation\n",
      "methylated xanthine conversion\n",
      "caffeine content measurements\n",
      "growth in caffeinated beverages\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00248/sb6b00248.xml\n",
      "introduction\n",
      "results and discussion\n",
      "light responsiveness for different split positions\n",
      "roles of light-induced protein–protein interaction (ppi) in pat7p-1\n",
      "roles of allosteric effects in photoactivation of the polymerase function\n",
      "time courses of the on/off process in different systems\n",
      "modularity of the combination between the split t7 rnap fragments and the regulatory domains\n",
      "conclusions\n",
      "materials and methods\n",
      "materials\n",
      "plasmid structures and molecular cloning\n",
      "light responsiveness assays\n",
      "time course analysis of gene expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00114/sb7b00114.xml\n",
      "results and discussion\n",
      "the e. coli formaldehyde-inducible promoter is an ideal candidate for engineering\n",
      "inverted repeats are central to p frm architecture and response\n",
      "high-diversity library generation ensures rich information output\n",
      "high-resolution binding sites and identification of mutations of interest from sort-seq data\n",
      "informed design of tunable formaldehyde-inducible promoters\n",
      "application of the engineered formaldehyde-responsive promoter enables higher methanol growth\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, and growth media\n",
      "promoter library generation\n",
      "flow cytometry and sorting\n",
      "next-generation sequencing and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00496/sb8b00496.xml\n",
      "results and discussion\n",
      "methods\n",
      "growth condition of anabaena strains\n",
      "plasmid construction\n",
      "enrichment of heterocysts and western blotting analysis\n",
      "rna extraction and rt-qpcr analysis\n",
      "quantification of ethanol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00293/sb7b00293.xml\n",
      "results and discussion\n",
      "transcription activation of wap core genes using crispr/dcas9-ω\n",
      "refactoring of self-protection genes under the control of a strong lysobacter promoter p hsaf\n",
      "yield improvement of the wap-8294a compounds in engineered strains\n",
      "distribution of wap-8294a compounds in the culture\n",
      "test of wap-8294a2 efficacy\n",
      "summary and concluding remarks\n",
      "methods\n",
      "bacterial strains, plasmids and growth conditions\n",
      "dna manipulation, sequencing and lysobacter transformation\n",
      "primers and pcr\n",
      "generation of strains dcas9-ω from l. enzymogenes oh11\n",
      "generation of strains dcas9-ω3/refactored from strain dcas9-ω3\n",
      "growth comparison between the wt and engineered strains\n",
      "rna isolation and qrt-pcr\n",
      "extraction and preparation of wap-8294a compounds\n",
      "antibacterial activity test\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400194g/sb400194g.xml\n",
      "results and discussion\n",
      "induction of hairy roots\n",
      "pcr analysis\n",
      "upregulation of pgmvd and pgfps gene expressions analyzed using qpcr\n",
      "hplc analysis of saponin\n",
      "gas chromatography–mass spectrometry analysis of triterpenes\n",
      "methods\n",
      "plant materials\n",
      "vector construction\n",
      "rna isolation and cdna synthase\n",
      "gene expression by quantitative real-time pcr\n",
      "preparation of a. rhizogenes\n",
      "genetic transformation of p. ginseng\n",
      "pcr analysis\n",
      "ginsenoside analysis in hplc\n",
      "extraction and derivatization of triterpene\n",
      "gas chromatography with time-of-flight mass spectrometry (gctof) analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400086p/sb400086p.xml\n",
      "results\n",
      "construction of a methylation sensor in e. coli\n",
      "construction of a methylation sensor in s. cerevisiae\n",
      "comparison of sensor responses to different methylating compounds\n",
      "biosensing applications\n",
      "discussion\n",
      "materials and methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "preparation of alkylating agents\n",
      "e. coli response function assays\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "s. cerevisiae response function assays\n",
      "cytometry and data analysis\n",
      "detection of mei production by mhts\n",
      "detection of mei contamination in soil samples\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5001092/sb5001092.xml\n",
      "methods\n",
      "materials\n",
      "photoisomerization of azobenzene\n",
      "transcription catalyzed by t7 rna polymerase\n",
      "gfp expression template including azo or s-dm-azo\n",
      "photoisomerization of azobenzene and its derivatives\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00358/sb7b00358.xml\n",
      "yadira boada\n",
      "thomas classen\n",
      "drew delorenzo\n",
      "juan antonio garcía\n",
      "anita loeschcke\n",
      "peter q. nguyen\n",
      "fabio pasin\n",
      "alejandro vignoni\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00219/sb8b00219.xml\n",
      "results\n",
      "fsm-resistant colonies\n",
      "culture selection\n",
      "create plasmid deep sequencing\n",
      "genomic sequencing of enriched mutants\n",
      "the p274 site\n",
      "discussion\n",
      "materials and methods\n",
      "plasmids and strains\n",
      "create cassette design\n",
      "target sites\n",
      "library preparation\n",
      "selection\n",
      "deep sequencing\n",
      "plasmid-based create tracking analysis\n",
      "processing of high-throughput sequencing reads and query matching\n",
      "data filtering and enrichment analysis\n",
      "genomic ispc variant sequencing analysis\n",
      "data processing and variant calling\n",
      "enrichment analysis\n",
      "mutants validation\n",
      "growth inhibition assay in e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00011/sb6b00011.xml\n",
      "results and discussion\n",
      "microfluidic device operation\n",
      "genetic engineering of e. coli\n",
      "improving heat-shock throughput\n",
      "optimization of dna concentration used for transformation\n",
      "transformation results for other plasmids\n",
      "kanamycin selection\n",
      "golden gate dna assembly reaction\n",
      "fungal transformation\n",
      "yeast transformation\n",
      "filamentous fungi transformation\n",
      "conclusions\n",
      "methods\n",
      "device assembly and operation\n",
      "microfluidic transformation\n",
      "e. coli\n",
      "s. cerevisiae\n",
      "a. niger\n",
      "transformation efficiency\n",
      "off-chip culture\n",
      "on-chip culture\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300037w/sb300037w.xml\n",
      "results and discussion\n",
      "reducing codon redundancy\n",
      "reducing screening effort\n",
      "comparison of 22c-trick and nnk libraries\n",
      "importance of quality control in library creation\n",
      "creation of other libraries using the 22c-trick\n",
      "influence of other parameters on the quality of 22c-trick libraries\n",
      "conclusion\n",
      "methods\n",
      "mutant libraries of the chmo (cyclohexanone monooxygenase gene from acinetobacter sp. strain ncimb 9871)\n",
      "mutant libraries of pamo (phenylacetone monooxygenase gene from thermobifida fusca )\n",
      "generation of p450-bm3 libraries\n",
      "obtaining the 22c-trick mixture of primers\n",
      "determination of annealing temperature for degenerated primers\n",
      "quick quality control\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00252/sb5b00252.xml\n",
      "results and discussion\n",
      "design of a multicellular memory device\n",
      "two-cell implementation of a biological memory device\n",
      "the multicellular memory device triggers a bistable dynamic response\n",
      "memory state changes can be externally triggered\n",
      "external switching between stable states\n",
      "implementation of a memory device in a microfluidic platform\n",
      "discussion\n",
      "methods\n",
      "engineered yeast cell library and cell growth conditions\n",
      "protease purification\n",
      "synthetic pheromones\n",
      "output detection by flow cytometry in single cells\n",
      "transfer function characterization of the engineered cells\n",
      "characterization of the engineered cells\n",
      "characterization of the intrinsic memory of the device\n",
      "in vivo bistability analyses\n",
      "experimental validation of the transitions between the stable states\n",
      "theoretical estimation of gfp fluorescence levels\n",
      "microfluidic setup and image quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300107h/sb300107h.xml\n",
      "methods\n",
      "plasmids and strains\n",
      "growth medium and condition\n",
      "relative transcription determination\n",
      "assay for self-lysis activity\n",
      "growth curve determination\n",
      "microscopy of devices\n",
      "biosafety and biosecurity considerations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00284/sb6b00284.xml\n",
      "results\n",
      "site-specific flavin transfer reaction of recombinant nqrc and apbe in mammalian cells\n",
      "er-targeted nqrc is retro-translocated to the cytosol and degraded by the proteasome\n",
      "nqrc-er/apbe-nes as an in cellulo proteasome inhibitor screening system\n",
      "cell-based detection of fmn-labeled nqrc by confocal microscope\n",
      "truncated rnfg is another promising fmn acceptor protein of apbe\n",
      "discussion\n",
      "conclusion\n",
      "experimental section\n",
      "plasmids and cloning\n",
      "mammalian cell culturing and transfection\n",
      "gel scanning and western blot analysis\n",
      "nqrc protein expression and purification\n",
      "size-exclusion chromatography\n",
      "cd analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400024s/sb400024s.xml\n",
      "motivation\n",
      "what is pigeon ?\n",
      "commands\n",
      "labels and colors\n",
      "relationships\n",
      "why pigeon ?\n",
      "extending pigeon\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400048b/sb400048b.xml\n",
      "methods\n",
      "strain, media, and growth conditions\n",
      "gene cloning, plasmid construction, and genetic manipulation\n",
      "transformation of a. nidulans\n",
      "rna extraction and reverse transcriptase pcr (rt-pcr)\n",
      "fermentation and lc–ms analysis\n",
      "neosartoricin b extraction and purification\n",
      "nmr characterization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500304m/sb500304m.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "parts acquisition\n",
      "gfp-actin\n",
      "mls-aco eforred\n",
      "transformation and expression in a. niger\n",
      "gfp-actin\n",
      "mls-aco eforred\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00416/sb7b00416.xml\n",
      "results and discussion\n",
      "constitutive promoter library\n",
      "heterologous and endogenous plasmid copy numbers\n",
      "antibiotic resistance markers\n",
      "genome recombineering\n",
      "neutral integration sites\n",
      "crispri-mediated gene repression\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, plasmids, and growth conditions\n",
      "growth and fluorescence measurements\n",
      "total dna extraction and quantitative pcr\n",
      "identification of neutral sites\n",
      "gene integration\n",
      "crispri\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00057/sb8b00057.xml\n",
      "results and discussion\n",
      "identification of an itaconic acid-inducible system\n",
      "itaconic acid-inducible gene expression is mediated by a lysr-type transcriptional regulator\n",
      "sensor characterization\n",
      "sensor specificity\n",
      "biosensor-assisted optimization of itaconic acid production\n",
      "correlation between biosensor output and itaconate concentration\n",
      "methods\n",
      "base strains and media\n",
      "cloning and transformation\n",
      "plasmid construction\n",
      "rfp fluorescence assay\n",
      "production of itaconate and hplc-uv analysis\n",
      "metabolite extraction\n",
      "calculation of intracellular itaconate concentration in cell culture\n",
      "flow cytometry\n",
      "calculation of half-maximal rfp expression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000884/sb5000884.xml\n",
      "results\n",
      "evolution experiment under reduced ribosome conditions\n",
      "biochemical analysis of the rna clones\n",
      "analysis of the rna structure of the clones\n",
      "translation and replication activities at the high ribosome concentration\n",
      "discussion\n",
      "methods\n",
      "evolution experiment\n",
      "sequence analysis\n",
      "tcrr reaction of the clones\n",
      "translation activity of the clones\n",
      "activity of replicase encoded in the clones ( supporting information figure s3 )\n",
      "replication activity of the clones under uncoupled conditions ( supporting information figure s4 )\n",
      "mathematical model of the translation-coupled minus rna synthesis\n",
      "shape analysis of rna clones\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00290/sb8b00290.xml\n",
      "results\n",
      "cho-k1 gfp clones exhibit variable response of gfp expression levels to hypothermia\n",
      "targeted sequencing identifies random integration sites of transgenes in the selected cho-k1 gfp clones\n",
      "targeted integration of transgenes into three identified integration sites reduces clonal variation in response to hypothermia\n",
      "random integration of transgenes induces rearrangements to the expression vector\n",
      "transgene expression response to hypothermia is affected by promoter selection\n",
      "deletion of a promoter or its elements alters the transgene expression pattern\n",
      "discussion\n",
      "methods\n",
      "plasmids\n",
      "cell lines and cell culture\n",
      "recombinant cell line generation\n",
      "targeted locus amplification (tla) based transgene and integration site sequencing\n",
      "fluorescence level analyses\n",
      "genomic dna extraction and pcr amplification of target regions\n",
      "quantitative real time pcr (qrt-pcr)\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00188/sb8b00188.xml\n",
      "results and discussion\n",
      "exploration of pegylation sites to develop pegylated ifns with equivalent antiviral activity to that of unmodified ifn-α2b\n",
      "antiviral activity, physicochemical properties, in vitro stability, and pharmacokinetics of our site-specific di-pegylated ifn with two linear 5-kda pegs\n",
      "experimental procedures\n",
      "materials\n",
      "methods\n",
      "protein expression, purification, and pegylation\n",
      "antiviral assay with replicon cells\n",
      "directed evolution\n",
      "evaluation of in vitro and in vivo stability\n",
      "maldi-ms analysis\n",
      "cd measurement\n",
      "statistical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500324p/sb500324p.xml\n",
      "results\n",
      "principle and design specifications of the mega deletion and replacement method\n",
      "mega deletion and replacement is efficient for e. coli genome reduction\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, media, and reagents\n",
      "construction of the assembly vector (pxx11) and the template plasmid (pxx22)\n",
      "assembly of the multiple essential genes\n",
      "preparation of the disruption cassettes and the targeted genomic deletion\n",
      "integration of the cloned essential genes into the chromosome\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00354/sb6b00354.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "plant transformation and agro-inoculation\n",
      "protein detection\n",
      "targeted genome mutagenesis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400067v/sb400067v.xml\n",
      "results and discussion\n",
      "simultaneous assembling of one dsdna fragment and the de novo synthesis of an additional sequence\n",
      "in situ generation of overlaps from oligonucleotides\n",
      "assembly of two inserts by in situ generation of overlaps and de novo assembly of a promoter and rbs (156 bp)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "dna fragment preparation\n",
      "oligonucleotide design for de novo synthesis\n",
      "assemblies\n",
      "bacterial transformation\n",
      "screening for full-length inserts assembled from oligonucleotides and nonoverlapping dsdna fragments\n",
      "in silico construct design\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00057/sb7b00057.xml\n",
      "results and discussion\n",
      "methods\n",
      "library construction\n",
      "in vitro transcription\n",
      "isolation of rna fragments\n",
      "reverse transcription and template switching reactions\n",
      "sequencing library preparation\n",
      "sequencing and data analysis\n",
      "page assay of aptazyme variants\n",
      "consensus structure prediction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00411/sb8b00411.xml\n",
      "results\n",
      "an optimized mito-crispr/cas9 system for mtdna editing in human cells and zebrafish\n",
      "mito-cas9 protein and exogenous dna colocalize with mitochondrial marker inhuman cells and grna exists in the isolated mitochondria\n",
      "mito-crispr/cas9 system efficiently decreased mtdna copy number\n",
      "mtdna mutations can be caused by mito-crispr/cas9 system and transmitted to offspring\n",
      "expression level of genes involved in nhej and hr pathways\n",
      "discussion\n",
      "materials and methods\n",
      "ethics\n",
      "vector construction\n",
      "transcription of mito-cas9 mrna and target grna in vitro\n",
      "zebrafish husbandry, embryo microinjection, and genome dna isolation\n",
      "cell culture, transfection, and the extraction of genome dna\n",
      "localization of mito-cas9 protein, oligo-hex, and grna\n",
      "quantification of the mtdna copy number in both human cells and zebrafish\n",
      "expression levels of genes involved in nhej and hr\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00017/sb6b00017.xml\n",
      "materials and methods\n",
      "dna template preparation\n",
      "preparation of grea, greb and ecrnap holoenzyme\n",
      "batch reaction setup and measurements\n",
      "flagellar gene network in a nanoreactor device\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500042m/sb500042m.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00342/sb8b00342.xml\n",
      "results and discussion\n",
      "susceptibility of model staphylococcal proteins to particular secreted proteases\n",
      "secretion of staphylococcal proteins by b. subtilis pg10\n",
      "enhanced secretion of isaa in b. subtilis pg10\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "protein stability tests\n",
      "construction of expression plasmids\n",
      "lithium dodecyl sulfate-polyacrylamide gel electrophoresis (lds-page) and western blotting\n",
      "protein production, localization, and quantification\n",
      "growth curves\n",
      "live cell array analyses\n",
      "plasmid copy number determination\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300120n/sb300120n.xml\n",
      "results and discussion\n",
      "photoregulation of rna digestion by modulating the topology of a hammerhead ribozyme\n",
      "molecular design of photoresponsive ribozyme with a controllable topological structure\n",
      "cleavage reactivity changes with light irradiation\n",
      "reversibly photoswitching of rna cleavage by rb8x\n",
      "photoswitching of rna digestion by using a photoresponsive dnazyme\n",
      "photoswitching of rna cleavage by a photoresponsive 8–17 dnazyme\n",
      "photoregulation of rna cleavage by photoresponsive 10–23 dnayzmes\n",
      "mechanisms mediating photoswitching of rna cleavage catalyzed by photoresponsive functional nucleic acids\n",
      "two factors are topologically constrained in trans form resulting in suppression of rna cleavage\n",
      "conclusions\n",
      "methods\n",
      "oligonucleotides and reagents\n",
      "rna cleavage by ribozyme or dnazyme\n",
      "gel shift assay\n",
      "circularization of dnazyme (10–23wdzv)\n",
      "digestion of photoresponsive ribozyme by exonuclease t\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00144/sb9b00144.xml\n",
      "results and discussion\n",
      "characterization of native s. cerevisiae promoters\n",
      "expanding the maximum dynamic range of a tf-based biosensor by evaluating the effect of bs positioning\n",
      "expanding the maximum dynamic range while still retaining the native strength of a promoter\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "strains, media, and culture conditions\n",
      "plasmid and strain construction\n",
      "fluorescence measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00158/sb8b00158.xml\n",
      "results and discussion\n",
      "experimental creation of a cis -hybrid strain\n",
      "in silico metabolic network reconstruction\n",
      "metabolic phenotypes and profiles of the cis -hybrid strain\n",
      "assessment of the phenotypes of the cis -hybrid strain\n",
      "growth profiles in synthetic laboratory media\n",
      "growth using root exudate as a nutrient source\n",
      "biofilm formation\n",
      "symbiotic phenotypes of the cis -hybrid strain\n",
      "conclusions\n",
      "methods\n",
      "microbiological and genetic methods\n",
      "cis -hybrid strain construction\n",
      "validation of the transplanted strain\n",
      "growth curves\n",
      "growth with root exudate\n",
      "plant symbiotic assays\n",
      "biofilm assays\n",
      "phenotype microarray\n",
      "nmr metabolomics of the cell lysates and media\n",
      "generation of the metabolic models\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00176/sb6b00176.xml\n",
      "results and discussion\n",
      "role of interdomain linkers in at domain replacements\n",
      "examination of at domain boundaries in a second pks system\n",
      "validation of at replacement strategy using several different at domains\n",
      "in vitro short-chain ketone production by at-swapped pkss\n",
      "microbial short-chain ketone production\n",
      "methods\n",
      "chemicals\n",
      "plasmids\n",
      "protein expression and purification\n",
      "in vitro polyketide biosynthesis\n",
      "microbial ketone production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00349/sb6b00349.xml\n",
      "results and discussion\n",
      "eliminating inducer depletion\n",
      "the torecad and torf promoters are induced over a range of tmao concentrations aerobically and anaerobically\n",
      "replacement of the native mtrcab promoter with p torf enables induction of extracellular electron transport with tmao\n",
      "inducing current production with tmao in the δ tor δ dms mutant with p torf :: mtrcab\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "plasmids and strain construction\n",
      "growth curves\n",
      "β-galactosidase assays\n",
      "quantification of fe­(iii) respiration\n",
      "bioreactor construction and chronoamperometry\n",
      "culturing conditions for bioreactors\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00217/sb8b00217.xml\n",
      "results and discussion\n",
      "design of protease-activatable 14-3-3 proteins\n",
      "binding of mono- and bivalent exos peptides to t14-3-3\n",
      "protease-triggered activation of exos-inhibited t14-3-3 scaffolds\n",
      "synthetic signaling cascade\n",
      "self-activation\n",
      "conclusion and outlook\n",
      "materials and methods\n",
      "protease cleavage and caspase-9 activation\n",
      "synthetic signaling cascade\n",
      "self-activation assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00100/sb9b00100.xml\n",
      "methods\n",
      "data representation, encoding, and decoding\n",
      "primer design\n",
      "emulsion pcr\n",
      "biotin–streptavidin file extractions\n",
      "fluorescein and digoxigenin file extractions\n",
      "oligo-d­(t) magnetic bead separation\n",
      "calculation of data quantity from total number of dna strands\n",
      "calculation of system capacity\n",
      "error prone pcr\n",
      "qpcr\n",
      "illumina library preparation\n",
      "error analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000959/sb4000959.xml\n",
      "results and discussion\n",
      "fusion of escherichia coli trans -encoded small-rna-derived scaffolds to artificial riboregulators\n",
      "effects of single-stranded au-rich sequences in micf scaffold\n",
      "improving tarna-micf fold expression by stabilization of the hairpin structures and introduction of a sequence with high affinity to hfq\n",
      "fusion of srna scaffolds to the transcriptional attenuator and their improvement\n",
      "substituting the scaffold region of the micf srna into an engineered micf scaffold\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "bacterial strains\n",
      "plasmids construction\n",
      "gfpuv assay\n",
      "northern blot analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002856/sb5002856.xml\n",
      "results and discussion\n",
      "functional characterization of retooled tetr homologues in hek293 cells\n",
      "construction of a dapg-inducible system\n",
      "measurement of 1-input response functions\n",
      "signal integration: construction of an “enhancer” promoter that responds to two activators\n",
      "expanding the mammalian parts toolbox and beyond\n",
      "methods\n",
      "cell culture, strains, and media\n",
      "mammalian genetic parts\n",
      "transfection, growth, and processing of cells\n",
      "flow cytometry\n",
      "fold change and circuit copy number calculations\n",
      "hill equation curve fitting\n",
      "calculation of fold-change\n",
      "microscope imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00364/sb6b00364.xml\n",
      "results\n",
      "mechanistic model for arsr–p ars regulation\n",
      "effect of arsr binding affinity on circuit output\n",
      "including an arsenite efflux system decreases sensitivity of the circuit\n",
      "possible cross-binding effects of double arsr alleles\n",
      "discussion\n",
      "methods\n",
      "strains, culturing procedures, cloning, and molecular techniques\n",
      "deletion of the chromosomal ars genes in e. coli\n",
      "construction of arsenic reporter plasmids\n",
      "arsenic bioreporter assays\n",
      "arsr circuit description and assumptions\n",
      "mathematical model\n",
      "model assumptions and constraints\n",
      "assumption 1\n",
      "assumption 2\n",
      "assumption 3\n",
      "assumption 4\n",
      "assumption 5\n",
      "assumption 6\n",
      "assumption 7\n",
      "assumption 8\n",
      "assumption 9\n",
      "assumption 10\n",
      "assumption 11\n",
      "assumption 12\n",
      "assumption 13\n",
      "assumption 14\n",
      "assumption 15\n",
      "constraint 1\n",
      "constraint 2\n",
      "constraint 3\n",
      "constraint 4\n",
      "parameter estimation\n",
      "sensitivity analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00247/sb9b00247.xml\n",
      "materials and methods\n",
      "chemical reagents\n",
      "microfluidic chip\n",
      "optical microscopy and image analysis\n",
      "filling protocol\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000518/sb3000518.xml\n",
      "methods\n",
      "materials\n",
      "assembly of the nanostructures\n",
      "afm imaging\n",
      "agarose gel electrophoresis\n",
      "design of scaples\n",
      "generation of random sequence design-scaffolds\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00387/sb6b00387.xml\n",
      "results and discussion\n",
      "formation of mtca in s. cerevisiae cultures\n",
      "expression of fgapt2 led to the formation of dimethylallyl-mtca in the yeast cultures\n",
      "increasing the product yields of 7-dma-mtcas by metabolic engineering\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "increasing the structural diversity of dma-mtcas\n",
      "isolation and structure elucidation of the dma-mtca derivatives\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "culture conditions\n",
      "strain construction\n",
      "construction of dmats expression plasmids\n",
      "detection of mtca formation in yeast cultures\n",
      "cultivation for detection of the prenylated products\n",
      "isolation of the prenylated products\n",
      "chemical formation of mtca in synthetic yeast medium\n",
      "hplc analysis\n",
      "lc–ms analysis\n",
      "nmr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00313/sb8b00313.xml\n",
      "results\n",
      "construction and characterization of an nadph auxotroph strain\n",
      "the nadphaux strain enables direct selection for nadph production by different enzymes and reduced substrates\n",
      "discussion\n",
      "materials and methods\n",
      "strains and gene deletions\n",
      "gene overexpression\n",
      "media and growth conditions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500305v/sb500305v.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna cloning\n",
      "cell culture and transfection\n",
      "illumination conditions\n",
      "reporter gene assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00278/sb8b00278.xml\n",
      "materials and methods\n",
      "plasmids, bacterial strains, and growth conditions\n",
      "cloning of pn272 cargo\n",
      "western blot analysis\n",
      "competitive binding assay\n",
      "cadmium binding\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400170r/sb400170r.xml\n",
      "yifan li\n",
      "tsung-yi lin (steven)\n",
      "zhenquan lin\n",
      "zengyi shao\n",
      "yan wang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002449/sb5002449.xml\n",
      "nathan crook\n",
      "fanny delegrange\n",
      "volker kurz\n",
      "edward m. nelson\n",
      "nicolas perry\n",
      "alexander schmitz\n",
      "hanna j. wagner\n",
      "sabrina wend\n",
      "masaki yamaguchi\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00086/sb5b00086.xml\n",
      "farhima akter\n",
      "zehua bao\n",
      "matthew begemann\n",
      "lara bereza-malcolm\n",
      "james chappell\n",
      "ryan clarke\n",
      "wouter engelen\n",
      "ashley franks\n",
      "andrew david garst\n",
      "brian janssen\n",
      "josé jiménez\n",
      "sung in lim\n",
      "gülay mann\n",
      "andrew markley\n",
      "kayzad soli nilgiriwala\n",
      "gur pines\n",
      "analise z. reeves\n",
      "melissa takahashi\n",
      "han xiao\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00254/sb6b00254.xml\n",
      "results and discussion\n",
      "design and screening of surface display epitopes\n",
      "extracellular factors occlude outer membrane accessibility in living cells\n",
      "lps extracellular occlusion of epitope\n",
      "identification and deletion of extracellular proteins\n",
      "elimination of the putative s-layer and oag improves surface display availability\n",
      "surface display mediates interactions with functionalized abiotic beads\n",
      "surface display mediates binding to engineered yeast binding partners\n",
      "discussion\n",
      "materials and methods\n",
      "s. elongatus cell culture and strains\n",
      "immunostaining of cyanobacterial cells\n",
      "cyanobacteria-bead binding assays\n",
      "yeast culturing and induction conditions\n",
      "cyanobacteria-yeast binding assays\n",
      "surface component stripping\n",
      "western blotting\n",
      "flow cytometry analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00047/sb6b00047.xml\n",
      "lydia m. contreras\n",
      "nicholas c. curtis\n",
      "gianfranco fiore\n",
      "taylor a. hatridge\n",
      "jingxuan he\n",
      "joanne m. ho\n",
      "giansimone perrino\n",
      "noah m. reynolds\n",
      "dieter söll\n",
      "sue zanne tan\n",
      "kevin a. vásquez\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00108/sb9b00108.xml\n",
      "results and discussion\n",
      "elimination of swimming capability by the deletion of large flagellar clusters\n",
      "reducing biofilm formation by deletion of adhesive proteins\n",
      "solvent tolerance and loss of the psty megaplasmid\n",
      "growth kinetics in minimal medium with glucose\n",
      "ttgghi transplantation and deletion of proviral dna segments\n",
      "genome reduction by consolidation of deletion targets\n",
      "metabolic engineering for the comparative production of phenol\n",
      "conclusions\n",
      "experimental procedures\n",
      "media and culture conditions\n",
      "plasmid cloning and strain engineering\n",
      "genome sequencing\n",
      "swim assay\n",
      "biofilm formation assay\n",
      "solvent tolerance tests\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00062/sb5b00062.xml\n",
      "results and discussion\n",
      "microfluidic dna assembly\n",
      "cross-contamination study\n",
      "transformation and sequencing\n",
      "future outlook\n",
      "methods\n",
      "reagents\n",
      "dna fragment preparation: golden gate assembly\n",
      "pcr amplification\n",
      "dpni and bsai digest and purification\n",
      "dna fragment preparation: gibson assembly\n",
      "pcr amplification\n",
      "dpni digest and purification\n",
      "dna fragment preparation: yeast assembly\n",
      "pcr amplification\n",
      "dpni digest and purification\n",
      "microfluidic device fabrication\n",
      "microfluidic dna assembly and transformation\n",
      "macroscale dna assembly and transformation\n",
      "colony picking and sequencing\n",
      "golden gate and gibson assemblies\n",
      "yeast assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00249/sb8b00249.xml\n",
      "results and discussion\n",
      "repression of genes not participating in l -methionine biosynthesis\n",
      "in situ promoter replacement of genes involving in the l -methionine biosynthesis\n",
      "combinatorial genetic perturbation to identify the sequential steps blocking the l -methionine biosynthesis\n",
      "the control mode of carbon flux distribution in module ii and iii\n",
      "altering the expression strength of sera fbr gene\n",
      "metabolomic response to the iterative genetic modifications\n",
      "production of l -methionine in 5-l bioreactor\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, plasmids and general techniques\n",
      "construction of sgrna-expressing plasmid library, pam and ppam\n",
      "gene knockout, chromosomal promoter replacement and in situ point mutation\n",
      "culture conditions\n",
      "rt-qpcr for mrna quantification\n",
      "metabolome analysis\n",
      "analytical procedures\n",
      "uhplc/tofms analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00107/sb6b00107.xml\n",
      "methods\n",
      "cell culture\n",
      "strain construction\n",
      "dialysis experiments\n",
      "chlorophyll analysis\n",
      "13 c bicarbonate experiments and phb isolation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00246/sb6b00246.xml\n",
      "effect of genome position on heterologous gene expression\n",
      "rewiring quorum sensing networks for increased protein expression\n",
      "engineering hybrid chemotaxis receptors in bacteria\n",
      "promoter engineering of natural product biosynthetic gene clusters in yeast\n",
      "grea and greb enhance expression of e. coli rna polymerase promoters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00135/sb9b00135.xml\n",
      "results and discussion\n",
      "reconstruction of the clb and olb pathways\n",
      "metabolic superiority of the olb pathway to the clb pathway\n",
      "efficiency of the olb pathway in fed-batch fermentation\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and media\n",
      "construction of plasmids and strains\n",
      "fermentation conditions\n",
      "biomass and metabolite analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400065q/sb400065q.xml\n",
      "results and discussion\n",
      "generation of initial library of mutagenized acrb\n",
      "selection via growth competition in n -butanol resulted in enhanced tolerance to alcohols\n",
      "a second round of mutagenesis and selection further increased tolerance to n -butanol and isobutanol\n",
      "butanol concentration within cells is lower for cells expressing var1-2 than for cells expressing wild type acrb\n",
      "individual mutations are responsible for growth enhancement in the presence of n -butanol\n",
      "elimination of the proton motive force disrupts the acrb variant-conferred tolerance phenotype\n",
      "elimination of the acrb proton relay functionality disrupts the acrb variant-conferred tolerance phenotype\n",
      "low-level overexpression of wtacrb and acrb variants does not affect membrane integrity\n",
      "acrb variants do not confer enhanced tolerance in δ acrb strains\n",
      "summary and concluding remarks\n",
      "methods\n",
      "strains and bacterial growth condtions\n",
      "plasmid construction\n",
      "library construction\n",
      "whole gene random mutagenesis library\n",
      "cpec-generated loop 1 library\n",
      "combinatorial site-directed mutagenesis of loop 2 residues\n",
      "exponential growth competition\n",
      "characterization of selection output\n",
      "construction of mutations in acrb\n",
      "microtiter plate-based growth assays\n",
      "end point growth assays\n",
      "extracellular n -butanol measurements\n",
      "membrane integrity assays\n",
      "western blotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300131w/sb300131w.xml\n",
      "methods\n",
      "primer design\n",
      "pcr amplification of dna fragments\n",
      "one-step iso assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400010m/sb400010m.xml\n",
      "results and discussion\n",
      "noncontiguous protein recombination library design\n",
      "stabilities of an informative subset\n",
      "modeling thermostability\n",
      "stabilizing point mutations\n",
      "summary and conclusions\n",
      "materials and methods\n",
      "noncontiguous recombination\n",
      "optimal experimental design\n",
      "gene synthesis\n",
      "protein expression\n",
      "thermostability residual activity assay ( t r50 measurement)\n",
      "thermostability activity assay ( t a50 measurement)\n",
      "linear regression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00184/sb5b00184.xml\n",
      "daphne che\n",
      "liting duan\n",
      "anna payne-tobin jost\n",
      "korbinian kapsner\n",
      "laura martini\n",
      "adam meyer\n",
      "alexander prokup\n",
      "steve shih\n",
      "friedrich simmel\n",
      "guoqiang zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00235/sb8b00235.xml\n",
      "results\n",
      "overview of the control platform\n",
      "control of gene expression in chinese hamster ovary cells\n",
      "control of gene expression in mouse embryonic stem cells\n",
      "single-cell control of mtor signaling pathway activity\n",
      "biohybrid oscillator in mouse embryonic stem cells\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00240/sb7b00240.xml\n",
      "results and discussion\n",
      "crispr-cas10 elicits robust targeting of virulent staphylococcal phages\n",
      "crispr-cas10 immunity facilitates the recovery of virulent phage recombinants\n",
      "alternative strategies to edit toxic phage genes\n",
      "editing s. aureus phages with a heterologous system\n",
      "crispr-cas10 protospacers are densely packed across phage genomes\n",
      "materials and methods\n",
      "strains and growth conditions\n",
      "spacer design\n",
      "construction of s. epidermidis targeting strains\n",
      "construction of s. epidermidis editing strains\n",
      "construction of s. aureus targeting and editing strains\n",
      "phage targeting and genome editing\n",
      "genome extraction and confirmation of recombinant phages\n",
      "python script for protospacer selection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000782/sb3000782.xml\n",
      "results and discussion\n",
      "design of a functional reversal of the β-oxidation cycle and in vitro characterization of its functional units\n",
      "thiolase\n",
      "3-hydroxyacyl-coa dehydrogenase\n",
      "enoyl-coa hydratase\n",
      "acyl-coa dehydrogenase/trans-enoyl-coa reductase\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "thioesterases (termination)\n",
      "in vivo assembly of functional units required for a one-turn reversal of the β-oxidation cycle: 4-c carboxylic acids as proxy for product synthesis\n",
      "synthesis of longer-chain products by operating multiple turns of a functional reversal of the β-oxidation cycle\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, and genetic methods\n",
      "culture medium and cultivation conditions\n",
      "analytical methods\n",
      "enzymatic characterization and assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500341a/sb500341a.xml\n",
      "results and discussion\n",
      "methods\n",
      "enzymes and reagents\n",
      "cloning of oled\n",
      "cloning of other gts\n",
      "dna for assay\n",
      "production of probe dna\n",
      "purification of oled\n",
      "purified protein assays\n",
      "qpcr assay protocol\n",
      "fluorescence assay protocol\n",
      "lcms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00041/sb8b00041.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00177/sb7b00177.xml\n",
      "results\n",
      "effects of genome reduction on cell growth\n",
      "effects of starvation induced random mutations\n",
      "correlated individual growth characteristics revealed changes in heterogeneity\n",
      "cell culturability of the populations\n",
      "discussion\n",
      "the single-cell approach enables novel measures of population heterogeneity\n",
      "the effects of genome reduction on cell growth\n",
      "mutations at different genetic backgrounds\n",
      "conclusion\n",
      "materials and methods\n",
      "fabrication of microfluidic platforms\n",
      "bacterial strains\n",
      "chemostat culture\n",
      "generating mutant populations\n",
      "on-chip bacteria culture\n",
      "optical image acquisition and analysis\n",
      "calculation of μ and λ parameters\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00396/sb8b00396.xml\n",
      "results and discussion\n",
      "characterization of the key propionyl-coa synthetases (pcs) in response to propionate addition\n",
      "propionate addition caused significantly hyperpropionylation of pcs enzymes\n",
      "enzyme-catalyzed propionylation inhibited the activities of sace_0337, sace_4729, and sace_3848 except for sace_1780\n",
      "engineering of propionate assimilation pathway based on propionylation modification\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains, growth conditions, and reagents\n",
      "rna preparation and real-time rt-pcr\n",
      "overproduction and purification of proteins\n",
      "overexpression of his-1780 from s. erythraea\n",
      "in vitro propionyl-coa synthetase assays\n",
      "in vitro enzymatic protein propionylation\n",
      "in vitro nonenzymatic protein propionylation\n",
      "western blot analysis\n",
      "mass spectrometry peptide fingerprinting\n",
      "fermentation and erythromycin determination\n",
      "propionyl-coa assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000675/sb5000675.xml\n",
      "our main contribution\n",
      "results and discussion\n",
      "system description\n",
      "oscillator network: design i\n",
      "design i and design ii: ordinary differential equation (ode) models\n",
      "ode model: accounting for the loading effects\n",
      "ode model: accounting for enzyme deactivation, waste product build-up, disturbances, and modeling uncertainties\n",
      "description of the in silico controller\n",
      "control systems: concepts and terminology\n",
      "synthesis of our l 1 adaptive controllers\n",
      "simulation scenarios\n",
      "simulation results for design i and design ii in isolation\n",
      "simulation results for design i driving dna tweezer\n",
      "speeding up the kim–winfree oscillator\n",
      "potential applications\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00121/sb6b00121.xml\n",
      "results and discussion\n",
      "strategy for optsseq of the homoethanol pathway\n",
      "submaximal strength, near-consensus promoters maximized ethanologenic growth rate\n",
      "optimal pdc rbss deviated from predictions but had maximal actual strength\n",
      "fe 2+ -dependent adhb is preferred over zn 2+ -dependent adha for ethanologenic growth\n",
      "translational coupling and operon polarity effects likely resulted in stringent selection of elements in abp cassettes relative to pba\n",
      "ppb1 enables optimal e. coli ethanologenesis with pdc and adhb levels comparable to z. mobilis\n",
      "e. coli homoethanologenic growth rate is limited by protein synthesis cost\n",
      "conclusions and prospects\n",
      "methods\n",
      "materials\n",
      "bacterial strains, plasmids, and genetic manipulations\n",
      "design and construction of ethanologenic gene cassette libraries\n",
      "growth enrichments\n",
      "ht-seq of libraries and bioinformatic analyses\n",
      "characterization of growth and fermentation\n",
      "measurements of total cellular protein\n",
      "protein quantitation by western blot\n",
      "measurments of promoter strength\n",
      "in-silico flux balance analysis of ethanologenic strains\n",
      "calculation of promoter consensus similarity scores and rnap·σ 70 binding strengths\n",
      "calculation of degrees of enrichment ( e x )\n",
      "model for cost of synthesizing ethanologenic enzymes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00499/sb8b00499.xml\n",
      "results and discussion\n",
      "adapting the multireporter bacterial 1-hybrid (b1h) to a b2h\n",
      "characterization of pdz specificity\n",
      "the b2h system captures a wide range of affinities\n",
      "predictions of known interactions are improved by models incorporating lower affinity binders\n",
      "the prediction of novel interaction partners underscores the utility of mr-b2h data\n",
      "the mr-b2h provides insight into the specificities of bacterial pdz domains\n",
      "conclusions\n",
      "materials and methods\n",
      "library and plasmid construction\n",
      "selections on plates\n",
      "selections in liquid culture\n",
      "next-generation sequencing\n",
      "mamth validation assays\n",
      "bioinformatics analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00360/sb8b00360.xml\n",
      "results\n",
      "testing feasibility of yl-tu assembly for modulating heterologous product synthesis\n",
      "boosting lycopene production with chimeric mva pathways containing different multigene combinations and copy numbers\n",
      "exploring significantly regulated intrinsic pathways relevant with boosting of lycopene synthesis\n",
      "modulating cell’s lipid amounts and composition along with enhancing synthesis of lipophilic terpene\n",
      "conclusion\n",
      "materials and methods\n",
      "strain and media\n",
      "design of standardized yl-tu for gene expression\n",
      "dna manipulation for construction of yl-tu\n",
      "e. coli and yeast transformation\n",
      "extraction and hplc detection of heterologous products\n",
      "extraction and gc detection of total fatty acids\n",
      "transcriptome sequencing and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00065/sb6b00065.xml\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid and strain construction\n",
      "transposon selection\n",
      "β-galactosidase and gfp assays\n",
      "dna copy numbers analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00424/sb8b00424.xml\n",
      "methods summary\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300062c/sb300062c.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00100/sb7b00100.xml\n",
      "results and discussion\n",
      "combining metrs fission with fusion to peptides\n",
      "fragmented metrs require assistance for activity\n",
      "altering metrs-521 substrate specificity\n",
      "using domain insertion to diversify ligand control\n",
      "implications\n",
      "methods\n",
      "materials\n",
      "library construction\n",
      "bacterial complementation\n",
      "growth analysis\n",
      "vector construction\n",
      "anl preparation\n",
      "anl protein labeling\n",
      "visualizing labeled proteins\n",
      "imaging of tamra-dibo modified proteins\n",
      "chemical-induced anl labeling\n",
      "immunoblot analysis\n",
      "translation initiation calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400131a/sb400131a.xml\n",
      "results and discussion\n",
      "linear dna can be protected from degradation in tx-tl\n",
      "dna degradation in tx-tl is incomplete from the 5′ or 3′ end\n",
      "linear dna is an alternative to plasmid dna for circuit prototyping\n",
      "linear dna can be rapidly assembled for prototyping circuits\n",
      "linear dna prototyping theoretically allows for large circuits to be tested in a single business day\n",
      "conclusion\n",
      "methods\n",
      "cell-free expression preparation and execution\n",
      "gams protein purification\n",
      "plasmid dna and pcr product preparation\n",
      "sequences used for steric protection\n",
      "in vitro linear dna assembly\n",
      "rapid assembly product protocol\n",
      "linear dna degradation assay\n",
      "in vivo promoter characterization\n",
      "tx-tl promoter characterization on linear and plasmid dna\n",
      "tx-tl promoter induction curves\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00020/sb7b00020.xml\n",
      "results\n",
      "evaluation of crispri-mediated gene knockdown in cho duxb11\n",
      "generation of stable cho clones coexpressing crispri, dhfr, and egfp\n",
      "crispri-mediated dhfr suppression enhanced recombinant protein expression\n",
      "crispri-mediated dhfr suppression augmented gene amplification\n",
      "crispri-mediated dhfr suppression did not affect the cell growth\n",
      "crispri-mediated dhfr suppression enhanced g-csf production\n",
      "discussion\n",
      "materials and methods\n",
      "plasmid construction\n",
      "cell culture, transfection, and selection\n",
      "fluorescence microscopy and flow cytometry\n",
      "quantitative real-time reverse transcription pcr (qrt-pcr)\n",
      "genomic copy number quantification by q-pcr\n",
      "analysis of g-csf by elisa\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00278/sb7b00278.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "materials\n",
      "dna design\n",
      "folding dna origami\n",
      "size exclusion liquid chromatography\n",
      "dna box concentration and the ratio to hiv-tat\n",
      "agarose gel electrophoresis\n",
      "circular dichroism and ultraviolet spectroscopy\n",
      "dynamic light scattering\n",
      "fluorescence spectroscopy\n",
      "atomic force microscopy\n",
      "transfection assays\n",
      "poisoned transfection\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00076/sb9b00076.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "general methods\n",
      "expression and purification of yfp-lbp fusion proteins\n",
      "fluorescence measurements\n",
      "uptake assay\n",
      "determination of uaa concentration in cell lysates\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00089/sb8b00089.xml\n",
      "methods\n",
      "landing pad site identification and vector construction\n",
      "cho cell culture\n",
      "transfection and flow cytometry\n",
      "fluorescence-activated cell sorting and lp cell clonal expansion\n",
      "genomic pcr\n",
      "southern blot\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00184/sb6b00184.xml\n",
      "results\n",
      "developing novel zinc finger protein-regulated constitutive promoters\n",
      "computational identification of promoter design features conferring zfp-mediated repression\n",
      "conversion of transcriptional repressors into ligand-responsive biosensors\n",
      "contributions of biosensor biophysical properties to biosensor performance\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains and culturing\n",
      "plasmid construction\n",
      "microplate-based fluorescence assays and analysis\n",
      "flow cytometry\n",
      "statistical analysis of promoter design features\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00199/sb5b00199.xml\n",
      "results and discussion\n",
      "genome-wide transcriptional response toward different carbon sources\n",
      "mut promoter analysis employing reporter genes\n",
      "p cat1 initiates transcription based on derepression, matching p aox1 for some specific examples\n",
      "regulatory implications and sequence features\n",
      "a set of strong transcriptional terminators for p. pastoris\n",
      "applying mut promoters and terminators for combinatorial pathway fine-tuning\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, plasmids, chemicals, and media\n",
      "p. pastoris transformation and screening\n",
      "cultivation conditions\n",
      "microarray analyses\n",
      "reporter activity measurements\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00213/sb6b00213.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00347/sb8b00347.xml\n",
      "methods\n",
      "plasmids and strains\n",
      "strain and plasmid construction\n",
      "strain cultivation and fluorescence assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00178/sb6b00178.xml\n",
      "introduction\n",
      "results\n",
      "computational design of synthetic core promoters\n",
      "assessing core promoter-crm structure in the p. pastoris p aox1 system\n",
      "establishing a baseline expression level\n",
      "synthetic core promoters under the control of the p. pastoris p aox1-r\n",
      "analysis of the top-ten synthetic core promoter sequences\n",
      "second round screening: top-ten synthetic core promoters in different yeasts and crms\n",
      "correlation between the activities of synthetic core promoters fused to different crms\n",
      "discussion\n",
      "functionality of synthetic core promoters\n",
      "no motifs except the tata box clearly affect expression\n",
      "the role of nucleosome occupancy\n",
      "effects of core promoter and 5′utr\n",
      "modularity of synthetic core promoters\n",
      "materials and methods\n",
      "strains\n",
      "vectors and cloning: controls and synthetic core promoters fused to the p aox1‑r\n",
      "controls and entry vectors to assess synthetic core promoters with different crms in p. pastoris and s. cerevisiae\n",
      "cloning a subset of synthetic core promoters with different crms in p. pastoris and s. cerevisiae\n",
      "transformation of p. pastoris and cultivations\n",
      "transformation of s. cerevisiae and cultivations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500074a/sb500074a.xml\n",
      "results and discussion\n",
      "functional reconstitution of rnai pathway\n",
      "suppressor analysis of the yku70δ mutation by rnai screening\n",
      "rnai-assisted genome evolution for improved hac tolerance\n",
      "superior hac tolerance of engineered strains by rage\n",
      "conclusion\n",
      "methods\n",
      "strains, media, and cultivation conditions\n",
      "dna manipulation\n",
      "reconstitution of the rnai machinery in s. cerevisiae\n",
      "construction of a genome-wide rnai library\n",
      "construction of a yeast knockdown library and screening\n",
      "second and third rounds of rage\n",
      "characterization of engineered strains for hac tolerance\n",
      "estimation of the gene knockdown efficiency\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00321/sb7b00321.xml\n",
      "switching protein localization by site-directed rna editing under control of light\n",
      "engineered crispr systems for next generation gene therapies\n",
      "a fluorescent split aptamer for visualizing rna–rna assembly in vivo\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00213/sb9b00213.xml\n",
      "results and discussion\n",
      "t7 rnap expression platform\n",
      "and genetic logic gate\n",
      "nand genetic logic gate\n",
      "synthetic iptg-inducible promoters\n",
      "imply genetic logic gate\n",
      "aromatic sensors\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, and growth conditions\n",
      "growth and fluorescence measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00115/sb8b00115.xml\n",
      "results\n",
      "analysis of the microarray transcriptome data of b. subtilis\n",
      "design and screening of a synthetic promoter library for b. subtilis\n",
      "characterization of the synthetic promoter library using rt-pcr and sds-page\n",
      "model-based statistical analysis of promoter sequences\n",
      "construction of tandem promoter clusters using the spl\n",
      "modification of the purine pathway for inosine production by fine-tuning the expression of pura using synthetic weak promoters\n",
      "production of acetoin by the expression of xylanase xyna driven by synthetic promoter clusters\n",
      "discussion\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, media, and growth conditions\n",
      "plasmids construction\n",
      "strain construction\n",
      "generation of the synthetic promoter library\n",
      "library screening using a gfp reporter gene assay\n",
      "model-based analysis of promoter sequences\n",
      "analysis of fermentation products and metabolites\n",
      "real-time quantitative pcr\n",
      "sds-page analysis of reporter proteins\n",
      "enzyme activity assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00009/sb7b00009.xml\n",
      "results and discussion\n",
      "biosensor design\n",
      "biosensor optimization by random rbs mutagenesis\n",
      "characterization of the biosensor\n",
      "sorting yeast producers with the help of e. coli biosensor cells\n",
      "sorting of yeast producer strain library\n",
      "methods\n",
      "microbial strains, media, and growth conditions\n",
      "recombinant dna work\n",
      "determination of fluorescence\n",
      "droplet generation\n",
      "picoliter droplet sorting\n",
      "incubation of sorted droplet emulsion\n",
      "strain verification after sorting\n",
      "sorting of producer cells from a small library\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00020/sb8b00020.xml\n",
      "results and discussion\n",
      "construction of the α-type haploid resource\n",
      "mating-type conversion of yeast strains from the α-type to the a -type\n",
      "polyploid generation via repetitive mating\n",
      "fluorescent assays to confirm the successful polyploid generation\n",
      "responsiveness of the generated triploid strain to environmental sugars\n",
      "limitations of the current system and suggestions for consolidating yeast genomes\n",
      "the availability of the currently constructed system to generate artificial polyploid yeasts\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and media\n",
      "construction of plasmids\n",
      "construction of yeast strains\n",
      "measuring fluorescence with a microplate reader\n",
      "mating assay\n",
      "counterselection using 5-foa to remove plasmids containing the ura3 selection marker gene\n",
      "microscopic observations\n",
      "flow cytometric analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00142/sb9b00142.xml\n",
      "methods\n",
      "design algorithm\n",
      "overview\n",
      "constraints\n",
      "sequence update\n",
      "scoring functions\n",
      "computation and code availability\n",
      "high-throughput array experiments\n",
      "chemical mapping experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00413/sb7b00413.xml\n",
      "results and discussion\n",
      "regulation of pcb production in mammalian cells by fd, fnr, and heme\n",
      "effects of pcya, ho1 and fd+fnr stoichiometry on pcb production levels\n",
      "mammalian phyb-pif gene switch using endogenously produced pcb\n",
      "light sensitivity of the mammalian phyb-pif3 gene switch using endogenously produced pcb\n",
      "endogenous mammalian phyb-pif3 gene switch bistability and reversibility with far-red light\n",
      "genetically encoded phyb-pif3 gene switch in several mammalian cell lines\n",
      "summary\n",
      "methods\n",
      "zinc-page-immunoprecipitation assays\n",
      "imaging pcb production\n",
      "cell culture, transfection, light induction and reporter gene assays\n",
      "luciferase activity assay\n",
      "illumination circuits and software\n",
      "kinetic model\n",
      "marvin\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000252/sb5000252.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna techniques, media and growth conditions\n",
      "dna microarrays\n",
      "total aerobic microbial count assays (tamc)\n",
      "construction of plasmids and strains\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00019/sb7b00019.xml\n",
      "results\n",
      "wt gfp exhibits smaller expression levels compared to gfp with an ncaa incorporated at position 35\n",
      "density induced translation arrest model predictions corresponds to counterintuitive protein expression patterns\n",
      "early reassigned amber stop codon rescues protein expression as predicted by the model\n",
      "slower transcription and translation initiation rates rescue protein expression\n",
      "an analysis of mutants and initiation rate variants suggests that dita influences mrna levels\n",
      "testing additional proteins supports the generality of the dita phenomenon\n",
      "discussion\n",
      "conclusions\n",
      "experimental procedures\n",
      "gfp and mrfp1 quantification and purity assessment\n",
      "gfp purification and mass spectrometry\n",
      "zymomonas mobilis alcohol dehydrogenase (zmadh) expression and quantification\n",
      "b1 domain of protein l (pl) expression and quantification\n",
      "mrna quantification\n",
      "modeling the expression density under the dita assumption\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00454/sb8b00454.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and plasmids used\n",
      "luminescence and fluorescence measurements\n",
      "crispri targeting ftsz in e. coli top10\n",
      "crispri targeting flic in e. coli mg1655\n",
      "flagellin extraction and observation on sds-page\n",
      "plasmid copy number measurement\n",
      "evaluation of genetic stability of the dual riboswitch construct\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00179/sb7b00179.xml\n",
      "results\n",
      "cytosine modification of phage t4 genome inhibits, but does not block, restriction by crispr-cas9 nuclease\n",
      "creation of specific mutants by editing of t4 genome using crispr-cas9\n",
      "the length of the homologous arms correlates with editing efficiency\n",
      "genome editing using two adjacent spacers\n",
      "determining the essentiality of rnlb gene by genome editing\n",
      "discussion\n",
      "methods\n",
      "bacterial and bacteriophage strains\n",
      "plasmids\n",
      "plaque assays\n",
      "dna sequencing of single plaques\n",
      "genome editing\n",
      "one-step growth experiment\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500363y/sb500363y.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results\n",
      "overview of the teto toolkit\n",
      "optimization of tet-off regulatory control in s. cerevisiae\n",
      "variants of the teto site that generate different expression levels\n",
      "lycopene pathway genes can be regulated by teto variants\n",
      "enhancing recombination in s. cerevisiae\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "transformation protocol\n",
      "oligonucleotides and dna sequencing\n",
      "teto regulation of his3\n",
      "ura3 allelic replacement assays\n",
      "rna extraction, cdna synthesis, and quantitative pcr\n",
      "luciferase assay\n",
      "lycopene measurement\n",
      "plasmid-based teto variant assay\n",
      "growth rate measurements\n",
      "crt gene integration\n",
      "teto toolkit implementation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00286/sb6b00286.xml\n",
      "results\n",
      "protein language and glyph set\n",
      "protein designer\n",
      "example a: protease sensor\n",
      "example b: light-inducible protein membrane localization\n",
      "example c: inducible artificial transcription factor\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400151d/sb400151d.xml\n",
      "belén calles\n",
      "meng chen\n",
      "fahim farzadfard\n",
      "brendan hussey\n",
      "richard kil\n",
      "chang liu\n",
      "david mcmillen\n",
      "felix moser\n",
      "liming wang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00042/sb9b00042.xml\n",
      "results and discussion\n",
      "construction of an initial oah-producing strain\n",
      "redistribution of metabolic flux to promote the biosynthesis of oah\n",
      "protein engineering of metxlm to improve its properties\n",
      "effects of optimized expression of favorable metxlm mutant on oah production\n",
      "increasing intracellular acetyl-coa level to enhance the production of oah\n",
      "engineering of nadph regeneration to improve the biosynthesis of oah\n",
      "fed-batch fermentation in a 7.5 l fermenter\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, plasmids, and culture conditions\n",
      "plasmids construction\n",
      "genome manipulation\n",
      "expression and purification of metx and enzyme activity assays\n",
      "evolutionary conservation analysis\n",
      "homology modeling and molecular docking\n",
      "site directed mutagenesis and saturation mutagenesis of metx\n",
      "thermal stability measurement\n",
      "measurement of intracellular nadph/nadp + and acetyl-coa\n",
      "batch and fed-batch cultivation\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00260/sb5b00260.xml\n",
      "johan andersen-ranberg\n",
      "lars m. blank\n",
      "amy ehrenworth\n",
      "elias englund\n",
      "jesús fernández-rodríguez\n",
      "thomas gorochowski\n",
      "kohei ike\n",
      "shigeko kawai-noma\n",
      "malathy krishnamurthy\n",
      "sijin li\n",
      "gabriel lopez\n",
      "kuntal mukherjee\n",
      "daisuke umeno\n",
      "nick wierckx\n",
      "lei yang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00013/sb9b00013.xml\n",
      "results and discussion\n",
      "design a hybrid tropone biosynthetic pathway\n",
      "enhancing e. coli native paa catabolism by promoter replacement\n",
      "tropone production from paa\n",
      "biosynthesis of paa\n",
      "assembly of the full pathway for de novo production of tropone\n",
      "materials and methods\n",
      "media and strains\n",
      "plasmid construction\n",
      "feeding experiments\n",
      "de novo production of paa and tropone\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00325/sb7b00325.xml\n",
      "results\n",
      "design of a gfp-specific nanodeg\n",
      "modulation of nanodeg rates of synthesis and degradation allows for quantitative control of cellular gfp levels\n",
      "modular design of the nanodeg system for predictable control of the target’s levels\n",
      "integration of the nanodeg system into a stimulus-dependent circuit for enhancing the output dynamics\n",
      "discussion\n",
      "methods\n",
      "plasmids and cloning procedures\n",
      "cell cultures, transfections and stable cell lines\n",
      "flow cytometric analyses and fluorescence-activated cell sorting (facs)\n",
      "photoactivation and fluorescence decay analyses\n",
      "quantitative real-time pcr analyses\n",
      "western blot analyses\n",
      "protein aggregation studies\n",
      "cycloheximide chase assay\n",
      "protein purification\n",
      "enzyme-linked immunosorbent assay (elisa)\n",
      "statistical analyses\n",
      "mathematical model\n",
      "sensitivity analysis\n",
      "data fitting and model parameter estimation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00199/sb8b00199.xml\n",
      "results and discussion\n",
      "signal peptide replacement of htxb for functional expression of the htx transporter in syn 7942\n",
      "identification of endogenous pi transporters in syn 7942\n",
      "construction of the pt-dependent syn 7942 strain\n",
      "availability of p sources for strain rh714\n",
      "escape frequency and long-term viability\n",
      "growth competition between rh714 and wild-type cyanobacteria\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "plasmid construction\n",
      "strain construction\n",
      "characterization of a pit homologue from syn 7942\n",
      "assessment of p availability\n",
      "escape assay and long-term viability\n",
      "growth competition assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400156v/sb400156v.xml\n",
      "results and discussion\n",
      "targeted enrichment to identify lpl determinants that work with the synthetic hsp system to enhance survival of e. coli under strong ethanol stress\n",
      "screening for growth: targeted enrichment to identify lpl determinants that work with phsp to enhance e. coli growth under moderate ethanol stress\n",
      "identified lpl genes represent a diverse pool of heterologous genes for engineering tolerance\n",
      "combining related traits leads to an improved complex trait: better survival and growth together\n",
      "genes imparting tolerance distinctly for growth vs viability\n",
      "more than ethanol: a broader alcohol tolerance\n",
      "tolerance and ability to produce higher product levels: engineering for improved ethanol tolerance leads to improved ethanol production\n",
      "building on the hsp strength: adding to it and/or benefitting from it\n",
      "“another” synthetic biology: building multigenic complex phenotypes by functional selection one “dna brick” at a time\n",
      "material and methods\n",
      "bacterial strains, plasmids, and primers\n",
      "analytical methods\n",
      "growth conditions\n",
      "dna isolation, manipulation, and cell transformations\n",
      "library construction\n",
      "plasmid construction\n",
      "alcohol tolerance assays\n",
      "ethanol fermentations to emulate the melle–boinot process\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00459/sb8b00459.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains\n",
      "gene cloning and plasmid construction\n",
      "culture medium, inoculation, and flask culture of recombinant b. subtilis\n",
      "analysis of cell density and surfactin production\n",
      "flow cytometry analysis\n",
      "accase assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00425/sb7b00425.xml\n",
      "results and discussion\n",
      "the sco6264 and sco6267 genes from s. coelicolor are essential for gbl production\n",
      "expression of the s. coelicolor gbl system in e. coli bl21 results in production of scb2\n",
      "production of scb2 in e. coli bl21 is robust through different temperature and medium conditions\n",
      "engineering of scbr to promote solubility in e. coli bl21\n",
      "crosstalk evaluation between the gbl and the ahl systems\n",
      "induction of the gbl genetic circuits\n",
      "toward establishing butyrolactone signaling circuits for synthetic biology\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "culture media and conditions\n",
      "expression of the gbl biosynthetic pathway in e. coli\n",
      "gbl extraction and purification\n",
      "kanamycin bioassays\n",
      "hplc and ms analysis\n",
      "scbr expression\n",
      "sds-page and western blot\n",
      "gel retardation assays\n",
      "relationship between signal input and response of gbl and ahl receptors\n",
      "structural model of scbr and docking\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00382/sb7b00382.xml\n",
      "introduction\n",
      "material and methods\n",
      "detailed steps for encoding binary data into the dna string\n",
      "decoding error-containing dna strings into binary data\n",
      "analysis of error tolerance by in silicon simulation\n",
      "in vivo verification of the error tolerance\n",
      "error-prone pcr experiment\n",
      "uv exposure experiment\n",
      "biological relevance analysis of sed3b encoded sequences\n",
      "secondary structure analysis\n",
      "implementation of an online encoding-decoding system with sed3b\n",
      "results\n",
      "principles of a self-error-detecting, three-base block encoding scheme\n",
      "high error tolerance revealed by in silicon simulations\n",
      "tolerance of substitution errors\n",
      "tolerance of insertion–deletion errors\n",
      "sed3b encoded dna sequences show low biological relevance\n",
      "sed3b encoded dna sequences show simple secondary structures\n",
      "reliable orthogonal information encoding in living cells using sed3b\n",
      "concluding remarks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400046t/sb400046t.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00153/sb8b00153.xml\n",
      "results and discussion\n",
      "methods\n",
      "cell culture and transfection\n",
      "plasmids\n",
      "flow cytometry\n",
      "qrt-pcr\n",
      "data analysis\n",
      "modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00453/sb7b00453.xml\n",
      "results and discussion\n",
      "overexpression of wax ester synthases for fatty acid ethyl ester production in y. lipolytica\n",
      "the effects of promoter on faees production\n",
      "increasing the supply of cytosolic precursors in the engineered y. lipolytica improved faees accumulation\n",
      "removing diacylglycerol acyltransferase (dgat) function in y. lipolytica affected faees production\n",
      "intervention in the β-oxidation pathway of y. lipolytica improved faees production\n",
      "the effects of altering exogenous ethanol or glucose concentration on cell growth and faees production\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and culture conditions\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "plasmids construction and yeast transformation\n",
      "analytical methods\n",
      "gas chromatography coupled with mass spectrometry (gc–ms) analysis of faees\n",
      "rna isolation and quantitative reverse transcription pcr (qrt-pcr)\n",
      "fluorescence microscopy analysis of gene knockout strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00330/sb8b00330.xml\n",
      "results\n",
      "effect of inactivating sequence length on virus formation and genome protection ability\n",
      "effect of inactivating sequence length on proteolytic susceptibility\n",
      "protease-activated transduction ability of provis mutants\n",
      "cell receptor usage by provis mutants\n",
      "transduction kinetics of provis mutants\n",
      "discussion\n",
      "conclusions\n",
      "materials and methods\n",
      "plasmid cloning\n",
      "virus production\n",
      "virus titer quantification\n",
      "cellular transduction assay\n",
      "nuclease protection assay\n",
      "western blot and silver stain\n",
      "proteolysis\n",
      "lectin competition assay\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00028/sb8b00028.xml\n",
      "results and discussion\n",
      "inducible activation of gene expression in the synthetic jcvi-syn1.0 cells\n",
      "inducible attenuation of gene expression via crispri in the jcvi-syn1.0 cells\n",
      "kinetics of inducible activation and attenuation of gene expression in the jcvi-syn1.0 cells\n",
      "inducible attenuation of gene expression via crispri in m. pneumoniae\n",
      "induced activation and attenuation of gene expression in the jcvi-syn3.0 cells\n",
      "crispri-mediated repression of ksga gene expression in the jcvi-syn3.0 cells\n",
      "additional consideration on crispri technologies and inducible systems in minimal cells\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "dna manipulation\n",
      "plasmid construction for m. pneumoniae experiments\n",
      "plasmid construction for experiments in jcvi-syn1.0 and jcvi-syn3.0\n",
      "mmoric plasmids for studies in the jcvi-syn1.0 cells\n",
      "landing pad-compatible constructs for studies in the jcvi-syn3.0 cells\n",
      "rna extraction and real-time pcr\n",
      "phase-contrast and epifluorescence microscopy\n",
      "analysis of mycoplasma mycoides strains\n",
      "time-course assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00067/sb8b00067.xml\n",
      "methods\n",
      "media, strains and plasmids\n",
      "dna manipulations\n",
      "in vitro crude extract enzyme assays\n",
      "feeding experiments\n",
      "de novo biosynthesis of pyrogallol\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001596/sb4001596.xml\n",
      "results and discussion\n",
      "decoder architecture and implementation\n",
      "decoder characterization\n",
      "conclusions\n",
      "methods\n",
      "mammalian cell culture and transient transfections\n",
      "fluorescence microscopy\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00250/sb6b00250.xml\n",
      "results and discussion\n",
      "ribosome binding sites strongly but unreliably control gene expression\n",
      "the 5′- and 3′-untranslated region (utr) influence protein expression\n",
      "variability in protein synthesis outweighs variability in rna synthesis\n",
      "a simple computational model that includes parameters for variability can reasonably predict cell-free expression\n",
      "genetic cascades increase the variability of gene expression\n",
      "a better understanding of the influence of rna folding on transcription-translation may improve predictability\n",
      "methods\n",
      "genetic constructs\n",
      "cell-free transcription-translation\n",
      "cell-free extract preparation\n",
      "genetically encoded pictures\n",
      "protein and rna standard curves\n",
      "computational kinetic model\n",
      "statistical analysis\n",
      "photobleaching\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00186/sb8b00186.xml\n",
      "results and discussion\n",
      "engineered cellulase signal peptide libraries for protein secretion in b. megaterium\n",
      "expression of cellulase signal peptide libraries identified constructs for secretion of active egi1 and cel9at in lb media\n",
      "expression of cellulase signal peptide libraries identified constructs for secretion of active egi1 and cel9at in m9+ minimal media\n",
      "egi1 and cel9at demonstrated different preferred conditions for cellulolytic activity that were balanced for synergistic behavior\n",
      "synergistic cellulolytic activity detected in reactions with different ratios of egi1:cel9at\n",
      "synergistic cellulase activity achieved in bacillus coculture of egi1 and cel9at strains\n",
      "discussion and conclusions\n",
      "methods\n",
      "bacterial strains, media, and growth conditions\n",
      "signal peptide library plasmid construction and transformation\n",
      "signal peptide library growth and protein expression\n",
      "cellulase activity detection using the dns assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400178c/sb400178c.xml\n",
      "promotercad application architecture (figure a)\n",
      "overview of promotercad data architecture (figure b,c)\n",
      "motif operation algorithm\n",
      "methods and results\n",
      "example 1. place as new database including sequence motifs with multiple sizes\n",
      "find place motifs in upstream sequences of arabidopsis gene loci\n",
      "format and load linkdata table (figure a, table s1, supporting information )\n",
      "load your data to your application\n",
      "example 2. pedb as a new motif database and gene atlas as a new expression database from mouse biological resources\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00528/sb8b00528.xml\n",
      "results and discussion\n",
      "engineering of the next-generation nirw-acs\n",
      "characterization of the engineered nirw-acs in vitro\n",
      "testing engineered nirw-acs in mammalian cell culture\n",
      "transcranial neuronal photoactivation via an engineered nirw-ac in mice\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "recombinant dna techniques\n",
      "screening system for bacterial ac activity\n",
      "protein purification\n",
      "spectroscopy\n",
      "ac enzymatic assays\n",
      "nirw-ac expression in hek293 cells\n",
      "aav injection in vivo\n",
      "eeg recordings in mice\n",
      "power spectrum\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00076/sb8b00076.xml\n",
      "results\n",
      "synthetic activators\n",
      "synthetic repressors\n",
      "establishment of a bistable switch\n",
      "bistable switch in control of a metabolic pathway\n",
      "discussion\n",
      "materials and methods\n",
      "strains\n",
      "media\n",
      "cloning\n",
      "transformations\n",
      "copy number analysis\n",
      "fluorescence measurement with fluorimeter\n",
      "transcription analysis and rt-pcr\n",
      "fluorescence flow cytometry\n",
      "violacein pathway metabolite analysis\n",
      "analysis of orthogonal activators\n",
      "growth curves\n",
      "analysis of orthogonal repressors\n",
      "analysis of the thi4 and gal1 promoter induction\n",
      "analysis of the bistable switch\n",
      "analysis of the violacein pathway control by the bistable switch\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00064/sb6b00064.xml\n",
      "results and discussion\n",
      "ectopic copies of oric in engineered e. coli strains are not functional\n",
      "v. cholerae - oriii insertions are functional replication origins in the e. coli chromosome context\n",
      "simulation of genomic copy number patterns of multiorigin chromosomes\n",
      "replication timing of two linear chromosomes in e. coli might be similar to replication of the two chromosomes in vibrio cholerae\n",
      "methods\n",
      "bacterial strains, media, plasmids and oligonucleotides\n",
      "strain construction\n",
      "flow cytometry\n",
      "comparative genomic hybridization\n",
      "simulation of genomic copy number dependent on two-origin replication\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500113b/sb500113b.xml\n",
      "evolvability\n",
      "self-organization\n",
      "responsiveness to environment\n",
      "bridging the function gap with inorganic materials\n",
      "a platform for engineering living functional materials\n",
      "outlook\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00103/sb9b00103.xml\n",
      "results\n",
      "generating microfluidic droplets for enzyme engineering\n",
      "e. coli lysis inside droplet microcompartments\n",
      "design of a fluorescent-activated droplet sorting (fads) device\n",
      "sorting fluorescent droplets\n",
      "establishing optical sensors for a diverse set of nucleic acid enzymes\n",
      "polymerase extension\n",
      "strand displacement\n",
      "restriction digestion\n",
      "dna ligation\n",
      "polymerase-mediated dna, rna, and xna synthesis from e. coli generated enzymes\n",
      "mock enrichment assay for dna synthesis activity\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00239/sb6b00239.xml\n",
      "methods\n",
      "reagents\n",
      "plasmid construction\n",
      "strains\n",
      "culture conditions\n",
      "quantification of extracellular metabolites\n",
      "measurement of photosynthetic oxygen evolution\n",
      "determination of chlorophyll content\n",
      "determination of ros production\n",
      "enzyme assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00371/sb6b00371.xml\n",
      "results and discussion\n",
      "library strategy and design\n",
      "library assembly\n",
      "yeast nif protein expression\n",
      "regression analysis\n",
      "nif protein processing\n",
      "targeting and processing of mitochondrial signals\n",
      "yeast nifdk formation\n",
      "nafy binding to yeast nifdk\n",
      "summary\n",
      "methods\n",
      "strains, media, and molecular biology for generation of yeast strains\n",
      "cluster genes and localization tags\n",
      "yeast gene expression parts\n",
      "library assembly\n",
      "terminator-like spacer design\n",
      "generation of plasmids for galactose-induced expression\n",
      "growth of yeast clones and preparation of nif total and mitochondria enriched protein extracts\n",
      "antibodies and immunoblotting quantification\n",
      "preparation of anaerobic yeast cell-free extracts\n",
      "anoxic native gels and nafy binding\n",
      "nifdk activity\n",
      "nifdk purification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300055e/sb300055e.xml\n",
      "results and discussion\n",
      "generation of a tunable glucose flux platform\n",
      "range of in vivo expression of glk\n",
      "glk expression controls gluconate productivity\n",
      "summary and concluding remarks\n",
      "methods\n",
      "strains and plasmids\n",
      "culture conditions\n",
      "quantification of mrna levels\n",
      "enzyme activity assays\n",
      "metabolite analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00309/sb8b00309.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "design of a pdr12 promoter-based biosensor for smcfa\n",
      "characterization and verification of functionality of the biosensor\n",
      "the biosensor also responds to c 6 and c 7 but not c 10 or c 12 fa\n",
      "detection of c 8 fa by the biosensor in s. cerevisiae culture supernatants\n",
      "methods\n",
      "strains and plasmid construction\n",
      "cultures for fatty acid production\n",
      "fatty acid extraction and quantification via gc\n",
      "cultures for the biosensor assay\n",
      "biosensor measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400133g/sb400133g.xml\n",
      "results and discussion\n",
      "prediction of native threonine/isoleucine binding sites of hsd\n",
      "deregulation of the native feedback inhibitions of cghsd by threonine and isoleucine\n",
      "design of a nonnatural lysine-sensitive cghsd\n",
      "discussion\n",
      "methods\n",
      "homology modeling\n",
      "plasmids construction\n",
      "protein expression and purification\n",
      "enzyme assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00317/sb8b00317.xml\n",
      "results\n",
      "screening and identification of a transcriptional regulator for a positive response to dhs\n",
      "design and construction of a synthetic biosensor to monitor exogenous and endogenous dhs\n",
      "development and application of a high-throughput screening platform based on a genetically encoded biosensor for dhs overproduction\n",
      "discussion\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains, media, and growth and fermentation conditions\n",
      "dna manipulation\n",
      "transcriptome analysis\n",
      "rt-qpcr analysis\n",
      "determination of cell growth and fluorescence\n",
      "mutagenesis and library screening\n",
      "metabolite analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00112/sb7b00112.xml\n",
      "results and discussion\n",
      "a single round of pr generates a comprehensive mutation library\n",
      "recombineering libraries can be finely controlled to alter the composition of mutations\n",
      "screening the ilov library identifies mutations conferring thermostability\n",
      "recombineering-based multiplexing allows rapid and robust directed evolution\n",
      "materials and methods\n",
      "strains and media\n",
      "recombineering library construction\n",
      "library sequencing and analysis\n",
      "in silico effective library size simulation\n",
      "thermostability screening\n",
      "protein expression and in vitro characterization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000589/sb3000589.xml\n",
      "methods\n",
      "plasmid construction\n",
      "plasmid and cell maintenance\n",
      "testing cell growth in culture\n",
      "testing circuit transformation efficiency\n",
      "testing viability of individual plasmids\n",
      "serial dilution experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00505/sb8b00505.xml\n",
      "results and discussion\n",
      "a semiquantitative, high-throughput assay for identifying potential defects in engineered strains\n",
      "early detection of bugs using the semiquantitative phenotype assay improves efficiency\n",
      "improved assembly efficiency after applying a more efficient and flexible assembly strategy\n",
      "methods\n",
      "media and growth conditions\n",
      "flocculation assay\n",
      "phenotypic profiling assay\n",
      "hierarchical clustering analysis\n",
      "combination of long synthetic segments by crispr/cas9-induced gene conversion assembly (ciga)\n",
      "refining the assembly strategy by integrating swap-in and ciga\n",
      "identifying negative epistasis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00032/sb6b00032.xml\n",
      "materials and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00092/sb5b00092.xml\n",
      "results and discussion\n",
      "product synthesis\n",
      "enzyme production and validation\n",
      "operon construction\n",
      "comparative cell extract biotransformations\n",
      "construct optimization\n",
      "biotransformation optimization\n",
      "side reactions\n",
      "conclusions\n",
      "methods\n",
      "general reagents and procedures\n",
      "synthesis of isomenthone\n",
      "synthesis of menthol and neomenthol\n",
      "synthesis of neoisomenthol\n",
      "synthesis of isomenthol\n",
      "gene synthesis and modifications\n",
      "protein production and purification\n",
      "protein detection and identification\n",
      "enzyme kinetics\n",
      "operon construction\n",
      "optimization of multigene expression\n",
      "biotransformation reactions\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500045w/sb500045w.xml\n",
      "materials and methods\n",
      "culturing, genetics, protein purification, and blot preparation\n",
      "tale binding specificity on nitrocellulose\n",
      "dual tale capture\n",
      "results\n",
      "tale proteins can be immobilized and retain target specificity\n",
      "capturing target dna with two tales\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00289/sb5b00289.xml\n",
      "results\n",
      "geraniol synthesis in s. cerevisiae\n",
      "10-hydroxygeraniol synthesis in s. cerevisiae\n",
      "the iridoid synthase cr is is a promiscuous reductase\n",
      "in vitro enzyme activity of cr 10hgo- cr is enzyme pair\n",
      "discussion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "integration of the erg20 k197e mutation\n",
      "analysis of monoterpenoid production in s. cerevisiae\n",
      "10-hydroxygeraniol oxidoreductase and iridoid synthase assays\n",
      "chemicals\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500020g/sb500020g.xml\n",
      "results and discussion\n",
      "genaration of the pdc-adhb fusion\n",
      "confirmation of fused protein state\n",
      "ethanol production in bacteria expressing fused and nonfused enzymes\n",
      "discussion\n",
      "methods\n",
      "strain and growth conditions\n",
      "ethanol production assay\n",
      "adhb activity assay\n",
      "page and activity stains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00175/sb6b00175.xml\n",
      "a uv–violet/green photoreversible transcriptional regulatory tool in e. coli\n",
      "rapid and efficient one-step metabolic pathway integration in e. coli\n",
      "fast characterization of inducible promoters and optimization of p -coumaric acid production\n",
      "refactoring the six-gene photosystem ii core in the chloroplast of green algae\n",
      "cyma and exogenous flavins improve extracellular electron transfer and couple it to cell growth\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00215/sb8b00215.xml\n",
      "results and discussion\n",
      "establishment of a baseline thioesterase\n",
      "development of a lipoic acid-based selection\n",
      "library of cpfatb1.2 mutants\n",
      "characterization of m4 variant in vivo\n",
      "characterization of m4-287 variant in vitro\n",
      "optimizing expression of m4-287 in e. coli\n",
      "structural modeling insights\n",
      "materials and methods\n",
      "chemicals, reagents, and media\n",
      "dna synthesis and cloning\n",
      "random mutagenesis of cpfatb1.2\n",
      "lipoic acid selection\n",
      "fatty acid production\n",
      "fatty acid extraction and quantification\n",
      "protein expression and purification of apo-acyl carrier protein (acp)\n",
      "protein expression and purification of vibrio harveyi aass and bacillus subtilis sfp, cpfatb1.2 and cpfatb1.2-m4-287\n",
      "synthesis of octanoyl-acp\n",
      "liquid chromatography of octanoyl-acp\n",
      "in vitro analysis of cpfatb1.2-m4-287\n",
      "structural modeling of cpfatb1.2-m3 and cpfatb1.2-m4-287\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00160/sb9b00160.xml\n",
      "results and discussion\n",
      "methods\n",
      "molecular biology\n",
      "extract preparation\n",
      "cell-free experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00251/sb7b00251.xml\n",
      "results\n",
      "point mutation of glt a by crispr\n",
      "replacement of lpd a with k.p.lpda (d354 k) by crispr\n",
      "integration of synthetic 1,4-bdo pathway into e. coli by crispr\n",
      "enhanced 1,4-bdo production by crispri-mediated knockdown of endogenous genes\n",
      "discussion\n",
      "materials and methods\n",
      "e. coli strains\n",
      "plasmids construction\n",
      "genome engineering and strain development\n",
      "verification of genome engineering\n",
      "qrt-pcr for mrna quantification\n",
      "fermentation conditions\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00218/sb9b00218.xml\n",
      "results\n",
      "leaky expression of i-ceui efficiently triggered the gene disruption activity of transposable elements\n",
      "insertion sequences is1 and is10 have different mechanisms of transposition\n",
      "insertion sequences in i-ceui restore host fitness\n",
      "proteomics analysis links stress response pathways and activation of iss\n",
      "discussion\n",
      "transposable elements as a host defensive system\n",
      "tes could be significant to synthetic biology and evolution\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "cloning of the i-ceui gene into plo11\n",
      "identification of insertion event and frequency of transposition\n",
      "growth kinetics\n",
      "sequencing the insertion sequence and nucleotide sequence analysis\n",
      "sample preparation for proteomics and lc–ms/ms analysis\n",
      "raw data analysis and protein identification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00205/sb5b00205.xml\n",
      "results\n",
      "pamdb: a quantitative database for parts and modules\n",
      "database organization\n",
      "database content\n",
      "parameter inference with pamdb\n",
      "uncertainty reduction through data integration\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "simultaneous vs sequential fitting\n",
      "performance impact of the universal model\n",
      "discussion\n",
      "methods\n",
      "data curation\n",
      "data integration\n",
      "database design and implementation\n",
      "model\n",
      "parameter estimation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00310/sb6b00310.xml\n",
      "results and discussion\n",
      "ecad12 chimera triggers a ca 2+ signal with high μm extracellular free ca 2+\n",
      "frb and fkbp12 chimera system triggers a ca 2+ signal with rapamycin\n",
      "stim1 chimera triggers a ca 2+ signal with edta\n",
      "cam and mlckp chimera system triggers a ca 2+ signal with low μm extracellular free ca 2+\n",
      "a synthetic circuit with ecad12rec and carq triggers dynamic blebbing\n",
      "conclusion\n",
      "materials and methods\n",
      "plasmid construction\n",
      "cell culture and transfection\n",
      "cycloheximide chase assay\n",
      "illumination and imaging\n",
      "cell stimulation\n",
      "signal processing\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00261/sb8b00261.xml\n",
      "results and discussion\n",
      "ccrc2 mediates efficient excision of scc mec from mrsa genome\n",
      "mutations in attl site impair the excision by ccrc2\n",
      "combining ccrc2/crispr–cas9 to eliminate scc mec inside mrsa\n",
      "conclusions\n",
      "methods\n",
      "media and reagents\n",
      "electroporation\n",
      "construction of allelic exchange plasmids\n",
      "construction of pse1- ccrc2\n",
      "allelic replacements in s. aureus\n",
      "determination of the scc mec excision frequency\n",
      "isolation of scc mec -cured clone\n",
      "purification of ccrc2 protein and anti-ccrc2 antibody preparation\n",
      "electrophoresis mobility shift assay\n",
      "construction of the scc mec killer plasmid\n",
      "determining the elimination of methicillin resistance in mrsa strains\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400109p/sb400109p.xml\n",
      "results\n",
      "design of the fairytale system\n",
      "assembly process\n",
      "functional validation\n",
      "discussion\n",
      "methods\n",
      "substrate plasmid construction\n",
      "receiver plasmid construction\n",
      "13- and 15-piece golden gate assembly\n",
      "golden gate transfer reaction\n",
      "assembly efficiency determination\n",
      "gfp talen reporter assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00373/sb6b00373.xml\n",
      "results and discussion\n",
      "screening of genomic insertion sites for the trna cua -aars* pair\n",
      "construction and examination of hiv-aars* mutants\n",
      "construction and examination of hiv-trna-azfrs mutants\n",
      "construction and multicycle replication of hiv-uag-trna-azfrs variants\n",
      "conclusions\n",
      "methods\n",
      "materials and general methods\n",
      "construction of hiv-1 mutants\n",
      "transfection and generation of hiv-1\n",
      "qualitative infection assay\n",
      "virus ultracentrifugation\n",
      "multicycle infection assay\n",
      "tissue culture infectious dose 50 (tcid 50 )\n",
      "fluorescence spectroscopy and cell imaging\n",
      "quantification of viral load\n",
      "analysis of azfrs transcription using quantitative pcr\n",
      "viral transcription and provirus detection using rnascope and dnascope in situ hybridization\n",
      "amplification and sequencing for aars-trna cua insertion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400152n/sb400152n.xml\n",
      "results and discussion\n",
      "computational method\n",
      "bayesian parameter inference\n",
      "growth models and data selection\n",
      "activity computation\n",
      "response models and model selection\n",
      "implementation and graphical interface\n",
      "receiver characterization\n",
      "characterization robustness\n",
      "enhanced 3oc6hsl sensitivity via inducible expression of luxr\n",
      "methods\n",
      "plasmids\n",
      "plate fluorometer assays\n",
      "controls\n",
      "computation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00331/sb7b00331.xml\n",
      "results and discussion\n",
      "novel pathway identification and theoretical comparison\n",
      "screening for and characterizing recombinant phenol hydroxylase activity in e. coli\n",
      "investigating “phenol-dependent” catechol production in e. coli\n",
      "investigating ma production via newly engineered pathways\n",
      "engineering a synthetic “metabolic funnel” to enhance ma production\n",
      "comparing pathway performance and future outlooks\n",
      "materials and methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "strain construction\n",
      "thermodynamic and elementary flux mode (efm) analysis\n",
      "assaying phenol hydroxylase and catechol 1,2-dioxygenase activity using whole resting cells\n",
      "catechol and ma production from glucose\n",
      "hplc metabolite analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500349h/sb500349h.xml\n",
      "ralph p. diensthuber\n",
      "christopher engelhard\n",
      "gopal p. pathak\n",
      "min-hyung ryu\n",
      "sebastian r. schmidl\n",
      "ravi u. sheth\n",
      "devin strickland\n",
      "jeffrey j. tabor\n",
      "hui wang\n",
      "jason j. yi\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00165/sb8b00165.xml\n",
      "results and discussion\n",
      "multi-input chimeric repressor systems\n",
      "multi-input hybrid promoter systems\n",
      "single-input experiments\n",
      "naı̈ve modeling\n",
      "a caveat\n",
      "implementation and prediction of two-input hybrid promoter systems\n",
      "three-input prediction and measurement\n",
      "noise in multi-input promoters\n",
      "discussion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "growth and induction measurements\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00387/sb7b00387.xml\n",
      "results and discussion\n",
      "translation rate versus structure as the optimization goal for a riboregulator\n",
      "the tx–tl system linearly amplifies the concentration of rna and quadratically amplifies the concentration of dna\n",
      "analytical model of translational riboregulation and quantitative definition of the dynamic range of a riboregulator\n",
      "the dynamic range of a riboregulator strongly depends on the concentration of d ta\n",
      "comparison of the dynamic range of riboregulators in vivo and in vitro\n",
      "translation from rna characterizes the reaction between the cis-repressed and the trans-activator rna\n",
      "conclusion\n",
      "methods\n",
      "dna and rna preparations\n",
      "preparation of the pure tx–tl system\n",
      "fluorescence measurements in real-time pcr machine\n",
      "data processing\n",
      "electrophoretic mobility shift assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00174/sb7b00174.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "reagents and media\n",
      "strains\n",
      "plasmids\n",
      "sos response assays\n",
      "cas9 killing assays\n",
      "recombineering\n",
      "guide library design and construction\n",
      "library selection\n",
      "next generation sequencing\n",
      "data processing and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00025/sb5b00025.xml\n",
      "results\n",
      "target choice: hydroxystrictosidine\n",
      "microbial synthesis of tetrahydrobiopterin in s. cerevisiae\n",
      "optimization of tetrahydrobiopterin biosynthesis\n",
      "tetrahydrobiopterin recycling for pterin-dependent amino acid mono-oxidation\n",
      "microbial synthesis of biogenic amines via pterin-dependent mono-oxidation\n",
      "microbial synthesis of the modified mia 10-hydroxystrictosidine\n",
      "determining strictosidine synthase functionality\n",
      "discussion\n",
      "methods\n",
      "microbial synthesis of tetrahydrobiopterin\n",
      "microbial synthesis of l-dopa, dopamine, serotonin, and hydroxystrictosidine\n",
      "biopterin quantification\n",
      "quantification of l-dopa, dopamine, and serotonin\n",
      "analysis of hydroxystrictosidine\n",
      "hydroxystrictosidine isomer ratios\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00235/sb6b00235.xml\n",
      "methods\n",
      "cultivation of synechocytis for cell free extract preparation\n",
      "cell free extract preparation\n",
      "preparation of 2,5-anhydro- d -mannitol-1,6-diphosphate\n",
      "enzymatic activity assays\n",
      "96-well plate cultivation and screening system\n",
      "strains cultivated in multi-cultivator\n",
      "sequencing of the ldh cassette\n",
      "validation of the results from 96-well plate cultivation system\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000832/sb3000832.xml\n",
      "results and discussion\n",
      "media dependence of the logic gates\n",
      "gate function in an industrial strain\n",
      "connecting genetic circuits: impact of media and strains in shake flask\n",
      "circuit dynamics in a 1 ml mtp microreactor\n",
      "circuit dynamics in a 10 l bioreactor\n",
      "methods\n",
      "strains and plasmids\n",
      "shake flask and tube experiments\n",
      "flow cytometry\n",
      "fermentation\n",
      "biolector microreactor experiment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002467/sb5002467.xml\n",
      "results and discussion\n",
      "quantitative substrate profiles in isat reactions\n",
      "extending the duration of isat reactions with substrate replenishment\n",
      "methods\n",
      "s150 extract preparation\n",
      "total protein of 70s ribosomes (tp70) preparation\n",
      "isat batch reactions\n",
      "isat fed-batch reactions\n",
      "isat semi-continuous reactions\n",
      "sfgfp quantification\n",
      "inorganic phosphate quantitative assay\n",
      "nucleotide, amino acid, and phosphoenolpyruvate concentration analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00513/sb8b00513.xml\n",
      "results\n",
      "osmolyte-enhanced gene expression of fluorescent proteins\n",
      "osmolyte-enhanced gene expression of β-galactosidase\n",
      "venus expression using different dna concentrations\n",
      "venus expression from mrna\n",
      "gene expression analysis with sds-page\n",
      "discussion\n",
      "methods\n",
      "chemical reagents\n",
      "construction of dna and mrna templates\n",
      "cell-free protein synthesis using the pure system\n",
      "fluorescence quantification\n",
      "quantification with sds-page\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00071/sb9b00071.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400122c/sb400122c.xml\n",
      "lun cui\n",
      "michele forlin\n",
      "roberta lentini\n",
      "sean sleight\n",
      "francois st-pierre\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00181/sb7b00181.xml\n",
      "lital alfonta\n",
      "katja backhaus\n",
      "anke becker\n",
      "yonatan chemla\n",
      "johannes döhlemann\n",
      "akihiro furuya\n",
      "aravind natarajan\n",
      "ulrike obst\n",
      "eden ozer\n",
      "orr schlesinger\n",
      "stephanie t. schmidt\n",
      "gao-yi tan\n",
      "marcel wagner\n",
      "yongyi wei\n",
      "guihua zeng\n",
      "jingwen zhou\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400180p/sb400180p.xml\n",
      "next generation needs\n",
      "vision\n",
      "leadership\n",
      "resources\n",
      "responsibility\n",
      "expectations\n",
      "metrics\n",
      "venues\n",
      "emerging opportunities\n",
      "summary\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00019/sb6b00019.xml\n",
      "results and discussion\n",
      "pdh with a scavenging lipoylation pathway\n",
      "pdh mutants insensitive to nadh inhibition\n",
      "development of the lipoic acid auxotrophic strain as an in vivo acetyl-coa assay system\n",
      "pdh with a semisynthetic lipoylation pathway\n",
      "pdh with a de novo synthetic lipoylation pathway\n",
      "pdh with an unnatural de novo synthetic lipoylation pathway\n",
      "conclusions\n",
      "methods\n",
      "strains, media, and cultivation conditions\n",
      "dna manipulation\n",
      "strain construction\n",
      "growth assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00254/sb5b00254.xml\n",
      "results and discussion\n",
      "theoretical analysis of spatial instabilities\n",
      "two-dimensional computational model\n",
      "concluding remarks\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00315/sb6b00315.xml\n",
      "related work\n",
      "simbiotics\n",
      "implementation\n",
      "modeling\n",
      "environment\n",
      "spherical cells\n",
      "rod-shaped cells\n",
      "cell neighborhood\n",
      "physics\n",
      "chemistry\n",
      "biology\n",
      "cell internal dynamics, sbml and gillespie integration\n",
      "cell growth and death\n",
      "cell division\n",
      "spherical cells\n",
      "rod-shaped cells\n",
      "membrane transport\n",
      "active motility\n",
      "extracellular polymeric substances\n",
      "features\n",
      "analysis suite\n",
      "microscopy image processing\n",
      "case studies\n",
      "bacterial coaggregation in a fluid\n",
      "analysis\n",
      "aggregation at varied population densities\n",
      "biofilm development\n",
      "analysis\n",
      "discussion\n",
      "data plotting\n",
      "simbiotics coaggregation model\n",
      "simbiotics biofilm model\n",
      "computational details\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "stress tests\n",
      "performance scaling\n",
      "sphere and rod comparison\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00470/sb8b00470.xml\n",
      "results\n",
      "optimizing cyp-cpr combinations\n",
      "kah82-cpr activity over time tested by staggered substrate supplementation\n",
      "kah82-cpr pairs coexpressed with atr2\n",
      "coexpression of kos, kahs and cprs in a kaurene-producing cell\n",
      "balancing stoichiometry of ko75-kah82-srcpr1 by copy number modulation\n",
      "extracellular pathway intermediates\n",
      "discussion\n",
      "materials and methods\n",
      "strains and genetic materials\n",
      "sequence analysis\n",
      "cloning and integration of pathway in s. cerevisiae\n",
      "growth conditions\n",
      "kaurene, kaurenal, kaurenoic acid, steviol, and steviol glucoside analysis\n",
      "microscopy imaging\n",
      "immunoblotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00200/sb7b00200.xml\n",
      "results and discussion\n",
      "probing the tunability of common antibiotic resistance markers\n",
      "exploration of an alternative translational coupling architecture\n",
      "antibiotic-resistance-based selection of high expressers from tir variation libraries\n",
      "optimization of nanobody production by synthetic evolution of the tir\n",
      "optimization of affibody production by synthetic evolution of the tir\n",
      "conclusions\n",
      "methods\n",
      "strains, media and growth conditions\n",
      "plasmid and strain construction\n",
      "library design and construction\n",
      "expression and selection\n",
      "mic determination\n",
      "flow cytometry\n",
      "western blot analysis\n",
      "analyses of expression levels of nanobody molecules\n",
      "in-gel fluorescence measurements\n",
      "expression of affibody molecules\n",
      "analyses of expression levels and purity of affibody molecules\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00299/sb6b00299.xml\n",
      "methods\n",
      "strains and reagents\n",
      "assembly of full and partial streptomyces sp. f-525 refactored pathways\n",
      "culturing of strains and compound purification\n",
      "nuclear magnetic resonance (nmr) and mass spectrometry analysis\n",
      "bioactivity assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00085/sb6b00085.xml\n",
      "results and discussion\n",
      "immobilization system\n",
      "flocculation system\n",
      "test of sand cementation with si-tag and mcfp3 producing cells\n",
      "light induction system\n",
      "conclusion\n",
      "methods\n",
      "transformation of yarrowia lipolytica and genomic dna extraction\n",
      "immunofluorescence labeling for surface display\n",
      "silica binding test\n",
      "flocculation protein detection\n",
      "biological cementation test\n",
      "light induction test\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300134j/sb300134j.xml\n",
      "jeffrey dietrich\n",
      "nimish gera\n",
      "matthew harger\n",
      "mahmud hussain\n",
      "ingrid swanson pultz\n",
      "justin siegel\n",
      "elizabeth specht\n",
      "shen-long tsai\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400148y/sb400148y.xml\n",
      "genetic sensor for strong methylating compounds\n",
      "reliable prediction of complex phenotypes\n",
      "noise attenuation in biological switches\n",
      "expanding the boolean logic of prokaryotic xylr\n",
      "engineering the esar promoter\n",
      "tunable and multifunctional eukaryotic transcription factors\n",
      "tuning biological response curves\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00410/sb7b00410.xml\n",
      "results and discussion\n",
      "evolution of a receptor for cystatin c, a peptide biomarker of kidney failure\n",
      "clinical relevance of evolved receptor\n",
      "step-size analysis\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, reagents, and media\n",
      "strain and plasmid construction\n",
      "ste2mutagenesis\n",
      "receptor induction and flow cytometry\n",
      "high throughput sorting by receptor activity\n",
      "receptor response modeling\n",
      "dose response curve calculations\n",
      "linear and dynamic range calculations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00041/sb5b00041.xml\n",
      "results and discussion\n",
      "modular design of riboregulators\n",
      "evolution of the cis -repressor\n",
      "riboregulated translation\n",
      "targeting of the tarna to specific crrna-regulated genes\n",
      "trans-activator sequence can dictate the extent of translation\n",
      "comparison of riboregulator designs\n",
      "conclusions\n",
      "methods\n",
      "materials\n",
      "plasmid construction and bacterial transformations\n",
      "gene expression assays\n",
      "luminescence assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00149/sb6b00149.xml\n",
      "results and discussion\n",
      "overview of the platform: a two-step mechanism for morphogenetic programming\n",
      "patterning layer: symmetry breaking function for the establishment of up to four different domains per colony\n",
      "actuation layer: an ubiquitously distributed plasmid to encode domain-responsive functions\n",
      "domain-specific regulation of colonies: metabolic complementation, rnapt7 activation and crispri-regulation of gene expression\n",
      "prototyping morphogenetic instructions by symmetry-breaking and domain-specific cell growth regulation\n",
      "conclusions\n",
      "methods\n",
      "dna construction\n",
      "growth condition\n",
      "fluorescence quantification using fluorometry\n",
      "microscopy and image analysis\n",
      "computational modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00147/sb5b00147.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "computational design of orthogonal repressors\n",
      "experimental methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00489/sb8b00489.xml\n",
      "results and discussion\n",
      "determination of the copy number of the lovastatin synthetic gene cluster\n",
      "producing monacolin j without lovastatin residues\n",
      "improving the production of monacolin j by overexpressing regulatory factors\n",
      "achieving better cell robustness for monacolin j production with the a. terreus mutant hz-δlovf4-love\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and media\n",
      "whole genome resequencing\n",
      "construction of a. terreus chassis cell\n",
      "deletion of the multiple copies of lovf\n",
      "site-specific integration of the transcriptional regulator expression cassettes\n",
      "fermentation analysis of the engineered a. terreus strains\n",
      "lc–ms analysis of statins\n",
      "hplc analysis of fermentation cultures\n",
      "quantitative real-time pcr (qrt-pcr)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00478/sb8b00478.xml\n",
      "results\n",
      "modification of the protoplast-based polyethylene glycol and ca 2+ transformation method in f. fujikuroi\n",
      "selection of an efficient nuclear localization signal for establishment of the crispr/cas9 system in f. fujikuroi\n",
      "crispr/cas9-based gene disruption using different promoters to drive sgrna expression\n",
      "multigene disruption using the efficient crispr/cas9 system in f. fujikuroi\n",
      "rewriting the metabolic pathways in f. fujikuroi to change its ga product profile\n",
      "discussion\n",
      "materials and methods\n",
      "strains and culture media\n",
      "plasmid construction\n",
      "sgrna in vitro transformation\n",
      "dry cell weight measurement\n",
      "nuclear localization signal prediction\n",
      "analysis of gibberellic acids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00222/sb6b00222.xml\n",
      "results\n",
      "facile reprogramming of target specificity\n",
      "targeting rna aptamers for selective inactivation by ribozyme k28min\n",
      "site-specific, differential regulation of a hammerhead ribozyme by covalent modification\n",
      "engineering constraints\n",
      "discussion and conclusions\n",
      "methods\n",
      "product formation by k28(1–77)c ribozyme mutants\n",
      "atp binding by atp aptamer\n",
      "mango aptamer fluorescence\n",
      "rzb-4 activity assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400013w/sb400013w.xml\n",
      "results and discussion\n",
      "description of the components and their elementary interactions\n",
      "description of multiple interactions\n",
      "transcriptional control\n",
      "induction\n",
      "model parameters\n",
      "standard free energies of dimer-operator binding\n",
      "activation by cap\n",
      "free energy of looping\n",
      "free energy of lac repressor tetramerization\n",
      "repressor-inducer dissociation constant\n",
      "repression data\n",
      "system behavior for dimeric species\n",
      "system behavior for tetrameric species\n",
      "effects of parameter inference\n",
      "effects of operator deletion residual binding\n",
      "from dimer to tetramer binding\n",
      "experimental variability and model accuracy\n",
      "induction curves and activity kinetics\n",
      "conclusions\n",
      "methods\n",
      "model implementation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00080/sb5b00080.xml\n",
      "results and discussion\n",
      "generation of the synthetic injector e. coli (siec) strain\n",
      "expression of t3ss injectisomes in siec strain\n",
      "injectisome assembly interferes with flagella in siec\n",
      "electron microscopy of the injectisomes assembled in siec\n",
      "injection of proteins into mammalian cells using siec\n",
      "concluding remarks\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmids, dna constructs, and primers\n",
      "e. coli genome modification and strain construction\n",
      "sds-page and western blot\n",
      "t3ss-mediated erythrocyte hemolysis\n",
      "infection of cell cultures and fluorescence confocal microscopy\n",
      "injectisome purification\n",
      "electron microscopy\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00089/sb6b00089.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "material and methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00450/sb7b00450.xml\n",
      "methods\n",
      "dna cloning\n",
      "mammalian cell culture and transfection\n",
      "protoplast preparation and transformation\n",
      "rna isolation and real-time quantitative pcr analysis\n",
      "analytics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002938/sb5002938.xml\n",
      "results and discussion\n",
      "combining ifp fission with fusion to peptides\n",
      "ifp tolerates fission when fused to peptides\n",
      "fragmented ifps require assistance for fluorescence\n",
      "combining chea/chey fusion with ifp fission\n",
      "comparison with previous protein fragmentation studies\n",
      "future applications of random fission and rational fusion\n",
      "materials and methods\n",
      "library construction\n",
      "screening\n",
      "fluorescence analysis\n",
      "vector construction\n",
      "western blotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00356/sb8b00356.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna constructs\n",
      "transfections\n",
      "reagents\n",
      "imaging\n",
      "cell speed analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00297/sb5b00297.xml\n",
      "rna parts and tools\n",
      "ribosome binding sites\n",
      "riboswitches\n",
      "translation riboregulators\n",
      "ribozymes\n",
      "transcriptional regulators\n",
      "crispr/cas9\n",
      "rna circuits\n",
      "ribosomal circuitry\n",
      "small rna circuitry\n",
      "riboswitch circuitry\n",
      "crispr/cas9 circuitry\n",
      "applications\n",
      "riboswitch sensors for bioproduction\n",
      "small rna regulators for metabolic engineering\n",
      "rna scaffolds for biotechnology\n",
      "molecular diagnostics\n",
      "smart therapeutics\n",
      "cellular rna editing\n",
      "genome engineering\n",
      "future directions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00022/sb9b00022.xml\n",
      "results\n",
      "modeling the search of a transcription factor for its target\n",
      "system sensitivity to intergenic distance modulation\n",
      "nonlinearity of regulation response to intergenic distance\n",
      "space as a design parameter\n",
      "discussion\n",
      "methods\n",
      "a facilitated diffusion model of gene regulation\n",
      "assumptions taken by the model\n",
      "calculation of tf distribution along the chromosome\n",
      "calculation of search times\n",
      "binding rates\n",
      "stochastic simulations\n",
      "homogeneous model\n",
      "inducible system\n",
      "availability of the source code\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00179/sb9b00179.xml\n",
      "genome editing using sequence-specific nucleases\n",
      "mechanism of the nhej pathway in eukaryotes\n",
      "components of the nhej pathway\n",
      "functional principle of the nhej pathway\n",
      "mechanism of the nhej pathway in prokaryotes\n",
      "composition of the nhej machinery\n",
      "functional principle of the bacterial nhej pathway\n",
      "strategies for applying the nhej pathway in prokaryotes\n",
      "the rationale for introducing heterologous nhej pathways into prokaryotes\n",
      "importance of endogenous nhej pathways in prokaryotes\n",
      "bacterial stress response\n",
      "bacterial stationary phase mutation\n",
      "direct proof of end joining by the nhej pathway in vitro and in vivo\n",
      "combination of endogenous nhej pathways with the crispr system\n",
      "combination of exogenous nhej pathways with the crispr system\n",
      "interactions between the nhej pathway and the crispr/cas system\n",
      "challenges facing the research on nhej pathways\n",
      "strategies and prospects of genome editing based on the nhej pathway\n",
      "improving the accuracy of the nhej pathway\n",
      "improving the efficiency of the nhej pathway\n",
      "concluding remarks\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00176/sb5b00176.xml\n",
      "results\n",
      "minimal inhibited and activated modules\n",
      "structural properties\n",
      "the concentration of titrating species modulates the dose response threshold and the delay of the time response\n",
      "the feedback interconnection of inhibitor and activator modules creates a negative feedback loop and is a structural oscillator\n",
      "structural analysis\n",
      "numerical analysis\n",
      "mutually inhibiting modules create a positive feedback loop and a structural bistable system\n",
      "numerical analysis\n",
      "discussion and conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400093x/sb400093x.xml\n",
      "methods\n",
      "plasmid construction\n",
      "construction of the phenylalanine auxotroph strain\n",
      "library construction and selection\n",
      "growth curve characterization\n",
      "enzyme assays using crude cell extract\n",
      "l -homophenylalanine production with keto acid feeding\n",
      "kinetic parameter characterization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00108/sb7b00108.xml\n",
      "results and discussion\n",
      "design of a lexa variant capable of orthogonal regulation\n",
      "construction of synthetic circuit inside cells\n",
      "small-molecule inducible uv resistance\n",
      "small-molecule inducible mutagenesis through the synthetic circuit\n",
      "genetic and environmental requirements for sos-mediated mutagenesis\n",
      "conclusion\n",
      "materials and methods\n",
      "lexa alignment\n",
      "lexa tev cloning and expression\n",
      "in vitro protein analysis\n",
      "strain generation for cell-based assays\n",
      "western blotting\n",
      "cell-based fluorescent assays\n",
      "dna repair and mutagenesis assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00273/sb7b00273.xml\n",
      "results\n",
      "construction of the optogenetic system in p. aeruginosa\n",
      "performance of light-activated system in p. aeruginosa\n",
      "characterization of the light-induced system at the single-cell level\n",
      "disassembly of biofilms through blue light induction\n",
      "discussion\n",
      "methods\n",
      "stains and growth conditions\n",
      "plasmids and strains construction\n",
      "construction of c-di-gmp reporter plasmids\n",
      "most appropriate gene expression level of pa2133\n",
      "flow cell experiment\n",
      "image acquisition\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00045/sb8b00045.xml\n",
      "results and discussion\n",
      "isolation of thylakoids\n",
      "preparation of ta-cells\n",
      "light-driven electron transfer in ta-cells\n",
      "the inhibition of light driven electron transfer of light reaction\n",
      "conclusions\n",
      "methods\n",
      "materials and characterization\n",
      "extraction of thylakoids from spinach\n",
      "preparation of ta-cells\n",
      "light reaction in ta-cells\n",
      "inhibition of light-driven electron transfer in ta-cells\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00053/sb6b00053.xml\n",
      "results\n",
      "four-helix bundle domain hybrid receptor\n",
      "hbm domain hybrid receptor\n",
      "nit domain hybrid receptor\n",
      "single pdc domain hybrid receptor\n",
      "double pdc domain hybrid receptor\n",
      "characterizing ligand specificity of hybrid receptors\n",
      "discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "molecular cloning\n",
      "fret measurements\n",
      "agar plate assay\n",
      "microfluidic experiments\n",
      "immunoblotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300117q/sb300117q.xml\n",
      "james m. clomburg\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "yan kung\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "yanjun liu\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "jin hwan park\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "weerawat runguphan\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "anna venancio-marques\n",
      "current position\n",
      "education\n",
      "nonscientific interests\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00368/sb8b00368.xml\n",
      "results and discussion\n",
      "design and in vitro characterization of lov2 loop hinge mutants\n",
      "rlr and rvh mutations in lans and linx improve control over localization\n",
      "lans and linx mutants demonstrate improved control over transcription\n",
      "linx mutants demonstrate improved control over histone modifications\n",
      "conclusion\n",
      "methods\n",
      "reagents\n",
      "dna cloning\n",
      "protein purification\n",
      "fluorescence polarization\n",
      "mammalian cell culture and transfections\n",
      "mammalian cell imaging\n",
      "mammalian cell quantification\n",
      "yeast transcription\n",
      "western blotting\n",
      "western blot quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00216/sb5b00216.xml\n",
      "results and discussion\n",
      "crispri-based rfp repression\n",
      "crispri-based regulation of amino acid production by c. glutamicum\n",
      "quantification of the mrna levels of genes targeted by crispri\n",
      "quantification of enzymatic activity\n",
      "conclusion\n",
      "materials and methods\n",
      "microorganisms, plasmids and growth conditions\n",
      "integration of prim3_rfp in c. glutamicum\n",
      "crispri-based regulation of rfp expression in c. glutamicum\n",
      "crispri-based regulation of amino acid production by c. glutamicum\n",
      "quantification of the mrna levels of genes targeted by crispri\n",
      "enzyme activity assays\n",
      "amino acid quantification\n",
      "statistical treatment of data\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00333/sb8b00333.xml\n",
      "results\n",
      "multiplex ligation fidelity data predicts multifragment dna assembly accuracy\n",
      "use of fidelity predictions enables 12- and 24-fragment one pot assembly of lac cassettes\n",
      "discussion\n",
      "materials and methods\n",
      "preparation and pacific biosciences smrt sequencing of ligation fidelity libraries\n",
      "preparation and smrt sequencing analysis of ten-fragment golden gate assemblies\n",
      "golden gate cloning of 12 and 24 fragment lac cassettes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00215/sb6b00215.xml\n",
      "methods\n",
      "cloning and protein purification\n",
      "sgrna production and in vitro measurement of luminescence\n",
      "search for grna candidates throughout the m. tuberculosis genome\n",
      "single-target assay and target-array assay of m. tuberculosis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00220/sb5b00220.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00002/sb5b00002.xml\n",
      "results and discussion\n",
      "introduction of the s. flexneri type 3 secretion system into e. coli\n",
      "regulation of expression of type 3 secretion in mt3 e. coli\n",
      "mt3 e. coli secrete native type 3 effectors\n",
      "mt3 e. coli can deliver effectors directly into the cytosol of mammalian cells\n",
      "development of a screening platform to identify optimal type 3 secretion sequence–target protein combinations\n",
      "type 3 secretion sequences can affect heterologous protein activity by altering protein stability or localization\n",
      "flexibility of recognition of heterologous proteins as type 3 secreted substrates\n",
      "mt3 e. coli invade but do not replicate or induce cytotoxicity of mammalian target cells\n",
      "summary\n",
      "methods\n",
      "construction of the t3ss capture vector\n",
      "generation of the pmt3ss plasmid\n",
      "integration of the shigella t3ss into the e. coli chromosome\n",
      "expression plasmids\n",
      "secretion assay\n",
      "translocation assay/ccf4 assay\n",
      "cytotoxicity assays\n",
      "immunofluorescence\n",
      "cell culture conditions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00439/sb7b00439.xml\n",
      "results and discussion\n",
      "design and characterization of a ccm sensor-selector\n",
      "sensor-selector validation in production strain\n",
      "ph tuning of the sensor-selector system\n",
      "high-throughput screening of a ccm production strain library\n",
      "methods\n",
      "strains, chemicals and media\n",
      "plasmids and strain construction\n",
      "library enrichment\n",
      "growth rate determination\n",
      "ccm production assays\n",
      "flow cytometry analysis of ccm administration assays and cocultures\n",
      "genomic dna extraction\n",
      "whole genome sequencing and copy number analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00162/sb5b00162.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00376/sb7b00376.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00238/sb8b00238.xml\n",
      "results\n",
      "mpusfus is functional in e. coli\n",
      "new molecular tools and protocols to engineer synechococcus sp. pcc 7002\n",
      "mpusfus is functional in synechococcus sp. pcc 7002\n",
      "discussion\n",
      "use of fusion genes for biotechnology\n",
      "methods\n",
      "construction of paq1bb vector for transformation of synechococcus sp. pcc 7002\n",
      "generation of codon-optimized mpusfus\n",
      "generation of transgenic synechococcus sp. pcc 7002 strains\n",
      "synechococcus sp. pcc 7002 growth\n",
      "generation of transgenic e. coli strain\n",
      "measurement of gfp\n",
      "mannitol analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00185/sb7b00185.xml\n",
      "results and discussion\n",
      "structural accessibility profiling suggests csrb substructures differentially contribute to srna function\n",
      "single nucleotide mutation of substructures produces a limited range of csrb regulatory activity\n",
      "rational modular redesign of csrb yields broad gradient of regulatory activity\n",
      "csrb variants tune control of multiple targets to impact phenotype\n",
      "conservation analysis of csrb offers insight into substructures’ sensitivity to engineering\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, plasmid construction, and culture conditions\n",
      "rna structure prediction\n",
      "selection of csrb substructures for engineering\n",
      "design of csrb single nucleotide mutations\n",
      "rational modular rna redesign builds csrb variant library\n",
      "fluorescent translational reporter assays\n",
      "glutamate metabolite assay\n",
      "csrb conservation analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00026/sb9b00026.xml\n",
      "results and discussion\n",
      "metabolic rewiring strategy\n",
      "component simplification for extracellular siderophore biosynthesis\n",
      "further enhancement of fsc production by sequential gene overexpression\n",
      "transcriptional analysis of l -ornithine and siderophore metabolic genes\n",
      "in vitro and in vivo active targeting of t. rubrum and c. albicans by fsc\n",
      "methods\n",
      "strains and media\n",
      "cre/ lox systems\n",
      "sequential genetic modification using the cre/ lox system\n",
      "analysis and quantification of the total siderophore, l -ornithine, fsc and fsb contents\n",
      "qpcr\n",
      "utilization of the [fe 3+ ] fsc chelates by t. rubrum and c. albicans as an iron source\n",
      "in vitro fluorescence tracing assay\n",
      "in vivo near-infrared fluorescence imaging assay\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00435/sb7b00435.xml\n",
      "results and discussion\n",
      "analysis of the e. coli rhabad promoter in a heterologous synechocystis context\n",
      "l -rhamnose is not metabolized by nor toxic to synechocystis\n",
      "characterization of rhabad promoter system in synechocystis\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "strain construction\n",
      "assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00085/sb8b00085.xml\n",
      "weixi liao\n",
      "chang liu\n",
      "stephen mann\n",
      "dastor margaux\n",
      "verónica s martínez\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00177/sb9b00177.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture\n",
      "production of replication-incompetent vsv vector by reverse genetics\n",
      "virus infection\n",
      "reversible gene regulation using the vsv vector\n",
      "quantification of viral rnas\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000673/sb3000673.xml\n",
      "a21978c (including daptomycin) and a54145\n",
      "rationale for the development of combinatorial biosynthesis of acidic lipopeptide antibiotic pathways\n",
      "a21978c and a54145 genes encode lipopeptides with identical peptide stereochemistry\n",
      "enzyme subunit, module, and domain organization of the a21978c and a54145 nrps genes\n",
      "genes involved in lipoinitiation\n",
      "genes involved in amino acid modifications\n",
      "protein–protein docking interactions\n",
      "mbth-like chaperones\n",
      "regulatory, editing, resistance, and transport devices\n",
      "microbial factories (chassis) and genetic toolkits (devises) for combinatorial biosynthesis\n",
      "combinatorial biosynthesis\n",
      "prospects for mutagenesis and selection for robust production\n",
      "discussion and concluding comments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400116x/sb400116x.xml\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400019s/sb400019s.xml\n",
      "results and discussion\n",
      "c-terminal phage vector display of the sortase substrate motif\n",
      "c-terminal sortase-mediated labeling of piii\n",
      "orthogonal labeling of three phage capsid proteins\n",
      "labeling of piii and pix with dna\n",
      "formation of ordered phage trimers\n",
      "conclusions\n",
      "methods\n",
      "phage engineering\n",
      "sortase-mediated reactions\n",
      "dna peptide conjugation\n",
      "atomic force microscopy and dynamic light scattering\n",
      "fluorescence microscopy\n",
      "miscellaneous\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00421/sb7b00421.xml\n",
      "results\n",
      "selection of candidate amber suppressor trna variants for surveying determinants of orthogonality\n",
      "analysis of high abundance suppressor trnas reveals that their orthogonality is impacted by anticodon modifying enzymes\n",
      "evolved trna suppressor variants show low abundance and demonstrate reduced dependence on post-transcriptional modification machinery\n",
      "low abundance trna s7 displays reduced dependence on ptm enzymes in the presence of its cognate aars\n",
      "post-transcriptional modification mapping by mass spectrometry confirms high abundance trnas are modified by miaa and trua\n",
      "modified nucleoside analysis in total trna pool indicates alterations in native e. coli ptm profiles in cells hosting heterologous suppressor trnas\n",
      "discussion\n",
      "materials and methods\n",
      "cell culture\n",
      "e. coli host strains\n",
      "quantification of trna variants\n",
      "orthogonality assays\n",
      "suppression assays\n",
      "purification of total trna (<250 nt)\n",
      "lc–ms/ms\n",
      "rna modification mapping by lc–ms/ms\n",
      "statistical analysis\n",
      "correlation between orthogonality and trna abundance\n",
      "orthogonality assays\n",
      "suppression assays\n",
      "total trna modification profiling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500047r/sb500047r.xml\n",
      "results and discussion\n",
      "methods\n",
      "bacterial strains, plasmids, growth conditions, and transformation procedures\n",
      "pcr, cloning, and site directed mutagenesis reactions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00405/sb7b00405.xml\n",
      "results and discussion\n",
      "gas reporter characterization\n",
      "gas reporting of signaling compounds\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "monitoring biological signal synthesis and degradation\n",
      "implications\n",
      "methods\n",
      "materials\n",
      "growth medium\n",
      "bacterial strains and plasmids\n",
      "headspace gas analysis\n",
      "efe and mht temperature dependence\n",
      "soil matrices\n",
      "constitutive gas production in soils\n",
      "soil bacteria extraction from soil\n",
      "measuring ahl following addition to soils\n",
      "ahl degradation by bacillus\n",
      "rhizobium ahl production dynamics\n",
      "ahl-sensing by shewanella oneidensis mr-1\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00204/sb8b00204.xml\n",
      "results and discussion\n",
      "two components of leakage translation\n",
      "suppression of the ltas by positive feedback derepression\n",
      "suppression of the ltabr by multiplexed tag insertion\n",
      "tunability, response rate, and reversibility\n",
      "effect of positive feedback derepression circuits in another uaars/trna cua system\n",
      "concluding remarks\n",
      "materials and methods\n",
      "strains, growth conditions, and transfection\n",
      "plasmid construction\n",
      "unnatural amino acids (uaa’s)\n",
      "egfp reporter assay\n",
      "western blot\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00377/sb8b00377.xml\n",
      "results and discussion\n",
      "spatial distribution introduces a new control knob to the conventional gene regulatory network\n",
      "dimensionality reduction enables steady-state analysis of multicompartmental cross-repressive networks\n",
      "graphical analysis of a dli circuit predicts two mechanisms of contrasting pattern formation\n",
      "dli network design and implementation require diffusible cross-repression and a geometric culturing platform\n",
      "bistable and contrasting gene expression was observed for dli circuit in liquid coculture\n",
      "bifurcation is observed for cells grown on the geometric culturing platform\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains, plasmid construction, and growth conditions\n",
      "plate reader assays\n",
      "flow cytometry\n",
      "construction of dli device\n",
      "dli assays\n",
      "computational modeling and simulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00052/sb6b00052.xml\n",
      "results\n",
      "pathway reconstruction for succinate production from acetate\n",
      "succinate production from acetate\n",
      "transcriptome and qpcr analysis of mg03 and mg1655\n",
      "overexpression of citrate synthase gene\n",
      "biosynthesis of succinate by resting cells\n",
      "discussion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "medium and cultivation\n",
      "analytical methods\n",
      "enzymatic activity assay\n",
      "rna-sequencing\n",
      "qrt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00232/sb9b00232.xml\n",
      "results\n",
      "construction of the crispri system\n",
      "tunable and durable crispri silencing of paaa suppresses growth on phenylacetic acid\n",
      "single-cell analysis reveals an “all or none” effect in b. cenocepacia k56–2\n",
      "broad host range of the mini-ctx system extends applicability to other species\n",
      "discussion\n",
      "methods\n",
      "strains, selective antibiotics, and growth conditions\n",
      "construction of psc-rhadcas9, pah-ctx1-rhadcas9, pah-ctx1-rhadcas9-native, and dcas9 insertional mutants\n",
      "design and construction of the grna-expressing plasmids\n",
      "construction of insertional mutants k56–2 flif::pah26 and k56–2 phbc::pah27\n",
      "assays for integration efficiency and stability of the mini-ctx1-based system\n",
      "growth assay with phenylacetic acid as the sole carbon source\n",
      "fluorescent microscopy and polyhydroxyalkanoate granule detection\n",
      "plate-based motility assay\n",
      "flagellum staining\n",
      "sds-page and immunoblotting\n",
      "rna extraction and reverse transcription quantitative pcr (rt-qpcr) analysis\n",
      "plasmid availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00350/sb7b00350.xml\n",
      "methods\n",
      "materials\n",
      "dna construction\n",
      "in vitro protein synthesis\n",
      "gtpase assay\n",
      "preparation of svs\n",
      "flotation assay\n",
      "protein synthesis inside gvs and imaging\n",
      "image analysis and quantification of membrane deformation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00140/sb9b00140.xml\n",
      "results and discussion\n",
      "reconstitution of spo0a-based gene expression regulation system in b. subtilis\n",
      "design and construction of phr60-rap60-spo0a qs system\n",
      "boosting the synthesis of mk-7 precursors in b. subtilis\n",
      "dynamic fine-tuning of mk-7 synthesis in b. subtilis by bifunctional phr60-rap60-spo0a qs system\n",
      "conclusions\n",
      "methods\n",
      "microorganism and medium\n",
      "shake flask culture of engineered b. subtilis\n",
      "b. subtilis transformation and dna assembly\n",
      "in situ replacement of promoter, genomic integration of heterogeneous genes and gene knockout\n",
      "mk-7 extraction\n",
      "sporulation assays\n",
      "analytical methods\n",
      "quantitative reverse-transcriptase pcr (qrt-pcr) analysis\n",
      "fed-batch culture in 15-l bioreactor\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000386/sb3000386.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300116y/sb300116y.xml\n",
      "results and discussion\n",
      "comprehensive analysis of terminator region activity\n",
      "most and least active 30 terminator regions in yeast\n",
      "least active terminators reduce the mrna levels\n",
      "biological characterization of the most and least active terminator regions\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "pcr-primer design\n",
      "first and second pcrs to amplify terminator regions\n",
      "gap-repair cloning\n",
      "flow cytometry and data processing\n",
      "construction and evaluation of stable transformants\n",
      "effects of promoter and reporter gene exchange on 3′-utr activities\n",
      "quantitative pcr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb2000287/sb2000287.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00211/sb6b00211.xml\n",
      "materials and methods\n",
      "yeast strains, selection, and anaerobic fermentation\n",
      "protein purification\n",
      "competition assay and kinetic parameters\n",
      "4- o -β- d -glucopyranosyl- d -xylose (gx) synthesis assay\n",
      "cbp homology and modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00416/sb8b00416.xml\n",
      "materials and methods\n",
      "bacteria growth conditions\n",
      "plasmids and bacterial strains\n",
      "preparation of pea/pa/da/l-dopa solutions\n",
      "comparisons of various sensor strains\n",
      "gfp and rfp intensity quantifications\n",
      "specificity of sensor strain toward various substrates\n",
      "preparation of environmental samples\n",
      "fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00006/sb7b00006.xml\n",
      "results\n",
      "putr library preparation\n",
      "transcriptional strength measurement\n",
      "fluorescence strength measurements across different growth phases\n",
      "comparing the results of rt-qpcr and fluorescence measurements\n",
      "putr engineering for enhanced gene expression\n",
      "rfp and β-galactosidase expression by engineered putrs\n",
      "discussion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "medium and culture conditions\n",
      "promoter library construction\n",
      "putr engineering\n",
      "real-time fluorescence quantitative pcr (rt-qpcr)\n",
      "fluorescence measurements\n",
      "measurement of β-galactosidase activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00163/sb5b00163.xml\n",
      "results and discussion\n",
      "representation\n",
      "synthesizing crns for computing polynomials\n",
      "designing the crn\n",
      "initialization\n",
      "proof based on the mass-action kinetics\n",
      "encoding and decoding\n",
      "encoding\n",
      "decoding\n",
      "methods and materials\n",
      "dna implementation\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00361/sb8b00361.xml\n",
      "results and discussion\n",
      "development of mi castar using the model tam gene cluster\n",
      "optimization of the initial in vitro cas9 gene cluster digestion step\n",
      "gene cluster reassembly with marker-free promoter cassettes\n",
      "systematic refactoring of the tam gene cluster using single-marker mi castar\n",
      "effect of bac size on refactoring efficiency\n",
      "comparison of mi castar and mcristar methods\n",
      "application of mi castar to the discovery of novel natural products from the ato gene cluster\n",
      "cloning of the ato gene cluster using tar\n",
      "parallelized, combinatorial refactoring of the ato gene cluster using mi castar\n",
      "analysis of metabolites produced by s. albus transformed with differentially refactored ato gene clusters\n",
      "atolypene isolation, structure elucidation, bioactivity, and biosynthesis\n",
      "conclusions\n",
      "materials and methods\n",
      "in vitro sgrna transcription and purification\n",
      "in vitro bac dna digestion with cas9\n",
      "pcr amplification of cluster-specific promoter cassettes and in-yeast assembly\n",
      "metabolite analysis of s. albus transformed with refactored tam gene clusters\n",
      "tar cloning of the ato gene cluster\n",
      "metabolite analysis of s. albus transformed with refactored ato gene clusters\n",
      "isolation of atolypene a and b\n",
      "pgme derivatization of atolypene a\n",
      "antimicrobial assays of atolypene a and b\n",
      "cytotoxicity assessment for atolypene a and atolypene b\n",
      "measurement of [ 14 c] l -leucine uptake\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00165/sb5b00165.xml\n",
      "michael asensio\n",
      "ana mafalda cavaleiro\n",
      "bernardo cervantes\n",
      "william deloache\n",
      "jeff glasgow\n",
      "christopher jakobson\n",
      "se hyeuk kim\n",
      "michael e. lee\n",
      "yunzi luo\n",
      "kiavash mirzadeh\n",
      "ragan pitner\n",
      "david ruano-gallego\n",
      "yaojun tong\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00005/sb9b00005.xml\n",
      "results and discussion\n",
      "deletions of ccpc and ccpn improved bacitracin production\n",
      "effects of nitrogen metabolism regulators gene cody , tnra , and glnr deletions on amino acid redistributions and bacitracin production\n",
      "strengthening cody, tnra, and glnr expression improved intracellular bcaa supplies for bacitracin synthesis\n",
      "deletion of phop improved bacitracin production\n",
      "combinatorial metabolic engineering to improve bacitracin yield\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, plasmids, and cultivation conditions\n",
      "gene knockout in b. licheniformis\n",
      "construction of gene expression vector\n",
      "construction of gene integration overexpression strain\n",
      "analytical methods\n",
      "transcriptional level assay\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00087/sb5b00087.xml\n",
      "results and discussion\n",
      "the genome calligrapher dna synthesis optimization web tool\n",
      "dna synthesis constraints across bacterial genomes\n",
      "level of recoding required for removal of dna synthesis constraints\n",
      "design, streamlining, and partial synthesis of a synthetic essential genome\n",
      "summary and conclusions\n",
      "methods\n",
      "web tool availability and license\n",
      "genome calligrapher web tool implementation\n",
      "calculation of codon usage tables\n",
      "design, refactoring and partial synthesis of the synthetic essential genome construct\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002533/sb5002533.xml\n",
      "results and discussion\n",
      "toy model\n",
      "circuit\n",
      "input/output dose response curve\n",
      "trade-offs with dynamic response\n",
      "band-pass filtering of stimulus amplitude\n",
      "methods\n",
      "circuit construction\n",
      "bacterial strains, media, and growth conditions\n",
      "steady state and dynamics experiments\n",
      "flowcytometry analyses\n",
      "modeling and simulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00432/sb8b00432.xml\n",
      "results and discussion\n",
      "implementation of the tn-core toolbox\n",
      "generation of core metabolic models\n",
      "evaluation of redundancy\n",
      "refinement of gprs in genome-scale metabolic models\n",
      "validation of tn-core functionality\n",
      "analysis of the accuracy of tn-core using a s. meliloti gsmr\n",
      "comparison of s. meliloti core model extraction by tn-core to other approaches\n",
      "validation of tn-core using p. aeruginosa and r. sphaeroides gsmrs\n",
      "limitations of tn-core for extraction of core metabolic models\n",
      "use of tn-core for identifying redundancies within a gsmr\n",
      "refinement of gsmrs with tn-seq data\n",
      "conclusions\n",
      "methods\n",
      "general metabolic modeling procedures\n",
      "metabolic models and -omics data sets\n",
      "core model extraction for benchmarking\n",
      "kegg functional mapping\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400003y/sb400003y.xml\n",
      "results and discussion\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in vitro expression of fluorescent proteins\n",
      "influence of sequences upstream of the ribosome binding site\n",
      "influence of sequences downstream of the ribosome binding site\n",
      "considerations for the assembly of in vitro genetic systems\n",
      "methods\n",
      "genetic constructs\n",
      "transcription-translation reactions\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00054/sb5b00054.xml\n",
      "results and discussion\n",
      "development of strand displacement selections for riboswitches\n",
      "model system for selection\n",
      "strand displacement can be used to select rna-based sensors\n",
      "strand displacement selection yields riboswitches that are active in vitro\n",
      "strand displacement selections for the engineering of nucleic acid circuitry\n",
      "methods\n",
      "chemicals, oligonucleotides, dna, and rna production\n",
      "real-time detection of strand displacement\n",
      "selection by strand displacement\n",
      "evaluation of selection cycles\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200001q/sb200001q.xml\n",
      "results and discussion\n",
      "design of artificial srna libraries\n",
      "screening of artificial srnas targeting ompf 5′ utr\n",
      "characterization of artificial srnas targeting ompf 5′ utr\n",
      "screening of artificial srnas targeting flic 5′ utr\n",
      "characterization of artificial srnas targeting flic 5′ utr\n",
      "mutational analysis of aflic-12.3\n",
      "summary and conclusion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "library construction\n",
      "library screening\n",
      "fluorescent reporter gene assays for artificial srna activities\n",
      "expression levels of outer membrane porins\n",
      "expression levels of flagellin\n",
      "swimming motility assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00165/sb6b00165.xml\n",
      "results and discussion\n",
      "design of short synthetic terminators\n",
      "activities of synthetic terminators evaluated by fluorescent protein expression\n",
      "design and construction of bac vectors for the assembly of transcription units and genomic integration\n",
      "transcriptional interference in tandem synthetic tus\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "design of random linkers and unss\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00526/sb8b00526.xml\n",
      "results\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500249g/sb500249g.xml\n",
      "results and discussion\n",
      "experimental design\n",
      "measurement of gene expression in the artificial cell\n",
      "correlated and uncorrelated noise in gene expression\n",
      "theoretical model for predicting the expression noise\n",
      "comparison with the model\n",
      "comparison with e. coli\n",
      "discussion\n",
      "methods\n",
      "construction and preparation of the plasmid\n",
      "in vitro transcription and translation system\n",
      "preparation of cell-seized vesicles for encapsulating the in vitro transcription and translation system\n",
      "gene expression in cell-seized vesicles and fcs measurement\n",
      "gene expression in e. coli.\n",
      "measurement of dna and mrna in e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000172/sb3000172.xml\n",
      "results\n",
      "conceptual design of a rac1-cam chimera\n",
      "racer is a ca 2+ -activated rac1 chimera\n",
      "racer is robustly activated by several distinct ca 2+ -mobilizing signals\n",
      "directed cell migration mediated by racer\n",
      "a ca 2+ -sensitive cdc42 using the racer design\n",
      "discussion\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture and transfection\n",
      "reagents used\n",
      "imaging and illumination\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00249/sb7b00249.xml\n",
      "results and discussion\n",
      "optimization of precursor supply for terpenoid production\n",
      "screening of optimal promoters necessary for terpenoid production\n",
      "optimized expression of terpenoid-biosynthetic genes by fur22\n",
      "materials and methods\n",
      "chemicals\n",
      "bacterial strains, plasmids, and culture conditions\n",
      "dna isolation\n",
      "in-frame deletion of fur1 gene\n",
      "vector construction and transformation\n",
      "total rna isolation for rna-seq analysis\n",
      "protein extraction for 2-d gel electrophoresis\n",
      "protein labeling and 2-d gel electrophoresis\n",
      "experimental design and data processing for 2-d profiling\n",
      "protein determination\n",
      "screening of intrinsic promoters for fq production\n",
      "evaluation of bc production\n",
      "metabolite extraction\n",
      "lc–ms analysis\n",
      "gc–ms analysis\n",
      "metabolite quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00437/sb8b00437.xml\n",
      "results and discussion\n",
      "development of a “two-spacers” strategy for genome editing in anabaena with a high success rate\n",
      "multiple genome editing in anabaena\n",
      "counter selection with sacb facilitates curing of editing plasmid from cells\n",
      "create conditional mutants of anabaena with crispr-cpf1\n",
      "delete large chromosomal regions with crispr-cpf1\n",
      "conclusions\n",
      "materials and methods\n",
      "growing cyanobacterial strains\n",
      "construction of vectors and editing plasmids\n",
      "construction of genome editing vectors\n",
      "construction of the editing plasmids for single gene deletion\n",
      "construction of the plasmids for hetr/hetn double mutation\n",
      "construction of the plasmid for making the conditional mutant of pola\n",
      "construction of the plasmid for large chromosomal region deletion\n",
      "construction of cyanobacterial strains\n",
      "testing the viability of trs- pola strain\n",
      "curing of the editing plasmid\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00465/sb7b00465.xml\n",
      "results and discussion\n",
      "model derivation and validation\n",
      "metabolic flux distributions\n",
      "theoretical optimal flux distribution\n",
      "experimentally constrained flux distribution\n",
      "analysis of cfps performance\n",
      "productivity\n",
      "energy efficiency\n",
      "global sensitivity analysis\n",
      "potential alternative metabolic optima\n",
      "summary and conclusions\n",
      "materials and methods\n",
      "glucose/nmp cell-free protein synthesis\n",
      "protein product and metabolite measurements\n",
      "formulation and solution of the model equations\n",
      "calculation of energy efficiency\n",
      "quantification of uncertainty\n",
      "global sensitivity analysis\n",
      "potential alternative optimal metabolic flux solutions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001917/sb4001917.xml\n",
      "methods\n",
      "materials\n",
      "preparation of the afce\n",
      "liposome preparation and condensation of macromolecules in liposomes\n",
      "quantification of gfp concentration\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00080/sb6b00080.xml\n",
      "results and discussion\n",
      "selecting a crispr/cas9 dna cleavage approach for multiplex promoter engineering\n",
      "development of mcristar using promoter cassettes containing auxotrophic markers\n",
      "development of single-marker mcristar\n",
      "validation of single-marker mcristar using additional biosynthetic gene clusters\n",
      "construction of marker-free promoter cassettes for use in the heterologous expression of refactored gene clusters in streptomyces sp\n",
      "activation of the silent tam gene cluster using single-marker mcristar\n",
      "conclusions\n",
      "methods\n",
      "yeast strain, plasmids, and culture conditions\n",
      "construction of the pathway-specific crispr plasmid\n",
      "mcristar protocol\n",
      "pcr amplification of cluster-specific promoter cassettes\n",
      "refactoring of the tam cluster using two rounds of mcristar\n",
      "hplc analysis of tetarimycin a production from refactored tam gene clusters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00366/sb7b00366.xml\n",
      "materials and methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "in vivo fluorescence measurements\n",
      "model feature quantification\n",
      "partial least-squares (pls) regression\n",
      "search algorithm for de novo design of 5′utrs\n",
      "cultivation of p -coumaric acid production strains\n",
      "detection and quantification of p -coumaric acid\n",
      "statistical calculations\n",
      "results and discussion\n",
      "development of the yutr calculator\n",
      "the yutr calculator compared to the dvir model\n",
      "universal applicability of the yutr calculator\n",
      "protein coding sequence influence and reverse engineering\n",
      "proof of concept: reliable p -coumaric acid production\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500306z/sb500306z.xml\n",
      "synthetic gene-overexpression for cell-type ratio control via reprogramming\n",
      "framework for engineering finite state machines\n",
      "signaling circuits with synthetic, post-translational, negating boolean logic devices\n",
      "strategy revealing phenotypic differences among synthetic oscillators\n",
      "modular, multi-input transcriptional logic gating\n",
      "expanding the logic of bacterial promoters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000564/sb4000564.xml\n",
      "response curves: models and mechanisms\n",
      "types of tuning\n",
      "vertical scaling\n",
      "vertical shifting\n",
      "vertical extension\n",
      "leakage\n",
      "horizontal scaling\n",
      "steepness\n",
      "dynamic range\n",
      "biological options for tuning\n",
      "transcription\n",
      "gene copy number and location\n",
      "promoter modifications\n",
      "operator site modification\n",
      "promoter escape\n",
      "modular transcription factor domains\n",
      "signal-sensor domains\n",
      "trans -activator and trans -silencer domains\n",
      "tal-effectors and zinc fingers\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "orthogonal rna polymerases\n",
      "nuclear localization and export sequences\n",
      "nuclesome-disfavoring sequences\n",
      "post-transcription\n",
      "ribosome binding site modifications\n",
      "orthogonal ribosomes\n",
      "start codon modification\n",
      "codon optimization\n",
      "riboregulators\n",
      "aptamers\n",
      "transcriptional attenuators\n",
      "mrna degradation control\n",
      "rna interference\n",
      "tunable intergenic regions\n",
      "protein-based systems\n",
      "post-translation\n",
      "dose-responsive enzymatic catalysis\n",
      "protein switches\n",
      "structure rescue\n",
      "protein half-life modulation\n",
      "scaffolding\n",
      "beyond rate tuning\n",
      "network extension\n",
      "linear cascades: sensitivity tuners\n",
      "feedback loops: engineered scaffold interactions\n",
      "feedback loops: bistability\n",
      "parallel pathways: reducing leakage\n",
      "consistency and modularity\n",
      "insulated promoter-gene cassettes\n",
      "promoter position relative to ori\n",
      "crispr editing\n",
      "ribozyme-based insulator parts\n",
      "noise\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00187/sb6b00187.xml\n",
      "results and discussion\n",
      "construction of t7 promoter library that covers a broad range of transcriptional dynamics\n",
      "plackett–burman design to screen the main effects that determine violacein pathway efficiency\n",
      "box–behnken design to determine the optimal expression level of the main pathway components\n",
      "correlation between gene expression and violacein production may follow weber’s law and statistical thermodynamic theory\n",
      "conclusions\n",
      "materials and methods\n",
      "t7 promoter library construction and screening\n",
      "epathbrick-directed modular assembly of the violacein pathway\n",
      "media and growth conditions\n",
      "metabolite extraction and analysis\n",
      "data analysis and model fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500262f/sb500262f.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "rationale for choosing chimera mutants\n",
      "assay for fluorescent protein production\n",
      "and transcriptional logic with laci/galr chimeras\n",
      "multi-input and transcriptional logic with laci/galr chimeras\n",
      "multi-orf and and transcriptional logic with laci/galr chimeras\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00087/sb6b00087.xml\n",
      "results and discussion\n",
      "search for parts in genomes of streptomycetes\n",
      "characterization of the regulatory mechanism of otrr\n",
      "determination of otrr operator otro and the core promoter of otrb p\n",
      "development of the otc-inducible expression system for streptomycetes\n",
      "evaluation of the performance of potr* in different streptomyces species\n",
      "application of potr* in activation and optimization of the silent jadomycin bgc\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "construction of plasmids\n",
      "expression and purification of otrr\n",
      "emsa\n",
      "luciferase assays in vivo using lux reporter systems\n",
      "dnase i footprinting assay\n",
      "determination of tss\n",
      "spr assays\n",
      "quantitative measurement of gfp fluorescence\n",
      "transcriptional analysis using rt-qpcr\n",
      "activation of jadomycin bgc and optimization of jadomycin production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00216/sb8b00216.xml\n",
      "results and discussion\n",
      "photoregulation of hyperstable g-quadruplexes\n",
      "polymerase stop assay of gq-installed templates\n",
      "tandem g-quadruplexes enhance inhibition of rna polymerase progression\n",
      "photoregulation of gene expression in zebrafish embryo\n",
      "conclusion\n",
      "methods\n",
      "photoisomerization\n",
      "cd spectroscopy and thermal denaturation experiments\n",
      "polymerase stop assay\n",
      "luciferase reporter assay\n",
      "construction of the gq-modified plasmids\n",
      "photoactivation and inactivation of plasmid in zebrafish embryos\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00266/sb5b00266.xml\n",
      "results and discussion\n",
      "panel of core promoters enables wide range of constitutive mammalian gene expression\n",
      "choice of core promoter significantly affects inducibility\n",
      "relative promoter inducibility is maintained across different host-cell types and associated response elements\n",
      "yb_tata combines high transcription rate with low basal expression to achieve superior inducibility\n",
      "rationally designed inducible promoter enables antigen-responsive t-cell activation specifically in hypoxic environments\n",
      "conclusion\n",
      "methods\n",
      "plasmid generation\n",
      "cell-line maintenance\n",
      "cell transfection\n",
      "lentivirus production and cell transduction\n",
      "gaussia luciferase assay\n",
      "t cell activation and immunostaining\n",
      "flow cytometry\n",
      "induction dynamics and mathematical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400192a/sb400192a.xml\n",
      "results and discussion\n",
      "construction of optochemically controlled nuclear import system\n",
      "optochemically controlled transcription factor activity\n",
      "optochemically controlled protease cleavage\n",
      "methods\n",
      "cloning\n",
      "transfection and tissue culture\n",
      "imaging and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00003/sb8b00003.xml\n",
      "results\n",
      "manipulation of c-di-gmp levels of engineered strain via optogenetic tools\n",
      "optimization of bioprinting conditions\n",
      "bioprinting living p. aeruginosa biofilms using optimized conditions\n",
      "discussion\n",
      "methods\n",
      "plasmids, strains and growth conditions\n",
      "bioprinting living biofilms\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00365/sb7b00365.xml\n",
      "methods\n",
      "materials and reagents\n",
      "cloning pdc and adh of zymomonas mobilis and ldh of bacillus subtilis var. natto\n",
      "expression and purification of pdc and ldh\n",
      "ethanol synthesis from lactate in the bulk system\n",
      "liposome preparation by droplet-transfer method\n",
      "evaluation of encapsulation rates of total enzymes used\n",
      "verification of enzymes digestion by proteinase k\n",
      "ethanol synthesis from lactate by using artcells\n",
      "ethanol synthesis using the products of cell-free protein expression\n",
      "simulations using cell designer\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00056/sb8b00056.xml\n",
      "methods\n",
      "cyanobacteria cultivation\n",
      "constructs and strains\n",
      "butanol and ethanol measurement\n",
      "minimization of metabolic adjustment (moma)\n",
      "design of sgrnas\n",
      "rt-qpcr\n",
      "nadph quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400140t/sb400140t.xml\n",
      "results and discussion\n",
      "rec.e13.δprfa as a chassis strain for cfps\n",
      "rf1 deletion enhances ppaf incorporation in cfps\n",
      "rf1-deficient cfps system increases ppaf incorporation at multiple sites\n",
      "in situ synthesis of optimized o-trna enhances ppaf incorporation\n",
      "quantitative mass spectrometry data discovers high suppression efficiencies for rf1-deficient extracts\n",
      "ppaf incorporation in pure system\n",
      "pacf incorporation\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "growth measurement\n",
      "cell extract preparation\n",
      "preparation of nsaas\n",
      "plasmid construction\n",
      "o-trna synthetase purification\n",
      "cfps reaction\n",
      "quantification of the synthesized sfgfp\n",
      "autoradiography analysis\n",
      "construction of linear dna templates for expressing o-trna\n",
      "cfps using pure system\n",
      "full-length sfgfp purification\n",
      "mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00366/sb6b00366.xml\n",
      "introduction\n",
      "results\n",
      "an engineered selection for extracellular protein expression\n",
      "selection of gain-of-function mutations that increase host secretion capacity\n",
      "mutant strains promote hypersecretion of yebf passenger protein\n",
      "mutant strains promote hypersecretion of yebf bearing n- linked glycans\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "selective plating of bacteria\n",
      "genetic selection of keio knockout library\n",
      "protein analysis\n",
      "characterization of outer membrane permeability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00295/sb7b00295.xml\n",
      "results\n",
      "design of chemical epigenetic modifiers\n",
      "recruitment of hdac activity can efficiently repress the cia:oct4 locus\n",
      "cem-mediated repression of egfp occurs in a variegated and whole-colony fashion\n",
      "h3k27ac levels decrease upon cem-recruitment\n",
      "discussion\n",
      "methods\n",
      "cell lines and infection\n",
      "construct design\n",
      "flow cytometry and facs\n",
      "image acquisition and quantification\n",
      "chip-qpcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00486/sb8b00486.xml\n",
      "results and discussion\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00076/sb5b00076.xml\n",
      "results and discussion\n",
      "generation of psii deletion strains\n",
      "construction of six-gene psii rescue vectors\n",
      "transformation of deletion strains with complementing gene sets and functional reconstitution of psii\n",
      "accumulation of core proteins of reconstituted psii\n",
      "methods\n",
      "strains, culture conditions, biolistic transformation, and pcr screens\n",
      "generation of psii deletion vectors and strains\n",
      "construction of psii expression vectors\n",
      "psii quantum yield ( f v / f m ) measurements\n",
      "protein preparation and elisa\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00180/sb6b00180.xml\n",
      "results and discussions\n",
      "generation of a human/plant hybrid cell line\n",
      "generation of a human-plant chromosome with centromeres from both organisms\n",
      "conversion of translocated plant chromosomes to two types of independent chromosomes through the successive cell divisions\n",
      "further characterization of pd chromosomes\n",
      "expression of arabidopsis genes in hybrid cells; the conservation of gene expression system between humans and plants\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "agrobacterium -mediated transformation of arabidopsis by the floral dip method\n",
      "cell culture\n",
      "cell fusion\n",
      "pcr analysis\n",
      "partially overlapping primer-based pcr (pop–pcr)\n",
      "fluorescent in situ hybridization (fish) and genomic in situ hybridization (gish)\n",
      "multicolour fish analysis (m-fish)\n",
      "combined immunostaining and fish\n",
      "classification of chromosomes types\n",
      "microarray analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00378/sb6b00378.xml\n",
      "results and discussion\n",
      "a novel strategy for genome engineering\n",
      "genome shuffling via p1 transduction\n",
      "genome-wide inactivation of is elements\n",
      "additional genomic modifications\n",
      "phenotypic characterization of the engineered strains\n",
      "carbon source utilization of bl21­(de3) and blk09\n",
      "mutation rates\n",
      "stability of plasmid-based protein expression\n",
      "efficiency of plasmid transformation\n",
      "stress-related phenotypes\n",
      "methods\n",
      "strains, plasmids, and media\n",
      "deletion design\n",
      "oligonucleotides\n",
      "p1 transduction\n",
      "crispr/cas-assisted mage\n",
      "translational knockout of error-prone dna polymerase genes\n",
      "sequencing\n",
      "measurement and calculation of growth parameters\n",
      "carbon source utilization\n",
      "measurement of mutation rates\n",
      "monitoring stability of plasmid-based protein expression\n",
      "determination of antibiotic and uv sensitivity\n",
      "measurement of electrocompetence\n",
      "measurement of tolerance to chemical stressors\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00096/sb6b00096.xml\n",
      "cell–cell communication in yeast\n",
      "engineering transcriptional regulator effector specificity\n",
      "automatic control of gene expression in mammalian cells\n",
      "a tunable protein piston that breaks membranes to release encapsulated cargo\n",
      "the all e. coli tx-tl toolbox 2.0: a platform for cell-free synthetic biology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400168u/sb400168u.xml\n",
      "results and discussion\n",
      "new forcefield charge parameters for 147 non-canonical amino acids\n",
      "predictive ability of approximate binding affinity in discriminating active and inactive analogs of compstatin\n",
      "predictive ability of binding free energy calculations in discriminating active and inactive analogs of compstatin\n",
      "discriminating differences in interactions in compstatin analogs with natural and modified amino acids\n",
      "ff_ncaa web interface\n",
      "methods\n",
      "procedure for calculation and derivation of forcefield parameters in forcefield_ncaa\n",
      "testing the parameters using approximate binding affinity and binding free energy calculations of 63 compstatin analogs\n",
      "selection of system for testing and extraction of data from the literature\n",
      "screening compstatin analogs with short molecular dynamics simulations and binding calculations\n",
      "approximate binding affinity calculations\n",
      "binding free energy calculations\n",
      "construction of receiver operating characteristic curves\n",
      "additional simulations for analysis of interaction free energies of compstatin derivatives and c3 residue pairs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00172/sb5b00172.xml\n",
      "results and discussion\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "preparation of zpsbkx nucleotides\n",
      "construction, expression, and purification of ndpk\n",
      "thin-layer chromatography assay\n",
      "luciferase assay\n",
      "modified 5′-nuclease assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500325e/sb500325e.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00136/sb7b00136.xml\n",
      "results and discussion\n",
      "constitution of a novel biosynthetic pathway for glutarate in e. coli\n",
      "identification of a functional decarboxylase for glutarate production\n",
      "de novo production of glutarate in e. coli\n",
      "constructing crispri system for gene repression\n",
      "production enhancement of glutarate via crispri\n",
      "conclusion\n",
      "materials and methods\n",
      "bacteria strains, plasmids and chemical reagents\n",
      "culture media and conditions\n",
      "plasmid construction\n",
      "protein purification and in vitro enzyme assay\n",
      "feeding experiments\n",
      "de novo production of glutarate\n",
      "hplc and ms analysis\n",
      "fluorescence assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00279/sb5b00279.xml\n",
      "introduction\n",
      "results and discussion\n",
      "the cyma, cymamtra, and cymamtr strains express the full-length and redox active cytochromes c\n",
      "fe­(iii) citrate reduction and cell survival are improved by coexpression of cyma\n",
      "fe 2 o 3 ( s ) reduction and cell survival are improved by coexpression of cyma\n",
      "the expression of cyma significantly increases extracellular electron transfer by enabling catalytic turnover of mtr\n",
      "the number of electron nanoconduits does not significantly affect solid fe 2 o 3 reduction and cell survival\n",
      "flavins couple solid fe 2 o 3 reduction to transient growth in the cymamtr strain\n",
      "opportunities for additional engineering of metal-reducing e. coli.\n",
      "conclusions\n",
      "methods\n",
      "plasmids and strains\n",
      "growth conditions and medium composition\n",
      "subcellular fractionation\n",
      "visible spectra of cytochrome samples by diffuse reflectance\n",
      "ecl to detect c -type cytochromes in whole cell lysates\n",
      "assaying soluble fe­(iii) citrate reduction and cell density of strains\n",
      "bulk fe 2 o 3 ( s ) reduction assay\n",
      "cyclic voltammetry of mtr and cymamtr strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00466/sb7b00466.xml\n",
      "results and discussion\n",
      "pathway design\n",
      "optimization of acyl-coa metabolite extraction\n",
      "engineered acyl-coa precursor supply\n",
      "isovaleryl-coa supply\n",
      "butyryl-coa and hexanoyl-coa production\n",
      "propionyl-coa production\n",
      "methylmalonyl-coa production\n",
      "conclusion\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "media and cultivation\n",
      "metabolite extraction\n",
      "lc–ms/ms analysis of acyl-coa esters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00008/sb9b00008.xml\n",
      "results and discussion\n",
      "identification of the residues forming the putative carboxylate binding pocket of atacs\n",
      "mutations that identify an important residue for catalysis\n",
      "mutations that switch substrate specificity to longer, linear chain carboxylate substrates\n",
      "mutations that switch substrate specificity to branched chain carboxylate substrates\n",
      "kinetics of atp and coa dependence\n",
      "conclusion\n",
      "methods\n",
      "generation of atacs variants\n",
      "overexpression and purification of wild-type and variant atacs proteins\n",
      "spectrophotometric atacs activity assay\n",
      "computational experiments\n",
      "circular dichroism\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00238/sb6b00238.xml\n",
      "lea de maddalena\n",
      "ariel hecht\n",
      "keith heyde\n",
      "zhen kang\n",
      "sung-ho paek\n",
      "benjamin philmus\n",
      "david n. quan\n",
      "felicia scott\n",
      "spencer r. scott\n",
      "miki tashiro\n",
      "matti turtola\n",
      "pawel l. urban\n",
      "patrick videau\n",
      "amin zargar\n",
      "ruihua zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00174/sb5b00174.xml\n",
      "results and discussion\n",
      "characterization of intrinsic terminators with impacts on the downstream and upstream genes\n",
      "construction of genetic filters as a design example of using terminators as regulatory parts\n",
      "termination leakage enables versatile regulation\n",
      "potential adjustment of genetic filters through the selection of proper terminators\n",
      "terminators might also affect the stability of upstream mrna and might result in significant influence\n",
      "demand for a standardized terminator measurement\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "growth medium and fluorescence assay\n",
      "sampling and transcription inhibition\n",
      "rna extraction and cdna synthesis\n",
      "mrna half-life determination by qrt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300039r/sb300039r.xml\n",
      "an in vitro autogene\n",
      "genetic control of metabolic flow\n",
      "small rnas that activate bacterial transcription\n",
      "tetracycline-regulated gene expression systems\n",
      "synthetic control of gene circuit modular design\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00078/sb8b00078.xml\n",
      "dna circuit design\n",
      "model generation\n",
      "analysis\n",
      "synthesis\n",
      "discussion\n",
      "availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00209/sb5b00209.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00266/sb7b00266.xml\n",
      "results\n",
      "engineering synthetic receptors by coupling split-dbds with a single domain antibody\n",
      "a prokaryotic transmembrane receptor using a single-domain antibody for ligand detection\n",
      "optimizing transmembrane receptor signal-to-noise ratio through linkers engineering\n",
      "connecting synthetic receptors to synthetic gene circuits\n",
      "discussion\n",
      "materials and methods\n",
      "plasmids and strains\n",
      "polymerase chain reaction\n",
      "gibson assembly\n",
      "electrotransformation\n",
      "functional characterization of synthetic receptors\n",
      "flow cytometry analysis\n",
      "calculation of relative promoter units (rpus)\n",
      "calculation of repression or activation fold\n",
      "western blotting\n",
      "preparation of cell wall deficient e. coli\n",
      "microscopy\n",
      "antibody labeling of cadc-vhh-caffeine expressed on cell wall deficient e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00300/sb8b00300.xml\n",
      "results and discussion\n",
      "bottom-up engineering of the σ 28 -oscillator\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "characterization and modification of oscillator properties\n",
      "effect of passive transcriptional control on a coupled oscillator\n",
      "conclusion\n",
      "methods\n",
      "preparation of dna templates\n",
      "preparation of cell lysate\n",
      "tx-tl reaction set up\n",
      "fabrication and design of microfluidic device\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00503/sb8b00503.xml\n",
      "results\n",
      "pulse count decoder\n",
      "limits of operation\n",
      "duty fraction decoder\n",
      "limits of operation\n",
      "time period decoder\n",
      "limits of operation\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00187/sb7b00187.xml\n",
      "the impact of dna topology and guide length on target selection by a cytosine-specific cas9\n",
      "a modular toolkit for generating pichia pastoris secretion libraries\n",
      "development of a biosensor concept to detect the production of cluster-specific secondary metabolites\n",
      "engineering iron responses in mammalian cells by signal-induced protein proximity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00334/sb7b00334.xml\n",
      "results\n",
      "screening medium-chain acyl-acp tes and e. coli hosts for medium-chain fatty acids production\n",
      "engineering abte for improved medium-chain fatty acids titers\n",
      "extending cultivation time to increase medium-chain fatty acids titers\n",
      "abte:g17r and abte:g17r/a165r mode of action\n",
      "detection of e. coli produced medium-chain fatty acids by yeast gpcr-based sensor\n",
      "discussion\n",
      "methods\n",
      "plasmid construction\n",
      "abte and acp expression and purification\n",
      "15 n-octanoyl-acpp synthesis\n",
      "nmr titration experiments\n",
      "mcfa production, derivatization and quantification\n",
      "mcfa production\n",
      "fatty acid analysis\n",
      "fatty acid derivatization\n",
      "fame quantification\n",
      "e. coli secreted mcfa detection by s. cerevisiae mcfa sensor\n",
      "e. coli mcfa production\n",
      "sensor preparation\n",
      "mcfa sensing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300049p/sb300049p.xml\n",
      "methods\n",
      "cell-free system and reaction\n",
      "bacteriophage titration and gel electrophoresis\n",
      "gel electrophoresis, radioactive incorporation, and transmission electron microscopy (tem)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002278/sb5002278.xml\n",
      "chun loong ho\n",
      "in young hwang\n",
      "elvin koh\n",
      "anand pai\n",
      "jasmine shong\n",
      "rafael silva-rocha\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500300d/sb500300d.xml\n",
      "results and discussion\n",
      "automated design of programmable pen toolbox networks\n",
      "robustness analysis and automated dna sequence design\n",
      "unproductive sequestration of ttrecj exonuclease\n",
      "conclusion\n",
      "methods\n",
      "integration of odes\n",
      "dna sequence design\n",
      "experimental characterization of polymerase and exonuclease\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00132/sb7b00132.xml\n",
      "gene and pathway discovery from new hosts or environments\n",
      "mechanisms of natural product biosynthesis\n",
      "combinatorial biosynthesis and pathway engineering\n",
      "expanding heterologous production systems to diverse hosts\n",
      "redesigning the genetics of natural product biosynthesis\n",
      "metabolic engineering for improved precursor supply\n",
      "new tools\n",
      "putting it all together\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00217/sb6b00217.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00044/sb5b00044.xml\n",
      "results\n",
      "a method for probabilistic analysis of localized circuits\n",
      "design and probabilistic analysis of elementary localized circuits\n",
      "probabilistic analysis of complex localized circuits\n",
      "probabilistic analysis of localized circuits with diffusible inputs\n",
      "experimental implementation strategies\n",
      "discussion\n",
      "methods\n",
      "biophysical approximation of the effect of localization on the kinetics of dna hairpin interactions\n",
      "probabilistic analysis of localized dna circuits\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00309/sb6b00309.xml\n",
      "methods\n",
      "mathematical modeling\n",
      "c. elegans maintenance\n",
      "ribonucleoprotein complex preparation and injection into c. elegans\n",
      "sequencing and analysis\n",
      "calculation of information content\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00133/sb5b00133.xml\n",
      "results and discussion\n",
      "design of t 2 rmr cassettes\n",
      "confirming t 2 rmr cassette function on a single gene scale\n",
      "rapid construction of genome-scale libraries\n",
      "validation of genome-scale fitness mapping\n",
      "further characterization of t 2 rmr library growth selections\n",
      "conclusions\n",
      "methods\n",
      "strains and dna\n",
      "β-galactosidase assays\n",
      "library construction and recombineering\n",
      "growth selections\n",
      "sequencing and fitness calculations\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300009t/sb300009t.xml\n",
      "results and discussion\n",
      "overview\n",
      "identification of dynamic and static biochemical models\n",
      "introduction\n",
      "enzymatic reaction scheme model of bba _ t 9002\n",
      "enzymatic reaction scheme model is not consistent with the hill equation\n",
      "identification of a data-driven dynamic model - the narmax framework\n",
      "introduction\n",
      "narmax model of bba _ t 9002\n",
      "consistent narmax model identification over a set of colonies\n",
      "summary\n",
      "methods\n",
      "experimental data\n",
      "dynamic modeling based on an enzymatic reaction scheme\n",
      "dynamic representation of the reaction scheme\n",
      "parameter estimation\n",
      "narmax modeling framework\n",
      "narmax model representation\n",
      "narmax model with static input nonlinearity\n",
      "parameter estimation and structure detection\n",
      "signal derivative estimation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500231j/sb500231j.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00229/sb9b00229.xml\n",
      "results and discussion\n",
      "establishment of a gene promoter tool set for use in s2973\n",
      "rational restructuring of the fam bgc and production of cis indole–isonitrile in s2973\n",
      "production of cis indole–isonitrile–gpp and hapalindole h in s2973\n",
      "production of 12- epi -hapalindole u by exchanging the fam cyclase genes\n",
      "s2973 hapalindole yields compared to native producers and other engineered microbes\n",
      "the yield-limiting step is different between the hapalindole-producing s2973 strains\n",
      "rational refactoring of the fam pathway to direct biosynthesis and a means to study hapalindole tailoring\n",
      "conclusions\n",
      "methods\n",
      "cyanobacterial culturing and conjugation\n",
      "construction of eyfp and fam expression vectors\n",
      "fluorescence measurements\n",
      "analysis and purification of hapalindoles from s2973 biomass\n",
      "expression and purification of famd2\n",
      "synthesis of geranyl diphosphate triammonium salt\n",
      "synthesis of ( z )-2-(3-indolyl)­vinyl isocyanide ( cis indole–isonitrile)\n",
      "in vitro aromatic prenyltransferase famd2 assay\n",
      "dowex 50wx8 resin preparation and gpp purification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4002008/sb4002008.xml\n",
      "results and discussion\n",
      "simple strategies for exploitative competition in a bacterial community\n",
      "analytical investigation of competition: constant growth versus susceptibleness\n",
      "searching for optimal competition strategies\n",
      "risk association of fitness-maximized strategies\n",
      "conclusion and outlook\n",
      "methods\n",
      "numerical methods and parameters\n",
      "analytical results\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00269/sb7b00269.xml\n",
      "results and discussion\n",
      "characterizing design properties of a modular cell and production modules\n",
      "modulating degrees of coupling of modular cell and production modules to enhance cell growth and product synthesis\n",
      "enabling modular cells for enzyme selection\n",
      "selection of pdcs with the modular cell tcs095 (de3)\n",
      "selection of pdcs with the modular cell tcs083 (de3)\n",
      "native adhe in the modular cell tcs083 (de3) is sufficient to facilitate pdc selection\n",
      "adaptive laboratory evolution of the coupled modular cells\n",
      "evolution of weakly coupled cells resulted in enhanced growth and ethanol production rates\n",
      "host adaption contributed to enhanced performance of coupled cells\n",
      "unbiased strain classification by principal component analysis revealed similar emergent features\n",
      "conclusion\n",
      "methods\n",
      "strains\n",
      "plasmid/pathway construction\n",
      "construction of pdc modules\n",
      "construction of adhb modules\n",
      "construction of the pdc and adhb ethanol modules\n",
      "medium and cell culturing\n",
      "culture media\n",
      "strain characterization\n",
      "adaptive laboratory evolution\n",
      "analytical methods\n",
      "cell growth\n",
      "high performance liquid chromatography (hplc)\n",
      "data analysis\n",
      "specific growth rate\n",
      "yield\n",
      "specific production/consumption rate\n",
      "growth generation\n",
      "principal component analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00323/sb7b00323.xml\n",
      "results and discussion\n",
      "gfp nanobody-based fusion protein generates a ca 2+ signal in response to gfp stimuli\n",
      "rfp nanobody-based fusion protein generates a ca 2+ signal in response to mcherry stimuli\n",
      "adalimumab-based fusion protein generates ca 2+ signal in response to tnfα\n",
      "anti-cd14 scfv-based fusion protein generates ca 2+ signal in response to soluble cd14 (scd14)\n",
      "scfvchi and carq rewires cell migration to sources of scd14\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture, transfection and stable cell line generation\n",
      "illumination and imaging\n",
      "cell stimulation\n",
      "data quantification\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00236/sb7b00236.xml\n",
      "the genetic constructor application\n",
      "extensions\n",
      "integration with foundries\n",
      "architecture/technical\n",
      "future\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00372/sb7b00372.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmids\n",
      "recombinant proteins\n",
      "cfps systems\n",
      "formation of guvs\n",
      "analysis of aggregates in guvs\n",
      "cell culture and transfection\n",
      "luciferase activity\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00014/sb6b00014.xml\n",
      "results and discussion\n",
      "the reaction network\n",
      "the control problem and the controller\n",
      "assumptions\n",
      "local asymptotic stability\n",
      "tracking and perfect adaptation properties\n",
      "metabolic load and cost of adaptation\n",
      "tuning the controller parameters\n",
      "practical implementation\n",
      "stochastic considerations\n",
      "control of a gene expression network with maturation\n",
      "control of a dimerization network\n",
      "dna implementation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00199/sb7b00199.xml\n",
      "results and discussion\n",
      "design and construction of boronated anticalins\n",
      "rational design and engineering of the borocalin sugar-binding site\n",
      "introduction of tryptophan as spectroscopic probe in proximity to the sugar-binding site\n",
      "effect of buffer ph on sugar-binding activity of borocalins\n",
      "fixing potential hindrance at the sugar-binding site of the borocalin\n",
      "binding studies with model cis -diol compounds as well as control lipocalins lacking the boronate group\n",
      "structural analyses of the borocalin\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500371v/sb500371v.xml\n",
      "qualitative and quantitative analysis using p systems\n",
      "simultaneous expression of multiple enzymes in yeast\n",
      "combinatorial assembly of large biochemical pathways into yeast chromosomes\n",
      "direct mutagenesis of thousands of genomic targets\n",
      "model-driven engineering of gene expression from rna replicons\n",
      "proposed data model for the next version of the synthetic biology open language\n",
      "a low cost, customizable turbidostat for synthetic circuit characterization\n",
      "meta-stochastic simulation of biochemical models\n",
      "distributed classifier based on genetically engineered bacterial cell cultures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00009/sb6b00009.xml\n",
      "results\n",
      "buffered dna strand displacement gates\n",
      "a buffered architecture for adaptive strand displacement networks\n",
      "an adaptive, buffered amplifier\n",
      "a strand displacement learning circuit\n",
      "predictor subcircuit\n",
      "feedback subcircuit\n",
      "results from simulations of the learning circuit\n",
      "investigating the effect of noise on learning circuit operation\n",
      "discussion\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "construction of chemical reaction models for learning circuit simulations\n",
      "execution of learning circuit simulations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00292/sb7b00292.xml\n",
      "results and discussion\n",
      "dynamic range of promoter regulator pairs\n",
      "two- and three-color pigment sensors\n",
      "tuning p znuc response with decoy operators\n",
      "growth and testing in serum\n",
      "conclusions\n",
      "methods\n",
      "plasmids and oligonucleotides\n",
      "molecular biology\n",
      "zur decoy array construction\n",
      "violacein extraction\n",
      "acetone extraction\n",
      "carotenoid analysis\n",
      "chelex 100 treatment\n",
      "modified m9 media\n",
      "cell culture\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400076r/sb400076r.xml\n",
      "results and discussion\n",
      "estimating capsid structural disruption\n",
      "analysis of chimera capsid assembly and genome packaging\n",
      "using genome protection as measure of capsid intactness\n",
      "ability of chimeras to bind heparin\n",
      "transduction of cells with chimeric capsids\n",
      "schema-based design rules\n",
      "materials and methods\n",
      "schema calculations\n",
      "cells\n",
      "aav cap gene vectors\n",
      "capsid gene chimera construction\n",
      "virus production\n",
      "quantifying genomic titers\n",
      "western blotting\n",
      "transmission electron microscopy\n",
      "nuclease protection assay\n",
      "heparin binding assay\n",
      "transduction assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500234s/sb500234s.xml\n",
      "results and discussion\n",
      "conditional origins of replication (cors)\n",
      "rich-media compatible auxotrophs\n",
      "toxin–antitoxin systems as a negative selection pressure\n",
      "geneguard genomic and vector cassettes\n",
      "using the geneguard system for heavy-metal biosensors\n",
      "methods\n",
      "media and general materials\n",
      "device construction\n",
      "flow cytometry\n",
      "genomic integration\n",
      "plasmid stability assay\n",
      "soil assay\n",
      "growth curves\n",
      "western blotting\n",
      "transformation assay\n",
      "biosensor assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00167/sb6b00167.xml\n",
      "results and discussion\n",
      "functional expression of rhodospirillum rubrum rubisco in s. cerevisiae\n",
      "effects of prk on sugar metabolism in s. cerevisiae\n",
      "completing the synthetic rppp in engineered s. cerevisiae\n",
      "ratio between accumulated co 2 and ethanol reveals recycling of co 2\n",
      "conclusions\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "crispr/cas-based genome modification\n",
      "enzyme activity assay\n",
      "anaerobic fermentation experiments\n",
      "analysis of the ratio between accumulated co 2 and ethanol ( r c/e )\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00406/sb8b00406.xml\n",
      "results and discussion\n",
      "identification of orthogonal probes with minimal background\n",
      "mirage quantitatively reports on cell presence and gene expression\n",
      "mirage can monitor reprogramming to pluripotency\n",
      "mirage reporters are multiplexable\n",
      "mirage reports on cell presence in vivo\n",
      "materials and methods\n",
      "cell culture, transfections, and selection\n",
      "stem cell differentiation and reprogramming\n",
      "bmp4-mediated differentiation\n",
      "differentiation of hescs into fibroblast-like cells\n",
      "reprogramming of hesc-derived fibroblasts into hipscs using sendai virus\n",
      "luciferase detection in vitro\n",
      "multiplexing test\n",
      "rna collection\n",
      "extracellular vesicle (ev) purification\n",
      "qrt-pcr\n",
      "mice\n",
      "imaging\n",
      "plasmid building/cloning\n",
      "tide analysis\n",
      "immunostaining\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00274/sb8b00274.xml\n",
      "results and discussion\n",
      "fusion of the tdot coiled-coil domain allows the production of fluorescent inclusion bodies in e. coli\n",
      "two fluorescent proteins can be colocalized in inclusion bodies\n",
      "proof-of-concept realization of a two-enzyme cascade\n",
      "pfbal and radh can be produced as catibs and co-catibs\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and media\n",
      "construction of expression plasmids\n",
      "production and purification of inclusion bodies (ibs) and soluble pf bal and r adh\n",
      "sodium dodecyl sulfate-polyacrylamide gel electrophoresis (sds-page) and protein concentration determination\n",
      "cell fractionation and determination of the fluorescence/activity distribution\n",
      "fluorescence spectrophotometry, fluorescence distribution and quantitative comparison\n",
      "online analysis of yfp and mcherry fluorescence during microtiter plate cultivations\n",
      "quantitative real-time pcr (qpcr)\n",
      "microscopy and live cell imaging\n",
      "microfluidic cultivation\n",
      "photometric pf bal and r adh activity assays for the analysis of cellular fractions\n",
      "pf bal and r adh activity assays for the determination of specific activities\n",
      "determination of the protein concentration\n",
      "two-enzyme cascade reaction\n",
      "high performance liquid chromatography analysis (hplc)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00314/sb8b00314.xml\n",
      "results and discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00395/sb6b00395.xml\n",
      "results and discussion\n",
      "3d printer and bioink\n",
      "printing reproducibility and resolution\n",
      "survival and metabolic activity of printed bacteria\n",
      "materials and methods\n",
      "printing system\n",
      "bacterial strains, plasmids, and culturing\n",
      "printer bioink\n",
      "printing substrate\n",
      "resolution and height measurements\n",
      "printing and imaging of layered alginate gels containing fluorescent bacteria\n",
      "bacterial survival\n",
      "production of rfp by printed bacteria\n",
      "statistical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00176/sb8b00176.xml\n",
      "results and discussion\n",
      "mining and sequence analysis of oxidosqualene cyclase (osc)\n",
      "expression, purification and in vitro characterization of oscs\n",
      "verification of oscs’ function and production of α-amyrin in engineered s. cerevisiae\n",
      "enabling alpha amyrin production by high-copy plasmids\n",
      "materials and methods\n",
      "phylogenetic analysis\n",
      "construction of expression plasmids\n",
      "construction of high-production strain\n",
      "qrt-pcr analysis\n",
      "expression and purification of ejas, mdosc1 and bmosc in e. coli rosetta (de3)\n",
      "expression and purification ejas, mdosc1 and bmosc in s. cerevisiae and pichia pastoris\n",
      "sds/page and western blotting for estimation of protein expression\n",
      "functional characterization by gc–ms\n",
      "analysis of enzyme specific activities in vitro\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001323/sb4001323.xml\n",
      "how does r2odna designer work?\n",
      "parameters\n",
      "initial sequence generation using monte carlo simulated annealing\n",
      "blast and network elimination\n",
      "job progress tracking\n",
      "reverse mode\n",
      "future development\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00397/sb6b00397.xml\n",
      "results\n",
      "two distinct noise regimes define pm output\n",
      "noise deconvolution and rate optimization\n",
      "system sensitivity to alterations in the number of transcription factor\n",
      "influence of intracellular regulator-promoter proximity on transcriptional output\n",
      "physical proximity between pm and xyls decreases transcriptional noise\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, growth conditions and genetic constructs\n",
      "single cell analysis by flow cytometry\n",
      "rna purification and real-time q-pcr\n",
      "kinetic reactions, simulation and optimization\n",
      "spatial protein trajectories\n",
      "visualization of the tol plasmid ( figure c)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00148/sb6b00148.xml\n",
      "methods\n",
      "strains and reagents\n",
      "plasmid construction\n",
      "shake flask cultivation of e. coli strains\n",
      "sds-page analysis of the protein profile\n",
      "ubiquinone extraction and measurement\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000355/sb4000355.xml\n",
      "results and discussion\n",
      "fluvirucin b 1 pks genes\n",
      "at selectivities\n",
      "module 3 substrate selectivity\n",
      "kr domains\n",
      "fluvirucin b 1 synthase cloning\n",
      "discussion\n",
      "conclusions\n",
      "experimental procedures\n",
      "materials\n",
      "bacterial strains, culture conditions, and dna purification\n",
      "cloning of module 1 of the fluvirucin b 1 polyketide synthase\n",
      "cloning of module 2 of the fluvirucin b 1 polyketide synthase\n",
      "cloning of module 3 of the fluvirucin b 1 polyketide synthase\n",
      "cloning of module 4 of fluvirucin b 1 polyketide synthase\n",
      "cloning of module 5 + te of the fluvirucin b 1 polyketide synthase\n",
      "cloning of flu ksat1, ksat3, ksat5 and flu acp1, acp3, acp5 domains of the fluvirucin b 1 polyketide synthase\n",
      "general procedure for protein expression and isolation\n",
      "synthesis of ( r )- and ( s )-3-hydroxybutyryl-snac\n",
      "( s )-3-hydroxybutyryl-snac\n",
      "( r )-3-hydroxybutyryl-snac\n",
      "synthesis of malonyl and substituted malonyl snac thoesters\n",
      "malonyl snac thioester\n",
      "methyl malonyl snac thioester\n",
      "ethyl malonyl snac thioester\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001068/sb4001068.xml\n",
      "synthetic biology: connecting science and technology\n",
      "vocabularies of science, engineering, and design\n",
      "the “central dogma” of science and society\n",
      "heterogeneous engineering\n",
      "speculative design\n",
      "collaboration: understanding problems and designing solutions\n",
      "conclusion: what i design, i understand?\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00237/sb8b00237.xml\n",
      "results\n",
      "schematic diagram of systematic strategies\n",
      "construction of a gene coexpression network\n",
      "identification of differential genetic interaction networks using crispr-based dual knockout\n",
      "distinct interaction patterns indicate specific inflammation-induced tumorigenesis mechanisms\n",
      "synergistic module-based drug prediction for inflammation-induced tumorigenesis\n",
      "discussion\n",
      "methods\n",
      "experimental materials\n",
      "cell lines and transfection\n",
      "cell viability assay\n",
      "quantitative rt-pcr\n",
      "western blot analysis\n",
      "fluorescence-activated cell sorting analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00353/sb6b00353.xml\n",
      "results and discussion\n",
      "identification of poorly expressed secondary metabolite biosynthesis gene clusters in streptomyces lividans tk24\n",
      "attempts on construction of the undecylprodegiosin biosensor\n",
      "construction of the coelimycin-specific biosensor\n",
      "testing of the coelimycin biosensor using artificial activation of the cpk gene cluster\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "rna isolation, library preparation and cdna sequencing\n",
      "read mapping, data visualization and analysis of gene expression\n",
      "dna manipulation\n",
      "construction of s. lividans redstrep1 strain\n",
      "construction of genes encoding hybrid proteins\n",
      "analysis of secondary metabolites produced by the recombinant strains\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00098/sb5b00098.xml\n",
      "results and discussion\n",
      "engineering triple operators using in silico optimization\n",
      "validation of plac-cfi1 activity in mutant strains of e. coli\n",
      "implementation of a modified optimization algorithm to allow cis -regulatory engineering\n",
      "accessing the cis -regulatory logic of p cfi20‑1 and p cfi22‑5 promoters in vivo\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "promoter activity assay\n",
      "flow cytometry\n",
      "data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00021/sb8b00021.xml\n",
      "results and discussion\n",
      "data structure for high-level protocols\n",
      "information integration via automated planning\n",
      "proof-of-principle applications\n",
      "conclusions\n",
      "methods\n",
      "software architecture\n",
      "experimental methods\n",
      "data analysis\n",
      "ph stability\n",
      "folding rate\n",
      "δ g of unfolding\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00377/sb6b00377.xml\n",
      "results and discussion\n",
      "elasticity estimation by lin-log approximation\n",
      "high concentration of atp inhibits fpps and ads activities\n",
      "pyrophosphate inhibits ads but not fpps\n",
      "enhance the rate of ad production\n",
      "enhance specific yield of ad by coimmobilized multienzymatic synthesis\n",
      "conclusion\n",
      "materials and methods\n",
      "bacteria strains and plasmids\n",
      "individual enzyme purification and reaction\n",
      "multienzyme reaction in solution phase\n",
      "coimmobilized multienzyme reaction\n",
      "analysis of metabolites\n",
      "lin-log approximation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00228/sb6b00228.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "expression and purification of the kinase variant\n",
      "tlc assay\n",
      "enzyme-coupled assay\n",
      "methods\n",
      "construction, expression, and purification of the kinase variant\n",
      "preparation of unnatural nucleosides\n",
      "tlc assays\n",
      "enzyme-coupled assay\n",
      "kinetic parameters\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00075/sb8b00075.xml\n",
      "results and discussion\n",
      "recruiting ols and oac for olivetolic acid production\n",
      "impact of precursor supply on olivetolic acid production in vivo\n",
      "integration of the synthetic olivetolic acid pathway with a β-oxidation reversal for precursor supply\n",
      "optimization of fermentation conditions for olivetolic acid production\n",
      "olivetolic acid fermentation in bioreactor under controlled conditions\n",
      "conclusions\n",
      "methods\n",
      "strains and culture conditions\n",
      "construction of plasmids\n",
      "in vitro production of olivetolic acid\n",
      "resting cell biotransformations for olivetolic acid production\n",
      "fermentation conditions for olivetolic acid production in shake flasks\n",
      "olivetolic acid fermentation in bioreactor with precise parameter control\n",
      "olivetolic acid/olivetol stability analysis\n",
      "gc-fid/ms analysis\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300076f/sb300076f.xml\n",
      "compartment size and multimeric protein synthesis\n",
      "test tube synthesis of the bacteriophage t7\n",
      "evolutionary stability of a refactored phage genome\n",
      "unsaturated lipidoids for efficient intracellular gene delivery\n",
      "engineering functional dna translocation\n",
      "synthetic biology to explain cellular information processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00129/sb8b00129.xml\n",
      "results\n",
      "isolation of active two-fragment tetr\n",
      "characterizing stetr repression\n",
      "stetr functions as a genetic and gate\n",
      "using stetr as a protein solubility sensor\n",
      "discussion\n",
      "materials and methods\n",
      "strain constructions\n",
      "library construction\n",
      "screening\n",
      "bacterial vector construction\n",
      "bacterial fluorescence measurements\n",
      "mammalian vector construction\n",
      "cell cultures and flow cytometry studies\n",
      "protein aggregation analyses\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00379/sb8b00379.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "crispr interference\n",
      "media and cultivation\n",
      "metabolite measurements\n",
      "flow cytometry\n",
      "next generation sequencing and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00275/sb7b00275.xml\n",
      "tamas feher\n",
      "paula gregorio-godoy\n",
      "martin gutiérrez\n",
      "patrick ho\n",
      "jyun-liang lin\n",
      "eleftherios terry papoutsakis\n",
      "julia rohlhill\n",
      "nicholas sandoval\n",
      "tatenda shopera\n",
      "kinga umenhoffer\n",
      "weiqian zeng\n",
      "jie zhu\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200015u/sb200015u.xml\n",
      "results and discussion\n",
      "synzip oligomerization state\n",
      "synzip interaction orientation\n",
      "synzip interaction affinities\n",
      "synzip interactions in a yeast two-hybrid assay\n",
      "repressing yeast α-factor response using synzips as recruitment domains for a negative pathway modulator\n",
      "synzip specification sheets\n",
      "discussion\n",
      "linear interactions: synzips 2:19:21:4\n",
      "ring interactions: synzips 18:19:21:20\n",
      "hub interactions: synzips 17:18:19:20\n",
      "orthogonal interactions: synzips 1:2, 3:4\n",
      "methods\n",
      "cloning, purification, and dye labeling\n",
      "size-exclusion chromatography (sec) assay\n",
      "fluorescence resonance energy transfer (fret) assay\n",
      "fluorescence polarization (fp) assay\n",
      "yeast two-hybrid assay\n",
      "mapk pathway testing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500256t/sb500256t.xml\n",
      "cem albayrak\n",
      "dario cecchi\n",
      "fabio chizzolini\n",
      "seok hoon hong\n",
      "sukanya iyer\n",
      "wei-cheng lu\n",
      "taiji okano\n",
      "dan siegal-gaskins\n",
      "haruka soga\n",
      "zoltan tuza\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00287/sb7b00287.xml\n",
      "results and discussion\n",
      "characterization of a prototype two-plasmid macrolide detection system in e. coli\n",
      "enhancing the sensitivity of mphr to erythromycin via multisite saturation mutagenesis and random mutagenesis\n",
      "random mutagenesis of the rbs of mphr\n",
      "narrowing the macrolide inducer specificity of mphr\n",
      "improving the sensitivity of mphr toward macrolides that are poor inducers of wild-type mphr\n",
      "validation of a biosensor microplate screen and growth selection\n",
      "discussion\n",
      "materials and methods\n",
      "strains, media, and chemicals\n",
      "error-prone pcr of mphr\n",
      "multi-site saturation mutagenesis of mphr\n",
      "site-directed mutagenesis of mphr\n",
      "fluorescence activated cell sorting (facs) of biosensor libraries\n",
      "general procedure for microplate screening of libraries of mphr variants\n",
      "dose–response analysis of wild-type and variant mphr\n",
      "aeromicrobium erythreum culturing and screening with the whole-cell biosensor strain\n",
      "growth selection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00390/sb6b00390.xml\n",
      "introduction\n",
      "capability of dna circuits\n",
      "digital and analog dna circuits\n",
      "our approach and contribution\n",
      "results and discussion\n",
      "a high-level chemical reaction network (crn) description of our architecture\n",
      "overview\n",
      "the chemical reaction network (crn)\n",
      "the input module computes a time point to trigger the stopper module\n",
      "the stopper module stops the output module at the designated time\n",
      "the output module produces the expected output\n",
      "examples of computing functions\n",
      "examples implemented by dna strand displacement circuits\n",
      "a dna-based autocatalytic amplifier\n",
      "building analog circuits by the dna-based autocatalytic amplifier\n",
      "tunable property of the amplifier\n",
      "reducing the leak in stopper module\n",
      "make reactions and fast\n",
      "analog dna circuits to compute sqrt­( x ), ln­( x ), and exp­( x )\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00032/sb9b00032.xml\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "microplate reader assays\n",
      "software implementation\n",
      "results\n",
      "characterization of luminescence bleed-through\n",
      "deconvolution strategy for luminescence bleed-through correction\n",
      "a minimization algorithm to compute the best kernel d-matrix\n",
      "robustness of the deconvolution process\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00043/sb5b00043.xml\n",
      "standardized parts\n",
      "promoters and terminators\n",
      "ribosome binding site\n",
      "rna regulators and post-translational control\n",
      "genome engineering\n",
      "multiple chromosomes and associated challenges\n",
      "genomic, expression, and deletions libraries in cyanobacteria\n",
      "multiplexing\n",
      "crispr-cas systems\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00403/sb7b00403.xml\n",
      "results and discussion\n",
      "search and data visualization\n",
      "submission and design documentation\n",
      "sharing and security\n",
      "computational api\n",
      "synthetic biology workflows\n",
      "discussion\n",
      "availability\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00219/sb9b00219.xml\n",
      "results and discussion\n",
      "biosynthesis of rosavin b in e. coli\n",
      "fed-batch fermentation for biosynthesis of cinnamyl alcohol glucosides\n",
      "biosynthesis of p -coumaryl alcohol diglucosides in e. coli\n",
      "biosynthesis of rosavin analogues in e. coli and production improvement\n",
      "methods\n",
      "plasmids, bacterial strains, and culture media\n",
      "plasmids construction\n",
      "shake flask cultivation of recombinant e. coli strains\n",
      "fed-batch fermentation\n",
      "compounds purification\n",
      "chemical analysis and quantification\n",
      "nmr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00114/sb9b00114.xml\n",
      "methods\n",
      "bacterial strains, plasmids, and growth conditions\n",
      "plasmid construction\n",
      "transformation\n",
      "recombineering\n",
      "mic assay\n",
      "catalase activity and h 2 o 2 stress assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00066/sb9b00066.xml\n",
      "introduction\n",
      "results and discussion\n",
      "cultivation and reporter systems\n",
      "characterization of cpcb promoter\n",
      "characterization of rbc promoter\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00141/sb5b00141.xml\n",
      "results and discussion\n",
      "implementation of the control platform\n",
      "experimental model\n",
      "control strategies: relay and pi controllers\n",
      "in silico set-point control of gene expression\n",
      "in vitro set-point control of gene expression\n",
      "concluding remarks\n",
      "methods\n",
      "cell culture procedures\n",
      "fluorescence time-lapse microscopy and control experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00123/sb9b00123.xml\n",
      "results and discussion\n",
      "crispr/cas9-mediated genome editing for integrating feedback-resistant aro4 and aro7\n",
      "styrene production in the engineered yeast cells\n",
      "engineering the budding yeast toward ( s )-phenylethanediol biosynthesis\n",
      "engineering the budding yeast toward ( r )-phenylethanediol biosynthesis\n",
      "conclusion\n",
      "methods\n",
      "genetic modification in budding yeast\n",
      "plasmid-based expression system for cascade biotransformation\n",
      "styrene biosynthesis by engineered yeast cells\n",
      "phenylethanediol production under shake-flasks\n",
      "chiral hplc for enantiopurity analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500270h/sb500270h.xml\n",
      "results and discussion\n",
      "rational design of tetracycline-dependent hammerhead ribozymes\n",
      "tetracycline ribozymes show dose-dependent regulation\n",
      "dependence of ribozyme regulation on position and copy number\n",
      "stable integration of the ribozymes into the genome of hela cells\n",
      "conclusion\n",
      "methods\n",
      "vector construction\n",
      "cell culture\n",
      "luciferase assay\n",
      "transient transfection, rna isolation, cdna synthesis, quantitative (q)­rt pcr\n",
      "in vitro transcription and cleavage kinetics\n",
      "generation of hela “flp-in reporter cell line”\n",
      "microscopy analysis\n",
      "fluorescence measurements\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500018f/sb500018f.xml\n",
      "results and discussion\n",
      "construction of bacterial ghost detection system\n",
      "thrombin detection using split egfp\n",
      "thrombin detection using β-lactamase testing strips\n",
      "discussion\n",
      "methods\n",
      "lysis gene e and inner-membrane scaffolds gene cloning\n",
      "protein expression and aptamer experiments\n",
      "β-lactamase testing strip\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300075t/sb300075t.xml\n",
      "methods\n",
      "par-par language syntax\n",
      "par-par compiler\n",
      "parser\n",
      "database manager\n",
      "optimizer\n",
      "translator\n",
      "par-par interfaces, workflow, license, and availability\n",
      "par-par software implementation\n",
      "par-par objects, memory allocation, and variable scope\n",
      "database\n",
      "command line interface\n",
      "web interface\n",
      "j5 license and availability\n",
      "j5 software implementation\n",
      "sequence availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00512/sb8b00512.xml\n",
      "results\n",
      "recombinase-based gene switch for controlling car expression\n",
      "induction of recombinase activity drives changes in car expression\n",
      "changes in car expression impact t cell responses to target antigen\n",
      "switch cells exhibit memory when drug is removed\n",
      "tunability of switching can be driven by drug dosage and duration\n",
      "discussion\n",
      "methods\n",
      "circuit construction\n",
      "primary t cell isolation and transduction\n",
      "jurkat t cell maintenance and transduction\n",
      "switch induction with 4-oht\n",
      "flow cytometry sorting and analysis\n",
      "car activation with plate-bound her2 protein\n",
      "il-2 elisa\n",
      "cell killing\n",
      "cell washing for memory and induction duration\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00430/sb8b00430.xml\n",
      "results and discussion\n",
      "cell-free gene expression from bacterial promoters in crude extract is limited\n",
      "postlysis processing enhances cfe yield from bacterial promoters\n",
      "yield improvements to cell-free gene expression are independent of regulatory element strength\n",
      "yield improvements to cell-free gene expression arise from enhanced transcriptional activity\n",
      "protocol sensitivity can be assessed in transcriptionally limiting conditions\n",
      "a wide array of transcriptional genetic circuits are functional in a modified cell-free extract preparation framework\n",
      "summary\n",
      "materials and methods\n",
      "plasmid construction and purification\n",
      "extract preparation\n",
      "cfe experiment\n",
      "rna purification and rna degradation experiments\n",
      "fluorescent standard calibration\n",
      "data availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00519/sb8b00519.xml\n",
      "results and discussion\n",
      "formation of caveolae in e. coli cytoplasm\n",
      "endocytosis of fatty acids\n",
      "constitutive expression of cav1\n",
      "biotransformation of ra is enhanced by endocytosing e. coli\n",
      "conversion of ra at high cell density\n",
      "biotransformation of 10-hydroxyoctadecanoic acid and castor oil by the endocytosing e. coli\n",
      "fatty acid uptake by e. coli expressing cav1 and/or fadl\n",
      "conclusions\n",
      "materials and methods\n",
      "chemicals and materials\n",
      "plasmid construction\n",
      "purification of heterologous caveolae\n",
      "analysis of fluorescence dye transport\n",
      "analysis of fatty acid transport\n",
      "whole-cell biotransformation\n",
      "high density cell cultivation\n",
      "biotransformation in a cell-recycle system\n",
      "analytical method\n",
      "sample preparation for transmission electron microscopy (tem) analyses\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00069/sb6b00069.xml\n",
      "methods\n",
      "growth conditions\n",
      "construction of mutants of rba. sphaeroides\n",
      "pigment analysis\n",
      "purification of wscp-his from rba. sphaeroides and identification by nanoflow lc–ms/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00181/sb6b00181.xml\n",
      "stijn aper\n",
      "adam paul arkin\n",
      "irem avcilar-kucukgoze\n",
      "marcelo c. bassalo\n",
      "roel a. l. bovenberg\n",
      "alexander brown\n",
      "patricia calero\n",
      "sebastian castillo-hair\n",
      "daniel a. charlebois\n",
      "junchen diao\n",
      "arnold j. m. driessen\n",
      "sheila i. jensen\n",
      "nilesh karalkar\n",
      "brian landry\n",
      "nicole a. leal\n",
      "jiazhang lian\n",
      "zhe lyu\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "michael magaraci\n",
      "dmitry nevozhay\n",
      "alex t. nielsen\n",
      "yvonne nygård\n",
      "belén pimentel\n",
      "carsten pohl\n",
      "mark preston\n",
      "john t. sexton\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00280/sb7b00280.xml\n",
      "materials and methods\n",
      "dna constructs and in vitro transcription\n",
      "cell culture\n",
      "preparation of rrl and cell lysate\n",
      "protein quantification\n",
      "rrl fractionation and reconstitution\n",
      "nmr acquisition\n",
      "results and discussion\n",
      "energetic metabolism evolution in rrl\n",
      "energy consumption is mostly independent from protein production yield of the native rrl\n",
      "major energy consuming processes associate with the ribosomal fraction\n",
      "riboproteome dilution is a compromise between translation-independent metabolism and protein synthesis\n",
      "evaluation of the hybrid cfps approach using ribosomal fractions from various cell types\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400134k/sb400134k.xml\n",
      "substructure of stfs\n",
      "rules and grammars\n",
      "rules for stf design\n",
      "stf grammar\n",
      "pids\n",
      "dbds\n",
      "grna\n",
      "eds\n",
      "reps\n",
      "genocad implementation\n",
      "future developments to the grammar\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00153/sb5b00153.xml\n",
      "methods\n",
      "plasmid construction\n",
      "purification of pks constructs\n",
      "purification of bordh2\n",
      "intermediate analysis of pks variants\n",
      "product analysis of thioesterase-harboring constructs\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00203/sb7b00203.xml\n",
      "results and discussion\n",
      "methods\n",
      "strains and growth conditions\n",
      "strain construction\n",
      "induction assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00386/sb8b00386.xml\n",
      "results\n",
      "structure-based designs and screening\n",
      "validating clips1 activity\n",
      "library screening for increased light versus dark activity\n",
      "growth curves and protein quantification in liquid culture\n",
      "polysome analysis\n",
      "effect on translation in vitro\n",
      "clips binding to eif4e in vitro\n",
      "discussion\n",
      "methods\n",
      "cloning\n",
      "library construction\n",
      "yeast growth assay\n",
      "growth curves in liquid culture\n",
      "polysome assays\n",
      "western blot analysis\n",
      "yeast extract preparation\n",
      "synthesis of mrna\n",
      "in vitro translation assay\n",
      "protein purification\n",
      "uv–vis characterization\n",
      "sec analysis of clips/eif4e interactions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00056/sb6b00056.xml\n",
      "methods\n",
      "cell culture and transfection\n",
      "plasmids and oligonucleotides\n",
      "pcr\n",
      "surveyor assay\n",
      "western blot\n",
      "quantification of integration efficiency\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002196/sb5002196.xml\n",
      "theory\n",
      "secondary structure model\n",
      "analyzing equilibrium base-pairing in a test tube\n",
      "test tube design problem specification\n",
      "test tube ensemble defect objective function\n",
      "algorithm\n",
      "overview\n",
      "test tube ensemble focusing\n",
      "hierarchical ensemble decomposition\n",
      "structure-guided decomposition of on-target complexes\n",
      "stop condition stringency\n",
      "efficient estimation of test tube ensemble properties\n",
      "complex partition function estimate\n",
      "complex pair probability matrix estimate\n",
      "complex concentration estimate using deflated mass constraints\n",
      "complex ensemble defect estimate\n",
      "test tube ensemble defect estimate\n",
      "sequence optimization at the leaves of the decomposition forest\n",
      "initialization\n",
      "leaf mutation\n",
      "leaf reoptimization\n",
      "subsequence merging, redecomposition, and reoptimization\n",
      "test tube evaluation, refocusing, and reoptimization\n",
      "hierarchical ensemble decomposition using multiple exclusive split-points\n",
      "structure-guided decomposition using a single split-point\n",
      "probability-guided decomposition using multiple exclusive split-points\n",
      "structure- and probability-guided decomposition using multiple exclusive split-points\n",
      "test tube ensemble defect estimation using multiple exclusive decompositions\n",
      "methods\n",
      "implementation\n",
      "target test tubes\n",
      "sequence design trials\n",
      "results and discussion\n",
      "algorithm performance for test tube design\n",
      "importance of designing against off-targets\n",
      "contributions of algorithmic ingredients\n",
      "robustness of predictions to model perturbations\n",
      "test tube design with competing on-target complexes\n",
      "test tube design with large numbers of on- and off-targets\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400101z/sb400101z.xml\n",
      "nikolaos anesiadis\n",
      "pablo ceres\n",
      "jin huh\n",
      "shridhar jayanthi\n",
      "henrike marie niederholtmeyer\n",
      "kayzad soli nilgiriwala\n",
      "taichi umeyama\n",
      "domitilla del vecchio\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00453/sb8b00453.xml\n",
      "results\n",
      "identification of putative safe harbor integration sites in the cho-s genome\n",
      "modular toolbox for generation of mcherry-encoding platform cell lines with a poly­(a) trap-based rmce landing pad\n",
      "validation of generated mcherry-encoding platform cell lines\n",
      "minimal clonal variation at defined integration sites\n",
      "proof-of-principle recombination into poly­(a) trap elicited tunable egfp expression patterns\n",
      "observation of site-dependent and construct-dependent egfp expression patterns\n",
      "similar clone-specific egfp expression patterns across all three integration sites\n",
      "long-term stability of egfp expression achieved in all three integration sites\n",
      "site-specific gene expression characterization for a secreted biotherapeutic\n",
      "discussion\n",
      "methods\n",
      "identification of integration sites\n",
      "plasmid design and construction\n",
      "cell cultivation\n",
      "platform cell line generation using crispr/cas9\n",
      "fluorescent level analysis\n",
      "confirmation of landing pad integration\n",
      "copy number analysis using qpcr\n",
      "batch cultivation with single-copy pcl clones\n",
      "subpopulation generation by recombinase-mediated cassette exchange (rmce) and bulk sorting\n",
      "copy number analysis on bulk-sorted populations using dpcr\n",
      "relative mrna expression level using rt-qpcr\n",
      "long-term cultivation\n",
      "short-term productivity assay and titer measurements\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003289/sb5003289.xml\n",
      "standards\n",
      "model annotation\n",
      "results and discussion\n",
      "example: 4-input and sensor\n",
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00402/sb8b00402.xml\n",
      "results and discussion\n",
      "direct cloning and heterologous expression of the native spinosad biosynthetic pathway in streptomyces\n",
      "enhanced spinosad production by overexpressing pks genes\n",
      "construction of the artificial spinosad gene cluster and exocet multipiece dna assembly\n",
      "enhanced spinosad production by the artificial gene cluster in s. albus j1074\n",
      "methods\n",
      "bacteria strains, plasmids, and growth conditions\n",
      "direct cloning of the spinosad gene cluster\n",
      "cloning of rhamnose biosynthetic genes and construction of the spinosad expression vector carrying all biosynthetic genes\n",
      "insertion of promoters erme *p, kaso p* and sf14p upstream of spna\n",
      "multipiece dna assembly\n",
      "construction of the artificial spinosad gene cluster\n",
      "fermentation and high performance liquid chromatography–mass spectrometry analysis of metabolites from wild type and engineered streptomyces strains\n",
      "real-time pcr and reverse transcription pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500268n/sb500268n.xml\n",
      "results and discussion\n",
      "methods\n",
      "transcription using standard, z-containing, and p-containing templates\n",
      "purification of rna\n",
      "rna digestion by p1 nuclease for hplc analysis\n",
      "maldi\n",
      "reverse transcription in the absence and presence of datp, dptp, or dztp\n",
      "rna labeling, alkaline hydrolysis, denaturing ribonuclease t1, and rnase a digestion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00115/sb5b00115.xml\n",
      "short synthetic terminators for improved heterologous gene expression in yeast\n",
      "functional exchangeability of oxidase and dehydrogenase reactions in the biosynthesis of hydroxyphenylglycine\n",
      "yeast golden gate for the efficient assembly of yeast transcription units\n",
      "quantifying impact of chromosome copy number on recombination\n",
      "red light-regulated reversible nuclear localization of proteins\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400081r/sb400081r.xml\n",
      "methods\n",
      "strain and plasmid construction\n",
      "saccharomyces cerevisiae\n",
      "hek293t cells\n",
      "fluorescence assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500236e/sb500236e.xml\n",
      "methods\n",
      "system equations\n",
      "nominal parameter values\n",
      "normalized system equations\n",
      "phenotypic deconstruction\n",
      "mathematically controlled comparisons\n",
      "computational procedures\n",
      "system design space\n",
      "routh criteria for local stability\n",
      "numerical methods\n",
      "results and discussion\n",
      "three architectures\n",
      "dual modes of molecular control\n",
      "regulation involving either activator or repressor\n",
      "regulation involving combinations of activator and repressor\n",
      "single negative feedback loop (s.1)\n",
      "single positive and negative feedback loops: activator-only control of repressor transcription\n",
      "activator-primary control of activator transcription (s.2)\n",
      "repressor-primary control of activator transcription (s.3)\n",
      "positive and nested negative feedback loops: activator-primary control of repressor transcription\n",
      "activator-primary control of activator transcription (s.4)\n",
      "repressor-primary control of activator transcription (s.5)\n",
      "positive and nested negative feedback loops: repressor-primary control of repressor transcription\n",
      "activator-primary control of activator transcription (s.6)\n",
      "repressor-primary control of activator transcription (s.7)\n",
      "comparisons\n",
      "delay\n",
      "global tolerances of the oscillatory phenotype\n",
      "circuits with and without the nested negative feedback loop\n",
      "insights in the context of synthetic projects aimed at realizing robust oscillations\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001799/sb4001799.xml\n",
      "results and discussion\n",
      "finite state machines\n",
      "modeling gene regulatory networks\n",
      "biomolecular parts\n",
      "boolean network model\n",
      "delay differential equation model\n",
      "general construction of a grn from an fsm specification\n",
      "boolean network model of the general construction\n",
      "dde model of the general construction\n",
      "example: two-state machine as a boolean network\n",
      "example: two-state machine as a system of ddes\n",
      "ddes approximate the behavior of the boolean network\n",
      "fsm framework in a cellular information processing context\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00174/sb6b00174.xml\n",
      "results and discussion\n",
      "genetic screening for the identification of factors that suppress the cytotoxicity caused by mp overexpression\n",
      "suptoxd and suptoxr accumulate enhanced levels of membrane-embedded br2 with the correct n out -c in topology\n",
      "suptoxd and suptoxr broadly enhance recombinant production for a variety of homologous and hetero­logous mps\n",
      "comparison of suptoxd and suptoxr with commercial strains frequently utilized for recombinant mp production\n",
      "methods\n",
      "bacterial strains\n",
      "plasmid constructions\n",
      "membrane protein overexpression in liquid cultures\n",
      "membrane isolation\n",
      "western blot and in-gel fluorescence analysis\n",
      "spheroplast generation and labeling\n",
      "bulk fluorescence measurements\n",
      "gfp fluorescence analysis by flow cytometry\n",
      "neurotensin labeling\n",
      "real-time pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00261/sb6b00261.xml\n",
      "results and discussion\n",
      "design and validation of a three-plasmid system for screening engineered srna variants\n",
      "retargeting stem-antisense srna variants to non-native mrna reporter genes\n",
      "screening of single-stem retargeted srna variants\n",
      "modularity of the stem-antisense motif\n",
      "combinatorial construction and screening of dual-acting retargeted srnas\n",
      "dual-acting srnas as tools\n",
      "methods\n",
      "bacterial strains, plasmids, and growth media\n",
      "simultaneous cell growth and fluorescence assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00324/sb6b00324.xml\n",
      "results and discussion\n",
      "methods\n",
      "pf ago expression and purification\n",
      "restriction digestion using pf ago/ares and native restriction enzymes\n",
      "pcr cloning using pf ago/ares\n",
      "pcr products digestion with pf ago\n",
      "puc19 digestion with pf ago/ares\n",
      "analysis of correct plasmid assembly using pf ago/ares\n",
      "dna fingerprinting using pf ago/ares\n",
      "genomic dna cloning using pf ago ares\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00253/sb7b00253.xml\n",
      "results\n",
      "construction of autolysis plasmids\n",
      "direct autolysate production by freeze-drying and rehydration in situ\n",
      "autolysate production by freeze–thawing\n",
      "expression from linear dna in autolysates in the presence of gams protein\n",
      "concentrated autolysates\n",
      "engineered clpxp-mediated protein degradation in autolysates\n",
      "vibrio fischeri quorum sensing system in autolysates\n",
      "β-galactosidase-free autolysates\n",
      "cell-free mercury­(ii) biosensors\n",
      "discussion\n",
      "methods\n",
      "cell strains, plasmids, and reagents\n",
      "e. coli genome editing\n",
      "autolysate preparation by freeze–thawing\n",
      "preparation of clpx-rich autolysate\n",
      "cell-free expression\n",
      "cell-free expression using linear templates\n",
      "assembly pcr\n",
      "expression and purification of gams\n",
      "clpx-rich autolysate activity assay\n",
      "ahl production in autolysate\n",
      "preparation of in situ autolysates by freeze-drying followed by rehydration\n",
      "mercury­(ii) biosensors\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00044/sb6b00044.xml\n",
      "methods\n",
      "strains, media, and reagents\n",
      "plasmid construction\n",
      "gene deletion by crispr/cas9 plasmids\n",
      "β-galactosidase assays\n",
      "phenotypic identification of cells with pyre deletion\n",
      "fermentation\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00129/sb7b00129.xml\n",
      "results and discussion\n",
      "design of a novel biosynthetic pathway for vanillyl alcohol production\n",
      "enzymatic activity assay of comt\n",
      "bioconversion of 3,4-dihydroxybenzoic acid or 3,4-dihydroxybenzyl alcohol into corresponding products\n",
      "de novo production of vanillyl alcohol\n",
      "enhancement of vanillyl alcohol production via modular optimization\n",
      "conclusion\n",
      "methods\n",
      "media, strains and plasmids\n",
      "dna manipulation\n",
      "in vitro comt enzyme assay\n",
      "toxicity test\n",
      "feeding experiments\n",
      "de novo production of vanillyl alcohol\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400052f/sb400052f.xml\n",
      "results and discussion\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, plasmids, and genes\n",
      "microfluidic device\n",
      "live cell lithography\n",
      "photopolymerization\n",
      "fluorescence imaging\n",
      "laser flow cytometry\n",
      "stochastic simulation of the dynamic response of bacteria to an inductant\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00109/sb8b00109.xml\n",
      "results and discussion\n",
      "identification of effective methanol assimilation pathway in e. coli\n",
      "implementation of methanol oxidation and formaldehyde assimilation under anaerobic condition\n",
      "13 c-labeling demonstrates that methanol is assimilated into intracellular metabolites\n",
      "high yield production of succinate driven by methanol\n",
      "nad­(h) and nadh/nad + ratio changed significantly in the methylotrophic e. coli\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and plasmids\n",
      "strain construction\n",
      "media and growth conditions\n",
      "enzyme assays\n",
      "anaerobic fermentation\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00257/sb5b00257.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300056x/sb300056x.xml\n",
      "modeling transcriptional cross-regulation\n",
      "genetic toggle switches\n",
      "cloning large gene clusters from e. coli\n",
      "assembling bacterial dna fragments in yeast\n",
      "a new platform for metabolic pathway engineering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00373/sb7b00373.xml\n",
      "results\n",
      "metabolic profile prediction of shmt flux disturbance by i cw773\n",
      "attenuation of shmt flux by replacing the promoter of glya\n",
      "introduction of gcvthp to recycle glycine for one-carbon unit generation\n",
      "cellular growth and metabolite profiles in response to reprogramming one-carbon metabolic pathways\n",
      "cell morphology restoration by improving the supply of glycine-derived one-carbon units\n",
      "changes in enzymes involved in one-carbon unit cycle are responsible for the homeostasis of one-carbon units\n",
      "the effect of reprogramming one-carbon unit pathways on l -serine accumulation in fed-batch fermentation\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and cell growth\n",
      "construction of plasmids and strains\n",
      "flux balance analysis\n",
      "shmt activity assay\n",
      "real-time reverse transcription-pcr\n",
      "western blotting\n",
      "13 c-labeling glycine tracing\n",
      "high-performance liquid chromatography-quadrupole time-of-flight/mass spectrometry (hplc-q-tof-ms) assay\n",
      "optical analysis of cells and cell size determination\n",
      "comparative proteomic analysis\n",
      "fermentation in the shake flask and bioreactor\n",
      "analytic method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200029a/sb200029a.xml\n",
      "design and strategy\n",
      "model of tetracycline regulated expression systems\n",
      "engineering class b and class e tta activators\n",
      "evaluation of the dual-targeting repressor/activator system\n",
      "improved fold-expression control achieved through repression of activator transcription\n",
      "discussion\n",
      "methods\n",
      "modeling\n",
      "vectors\n",
      "cell engineering and culture\n",
      "doxycycline expression response measurements\n",
      "quantification of mrna levels\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00191/sb6b00191.xml\n",
      "methods\n",
      "plasmid editing, growth conditions, and data collection\n",
      "limit of detection assay\n",
      "timecourse assay\n",
      "orthogonality assay\n",
      "ratio assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00435/sb8b00435.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "materials\n",
      "preparation of vesicles\n",
      "preparation of micelles\n",
      "calcein release assays with α-hemolysin\n",
      "fret assays for oa uptake\n",
      "protein expression and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00144/sb7b00144.xml\n",
      "results and discussion\n",
      "design of a plasmid vector for simultaneous secretion and cell surface display\n",
      "f2a peptide enables simultaneous cell surface display and soluble secretion of sso7d mutants\n",
      "a combinatorial library of sso7d mutants incorporating the f2a peptide can be screened for high-affinity binders\n",
      "f2a peptide enables simultaneous yeast surface display and secretion of a dimeric enzyme\n",
      "f2a peptide enables simultaneous cell surface display and secretion of proteins when expressed as n- or c-terminal fusions to aga2\n",
      "conclusions\n",
      "materials and methods\n",
      "reagents\n",
      "construction of plasmid vectors containing the f2a peptide sequence\n",
      "yeast surface display of proteins in f2a for flow cytometry analysis\n",
      "capture and detection of solubly secreted sso7dstrep and sso7dhfc on his-tag isolation beads\n",
      "capture of solubly secreted sso7dhfc and sso7dstrep with streptavidin-coated beads\n",
      "purification of solubly secreted sso7dstrep from yeast culture supernatant\n",
      "generation of a combinatorial library using error-prone pcr\n",
      "combinatorial library screening\n",
      "flow cytometry analysis of enrichment for higher affinity lysozyme binders by facs\n",
      "purification and validation of soluble lysozyme binders from yeast cell culture supernatant\n",
      "cloning of gox into pct-nt-f2a and pct-ct-f2a\n",
      "measurement of gox activity\n",
      "purification of sso7dhfc-gox and gox\n",
      "immunoblotting analysis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00137/sb9b00137.xml\n",
      "results and discussion\n",
      "a putative glycine riboswitch controls a glycine symporter\n",
      "characterization of glycine riboswitch-based biosensors\n",
      "selection of glycine-off riboswitches with improved performance\n",
      "dose–response of glycine-off riboswitch clones\n",
      "glycine-off riboswitch for metabolic control in e. coli\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, plasmids, and media\n",
      "identification of glycine riboswitches in c. pasteurianum dsm 525 (atcc 6013)\n",
      "growth conditions and reporter assays\n",
      "construction of glycine-off riboswitch library\n",
      "dual genetic selection of glycine-off riboswitches\n",
      "screening and characterization of glycine-off riboswitches\n",
      "construction of e. coli glycine-off riboswitch- hemb mutant\n",
      "fermentation and analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00455/sb7b00455.xml\n",
      "results\n",
      "engineering a thermal gene switch\n",
      "triggering cellular activity with pulses of heat\n",
      "photothermal targeting of jurkat t cells\n",
      "thermal pulse trains for long-term control of jurkat t cells in vivo\n",
      "discussion\n",
      "methods\n",
      "plasmid construction and viral production\n",
      "preparation of aunrs\n",
      "viability studies\n",
      "in vitro heating assays\n",
      "in vivo laser heating\n",
      "in vivo bioluminescence and imaging\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000838/sb4000838.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001245/sb4001245.xml\n",
      "methods\n",
      "microorganisms and media\n",
      "expression elements\n",
      "expression vectors\n",
      "expression experiments\n",
      "plasmid stability experiments\n",
      "biolector biomass conversion to od 600 values\n",
      "calculating growth and maximum sfgfp production rates\n",
      "analysis of single-cell morphology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00415/sb7b00415.xml\n",
      "results and discussion\n",
      "the tf circuit generates a ca 2+ signal in a magnetic field\n",
      "the tf circuit with carq exhibits dynamic protrusions in a magnetic field\n",
      "the tf-carq cells exhibit wound healing in a magnetic field\n",
      "tf-carq cells exhibit transwell migration and directed cell spreading in a magnetic field\n",
      "conclusion\n",
      "experimental procedures\n",
      "plasmid construction\n",
      "cell culture, transfection and preparation\n",
      "illumination and imaging\n",
      "magnetic field and temperature measurements\n",
      "magnetic field stimulation\n",
      "ca 2+ imaging and dynamic protrusions\n",
      "wound healing\n",
      "boyden chamber assay\n",
      "rose diagram\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00296/sb6b00296.xml\n",
      "results and discussion\n",
      "timing analysis\n",
      "simulation results\n",
      "summary\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00055/sb9b00055.xml\n",
      "results\n",
      "experimental procedures\n",
      "cpp-mediated transformation for e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000788/sb4000788.xml\n",
      "results and discussion\n",
      "reconstruction of the we production pathway\n",
      "studying the functionality and inducibility of ara-luxcde complex\n",
      "temperature dependency of we production\n",
      "qualitative analysis of lipids produced by the synthetic pathway\n",
      "conclusions\n",
      "methods\n",
      "strains\n",
      "plasmid construction and transformation\n",
      "medium composition\n",
      "functional verification of the construct\n",
      "cultivations\n",
      "lipid extraction and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300091d/sb300091d.xml\n",
      "results and discussion\n",
      "engineering 1-butanol-dependent growth in e. coli\n",
      "engineering dicarboxylic acid-dependent growth in e. coli\n",
      "increasing e. coli 1-butanol biosynthesis using a biosensor-mediated screen\n",
      "synthetic selection for 1-butanol biosynthesis in engineered e. coli\n",
      "methods\n",
      "characterization of butanol- and dicarboxylic acid-dependent e. coli growth\n",
      "characterization of gfp-based biosensor\n",
      "liquid culture screening assays\n",
      "1-butanol biosynthesis selection assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00001/sb9b00001.xml\n",
      "results and discussion\n",
      "design principle of synthetic inducible regulatory system: modular design approach\n",
      "dynamic ranges of the induction modules: characterization of leakiness and inducibility\n",
      "inducer sensitivities of the induction modules: characterization of inducer concentration-dependent indigoidine production\n",
      "module compatibility: the cmt induction module combined with diverse constitutive rss\n",
      "refactoring of the act cluster: crispr/cas9-aided insertion of the operator sequence\n",
      "optimization of the repressor expression: further improvement of the dynamic range\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and culture conditions\n",
      "dna synthesis and molecular cloning\n",
      "measurement of indigoidine production\n",
      "creation of cymr-expressing streptomyces\n",
      "refactoring of the act cluster\n",
      "measurement of act production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00048/sb8b00048.xml\n",
      "results and discussion\n",
      "selection scheme for dttp uptake in e. coli\n",
      "construction and selection of dttp-auxotrophic strains\n",
      "nutritional containment of thya tdk dual deletants\n",
      "uptake of radio-labeled canonical dntps\n",
      "uptake of nucleoside triphosphates with altered sugar moieties\n",
      "uptake of a mutagenic nucleoside triphosphate\n",
      "directed evolution of dttp auxotrophic strains\n",
      "conclusion\n",
      "materials and methods\n",
      "strains, reagents and growth conditions\n",
      "insertion of ntt2 genes into the chromosome\n",
      "viability of tdk thya deletant strains in rich medium\n",
      "labeled dntp transport\n",
      "growth inhibition assays on plates\n",
      "decitabine triphosphate mutagenesis\n",
      "minimal inhibitory concentration\n",
      "serial batch cultivation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00274/sb7b00274.xml\n",
      "design of a toolbox of rna thermometers\n",
      "a new improved and extended version of the multicell bacterial simulator gro\n",
      "amplifying riboswitch signal output using cellular wiring\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00059/sb9b00059.xml\n",
      "results\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00028/sb6b00028.xml\n",
      "results and discussion\n",
      "primer-aided truncation for the creation of hybrid proteins (patchy)\n",
      "linker length governs activity and signal response in light-regulated histidine kinases\n",
      "synthetic biology recapitulates and transcends natural systems\n",
      "applications of patchy\n",
      "methods\n",
      "generation of itchy and patchy linker libraries\n",
      "library screening\n",
      "multiple sequence alignment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002592/sb5002592.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00440/sb7b00440.xml\n",
      "results and discussion\n",
      "using simplified computational models to guide nar network design\n",
      "optimization of rna parts for nar network construction\n",
      "accurate functional prediction of a nar network in tx-tl by a quantitative model\n",
      "an srna transcriptional nar network functions in e. coli\n",
      "noise reduction properties of srna transcriptional nar networks\n",
      "conclusion\n",
      "methods\n",
      "plasmid construction and purification\n",
      "tx-tl experiments\n",
      "network/model simulations\n",
      "parameterization experiment design\n",
      "parameter estimation\n",
      "strains, growth media and in vivo gene expression experiments\n",
      "response-time calculation\n",
      "signal noise analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00292/sb6b00292.xml\n",
      "results and discussion\n",
      "expression, secretion, and conjugation of spytag-spycatcher fusion proteins\n",
      "engineering xyna thermotolerance by spyring cyclization\n",
      "a modular method for extracellular protein–protein conjugation from cellular consortia\n",
      "tuning conjugation through consortia composition\n",
      "conclusion\n",
      "methods\n",
      "strains and plasmids\n",
      "plasmid construction\n",
      "protein expression and coculturing\n",
      "sds-page and western blotting\n",
      "enzyme activity assays\n",
      "protein purification\n",
      "sec-mals sample preparation\n",
      "enzyme thermotolerance assays\n",
      "coculture conditions, copurifications, and enzyme assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000063/sb3000063.xml\n",
      "surface structure of the native connector (p10)\n",
      "integration strategy: hydrophilicing the membrane with oxidized lipids\n",
      "results and discussion\n",
      "membrane integration in guvs: inmunofluorescence detection\n",
      "permeability tests\n",
      "size-probe retention in luvs\n",
      "pore connection to internal compartments\n",
      "permeability kinetics in guvs\n",
      "structural analysis by afm\n",
      "conclusions\n",
      "methods\n",
      "chemicals\n",
      "connector production\n",
      "buffer\n",
      "large unilamellar vesicles\n",
      "giant unilamellar vesicles (guvs)\n",
      "anti-p10 inmunofluorescence\n",
      "optical microscopy\n",
      "luvs for confocal microscopy\n",
      "supported planar lipid bilayers\n",
      "atomic force microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00105/sb6b00105.xml\n",
      "introduction\n",
      "results and discussion\n",
      "ala production by e. coli expressing yeast mitochondrial ala synthase\n",
      "enhanced ala production by expressing codon optimized hem1 in various recombinant e. coli\n",
      "ala production by e. coli under a t7 enhanced and antibiotic-free system\n",
      "ala production by an autoinduction (iptg-free) system under better growth conditions\n",
      "coproduction of ala and poly­(3-hydroxybutyrate) or phb by recombinant e. coli\n",
      "halomonas td as a low-cost platform for coproduction of ala and phb\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "construction of essential gene containing plasmids and study on plasmid stability\n",
      "construction of recombinant halomonas td01\n",
      "shake flask studies on ala accumulation\n",
      "coproduction of ala and phb by recombinant e. coli and halomonas td01\n",
      "analytical method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00154/sb5b00154.xml\n",
      "results and discussion\n",
      "efflux pump alters the dose–response of the nr synthetic gene circuit\n",
      "a mathematical model of the pump-mediated dose–response change\n",
      "efflux pump alters the dose–response of the nf synthetic gene circuit\n",
      "efflux pump mutants reveal that pump function is insufficient to explain dose–response differences\n",
      "experimental confirmation of lower tetr expression in nrpump and nfpump strains\n",
      "discussion and outlook\n",
      "methods\n",
      "synthetic gene circuit construction\n",
      "strains and medium\n",
      "reporter gene expression measurement\n",
      "data analysis\n",
      "computational modeling and data fitting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00364/sb8b00364.xml\n",
      "results and discussion\n",
      "function verification of pakω-ta and its ph and temperature dependence\n",
      "substrate spectrum of pakω-ta and amino donor select\n",
      "design of a three-enzyme biocatalysis cascade system\n",
      "optimization of the expression level of mnadh in coexpression e. coli\n",
      "the construction of redox self-sufficient system and key role of glutamate dehydrogenase\n",
      "coexpression of four enzymes in e. coli and one-step whole-cell bioconversion of epoxide to 1,2-amino alcohol\n",
      "preparation of 1,2-amino alcohols ( s )- 2d , ( s )- 3d , ( s )- 4d , and ( s )- 5d through whole-cell catalysis\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains, plasmids, and primers\n",
      "materials and media\n",
      "recruitment of the pakω-ta gene by homology search\n",
      "construction and expression of recombinant plasmids\n",
      "enzyme activity assays\n",
      "purification and characterization of the recombinant enzyme pakω-ta\n",
      "bioconversion conditions\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00113/sb7b00113.xml\n",
      "results and discussion\n",
      "general considerations\n",
      "editing-dependent inclusion of the nls into the c-terminus under transient expression\n",
      "c-terminal nls-inclusion works also under genomic expression\n",
      "editing-dependent inclusion of the nls into the n-terminus (transient expression)\n",
      "editing in the 5′-utr under genomic expression requires an activated deaminase\n",
      "isoform switching can be controlled by light\n",
      "5′-utr editing enables to switch localization from the cytoplasm to the outer membrane\n",
      "translocation to the outer membrane can be controlled by light\n",
      "conclusion and outlook\n",
      "methods\n",
      "editing under transient expression\n",
      "editing under genomic expression\n",
      "light-induced rna editing\n",
      "bg-fitc staining\n",
      "immunofluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00142/sb6b00142.xml\n",
      "introduction\n",
      "results\n",
      "overexpression of fus3 reduces pathway output\n",
      "overexpressed fus3 mutants can rescue pathway activity in different ways\n",
      "further analysis of psc containing variants suggests readthrough\n",
      "readthrough of psc leads to full-length active kinases\n",
      "psc containing variants can mediate a full mating response\n",
      "pscs are preferentially tolerated at surface residues\n",
      "discussion\n",
      "methods\n",
      "yeast strains\n",
      "library construction\n",
      "site directed mutagenesis\n",
      "transformation\n",
      "fluorescence-activated cell sorting\n",
      "flow cytometry\n",
      "western blots\n",
      "mating assay\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00165/sb9b00165.xml\n",
      "results\n",
      "construction of synthetic srnas for scaffold engineering\n",
      "effect of mutation in the a/u-rich sequence on repression efficiency\n",
      "effect of the length of the hairpin stem on gene repression ability\n",
      "effect of mutations in the loop sequence on repression levels\n",
      "a new srna scaffold with three mutated modules for expanding the srna repression range\n",
      "application of the engineered srna scaffold to cadaverine-producing strain\n",
      "discussion\n",
      "methods\n",
      "strains and culture conditions\n",
      "plasmid construction\n",
      "gene repression activity measurement using a fluorescence protein\n",
      "analytical methods\n",
      "hfq-srna elisa assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00127/sb9b00127.xml\n",
      "results\n",
      "schematic representation of combinatorial genetically encoded library for sequence-specific mucins\n",
      "constructing and validating the surface expression of sequence-specific mucins\n",
      "substituting the potential glycosylation sites with alanine in the mucin polymer backbone tunes o-glycan maturation\n",
      "designer mucin domains reveal sequence-specific effects on glycosylation\n",
      "tuning mucin glycosylation through cytoplasmic tail engineering\n",
      "discussions\n",
      "materials and methods\n",
      "antibodies and reagents\n",
      "gene design and assembly of muc1 tandem repeat domains\n",
      "pcr and golden gate assembly of extended synthetic tandem repeats\n",
      "cell lines, culture, and transfection\n",
      "western blot analysis\n",
      "periodate labeling of cell surface sialic acids\n",
      "immunoprecipitation\n",
      "sialidase treatment of hek293ts\n",
      "immunofluorescence\n",
      "secreted mucin protein expression and purification\n",
      "o-glycan profiling of secreted mucin protein\n",
      "cellular o-glycome reporter/amplification (cora)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500079f/sb500079f.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "combinatorial assembly of mitochondrion-based isobutanol pathway into yeast chromosomes\n",
      "combinatorial assembly of the mevalonate pathway into yeast chromosomes\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, and reagents\n",
      "plasmid construction\n",
      "yeast transformation and library construction\n",
      "pcr verification of genome integration events\n",
      "isobutanol production in engineered yeast cells\n",
      "amorpha-4,11-diene production in engineered yeast cells\n",
      "rna extraction and qrt-pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00221/sb5b00221.xml\n",
      "hanna aucoin\n",
      "robert chen\n",
      "tadas jakočiu̅nas\n",
      "justin r. klesmith\n",
      "nicole nozzi\n",
      "harneet rishi\n",
      "kylie standage-beier\n",
      "xiao wang\n",
      "jie zhang\n",
      "qi zhang\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300093y/sb300093y.xml\n",
      "results and discussion\n",
      "benchmark synthetic network\n",
      "characterization of the synthetic circuit\n",
      "reverse engineering approach\n",
      "network reconstruction using flow cytometry data\n",
      "network reconstruction using mrna measurements\n",
      "conclusion\n",
      "methods\n",
      "bootstrap\n",
      "stable transfection and cell line\n",
      "fluorescence microscopy\n",
      "flow cytometry\n",
      "quantitative rt-pcr\n",
      "resampling rate selection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00210/sb9b00210.xml\n",
      "results\n",
      "h 2 s is a metabolite proportional to e. coli cell density\n",
      "converting h 2 s into intracellularly detectable hs n h\n",
      "screening for functional hs n h-sensitive tfs\n",
      "construction of artificial sensor-actuator elements\n",
      "assembly of h 2 s-mediated gene circuit and test of its qs-like functions\n",
      "tuning dynamics of h 2 s-mediated gene circuit\n",
      "application of h 2 s-mediated gene circuit in cell factory construction\n",
      "discussion\n",
      "methods\n",
      "strains, plasmids, and chemicals\n",
      "h 2 s and hs n h analysis\n",
      "fluorescence analysis\n",
      "xylose and xylonic acid analysis\n",
      "microfluidic cultivation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00255/sb9b00255.xml\n",
      "results\n",
      "genomic contexts of csir and gcdr homologues and prediction of p. putida binding sites\n",
      "biochemical characterization of csir and gcdr\n",
      "development of two glutarate biosensor vectors\n",
      "developing metrics to quantify biosensor performance\n",
      "responsiveness of glutarate biosensors to lysine metabolites in p. putida\n",
      "discussion\n",
      "methods\n",
      "media, chemicals, and culture conditions\n",
      "strains and plasmids\n",
      "expression and purification of proteins\n",
      "plate based growth and fluorescence assays in p. putida\n",
      "transcriptional fusion fluorescence assays\n",
      "electrophoretic mobility shift assays\n",
      "measurement of 5-aminovalerate\n",
      "analysis of biosensor parameters\n",
      "bioinformatic analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00193/sb5b00193.xml\n",
      "results and discussion\n",
      "establishing a model multidomain protein system that can display changes in solubility via synonymous codon mutations within the linker-coding region\n",
      "changes in solubility due to random synonymous mutations correlate with changes in affinity between the ribosomal asd sequence and the mrna linker-coding region\n",
      "rational design of a large linker library that exhibits a wide range of asd affinities\n",
      "affinity between asd sequence and mrna is a good predictor of multidomain protein solubility\n",
      "effect of a longer high-affinity linker on protein solubility\n",
      "analysis of naturally occurring multidomain protein shows that linker regions separating highly structured domains have higher codon affinities to the anti-shine–dalgarno sequence\n",
      "conclusions\n",
      "methods\n",
      "plasmid and bacterial strains\n",
      "construction of the random linker coding region library\n",
      "screening of random linker-coding region library\n",
      "confirmation of fluorescence patterns by 96-well plate reader experiments\n",
      "computational prediction of mrna-anti-shine–dalgarno affinity\n",
      "construction of the rational linker coding region library\n",
      "flow cytometry screening of the rational library\n",
      "western blotting analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00274/sb5b00274.xml\n",
      "results\n",
      "gfp fluorescence depends on the adx redox state\n",
      "reduced state time depends on nadph and adx concentrations\n",
      "fluorescence unquenching is observed in fusions of gfp with plant ferredoxin\n",
      "stronger quenching is observed with shorter linkers\n",
      "other fluorescent proteins also can be reversibly unquenched\n",
      "a model for the reactions of electron transport by adx\n",
      "duration of unquenching is sensitive to cyp enzyme and substrate concentrations\n",
      "gfp-p450 fusion proteins exhibit redox state dependent fluorescent changes\n",
      "discussion\n",
      "materials and methods\n",
      "plasmid construction\n",
      "gfp-adx fluorescence measurements\n",
      "gfp-adx reduction with sodium dithionite\n",
      "steroid fluorescence assay\n",
      "gfp-adx fusion cleavage with tev protease\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00201/sb8b00201.xml\n",
      "background\n",
      "rule-based modeling of genetic processes\n",
      "the κ language\n",
      "biological parts and annotation\n",
      "a language for synthetic gene circuits\n",
      "desired language features\n",
      "vocabulary terms\n",
      "model description\n",
      "a part description\n",
      "a more complex part description\n",
      "host and protein–protein interactions\n",
      "protein fusion\n",
      "other parts\n",
      "output representation\n",
      "genetic circuit compiler\n",
      "semantic inference\n",
      "internal representation\n",
      "template substitution\n",
      "derivation of declarations\n",
      "initialization\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00368/sb6b00368.xml\n",
      "methods\n",
      "bacterial strains, media, and reagents\n",
      "plasmid construction\n",
      "reporter strain construction\n",
      "purification of eecpf1 and eedcpf1\n",
      "in vitro nuclease activity assays\n",
      "fluorescence assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500364q/sb500364q.xml\n",
      "methods\n",
      "strains, media, and culture conditions\n",
      "strains and plasmid construction\n",
      "fermentation experiments\n",
      "enzymatic assay of β-glucosidase activity\n",
      "determination of genomic copy numbers of gh1-1 and cdt-1 by quantitative pcr\n",
      "analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00150/sb9b00150.xml\n",
      "results and discussion\n",
      "determination of suitable replicon and antibiotic concentration\n",
      "construction of an inducible promoter system\n",
      "evaluation of sgrna availability in e. limosum genome\n",
      "precise gene deletion using crispr-cas9-mediated homologous recombination\n",
      "fold and acsa are essential genes for autotrophic growth in e. limosum\n",
      "effect of crispr interference system on target gene repression\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains and growth conditions\n",
      "transformation\n",
      "construction of inducible promoter plasmid\n",
      "construction of pjir-cas9 (dcas9) plasmid and knock-out template\n",
      "design of sgrnas from e. limosum genome sequence\n",
      "dna extraction\n",
      "western blot\n",
      "quantitative rt-pcr\n",
      "crispri competitive growth assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00279/sb8b00279.xml\n",
      "results and discussion\n",
      "design and implementation of a noise tuner\n",
      "model of expression noise\n",
      "tuning of information flow through a signaling pathway\n",
      "conclusion\n",
      "methods\n",
      "plasmids and strains\n",
      "media and growth conditions\n",
      "flow cytometry measurements and noise calculation\n",
      "stochastic simulation\n",
      "calculation of mutual information\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500194z/sb500194z.xml\n",
      "amuser web server\n",
      "input format\n",
      "output construct options\n",
      "site-directed mutagenesis\n",
      "degenerate nucleotides\n",
      "single insert combinatorial assembly\n",
      "pcr settings\n",
      "web interface\n",
      "experimental demonstration\n",
      "discussion\n",
      "methods\n",
      "software\n",
      "primer design for experimental demo\n",
      "pcr amplification of dna fragments\n",
      "one-step user assembly\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00225/sb7b00225.xml\n",
      "results\n",
      "development of a light inducible optotgfbrs system\n",
      "light induces smad2 phosphorylation and downstream gene expression with optotgfbrs\n",
      "control of tgf-β/smad signaling in time and space with optotgfbrs at the single cell level\n",
      "generating complex smad2 signaling dynamics by manipulating the pattern of light stimulations\n",
      "discussion\n",
      "methods\n",
      "cell culture and transient transfection\n",
      "plasmids preparation\n",
      "stable cell line generation\n",
      "cell lysate preparation and western blot\n",
      "leds illumination box\n",
      "cell imaging and photoactivation\n",
      "microscope imaging analysis\n",
      "cell viability assay\n",
      "rna isolation and qpcr analysis\n",
      "mathematical modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00130/sb8b00130.xml\n",
      "results and discussion\n",
      "kinetic analysis of key metabolites in cytosol and mitochondria of a. oryzae\n",
      "effects of overexpressing pyc in cytosol and mitochondria on malate synthesis\n",
      "engineering the oxidative branch of tca cycle and glyoxylate bypass in mitochondria\n",
      "effects of down-regulating the oxidative tca cycle by rna interference (rnai) on malate synthesis\n",
      "engineering the exchange of succinate and fumarate between cytosol and mitochondria\n",
      "engineering redox metabolism for improved l -malate production\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, plasmids, and transformation\n",
      "cultivation conditions\n",
      "purification of mitochondria\n",
      "metabolite analysis\n",
      "rna extraction and transcription analysis\n",
      "enzyme activity assay and determination of nadh/nad + concentrations\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00168/sb8b00168.xml\n",
      "results and discussion\n",
      "characterization of constitutive promoters in acetobacteraceae\n",
      "characterization of inducible promoters in acetobacteraceae\n",
      "characterization of natural and synthetic terminators in acetobacteraceae\n",
      "characterization of ribosome binding sites in acetobacteraceae\n",
      "characterization of protein degradation tags in acetobacteraceae\n",
      "synthesis of a cellulose-chitin copolymer\n",
      "modulation of the bacterial cellulose biosynthetic pathway\n",
      "implementation of crispr interference\n",
      "materials and methods\n",
      "bacterial strains and media\n",
      "plasmid construction\n",
      "preparation of gluconacetobacter electrocompetent cells and electroporation\n",
      "characterization of rbs variants, constitutive anderson promoters, degradation tags, inducible promoters, and terminators constructs\n",
      "determination of cellulose pellicle weight\n",
      "fluorescence microscopy\n",
      "scanning electron microscopy\n",
      "lectin staining\n",
      "quantitative real-time pcr (qrt-pcr)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00028/sb9b00028.xml\n",
      "results\n",
      "sequence ordering effect on the thermostability of tna:dna duplexes\n",
      "purine content effect on duplex stability and conformation\n",
      "gc content effect on duplex stability\n",
      "interplay of duplex length with purine content in determining stability\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "synthesis of oligonucleotides\n",
      "melting curve experiments and determination of thermodynamic parameters\n",
      "cd spectroscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00171/sb7b00171.xml\n",
      "results and discussion\n",
      "methods\n",
      "reagents\n",
      "microorganisms and culturing conditions\n",
      "plasmid design, construction, and transformation\n",
      "promoter and riboswitch evaluation\n",
      "chromosomal insertion of theo-f riboswitch\n",
      "predation assays in liquid culture and on agar plates\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00167/sb9b00167.xml\n",
      "materials and methods\n",
      "construction of the prsfgfp-nisb backbone and pacyc-nisc\n",
      "construction of lanthipeptide expressing strains\n",
      "protein expression and purification\n",
      "antimicrobial activity\n",
      "optimization of protease digestion and gfp-pnisin expression\n",
      "sds-page\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00386/sb7b00386.xml\n",
      "material and methods\n",
      "detection of translated orfs from ribo-seq data and related analysis\n",
      "noncoding peptides (ncpep)-protein interaction (ncppi) network prediction and ncpep analysis\n",
      "database implementation and network visualization\n",
      "statistical data analyses\n",
      "results\n",
      "identification of putative ncrnas and their translation products\n",
      "characteristics of putative ncrnas\n",
      "ncppi network prediction\n",
      "ncppi network-related peptide analysis\n",
      "visualization of ncpeps and ncppi network\n",
      "case studies of ncppi network\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00010/sb8b00010.xml\n",
      "meike baumgart\n",
      "nicholas c. o. lee\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00191/sb5b00191.xml\n",
      "methods\n",
      "culture conditions\n",
      "expression clone construction\n",
      "transformation of diatoms\n",
      "isolation of diatom biosilica frustules\n",
      "pdmpo and ascy3e labeling of live diatoms\n",
      "ascy3 labeling of isolated diatom biosilica\n",
      "fluorescent antigen synthesis and binding to single chain antibodies\n",
      "epifluorescence microscopy\n",
      "3d-sim\n",
      "fluorescence lifetime measurements\n",
      "analysis of fluorescence intensity decay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00259/sb7b00259.xml\n",
      "results\n",
      "conclusion\n",
      "methods\n",
      "plasmid and strain construction\n",
      "fluorescence measurements\n",
      "growth medium\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00230/sb5b00230.xml\n",
      "results and discussion\n",
      "beti-regulated t5 promoter system\n",
      "parallel selection for functional bet-on/bet-off mutants\n",
      "characterization of beti variants\n",
      "monodispersed and graded response of beti mutations to choline concentration\n",
      "strain-dependence in the switching properties of bet-switches\n",
      "bet-switches as pathway controllers\n",
      "two-input boolean logic gates consisting of synthetic choline/tetracycline promoters\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, media, and chemicals\n",
      "plasmid construction\n",
      "on-/off-selection of t5-based beton/off switches\n",
      "fluorescence analysis\n",
      "carotenoid pigment analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000077/sb5000077.xml\n",
      "results and discussion\n",
      "methods\n",
      "plasmid assembly\n",
      "ethylene production\n",
      "fruit ripening\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00376/sb6b00376.xml\n",
      "methods\n",
      "barcode construction\n",
      "library construction\n",
      "primer and linker design\n",
      "primer design software\n",
      "library genotyping\n",
      "selection\n",
      "quantification\n",
      "genotype data analysis\n",
      "noise calculations\n",
      "quantitative data analysis\n",
      "genotype error calling model\n",
      "copy number variation noise simulation\n",
      "coverage versus reads simulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300010a/sb300010a.xml\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00330/sb6b00330.xml\n",
      "result and discussion\n",
      "construction of sa. spinosa nrrl 18395 bac library and screening of spinosyn biosynthesis gene cluster\n",
      "introduction of spinosyn biosynthetic gene cluster into streptomyces by triparental conjugation\n",
      "transcriptional and translational analysis of biosynthetic genes in streptomyces and identification of heterologous production of spinosyn\n",
      "omics analysis revealed the rate-limiting steps in heterologous production of spinosyn\n",
      "overexpression of sugar biosynthetic module, methyltransferase and polyketide synthase to promote spinosyn biosynthesis\n",
      "methods\n",
      "microorganisms, fermentation and analysis\n",
      "bac library construction\n",
      "conjugation of bac plasmid to streptomyces\n",
      "rna isolation and reverse transcription-pcr (rt-pcr)\n",
      "targeted proteomics analysis\n",
      "targeted metabolomics analysis\n",
      "gene overexpression plasmid construction\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000614/sb4000614.xml\n",
      "pigeon: a design visualizer\n",
      "“addicted” bacteria for decaffeination and caffeine measurement\n",
      "redirecting photosynthesis for natural product synthesis\n",
      "a solid-phase platform for multipart gene assembly\n",
      "quantifying translational coupling in e. coli\n",
      "a “terminatome” toolbox\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300052u/sb300052u.xml\n",
      "refactoring the gene vii/gene ix overlap\n",
      "refactored m13 is capable of p9modification and peptide display\n",
      "refactored m13 enables cancer cell targeting, imaging, and drug delivery\n",
      "methods\n",
      "refactoring p9 and p7 of m13\n",
      "engineering p9 for n-terminal display\n",
      "construction of m13 vector for p3/p8 display\n",
      "phage amplification and purification\n",
      "biotinylation of m13-983\n",
      "western blotting and elisa for bap functionality on re-engineered p9\n",
      "conjugation for imaging and therapy in vitro\n",
      "imaging and cytotoxicity assay\n",
      "cell lines and culture\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00092/sb9b00092.xml\n",
      "results\n",
      "representing combinatorial design\n",
      "use case: pathway design\n",
      "use case: genetic circuit design\n",
      "use case: sboldesigner\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00192/sb8b00192.xml\n",
      "methods\n",
      "cell cultures\n",
      "reagents\n",
      "plasmids\n",
      "preparation of the gedex system (extracellular vesicle isolation)\n",
      "luciferase activity assay\n",
      "semiquantitative pcr\n",
      "t7e1 assay\n",
      "elisa assays\n",
      "immunoblotting\n",
      "electron microscopy\n",
      "animals\n",
      "in vivo luciferase imaging\n",
      "in vivo tracking of extracellular vesicles\n",
      "acute liver injury-induced mouse model\n",
      "histology of mouse liver\n",
      "statistical analyses\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00306/sb6b00306.xml\n",
      "results and discussion\n",
      "forward problem: finding pareto optimal synthetic circuits\n",
      "design of an oscillator with optimal properties\n",
      "reverse design: inferring design principles of gene regulatory networks\n",
      "finding motifs for stripe formation\n",
      "finding motifs for rapid adaptation\n",
      "finding motifs for fold-change detection\n",
      "methods\n",
      "multiobjective mixed-integer nonlinear programming formulation of the automated design problem\n",
      "computing the pareto front of optimal trade-offs\n",
      "knowledge discovery from the pareto front\n",
      "selection of best circuit(s) for implementation in forward design\n",
      "inferring design principles in forward and reverse design\n",
      "clustering of solutions in the decision space\n",
      "heat maps and cp lines\n",
      "flexible pattern mining\n",
      "computational efficiency and scalability of the method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00201/sb5b00201.xml\n",
      "methods\n",
      "chemicals and media\n",
      "plasmids construction\n",
      "strains and culture conditions\n",
      "batch fermentations\n",
      "fed-batch fermentation\n",
      "gas chromatography (gc) quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3001276/sb3001276.xml\n",
      "a heritable recombination system in yeast\n",
      "dimerization-dependent green and yellow fluorescent proteins\n",
      "a new platform for tumor-cell imaging and drug delivery\n",
      "organization of a synthetic alphoid dna array in a human artificial chromosome\n",
      "scalable plasmid transfer using engineered phagemids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00521/sb8b00521.xml\n",
      "results\n",
      "validation of the crucial recognition sequence of the rna replication system (rrs)\n",
      "construction and characterization of the rna replication system (rrs)\n",
      "identification of rnc gene as a knockout target to enhance rrs function in host cells\n",
      "universality test of rrs\n",
      "further application broadening of rrs\n",
      "extended application of rrs in metabolic engineering\n",
      "discussion\n",
      "materials and methods\n",
      "bacterial culture\n",
      "bacterial strains and plasmids\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "insertion of constructs into the genome of e. coli\n",
      "flow-cytometry (fcm) measurement\n",
      "rna purification and quantitative real-time polymerase chain reaction (qrt-pcr)\n",
      "western blot analysis\n",
      "northern blot analysis\n",
      "phap protein production and detection\n",
      "shake-flask studies of ala production\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300092n/sb300092n.xml\n",
      "synthetic biology and antibiotics\n",
      "glycopeptides: a model platform to explore antibiotic synthetic biology\n",
      "gpa primary modifications\n",
      "halogenation\n",
      "sulfation\n",
      "methylation\n",
      "glycosylation\n",
      "gpa secondary modifications\n",
      "gpa production chassis\n",
      "conclusions and caveats\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300060e/sb300060e.xml\n",
      "experimental results\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500358y/sb500358y.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500024b/sb500024b.xml\n",
      "results and discussion\n",
      "scaffold-based circuit design and implementation\n",
      "modeling dynamics and steady state circuit behavior\n",
      "characterization of step response\n",
      "circuit closely follows three step induction\n",
      "inducer diffusion rates contribute to cumulative effect of sequential pulses\n",
      "model-informed exploration of parameter space\n",
      "scaffold-based circuits for rapid feedback\n",
      "methods\n",
      "cell strain and media\n",
      "plasmids\n",
      "plate reader experiments\n",
      "western blots\n",
      "microscopy\n",
      "model implementation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00328/sb6b00328.xml\n",
      "results\n",
      "design and characterization of a novel l -phe biosensor\n",
      "biosensor response to extracellular l -phe\n",
      "improving the sensitivity and dynamic range of the biosensors\n",
      "biosensor-driven l -phe synthetic pathway optimization\n",
      "use of the biosensor to screen hyperproducing strains from a random mutagenesis library\n",
      "elucidation of evolved enzyme in the l -phe biosynthesis pathway\n",
      "altering the transcription level of key enzymes to improve l -phe production\n",
      "discussion\n",
      "methods\n",
      "bacterial strains, plasmids, and cultivation medium\n",
      "design and optimization of biosensors\n",
      "dipeptide feeding assay\n",
      "preparation of cell extracts to detect intracellular l -phe concentration\n",
      "determination of l -phe production in 96-well microtiter plates\n",
      "generation of the synthetic rbs library\n",
      "construction of the random mutagenesis library and high-throughput screening\n",
      "construction of back mutants of hdh6-d12 by crispr-cas9-mediated genome editing\n",
      "construction of plasmids for verifying the function of snps\n",
      "enzymatic activity analysis of ahas\n",
      "quantitative real-time pcr analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00038/sb7b00038.xml\n",
      "jonathan boreyko\n",
      "jackson cahn\n",
      "patrick caveney\n",
      "michael chavez\n",
      "quan chen\n",
      "charles chin\n",
      "kulika chomvong\n",
      "robert j. citorik\n",
      "c. patrick collier\n",
      "rené daer\n",
      "derek fedeson\n",
      "joshua fern\n",
      "dimitra gialama\n",
      "tiyun han\n",
      "karmella haynes\n",
      "jonathan ho\n",
      "s. elizabeth norred\n",
      "scott retterer\n",
      "dominic scalise\n",
      "michael simpson\n",
      "cheemeng tan\n",
      "peng-fei xia\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00205/sb6b00205.xml\n",
      "results and discussion\n",
      "construction of a phlf-based transcriptional regulator and a phlf operator-embedded promoter for the dapg-off system\n",
      "performance of the dapg-off switch with a gfp reporter\n",
      "dapg-off system with an ade2 reporter\n",
      "construction of off-switches using other tetr homologues\n",
      "orthogonality of the yeast off-switches\n",
      "construction of two components for the dapg-on system\n",
      "dapg-on system with an ade2 reporter\n",
      "perspectives\n",
      "methods\n",
      "media\n",
      "plasmids\n",
      "yeast strains\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00341/sb6b00341.xml\n",
      "methods\n",
      "general methods and strain cultivation conditions\n",
      "chemical synthesis of 2-(prop-2-yn-1-yl)­malonic acid\n",
      "cloning of the malonyl-coa synthetase expression construct\n",
      "feeding of allylmalonic acid or propargylmalonic acid to s. collinus tü 365 and s. collinus prm4.2- matb\n",
      "scale-up of feeding and extraction of kirromycin derivatives\n",
      "purification of allyl-kirromycin and propargyl-kirromycin\n",
      "high resolution mass spectrometry (hrms) analysis of kirromycins\n",
      "nuclear magnetic resonance (nmr) spectroscopy of kirromycins\n",
      "cuaac reaction of propargyl-kirromycin and coumarin-343-azide and product purification\n",
      "in vitro activity testing of kirromycin and kirromycin derivatives\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00127/sb8b00127.xml\n",
      "results and discussion\n",
      "data collection and analysis\n",
      "model construction and optimization\n",
      "engineering 5′utrs using nusar\n",
      "evaluation of 5′utr elements derived from nusar\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00007/sb9b00007.xml\n",
      "results and discussion\n",
      "carbon catabolite repression limits sugar coutilization in e. coli fermentation monocultures\n",
      "development of catabolically orthogonal specialist strains from e. coli w\n",
      "engineering and optimizing a synthetic coculture of wild-type derived, catabolically orthogonal specialist strains for glucose–xylose coutilization\n",
      "development of catabolically orthogonal specialist strains from ethanologenic e. coli\n",
      "engineering and optimizing a synthetic ethanologenic e. coli coculture\n",
      "population dynamics of ethanologenic coculture systems during the fermentation of glucose–xylose mixtures\n",
      "methods\n",
      "strain construction\n",
      "fermentation conditions and media\n",
      "analytical methods and data analysis\n",
      "adaptive laboratory evolution to improve glucose catabolism\n",
      "quantification of viable cells of individual specialist strains in cocultures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00036/sb6b00036.xml\n",
      "results and discussion\n",
      "hfq sites\n",
      "target binding region design\n",
      "target location\n",
      "mismatch\n",
      "length\n",
      "thermodynamics\n",
      "ribosome interactions\n",
      "yunr motif\n",
      "srcc and normalization\n",
      "development of asrna design rules\n",
      "off-target effects\n",
      "validation of design rules\n",
      "complex genetic circuit construction\n",
      "conclusions\n",
      "materials and methods\n",
      "sequence design\n",
      "plasmid construction\n",
      "reporter fluorescence measurements to determine percent repression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00408/sb7b00408.xml\n",
      "results and discussion\n",
      "codon selection\n",
      "orthogonality among otss\n",
      "simultaneously installing serine phosphorylation and lysine acetylation in the sfgfp\n",
      "simultaneously incorporating ack and sep into malate dehydrogenase (mdh)\n",
      "studying effects of acetylation and phosphorylation on mdh\n",
      "methods\n",
      "chemicals and materials\n",
      "plasmid construction\n",
      "the sfgfp readthrough assay\n",
      "protein expression, purification and characterization\n",
      "mass spectrometry (ms) analyses\n",
      "the malate dehydrogenase (mdh) activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00075/sb5b00075.xml\n",
      "results and discussion\n",
      "dual genetic selection of ecrs-on- teta library\n",
      "characterization of ecrs-on- lacz\n",
      "lysine-on riboswitch for metabolic control in c. glutamicum\n",
      "batch fermentations with enhanced lysine production\n",
      "methods\n",
      "bacterial strains, plasmids, media, and growth conditions\n",
      "dna manipulation\n",
      "construction of the lysine-on riboswitch library\n",
      "dual genetic selection\n",
      "construction of lysine-on riboswitch- lacz fusions\n",
      "construction of c. glutamicum lysine-on riboswitch- lyse mutants\n",
      "fermentation and analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00207/sb8b00207.xml\n",
      "materials and methods\n",
      "plasmid construction\n",
      "cultivation of yeast\n",
      "cytometer measurement\n",
      "machine learning\n",
      "mlgeneral consideration\n",
      "mldetailed description\n",
      "evaluation of classification\n",
      "software\n",
      "results and discussion\n",
      "design of the studythe tandem riboswitch\n",
      "machine learning workflow\n",
      "distribution of all biophysical parameters used for the random forest classification\n",
      "analysis of the random forest classifier\n",
      "a second classifier for the last optimization round and calculation of the h-bond score hs\n",
      "a riboswitch with 40x dynamic range\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00121/sb7b00121.xml\n",
      "methods\n",
      "implementation\n",
      "cellular growth\n",
      "cell–cell mechanics\n",
      "environmental geometry\n",
      "grn implementation and simulation\n",
      "case study\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300027y/sb300027y.xml\n",
      "methods\n",
      "translation strength prediction and rbs sequence designing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00086/sb7b00086.xml\n",
      "results\n",
      "formate assimilation relieves c1-auxotrophy\n",
      "formate assimilation relieves serine-auxotrophy\n",
      "threonine biosynthesis and degradation relieves glycine-auxotrophy in a formate-dependent strain\n",
      "rewiring central metabolism via endogenous production and consumption of formate\n",
      "discussion\n",
      "materials and methods\n",
      "reagents\n",
      "synthetic-operon construction\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bacterial strains\n",
      "growth conditions\n",
      "carbon labeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400092e/sb400092e.xml\n",
      "multichange isothermal mutagenesis\n",
      "facile engineering of novel genetic regulatory devices\n",
      "a genetic circuit for dynamic metabolic engineering\n",
      "real-time mrna measurement using binary probes\n",
      "synthetic gene circuit-mediated monitoring of endogenous metabolites\n",
      "eliminating redundant and stop codons\n",
      "modular design of a synthetic payload delivery device\n",
      "retroactivity controls the temporal dynamics of gene transcription\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00373/sb8b00373.xml\n",
      "results and discussion\n",
      "salanto: a versatile bioinformatical pipeline for analysis of shuffled aav capsids\n",
      "standardized plasmids for routine aav dna shuffling\n",
      "robust protocols for aav dna fragmentation and reassembly\n",
      "improved recombination with sequence-adapted aav4 and aav5 cap\n",
      "dna barcoding as a novel means to track cocktails of aav capsid libraries\n",
      "methods\n",
      "plasmids\n",
      "cloning of cap -aap hybrids\n",
      "amplification of cap sequences for dna shuffling\n",
      "aav cap gene fragmentation\n",
      "reassembly of full-length cap and cloning into plib\n",
      "cell culture\n",
      "virus production and purification\n",
      "titration of aav virus stocks\n",
      "salanto\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300031n/sb300031n.xml\n",
      "results\n",
      "modeling framework\n",
      "cell biophysics\n",
      "microcolony morphology\n",
      "rule-based cell behavior\n",
      "programmed growth\n",
      "intracellular dynamics and signaling\n",
      "domain boundary detection\n",
      "discussion\n",
      "methods\n",
      "computation\n",
      "microbial cultures\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001992/sb4001992.xml\n",
      "results and discussion\n",
      "optimization of lcr assembly\n",
      "characterization of lcr assembly\n",
      "comparison of lcr assembly against existing dna assembly methods\n",
      "methods\n",
      "amplification primers, bridging oligos, and overlap sequences\n",
      "pcr amplification, purification, and quantification of dna parts\n",
      "dna assembly via ligase cycling reaction (lcr)\n",
      "dna assembly via gibson isothermal assembly\n",
      "dna assembly via circular polymerase extension cloning (cpec)\n",
      "dna assembly via yeast homologous recombination\n",
      "transformation to e. coli , colony counting, and restriction endonuclease dna fragment analysis\n",
      "sequencing of selected clones\n",
      "optimization of lcr assembly efficiency\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300029m/sb300029m.xml\n",
      "results and discussion\n",
      "construction of a superlibrary of alternate scaffolds using yeast surface display\n",
      "isolation of binders to model targets from the superlibrary\n",
      "role of scaffold versus sequence diversity in combinatorial libraries\n",
      "analysis of specificity and thermal stability\n",
      "conclusions\n",
      "methods\n",
      "construction of yeast surface display libraries\n",
      "library screening\n",
      "estimation of k d\n",
      "biophysical characterization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300096v/sb300096v.xml\n",
      "genetic circuits in salmonella\n",
      "gene expression dynamics of tumor-targeted bacteria\n",
      "maintenance of multiple plasmids in e. coli\n",
      "engineered e. coli to detect gut inflammation\n",
      "integrated digital circuits to scale-up molecular computation\n",
      "rapid synthesis of eukaryotic promoter libraries\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00200/sb6b00200.xml\n",
      "results and discussion\n",
      "a high-level design workflow for dna synthesis and assembly\n",
      "boost automates design for synthesis and assembly\n",
      "boost results in efficiency gains\n",
      "boost-related software tools\n",
      "future plans\n",
      "methods\n",
      "sequence exchange formats\n",
      "reverse-translation and codon juggling\n",
      "verification of a dna sequence against common dna synthesis constraints\n",
      "reporting sequence regions that violate gene synthesis constraints\n",
      "modifying coding sequences to comply with gene synthesis constraints\n",
      "partitioning dna sequences into synthesizable building blocks\n",
      "implementation details\n",
      "availability and system requirements\n",
      "licensing information\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00061/sb9b00061.xml\n",
      "results and discussion\n",
      "bioinformatic identification of putative promoters from the core genome of four geobacillus species\n",
      "in vivo characterization of putative promoters\n",
      "sequence-function modeling\n",
      "partition modeling\n",
      "artificial neural network and partial least squares sequence-function modeling\n",
      "predicting the function of previously uncharacterized promoters\n",
      "future applications of promoter sequence-function modeling\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains and plasmids\n",
      "growth media\n",
      "bioinformatic identification of putative promoters from the core genome of four geobacillus species\n",
      "bioinformatic identification of putative promoter sequences from bacteriophage\n",
      "selection, synthesis, and cloning of putative promoters for in vivo characterization\n",
      "transformation of chemically competent e. coli\n",
      "conjugal transformation of g. thermoglucosidasius\n",
      "in vivo characterization of promoter activity\n",
      "promoter sequence-function modeling\n",
      "partition modeling\n",
      "selection of an independent test set for pls and ann modeling\n",
      "partial least squares sequence-function modeling\n",
      "artificial neural network sequence–function modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500361d/sb500361d.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00118/sb8b00118.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "yeast media, plasmids and strains\n",
      "gpr1 mutagenesis library and screening\n",
      "suc2 leucine reporter assay\n",
      "glucose activation assay\n",
      "deep sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00179/sb8b00179.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "media and growth conditions\n",
      "strains and plasmids\n",
      "aerobic and anaerobic growth rescue\n",
      "selection of d -ldh library\n",
      "purification and characterization of d -ldh variants\n",
      "rosetta modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300054p/sb300054p.xml\n",
      "transcription of coi triggers the p1 lytic cycle\n",
      "cin is a cis element that improves phagemid transduction\n",
      "phagemids exhibit robust and faithful transduction\n",
      "summary and conclusions\n",
      "methods\n",
      "plasmids, strains, phage, and growth media\n",
      "generation of phage lysates\n",
      "transduction of phage lysates\n",
      "timecourse of phage lysis\n",
      "measurement of relative promoter units (rpu)\n",
      "visualization of serial lysis\n",
      "phagemid library construction and enrichment\n",
      "competition between phagemids\n",
      "construction of library of gfp expressing phagemids\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500314r/sb500314r.xml\n",
      "results and discussion\n",
      "methods\n",
      "rna synthesis\n",
      "rna circuit reactions\n",
      "rna and dna sequences used in this study\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00024/sb9b00024.xml\n",
      "results and discussion\n",
      "analysis of the erythromycin bgc promoters and combination of bdps\n",
      "crispr-cas9 strategy for bdp knock-in\n",
      "bdp knock-in activated the erythromycin bgc\n",
      "optimized the metabolic flow to increase erythromycin production\n",
      "methods\n",
      "microorganisms and culture medium\n",
      "construction of promoter test plasmids\n",
      "bidirectional promoter knock-in with crispr-cas9\n",
      "rna extraction, cdna synthesis, and quantitative real-time pcr analysis\n",
      "biomass and evaluation of secondary metabolites\n",
      "quantitative measurement of fluorescence\n",
      "crispr interference\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00455/sb8b00455.xml\n",
      "results\n",
      "pam-proximal mismatches are deleterious to acecas9 activity in bacterial cells\n",
      "phosphate lock variants tolerate −1 mismatch\n",
      "the sizes of phosphate lock residues impact sensitivity to mismatches\n",
      "phosphate lock residues impact sensitivity to mismatches at other positions\n",
      "buried surface area of the target is correlated to off-target sensitivity\n",
      "discussion\n",
      "materials and methods\n",
      "plasmid and library construction\n",
      "in vivo selection through survival assay\n",
      "next generation sequencing\n",
      "protein expression and purification\n",
      "in vitro transcription and purification of sgrna\n",
      "plasmid dna cleavage assay\n",
      "acecas9 structural model\n",
      "calculation of buried surface areas\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500287j/sb500287j.xml\n",
      "jaume bacardit\n",
      "claire baudier\n",
      "tuval ben-yehezkel\n",
      "jonathan blakes\n",
      "michael guinn\n",
      "linh huynh\n",
      "jongmin kim\n",
      "vishwesh kulkarni\n",
      "pierre parutto\n",
      "ofir raz\n",
      "pawel widera\n",
      "boyan yordanov\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500322u/sb500322u.xml\n",
      "results and discussion\n",
      "methods\n",
      "cells\n",
      "viral production and transduction\n",
      "myod-directed genetic reprogramming\n",
      "quantitative reverse transcription pcr\n",
      "western blot\n",
      "immunofluorescence staining\n",
      "quantitative cell fusion assay\n",
      "rna-seq\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00099/sb9b00099.xml\n",
      "results and discussion\n",
      "collection of optimal growth temperature and proteomes of microorganisms\n",
      "ogt can be accurately predicted from amino acid composition of the proteome\n",
      "validation of the svr model for growth temperature prediction\n",
      "improved estimation of enzyme temperature optima using machine learning\n",
      "annotating enzymes in brenda using ogt and predicted t opt\n",
      "tome: a command line tool for ogt prediction and identification of enzyme homologues with different t opt\n",
      "methods\n",
      "software\n",
      "proteome data set\n",
      "estimation of threshold\n",
      "machine learning workflow for ogt model\n",
      "ogt prediction validation\n",
      "machine learning workflow for t opt model\n",
      "brenda annotation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00193/sb6b00193.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "oligonucleotides synthesis\n",
      "synthesis of dktp and dxtp\n",
      "page purification of the template and primer\n",
      "enzyme purification\n",
      "labeling of the primer for primer extension reaction\n",
      "primer extension\n",
      "ligation of kx containing inserts in a p15a3 vector\n",
      "media and electrocompetent cells\n",
      "transformation of the ligated sequences\n",
      "pcr for insert control\n",
      "plasmid preparation and sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400027x/sb400027x.xml\n",
      "synthetic biology\n",
      "microbial cell factories\n",
      "artificial organisms\n",
      "more to come\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00266/sb8b00266.xml\n",
      "methods\n",
      "preparation of dna template arrays and generation of mrna arrays\n",
      "generation of reporter-protein arrays\n",
      "mrna array probing\n",
      "reporter-protein array probing\n",
      "visualization of the arrays\n",
      "quantification of the arrays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00355/sb7b00355.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00288/sb8b00288.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4001307/sb4001307.xml\n",
      "methods\n",
      "dynamic modeling of microbial communities using d-optcom\n",
      "dynamic modeling in the absence of uptake kinetics\n",
      "results and discussion\n",
      "co-growth of auxotrophic e. coli mutant pairs\n",
      "dynamic analysis of subsurface uranium-reducing communities\n",
      "comparison of optcom and d-optcom predictions\n",
      "analysis of the impact of lactate vs acetate amendment on the biomass composition\n",
      "modeling the uranium reduction capability of the community\n",
      "summary and conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300097z/sb300097z.xml\n",
      "eric archer\n",
      "tal danino\n",
      "truong d. nguyen-huu\n",
      "robert penchovsky\n",
      "arthur prindle\n",
      "arun stephen rajkumar\n",
      "calvin schmidt\n",
      "david shis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00146/sb9b00146.xml\n",
      "results and discussion\n",
      "strain ratio stability\n",
      "spatial patterning\n",
      "communication\n",
      "concluding remarks\n",
      "methods\n",
      "construction of noncommunicating strains\n",
      "construction of sender and receiver strains\n",
      "data analysis and microfluidic device construction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00239/sb8b00239.xml\n",
      "results\n",
      "genes from the facultative anaerobe show better performance\n",
      "spatial segregation enhances 1,3-pdo production by heterocysts\n",
      "optimization of 1,3-pdo production in engineered anabaena pcc7120\n",
      "the extremely oxygen-sensitive enzyme is active in heterocysts\n",
      "tolerance of anabaena pcc7120 to the intermediate and end-product\n",
      "discussion\n",
      "conclusions\n",
      "methods\n",
      "chemicals, strains and culture conditions\n",
      "plasmid construction\n",
      "construction of engineered s. elongatus pcc7942\n",
      "construction of engineered anabaena pcc7120\n",
      "rt-qpcr\n",
      "analytical techniques\n",
      "data process and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00033/sb5b00033.xml\n",
      "methods\n",
      "molecular cloning\n",
      "expression\n",
      "in-gel fluorescence and western blotting\n",
      "cell sorting of libraries with facs\n",
      "high-throughput sequencing and analysis\n",
      "free energy measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00122/sb7b00122.xml\n",
      "results and discussion\n",
      "multiple synthetic amino acid sequence tags target fluorescent proteins for degradation by different proteases\n",
      "the main proteases of an e. coli cell can be overloaded when proteins are designed to be targeted to a common protease via engineered degradation tags\n",
      "the main proteases of e. coli can exhibit different levels of crosstalk depending on the degradation tags used\n",
      "single-cell data from microscopy slides support the high-throughput microplate data\n",
      "conclusion\n",
      "methods\n",
      "reagents\n",
      "strains and plasmids\n",
      "absorbance and fluorescence measurements with a microplate reader\n",
      "single-cell snapshots\n",
      "analysis of mean fluorescence\n",
      "model fitting to single tag microplate data\n",
      "model fitting to dual tag microplate data\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00037/sb5b00037.xml\n",
      "results and discussion\n",
      "encapsulation of alkaline phosphatase in mutated ms2 capsids\n",
      "capsid mutations impact kinetics of encapsulated phoa\n",
      "modeling suggests a role for phosphate transport in kinetic differences\n",
      "conclusions\n",
      "methods\n",
      "reagents\n",
      "high-performance liquid chromatography\n",
      "dynamic light scattering\n",
      "enzyme kinetics\n",
      "gel analyses\n",
      "uv–vis spectroscopy\n",
      "transmission electron microscopy\n",
      "generation of ms2 mutants\n",
      "expression and purification of ms2\n",
      "generation of phoa wt -neg and phoa d153g/d330n -neg\n",
      "expression and purification of phoa wt -neg and phoa d153g/d330n -neg\n",
      "fitc labeling of phoa wt -neg\n",
      "encapsulation of enzymes in ms2 mutants\n",
      "purification of encapsulated enzymes\n",
      "evaluation of stability of ms2-encapsulated phoa wt -neg\n",
      "enzyme kinetics\n",
      "experimental determination of inhibition constants ( k i ) for free enzymes\n",
      "nanoreactor kinetics modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5003604/sb5003604.xml\n",
      "omar akbari\n",
      "nichole daringer\n",
      "rachel m. dudek\n",
      "kei fujiwara\n",
      "amar ghodasara\n",
      "george a. khoury\n",
      "joshua leonard\n",
      "john m. marshall\n",
      "tushar patel\n",
      "kelly schwarz\n",
      "brynne stanton\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00185/sb8b00185.xml\n",
      "results and discussion\n",
      "development of a scalable hairpin construction protocol\n",
      "design of hairpin testing experiments\n",
      "characterizing the hairpin libraries\n",
      "examining hairpin stability and effects of distance from the target site to the orf\n",
      "targeting nuclear-localized transcripts\n",
      "conclusions\n",
      "materials and methods\n",
      "additional hairpin construction protocol details\n",
      "strain and construct information\n",
      "hairpin stem sequences\n",
      "library characterization\n",
      "distance from stop codon tests\n",
      "repression stability testing\n",
      "nuclear rnai experiments\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00264/sb7b00264.xml\n",
      "results and discussion\n",
      "enzyme and substrate preparation\n",
      "amino acid preferences for cyclization\n",
      "cyclization of sequences containing unusual amino acids\n",
      "core domain length requirement\n",
      "synthesis of naturally occurring cycloamanides\n",
      "cyclization of the phalloidin sequence with d -threonine\n",
      "cyclic peptide library production\n",
      "concluding remarks\n",
      "methods\n",
      "enzyme preparation\n",
      "dna constructs\n",
      "preparation of popb substrates\n",
      "cyclization assays and lc–ms\n",
      "library preparation and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5002505/sb5002505.xml\n",
      "results and discussion\n",
      "phylogenetic analysis of aaahs\n",
      "bioprospection and reconstitution of prokaryotic p4hs in e. coli\n",
      "modification of the xcp4h substrate preference through protein engineering\n",
      "de novo microbial synthesis of 5-htp via metabolic engineering\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "experimental materials\n",
      "dna manipulation\n",
      "construction of the phylogenetic tree and homology modeling\n",
      "in vivo assays of wild-type and mutant p4hs\n",
      "bioconversion of tryptophan to 5-htp\n",
      "de novo production of tryptophan and 5-htp\n",
      "hplc analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00394/sb7b00394.xml\n",
      "results and discussion\n",
      "control over tcr activity\n",
      "dynamics of zap70 switch\n",
      "modulation of tcr activity\n",
      "zap70 switch and cars\n",
      "methods\n",
      "plasmids\n",
      "cell culture\n",
      "lentivirus generation and transduction\n",
      "surface and intracellular staining\n",
      "flow cytometry\n",
      "il-2 elisa\n",
      "calcium dynamics\n",
      "reactivation of zap70 switch\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00278/sb6b00278.xml\n",
      "sbolme\n",
      "types of metabolic parts\n",
      "chemical compounds\n",
      "ec numbers\n",
      "metabolic proteins\n",
      "metabolic reactions\n",
      "metabolic parts exploration\n",
      "pathway design using sbolme parts\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00227/sb8b00227.xml\n",
      "results\n",
      "development of a multivariate model in e. coli dh10b\n",
      "validation of the model using chromosomal gene targets\n",
      "application of the model to control a toxin-antitoxin system\n",
      "expanding the model to different organisms\n",
      "discussion\n",
      "conclusion\n",
      "methods\n",
      "strains and culture media\n",
      "asrna design and plasmid construction\n",
      "chromosomal rfp or gfpmut3 reporter strain construction\n",
      "fluorescence and absorbance measurements\n",
      "quantification of β-glucuronidase\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00400/sb8b00400.xml\n",
      "results\n",
      "ucr mutations increase orthorep expression\n",
      "genetically encoded poly­(a) tails boost expression of genes encoded in orthorep\n",
      "promoters and poly­(a) tails can be combined to yield a panel of genetic constructs for customized and high expression levels from orthorep\n",
      "discussion and conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400055h/sb400055h.xml\n",
      "results and discussion\n",
      "engineering and characterization of randomized cmy circuits\n",
      "evolution experiments on randomized cmy circuits\n",
      "competitive fitness between the ancestor and mutants in an evolved population\n",
      "overall conclusions and future directions\n",
      "methods\n",
      "parts, vectors, and plasmid engineering\n",
      "genetic circuit characterization\n",
      "cmy color visualization and inducer experiments\n",
      "evolution experiments\n",
      "notes on the 24 h growth cycle\n",
      "visualizing cell color in evolved populations and clones\n",
      "sequencing plasmids in evolved clones\n",
      "direct competition experiments to measure fitness\n",
      "strain availability\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300094q/sb300094q.xml\n",
      "metabolic engineering\n",
      "synthetic biology\n",
      "biobricks and synthetic cells\n",
      "marveling nonlinear system dynamics\n",
      "two paradigms: unit operations versus electronic circuits\n",
      "epilogue\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb4000096/sb4000096.xml\n",
      "results and discussion\n",
      "considerations in design of modular expression platforms\n",
      "chimeric riboswitches regulate transcription in vitro\n",
      "synthetic aptamers can control the regulatory switch\n",
      "chimeric riboswitches function in vivo\n",
      "optimizing riboswitch performance by altering p1 helix stability\n",
      "conclusion\n",
      "methods\n",
      "synthesis of dna templates for transcription\n",
      "in vitro transcription assays\n",
      "in vivo assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00137/sb6b00137.xml\n",
      "alex campbell\n",
      "sara cleto\n",
      "christopher ede\n",
      "philip gach\n",
      "nicholas gold\n",
      "ting gong\n",
      "björn johansson\n",
      "filipa pereira\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00293/sb9b00293.xml\n",
      "results and discussion\n",
      "efficiency of crrna-guided genome cleavage with the cas9 rnp complex\n",
      "native integration efficiency of the engineered group ii intron in pacdura3_635slacz without counterselection\n",
      "delivery of a group ii intron in combination with the crispr/cas9 counterselection method\n",
      "delivery of a group ii intron alone or in combination with crispr/cas9 counterselection in e. coli bl21/de3 δ reca\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and media\n",
      "plasmid construction\n",
      "interference assay of selected spacers\n",
      "insertion assay\n",
      "counterselection assay\n",
      "analysis of integration by colony pcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400135t/sb400135t.xml\n",
      "results and discussion\n",
      "graph construction\n",
      "partitioning and decomposition\n",
      "matching and covering\n",
      "test cases\n",
      "discussion\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "methods\n",
      "matching\n",
      "covering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400182x/sb400182x.xml\n",
      "results and discussion\n",
      "engineering a potent chimeric nirw light-activated dgc\n",
      "improving the photodynamic range of a light-activated synthetic c-di-gmp module\n",
      "kinetics of the nirw light-induced c-di-gmp accumulation in e. coli\n",
      "nirw light-activated, c-di-gmp-mediated gene expression\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid and strain construction\n",
      "protein overexpression, purification, and spectroscopy\n",
      "motility assays\n",
      "enzymatic assays\n",
      "intracellular c-di-gmp measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00131/sb5b00131.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00423/sb7b00423.xml\n",
      "results\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400197f/sb400197f.xml\n",
      "results and discussion\n",
      "preparation of recombinant proteins for chimeric enzyme complex components\n",
      "kinetic parameters of two chimeric enzymes, acs-ctdoc and aco-acdoc\n",
      "binding analysis within the chimeric enzyme complex\n",
      "coexpression of the three component proteins of the chimeric enzyme complex in cyanobacteria\n",
      "ethylene production in cyanobacteria expressing a chimeric enzyme complex\n",
      "enhanced ethylene production with or without forming a chimeric enzyme complex in cyanobacteria\n",
      "conclusion\n",
      "methods\n",
      "chemicals and reagents\n",
      "bacterial strains and culture conditions\n",
      "construction of plasmids\n",
      "production and purification of recombinant proteins\n",
      "gel filtration\n",
      "nondenaturing page\n",
      "enzyme activity assays\n",
      "gas chromatography\n",
      "cloning of acs-ctdoc, aco-acdoc , and cip2 genes into the puc303 plasmid\n",
      "transformation and selection of cyanobacteria\n",
      "detection of recombinant proteins in cyanobacteria (western blot analysis)\n",
      "in vivo assay of ethylene-forming activity by cyanobacteria\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00331/sb6b00331.xml\n",
      "key elements in genome integration\n",
      "selection methods for genome integration\n",
      "effects of different integration loci\n",
      "effects of different double-stranded break mechanisms\n",
      "effects of the inset size and the size of the genomic region to be replaced\n",
      "integration via homologous recombination\n",
      "mechanism of hr\n",
      "suppression of nhej\n",
      "extension of the homology arms\n",
      "design of donor dna\n",
      "use of a chemical hr enhancer\n",
      "tuning of the antibiotic concentrations\n",
      "integration via nonhomologous end joining\n",
      "mechanism of nhej\n",
      "nhej applications\n",
      "increasing precision and efficiency of nhej-mediated integration\n",
      "integration via microhomology-mediated end joining\n",
      "mechanism of mmej\n",
      "mmej applications\n",
      "increasing precision and efficiency of mmej-mediated integration\n",
      "integration via alternative mechanisms\n",
      "integration via single-stranded oligodeoxyribonucleotides (ssodns)\n",
      "integration via nuclease-assisted recombination\n",
      "choosing the right integration platform\n",
      "strain and pathway engineering through nuclease-assisted genome integration methods\n",
      "in vivo assembly and integration of large biochemical pathways\n",
      "multicopy integration of large pathways\n",
      "library integration and screening\n",
      "perspectives\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00003/sb9b00003.xml\n",
      "results and discussion\n",
      "selection of enzymes for the decarboxylation of 4-hydroxy-2-ketobutyrate\n",
      "selection of enzymes for the deamination of homoserine\n",
      "construction of strains for the direct production of 1,3-pdo from glucose\n",
      "protein engineering of pyruvate decarboxylase to increase 1,3-pdo production\n",
      "construction of fusion protein to increase 1,3-pdo production\n",
      "conclusion\n",
      "methods\n",
      "strains and plasmids\n",
      "plasmid construction\n",
      "protein expression and enzyme assay\n",
      "culture condition and analytical method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00034/sb9b00034.xml\n",
      "results and discussion\n",
      "shaking strategy for the formation of guv-based synthetic cells\n",
      "characterization of guvs formed by the shaking strategy\n",
      "confirmation of unilamellarity\n",
      "reconstitution and encapsulation\n",
      "conclusion\n",
      "methods\n",
      "suv formation\n",
      "guv formation\n",
      "confocal fluorescence microscopy\n",
      "frap measurements\n",
      "cryotem\n",
      "dye influx experiments\n",
      "zeta potential measurements\n",
      "reconstitution and encapsulation\n",
      "integrin reconstitution\n",
      "cholesterol-tagged dna\n",
      "pyranine encapsulation\n",
      "mrna encapsulation\n",
      "polystyrene beads encapsulation\n",
      "mitochondria encapsulation\n",
      "e. coli\n",
      "guvs encapsulation\n",
      "lipid tubulation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00269/sb6b00269.xml\n",
      "robert gnügge\n",
      "ángel goñi-moreno\n",
      "noushin hadadi\n",
      "jasmin hafner\n",
      "mustafa khammash\n",
      "tina lebar\n",
      "vincent libis\n",
      "sarah milbredt\n",
      "robert ohlendorf\n",
      "maja rennig\n",
      "torsten waldminghaus\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00104/sb6b00104.xml\n",
      "methods\n",
      "strains and growth conditions\n",
      "construction of a synthetic amylase\n",
      "amylase activity assays\n",
      "construction of a luciferase reporter\n",
      "luciferase activity assay\n",
      "western blot analysis\n",
      "immunoassay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500327c/sb500327c.xml\n",
      "rule-based design of synthetic transcription factors\n",
      "transcription activator-like effectors\n",
      "regulation of endogenous human gene expression\n",
      "optochemical translocation to control protein function\n",
      "development of an artificial calcium-dependent transcription factor\n",
      "combinatorial biosynthesis of cyclic lipopeptide antibiotics\n",
      "modular construction of a functional artificial epothilone polyketide pathway\n",
      "enhanced triterpene accumulation in panax ginseng hairy roots\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300123m/sb300123m.xml\n",
      "results and discussion\n",
      "methods\n",
      "insect genetics, rearing, and transformation\n",
      "amplification and cloning of pgdsx and transcripts\n",
      "plasmid construction\n",
      "laboratory experiments with insects\n",
      "diamondback moth longevity test\n",
      "diamondback moth mating tests\n",
      "molecular analysis of transformed strains\n",
      "statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00279/sb7b00279.xml\n",
      "results\n",
      "cloning of the m. florum genome in yeast\n",
      "impact of the additional bacterial genome on yeast growth\n",
      "transcriptome profiling of yeast harboring a bacterial genome\n",
      "transplantation of the m. florum genome into m. capricolum\n",
      "discussion\n",
      "methods\n",
      "strains and culture conditions\n",
      "molecular biology methods\n",
      "construction of pmfl-ach and the donor strain\n",
      "whole-genome cloning and engineering\n",
      "genome transplantation in mycoplasma capricolum\n",
      "dna-seq and mutation detection\n",
      "rna-seq and differential expression analysis\n",
      "growth analysis\n",
      "cell cycle analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5000115/sb5000115.xml\n",
      "results and discussion\n",
      "in vitro determination of termination efficiency\n",
      "20-l labscale bioreactor cultivations\n",
      "quantification of rna levels\n",
      "conclusion\n",
      "methods\n",
      "strains and plasmids\n",
      "in vitro transcription assay\n",
      "sample preparation\n",
      "transcription analysis and calculation of termination efficiency\n",
      "quantitation of rna levels\n",
      "rna isolation from fermentation samples\n",
      "nanostring ncounter analysis system\n",
      "medium, composition, cultivation, and sampling\n",
      "sampling for process parameter analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00472/sb8b00472.xml\n",
      "results and discussion\n",
      "deleting omv-related genes enabled β-carotene excretion\n",
      "further increase of β-carotene excretion by the combined deletion of tola , tolr , and nlpi genes\n",
      "modulating membrane synthesis pathways facilitated amvts-mediated β-carotene excretion\n",
      "further increase of β-carotene excretion by combined overexpression of membrane synthesis genes\n",
      "microscopic observation of e. coli cells that excrete β-carotene via amvts\n",
      "the secreted β-carotene was mainly associated with the excreted omvs\n",
      "amvts improved β-carotene production in the hyperproducer strain\n",
      "conclusions\n",
      "materials and methods\n",
      "strains, media, and culture conditions\n",
      "plasmid construction\n",
      "genome editing\n",
      "analysis of β-carotene production and secreted β-carotene content\n",
      "extraction of omvs from engineered β-carotene-producing e. coli strains\n",
      "proteins extraction and sample preparation\n",
      "observation of the cell morphology via transmission electron microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300061x/sb300061x.xml\n",
      "methods and results\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00377/sb7b00377.xml\n",
      "results and discussion\n",
      "methods\n",
      "culturing cyanobacteria\n",
      "strains\n",
      "exponential growth rate experiments\n",
      "microscopy\n",
      "transformation efficiency assay\n",
      "l -lactate production assay\n",
      "l -lactate quantification\n",
      "horizontal gene transfer assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00017/sb9b00017.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "strains and plasmids\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cultivation and expression conditions\n",
      "facs counter-selection of improved riboswitch variants\n",
      "plate-based screening protocol\n",
      "design of experiments\n",
      "standard least squares model fitting\n",
      "stress response promoter testing: phosphate starvation coupled riboswitch function assay\n",
      "hyperosmotic shock activation of p osmy assay\n",
      "hydrogen peroxide induction of p soxs assay\n",
      "p csta induction assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00331/sb8b00331.xml\n",
      "results\n",
      "eat1 is the primary pathway for ethyl, isoamyl, and phenylethyl acetate ester biosynthesis\n",
      "mitochondrial localization of eat1 is necessary for high ethyl acetate production\n",
      "tca cycle and electron transport chain (etc) genetic targets for modulating ethyl acetate precursors\n",
      "multiplexed crispri for transcriptional regulation of ethyl acetate biosynthesis\n",
      "discussion\n",
      "methods\n",
      "strains and culturing conditions\n",
      "molecular cloning and plasmids construction\n",
      "transformation of k. marxianus and s. cerevisiae\n",
      "strain construction\n",
      "eat1 protein sequence analysis\n",
      "headspace gas chromatography\n",
      "reverse transcription quantitative pcr (qrt-pcr)\n",
      "mitotracker staining\n",
      "fluorescence microscopy\n",
      "western blot analysis\n",
      "protein expression and purification\n",
      "enzyme kinetics\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00031/sb6b00031.xml\n",
      "methods\n",
      "bacterial strains and growth\n",
      "molecular biology\n",
      "ecoflex assembly conditions\n",
      "flow cytometry\n",
      "cell-free protein synthesis\n",
      "in vivo growth and fluorescence measurements\n",
      "protein production and purification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00404/sb7b00404.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "reagents and enzymes\n",
      "plasmid dna constructs\n",
      "cell culture and transfection\n",
      "aav packaging, purification, and infection\n",
      "flow cytometry\n",
      "fluorescence microscopy\n",
      "adjacent residue searching\n",
      "rna purification and quantitative pcr\n",
      "t7e1 dna cleavage assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00281/sb7b00281.xml\n",
      "introduction\n",
      "results and discussion\n",
      "design of a bivalent synthetic chromatin-based transcriptional regulator\n",
      "bivalency strengthens the avidity of the pc-fusion for h3k27me3\n",
      "bivalent pc 2 tf shows cooperative, on-target binding with solid phase target ligands\n",
      "bivalent pc 2 tf activates a target gene in a partially silenced state\n",
      "conclusions\n",
      "methods\n",
      "plasmid constructs for txtl and bacterial expression\n",
      "txtl: cell-free expression\n",
      "e. coli expression and purification of proteins\n",
      "enzyme-linked immunosorbent assays (elisas)\n",
      "peptide microspot arrays\n",
      "plasmid constructs for mammalian expression\n",
      "cell culture and transfection\n",
      "rt-qpcr\n",
      "western blots\n",
      "imaging and flow cytometry\n",
      "luciferase assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00074/sb8b00074.xml\n",
      "methods\n",
      "plasmid construction\n",
      "transcription of crrnas and purification of cas12a protein\n",
      "the cads method\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400121x/sb400121x.xml\n",
      "one-step cloning and chromosomal integration of dna\n",
      "fluorescent proteins and in vitro genetic organization for cell-free synthetic biology\n",
      "probing cell-free gene expression noise in femtoliter volumes\n",
      "self-assembly of end-to-end multiphage structures\n",
      "pcr amplification of 4′-thiodna\n",
      "randomized biobrick assembly\n",
      "evolutionary stability dynamics and competitive fitness of engineered e. coli\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00279/sb6b00279.xml\n",
      "results and discussion\n",
      "development of hybrid promoters that exhibit synergistic activation by two tfs\n",
      "evaluating multiplexed receptor signaling with hybrid promoters\n",
      "precise promoter characterization using model-guided single-cell analysis\n",
      "elucidating properties of multiplexed receptor performance using a mechanistic model\n",
      "promoter and receptor engineering strategies to improve and gate functionality\n",
      "conclusions\n",
      "materials and methods\n",
      "dna constructs\n",
      "cell culture and transfection\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00093/sb9b00093.xml\n",
      "results and discussion\n",
      "site-directed mutagenesis of wild-type promoter in arsr gene\n",
      "addition of arsr-binding site resulted in decrease of background noise\n",
      "dose-dependent response to arsenite\n",
      "dynamics and specificity of arsenite biosensor\n",
      "real sample analysis\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, growth conditions, and reagents\n",
      "construction of arsenite reporter plasmids\n",
      "fluorescence measurement and data analysis\n",
      "real sample analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500065c/sb500065c.xml\n",
      "jason boock\n",
      "sunil chandran\n",
      "stefan de kok\n",
      "charles haitjema\n",
      "jing liang\n",
      "nicholas andrew roehner\n",
      "leslie stanton\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00056/sb7b00056.xml\n",
      "results and discussion\n",
      "construction of the reporter expression system\n",
      "improvement of fluorescence intensity by exchanging the terminator\n",
      "redesign of operation according to promoter characteristics\n",
      "detection of the synthetic antagonist against the model gpcr according to the redesigned scheme\n",
      "introduction of short incubation with ligands for reducing background signal\n",
      "concluding remarks\n",
      "methods\n",
      "strains and media\n",
      "construction of plasmids\n",
      "gpcr ligands\n",
      "fluorescent reporter assay\n",
      "microscopic observation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300095m/sb300095m.xml\n",
      "methods\n",
      "an overview of the integrated sbrome framework\n",
      "graph representation and transformation (figure )\n",
      "module query\n",
      "circuit partitioning\n",
      "mutant search\n",
      "gene expression model\n",
      "complexity analysis\n",
      "results\n",
      "multiplexer circuit\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00022/sb7b00022.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "plasmid construction\n",
      "cell culture\n",
      "dish coating\n",
      "tirf translocation assay\n",
      "confocal laser scanning microscopy imaging\n",
      "a half-life evaluation for off kinetics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00451/sb7b00451.xml\n",
      "results\n",
      "engineering reciprocal translocations in drosophila\n",
      "predicted trade-offs associated with translocation-based gene drive\n",
      "discussion\n",
      "materials and methods\n",
      "construct assembly\n",
      "fly culture and strains\n",
      "embryo and adult viability determination\n",
      "population cage experiments\n",
      "theoretical framework\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500302y/sb500302y.xml\n",
      "results and discussion\n",
      "protease-alleviated spatial sequestration (pass) platform design\n",
      "inducible pass construct expression and cleavage\n",
      "subcellular localization of pass constructs\n",
      "pass-regulated repression of gene expression\n",
      "pass-regulated induction of gene expression\n",
      "conclusions\n",
      "methods\n",
      "media, growth conditions, and strains\n",
      "dna constructs\n",
      "microplate fluorescence assays\n",
      "spheroplast production\n",
      "fluorescence microscopy\n",
      "western blots\n",
      "flow cytometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400047f/sb400047f.xml\n",
      "methodologies for comparison and optimization of photosynthesis models\n",
      "reverse engineering validation in human cells\n",
      "rational design of protein coding gene libraries\n",
      "sbrome: a framework for automated biosystems design\n",
      "autobiocad: full biodesign automation of genetic circuits\n",
      "dynamic modeling within ibiosim\n",
      "discrete-time signal processing with dna\n",
      "par-par laboratory automation platform\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00007/sb6b00007.xml\n",
      "results and discussion\n",
      "introduction of heterologous pathway to produce 3-hp from glucose\n",
      "redesign of strain to enhance 3-hp production\n",
      "testing combinations of downstream enzymes from various bacteria and overexpression of succinate dehydrogenase to increase 3-hp production\n",
      "screening of candidate semialdehyde dehydrogenases to produce ma\n",
      "fed-batch cultures\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and plasmids\n",
      "cultivation conditions\n",
      "gene manipulations\n",
      "analytical procedures\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00442/sb7b00442.xml\n",
      "architecture of candidate negative feedback oscillators\n",
      "influence of time constants on the oscillatory regime\n",
      "examples\n",
      "goodwin oscillator\n",
      "a two-node oscillator\n",
      "conclusion and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00270/sb7b00270.xml\n",
      "recent strategies for improving specificity of the crispr/cas9 system\n",
      "comparing the methods to improve specificity requires standardization\n",
      "accuracy and sensitivity of the methods for determining off-target mutations\n",
      "data standardization and pooling to improve computational prediction of off-target sites\n",
      "targeting specificity of donor dna\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300033q/sb300033q.xml\n",
      "need for integrating sequence information with mathematical models\n",
      "shortcomings of existing methods\n",
      "a new approach\n",
      "“conceptual model” versus “model”\n",
      "analysis of conceptual models at present\n",
      "overview\n",
      "ontology\n",
      "relationship to existing ontologies\n",
      "an example where existing ontologies fail\n",
      "local parameters\n",
      "local parameters are a way of organizing a model\n",
      "parameter names are provided by the software tool\n",
      "hierarchical modeling\n",
      "automatic mapping to multiple models\n",
      "automatic mapping of parameters\n",
      "parent/child relationships\n",
      "implementation\n",
      "quorum sensing example\n",
      "mapping to two mathematical models of protein production\n",
      "representing modules\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00541/sb8b00541.xml\n",
      "results and discussion\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300018h/sb300018h.xml\n",
      "results\n",
      "overview of design and analysis\n",
      "system specification and reaction mechanisms\n",
      "sequence design\n",
      "reaction mechanism validation\n",
      "system-level characterization of bistability\n",
      "quantitative modeling and prediction\n",
      "a simple kinetic model for the self-activating switch\n",
      "conclusions\n",
      "materials and methods\n",
      "dna oligonucleotides and enzymes\n",
      "transcription\n",
      "data acquisition\n",
      "model simulation\n",
      "dna sequences\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00308/sb7b00308.xml\n",
      "serine integrase site-specific recombination\n",
      "genome engineering with serine integrases\n",
      "dna assembly by site-specific recombination\n",
      "serine integrase logic and memory devices\n",
      "prospects for serine integrases\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00175/sb9b00175.xml\n",
      "results and discussion\n",
      "testing photoactivatable split-cre systems\n",
      "testing cre-ert2 systems\n",
      "design and function of tampa-cre\n",
      "optimizing tamoxifen and blue light stimulation\n",
      "design and function of the car reporter\n",
      "tampa-cre drives car-mediated t cell activation\n",
      "summary and future directions\n",
      "materials and methods\n",
      "cre and car construct design\n",
      "cell culture and reagents\n",
      "lentivirus production\n",
      "lentiviral transduction in cell lines\n",
      "light stimulation\n",
      "hek 293t experiments\n",
      "jurkat experiments\n",
      "microscopy\n",
      "flow cytometry measurements\n",
      "data representation and statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00246/sb5b00246.xml\n",
      "background and motivation\n",
      "from the individual cell to patterns\n",
      "the perspective from artificial development\n",
      "morphogenetic synthetic biology\n",
      "results\n",
      "homeostatic core: morphobrick 1\n",
      "morphobrick-1 homeostatic development\n",
      "morphobrick-1 sensitivity analysis\n",
      "morphobrick-1 genomic variations: transition thresholds\n",
      "morphobrick-1 environmental variations: chemical rates\n",
      "morphobrick-1 genomic mutation: number of types\n",
      "limb growth: morphobrick 2\n",
      "morphobrick-2 sensitivity analysis\n",
      "morphobrick-2 genomic variations: transition thresholds\n",
      "morphobrick-2 environmental variations: chemical rates\n",
      "precursor cells and assembly: morphobrick 3\n",
      "morphobrick-3 segmentation\n",
      "morphobrick-3 precursor cell positioning\n",
      "class-i shapes: radially symmetric wheels\n",
      "morphobrick-3 precursor cell differentiation\n",
      "class-ii shapes: asymmetric wheels\n",
      "discussion\n",
      "robustness of leader singularity\n",
      "multiplicity of chemical species\n",
      "synchrony of bacterial population\n",
      "midplane feasibility\n",
      "future work\n",
      "toward higher-level classes\n",
      "polymorphism, polyphenism, and evolution\n",
      "methods\n",
      "model of bacterial behavior ontology\n",
      "genomic representation\n",
      "higher-level biological functions\n",
      "hierarchical composition\n",
      "chemical environment\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300041z/sb300041z.xml\n",
      "results and discussion\n",
      "detecting synthesis of tetrameric gus and gal in microcompartments\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "preparation method and surface properties of the microcompartments do not affect intracompartmental enzyme synthesis\n",
      "synthesis of tetrameric gus at lower dna concentration is accelerated in smaller microcompartments\n",
      "mechanism of the acceleration of tetrameric gus synthesis in smaller compartments\n",
      "mechanism underlying the lack of acceleration of tetrameric gal synthesis in smaller compartments\n",
      "qualitative evaluation of the acceleration mechanism\n",
      "gus and gal synthesis is a fourth and first-order reaction, respectively\n",
      "conclusion\n",
      "methods\n",
      "in vitro transcription and translation\n",
      "water-in-oil (w/o) emulsion droplet preparation\n",
      "microscopic observation\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00018/sb5b00018.xml\n",
      "the ant application\n",
      "how does ant work?\n",
      "parameters\n",
      "working with the gui\n",
      "working with the api\n",
      "algorithm for generating a degenerate codon\n",
      "algorithm for evaluating a codon\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00239/sb7b00239.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00096/sb5b00096.xml\n",
      "materials and methods\n",
      "oligonucleotides\n",
      "plasmids\n",
      "toxin-neutralization assays\n",
      "protein–protein interaction assays\n",
      "ubiquitination assays\n",
      "protein stability assays\n",
      "cell proliferation and apoptosis assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00540/sb8b00540.xml\n",
      "a new biology for a new century\n",
      "obstacles to an exponential increase in synthetic biology productivity\n",
      "machine learning’s predictive capabilities\n",
      "machine learning needs automation to be truly effective\n",
      "predictive synthetic biology will dramatically impact biology and inspire computer science\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00125/sb9b00125.xml\n",
      "results\n",
      "system description\n",
      "ode model\n",
      "steady-state analysis\n",
      "performance bounds\n",
      "closed loop stability\n",
      "dynamic simulation\n",
      "stochastic simulation\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00250/sb5b00250.xml\n",
      "results and discussion\n",
      "ypk strategy\n",
      "promoter activity measurement\n",
      "construction of a four-gene d -xylose metabolic pathway in s. cerevisiae\n",
      "construction of an eight-gene d -xylose metabolic pathway\n",
      "random assembly of d -xylose-utilizing pathways\n",
      "ypkpathway software and executable documentation\n",
      "comparison between ypk and existing methods\n",
      "final remarks\n",
      "methods\n",
      "strains, plasmids, and cell cultivation\n",
      "construction of the pypka vector\n",
      "cloning of genetic elements in pypka\n",
      "construction of the pypk0 e. coli / s. cerevisiae shuttle vector and its derivatives\n",
      "construction of single-gene expression vectors\n",
      "rational pathway design\n",
      "physiological characterization\n",
      "ypkpathway software\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500062p/sb500062p.xml\n",
      "results and discussion\n",
      "in vivo selection method\n",
      "regulation of gal4 expression with 3′-utr hammerhead ribozymes\n",
      "in vivo selection of theophylline-dependent type 1 hhrs\n",
      "development of neomycin-dependent hammerhead ribozymes\n",
      "methods\n",
      "general considerations\n",
      "plasmid construction\n",
      "construction and screening of aptazyme libraries\n",
      "reporter gene assays in s. cerevisiae\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300087n/sb300087n.xml\n",
      "methodology\n",
      "synchronous framework\n",
      "asynchronous framework\n",
      "clock\n",
      "delay elements\n",
      "synchronous delay elements\n",
      "asynchronous delay elements\n",
      "computations\n",
      "biquad filter\n",
      "synchronous biquad filter\n",
      "asynchronous biquad filter\n",
      "8-point rfft\n",
      "simulation results\n",
      "mapping to dna strand displacement\n",
      "simulation results\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00251/sb8b00251.xml\n",
      "results\n",
      "standard and content of the chlamydomonas moclo kit\n",
      "constitutive promoters and reporter genes\n",
      "control of gene expression\n",
      "multiple fusion tags for detection and purification of gene products\n",
      "visualization and targeting of proteins in living cells\n",
      "discussion\n",
      "methods\n",
      "escherichia coli and chlamydomonas reinhardtii strains, transformation, and growth conditions\n",
      "design\n",
      "parts repository\n",
      "parts cloning\n",
      "moclo assembly conditions\n",
      "quality control of generated dnas\n",
      "nanoluc activity determination\n",
      "absorbance measurement of cultures growing in microtiter plates\n",
      "rna extraction and mirna-mediated cleavage mapping\n",
      "immuno-blotting\n",
      "microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00070/sb8b00070.xml\n",
      "results\n",
      "phylogenetic analysis of the 4-hba polyprenyltransferase ximb\n",
      "catalytic properties of ximb\n",
      "enhancing gba formation by increasing gpp supply in e. coli\n",
      "engineering chorismate pathway to increase 4-hba production\n",
      "targeted production of prenylated 4-hba derivatives in vivo\n",
      "feeding with 4-hba derivatives for the production of gba analogues\n",
      "discussion\n",
      "methods\n",
      "strains and plasmids\n",
      "culture conditions\n",
      "chemicals\n",
      "phylogenetic analysis and multiple sequence alignment\n",
      "in vitro assay of ximb\n",
      "western blot detection of ximb\n",
      "plasmid construction\n",
      "extraction of cell cultures and hplc analysis\n",
      "uplc-qtof-ms analysis\n",
      "nmr analysis\n",
      "isolation of gba derivatives\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00356/sb7b00356.xml\n",
      "results and discussion\n",
      "design strategy for temperature-responsive transcription terminators\n",
      "the ibpa terminator is temperature-responsive and regulates transcription in vivo\n",
      "comparative analysis of single point mutations on rna thermometer and transcription terminator activity\n",
      "the ibpa thermoterm is functional in different genetic contexts\n",
      "the ibpa hairpin unfolds after a temperature upshift in vitro\n",
      "methods\n",
      "bacterial growth conditions\n",
      "plasmid construction\n",
      "β-galactosidase activity assay\n",
      "rna preparation\n",
      "northern blot analysis\n",
      "preparation of protein extracts and western blot analysis\n",
      "enzymatic rna structure probing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00388/sb6b00388.xml\n",
      "results and discussion\n",
      "introduction of crispr-cas9 into l. lactis mg1363\n",
      "efficiency of pnzcas9 and pl2cas9 protection against phage infection\n",
      "sequence analysis of escaping phages\n",
      "genome engineering of the virulent phage p2\n",
      "deletion\n",
      "point mutation\n",
      "insertion\n",
      "targeting other phage p2 genes\n",
      "sequential mutations\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains, phages, and growth conditions\n",
      "reagents and enzymes\n",
      "dna sequencing and analysis\n",
      "construction of pnzcas9\n",
      "construction of pl2cas9\n",
      "new spacer cloning in pnzcas9 and pl2cas9\n",
      "construction of homologous recombination templates\n",
      "analysis of recombinant phages\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00322/sb6b00322.xml\n",
      "results and discussion\n",
      "construction of a repressor gate operating by not and nor logic\n",
      "transient characterization of the neotc gate\n",
      "performance characterization\n",
      "hierarchical stochastic models\n",
      "in silico completion of device characterization\n",
      "computer-aided redesign\n",
      "materials and methods\n",
      "algorithms for calibration and in silico characterization\n",
      "cloning\n",
      "p adh1 -driven device\n",
      "p gal1 -driven device\n",
      "genome manipulations\n",
      "cultivation of yeast\n",
      "cytometry\n",
      "time-lapse microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00295/sb8b00295.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00209/sb7b00209.xml\n",
      "results\n",
      "construction of plasmids for the iis-alphoid teto -hac system\n",
      "description of the iis-alphoid teto -hac system\n",
      "proof of site-specific recombination using the iis-alphoid teto -hac system\n",
      "vhl gene reconstruction using iis-alphoid teto -hac system\n",
      "round 1\n",
      "round 2\n",
      "round 3\n",
      "discussion\n",
      "methods\n",
      "cell lines and culture\n",
      "construction of plasmids for the iis-alphoid teto -hac system\n",
      "insertion of the platform cassette a037 into the alphoid teto -hac in chicken dt40 cells\n",
      "one platform cassette per alphoid teto -hac, per cell\n",
      "microcell-mediated chromosome transfer (mmct) technique\n",
      "fluorescence in situ hybridization (fish)\n",
      "plasmid dna transfection and loading into alphoid teto -hac\n",
      "pcr reaction\n",
      "construction of vhl gene fragment-containing plasmids and loading into alphoid teto -hac\n",
      "rt-pcr reaction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00226/sb8b00226.xml\n",
      "results and discussion\n",
      "selection of an improved secretion signal\n",
      "lp ibp fusions and efficient recombinant protein secretion\n",
      "glycosylation, protein structure, and efficient export\n",
      "fusion protein design and cultivation for secreted rp production\n",
      "protein extraction from c. reinhardtii culture mediated by atps\n",
      "methods\n",
      "dna assembly, construct design, and cloning\n",
      "c. reinhardtii cultivation, transformation and mutant screening\n",
      "protein extraction and sds-page\n",
      "detergent-based aqueous two-phase system (atps)\n",
      "wound healing assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00248/sb9b00248.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results and discussion\n",
      "in vitro packaging mediated approach\n",
      "targeted cloning of natural product pathways\n",
      "in vitro editing for heterologous expression of pathways\n",
      "conclusion\n",
      "materials and methods\n",
      "gdna isolation\n",
      "preparation of cas9 protein and sgrnas\n",
      "protocols of in vitro packaging mediated approach\n",
      "in vitro crispr/cas9-mediated editing of stu , heterologous expression, and detection\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00138/sb5b00138.xml\n",
      "materials and methods\n",
      "chemicals\n",
      "molecular phylogenetic analysis by maximum likelihood\n",
      "expression and purification of sats\n",
      "enzymatic activity and the kinetic parameter assays\n",
      "stereopurity determination of β-valienamine by hplc with precolumn derivatization using opa\n",
      "β-valienamine purification from the reaction mixture and the fermentation broth of δ valc :: btrr\n",
      "high resolution-mass spectrometry (hr-ms) and nuclear magnetic resonance spectrometry (nmr)\n",
      "construction of δ valc :: btrr mutant\n",
      "fermentation of s. hygroscopicus 5008 and its mutants\n",
      "validamycin a analysis by hplc\n",
      "valienone analysis by hplc with dnph precolumn derivatization\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00062/sb9b00062.xml\n",
      "materials and methods\n",
      "cell strains and plasmid constructions\n",
      "expression and labeling of surface displayed sfgfp and tevp\n",
      "trypsin protease digestion of surface-exposed sfgfp\n",
      "heat release of surface displayed sfgfp and tevp\n",
      "biochemical release of surface displayed sfgfp with tevp and tevp activity in different growth media\n",
      "protein precipitation and western blotting\n",
      "molecular modeling of tevp-ag43 160n fusion\n",
      "coexpression of ag43 160n sfgfp and tevp and release kinetics\n",
      "sequential secretion of mcherry and sfgfp\n",
      "statistical analysis\n",
      "results and discussion\n",
      "cell surface display of sfgfp via ag43 autotransporter protein\n",
      "release of surface displayed proteins\n",
      "cleavage of displayed protein with tevp in different growth media\n",
      "codisplay of tevp and sfgfp on the cell surface for controlled release\n",
      "sequential secretion of multiple proteins\n",
      "conclusion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00149/sb9b00149.xml\n",
      "materials and methods\n",
      "bacterial strains, plasmids, media, and growth conditions\n",
      "molecular biology\n",
      "standard techniques for molecular cloning\n",
      "error-prone pcr and ammonia lyase library construction\n",
      "chromosomal deletions\n",
      "fluorescence activated cell sorting (facs)\n",
      "protein purification and enzyme assays\n",
      "chemical analyses\n",
      "bioinformatic methods\n",
      "results\n",
      "construction and characterization of the trans -cinnamic acid biosensor psenca\n",
      "optimization of heterologous gene expression enables ca production and biosensor-mediated product detection in e. coli\n",
      "prevention of biosensor cross-talk is a prerequisite for any facs applications using psenca\n",
      "directed evolution of an ammonia lyase by multistep facs screening\n",
      "discussion\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00102/sb7b00102.xml\n",
      "results and discussion\n",
      "construction of the tunable mlaci expression system\n",
      "performance of the mlaci titratable system in conjunction with commonly used expression vectors\n",
      "cell-to-cell homogeneous gene expression mediated by the mlaci titratable expression\n",
      "comparison with another expression system based on the modulation of t7 rnap activity\n",
      "enhanced production of membrane proteins\n",
      "overcoming transformation toxicity by the mlaci expression system\n",
      "conclusions\n",
      "methods\n",
      "strains, plasmids, and culture conditions\n",
      "construction of tunable mlaci expression plasmids\n",
      "cloning of membrane proteins\n",
      "gfp fluorescence detection and flow cytometry analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00327/sb6b00327.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00058/sb9b00058.xml\n",
      "results and discussion\n",
      "overview of synchronizer\n",
      "development of magnetic-capped bacteria\n",
      "device design\n",
      "microfluidic synchronizer performance\n",
      "synchrony validation\n",
      "concluding remarks\n",
      "methods\n",
      "plasmid constructs\n",
      "outer membrane protein isolation and western blot\n",
      "preparation of biotinylated anti-gfp antibody attached bacteria\n",
      "immobilization of mfns and optimization\n",
      "fabrication\n",
      "device operation\n",
      "cell collection and count\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00236/sb8b00236.xml\n",
      "results\n",
      "spacer variations result in nonresponsive smad3 reporters\n",
      "spacer composition predicts sensitivity\n",
      "binding of p53 predicts nonresponsive reporters\n",
      "multivariate tf activity identifies central connections between factors\n",
      "genetic reporters identify dynamic emt signature\n",
      "discussion\n",
      "methods\n",
      "cells and reagents\n",
      "reporter construction\n",
      "lentivirus\n",
      "reporter arrays\n",
      "transcription factor binding analysis\n",
      "classification analysis\n",
      "network analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00228/sb5b00228.xml\n",
      "results\n",
      "optimization of xylose utilization (xu) module\n",
      "optimization of xylose transport (xt) module\n",
      "optimization of xylan degradation module\n",
      "integration of all optimized modules for xylan utilization as the sole carbon source\n",
      "demonstration of a consolidated bioprocess using hemicellulose as feedstock\n",
      "discussion\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains and plasmid manipulation\n",
      "media and cultivation conditions\n",
      "protein preparation and analysis\n",
      "sugar concentration analysis\n",
      "analysis of lysine in the culture medium\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00138/sb7b00138.xml\n",
      "results\n",
      "predictions of nucleosome occupancy correlate to measured terminator strength\n",
      "terminators designed for decreased nucleosome affinity have increased protein production\n",
      "designed terminators display decreased nucleosome occupancy\n",
      "designed terminators exhibit higher mrna levels and improved termination efficiency\n",
      "strong terminators can be designed from randomized terminator sequences\n",
      "native terminators can also be improved by optimizing for nucleosome depletion\n",
      "discussion\n",
      "methods\n",
      "strains and media\n",
      "plasmid construction\n",
      "computational methods\n",
      "flow cytometry\n",
      "quantitative pcr and mrna half-life measurements\n",
      "nucleosome mapping\n",
      "itaconic acid quantification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00222/sb7b00222.xml\n",
      "methods\n",
      "design of the network motif\n",
      "modeling and analysis approach\n",
      "results\n",
      "a transcriptional network with competitive push signals\n",
      "baseline and amplitude of the input signal reduce the bistability region of the core toggle switch and reduce the period-doubling regime\n",
      "period-doubling occurs with transient loss of bistability\n",
      "a transcriptional network with noncompetitive push signals\n",
      "the period-doubling region is robust with respect to input amplitude, period, and baseline\n",
      "a realization based on monomeric regulators\n",
      "input amplitude and baseline alter the bistable regime\n",
      "discussion and conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00448/sb8b00448.xml\n",
      "results and discussion\n",
      "design and construction of caulobacter crescentus s-layer variants for surface display\n",
      "engineered s-layers specifically display proteins ligated to the cell surface\n",
      "density of attached materials is controlled by insertion location\n",
      "arraying hard and soft materials on the surface of engineered cells\n",
      "nanoparticle attachment does not affect cell viability\n",
      "advancement of rsaa s-layer as a platform for controlled material assembly\n",
      "engineered c. crescentus opens new possibilities for hierarchical assembly of hybrid living materials\n",
      "conclusions\n",
      "methods\n",
      "strains\n",
      "plasmid construction\n",
      "genome engineering of c. crescentus\n",
      "monitoring ligation of spycatcher-fusions to c. crescentus\n",
      "cell viability assay\n",
      "in situ atomic force microscope (afm) imaging\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00247/sb7b00247.xml\n",
      "results and discussion\n",
      "establishment and characterization of the growth-based selection platform\n",
      "evaluation of the anti-escape ability of circuits based on carbon source utilization and antibiotic resistance\n",
      "application of the selection platform to optimize the l -trp biosynthetic pathway\n",
      "application of the selection platform for mutational breeding of l -trp high producers at the level of the genome\n",
      "methods\n",
      "strains and culture conditions\n",
      "media\n",
      "plasmid construction\n",
      "construction of pmc, ptet, and pmc2\n",
      "construction of pedwt and pedfbr\n",
      "construction of the plasmid library of ped-s40ssm\n",
      "preparation of the error-prone trpe library, ped-library\n",
      "characterization of pmc\n",
      "growth selection for pmc\n",
      "generation of mutational library by dnaq926\n",
      "generation of mutational library by artp\n",
      "characterization and evolution process for the ptet circuit\n",
      "quantification of extracellular l -trp and maltose\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00352/sb6b00352.xml\n",
      "combinatorial pairwise assembly efficiently generates high affinity binders\n",
      "design and assembly of dna sequence libraries for chromosomal insertion in bacteria\n",
      "protein synthesis in coupled and uncoupled cell-free prokaryotic gene expression systems\n",
      "biosynthesis of the high-value plant secondary product benzyl isothiocyanate\n",
      "the geobacillus plasmid set: a modular toolkit for thermophile engineering\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00086/sb8b00086.xml\n",
      "methods\n",
      "construction of plasmids and transformation\n",
      "media and cultivations\n",
      "individual protein expression and purification\n",
      "enzymatic activity assay\n",
      "metabolites and glucose quantification\n",
      "dynamic light scattering assay\n",
      "scanning electron, field-emission scanning electron and atomic force microscopy\n",
      "fluorescence complementation assay\n",
      "crispr-cas9 mediated genome editing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400102w/sb400102w.xml\n",
      "results and discussion\n",
      "methods\n",
      "enzymes, chemicals, and strain\n",
      "plasmid construction\n",
      "assay protocol\n",
      "fluorescence measurements\n",
      "numerics\n",
      "determination of the region of the single stable stationary point achieved by gene overexpression\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300130z/sb300130z.xml\n",
      "yidan ding\n",
      "joshua t. kittleson\n",
      "dante w. romanini\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00223/sb8b00223.xml\n",
      "results\n",
      "design of the spy-sup polymer network\n",
      "gelation and working conditions of the spy-sup polymer\n",
      "uranyl adsorption efficiency of the spy–sup network\n",
      "retrieval of the spy–msa polymer network\n",
      "uranyl retrieval by a bifunctional spy–sup–msa network\n",
      "bacterial autosecretion of components for the spy–sup network\n",
      "discussion\n",
      "material and methods\n",
      "cloning and protein purification\n",
      "temperature and ph gradient assays\n",
      "uranyl binding assay\n",
      "biotinylated magnetic beads preparation\n",
      "msa retrieval efficiency assay\n",
      "uranyl retrieval efficiency by the bifunctional spy–sup–msa polymer\n",
      "isolation of cellular fractions of the secreted proteins\n",
      "western blot for protein secretion assay\n",
      "modeling and calculation of gel point cld and molecular weight distribution\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400079u/sb400079u.xml\n",
      "in vivo approaches for chromosome fragmentation\n",
      "results and discussion\n",
      "generation of strains harboring different combinations of genome-engineering elements\n",
      "use of a heterologous origin of replication\n",
      "phenotypes of strain 22 harboring a teln gene\n",
      "methods\n",
      "strain engineering\n",
      "southern blot\n",
      "pulse-field gel electrophoresis\n",
      "grow curves and microscopy\n",
      "other reagents\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300133r/sb300133r.xml\n",
      "expanding an alkane biosynthetic pathway\n",
      "screens and selections for small molecule biosynthesis\n",
      "a bacterial biosynthetic pathway tool\n",
      "functional display of complex cellulosomes on yeast\n",
      "scaffold diversification of a hyperthermophilic protein superlibrary\n",
      "novel regulatory elements in chlamydomonas chloroplast mrna\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500350t/sb500350t.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00108/sb6b00108.xml\n",
      "the wet lab accelerator application\n",
      "autoprotocol and omniprotocol\n",
      "future work\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00135/sb5b00135.xml\n",
      "results and discussion\n",
      "experimental testbed for the assessment of control strategies\n",
      "controlling gene expression from the gal1 promoter: set-point and tracking control tasks\n",
      "control algorithms\n",
      "set-point control experiments\n",
      "signal tracking control experiments\n",
      "conclusions\n",
      "methods\n",
      "yeast strain and cell culture\n",
      "microfluidics\n",
      "microscopy and image analysis\n",
      "control strategies implementation\n",
      "pi\n",
      "mpc\n",
      "zad\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00119/sb7b00119.xml\n",
      "results and discussion\n",
      "resource-coupled circuit interference\n",
      "effect of the circuit topology on the coupling correlation\n",
      "negative feedback loops can reduce resource-associated coupling in genetic circuits\n",
      "methods\n",
      "strains and growth media\n",
      "plasmid design and circuit construction\n",
      "fluorescence measurements\n",
      "conversion of rfp and gfp arbitrary units into relative expression units (reus)\n",
      "computational modeling\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00068/sb5b00068.xml\n",
      "results and discussion\n",
      "methods\n",
      "efm calculator implementation\n",
      "simple sequence repeat (ssr) predictions\n",
      "repeat-mediated deletion (rmd) predictions\n",
      "relative instability prediction (rip) score\n",
      "igem registry analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00319/sb6b00319.xml\n",
      "results and discussion\n",
      "design of the artificial gene operon assembly system (agos)\n",
      "reconstruction of the novobiocin biosynthetic gene cluster by agos\n",
      "proof-of-concept\n",
      "testing the flexibility of agos: omitting terminator sequences\n",
      "conclusion\n",
      "material and methods\n",
      "bacterial strains, culture conditions, plasmids, and cosmids\n",
      "construction of the entry plasmids and destination vector\n",
      "generation of artificial gene operons\n",
      "assembly of the novobiocin gene cluster\n",
      "generation of mrslw24\n",
      "heterologous expression\n",
      "lc–ms and lc–ms/ms\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400171j/sb400171j.xml\n",
      "analysis of the fluvirucin b1 polyketide synthase\n",
      "discovery of cryptic polyketide metabolites from dermatophytes\n",
      "antimicrobial peptides targeting gram-negative pathogens\n",
      "refactoring silent biosynthetic gene clusters\n",
      "a facile method for site-specific gene integration\n",
      "multiplex iterative plasmid engineering\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00248/sb5b00248.xml\n",
      "gpcr-based chemical biosensors for medium-chain fatty acids\n",
      "synthetic auxotrophs for a bl21(de3) biosafety strain\n",
      "tn7-based device for calibrated heterologous gene expression\n",
      "development of a synthetic malonyl-coa sensor in yeast\n",
      "evolutionary design of choline-inducible and -repressible t7-based induction systems\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00198/sb8b00198.xml\n",
      "methods\n",
      "strains, chemicals, and genes\n",
      "construction of a curcuminoids-producing strain for mage\n",
      "design of the mage oligomers and mage experiments\n",
      "screening of curcumin-overproducing variants\n",
      "quantification of curcuminoid production in the selected variants\n",
      "calculation of the translation efficiency of each gene in the parents strain and mutants\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00010/sb6b00010.xml\n",
      "results and discussion\n",
      "methods\n",
      "dna constructs\n",
      "in vitro transcription reaction\n",
      "cell lysate (25 mg ml –1 ) preparation\n",
      "in vitro transcription/translation reaction\n",
      "in vitro translation reaction\n",
      "mrna production\n",
      "uncoupling\n",
      "ribosome extraction\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300001c/sb300001c.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300053s/sb300053s.xml\n",
      "methods\n",
      "computational design of oligonucleotide-sensing allosteric ribozymes\n",
      "preparations of nucleic acids\n",
      "allosteric ribozyme assays\n",
      "results and discussion\n",
      "design and experimental validation of rna length sensing ribozymes\n",
      "ribozyme detection of rnas with triplet repeats observed in opmd disorder\n",
      "integrated three-input molecular and gate\n",
      "integrated two-input molecular multiplexer\n",
      "molecular multiplexer–decoder circuit\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00240/sb5b00240.xml\n",
      "results and discussion\n",
      "the basic principle and feature of the recode method\n",
      "optimization of the constructed recode reaction system for multiple site-directed mutagenesis\n",
      "application of recode to rapidly construct a diverse mutant library of regulatory elements\n",
      "application of recode to rapidly broaden the substrate specificity of the leech hyaluronidase\n",
      "combinatorial engineering of regulatory elements and enzymes to optimize synthetic pathway\n",
      "discussion\n",
      "conclusion\n",
      "materials and methods\n",
      "bacterial strains and plasmids\n",
      "media and culture conditions\n",
      "primer design\n",
      "recode manipulation\n",
      "variants library construction and screening\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb200005w/sb200005w.xml\n",
      "methods\n",
      "protein and rna preparation\n",
      "surface plasmon resonance\n",
      "cell culture\n",
      "fluorescence-activated cell sorting (facs)\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00120/sb6b00120.xml\n",
      "methods\n",
      "materials\n",
      "cell culture\n",
      "creation of cell lines that stably express hyper, sypher, and daao\n",
      "intracellular generation of h 2 o 2 using d -amino acid oxidase\n",
      "quantification of hyper and sypher signals using fluorescence microscopy\n",
      "converting hyper’s ratiometric fold-change to intracellular concentration of h 2 o 2\n",
      "annexin v/pi apoptosis assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00048/sb7b00048.xml\n",
      "results and discussion\n",
      "selection of n ε -acetyl- l -lysine for amber suppression\n",
      "development of a luminescence-based amber suppression reporter\n",
      "adjustment of the amber codon context permits modulation of protein output\n",
      "a diproline amber context allows tight regulation of β-galactosidase protein levels\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chromosomal integration of ackrs enhances regulated amber suppression\n",
      "four broad-host-range cloning vectors allow amber suppression in diverse bacterial species\n",
      "concluding remarks\n",
      "materials and methods\n",
      "bacterial stains, oligonucleotides and plasmids\n",
      "molecular biological methods\n",
      "plasmid and strain construction\n",
      "lux-based luminescence activity assay\n",
      "nanoluc luciferase assay in s. enterica and v. cholerae\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00261/sb5b00261.xml\n",
      "methods\n",
      "plasmid construction\n",
      "ai-2 assay\n",
      "protein expression assays\n",
      "rna isolation and analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00115/sb7b00115.xml\n",
      "results and discussion\n",
      "conclusion\n",
      "methods\n",
      "synthesis of unnatural triphosphates\n",
      "steady-state kinetics assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00138/sb8b00138.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00487/sb8b00487.xml\n",
      "results and discussion\n",
      "materials and methods\n",
      "media, plasmids, strains\n",
      "identification of riboswitches in thermophilic bacteria\n",
      "riboswitch activity assays in geobacillus thermoglucosidasius\n",
      "clostridium thermocellum growth\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300007g/sb300007g.xml\n",
      "artificial cellular architecture\n",
      "understanding ultrasensitivity\n",
      "degrading hydrocarbons\n",
      "highly modified proteins for multienzyme catalysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00245/sb6b00245.xml\n",
      "results\n",
      "freeing agg codon for uaa incorporation\n",
      "reassigning agg codon to azf\n",
      "reassignment of agg codon to bodipy-fl uaa derivative\n",
      "combination of agg and amber -codon reassignment for dual protein labeling\n",
      "discussion\n",
      "materials and methods\n",
      "materials\n",
      "trna production\n",
      "total trna preparation\n",
      "specific trna depletion in total trna mixture\n",
      "trna aminoacylation and fluorophore conjugation\n",
      "in vitro protein translation assay\n",
      "protein labeling\n",
      "smfret measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00011/sb7b00011.xml\n",
      "potential of clustered regularly interspaced short palindromic repeats (crispr) as next generation gene therapy\n",
      "in vivo gene editing enabled by crispr\n",
      "challenges of crispr-based gene therapies and engineering approaches to address them\n",
      "challenge 1: diminishing off-target activity in the genome\n",
      "challenge 2: enhancing gene editing efficiency in vivo\n",
      "challenge 3: efficient transcriptional modulation\n",
      "challenge 4: delivery\n",
      "challenge 5: immune response to the gene editing tool\n",
      "challenge 6: spatiotemporal control: rational engineering approaches for spatiotemporal control of crispr\n",
      "opportunities and future perspectives\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400112u/sb400112u.xml\n",
      "results and discussion\n",
      "design and optimization of the genetic circuit\n",
      "application of the genetic circuit for cell-based enzyme assay\n",
      "testing the genetic circuit as generalized enzyme screening system\n",
      "high-throughput screening from a metagenome library\n",
      "perspectives for a generalized and high-throughput screening system\n",
      "methods\n",
      "materials\n",
      "tyrosine phenol-lyases\n",
      "lipase\n",
      "cellulase\n",
      "mph\n",
      "metagenomic screening\n",
      "analysis\n",
      "fluorescence microscope\n",
      "fluorescence plate reader\n",
      "high pressure liquid chromatography (hplc)\n",
      "pnp colorimetric assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00332/sb8b00332.xml\n",
      "results\n",
      "design and genomic insertion of extra copies of trna genes\n",
      "growth properties of sixpack and control strains\n",
      "elevated expression of rare trnas\n",
      "individual impact of elevated levels of the rare trnas on protein expression\n",
      "heterologous protein expression performance of sixpack and control strains\n",
      "discussion\n",
      "materials and methods\n",
      "medium\n",
      "e. coli strains\n",
      "genomic construct\n",
      "expression plasmids\n",
      "quantitative real-time pcr (qrt-pcr)\n",
      "measuring and calculating growth parameters\n",
      "fluorescence measurements\n",
      "protein expression and whole cell lysate preparation\n",
      "sds-page and western blotting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00040/sb5b00040.xml\n",
      "methods\n",
      "magnet vector construction\n",
      "lentivirus production\n",
      "stereotaxic surgeries\n",
      "immunohistochemistry\n",
      "confocal imaging and quantification\n",
      "pearson’s chi squared test\n",
      "cortical neuron culture transfection and electrophysiology\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00237/sb5b00237.xml\n",
      "methods\n",
      "cloning\n",
      "r body expression and purification\n",
      "electron microscopy\n",
      "circular dichroism\n",
      "light microscopy\n",
      "spectrophotometric assay\n",
      "screen for mutants defective in ph response\n",
      "sequence analysis\n",
      "spheroplasting\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00523/sb8b00523.xml\n",
      "results and discussion\n",
      "general descriptions of bmss\n",
      "inducible promoter systems\n",
      "constitutive promoters and rbss library\n",
      "logic gates\n",
      "not gates (inverter)\n",
      "and gates\n",
      "or gates\n",
      "cross-platform implementations\n",
      "discussion\n",
      "methods\n",
      "plasmids construction\n",
      "growth and characterization conditions\n",
      "installations\n",
      "python tools\n",
      "input data curation\n",
      "full system workflow\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500247a/sb500247a.xml\n",
      "results and discussion\n",
      "methods\n",
      "protein crystallization and structure determination\n",
      "homology modeling\n",
      "plasmid construction and generation of the mutant libraries\n",
      "in vivo hydroxylation of l -pip\n",
      "construction and screening of the smp4h mutant library\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00081/sb6b00081.xml\n",
      "results and discussion\n",
      "construction of prouser vectors: a user compatible set of broad-host-range vectors for synthetic biology\n",
      "comparison of different expression systems in p. putida kt2440\n",
      "investigation of single-cell heterogeneity\n",
      "orthogonality of promoters pm and prhab\n",
      "construction of a double deletion strain p. putida kt2440δ fcs δ hpd for production of p -coumaric acid\n",
      "use of pm and prhab promoters for independent expression of p -coumaric acid and tyrosine pathways\n",
      "optimization of the production of p -coumaric acid in p. putida kt2440δ fcs δ hpd through modulation of the inducers\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmids and strains constructions\n",
      "expression assays\n",
      "population assessment using flow cytometry\n",
      "production assays\n",
      "hplc analysis\n",
      "availability of prouser vectors\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00119/sb6b00119.xml\n",
      "results\n",
      "generation of cyp719 and nmt enzyme libraries\n",
      "selection of replacement ring a-closing cyp719s\n",
      "engineering of a non-natural stylopine synthesis pathway\n",
      "combinatorial testing of ring a- and ring b-closing cyp719s\n",
      "selection of replacement tnmt\n",
      "retrofitting and testing an optimized dihydrosanguinarine producing strain\n",
      "discussion\n",
      "materials and methods\n",
      "transcriptome data analysis\n",
      "construction of plasmids\n",
      "chromosomal integration of genes and multigene pathways\n",
      "bia culture supplementation assays\n",
      "liquid chromatography–mass spectrometry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00231/sb5b00231.xml\n",
      "leakage problem\n",
      "thermal fluctuations in dna\n",
      "insight from secondary structure\n",
      "results and discussion\n",
      "effect of mismatch on leakage rate constants\n",
      "availability\n",
      "base modifications at the 5′ end of the fuel\n",
      "base modifications at the 3′ end of the fuel\n",
      "base modifications of the fuel at the nick location\n",
      "mutual availability\n",
      "catalyzed reactions\n",
      "system performance\n",
      "availability analysis of other networks\n",
      "conclusions\n",
      "methods\n",
      "chemicals and dna complex purification\n",
      "spectrofluorimetry\n",
      "reaction measurements\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00084/sb5b00084.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300042w/sb300042w.xml\n",
      "results and discussion\n",
      "lamb-5rgbp is expressed in the outer membrane of s. oneidensis and increases attachment to gold surfaces\n",
      "expression of lamb and lamb-5rgbp is associated with the loss of outer membrane cytochromes required for extracellular respiration\n",
      "expression of lamb-5rgbp does not significantly increase electron transfer to gold electrodes\n",
      "expression of lamb-5rgbp also reduces electron transfer to graphitic carbon electrodes\n",
      "expression of lamb and lamb-5rgbp decrease rates of flavin and iron oxide reduction\n",
      "riboflavin remains stable in electrochemical cells\n",
      "cells expressing lamb-5rgbp are repelled by gold surfaces poised at positive potentials\n",
      "methods\n",
      "strains and growth conditions\n",
      "reagents\n",
      "vector construction\n",
      "membrane fractionation and characterization\n",
      "preparation of gold(111) surfaces\n",
      "gold-binding assays\n",
      "heme staining\n",
      "western blots\n",
      "riboflavin reduction assay\n",
      "bioreactor construction\n",
      "oxidation of riboflavin by gold(111) electrodes\n",
      "current analysis in reactors poised during attachment\n",
      "current analysis in reactors not poised during attachment\n",
      "imaging of electrodes\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400031e/sb400031e.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00027/sb9b00027.xml\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "materials and methods\n",
      "reporter strain construction\n",
      "yfp reporter assays\n",
      "vanillin-β-glucoside production strains\n",
      "bioreactor cultures and hplc analysis of metabolites\n",
      "rnaseq and transcriptome data analysis\n",
      "database for rna-seq data\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400037a/sb400037a.xml\n",
      "methods\n",
      "library and plasmid construction\n",
      "cell culture and transfection\n",
      "egfp assay\n",
      "seap assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00187/sb9b00187.xml\n",
      "materials and methods\n",
      "chemicals and reagents\n",
      "cloning of plasmids\n",
      "recombinant protein expression and purification\n",
      "protein purification\n",
      "isopeptide bond formation between spytag-adh and eutm-spycatcher tested by sds-page\n",
      "negative stain transmission electron microscopy\n",
      "activity and stability of (spytag)-adh immobilized on scaffolds\n",
      "results and discussion\n",
      "design and in vitro characterization of a toolbox of protein scaffolds\n",
      "rapid covalent enzyme immobilization on eutm-spycatcher scaffolds\n",
      "eutm-spycatcher scaffolds have different effects on the activity of free and immobilized enzyme\n",
      "eutm-spycatcher scaffolds increase enzyme stability\n",
      "conclusions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00110/sb8b00110.xml\n",
      "results and discussion\n",
      "design of a synthetic expression module (sem) library\n",
      "quantification of the sem library\n",
      "identification and ranking of elements\n",
      "contribution of individual elements\n",
      "expression in log and stationary growth\n",
      "application of strong sems for xylanase production\n",
      "conclusion\n",
      "methods\n",
      "bacterial strains and growth conditions\n",
      "plasmid and strain construction\n",
      "stargate and library cloning\n",
      "xylanase assay\n",
      "microscopy and data analysis\n",
      "bioinformatics design\n",
      "validation sem library\n",
      "fluorescence-activated cell sorting (facs)\n",
      "fluorescence-activated cell scan (facscan)\n",
      "amplification and barcoding of the sem library\n",
      "sequencing of sem library\n",
      "prediction of translation rates and total free energy exchange between ribosome and mrna\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00219/sb7b00219.xml\n",
      "results and discussion\n",
      "characterization of the ahl-responsive lasrv biosensor\n",
      "investigation of ahls in cf patient sputum samples\n",
      "optimizing the cell-free platform for measurement of ahls from cf sputum samples\n",
      "clinical relevance of qs molecules\n",
      "translating lab-based cell-free biosensors to practice\n",
      "methods\n",
      "the lasrv biosensor device\n",
      "commercial e. coli cell-free reactions\n",
      "tx-tl cell-free reactions\n",
      "collection of cf patient samples\n",
      "ahl extraction from cf sputum\n",
      "testing of sputum sample extract by lasrv biosensor\n",
      "efficiency of ahl recovery by solvent extraction\n",
      "comparison of different cell-free systems activity\n",
      "mass spectrometry analysis of extracted ahls\n",
      "data analysis and statistics\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00515/sb8b00515.xml\n",
      "introduction\n",
      "results and discussion\n",
      "materials and methods\n",
      "strains used and growth conditions\n",
      "plasmid construction\n",
      "conjugation and syn6803 exconjugants selection\n",
      "rna isolation and real-time qrt-pcr to assay t7rnap transcript\n",
      "gfp assay through microplate reader\n",
      "cell extract and western blot\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400136f/sb400136f.xml\n",
      "photosynthesis\n",
      "photosynthesis and the photosystems\n",
      "coupling of psi and hydrogenases\n",
      "bioactive natural compounds\n",
      "moving from bulk energy production toward high-value fine chemicals\n",
      "different groups of bioactive natural compounds\n",
      "applications of bioactive natural compounds\n",
      "high value–low yield\n",
      "cytochrome p450s\n",
      "function of p450s\n",
      "classification and redox partners\n",
      "plant p450s and biosynthesis of natural bioactive compounds\n",
      "heterologous expression of plant cytochrome p450s in microbial hosts\n",
      "supply of reducing equivalents\n",
      "redirecting photosynthetic reducing equivalents\n",
      "combining cytochromes p450 with the photosynthetic machinery\n",
      "conclusions, perspectives, and challenges\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300114d/sb300114d.xml\n",
      "results and discussion\n",
      "discovery of a promoter element in bba_c0051\n",
      "determining the location of the active promoter\n",
      "experimentally determining the transcript start site\n",
      "informatic analysis of the pkat\n",
      "conditional characterization of the promoter\n",
      "in silico detection of putative promoters\n",
      "other biobrick parts with similar motifs\n",
      "summary and conclusion\n",
      "methods\n",
      "molecular assembly\n",
      "site directed mutagenesis\n",
      "determining transcript start site using 5′-race\n",
      "fluorescence and optical density measurements\n",
      "motif search using mast\n",
      "discovery of parts containing hidden barcodes in parts registry\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00030/sb6b00030.xml\n",
      "results\n",
      "construction and validation of a rhabad reporter construct\n",
      "screening analogues of l -rhamnose as inducers of the rhabad promoter system\n",
      "decoupling the rhabad promoter from the rhasr regulatory cascade reveals effective inducers\n",
      "characterization and comparison of promising inducers\n",
      "discussion\n",
      "materials and methods\n",
      "rare sugars\n",
      "bacterial strains and growth conditions\n",
      "plasmid construction\n",
      "assays\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb400023u/sb400023u.xml\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00316/sb7b00316.xml\n",
      "results and discussion\n",
      "cfps-based screening system\n",
      "cfps of screening pool\n",
      "pegylation efficiency screening\n",
      "stability screening\n",
      "activity screening\n",
      "evaluating common pegylation guidelines with experimental screen results\n",
      "enhancing cfps-based screen with coarse-grain simulation\n",
      "conclusions\n",
      "methods\n",
      "extract preparation\n",
      "plasmid preparation\n",
      "cell-free protein synthesis\n",
      "t4 lysozyme purification\n",
      "pegylation reactions\n",
      "pegylation efficiency analysis\n",
      "stability analysis\n",
      "stability assay\n",
      "simulation-predicted t m\n",
      "activity assay\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5001356/sb5001356.xml\n",
      "methods\n",
      "antibodies and reagents\n",
      "molecular biology\n",
      "light irradiation assay\n",
      "dissociated neuron cultures\n",
      "immunohistochemistry\n",
      "live cell imaging\n",
      "image analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00042/sb7b00042.xml\n",
      "results and discussion\n",
      "the pore of pdua is different from the pore of eutm\n",
      "the growth of s. enterica on 1,2-pd is improved in δ pdua::eutm strain\n",
      "eutm is localized to the pdu mcp in the s. enterica δ pdua::eutm strain\n",
      "pdua k26a and eutm k24a mutations both abrogate mcp formation\n",
      "limiting coenzyme b 12 condition confirms δ pdua::eutm mcps form a complete compartment shell\n",
      "the effects of eutm substitution are recapitulated by a point mutation in the pdua pore\n",
      "the eutm substitution and the pdua k37q pore mutant change the accumulation of downstream metabolites\n",
      "shell protein pore residues can be altered to control encapsulated enzyme function\n",
      "pore residues of pdua evolve rapidly\n",
      "pdua pore mutants conferring faster growth on 1,2-pd are not abundant among pdua homologues\n",
      "conclusions\n",
      "methods\n",
      "bacterial strains, media, and growth conditions\n",
      "high performance liquid chromatography (hplc) media analysis\n",
      "mcp purification\n",
      "transmission electron microscopy\n",
      "immunostaining\n",
      "mass spectroscopy\n",
      "protein electrophoresis and western blot\n",
      "fluorescence microscopy\n",
      "multiple sequence alignment construction and evolutionary rate analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00146/sb6b00146.xml\n",
      "visual representation of genetic designs\n",
      "digital representation of sequence information\n",
      "workflow for genetic design deposit and review\n",
      "future directions\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00045/sb7b00045.xml\n",
      "results\n",
      "discussion\n",
      "methods\n",
      "design principles and construction of cages\n",
      "construction, configuration change and fret studies of the cages\n",
      "in vitro rna transcription, gel electrophoresis, and qpcr\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00034/sb5b00034.xml\n",
      "results and discussion\n",
      "primer design\n",
      "conclusion\n",
      "methods\n",
      "materials\n",
      "growth and purification of wild-yype m13 (wtm13)\n",
      "m13 conjugation of bifunctional linkage\n",
      "oligonucleotide synthesis, purification, and characterization\n",
      "bioconjugation\n",
      "size exclusion chromatography\n",
      "sds–page\n",
      "measurement of m13/dna concentration\n",
      "measurement of ld spectra\n",
      "fluorescence measurements\n",
      "polymerase chain reaction (pcr)\n",
      "agarose gel electrophoresis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00477/sb8b00477.xml\n",
      "results and discussion\n",
      "measuring transcriptional interference\n",
      "spontaneous termination in untranslated regions\n",
      "modeling of transcriptional interference\n",
      "methods\n",
      "assays\n",
      "stochastic simulations\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000639/sb3000639.xml\n",
      "methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00102/sb8b00102.xml\n",
      "results and discussion\n",
      "construction and characterization of a promoter library for e. coli\n",
      "characterization of the promoter library in different bacteria\n",
      "application of the promoter library to synthesize pha in h. bluephagenesis\n",
      "pha production by recombinant h. bluephagenesis with optimized promoter\n",
      "performance of h. bluephagenesis with optimized promoter in a 7 l fermentor\n",
      "conclusions\n",
      "materials and methods\n",
      "bacterial strains, media and chemicals\n",
      "plasmid library construction and conjugation\n",
      "flow cytometry and characterization of promoter strengths\n",
      "genome integration\n",
      "shake flask experiment\n",
      "fermentor studies\n",
      "pha analytical methods\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00078/sb7b00078.xml\n",
      "methods\n",
      "peptide design and preparation\n",
      "screening of synthetic α-factor analogues\n",
      "construction of expression plasmids\n",
      "protein expression and purification\n",
      "in vitro reconstitution of functional α-factor from split intein fusions\n",
      "yeast growth arrest assays using products of intein-catalyzed reactions\n",
      "transcriptional induction of gfp in yeast\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb300110b/sb300110b.xml\n",
      "results and discussion\n",
      "circuit topology\n",
      "secreted α-pheromone elicits a population-density-dependent cell-to-cell communication response\n",
      "positive feedback quorum sensing\n",
      "aromatic amino acid induced quorum sensing\n",
      "summary and conclusions\n",
      "methods\n",
      "media\n",
      "strains and plasmids\n",
      "sender–receiver assays\n",
      "flow cytometry\n",
      "fermentation conditions\n",
      "statistical analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb6b00040/sb6b00040.xml\n",
      "results\n",
      "a minimal model of ribosome allocation dynamics\n",
      "systematic experimental exploration of expression space\n",
      "impact of additional stress on expression dynamics\n",
      "discussion\n",
      "methods\n",
      "expression vectors\n",
      "biolector experiments\n",
      "temperature stress experiments\n",
      "in gel fluorescence and immunodetection\n",
      "qrt-pcr\n",
      "model fitting\n",
      "model simulation and data analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00533/sb8b00533.xml\n",
      "calibrated interpretation of flow cytometry data\n",
      "automation-assisted gating\n",
      "compensation for autofluorescence and spectral overlap\n",
      "unit calibration\n",
      "example application to experimental data\n",
      "tasbe architecture and applications\n",
      "analytical workflows\n",
      "interactive user interface\n",
      "workflow integration\n",
      "discussion\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500255k/sb500255k.xml\n",
      "results and discussion\n",
      "identification of the minimum length of a targeting sequence\n",
      "design of the hi-crispr system\n",
      "discovery of a cas9 variant\n",
      "determination of the optimal length of an hr disruption donor\n",
      "stimulation of gene disruption by increasing the plasmid copy number\n",
      "one-step multiple gene disruption by the hi-crispr system\n",
      "investigation of increased multiplexing\n",
      "methods\n",
      "strains and media\n",
      "crispr–cas target site selection and donor design\n",
      "plasmid construction\n",
      "yeast transformation\n",
      "calculation of single gene disruption efficiency\n",
      "calculation of the triple gene disruption efficiency\n",
      "yeast genomic dna extraction and target site sequencing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb8b00510/sb8b00510.xml\n",
      "results and discussion\n",
      "construction and initial characterization of a reporter strain for screening of a pyc mutant library\n",
      "testing the reporter strain by comparison of wild-type pcx and the pcx p458s variant\n",
      "construction and screening of a pyc mutant library\n",
      "testing of clones isolated from the pyc mutant library regarding growth and l -lysine production\n",
      "analysis of isolated clones after reconstruction\n",
      "chromosomal integration of two pyc mutations and analysis of their impact on growth and l -lysine production\n",
      "alignment and structural modeling of pcx\n",
      "concluding remarks\n",
      "methods\n",
      "bacterial strains, plasmids, growth conditions, recombinant dna techniques, and lysine quantification\n",
      "library construction\n",
      "facs analysis\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00462/sb7b00462.xml\n",
      "results\n",
      "effect of high-level dcas9 on cell growth\n",
      "effect of dcas9 expression levels on cell morphology\n",
      "changes in mrna transcript levels induced by dcas9\n",
      "effect of selected degs on cell morphology\n",
      "location analysis of dcas9\n",
      "discussion\n",
      "methods\n",
      "strains and cell culture\n",
      "western blotting\n",
      "cell imaging\n",
      "flow cytometry\n",
      "rna-seq\n",
      "chromatin immunoprecipitation (chip) sequencing\n",
      "data processing\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500225r/sb500225r.xml\n",
      "methods\n",
      "bacterial strains and culture conditions\n",
      "cloning and plasmid construction\n",
      "tricine-sds-page\n",
      "nisin induction experiment\n",
      "nisin bioassay\n",
      "kinetics of nisin production\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb9b00131/sb9b00131.xml\n",
      "results and discussion\n",
      "wires decorated with a six-histidine peptide (his-tag)\n",
      "wires decorated with two different peptides\n",
      "conclusions\n",
      "materials and methods\n",
      "strains and growth conditions\n",
      "construction of g. sulfurreducens pila-wt/pila-6his strain\n",
      "construction of g. sulfurreducens pila-wt/pila-6his/pila-ha strain\n",
      "western blot analysis\n",
      "immunogold labeling\n",
      "ni 2+ -binding assay\n",
      "current production\n",
      "c-afm of individual e-pns\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb3000645/sb3000645.xml\n",
      "bistability of an in vitro autoregulatory switch\n",
      "engineering of biological circuits–a workflow\n",
      "automated part selection for regulatory networks\n",
      "modeling of microbial biofilms\n",
      "hierarchical modeling for synthetic biology\n",
      "simulation of multicelled behaviors\n",
      "nonlinear dynamic models of synthetic bioparts\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb500195w/sb500195w.xml\n",
      "methods\n",
      "materials and general methods\n",
      "library construction\n",
      "positive selection\n",
      "negative selection assay\n",
      "hit verification\n",
      "fluorescence analysis of bacterial culture\n",
      "protein expression and purification\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb7b00167/sb7b00167.xml\n",
      "results and discussion\n",
      "different pdu operon designs lead to diverse fluorescence patterns in c. glutamicum\n",
      "optimization of protein stoichiometry for compartment assembly\n",
      "chromosomal integration of the pduabjknt shell operon\n",
      "c-terminal targeting to bmcs is possible with native and non-native encapsulation peptides\n",
      "implementation of protein scaffolds for bmc targeting\n",
      "pdua and pduj form filaments in c. glutamicum\n",
      "n-terminal targeting peptides recruit eyfp to pdua scaffolds\n",
      "c-terminal targeting to pdua is possible by using pdz and gbd interactions\n",
      "impact of pduabjknt bmcs on ethanol production in c. glutamicum\n",
      "conclusion\n",
      "material and methods\n",
      "bacterial strains, plasmids and growth conditions\n",
      "recombinant dna work and construction of chromosomal insertion strains\n",
      "adhb and pdc enzyme assay\n",
      "transmission electron microscopy\n",
      "fluorescence microscopy\n",
      "\n",
      "parsing /home/jiawei/sbks-ucsd/biobert/acs-data/unzip-files/sb5b00276/sb5b00276.xml\n",
      "results\n",
      "discussion\n",
      "experimental section\n",
      "materials\n",
      "general procedures (organic synthesis)\n",
      "synthesis of 5′- o -(4,4′-dimethoxytrityl)- n 6 [(diisobutylamino)­methylidene]- o 2 -(diphenylcarbamoyl)-3′- o -acetyl-7-deazaisoguanosine ( 2 )\n",
      "3′- o -acetyl-2′-deoxy- n 6 [(diisobutylamino)­methylidene]- o 2 -(diphenylcarbamoyl)-7-deazaisoguanosine ( 3 )\n",
      "7-deaza-2′-deoxyisoguanosine 5′-triphosphate­( 4 )\n",
      "pcr and acid hydrolysis conditions\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# parse the files\n",
    "for xml_path in tqdm(xml_paths):\n",
    "    \n",
    "    print(\"\\nparsing %s\" % xml_path)\n",
    "    \n",
    "    # get the root of the xml\n",
    "    root = etree.parse(xml_path).getroot()\n",
    "    \n",
    "    # create a dictionary holding the xml data\n",
    "    xml_data = {\n",
    "        \"keywords\": extract_keywords(root),\n",
    "        \"abstract\": extract_abstract(root),\n",
    "        \"body\": extract_body(root)\n",
    "    }\n",
    "    \n",
    "    # pickle the data\n",
    "    # name the file to <pub #>.pkl\n",
    "    pub_num = xml_path.split(\"/\")[-1].split(\".\")[0]\n",
    "    with open(os.path.join(output_path, pub_num + \".pkl\"), \"wb\") as f:\n",
    "        pickle.dump(xml_data, f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pkl files found: 1545\n"
     ]
    }
   ],
   "source": [
    "# load some file to check\n",
    "pkl_paths = []\n",
    "collect_files(output_path, pkl_paths, pattern=\".*\\.pkl$\", collect_dirs=False)\n",
    "print(\"pkl files found: %d\" % len(pkl_paths))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'abstract': ['Metabolic engineering and synthetic biology usually require '\n",
      "              'universal expression systems for stable and efficient gene '\n",
      "              'expression in various organisms. In this study, a '\n",
      "              'host-independent and stable T7 expression system had been '\n",
      "              'developed by integrating T7 RNA polymerase and its cognate '\n",
      "              'transcriptional units in single plasmid. The expression of T7 '\n",
      "              'RNA polymerase was restricted below its lethal threshold using '\n",
      "              'a T7 RNA polymerase antisense gene cassette, which allowed long '\n",
      "              'periods of cultivation and protein production. In addition, by '\n",
      "              'designing ribosome binding sites, we further tuned the '\n",
      "              'expression capacity of this novel T7 system within a wide '\n",
      "              'range. This host-independent expression system efficiently '\n",
      "              'expressed genes in five different Gram-negative strains and one '\n",
      "              'Gram-positive strain and was also shown to be applicable in a '\n",
      "              'real industrial d-p-hydroxyphenylglycine production system.'],\n",
      " 'body': [{'section': [],\n",
      "           'text': 'Engineering complex genetic circuits and metabolic '\n",
      "                   'pathways in synthetic biology and metabolic engineering '\n",
      "                   'has become easier and more efficient with the rapid '\n",
      "                   'development of biotechnologies. The typical procedure used '\n",
      "                   'in synthetic biology and metabolic engineering is to '\n",
      "                   'reconstruct metabolic networks in a host by regulating '\n",
      "                   'protein expression (or protein activity) or to design a '\n",
      "                   'metabolic pathway, resulting in the desired functions or '\n",
      "                   'products. Most applications of reconstructed networks are '\n",
      "                   'mainly restricted to several model organisms such as '\n",
      "                   'Escherichia coli and Saccharomyces cerevisiae. This '\n",
      "                   'restriction is due to the limited availability of genetic '\n",
      "                   'elements and accessibility of genomic and metabolic data '\n",
      "                   'in noncanonical organisms, i.e., their complicated genetic '\n",
      "                   'backgrounds. However, noncanonical organisms usually have '\n",
      "                   'advantages over model organisms; for instance, they '\n",
      "                   'naturally produce and tolerate high concentrations of '\n",
      "                   'desired products and live in environments with higher or '\n",
      "                   'lower pH and temperature values, which are valuable traits '\n",
      "                   'for industrial biotechnology. Thus, developing some '\n",
      "                   'universal genetic elements and circuits, i.e., '\n",
      "                   'host-independent systems, for both model and noncanonical '\n",
      "                   'organisms is essential and will broaden the choice of '\n",
      "                   'hosts applied in synthetic biology and metabolic '\n",
      "                   'engineering.'},\n",
      "          {'section': [],\n",
      "           'text': 'T7 RNAP, originating from bacteriophage T7, has been long '\n",
      "                   'studied and widely applied in engineering genetic circuits '\n",
      "                   'and metabolic pathways in roles such as controllers, logic '\n",
      "                   'gates, resource allocators, and orthogonal systems. '\n",
      "                   '(Figure S1) Due to its orthogonality, transcriptional '\n",
      "                   'efficiency and promoter specificity, T7 RNAP is an '\n",
      "                   'excellent candidate for the construction of an ideal '\n",
      "                   'host-independent expression system. T7 RNAP functions in a '\n",
      "                   'variety of hosts, including prokaryotic, eukaryotic and '\n",
      "                   'even cell-free systems. The most common way to utilize the '\n",
      "                   'T7 expression system relies on a classical approach where '\n",
      "                   'it is used by combining DE3 lysogenic hosts (integrating '\n",
      "                   'the T7 RNAP gene into the host genome) and cognate '\n",
      "                   'plasmids containing T7 promoter-induced transcription '\n",
      "                   'units. Although the lethal effect of T7 RNAP is attenuated '\n",
      "                   'by the reduced amount of T7 RNAP produced from a single '\n",
      "                   'copy of the T7 RNAP gene in the genome, the dependence on '\n",
      "                   'the preconstruction of DE3 lysogenic hosts limits its '\n",
      "                   'universal application in noncanonical hosts. Since the '\n",
      "                   'discovery of the T7 system, numerous efforts have been '\n",
      "                   'made to construct host-independent T7 expression systems '\n",
      "                   'for extending its application to various hosts. However, '\n",
      "                   'the majority of these efforts failed due to excessive '\n",
      "                   'expression of T7 RNAP, which killed the host cells or '\n",
      "                   'caused mutations in the T7 RNAP gene under the biological '\n",
      "                   'stress. A few reported successful attempts include the '\n",
      "                   'establishment of a two-vector system (low-copy-number and '\n",
      "                   'high-copy-number plasmids) and the mutation or splitting '\n",
      "                   'of T7 RNAP to reduce lethal effects and biological stress. '\n",
      "                   'Owing to the difficulties in finding two biocompatible '\n",
      "                   'plasmids for noncanonical hosts or sacrificing the high '\n",
      "                   'transcriptional activity of T7 RNAP, these attempts do not '\n",
      "                   'extend the application of the T7 system well. Recently, '\n",
      "                   'the work of Manish and Zhao paved a new way for developing '\n",
      "                   'T7 expression systems in an extensive range of hosts; '\n",
      "                   'although feedback-circuit regulation and gene mining were '\n",
      "                   'achieved with T7 systems, the two-vector system and '\n",
      "                   'host-dependent system are still used in these studies. '\n",
      "                   'These limitations underline the significance to facilitate '\n",
      "                   'the development of novel host-independent T7 expression '\n",
      "                   'systems through the integration of full-length original T7 '\n",
      "                   'RNA polymerase and its cognate transcriptional units into '\n",
      "                   'a single plasmid to meet the long-standing request of '\n",
      "                   'applying T7 systems as a universal genetic element in '\n",
      "                   'various organisms.'},\n",
      "          {'section': [],\n",
      "           'text': 'Here, we report the implementation of a stable and '\n",
      "                   'host-independent expression system containing a T7 RNAP '\n",
      "                   'expression cassette and its cognate transcriptional units '\n",
      "                   'in single plasmid. The critical idea of our design is to '\n",
      "                   'strictly regulate the amount of T7 RNAP produced by a '\n",
      "                   'single plasmid system to be below the lethal and '\n",
      "                   'biological stress levels. The host-independent T7 '\n",
      "                   'expression system (HITES) efficiently overexpressed '\n",
      "                   'heterologous proteins in various Gram-positive and '\n",
      "                   'Gram-negative bacteria, and it was also stable in serial '\n",
      "                   'subculture.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Scheme describes the operating principle of HITES designed '\n",
      "                   'in this study. The T7 RNAP gene is transcribed into a '\n",
      "                   'large number of mRNA molecules under the control of the '\n",
      "                   'lac promoter (the universality of the lac promoter has '\n",
      "                   'been evaluated on the basis of the data in Table S1). At '\n",
      "                   'the same time, the antisense gene of T7 RNAP in the same '\n",
      "                   'plasmid is transcribed into antisense RNA (asRNA) '\n",
      "                   'molecules, which can reduce the expression of the T7 RNAP '\n",
      "                   'protein by annealing to its cognate mRNA. Consequently, '\n",
      "                   'the amount of T7 RNAP will be limited below the lethal '\n",
      "                   'threshold, which will guarantee the survival of the host. '\n",
      "                   'The limited amount of T7 RNAP can then recognize the T7 '\n",
      "                   'promoter on the same plasmid and initiate the '\n",
      "                   'transcription of the target gene (gfp in this case) at a '\n",
      "                   'proper speed. This design allows this HITES to be '\n",
      "                   'integrated into any plasmid and work effectively in '\n",
      "                   'corresponding hosts.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'a CAP: the CAP binding site of the lac promoter, TT: '\n",
      "                   'terminator. Elements marked in red fonts were engineered '\n",
      "                   'in the following section to generate a stable HITES.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'CAP: the CAP binding site of the lac promoter, TT: '\n",
      "                   'terminator. Elements marked in red fonts were engineered '\n",
      "                   'in the following section to generate a stable HITES.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'The construction of the first-generation HITES has been '\n",
      "                   'reported in previous work. However, its efficacy was '\n",
      "                   'impaired because of its instability during serial '\n",
      "                   'subculture (Figure b). The proportion of GFP-free cells '\n",
      "                   'increased significantly even after the first subculture, '\n",
      "                   'suggesting that the target gene (gfp) was not expressed '\n",
      "                   'properly in host cells and that the T7 RNAP gene in the '\n",
      "                   'HITES might have been mutated; this latter possibility was '\n",
      "                   'confirmed by plasmid sequencing. The stability of the '\n",
      "                   'first-generation HITES was significantly improved when GLB '\n",
      "                   'medium (LB medium contain 1% glucose) was used instead of '\n",
      "                   'LB medium (Figure c), but this strategy did not satisfy '\n",
      "                   'the laboratory and industrial demands of long serial '\n",
      "                   'cultivation.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Instability of the first-generation HITES. (a) Schematic '\n",
      "                   'representation of the subculture experiment. (b) The '\n",
      "                   'fluorescent cell distribution after each subculture in LB '\n",
      "                   'medium. (c) The fluorescent cell distribution after each '\n",
      "                   'subculture in GLB medium (LB medium containing 1% '\n",
      "                   'glucose).'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'E. coli K-12 and E. coli B are extensively used in the '\n",
      "                   'laboratory and industry. Since E. coli JM109 and BL21 are '\n",
      "                   'derived from E. coli K-12 and B strains, respectively, and '\n",
      "                   'both have corresponding DE3 lysogenic hosts, they were '\n",
      "                   'selected for testing the expression efficiency and '\n",
      "                   'stability of HITES. As shown in Figure a and d, the amount '\n",
      "                   'of T7 RNAP in hosts harboring the first-generation HITES '\n",
      "                   '(A4A27) was much higher than that in DE3 lysogenic hosts, '\n",
      "                   'indicating that the elements in the first-generation HITES '\n",
      "                   'were not sufficient to limit the amount of T7 RNAP to '\n",
      "                   'nonlethal levels and that a more elaborate regulation of '\n",
      "                   'T7 RNAP expression was required for improving the '\n",
      "                   'stability of plasmids harboring a HITES. In the following '\n",
      "                   'section, we dissect the contribution of each element, '\n",
      "                   'including the asRNA, promoter and terminator, to the '\n",
      "                   'amount of T7 RNAP and determine how changes in the T7 RNAP '\n",
      "                   'level influenced the expression performance and stability '\n",
      "                   'of the HITES.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Construction and verification of a stable HITES. (a) The '\n",
      "                   'amount of T7 RNAP in JM109 host cells with different '\n",
      "                   'modifications (asRNA sequence, CAP deletion, and '\n",
      "                   'terminator replacement) was analyzed by Western blot using '\n",
      "                   'monoclonal antibodies against T7 RNAP. JM109: JM109 '\n",
      "                   'harboring pET30-GFP, negative control; JM109\\xad(DE3): '\n",
      "                   'JM109\\xad(DE3) harboring pET30-GFP, positive control; '\n",
      "                   'JM109/A4A27: JM109 harboring the first-generation HITES; '\n",
      "                   'JM109/A4A27-Hfq: JM109 harboring a HITES with synthetic '\n",
      "                   'asRNA; JM109/A4A27-ΔCAP: JM109 harboring a HITES with CAP '\n",
      "                   'deletion; JM109/A4A27-Hfq-ΔCAP: JM109 harboring a HITES '\n",
      "                   'with synthetic asRNA and CAP deletion; JM109/HITES: JM109 '\n",
      "                   'harboring a HITES with synthetic asRNA, CAP deletion and '\n",
      "                   'terminator replacement. (b) The RFI of JM109 host cells '\n",
      "                   'with different modifications (asRNA sequence, CAP '\n",
      "                   'deletion, and terminator replacement). (c) Single-cell GFP '\n",
      "                   'fluorescence distributions of JM109 host cells with '\n",
      "                   'different modifications (asRNA sequence, CAP deletion, and '\n",
      "                   'terminator replacement). (d) The amount of T7 RNAP in BL21 '\n",
      "                   'host cells with different modifications (asRNA sequence, '\n",
      "                   'CAP deletion, and terminator replacement) was analyzed by '\n",
      "                   'Western blot using monoclonal antibodies against T7 RNA '\n",
      "                   'polymerase. (e) The RFI of BL21 host cells with different '\n",
      "                   'modifications (asRNA sequence, CAP deletion, and '\n",
      "                   'terminator replacement). (f) Single-cell GFP fluorescence '\n",
      "                   'distributions of BL21 host cells with different '\n",
      "                   'modifications (asRNA sequence, CAP deletion, and '\n",
      "                   'terminator replacement).'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'In the first-generation HITES, a 24 nt long asRNA (A4A27) '\n",
      "                   'was used without any supporting sequences, resulting in a '\n",
      "                   'low binding capacity and instability for this asRNA. In '\n",
      "                   'this study, a scaffold sequence for recruiting Hfq protein '\n",
      "                   'was added to improve the inhibitory effect of asRNA, '\n",
      "                   'yielding the newly designed synthetic asRNA, A4A27-Hfq. '\n",
      "                   'Hfq protein enhances the hybridization of asRNA-target '\n",
      "                   'mRNA, stabilizes bound asRNA, and facilitates the '\n",
      "                   'degradation of target mRNA by recruiting RNases. The '\n",
      "                   'introduction of the Hfq binding sequence greatly improved '\n",
      "                   'the inhibition of T7RNAP by asRNA in both JM109 (Figure a) '\n",
      "                   'and BL21 (Figure d) cells, and the relative fluorescence '\n",
      "                   'intensity (RFI) of the GFP slightly increased (Figure b '\n",
      "                   'and e). In addition, the GFP fluorescence distribution in '\n",
      "                   'BL21 cells was wider than that in JM109 (Figure c and f), '\n",
      "                   'this might be due to the differences in the genotype and '\n",
      "                   'genetic background of these two strains, leading to the '\n",
      "                   'differences in probabilities of plasmid mutation and '\n",
      "                   'recombination. However, since the amount of T7 RNAP was '\n",
      "                   'still relatively high, additional strategies should be '\n",
      "                   'taken to further reduce its level.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'The increased stability of the first-generation HITES when '\n",
      "                   'GLB medium was used indicated that glucose influenced the '\n",
      "                   'activity of the lac promoter via the glucose effect. The '\n",
      "                   'stability of the HITES could thus be improved by a '\n",
      "                   'promoter engineering strategy, which in this case aimed to '\n",
      "                   'restrict the activity of the lac promoter to a required '\n",
      "                   'level. An analysis of lac promoter structure revealed two '\n",
      "                   'binding sites in the lac promoter: the catabolite '\n",
      "                   'activator protein (CAP) binding site and the host RNA '\n",
      "                   'polymerase binding site, both of which operate '\n",
      "                   'coordinately to enhance the expression of lac '\n",
      "                   'promoter-controlled genes. Mutations or deletion of the '\n",
      "                   'CAP binding site could reduce the activity of the lac '\n",
      "                   'promoter by interfering with the interaction between RNA '\n",
      "                   'polymerase and the lac promoter. Therefore, the CAP site '\n",
      "                   'was deleted from the lac promoter that initiated T7 RNAP '\n",
      "                   'transcription to generate A4A27-ΔCAP and A4A27-Hfq-ΔCAP. '\n",
      "                   'As shown in Figure a, the expression of T7 RNAP was '\n",
      "                   'significantly reduced in JM109 cells with a CAP site '\n",
      "                   'deletion. The final concentration of T7 RNAP in '\n",
      "                   'JM109/A4A27-Hfq-ΔCAP cells (with both synthetic asRNA and '\n",
      "                   'a CAP site deletion) was approximately similar to that of '\n",
      "                   'JM109\\xad(DE3). With the decrease in T7 RNAP expression, '\n",
      "                   'the RFI of cells (Figure b) increased, and the '\n",
      "                   'distribution of fluorescent cells (Figure c) significantly '\n",
      "                   'improved. Similar results were also obtained in BL21 cells '\n",
      "                   '(Figure d, e and f). These results indicated that the '\n",
      "                   'synthetic asRNA and CAP site deletion effectively reduced '\n",
      "                   'the amount of T7 RNAP below the lethal threshold.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Stability of the HITES in E. coli JM109 during long '\n",
      "                   'periods of serial subculture. (a) The fluorescence '\n",
      "                   'intensity changes in JM109\\xad(DE3) and JM109/HITES cells '\n",
      "                   'measured by spectrophotometer after each subculture. (b) '\n",
      "                   'Single-cell GFP fluorescence distributions of '\n",
      "                   'JM109\\xad(DE3) and JM109/HITES cells measured by flow '\n",
      "                   'cytometer. The numbers 1, 5, 10, 15, 20, and 22 represent '\n",
      "                   'the 1st, 5th, 10th, 15th, 20th, and 22nd subculture, '\n",
      "                   'respectively. (c) The ratio of fluorescent JM109\\xad(DE3) '\n",
      "                   'and JM109/HITES cells after each subculture, which was '\n",
      "                   'determined from the results of a flow cytometer.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'However, when compared with the RFI of DE3 lysogenic '\n",
      "                   'hosts, the RFI of A4A27-Hfq-ΔCAP was relatively low '\n",
      "                   '(Figure b and e). According to the results above, this '\n",
      "                   'difference should not be mainly due to the excessive '\n",
      "                   'expression of T7 RNAP. One possible explanation was that '\n",
      "                   'the integration of the T7 RNAP gene with its cognate T7 '\n",
      "                   'promoter in one plasmid in our system caused the context '\n",
      "                   'effects of each component to impair the stability and '\n",
      "                   'overall performance of the HITES. Transcriptional '\n",
      "                   'read-through by T7 RNAP is a common phenomenon even when '\n",
      "                   'the T7 terminator (the termination efficiency was '\n",
      "                   'approximately 70%) is used. This read-through by T7 RNAP '\n",
      "                   'could result in the up-regulation of backbone sequences in '\n",
      "                   'the plasmid-based expression system and could hence lead '\n",
      "                   'to the abnormal expression of other components on the '\n",
      "                   'plasmid. In order to reduce T7 RNAP read-through, we '\n",
      "                   'therefore replaced the T7 terminator with a synthetic T7 '\n",
      "                   'termination signal (Tz) containing the two transcriptional '\n",
      "                   'terminators rrnBT1 and T7, whose termination efficiency is '\n",
      "                   'nearly 99% (the final unit with the designed asRNA, CAP '\n",
      "                   'site deletion and terminator replacement was denoted the '\n",
      "                   'HITES). As shown in Figure b, the RFI of JM109/HITES '\n",
      "                   'increased by 76% with the enhancement of termination '\n",
      "                   'efficiency and reached a level comparable to that of '\n",
      "                   'JM109\\xad(DE3). The fluorescent JM109/HITES cells were '\n",
      "                   'uniformly distributed with no obvious nonfluorescent cells '\n",
      "                   '(Figure c). Similar results were observed in BL21 cells '\n",
      "                   '(Figure d, e and f) when they had adopted the HITES. We '\n",
      "                   'successfully demonstrated that the performance of the '\n",
      "                   'HITES was significantly improved by applying the Tz '\n",
      "                   'terminator to overcome read-through transcription of T7 '\n",
      "                   'RNAP.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Considering that gene overexpression can cause stress that '\n",
      "                   'impairs cell growth, limiting stress to a reasonable level '\n",
      "                   'is essential. The stress of the HITES on host growth was '\n",
      "                   'thus investigated. (Figure S2) Our results showed that the '\n",
      "                   'growth of JM109/HITES and JM109\\xad(DE3) cells did not '\n",
      "                   'significantly differ (Figure S2a). Interestingly, '\n",
      "                   'BL21/HITES cells grew better and had higher cell densities '\n",
      "                   'than the BL21\\xad(DE3) strain (Figure S2b). On the other '\n",
      "                   'hand, since isopropyl β-d-1-thiogalactopyranoside (IPTG) '\n",
      "                   'is not suitable for large-scale industrial protein '\n",
      "                   'production because of its toxicity and high cost, we used '\n",
      "                   'lactose as an expression inducer instead of IPTG; in this '\n",
      "                   'case, the HITES still effectively expressed GFP comparable '\n",
      "                   'to DE3 lysogenic hosts (Figure S3). These results showed '\n",
      "                   'that the performance of hosts harboring the HITES were as '\n",
      "                   'good as conventional DE3 lysogenic hosts. More '\n",
      "                   'importantly, our results demonstrated that T7 expression '\n",
      "                   'system could also be applied in a “one-element '\n",
      "                   'system”\\ue5f8a single plasmid system, rather than a '\n",
      "                   '“two-element system”, which contains both a DE3 lysogenic '\n",
      "                   'host (that include a chromosomal T7 RNAP) and a plasmid '\n",
      "                   '(including a T7 promoter and the target gene).'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Genetically engineered bacteria need to be continuously '\n",
      "                   'subcultured in both laboratory research and industrial '\n",
      "                   'production. Therefore, the plasmid harboring the HITES '\n",
      "                   'must be genetically stable in host cells and not mutate. A '\n",
      "                   'serial subculture experiment was carried out to test the '\n",
      "                   'stability of the HITES. As shown in Figure a, the RFI '\n",
      "                   'fluctuated within a reasonable range, and neither '\n",
      "                   'JM109\\xad(DE3) nor JM109/HITES cells clearly declined in '\n",
      "                   'RFI. Meanwhile, nonfluorescent cells did not appear in '\n",
      "                   'Figure b, and the ratio of fluorescent cells remained '\n",
      "                   'constant (Figure c). These results underlined the '\n",
      "                   'stability of the HITES in E. coli JM109 strains.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'The stability of the HITES was further tested in E. coli '\n",
      "                   'BL21 strains. Figure a shows that the RFI of BL21/HITES '\n",
      "                   'cells fluctuated within a narrow range. As a positive '\n",
      "                   'control, the RFI of BL21\\xad(DE3) clearly declined over '\n",
      "                   'long periods of serial subculture. Similarly, the '\n",
      "                   'distribution of fluorescent cells remained fairly constant '\n",
      "                   'in BL21/HITES cells. In contrast, a small number of '\n",
      "                   'nonfluorescent cells appeared in BL21\\xad(DE3) after the '\n",
      "                   '15th subculture (Figure b), and a clear increase in '\n",
      "                   'nonfluorescent cells started. The fluorescence intensities '\n",
      "                   'of single cells started to decline significantly in '\n",
      "                   'BL21\\xad(DE3) cells (Figure b). As shown in Figure c, the '\n",
      "                   'ratio of fluorescent BL21/HITES cells stayed higher than '\n",
      "                   'this ratio for BL21\\xad(DE3) cells after each subculture. '\n",
      "                   'This behavior might be because these conventional '\n",
      "                   'BL21\\xad(DE3) cells were not stable over long periods of '\n",
      "                   'cultivation, which was identical to previously reported '\n",
      "                   'results. This decrease in target gene expression in DE3 '\n",
      "                   'hosts was attributed to chromosomal mutations that '\n",
      "                   'diminish the level of functional T7 RNA polymerase. In '\n",
      "                   'this context, the HITES exhibited better stability in E. '\n",
      "                   'coli BL21 cells than BL21\\xad(DE3) cells, which implied '\n",
      "                   'that the expression level of T7 RNAP from the HITES was '\n",
      "                   'less toxic in E. coli BL21 than the level of T7 RNAP in '\n",
      "                   'BL21\\xad(DE3) cells. Altogether, the HITES showed an '\n",
      "                   'excellent expression capacity and stability by inhibiting '\n",
      "                   'the expression of T7 RNAP with optimized asRNA, promoter '\n",
      "                   'and terminator engineering. All these results support the '\n",
      "                   'hypothesis that limiting the amount of T7 RNAP is the key '\n",
      "                   'to successfully constructing stable HITESs, which provides '\n",
      "                   'guidance for other researchers to develop T7 expression '\n",
      "                   'systems.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Stability of the HITES in E. coli BL21 during long periods '\n",
      "                   'of serial subculture. (a) The fluorescence intensity '\n",
      "                   'changes of BL21\\xad(DE3) and BL21/HITES cells measured by '\n",
      "                   'spectrophotometer after each subculture. (b) Single-cell '\n",
      "                   'GFP fluorescence distributions of BL21\\xad(DE3) and '\n",
      "                   'BL21/HITES cells measured by flow cytometer. The numbers '\n",
      "                   '1, 5, 10, 15, 20, and 22 represent the 1st, 5th, 10th, '\n",
      "                   '15th, 20th, and 22nd subculture, respectively. (c) Ratio '\n",
      "                   'of fluorescent BL21\\xad(DE3) and BL21/HITES cells after '\n",
      "                   'each subculture, which was determined using a flow '\n",
      "                   'cytometer.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'To demonstrate that the HITES was able to work efficiently '\n",
      "                   'in different host cells, we tested it in three different '\n",
      "                   'Gram-negative bacteria, including Pseudomonas putida '\n",
      "                   'KT2440, Tatumella morbirosei LMG 23359, Sinorhizobium '\n",
      "                   'TH572, and one Gram-positive bacterium, Corynebacterium '\n",
      "                   'glutamicum RES167, in addition to E. coli JM109 and BL21. '\n",
      "                   'P. putida and C. glutamicum have been widely used in '\n",
      "                   'laboratory research and industrial production. '\n",
      "                   'Sinorhizobium is an important wild-type strain for the '\n",
      "                   'industrial production of d-p-hydroxyphenylglycine (d-HPG), '\n",
      "                   'while T. morbirosei has great potential for replacing the '\n",
      "                   'conventional two-step fermentation process in the vitamin '\n",
      "                   'C industry with a one-step process. To test the efficacy '\n",
      "                   'of the HITES, we first integrated it with a gfp reporter '\n",
      "                   'gene into a single plasmid that survives in corresponding '\n",
      "                   'host and then transformed these plasmids into '\n",
      "                   'corresponding host cells separately. Confocal fluorescence '\n",
      "                   'microscopy imaging (Figure a) showed that almost all cells '\n",
      "                   'exhibited significant GFP fluorescence and that the RFI of '\n",
      "                   'hosts harboring the HITES was significantly higher than '\n",
      "                   'that of the control (Figure b). These results indicated '\n",
      "                   'that the HITES could be universally used in different '\n",
      "                   'Gram-positive and Gram-negative species.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Universality of the HITES in different Gram-positive and '\n",
      "                   'Gram-negative bacteria. (a) Confocal fluorescence '\n",
      "                   'micrograph analysis showing GFP fluorescence in different '\n",
      "                   'host cells. The merged images of the confocal fluorescence '\n",
      "                   'micrographs and the differential interference contrast '\n",
      "                   '(DIC) micrographs are shown. Scale bars, 2 μm. (b) GFP '\n",
      "                   'fluorescence intensity of different host cells measured by '\n",
      "                   'spectrophotometer. (c) SDS-PAGE analysis of luxAB '\n",
      "                   'expression in P. putida, T. morbirosei, C. glutamicum, E. '\n",
      "                   'coli JM109 and BL21 cells.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'On the other hand, since metabolic engineering often '\n",
      "                   'requires multistep enzymatic catalytic reactions for the '\n",
      "                   'production of desired chemicals, demonstrating that the '\n",
      "                   'HITES can be used to express multiple genes in a '\n",
      "                   'multicistron structure in a single plasmid is important. '\n",
      "                   'To verify this ability, lux AB genes encoding a 41 kDa α '\n",
      "                   'subunit and a 37.68 kDa β subunit were selected as '\n",
      "                   'reporter proteins. SDS-PAGE (Figure c) showed the high '\n",
      "                   'expression levels of lux AB genes in P. putida, T. '\n",
      "                   'morbirosei, C. glutamicum, and E. coli. In summary, the '\n",
      "                   'HITES showed excellent performances (1) in different '\n",
      "                   'Gram-positive and Gram-negative strains; (2) in different '\n",
      "                   'laboratory and industrial strains; and (3) in single-gene '\n",
      "                   'and multigene expression systems. These results underlined '\n",
      "                   'the potential of using the HITES in different species and '\n",
      "                   'multigene expression systems for laboratory research and '\n",
      "                   'industrial production.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'The expression of enzymes involved in a desired pathway '\n",
      "                   'must always be finely tuned for an optimized and balanced '\n",
      "                   'pathway flux. Inclusion bodies consisting of inactive '\n",
      "                   'protein aggregates are easily formed using conventional T7 '\n",
      "                   'expression systems, and multiple strategies had been '\n",
      "                   'adopted to prevent the formation of inclusion bodies by '\n",
      "                   'slowing down the rate of protein synthesis. Here, tuning '\n",
      "                   'the amount of T7 RNAP within a wide range was hypothesized '\n",
      "                   'to precisely regulate the expression of T7 '\n",
      "                   'promoter-cognate genes. The sequence of ribosome binding '\n",
      "                   'sites (RBSs) have been shown to strongly affect gene '\n",
      "                   'expression at the translational level. Therefore, we '\n",
      "                   'designed a series of RBS sequences (Figure b) with '\n",
      "                   'different putative T7 RNAP translation rates using the RBS '\n",
      "                   'Calculator v2.0 model (Table S2). As shown in Figure c and '\n",
      "                   'd, JM109 and BL21 cells modulated by HITESs containing '\n",
      "                   'different RBSs exhibited significant differences in GFP '\n",
      "                   'fluorescence, with RFI values ranging from 0.63 to 200.08% '\n",
      "                   'and from 0.36 to 149.50%, respectively, relative to the '\n",
      "                   'RFI of DE3 lysogenic hosts. This result not only confirmed '\n",
      "                   'the strong effect of the RBS sequence on regulating T7 '\n",
      "                   'RNAP expression but also provided a good strategy for '\n",
      "                   'refining the efficacy of HITES in various hosts to meet '\n",
      "                   'the needs of different cases of metabolic engineering, '\n",
      "                   'which will be further proven in the following section.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Regulation performance of HITESs with different T7 RNAP '\n",
      "                   'translation rates. (a) Schematic diagram of how the RBS of '\n",
      "                   'T7 RNAP gene could affect its expression in HITESs. (b) '\n",
      "                   'Putative translation initiation rate of the designed RBSs '\n",
      "                   'according to the RBS Calculator v2.0 model. (c) GFP '\n",
      "                   'fluorescence intensity of JM109 cells harboring HITESs '\n",
      "                   'with different RBSs measured by spectrophotometer. (d) GFP '\n",
      "                   'fluorescence intensity of BL21 cells harboring HITESs with '\n",
      "                   'different RBSs measured by spectrophotometer.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'The expression of different genes in metabolic networks '\n",
      "                   'should vary on a case-by-case basis. For instance, '\n",
      "                   'inducible expression is more effective when gene '\n",
      "                   'expression is regulated at specific growth stages, whereas '\n",
      "                   'for the production of inexpensive products (where the cost '\n",
      "                   'of inducer is the main concern), constitutive expression '\n",
      "                   'is commonly used for gene expression. Sinorhizobium TH572 '\n",
      "                   'is an industrial bacterium used to produce d-HPG, which is '\n",
      "                   'the precursor of amoxicillin in the biopharmaceutical '\n",
      "                   'industry. A commonly used method to produce d-HPG depends '\n",
      "                   'on an inherent expression system in Sinorhizobium TH572 '\n",
      "                   'that expresses the enzymes d-hydantoinase (d-Hase) and '\n",
      "                   'd-carbamoylase (d-Case) (see Figure a). However, the '\n",
      "                   'requirement of the toxic, unstable and expensive inducer '\n",
      "                   '5-(2-methylthioethyl)\\xadhydantoin made this inherent '\n",
      "                   'expression system less practical in industrial production. '\n",
      "                   'This limit can be avoided if a constitutive HITES where '\n",
      "                   'd-Hase and d-Case are constitutively expressed by T7 RNAP '\n",
      "                   'without inducers is used as a substitute. For this '\n",
      "                   'purpose, the lacI gene, encoding a lactose repressor, was '\n",
      "                   'deleted from HITES to realize constitutive gene '\n",
      "                   'expression. The deletion of lacI caused a sharp decrease '\n",
      "                   'in RFI in serial subcultures (Figure a), with '\n",
      "                   'nonfluorescent cells appearing and eventually becoming the '\n",
      "                   'whole population (Figure b), indicating the instability of '\n",
      "                   'the HITES-Δ lacI construct. Solving this problem requires '\n",
      "                   'reducing the amount of T7 RNAP present.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Construction of a stable, constitutive HITES in '\n",
      "                   'Sinorhizobium TH572. (a) The fluorescence intensity '\n",
      "                   'changes of Sinorhizobium/HITES-ΔlacI measured by '\n",
      "                   'spectrophotometer after each subculture. (b) Single-cell '\n",
      "                   'GFP fluorescence distributions of '\n",
      "                   'Sinorhizobium/HITES-ΔlacI cells measured by flow '\n",
      "                   'cytometer. The numbers 1, 2, 3, 4, 5, and 8 represent the '\n",
      "                   '1st, 2nd, 3rd, 4th, 5th, and 8th subculture, respectively. '\n",
      "                   '(c) Putative translation initiation rate of the designed '\n",
      "                   'RBSs according to the RBS Calculator v2.0 model. (d) The '\n",
      "                   'amount of T7 RNAP in Sinorhizobium harboring constitutive '\n",
      "                   'HITESs with different RBSs was analyzed by Western blot '\n",
      "                   'using monoclonal antibodies against T7 RNAP.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'As described above, the designed RBSs in the former '\n",
      "                   'section tuned the T7 RNAP expression level. Five RBSs with '\n",
      "                   'lower translation rates (RBS2–6) were selected for '\n",
      "                   'constructing a stable constitutive HITES (Figure c). The '\n",
      "                   'results in Figure d indicated that the expression of T7 '\n",
      "                   'RNAP decreased corresponding to the putative translation '\n",
      "                   'rate of the five RBSs, among which RBS2 caused the '\n",
      "                   'strongest reduction. The following subculture experiment '\n",
      "                   'demonstrated that RBSs with lower translation rates slowed '\n",
      "                   'the fluorescent decay (Figure a–e, left panel) and that '\n",
      "                   'the RFI of cells with HITES-RSB2-Δ lacI remained steady '\n",
      "                   'during long periods of serial subculture (Figure a, left '\n",
      "                   'panel). The distribution of fluorescent cells showed the '\n",
      "                   'same trend (Figure a–e, right panel), and no clearly '\n",
      "                   'nonfluorescent cells were observed when the HITES-RSB2-Δ '\n",
      "                   'lacI construct was used (Figure a, right panel). Figure f '\n",
      "                   'further revealed that the ratio of fluorescent cells was '\n",
      "                   'significantly higher when RBSs with lower translation '\n",
      "                   'rates were applied. The constitutive expression from '\n",
      "                   'HITES-RSB2-Δ lacI was stable for at least 20 subculture '\n",
      "                   'cycles, which should satisfy the demands of large-scale '\n",
      "                   'industrial production, assuming that cells require 27 '\n",
      "                   'generations to grow from a 1 mL seed culture to a 100 m 3 '\n",
      "                   'culture with the same cell density. In addition, the '\n",
      "                   'inoculum volume was 0.1% per subculture in the previous '\n",
      "                   'subculture experiment, indicating that the cells grew at a '\n",
      "                   'speed of approximately 10 generations per subculture. '\n",
      "                   'Thus, the fermentation could theoretically be carried out '\n",
      "                   'steadily if the HITES-RSB2-Δ lacI construct is stable for '\n",
      "                   'at least 3 subcultures, which is far less than 20 '\n",
      "                   'subcultures demonstrated above. In conclusion, by using '\n",
      "                   'RBSs with different translation rates, the speed of T7 '\n",
      "                   'RNAP expression was adjusted to a reasonable level in a '\n",
      "                   'given host to achieve stable protein expression.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Stability of constitutive HITESs with different RBSs. '\n",
      "                   '(a–e) Left: The fluorescence intensity changes of '\n",
      "                   'Sinorhizobium cells harboring constitutive HITESs with '\n",
      "                   'RBS2, RBS3, RBS4, RBS5, and RBS6, respectively. Right: '\n",
      "                   'Single-cell GFP fluorescence distributions of '\n",
      "                   'Sinorhizobium harboring constitutive HITESs with RBS2, '\n",
      "                   'RBS3, RBS4, RBS5, and RBS6, respectively. Numbers on the '\n",
      "                   'graph represent the iteration number of the subculture. '\n",
      "                   '(f) Ratio of fluorescent Sinorhizobium cells harboring '\n",
      "                   'constitutive HITES with different RBSs after each '\n",
      "                   'subculture as counted by a flow cytometer.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'On the basis of the results above, the stable, '\n",
      "                   'constitutively expressing HITES-RSB2-Δ lacI construct was '\n",
      "                   'utilized to overexpress d-Hase and d-Case in Sinorhizobium '\n",
      "                   'TH752. A recombinant strain used in industrial production, '\n",
      "                   'Sinorhizobium/Hp-C2H, that harbors the inherent inducible '\n",
      "                   'promoter was selected as a positive control to investigate '\n",
      "                   'the expression capacity of HITES-RSB2-Δ lacI in TH752 '\n",
      "                   'cells. SDS-PAGE analysis (Figure b) showed that the '\n",
      "                   'expression level of d-Hase and d-Case was slightly higher '\n",
      "                   'in cells with this construct than in the positive control '\n",
      "                   '(Sinorhizobium/Hp-C2H). The combined enzyme activity '\n",
      "                   '(which was used to evaluate the amount of the active '\n",
      "                   'enzymes) of Sinorhizobium/Hp-C2H and '\n",
      "                   'Sinorhizobium/HITES-RBS2-Δ lacI-C2H was 1.00 U and 1.11 U, '\n",
      "                   'respectively (Figure c). To conclude, the application of '\n",
      "                   'the stable, constitutive HITES in TH572 not only released '\n",
      "                   'd-HPG production from the addition of inducer but also '\n",
      "                   'exhibited a combined enzyme activity 10.74% greater than '\n",
      "                   'that of the existing industrial strain, demonstrating the '\n",
      "                   'industrial value of HITESs in different wild-type '\n",
      "                   'strains.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Constitutive expression of d-Hase and d-Case in '\n",
      "                   'Sinorhizobium TH572. (a) Schematic diagram of d-HPG '\n",
      "                   'production via the catalysis of two sequentially acting '\n",
      "                   'enzymes, d-Hase and d-Case. (b) SDS-PAGE analysis of '\n",
      "                   'd-Hase and d-Case expression in wild-type Sinorhizobium '\n",
      "                   '(negative control), Sinorhizobium/Hp-C2H (positive '\n",
      "                   'control), and Sinorhizobium/HITES-RBS2-ΔlacI-C2H cells. '\n",
      "                   '(c) Combined enzyme activity of the corresponding strains '\n",
      "                   'measured by HPLC.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'Here, a novel host-independent T7 expression system '\n",
      "                   'integrating the T7 RNAP gene and its cognate T7 promoter '\n",
      "                   'into the same unit with antisense RNA regulation has been '\n",
      "                   'developed. This HITES was integrated into a single plasmid '\n",
      "                   'for effective and stable work in host cells, freeing the '\n",
      "                   'application of the T7 system from relying on DE3 lysogenic '\n",
      "                   'strains. HITES has high transcriptional efficiency, '\n",
      "                   'flexible regulation performance and long-lasting genetic '\n",
      "                   'stability during subculture. The HITES also worked '\n",
      "                   'effectively in one Gram-positive and five different '\n",
      "                   'Gram-negative strains, indicating its universality and '\n",
      "                   'powerful application potential in different prokaryotic '\n",
      "                   'hosts.'},\n",
      "          {'section': ['result', 'discussion'],\n",
      "           'text': 'A recent publication reported the Universal Bacterial '\n",
      "                   'Expression Resource (UBER), which is an autonomous '\n",
      "                   'self-regulated T7 RNAP expression system that functions by '\n",
      "                   'combining mixed feedback control loops and cross-species '\n",
      "                   'translation signals, and reported it functioning in E. '\n",
      "                   'coli, Bacillus subtilis and P. putida. However, there are '\n",
      "                   'many advantages of our work comparing to theirs regarding '\n",
      "                   'the construction strategy and application fields. First, '\n",
      "                   'HITES can be integrated into a single plasmid for stable '\n",
      "                   'expression in different hosts, whereas the UBER usually '\n",
      "                   'requires two biocompatible plasmids to achieve the '\n",
      "                   'efficient expression of target genes, which limits its '\n",
      "                   'application in noncanonical hosts. Second, the normal '\n",
      "                   'function of the UBER relies on the optimization of both '\n",
      "                   'positive and negative feedback loop strengths when applied '\n",
      "                   'in a new host, while the expression capacity in our HITES '\n",
      "                   'system is effectively tuned by simple RBS engineering. '\n",
      "                   'Lastly, unlike the UBER, which is a constitutive '\n",
      "                   'expression system with no operon to control target gene '\n",
      "                   'expression, we have developed ready-to-use, constitutive '\n",
      "                   'HITESs and inducible HITESs for application to different '\n",
      "                   'metabolic engineering situations.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'E. coli JM109, E. coli BL21, P. putida KT2440, T. '\n",
      "                   'morbirosei LMG 23359, and Sinorhizobium TH572 were '\n",
      "                   'cultured in LB broth. C. glutamicum RES167 was cultured in '\n",
      "                   'LBGU medium (10 g of tryptone, 10 g of NaCl, 5 g of yeast '\n",
      "                   'extract, 10 g of glucose and 2 g of urea per liter). Solid '\n",
      "                   'media on plates were prepared by adding 1.5% w/v agar. E. '\n",
      "                   'coli cells were grown at 37 °C, P. putida, Sinorhizobium '\n",
      "                   'and C. glutamicum were grown at 30 °C, and T. morbirosei '\n",
      "                   'cells were grown at 28 °C. Antibiotic selection for E. '\n",
      "                   'coli with the single plasmid pET30-HITES was performed '\n",
      "                   'with 50 μg/mL kanamycin. Antibiotic selection for T. '\n",
      "                   'morbirosei, P. putida and Sinorhizobium with the single '\n",
      "                   'plasmid pBBR1MCS5-KT-HITES or pBBR1MCS5-Sino-HITES was '\n",
      "                   'performed with 30 μg/mL gentamicin. Antibiotic selection '\n",
      "                   'for C. glutamicum with the single plasmid pXMJ19-HITES was '\n",
      "                   'performed with 12.5 μg/mL chloramphenicol.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'The coding sequence of synthetic small RNA was synthesized '\n",
      "                   'by assembly PCR according to primers designed using '\n",
      "                   'DNAWorks (v3.2.4). The various plasmids with CAP deletion, '\n",
      "                   'terminator replacement, and designed RBSs were constructed '\n",
      "                   'using standard cloning techniques, including PCR, '\n",
      "                   'restriction enzyme digestion and ligation. For application '\n",
      "                   'in P. putida and T. morbirosei, a HITES fragment was '\n",
      "                   'amplified from pET30-HITES with primers HITES-KT-F and '\n",
      "                   'HITES-KT-R, the plasmid fragment was amplified from '\n",
      "                   'pBBR1MCS-5 with primers pBBR1MCS5-KT-F and pBBR1MCS5-KT-R, '\n",
      "                   'and the HITES fragment was cloned into the plasmid '\n",
      "                   'fragment by Gibson assembly, resulting in '\n",
      "                   'pBBR1MCS5-KT-HITES. (Figure S5) For application in '\n",
      "                   'Sinorhizobium, a HITES fragment was amplified from '\n",
      "                   'pET30-HITES with primers HITES-Sino-F and HITES-Sino-R, '\n",
      "                   'the plasmid fragment was amplified from pBBR1MCS-5 with '\n",
      "                   'primers pBBR1MCS5-Sino-F and pBBR1MCS5-Sino-R, and the '\n",
      "                   'HITES fragment was cloned into the plasmid fragment by '\n",
      "                   'Gibson assembly, resulting in pBBR1MCS5-Sino-HITES. '\n",
      "                   '(Figure S5) For application in C. glutamicum, a HITES '\n",
      "                   'fragment was amplified from pET30-HITES with primers '\n",
      "                   'HITES-RES-F and HITES-RES-R, the plasmid pXMJ19 was double '\n",
      "                   'digested with Apa I and EcoR I to obtain the plasmid '\n",
      "                   'fragment, and the HITES fragment was cloned into the '\n",
      "                   'plasmid fragment by Gibson assembly, resulting in '\n",
      "                   'pXMJ19-HITES. (Figure S5) For construction of constitutive '\n",
      "                   'HITES, the primers Delete LacI-F and Delete LacI-R were '\n",
      "                   'gradient annealed, and the plasmid pBBR1MCS5-Sino-HITES '\n",
      "                   'was digested with Mlu I/SgrA I. The annealed DNA fragment '\n",
      "                   'and the digested plasmid were ligated with T4 DNA ligase '\n",
      "                   'to generate plasmid pBBR1MCS5-Sino-HITES-Δ lacI. The '\n",
      "                   'recombinant plasmids were sequenced after their '\n",
      "                   'construction. Cloning was conducted in E. coli DH5α cells. '\n",
      "                   'The sequence of pBBR1MSC-5 and pXMJ19 could be referred to '\n",
      "                   'the GenBank accession number U25061.1 and AJ133195.1.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'E. coli cells were induced with 0.2 mM IPTG at 30 °C for '\n",
      "                   '18 h. P. putida and Sinorhizobium cells were induced with '\n",
      "                   '0.2 mM IPTG at 30 °C for 12 and 24 h, respectively. T. '\n",
      "                   'morbirosei cells were induced with 0.2 mM IPTG at 28 °C '\n",
      "                   'for 18 h. C. glutamicum cells were induced with 0.2 mM '\n",
      "                   'IPTG at 30 °C for 18 h. IPTG was added at the beginning of '\n",
      "                   'each culture for all inductions.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'To facilitate comparisons of different hosts, the '\n",
      "                   'fluorescence intensity was normalized as the fluorescence '\n",
      "                   'intensity per OD 600. For all the strains used in this '\n",
      "                   'study, OD 600 and fluorescence measurements were recorded '\n",
      "                   'using an Infinite M200PRO spectrophotometer (TECAN). '\n",
      "                   'Samples were diluted to the proper concentration (OD 600 = '\n",
      "                   '0.3–0.8) before measurement. Samples of 200 μL volumes '\n",
      "                   '(triplicate) were transferred to a 96 well transparent '\n",
      "                   'microtiter plate for OD 600 measurements, and a 96 well '\n",
      "                   'black microtiter plate for GFP fluorescence measurements. '\n",
      "                   'The excitation and emission wavelengths were 485 and 535 '\n",
      "                   'nm, respectively. The fluorescence and OD 600 measurements '\n",
      "                   'of the plate wells were conducted after high orbital '\n",
      "                   'shaking. Single-cell fluorescence distributions were '\n",
      "                   'measured using an S3e Cell Sorter (Bio-Rad). The samples '\n",
      "                   'were diluted to 10 6 cells/mL and filtered before '\n",
      "                   'injection. The number of collected cells was set to 30 '\n",
      "                   '000. In addition, the ratio of fluorescent cells was '\n",
      "                   'counted from the results of a flow cytometer.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'Growth curves were recorded using a Bioscreen C (Oy Growth '\n",
      "                   'Curves Ab Ltd.). Overnight cultures were diluted into LB '\n",
      "                   'medium (final volume 300 μL) to a final OD 600 of 0.05. '\n",
      "                   'The incubation temperature was 30 °C, and the lid '\n",
      "                   'temperature was 31 °C. The plate was then cultured with '\n",
      "                   'high orbital shaking, and its OD 600 was measured every 20 '\n",
      "                   'min.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'The OD 600 values of the cells were measured using an '\n",
      "                   'Infinite M200PRO spectrophotometer. Cells with an OD 600 '\n",
      "                   'of 30 were harvested by centrifugation and resuspended to '\n",
      "                   'a final volume of 300 μL. Then, 75 μL of 5 × SDS-PAGE '\n",
      "                   'loading buffer was added to the samples, which were then '\n",
      "                   'boiled for 10 min to lyse the cells. The samples were '\n",
      "                   'subsequently centrifuged at 13 000 g for 2 min. The '\n",
      "                   'supernatant contents were separated by 12% SDS-PAGE under '\n",
      "                   'reducing conditions and transferred to a polyvinylidene '\n",
      "                   'difluoride (PVDF) membrane (0.45 μm) using 300 mA of '\n",
      "                   'current for 2 h. Membranes were divided into two parts '\n",
      "                   '(the upper part, containing T7 RNAP, and the other part, '\n",
      "                   'containing glyceraldehyde 3-phosphate dehydrogenase '\n",
      "                   '(GAPDH)) and blocked with 5% skim milk powder in TBST (10 '\n",
      "                   'mM Tris-HCl, pH 7.5, 150 mM NaCl, and 0.05% Tween 20) '\n",
      "                   'overnight. Membranes were incubated with the indicated '\n",
      "                   'antibodies at room temperature for 2 h. T7 RNAP was '\n",
      "                   'incubated with T7 RNAP monoclonal antibody (Novagen, '\n",
      "                   'Catalog Number 70566), and GAPDH (selected as reference '\n",
      "                   'protein) was incubated with GAPDH rabbit polyclonal '\n",
      "                   'antibody (Beijing Biodragon Immunotechnologies Co., Ltd., '\n",
      "                   'Catalog Number B1421). The membranes were then washed five '\n",
      "                   'times and incubated with peroxidase-conjugated secondary '\n",
      "                   'antibody (ZsBio Ltd.). Finally, proteins were detected '\n",
      "                   'with High-sig Enhanced Chemiluminescence (ECL) Western '\n",
      "                   'Blotting Substrate (Tanon Science & Technology Co., Ltd., '\n",
      "                   'Catalog Number 180–501).'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'Subculture experiments were carried out according to the '\n",
      "                   'procedure shown in Figure a. For E. coli, the seed was '\n",
      "                   'cultured in LB medium at 37 °C for 12 h, and the culture '\n",
      "                   'was then transferred to LB medium with 0.2 mM IPTG at 30 '\n",
      "                   '°C for 24 h. For Sinorhizobium, the seed was cultured in '\n",
      "                   'LB medium at 30 °C for 24 h, and the culture was then '\n",
      "                   'transferred to LB medium with 0.2 mM IPTG at 30 °C for 24 '\n",
      "                   'h.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'Recombinant or wild-type Sinorhizobium TH572 cells with an '\n",
      "                   'OD 600 of 30 were harvested by centrifugation, and then '\n",
      "                   'washed and resuspended in 1/15 M Na 2 HPO 4 –KH 2 PO 4 '\n",
      "                   'buffer (pH 8.0) to a final volume of 1 mL. Catalytic '\n",
      "                   'reactions were conducted with 9 mL of 0.3% d, l-HPH '\n",
      "                   'solution as the substrate at 33 °C and 150 rpm for 30 min. '\n",
      "                   'Next, 400 μL of 6 M HCl was added to terminate the '\n",
      "                   'reaction. The reaction mixture was then centrifuged and '\n",
      "                   'cooled in ice.'},\n",
      "          {'section': ['method'],\n",
      "           'text': 'The concentrations of d-HPG were measured using '\n",
      "                   'high-performance liquid chromatography (SHIMADZU, '\n",
      "                   'LC-10ATVP). Chromatographic conditions were column: '\n",
      "                   'Inertsil ODS-2 packed column (5 μm, 4.6 × 250 mm, Catalog '\n",
      "                   'Number 5020–01128); detector: UV–visible detector, 210 nm; '\n",
      "                   'flow rate: 1.0 mL/min; mobile phase: '\n",
      "                   'water/acetonitrile/phosphoric acid (96:4:0.01, v/v/v). The '\n",
      "                   'standard curve of d-HPG standards was constructed under '\n",
      "                   'the same chromatographic conditions. In addition, the '\n",
      "                   'concentrations of d-HPG in the samples were calculated '\n",
      "                   'from the standard curve. The combined enzyme activity was '\n",
      "                   'calculated according to the following equation: combined '\n",
      "                   'enzyme activity (U/mL) = c S · D S M D · V 2 V 1 · t c S: '\n",
      "                   'The concentration of d-HPG in the sample, g/mL. M D: The '\n",
      "                   'molar mass of d-HPG, 167.16 g/mol. D S: The dilution '\n",
      "                   'factor of the sample. V 2: The volume of the reaction '\n",
      "                   'mixture, 10 mL in our study. V 1: The volume of the broth '\n",
      "                   'diluted in the reaction mixture, 1 mL in our study. t: The '\n",
      "                   'reaction time, 30 min in our study.'}],\n",
      " 'keywords': ['host-independent expression system',\n",
      "              'stable expression',\n",
      "              'single plasmid',\n",
      "              'T7 RNA polymerase',\n",
      "              'antisense RNA']}\n"
     ]
    }
   ],
   "source": [
    "with open(pkl_paths[101], \"rb\") as f:\n",
    "    pprint(pickle.load(f))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
